{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load metadata\n",
    "metadata = pd.read_csv(\"data/archive/metadata.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>source_x</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>license</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "      <th>Microsoft Academic Paper ID</th>\n",
       "      <th>WHO #Covidence</th>\n",
       "      <th>has_pdf_parse</th>\n",
       "      <th>has_pmc_xml_parse</th>\n",
       "      <th>full_text_file</th>\n",
       "      <th>url</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>xqhn0vbp</td>\n",
       "      <td>1e1286db212100993d03cc22374b624f7caee956</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Airborne rhinovirus detection and effect of ul...</td>\n",
       "      <td>10.1186/1471-2458-3-5</td>\n",
       "      <td>PMC140314</td>\n",
       "      <td>12525263.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>BACKGROUND: Rhinovirus, the most common cause ...</td>\n",
       "      <td>2003-01-13</td>\n",
       "      <td>Myatt, Theodore A; Johnston, Sebastian L; Rudn...</td>\n",
       "      <td>BMC Public Health</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>gi6uaa83</td>\n",
       "      <td>8ae137c8da1607b3a8e4c946c07ca8bda67f88ac</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Discovering human history from stomach bacteria</td>\n",
       "      <td>10.1186/gb-2003-4-5-213</td>\n",
       "      <td>PMC156578</td>\n",
       "      <td>12734001.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Recent analyses of human pathogens have reveal...</td>\n",
       "      <td>2003-04-28</td>\n",
       "      <td>Disotell, Todd R</td>\n",
       "      <td>Genome Biol</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>fy4w7xz8</td>\n",
       "      <td>0104f6ceccf92ae8567a0102f89cbb976969a774</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Association of HLA class I with severe acute r...</td>\n",
       "      <td>10.1186/1471-2350-4-9</td>\n",
       "      <td>PMC212558</td>\n",
       "      <td>12969506.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>BACKGROUND: The human leukocyte antigen (HLA) ...</td>\n",
       "      <td>2003-09-12</td>\n",
       "      <td>Lin, Marie; Tseng, Hsiang-Kuang; Trejaut, Jean...</td>\n",
       "      <td>BMC Med Genet</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0qaoam29</td>\n",
       "      <td>5b68a553a7cbbea13472721cd1ad617d42b40c26</td>\n",
       "      <td>PMC</td>\n",
       "      <td>A double epidemic model for the SARS propagation</td>\n",
       "      <td>10.1186/1471-2334-3-19</td>\n",
       "      <td>PMC222908</td>\n",
       "      <td>12964944.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>BACKGROUND: An epidemic of a Severe Acute Resp...</td>\n",
       "      <td>2003-09-10</td>\n",
       "      <td>Ng, Tuen Wai; Turinici, Gabriel; Danchin, Antoine</td>\n",
       "      <td>BMC Infect Dis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>qj4dh6rg</td>\n",
       "      <td>3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Cloaked similarity between HIV-1 and SARS-CoV ...</td>\n",
       "      <td>10.1186/1471-2180-3-20</td>\n",
       "      <td>PMC222911</td>\n",
       "      <td>14499001.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>BACKGROUND: Severe acute respiratory syndrome ...</td>\n",
       "      <td>2003-09-21</td>\n",
       "      <td>Kliger, Yossef; Levanon, Erez Y</td>\n",
       "      <td>BMC Microbiol</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   cord_uid                                       sha source_x  \\\n",
       "0  xqhn0vbp  1e1286db212100993d03cc22374b624f7caee956      PMC   \n",
       "1  gi6uaa83  8ae137c8da1607b3a8e4c946c07ca8bda67f88ac      PMC   \n",
       "3  fy4w7xz8  0104f6ceccf92ae8567a0102f89cbb976969a774      PMC   \n",
       "4  0qaoam29  5b68a553a7cbbea13472721cd1ad617d42b40c26      PMC   \n",
       "5  qj4dh6rg  3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c      PMC   \n",
       "\n",
       "                                               title                      doi  \\\n",
       "0  Airborne rhinovirus detection and effect of ul...    10.1186/1471-2458-3-5   \n",
       "1    Discovering human history from stomach bacteria  10.1186/gb-2003-4-5-213   \n",
       "3  Association of HLA class I with severe acute r...    10.1186/1471-2350-4-9   \n",
       "4   A double epidemic model for the SARS propagation   10.1186/1471-2334-3-19   \n",
       "5  Cloaked similarity between HIV-1 and SARS-CoV ...   10.1186/1471-2180-3-20   \n",
       "\n",
       "       pmcid   pubmed_id license  \\\n",
       "0  PMC140314  12525263.0   no-cc   \n",
       "1  PMC156578  12734001.0   no-cc   \n",
       "3  PMC212558  12969506.0   no-cc   \n",
       "4  PMC222908  12964944.0   no-cc   \n",
       "5  PMC222911  14499001.0   no-cc   \n",
       "\n",
       "                                            abstract publish_time  \\\n",
       "0  BACKGROUND: Rhinovirus, the most common cause ...   2003-01-13   \n",
       "1  Recent analyses of human pathogens have reveal...   2003-04-28   \n",
       "3  BACKGROUND: The human leukocyte antigen (HLA) ...   2003-09-12   \n",
       "4  BACKGROUND: An epidemic of a Severe Acute Resp...   2003-09-10   \n",
       "5  BACKGROUND: Severe acute respiratory syndrome ...   2003-09-21   \n",
       "\n",
       "                                             authors            journal  \\\n",
       "0  Myatt, Theodore A; Johnston, Sebastian L; Rudn...  BMC Public Health   \n",
       "1                                   Disotell, Todd R        Genome Biol   \n",
       "3  Lin, Marie; Tseng, Hsiang-Kuang; Trejaut, Jean...      BMC Med Genet   \n",
       "4  Ng, Tuen Wai; Turinici, Gabriel; Danchin, Antoine     BMC Infect Dis   \n",
       "5                    Kliger, Yossef; Levanon, Erez Y      BMC Microbiol   \n",
       "\n",
       "   Microsoft Academic Paper ID WHO #Covidence  has_pdf_parse  \\\n",
       "0                          NaN            NaN           True   \n",
       "1                          NaN            NaN           True   \n",
       "3                          NaN            NaN           True   \n",
       "4                          NaN            NaN           True   \n",
       "5                          NaN            NaN           True   \n",
       "\n",
       "   has_pmc_xml_parse  full_text_file  \\\n",
       "0               True  custom_license   \n",
       "1               True  custom_license   \n",
       "3               True  custom_license   \n",
       "4               True  custom_license   \n",
       "5               True  custom_license   \n",
       "\n",
       "                                                 url  \n",
       "0  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "1  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "3  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...  \n",
       "4  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...  \n",
       "5  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2...  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cleaned_data = metadata.dropna(subset=['cord_uid', 'sha', 'title', 'abstract', 'publish_time', 'authors'])\n",
    "# cleaned_data.sample(1000, random_state=73)\n",
    "cleaned_data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/dr/fn2_r29930s6nq_wqwxq_ckm0000gn/T/ipykernel_20848/1874208079.py:48: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  cleaned_data[\"body_text\"] = cleaned_data[\"sha\"].apply(lambda x: extract_body_text_from_json(sha_to_path.get(x)) if pd.notnull(x) and x in sha_to_path else None)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CSV file with title, abstract, and body_text has been created successfully!\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import json\n",
    "import pandas as pd\n",
    "\n",
    "# Path to the metadata file and the full-text directories\n",
    "json_dirs = [\n",
    "    \"data/archive/biorxiv_medrxiv/biorxiv_medrxiv/pdf_json\",\n",
    "    \"data/archive/comm_use_subset/comm_use_subset/pdf_json\",\n",
    "    \"data/archive/custom_license/custom_license/pdf_json\",\n",
    "    \"data/archive/noncomm_use_subset/noncomm_use_subset/pdf_json\"\n",
    "]\n",
    "\n",
    "# Function to extract body text from JSON\n",
    "def extract_body_text_from_json(file_path):\n",
    "    \"\"\"\n",
    "    Extracts body text from a JSON file.\n",
    "\n",
    "    Args:\n",
    "        file_path (str): Path to the JSON file.\n",
    "\n",
    "    Returns:\n",
    "        str: The body text from the JSON file.\n",
    "    \"\"\"\n",
    "    try:\n",
    "        with open(file_path, 'r', encoding='utf-8') as f:\n",
    "            data = json.load(f)\n",
    "\n",
    "        # Extract body text\n",
    "        body_text = \"\"\n",
    "        if \"body_text\" in data:\n",
    "            body_text = \" \".join([entry.get(\"text\", \"\") for entry in data[\"body_text\"]])\n",
    "\n",
    "        return body_text\n",
    "    except Exception as e:\n",
    "        print(f\"Error reading {file_path}: {e}\")\n",
    "        return None\n",
    "\n",
    "# Map SHA values to JSON paths\n",
    "sha_to_path = {}\n",
    "for json_dir in json_dirs:\n",
    "    for root, _, files in os.walk(json_dir):\n",
    "        for file in files:\n",
    "            if file.endswith(\".json\"):\n",
    "                sha = os.path.splitext(file)[0]\n",
    "                sha_to_path[sha] = os.path.join(root, file)\n",
    "\n",
    "# Add body_text to the metadata\n",
    "cleaned_data[\"body_text\"] = cleaned_data[\"sha\"].apply(lambda x: extract_body_text_from_json(sha_to_path.get(x)) if pd.notnull(x) and x in sha_to_path else None)\n",
    "\n",
    "cleaned_data = cleaned_data.drop(columns=[\"source_x\", \"doi\", \"pmcid\", \"pubmed_id\", \"license\", \"journal\", \"Microsoft Academic Paper ID\", \"WHO #Covidence\",  \"has_pdf_parse\", \"has_pmc_xml_parse\", \"full_text_file\", \"url\"])\n",
    "# Save to a new CSV with the required columns\n",
    "cleaned_data.to_csv(\"cleaned_data.csv\", index=False)\n",
    "\n",
    "print(\"CSV file with title, abstract, and body_text has been created successfully!\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>body_text</th>\n",
       "      <th>document</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>xqhn0vbp</td>\n",
       "      <td>1e1286db212100993d03cc22374b624f7caee956</td>\n",
       "      <td>Airborne rhinovirus detection and effect of ul...</td>\n",
       "      <td>BACKGROUND: Rhinovirus, the most common cause ...</td>\n",
       "      <td>2003-01-13</td>\n",
       "      <td>Myatt, Theodore A; Johnston, Sebastian L; Rudn...</td>\n",
       "      <td>Rhinoviruses have been associated with 40% to ...</td>\n",
       "      <td>BACKGROUND: Rhinovirus, the most common cause ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>gi6uaa83</td>\n",
       "      <td>8ae137c8da1607b3a8e4c946c07ca8bda67f88ac</td>\n",
       "      <td>Discovering human history from stomach bacteria</td>\n",
       "      <td>Recent analyses of human pathogens have reveal...</td>\n",
       "      <td>2003-04-28</td>\n",
       "      <td>Disotell, Todd R</td>\n",
       "      <td>Charles Darwin recognized that the distributio...</td>\n",
       "      <td>Recent analyses of human pathogens have reveal...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fy4w7xz8</td>\n",
       "      <td>0104f6ceccf92ae8567a0102f89cbb976969a774</td>\n",
       "      <td>Association of HLA class I with severe acute r...</td>\n",
       "      <td>BACKGROUND: The human leukocyte antigen (HLA) ...</td>\n",
       "      <td>2003-09-12</td>\n",
       "      <td>Lin, Marie; Tseng, Hsiang-Kuang; Trejaut, Jean...</td>\n",
       "      <td>patient group, a further significant increase ...</td>\n",
       "      <td>BACKGROUND: The human leukocyte antigen (HLA) ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0qaoam29</td>\n",
       "      <td>5b68a553a7cbbea13472721cd1ad617d42b40c26</td>\n",
       "      <td>A double epidemic model for the SARS propagation</td>\n",
       "      <td>BACKGROUND: An epidemic of a Severe Acute Resp...</td>\n",
       "      <td>2003-09-10</td>\n",
       "      <td>Ng, Tuen Wai; Turinici, Gabriel; Danchin, Antoine</td>\n",
       "      <td>Since November 2002 (and perhaps earlier) an o...</td>\n",
       "      <td>BACKGROUND: An epidemic of a Severe Acute Resp...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>qj4dh6rg</td>\n",
       "      <td>3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c</td>\n",
       "      <td>Cloaked similarity between HIV-1 and SARS-CoV ...</td>\n",
       "      <td>BACKGROUND: Severe acute respiratory syndrome ...</td>\n",
       "      <td>2003-09-21</td>\n",
       "      <td>Kliger, Yossef; Levanon, Erez Y</td>\n",
       "      <td>Infection by many enveloped viruses requires f...</td>\n",
       "      <td>BACKGROUND: Severe acute respiratory syndrome ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   cord_uid                                       sha  \\\n",
       "0  xqhn0vbp  1e1286db212100993d03cc22374b624f7caee956   \n",
       "1  gi6uaa83  8ae137c8da1607b3a8e4c946c07ca8bda67f88ac   \n",
       "2  fy4w7xz8  0104f6ceccf92ae8567a0102f89cbb976969a774   \n",
       "3  0qaoam29  5b68a553a7cbbea13472721cd1ad617d42b40c26   \n",
       "4  qj4dh6rg  3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c   \n",
       "\n",
       "                                               title  \\\n",
       "0  Airborne rhinovirus detection and effect of ul...   \n",
       "1    Discovering human history from stomach bacteria   \n",
       "2  Association of HLA class I with severe acute r...   \n",
       "3   A double epidemic model for the SARS propagation   \n",
       "4  Cloaked similarity between HIV-1 and SARS-CoV ...   \n",
       "\n",
       "                                            abstract publish_time  \\\n",
       "0  BACKGROUND: Rhinovirus, the most common cause ...   2003-01-13   \n",
       "1  Recent analyses of human pathogens have reveal...   2003-04-28   \n",
       "2  BACKGROUND: The human leukocyte antigen (HLA) ...   2003-09-12   \n",
       "3  BACKGROUND: An epidemic of a Severe Acute Resp...   2003-09-10   \n",
       "4  BACKGROUND: Severe acute respiratory syndrome ...   2003-09-21   \n",
       "\n",
       "                                             authors  \\\n",
       "0  Myatt, Theodore A; Johnston, Sebastian L; Rudn...   \n",
       "1                                   Disotell, Todd R   \n",
       "2  Lin, Marie; Tseng, Hsiang-Kuang; Trejaut, Jean...   \n",
       "3  Ng, Tuen Wai; Turinici, Gabriel; Danchin, Antoine   \n",
       "4                    Kliger, Yossef; Levanon, Erez Y   \n",
       "\n",
       "                                           body_text  \\\n",
       "0  Rhinoviruses have been associated with 40% to ...   \n",
       "1  Charles Darwin recognized that the distributio...   \n",
       "2  patient group, a further significant increase ...   \n",
       "3  Since November 2002 (and perhaps earlier) an o...   \n",
       "4  Infection by many enveloped viruses requires f...   \n",
       "\n",
       "                                            document  \n",
       "0  BACKGROUND: Rhinovirus, the most common cause ...  \n",
       "1  Recent analyses of human pathogens have reveal...  \n",
       "2  BACKGROUND: The human leukocyte antigen (HLA) ...  \n",
       "3  BACKGROUND: An epidemic of a Severe Acute Resp...  \n",
       "4  BACKGROUND: Severe acute respiratory syndrome ...  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "c_df = pd.read_csv(\"cleaned_data.csv\")\n",
    "\n",
    "# Combine 'abstract' and 'body_text' columns into a new column 'document'\n",
    "c_df['document'] = c_df['abstract'] + \" \" + c_df['body_text']\n",
    "\n",
    "# Display the updated DataFrame\n",
    "c_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DataFrame saved as 'cleaned_data_with_document.csv'\n"
     ]
    }
   ],
   "source": [
    "# Save the updated DataFrame to a CSV file\n",
    "c_df.to_csv(\"cleaned_data_with_document.csv\", index=False)\n",
    "\n",
    "print(\"DataFrame saved as 'cleaned_data_with_document.csv'\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## entity and entity type extraction "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "cleaned_data = pd.read_csv(\"cleaned_data_with_document.csv\")\n",
    "documents = cleaned_data[\"document\"].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gliner import GLiNER\n",
    "\n",
    "model = GLiNER.from_pretrained(\"urchade/gliner_medium-v2.1\")\n",
    "\n",
    "labels = [\n",
    "    \"virus\", \"disease\", \"symptom\", \"drug\", \"vaccine\", \n",
    "    \"gene\", \"protein\", \"diagnostic\", \"treatment\", \"transmission\",\n",
    "    \"epidemiology\", \"public health\", \"host\", \"pathway\", \"immune response\",\n",
    "    \"clinical trial\", \"organization\", \"measurement\", \"publication\", \"environment\", \n",
    "    \"country\", \"region\", \"city\", \"person\", \"location\", \"event\", \"method\", \n",
    "]\n",
    "\n",
    "\n",
    "## This just for Demo for the first document\n",
    "# for document in documents[:1]:\n",
    "#     entities = model.predict_entities(document, labels)\n",
    "#     for entity in entities:\n",
    "#         print(entity[\"text\"], \"=>\", entity[\"label\"])\n",
    "\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def merge_entities(entities):\n",
    "    if not entities:\n",
    "        return []\n",
    "    merged = []\n",
    "    current = entities[0]\n",
    "    for next_entity in entities[1:]:\n",
    "        if next_entity['label'] == current['label'] and (next_entity['start'] == current['end'] + 1 or next_entity['start'] == current['end']):\n",
    "            current['text'] = text[current['start']: next_entity['end']].strip()\n",
    "            current['end'] = next_entity['end']\n",
    "        else:\n",
    "            merged.append(current)\n",
    "            current = next_entity\n",
    "    # Append the last entity\n",
    "    merged.append(current)\n",
    "    return merged"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "132"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "\n",
    "text_splitter = RecursiveCharacterTextSplitter(\n",
    "    separators=[\"\\n\\n\", \"\\n\", r\"(?<=[.?!])\\s+\"],                                   \n",
    "                  keep_separator=False, is_separator_regex=True,\n",
    "                  chunk_size=200, chunk_overlap=20)\n",
    "\n",
    "\n",
    "chunks = text_splitter.split_text(documents[0])\n",
    "len(chunks)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "\n",
    "for doc_index, document in enumerate(documents[:5]):\n",
    "        print(f\"Processing document {doc_index + 1}/{len(documents)}...\")\n",
    "        \n",
    "        # Split document into chunks\n",
    "        chunks = text_splitter.split_text(document)\n",
    "\n",
    "        # Initialize storage for entities\n",
    "        chunks_entities = []\n",
    "        entity_list = []\n",
    "        duplicates = set()\n",
    "\n",
    "        # Process each chunk\n",
    "        for text in tqdm(chunks, desc=f\"Processing chunks for document {doc_index}\"):\n",
    "            # Predict entities\n",
    "            entities = model.predict_entities(text, labels, threshold=0.7)\n",
    "            entities = merge_entities(entities)  # Merge overlapping/duplicate entities\n",
    "\n",
    "            # Process entities\n",
    "            chunk_entities = set()\n",
    "            for entity in entities:\n",
    "                chunk_entities.add(entity[\"text\"])\n",
    "                if entity[\"text\"] in duplicates:\n",
    "                    continue\n",
    "                duplicates.add(entity[\"text\"])\n",
    "                entity_list.append({\"text\": entity[\"text\"], \"label\": entity[\"label\"]})\n",
    "\n",
    "            # Store entities for the chunk\n",
    "            chunks_entities.append(list(chunk_entities))\n",
    "\n",
    "        # Save results for the current document\n",
    "        output_file = f\"entity_type_list/document_{doc_index + 1}_entities.json\"\n",
    "        result = {\n",
    "            \"document_index\": doc_index,\n",
    "            \"chunks_entities\": chunks_entities,\n",
    "            \"entity_list\": entity_list\n",
    "        }\n",
    "        with open(output_file, \"w\") as f:\n",
    "            json.dump(result, f, indent=4)\n",
    "        \n",
    "        print(f\"Entities for document {doc_index + 1} saved to {output_file}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results saved to entity_list.csv\n"
     ]
    }
   ],
   "source": [
    "import csv\n",
    "\n",
    "# Assuming entity_list is your final output\n",
    "output_file = \"entity_list.csv\"\n",
    "\n",
    "with open(output_file, mode=\"w\", newline=\"\") as f:\n",
    "    writer = csv.writer(f)\n",
    "    writer.writerow([\"Entity Text\", \"Entity Label\"])  # Header\n",
    "    writer.writerows(entity_list)\n",
    "\n",
    "print(f\"Results saved to {output_file}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Using Rebel For Relation Extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/opt/anaconda3/envs/KG_env/lib/python3.11/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "from tqdm.auto import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "cleaned_data = pd.read_csv(\"cleaned_data_with_document.csv\")\n",
    "documents = cleaned_data[\"document\"].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from huggingface_hub import login\n",
    "\n",
    "# Log in to Hugging Face Hub\n",
    "login(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chunks saved to output_from_using_REBEL/chunks.json\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "import hashlib\n",
    "from transformers import AutoTokenizer\n",
    "\n",
    "data = []  # Initialize data list\n",
    "m = hashlib.md5()  # Hashing object\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"bert-base-uncased\")\n",
    "\n",
    "def bert_len(text):\n",
    "    tokens = tokenizer.encode(text)\n",
    "    return len(tokens)\n",
    "\n",
    "def create_chunk_dataset(content):\n",
    "    m.update(content.encode('utf-8'))\n",
    "    uid = m.hexdigest()[:12]\n",
    "    text_splitter = RecursiveCharacterTextSplitter(\n",
    "        separators=[\"\\n\\n\", \"\\n\", r\"(?<=[.?!])\\s+\"],                                   \n",
    "        keep_separator=False, is_separator_regex=True,\n",
    "        chunk_size=200, chunk_overlap=20\n",
    "    )\n",
    "    chunks = text_splitter.split_text(content)\n",
    "    for i, chunk in enumerate(chunks):\n",
    "        data.append({\n",
    "            'id': f'{uid}-{i}',\n",
    "            'text': chunk\n",
    "        })\n",
    "\n",
    "# Filter out NaN values from documents\n",
    "filtered_documents = [doc for doc in documents[:100] if isinstance(doc, str)]\n",
    "\n",
    "# Process your data\n",
    "for document in filtered_documents:  # Assuming documents is your input data list\n",
    "    create_chunk_dataset(document)\n",
    "\n",
    "# Save chunks to a `.jsonl` file\n",
    "filename = 'output_from_using_REBEL/chunks.json'\n",
    "with open(filename, 'w') as outfile:\n",
    "    for chunk in data:\n",
    "        outfile.write(json.dumps(chunk) + '\\n')\n",
    "\n",
    "print(f\"Chunks saved to {filename}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "14660"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_triplets(text):\n",
    "    triplets = []\n",
    "    relation, subject, relation, object_ = '', '', '', ''\n",
    "    text = text.strip()\n",
    "    current = 'x'\n",
    "    for token in text.replace(\"<s>\", \"\").replace(\"<pad>\", \"\").replace(\"</s>\", \"\").split():\n",
    "        if token == \"<triplet>\":\n",
    "            current = 't'\n",
    "            if relation != '':\n",
    "                triplets.append({'head': subject.strip(), 'type': relation.strip(),'tail': object_.strip()})\n",
    "                relation = ''\n",
    "            subject = ''\n",
    "        elif token == \"<subj>\":\n",
    "            current = 's'\n",
    "            if relation != '':\n",
    "                triplets.append({'head': subject.strip(), 'type': relation.strip(),'tail': object_.strip()})\n",
    "            object_ = ''\n",
    "        elif token == \"<obj>\":\n",
    "            current = 'o'\n",
    "            relation = ''\n",
    "        else:\n",
    "            if current == 't':\n",
    "                subject += ' ' + token\n",
    "            elif current == 's':\n",
    "                object_ += ' ' + token\n",
    "            elif current == 'o':\n",
    "                relation += ' ' + token\n",
    "    if subject != '' and relation != '' and object_ != '':\n",
    "        triplets.append({'head': subject.strip(), 'type': relation.strip(),'tail': object_.strip()})\n",
    "    return triplets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from transformers import AutoModelForSeq2SeqLM, AutoTokenizer\n",
    "\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"Babelscape/rebel-large\")\n",
    "model = AutoModelForSeq2SeqLM.from_pretrained(\"Babelscape/rebel-large\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gen_kwargs = {\n",
    "    \"max_length\": 256,\n",
    "    \"length_penalty\": 0,\n",
    "    \"num_beams\": 3,\n",
    "    \"num_return_sequences\":1,\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "14660"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 7330/7330 [3:00:17<00:00,  1.48s/it]  \n"
     ]
    }
   ],
   "source": [
    "def generate_triples(texts):\n",
    "\n",
    "    model_inputs = tokenizer(\n",
    "        texts, max_length=512, padding=True, truncation=True, return_tensors=\"pt\"\n",
    "    )\n",
    "    generated_tokens = model.generate(\n",
    "        model_inputs[\"input_ids\"].to(model.device),\n",
    "        attention_mask=model_inputs[\"attention_mask\"].to(model.device),\n",
    "        **gen_kwargs\n",
    "    )\n",
    "    decoded_preds = tokenizer.batch_decode(generated_tokens, skip_special_tokens=False)\n",
    "    for idx, sentence in enumerate(decoded_preds):\n",
    "        et = extract_triplets(sentence)\n",
    "        for t in et:\n",
    "            triples.append((t[\"head\"], t[\"type\"], t[\"tail\"], texts))\n",
    "\n",
    "\n",
    "triples = []\n",
    "\n",
    "\n",
    "for i in tqdm(range(0, len(data), 2)):\n",
    "    try:\n",
    "        texts = [data[i][\"text\"], data[i + 1][\"text\"]]\n",
    "    except:\n",
    "        texts = [data[i][\"text\"]]\n",
    "\n",
    "    generate_triples(texts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16088"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(triples)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('Rhinovirus',\n",
       "  'has effect',\n",
       "  'upper respiratory tract infections',\n",
       "  ['BACKGROUND: Rhinovirus, the most common cause of upper respiratory tract infections, has been implicated in asthma exacerbations and possibly asthma deaths.',\n",
       "   'Although the method of transmission of rhinoviruses is disputed, several studies have demonstrated that aerosol transmission is a likely method of transmission among adults.']),\n",
       " ('aerosol transmission',\n",
       "  'facet of',\n",
       "  'rhinovirus',\n",
       "  ['BACKGROUND: Rhinovirus, the most common cause of upper respiratory tract infections, has been implicated in asthma exacerbations and possibly asthma deaths.',\n",
       "   'Although the method of transmission of rhinoviruses is disputed, several studies have demonstrated that aerosol transmission is a likely method of transmission among adults.']),\n",
       " ('detecting infectious agents',\n",
       "  'studies',\n",
       "  'detecting aerosolized rhinovirus',\n",
       "  ['As a first step in studies of possible airborne rhinovirus transmission, we developed methods to detect aerosolized rhinovirus by extending existing technology for detecting infectious agents in nasal specimens.',\n",
       "   'METHODS: We aerosolized rhinovirus in a small aerosol chamber.(?<=[.?!])\\\\s+Experiments were conducted with decreasing concentrations of rhinovirus.']),\n",
       " ('aerosol chamber',\n",
       "  'use',\n",
       "  'aerosol',\n",
       "  ['As a first step in studies of possible airborne rhinovirus transmission, we developed methods to detect aerosolized rhinovirus by extending existing technology for detecting infectious agents in nasal specimens.',\n",
       "   'METHODS: We aerosolized rhinovirus in a small aerosol chamber.(?<=[.?!])\\\\s+Experiments were conducted with decreasing concentrations of rhinovirus.']),\n",
       " ('rhinoviral',\n",
       "  'subclass of',\n",
       "  'aerosols',\n",
       "  ['To determine the effect of UV irradiation on detection of rhinoviral aerosols, we also conducted experiments in which we exposed aerosols to a UV dose of 684 mJ/m(2).',\n",
       "   'Aerosols were collected on Teflon filters and rhinovirus recovered in Qiagen AVL buffer using the Qiagen QIAamp Viral RNA Kit (Qiagen Corp., Valencia, California) followed by semi-nested RT-PCR and detection by gel electrophoresis.']),\n",
       " ('Qiagen Corp',\n",
       "  'headquarters location',\n",
       "  'Valencia, California',\n",
       "  ['To determine the effect of UV irradiation on detection of rhinoviral aerosols, we also conducted experiments in which we exposed aerosols to a UV dose of 684 mJ/m(2).',\n",
       "   'Aerosols were collected on Teflon filters and rhinovirus recovered in Qiagen AVL buffer using the Qiagen QIAamp Viral RNA Kit (Qiagen Corp., Valencia, California) followed by semi-nested RT-PCR and detection by gel electrophoresis.']),\n",
       " ('TCID(50)',\n",
       "  'facet of',\n",
       "  'aerosolized rhinovirus',\n",
       "  ['RESULTS: We obtained positive results from filter samples that had collected at least 1.3 TCID(50 )of aerosolized rhinovirus.',\n",
       "   'Ultraviolet irradiation of airborne virus at doses much greater than those used in upper-room UV germicidal irradiation applications did not inhibit subsequent detection with the RT-PCR assay.']),\n",
       " ('germicidal',\n",
       "  'uses',\n",
       "  'Ultraviolet',\n",
       "  ['RESULTS: We obtained positive results from filter samples that had collected at least 1.3 TCID(50 )of aerosolized rhinovirus.',\n",
       "   'Ultraviolet irradiation of airborne virus at doses much greater than those used in upper-room UV germicidal irradiation applications did not inhibit subsequent detection with the RT-PCR assay.']),\n",
       " ('rhinovirus',\n",
       "  'subclass of',\n",
       "  'airborne viruses',\n",
       "  ['CONCLUSION: The air sampling and extraction methodology developed in this study should be applicable to the detection of rhinovirus and other airborne viruses in the indoor air of offices and schools.',\n",
       "   'This method, however, cannot distinguish UV inactivated virus from infectious viral particles.']),\n",
       " ('infectious viral particles',\n",
       "  'subclass of',\n",
       "  'virus',\n",
       "  ['CONCLUSION: The air sampling and extraction methodology developed in this study should be applicable to the detection of rhinovirus and other airborne viruses in the indoor air of offices and schools.',\n",
       "   'This method, however, cannot distinguish UV inactivated virus from infectious viral particles.']),\n",
       " ('common cold',\n",
       "  'has cause',\n",
       "  'Rhinovirus',\n",
       "  ['Rhinoviruses have been associated with 40% to 65% of \"common colds\" studied by methods of virus culture [1, 2] .',\n",
       "   'With the use of reverse transcriptase -polymerase chain reaction (RT-PCR), rhinovirus has been shown to be associated with 53% of all colds throughout the year and up to 92% of colds during the autumn outbreak period [3] .']),\n",
       " ('autumn outbreak period',\n",
       "  'has cause',\n",
       "  'rhinovirus',\n",
       "  ['Rhinoviruses have been associated with 40% to 65% of \"common colds\" studied by methods of virus culture [1, 2] .',\n",
       "   'With the use of reverse transcriptase -polymerase chain reaction (RT-PCR), rhinovirus has been shown to be associated with 53% of all colds throughout the year and up to 92% of colds during the autumn outbreak period [3] .']),\n",
       " ('asthma',\n",
       "  'has cause',\n",
       "  'Rhinvirus',\n",
       "  ['Rhinviruses have been implicated in the exacerbation of asthma symptoms [4] and may even play a role in asthma related deaths [5] .',\n",
       "   'Due to their role in the exacerbation of asthma symptoms, rhinoviruses are a major cause of pediatric hospitalizations [6] .']),\n",
       " ('asthma',\n",
       "  'has cause',\n",
       "  'rhinovirus',\n",
       "  ['Rhinviruses have been implicated in the exacerbation of asthma symptoms [4] and may even play a role in asthma related deaths [5] .',\n",
       "   'Due to their role in the exacerbation of asthma symptoms, rhinoviruses are a major cause of pediatric hospitalizations [6] .']),\n",
       " ('rhinovirus',\n",
       "  'has effect',\n",
       "  'asthma',\n",
       "  ['Rhinviruses have been implicated in the exacerbation of asthma symptoms [4] and may even play a role in asthma related deaths [5] .',\n",
       "   'Due to their role in the exacerbation of asthma symptoms, rhinoviruses are a major cause of pediatric hospitalizations [6] .']),\n",
       " ('droplets',\n",
       "  'has effect',\n",
       "  'contact',\n",
       "  [\"Although rhinoviruses have been thought to be transmitted via large droplets or contact [7] , studies in the 1980's demonstrated that fomites are not an effective means of transmitting rhinovirus infections [8] [9] [10] .\",\n",
       "   'While direct inoculation of respiratory secretions has been shown to transmit infection, this is not a likely route of transmission in older children and adults as survival on hands is brief [9] .']),\n",
       " ('transmission',\n",
       "  'has cause',\n",
       "  'infection',\n",
       "  [\"Although rhinoviruses have been thought to be transmitted via large droplets or contact [7] , studies in the 1980's demonstrated that fomites are not an effective means of transmitting rhinovirus infections [8] [9] [10] .\",\n",
       "   'While direct inoculation of respiratory secretions has been shown to transmit infection, this is not a likely route of transmission in older children and adults as survival on hands is brief [9] .']),\n",
       " ('one experimental study',\n",
       "  'main subject',\n",
       "  'Airborne transmission of infection',\n",
       "  ['Airborne transmission of infection was clearly demonstrated in one experimental study on adults [8] .',\n",
       "   'However, in the absence of epidemiological studies designed to conclusively identify modes of transmission, the role of airborne transmission in viral respiratory infection remains controversial.']),\n",
       " ('airborne transmission',\n",
       "  'instance of',\n",
       "  'modes of transmission',\n",
       "  ['Airborne transmission of infection was clearly demonstrated in one experimental study on adults [8] .',\n",
       "   'However, in the absence of epidemiological studies designed to conclusively identify modes of transmission, the role of airborne transmission in viral respiratory infection remains controversial.']),\n",
       " ('detection',\n",
       "  'part of',\n",
       "  'epidemiological',\n",
       "  ['An impediment to future studies of airborne transmission of respiratory infections is the lack of established methods for the detection of airborne respiratory microorganisms appropriate for epidemiological studies.',\n",
       "   'Several laboratory studies have been conducted in which aerosolized respiratory viruses have been collected and quantified with culture based methods [11] [12] [13] .']),\n",
       " ('quantified',\n",
       "  'subclass of',\n",
       "  'laboratory studies',\n",
       "  ['An impediment to future studies of airborne transmission of respiratory infections is the lack of established methods for the detection of airborne respiratory microorganisms appropriate for epidemiological studies.',\n",
       "   'Several laboratory studies have been conducted in which aerosolized respiratory viruses have been collected and quantified with culture based methods [11] [12] [13] .']),\n",
       " ('infectivity',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['These studies were conducted to determine the effects of temperature and relative humidity on the infectivity of respiratory viruses, but their methods are not suitable for the extremely low concentrations of virus in field experiments.',\n",
       "   'PCR based assays have been employed to detect airborne respiratory viruses in a number of small studies conducted in hospital rooms of infected patients [14] [15] [16] [17] and in pig houses [18] .']),\n",
       " ('respiratory virus',\n",
       "  'subclass of',\n",
       "  'airborne',\n",
       "  ['These studies were conducted to determine the effects of temperature and relative humidity on the infectivity of respiratory viruses, but their methods are not suitable for the extremely low concentrations of virus in field experiments.',\n",
       "   'PCR based assays have been employed to detect airborne respiratory viruses in a number of small studies conducted in hospital rooms of infected patients [14] [15] [16] [17] and in pig houses [18] .']),\n",
       " ('ultraviolet germicidal irradation',\n",
       "  'instance of',\n",
       "  'inactivated',\n",
       "  [\"To determine if airborne microorganisms can be inactivated, and therefore, prevent infection transmission, studies during the 1940's-1970's used ultraviolet germicidal irradation (UVGI) to inactivated airborne microorganisms under experimental conditions [19] [20] [21] [22] and reduced airborne infections in a number of settings, such as hospitals [23] , military housing [24] , and classrooms [25] .\",\n",
       "   'However, UVGI was not effective against the highly contagious measles virus when children had extensive contact in buses and crowded tenements [26, 27] .']),\n",
       " ('measles virus',\n",
       "  'has effect',\n",
       "  'highly contagious',\n",
       "  [\"To determine if airborne microorganisms can be inactivated, and therefore, prevent infection transmission, studies during the 1940's-1970's used ultraviolet germicidal irradation (UVGI) to inactivated airborne microorganisms under experimental conditions [19] [20] [21] [22] and reduced airborne infections in a number of settings, such as hospitals [23] , military housing [24] , and classrooms [25] .\",\n",
       "   'However, UVGI was not effective against the highly contagious measles virus when children had extensive contact in buses and crowded tenements [26, 27] .']),\n",
       " ('molecular',\n",
       "  'subclass of',\n",
       "  'methods',\n",
       "  ['Thus, it fell into disfavor and has not been extensively employed nor has it been studied using molecular methods of detecting its efficacy.',\n",
       "   'Because we anticipate future studies of UVGI will use PCR based detection methods, it is important to determine how PCR assays react to UVGI damaged genetic material.']),\n",
       " ('PCR',\n",
       "  'use',\n",
       "  'detection',\n",
       "  ['Thus, it fell into disfavor and has not been extensively employed nor has it been studied using molecular methods of detecting its efficacy.',\n",
       "   'Because we anticipate future studies of UVGI will use PCR based detection methods, it is important to determine how PCR assays react to UVGI damaged genetic material.']),\n",
       " ('dimers',\n",
       "  'subclass of',\n",
       "  'DNA',\n",
       "  ['In DNA, UVGI predominately produces damage in the form of cyclobutylpyrimidine dimers and (6-4) photoproducts.',\n",
       "   'These photoproducts disrupt the replication of the genetic material and, therefore, inhibit the replication.(?<=[.?!])\\\\s+There is limited information regarding the effect of UVGI on RNA.']),\n",
       " ('RNA',\n",
       "  'part of',\n",
       "  'genetic material',\n",
       "  ['In DNA, UVGI predominately produces damage in the form of cyclobutylpyrimidine dimers and (6-4) photoproducts.',\n",
       "   'These photoproducts disrupt the replication of the genetic material and, therefore, inhibit the replication.(?<=[.?!])\\\\s+There is limited information regarding the effect of UVGI on RNA.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['There are conflicting studies on viruses with RNA genomes that have attempted to determine if PCR assays are inhibited by UVGI.',\n",
       "   'Two studies involving hepatitis A virus (single-stranded positive sense RNA genome) and rotavirus (double-stranded RNA genome), showed their PCR assays to be inhibited by UV inactivation of the viruses [28, 29] .']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['There are conflicting studies on viruses with RNA genomes that have attempted to determine if PCR assays are inhibited by UVGI.',\n",
       "   'Two studies involving hepatitis A virus (single-stranded positive sense RNA genome) and rotavirus (double-stranded RNA genome), showed their PCR assays to be inhibited by UV inactivation of the viruses [28, 29] .']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Conversely, Ma et al showed no effect of UV on a PCR assay when they irradiated poliovirus (singlestranded positive sense RNA genome) [30] .',\n",
       "   'In this study, we present an extremely sensitive PCR based assay to detect airborne rhinovirus and demonstrate the effect of ultraviolet irradiation on the assay.']),\n",
       " ('PCR',\n",
       "  'subclass of',\n",
       "  'assay',\n",
       "  ['Conversely, Ma et al showed no effect of UV on a PCR assay when they irradiated poliovirus (singlestranded positive sense RNA genome) [30] .',\n",
       "   'In this study, we present an extremely sensitive PCR based assay to detect airborne rhinovirus and demonstrate the effect of ultraviolet irradiation on the assay.']),\n",
       " ('Manassas',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Virginia',\n",
       "  ['We obtained rhinovirus 16 strain 11757 from ATCC (Manassas, Virginia) with a titer of 10 7.25 particles per 0.2 ml.',\n",
       "   'The suspension was diluted with phosphate buffered saline (PBS) to create four different concentrations to be used in the experiment.']),\n",
       " ('Virginia',\n",
       "  'contains administrative territorial entity',\n",
       "  'Manassas',\n",
       "  ['We obtained rhinovirus 16 strain 11757 from ATCC (Manassas, Virginia) with a titer of 10 7.25 particles per 0.2 ml.',\n",
       "   'The suspension was diluted with phosphate buffered saline (PBS) to create four different concentrations to be used in the experiment.']),\n",
       " ('PBS',\n",
       "  'subclass of',\n",
       "  'phosphate buffered saline',\n",
       "  ['We obtained rhinovirus 16 strain 11757 from ATCC (Manassas, Virginia) with a titer of 10 7.25 particles per 0.2 ml.',\n",
       "   'The suspension was diluted with phosphate buffered saline (PBS) to create four different concentrations to be used in the experiment.']),\n",
       " ('BGI',\n",
       "  'headquarters location',\n",
       "  'Waltham, MA',\n",
       "  ['We generated a rhinoviral aerosol with a 6-jet Collison nebulizer (model CN-38, BGI, Waltham, MA), which generates 2-µm mass-median diameter (MMD) droplets when run at 20 psig of air pressure [31] .',\n",
       "   'The nebulizing solution consists of PBS, diluted virus, and 0.05 g/l uranine.']),\n",
       " ('PBS',\n",
       "  'has part',\n",
       "  'uranine',\n",
       "  ['We generated a rhinoviral aerosol with a 6-jet Collison nebulizer (model CN-38, BGI, Waltham, MA), which generates 2-µm mass-median diameter (MMD) droplets when run at 20 psig of air pressure [31] .',\n",
       "   'The nebulizing solution consists of PBS, diluted virus, and 0.05 g/l uranine.']),\n",
       " ('fluorescent',\n",
       "  'subclass of',\n",
       "  'tracer',\n",
       "  ['We used the uranine as a fluorescent tracer, so that some filters could be used to determine the percentage of actual collection versus theoretical collection.',\n",
       "   'The aerosol was generated in a specially designed aerosolization chamber that had been used in studies of UV sensitivity of Serratia marcescens and bacilli Calmette-Guérin ( Figure 1 ) [32] .']),\n",
       " ('bacilli Calmette-Guérin',\n",
       "  'parent taxon',\n",
       "  'Serratia marcescens',\n",
       "  ['We used the uranine as a fluorescent tracer, so that some filters could be used to determine the percentage of actual collection versus theoretical collection.',\n",
       "   'The aerosol was generated in a specially designed aerosolization chamber that had been used in studies of UV sensitivity of Serratia marcescens and bacilli Calmette-Guérin ( Figure 1 ) [32] .']),\n",
       " ('drying jar',\n",
       "  'product or material produced',\n",
       "  'aerosol',\n",
       "  ['The chamber consists of four parts: a drying jar, a 165 mm square duct where the aerosol first enters the chamber, a UV exposure chamber, and sampling ports.',\n",
       "   'We attached a Collison nebulizer to the drying jar, where the aerosol is given time to dry, before entering the square duct through a 9-port manifold, which creates a uniformly distributed aerosol.']),\n",
       " ('Collison',\n",
       "  'subclass of',\n",
       "  'nebulizer',\n",
       "  ['The chamber consists of four parts: a drying jar, a 165 mm square duct where the aerosol first enters the chamber, a UV exposure chamber, and sampling ports.',\n",
       "   'We attached a Collison nebulizer to the drying jar, where the aerosol is given time to dry, before entering the square duct through a 9-port manifold, which creates a uniformly distributed aerosol.']),\n",
       " ('UV exposure chamber',\n",
       "  'subclass of',\n",
       "  'aerosol',\n",
       "  ['The aerosol next enters the UV exposure chamber.',\n",
       "   'The UV exposure chamber is comprised of a fused quartz UV exposure window (279 × 254 mm), on which a UV fixture containing six 36-watt UV lamps (Lumalier, Memphis, Tennessee) sits.']),\n",
       " ('Lumalier',\n",
       "  'headquarters location',\n",
       "  'Memphis, Tennessee',\n",
       "  ['The aerosol next enters the UV exposure chamber.',\n",
       "   'The UV exposure chamber is comprised of a fused quartz UV exposure window (279 × 254 mm), on which a UV fixture containing six 36-watt UV lamps (Lumalier, Memphis, Tennessee) sits.']),\n",
       " ('UV fixture',\n",
       "  'facet of',\n",
       "  'irradiance',\n",
       "  ['We inserted two types of screens beneath the UV fixture to adjust the level of irradiance.(?<=[.?!])\\\\s+Downstream of the exposure chamber a slide contains three ports.',\n",
       "   'One port is used to collect the aerosol sample with our sampling filters (Teflo 2.0 µm pore, Pall Gelman, Ann Arbor, Michigan) mounted in filter cassettes (Millipore Corp., Bedford, Massachusetts) while the other ports can be loaded with two Teflo filters in series, to allow a bypass while changing sampling filters.']),\n",
       " ('Pall Gelman',\n",
       "  'headquarters location',\n",
       "  'Ann Arbor, Michigan',\n",
       "  ['We inserted two types of screens beneath the UV fixture to adjust the level of irradiance.(?<=[.?!])\\\\s+Downstream of the exposure chamber a slide contains three ports.',\n",
       "   'One port is used to collect the aerosol sample with our sampling filters (Teflo 2.0 µm pore, Pall Gelman, Ann Arbor, Michigan) mounted in filter cassettes (Millipore Corp., Bedford, Massachusetts) while the other ports can be loaded with two Teflo filters in series, to allow a bypass while changing sampling filters.']),\n",
       " ('filters',\n",
       "  'part of',\n",
       "  'aerosol',\n",
       "  ['Two filters were used in the bypass to ensure that no aerosol escaped the chamber.(?<=[.?!])\\\\s+The multiple ports allow changing of filters without interrupting the aerosol flow.',\n",
       "   'Makeup air passes through a Teflon filter prior to entering the chamber to prevent environmental microorganisms from entering the chamber.']),\n",
       " ('Teflon filter',\n",
       "  'use',\n",
       "  'Makeup air',\n",
       "  ['Two filters were used in the bypass to ensure that no aerosol escaped the chamber.(?<=[.?!])\\\\s+The multiple ports allow changing of filters without interrupting the aerosol flow.',\n",
       "   'Makeup air passes through a Teflon filter prior to entering the chamber to prevent environmental microorganisms from entering the chamber.']),\n",
       " ('Rotameter',\n",
       "  'uses',\n",
       "  'Filtered air',\n",
       "  ['Filtered air was pumped through the chamber at a rate of 8.5 l/min monitored by a rotameter.',\n",
       "   'To provide additional protection, the experimental chamber was housed within a Class II, Type A biosafety cabinet.']),\n",
       " ('Type A biosafety cabinet',\n",
       "  'subclass of',\n",
       "  'Class II',\n",
       "  ['Filtered air was pumped through the chamber at a rate of 8.5 l/min monitored by a rotameter.',\n",
       "   'To provide additional protection, the experimental chamber was housed within a Class II, Type A biosafety cabinet.']),\n",
       " ('Onset Computer Corp',\n",
       "  'headquarters location',\n",
       "  'Bourne, Massachusetts',\n",
       "  ['Although temperature and %RH were not controlled, they were measured with continuous monitors (HOBO H08-004-02 four channel data loggers, Onset Computer Corp., Bourne, Massachusetts) downstream of the sampling filters.',\n",
       "   'Airflow through the chamber was maintained at 8.5 l/min for 10 minutes.']),\n",
       " ('Airflow',\n",
       "  'instance of',\n",
       "  'l/min',\n",
       "  ['Although temperature and %RH were not controlled, they were measured with continuous monitors (HOBO H08-004-02 four channel data loggers, Onset Computer Corp., Bourne, Massachusetts) downstream of the sampling filters.',\n",
       "   'Airflow through the chamber was maintained at 8.5 l/min for 10 minutes.']),\n",
       " ('bypass filter',\n",
       "  'use',\n",
       "  'aerosolization',\n",
       "  ['We then began the aerosolization of the rhinoviral solution and allowed the aerosol to flow through the bypass filter for 20 minutes.(?<=[.?!])\\\\s+Air samples were then collected for 10 minutes.',\n",
       "   'For sampling runs that included testing for effects of UV exposure, the UV radiation was turned on and allowed to warm up for 10 minutes while airflow was directed through the bypass filter.']),\n",
       " ('UV',\n",
       "  'has part',\n",
       "  'UV radiation',\n",
       "  ['We then began the aerosolization of the rhinoviral solution and allowed the aerosol to flow through the bypass filter for 20 minutes.(?<=[.?!])\\\\s+Air samples were then collected for 10 minutes.',\n",
       "   'For sampling runs that included testing for effects of UV exposure, the UV radiation was turned on and allowed to warm up for 10 minutes while airflow was directed through the bypass filter.']),\n",
       " ('particle samples',\n",
       "  'subclass of',\n",
       "  'air sample',\n",
       "  ['An air sample was then collected for 10 minutes.(?<=[.?!])\\\\s+Three replicates of UV exposed and nonexposed particle samples were collected during each sampling run.',\n",
       "   'Aerosols were exposed to an irradiance level of 30 mW/m 2 for 22.8 s for a UV dose of 684 mJ/m 2 .']),\n",
       " ('UV',\n",
       "  'has effect',\n",
       "  'irradiance',\n",
       "  ['An air sample was then collected for 10 minutes.(?<=[.?!])\\\\s+Three replicates of UV exposed and nonexposed particle samples were collected during each sampling run.',\n",
       "   'Aerosols were exposed to an irradiance level of 30 mW/m 2 for 22.8 s for a UV dose of 684 mJ/m 2 .']),\n",
       " ('semi-nested',\n",
       "  'subclass of',\n",
       "  'RT-PCR',\n",
       "  ['Prior to the aerosol experiments, we spiked filters with progressively more dilute viral suspensions to determine the limit of detection of our extraction and semi-nested RT-PCR assay.',\n",
       "   'For spiking, 1 µl methanol was placed on the filter as a wetting agent followed by 1 µl of diluted virus.(?<=[.?!])\\\\s+Filters were and allowed to dry prior to extraction.']),\n",
       " ('methanol',\n",
       "  'subject has role',\n",
       "  'wetting agent',\n",
       "  ['Prior to the aerosol experiments, we spiked filters with progressively more dilute viral suspensions to determine the limit of detection of our extraction and semi-nested RT-PCR assay.',\n",
       "   'For spiking, 1 µl methanol was placed on the filter as a wetting agent followed by 1 µl of diluted virus.(?<=[.?!])\\\\s+Filters were and allowed to dry prior to extraction.']),\n",
       " ('spiked',\n",
       "  'subclass of',\n",
       "  'control',\n",
       "  ['We also spiked one filter prior to each aerosol chamber run as a positive control.',\n",
       "   'To test the stability of virus on the filters, a test was conducted in which spiked filters were used to collect air samples for one week in our laboratory, which has single pass air and is therefore an unlikely source of rhinovirus.']),\n",
       " ('single pass air',\n",
       "  'subclass of',\n",
       "  'air samples',\n",
       "  ['We also spiked one filter prior to each aerosol chamber run as a positive control.',\n",
       "   'To test the stability of virus on the filters, a test was conducted in which spiked filters were used to collect air samples for one week in our laboratory, which has single pass air and is therefore an unlikely source of rhinovirus.']),\n",
       " ('petri dish',\n",
       "  'length',\n",
       "  '60',\n",
       "  ['We placed the filters face down in a 60 mm petri dish using sterile forceps.',\n",
       "   'We then added 560 µl of prepared AVL buffer containing carrier RNA (Qiagen QIAamp Viral RNA Mini Kit) and 140 µl phosphate buffered saline (PBS).']),\n",
       " ('Qiagen QIAamp Viral RNA Mini Kit',\n",
       "  'subclass of',\n",
       "  'carrier RNA',\n",
       "  ['We placed the filters face down in a 60 mm petri dish using sterile forceps.',\n",
       "   'We then added 560 µl of prepared AVL buffer containing carrier RNA (Qiagen QIAamp Viral RNA Mini Kit) and 140 µl phosphate buffered saline (PBS).']),\n",
       " ('VWR Scientific Products',\n",
       "  'headquarters location',\n",
       "  'Thorofare, New Jersey',\n",
       "  ['After placing the lids on the dishes, the dishes were secured to an orbital tabletop shaker (VWR Scientific Products, Thorofare, New Jersey) and rotated for 20 minutes at 360 RPM.',\n",
       "   'After shaking, we pipeted the eluted material from the petri dish into a sterile microfuge tube.']),\n",
       " ('microfuge',\n",
       "  'subclass of',\n",
       "  'pipeted',\n",
       "  ['After placing the lids on the dishes, the dishes were secured to an orbital tabletop shaker (VWR Scientific Products, Thorofare, New Jersey) and rotated for 20 minutes at 360 RPM.',\n",
       "   'After shaking, we pipeted the eluted material from the petri dish into a sterile microfuge tube.']),\n",
       " ('QIAamp Viral RNA Mini Kit',\n",
       "  'manufacturer',\n",
       "  'Qiagen',\n",
       "  [\"We extracted the RNA from the filter samples with the Qiagen QIAamp Viral RNA Mini Kit following manufacturer's protocol.\",\n",
       "   'The use of the Qiagen QIAamp Viral RNA Mini Kit resulted in 80 µl of purified viral RNA.']),\n",
       " ('QIAamp Viral RNA Mini Kit',\n",
       "  'manufacturer',\n",
       "  'Qiagen',\n",
       "  [\"We extracted the RNA from the filter samples with the Qiagen QIAamp Viral RNA Mini Kit following manufacturer's protocol.\",\n",
       "   'The use of the Qiagen QIAamp Viral RNA Mini Kit resulted in 80 µl of purified viral RNA.']),\n",
       " ('Invitrogen SuperScript',\n",
       "  'use',\n",
       "  'RT-PCR',\n",
       "  [\"Following extraction, we first amplified the picornavirus RNA using primers OL26 (5'-GCA CTT CTG TTT CCC C-3') [33] and OL27 (5'-CGG ACA CCC AAA GTA G-3') [33] using the Invitrogen SuperScript One-Step RT-PCR with PLATINUM Taq  Amplified products were detected by electrophoresis analysis on 2% agarose gel containing 8 µl of ethidium bromide per 200 ml of gel followed by examination under UV light.\",\n",
       "   \"The OL26 and OL27 primer pair generates a 388 base pair amplicon from a region of the 5' noncoding region of the picornavirus genome.\"]),\n",
       " (\"5' noncoding region\",\n",
       "  'part of',\n",
       "  'genome',\n",
       "  [\"Following extraction, we first amplified the picornavirus RNA using primers OL26 (5'-GCA CTT CTG TTT CCC C-3') [33] and OL27 (5'-CGG ACA CCC AAA GTA G-3') [33] using the Invitrogen SuperScript One-Step RT-PCR with PLATINUM Taq  Amplified products were detected by electrophoresis analysis on 2% agarose gel containing 8 µl of ethidium bromide per 200 ml of gel followed by examination under UV light.\",\n",
       "   \"The OL26 and OL27 primer pair generates a 388 base pair amplicon from a region of the 5' noncoding region of the picornavirus genome.\"]),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  \"5' noncoding region\",\n",
       "  [\"Following extraction, we first amplified the picornavirus RNA using primers OL26 (5'-GCA CTT CTG TTT CCC C-3') [33] and OL27 (5'-CGG ACA CCC AAA GTA G-3') [33] using the Invitrogen SuperScript One-Step RT-PCR with PLATINUM Taq  Amplified products were detected by electrophoresis analysis on 2% agarose gel containing 8 µl of ethidium bromide per 200 ml of gel followed by examination under UV light.\",\n",
       "   \"The OL26 and OL27 primer pair generates a 388 base pair amplicon from a region of the 5' noncoding region of the picornavirus genome.\"]),\n",
       " ('OL26',\n",
       "  'followed by',\n",
       "  'JWA-1b',\n",
       "  ['The semi-nested primer pair of OL26 and JWA-1b generates an amplicon of ~292 base pairs from within the original amplicon.',\n",
       "   'The 292 bp amplicon does not differentiate between rhinovirus and enterovirus [34] .']),\n",
       " ('rhinovirus',\n",
       "  'different from',\n",
       "  'enterovirus',\n",
       "  ['The semi-nested primer pair of OL26 and JWA-1b generates an amplicon of ~292 base pairs from within the original amplicon.',\n",
       "   'The 292 bp amplicon does not differentiate between rhinovirus and enterovirus [34] .']),\n",
       " ('enterovirus',\n",
       "  'different from',\n",
       "  'rhinovirus',\n",
       "  ['The semi-nested primer pair of OL26 and JWA-1b generates an amplicon of ~292 base pairs from within the original amplicon.',\n",
       "   'The 292 bp amplicon does not differentiate between rhinovirus and enterovirus [34] .']),\n",
       " ('Uranine',\n",
       "  'subject has role',\n",
       "  'fluorescent tracer',\n",
       "  ['Uranine was used as a fluorescent tracer, which allowed us to determine a percent yield for each sampling run.',\n",
       "   'To determine the amount of aerosol collected on each filter, uranine analysis was conducted on three filters per sampling run, one each from the beginning, middle and end of each run.']),\n",
       " ('uranine analysis',\n",
       "  'facet of',\n",
       "  'aerosol',\n",
       "  ['Uranine was used as a fluorescent tracer, which allowed us to determine a percent yield for each sampling run.',\n",
       "   'To determine the amount of aerosol collected on each filter, uranine analysis was conducted on three filters per sampling run, one each from the beginning, middle and end of each run.']),\n",
       " ('100 mm test tube',\n",
       "  'use',\n",
       "  'uranine analysis',\n",
       "  ['The filters used for uranine analysis were removed from cassette holders and placed into a 100 mm test tube with sterile forceps.',\n",
       "   'We added 5 ml of PBS to each tube and sonicated for an hour with vortexing every 15 minutes.(?<=[.?!])\\\\s+We also collected samples of the nebulizing fluid before and after nebulization.']),\n",
       " ('nebulization',\n",
       "  'has effect',\n",
       "  'nebulizing fluid',\n",
       "  ['The filters used for uranine analysis were removed from cassette holders and placed into a 100 mm test tube with sterile forceps.',\n",
       "   'We added 5 ml of PBS to each tube and sonicated for an hour with vortexing every 15 minutes.(?<=[.?!])\\\\s+We also collected samples of the nebulizing fluid before and after nebulization.']),\n",
       " ('NaOH',\n",
       "  'has part',\n",
       "  'N',\n",
       "  ['Filter samples and nebulizing fluid samples were diluted with 0.01 N NaOH and analyzed with a spectrofluorometer (excitation 485 nm and emission 535 nm) (Tecan Corp.',\n",
       "   'Spectrafluor Plus, Mannedorf, Switzerland).(?<=[.?!])\\\\s+We averaged the results of the three filters to calculate an estimated percent yield for the sampling run.']),\n",
       " ('Mannedorf',\n",
       "  'country',\n",
       "  'Switzerland',\n",
       "  ['Filter samples and nebulizing fluid samples were diluted with 0.01 N NaOH and analyzed with a spectrofluorometer (excitation 485 nm and emission 535 nm) (Tecan Corp.',\n",
       "   'Spectrafluor Plus, Mannedorf, Switzerland).(?<=[.?!])\\\\s+We averaged the results of the three filters to calculate an estimated percent yield for the sampling run.']),\n",
       " ('fluorescence',\n",
       "  'subclass of',\n",
       "  'fluorescence',\n",
       "  ['The amount of uranine nebulized during each sampling run was determined by measuring the amount of fluid consumed and by measuring the fluorescence of nebulizing fluid before and after each sampling run.',\n",
       "   'Using the total aerosolization time during the sampling run, we determined the amount of uranine that theoretically would be on each filter if we collected 100% of the aerosol generated during the 10-minute exposure of each filter.']),\n",
       " ('aerosol',\n",
       "  'has part',\n",
       "  'uranine',\n",
       "  ['The amount of uranine nebulized during each sampling run was determined by measuring the amount of fluid consumed and by measuring the fluorescence of nebulizing fluid before and after each sampling run.',\n",
       "   'Using the total aerosolization time during the sampling run, we determined the amount of uranine that theoretically would be on each filter if we collected 100% of the aerosol generated during the 10-minute exposure of each filter.']),\n",
       " ('filter samples',\n",
       "  'has part',\n",
       "  'uranine',\n",
       "  ['Using the filter samples that were analyzed for uranine, we determined the actual amount of uranine collected per sample.',\n",
       "   'We averaged the three filter samples per run and divided by the total amount of uranine consumed to determine a percent yield (observed/expected) for the sampling run.']),\n",
       " ('filter samples',\n",
       "  'subclass of',\n",
       "  'sampling',\n",
       "  ['Using the filter samples that were analyzed for uranine, we determined the actual amount of uranine collected per sample.',\n",
       "   'We averaged the three filter samples per run and divided by the total amount of uranine consumed to determine a percent yield (observed/expected) for the sampling run.']),\n",
       " ('nebulizer',\n",
       "  'has part',\n",
       "  'filter',\n",
       "  ['We computed the expected number of TCID 50 on each filter sample from the known TCID 50 and fluid volume added to the nebulizer, the amount of nebulizing fluid consumed per minute, and the sampling time.',\n",
       "   'The expected TCID 50 was adjusted for the yield for the uranine samples.']),\n",
       " ('TCID',\n",
       "  'facet of',\n",
       "  'yield',\n",
       "  ['We computed the expected number of TCID 50 on each filter sample from the known TCID 50 and fluid volume added to the nebulizer, the amount of nebulizing fluid consumed per minute, and the sampling time.',\n",
       "   'The expected TCID 50 was adjusted for the yield for the uranine samples.']),\n",
       " ('dilution gradient filter',\n",
       "  'subclass of',\n",
       "  'spiking',\n",
       "  [\"We were able to obtain from the dilution gradient filter spiking experiments a positive band from a 1:500,000 dilution of virus, which translates into a detected amount of approximately 0.77 TCID 50 or 0.39 PFU's per filter.\",\n",
       "   'The stability tests, in which, filters spiked with 25 TCID 50 /filter were used to collect air samples for 40 hours did not display any difference in band strength compared to control filters.']),\n",
       " ('TCID 50',\n",
       "  'instance of',\n",
       "  'filter',\n",
       "  [\"We were able to obtain from the dilution gradient filter spiking experiments a positive band from a 1:500,000 dilution of virus, which translates into a detected amount of approximately 0.77 TCID 50 or 0.39 PFU's per filter.\",\n",
       "   'The stability tests, in which, filters spiked with 25 TCID 50 /filter were used to collect air samples for 40 hours did not display any difference in band strength compared to control filters.']),\n",
       " ('aerosol',\n",
       "  'has part',\n",
       "  'virus',\n",
       "  ['Figure 2 displays the results from the aerosol experiments with the lowest concentration of virus detected.',\n",
       "   'Table 1 summarizes results of the aerosol experiments, including the expected TCID 50 captured on each filter during each sampling run and the number of filters that produced positive RT-PCR results.']),\n",
       " ('RT-PCR',\n",
       "  'uses',\n",
       "  'sampling',\n",
       "  ['Figure 2 displays the results from the aerosol experiments with the lowest concentration of virus detected.',\n",
       "   'Table 1 summarizes results of the aerosol experiments, including the expected TCID 50 captured on each filter during each sampling run and the number of filters that produced positive RT-PCR results.']),\n",
       " ('RT-PCR',\n",
       "  'has part',\n",
       "  'primers',\n",
       "  ['The limit of detection of our semi-nested RT-PCR assay with primers OL26, OL27 and JWA-1b is in the range of 1.3 TCID 50 /filter, yielding a clearly visible 292 bp band.',\n",
       "   'No signal was detected from samples with 0.5 TCID 50 /filter.(?<=[.?!])\\\\s+The percent yields for each sampling run, were low, ranging from 4% to 10%.']),\n",
       " ('primers',\n",
       "  'part of',\n",
       "  'RT-PCR',\n",
       "  ['The limit of detection of our semi-nested RT-PCR assay with primers OL26, OL27 and JWA-1b is in the range of 1.3 TCID 50 /filter, yielding a clearly visible 292 bp band.',\n",
       "   'No signal was detected from samples with 0.5 TCID 50 /filter.(?<=[.?!])\\\\s+The percent yields for each sampling run, were low, ranging from 4% to 10%.']),\n",
       " ('TCID',\n",
       "  'length',\n",
       "  '50',\n",
       "  ['The limit of detection of our semi-nested RT-PCR assay with primers OL26, OL27 and JWA-1b is in the range of 1.3 TCID 50 /filter, yielding a clearly visible 292 bp band.',\n",
       "   'No signal was detected from samples with 0.5 TCID 50 /filter.(?<=[.?!])\\\\s+The percent yields for each sampling run, were low, ranging from 4% to 10%.']),\n",
       " ('temperature',\n",
       "  'different from',\n",
       "  '%RH',\n",
       "  ['Temperature and relative humidity were recorded in the air downstream of the chamber.',\n",
       "   'The average temperature inside the chamber was very consistent, ranging from 26.4°C to 27°C and the %RH ranged from 24.9 to 50.7.']),\n",
       " ('aerosol',\n",
       "  'has part',\n",
       "  'UV',\n",
       "  ['Figure 3 shows the results from test of the effect of UV exposure on aerosolized rhinovirus.(?<=[.?!])\\\\s+Lanes 5 and 6 contain PCR products from filters containing UV exposed aerosol.',\n",
       "   'No difference was detected between the UV exposed and nonexposed filter samples.']),\n",
       " ('UV',\n",
       "  'has part',\n",
       "  'filter',\n",
       "  ['Figure 3 shows the results from test of the effect of UV exposure on aerosolized rhinovirus.(?<=[.?!])\\\\s+Lanes 5 and 6 contain PCR products from filters containing UV exposed aerosol.',\n",
       "   'No difference was detected between the UV exposed and nonexposed filter samples.']),\n",
       " ('extraction and detection assay',\n",
       "  'uses',\n",
       "  'viral suspension',\n",
       "  ['In order to determine if the uranine in the viral suspension was inhibiting the extraction and detection assay, two otherwise identical runs were conducted -one with and one without uranine.',\n",
       "   'Each sampling run was conducted with nebulizing solution containing 10.1 TCID 50 per ml.']),\n",
       " ('nebulizing solution',\n",
       "  'has part',\n",
       "  'TCID',\n",
       "  ['In order to determine if the uranine in the viral suspension was inhibiting the extraction and detection assay, two otherwise identical runs were conducted -one with and one without uranine.',\n",
       "   'Each sampling run was conducted with nebulizing solution containing 10.1 TCID 50 per ml.']),\n",
       " ('RT-PCR',\n",
       "  'use',\n",
       "  'extraction',\n",
       "  ['Results of the two runs were identical, demonstrating that the uranine was not inhibitory to the extraction or RT-PCR assay.',\n",
       "   'The majority of rhinoviral transmission among adults is thought to occur via airborne infectious droplet nuclei [8] [9] [10] .']),\n",
       " ('airborne infectious droplet nuclei',\n",
       "  'subclass of',\n",
       "  'rhinoviral transmission',\n",
       "  ['Results of the two runs were identical, demonstrating that the uranine was not inhibitory to the extraction or RT-PCR assay.',\n",
       "   'The majority of rhinoviral transmission among adults is thought to occur via airborne infectious droplet nuclei [8] [9] [10] .']),\n",
       " ('RT-PCR',\n",
       "  'uses',\n",
       "  'amplification',\n",
       "  ['This study was conducted to assess the ability to detect airborne rhinovirus on air sampling filters and documents the first successful attempt to detect airborne rhinovirus using RT-PCR amplification-based techniques.',\n",
       "   'These techniques allow for a highly sensitive method to conduct air sampling for rhinovirus.']),\n",
       " ('air sampling',\n",
       "  'facet of',\n",
       "  'rhinovirus',\n",
       "  ['This study was conducted to assess the ability to detect airborne rhinovirus on air sampling filters and documents the first successful attempt to detect airborne rhinovirus using RT-PCR amplification-based techniques.',\n",
       "   'These techniques allow for a highly sensitive method to conduct air sampling for rhinovirus.']),\n",
       " ('enteroviral aerosols',\n",
       "  'subclass of',\n",
       "  'aerosolized',\n",
       "  ['This assay also is capable of detecting aerosolized enterovirus, although future studies are necessary to confirm the limit of detection for enteroviral aerosols Our limit of detection for the collection, extraction and amplification of aerosolized rhinovirus was 1.3 TCID 50 .',\n",
       "   'The spiking experiments we conducted had a limit of detection of approximately 0.8 TCID 50 or 60% of what was detected when the virus was aerosolized.']),\n",
       " ('aerosolized',\n",
       "  'subclass of',\n",
       "  'spiking',\n",
       "  ['This assay also is capable of detecting aerosolized enterovirus, although future studies are necessary to confirm the limit of detection for enteroviral aerosols Our limit of detection for the collection, extraction and amplification of aerosolized rhinovirus was 1.3 TCID 50 .',\n",
       "   'The spiking experiments we conducted had a limit of detection of approximately 0.8 TCID 50 or 60% of what was detected when the virus was aerosolized.']),\n",
       " ('University of Utah',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Salt Lake City',\n",
       "  ['These results can be interpreted in several ways.',\n",
       "   'One possibility is that the virus may be damaged during the process of aerosoliza-tion in the Collison nebulizer, and that the RNA from the damaged virus cannot be amplified.']),\n",
       " ('Collison',\n",
       "  'subclass of',\n",
       "  'nebulizer',\n",
       "  ['These results can be interpreted in several ways.',\n",
       "   'One possibility is that the virus may be damaged during the process of aerosoliza-tion in the Collison nebulizer, and that the RNA from the damaged virus cannot be amplified.']),\n",
       " ('nebulization solution',\n",
       "  'has effect',\n",
       "  'aerosolization',\n",
       "  ['This explanation would account for the results shown in Figure 2 where the nebulization solution prior to aerosolization yielded a positive result while after aerosolization we obtained a negative result.',\n",
       "   'Another possible explanation is that it is easier to extract the virus from a dried liquid droplet, as was done in the spiking experiments, than from an aerosol spread uniformly over the filter surface.']),\n",
       " ('spiking experiments',\n",
       "  'uses',\n",
       "  'aerosol',\n",
       "  ['This explanation would account for the results shown in Figure 2 where the nebulization solution prior to aerosolization yielded a positive result while after aerosolization we obtained a negative result.',\n",
       "   'Another possible explanation is that it is easier to extract the virus from a dried liquid droplet, as was done in the spiking experiments, than from an aerosol spread uniformly over the filter surface.']),\n",
       " ('aerosol',\n",
       "  'subclass of',\n",
       "  'expelled',\n",
       "  ['We were unable to recover the majority of aerosol that was expelled into the chamber.',\n",
       "   'The main reason for the low yields appears to be the result of large droplets falling out in the drying jar prior to entry into the exposure chamber.']),\n",
       " ('drying jar',\n",
       "  'has part',\n",
       "  'exposure chamber',\n",
       "  ['We were unable to recover the majority of aerosol that was expelled into the chamber.',\n",
       "   'The main reason for the low yields appears to be the result of large droplets falling out in the drying jar prior to entry into the exposure chamber.']),\n",
       " ('UV',\n",
       "  'has effect',\n",
       "  'yield',\n",
       "  ['Because the yields were the same regardless of the UV status, the low yields should not affect our results.',\n",
       "   'Previous studies have shown that UV inhibits PCR assays in single and double stranded RNA viruses, [28, 29] while our RT-PCR method did not detect a difference between UV exposed and nonexposed rhinoviral aerosols.']),\n",
       " ('double stranded RNA',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['Because the yields were the same regardless of the UV status, the low yields should not affect our results.',\n",
       "   'Previous studies have shown that UV inhibits PCR assays in single and double stranded RNA viruses, [28, 29] while our RT-PCR method did not detect a difference between UV exposed and nonexposed rhinoviral aerosols.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'dose',\n",
       "  ['The most likely reason for this is a matter of dose, although it is possible that characteristics of the particular viruses and PCR methods could also explain the results.',\n",
       "   'Neither of the previous studies were conducted on aerosolized virus and the dose of UV was several orders of magnitude greater.']),\n",
       " ('aerosolized',\n",
       "  'subclass of',\n",
       "  'virus',\n",
       "  ['The most likely reason for this is a matter of dose, although it is possible that characteristics of the particular viruses and PCR methods could also explain the results.',\n",
       "   'Neither of the previous studies were conducted on aerosolized virus and the dose of UV was several orders of magnitude greater.']),\n",
       " ('aerosol',\n",
       "  'has part',\n",
       "  'UV',\n",
       "  ['This study, did not determine whether the rhinovirus aerosol was capable of infection after exposure to the high UV dose used in these experiments.',\n",
       "   'It is possible that the UV irradiation damaged other regions of the viral RNA and not the region that our primers amplified, or that the amount of damage required to prevent RT-PCR amplification is greater than the amount of damage sufficient to prevent infection.']),\n",
       " ('RNA',\n",
       "  'part of',\n",
       "  'viral',\n",
       "  ['This study, did not determine whether the rhinovirus aerosol was capable of infection after exposure to the high UV dose used in these experiments.',\n",
       "   'It is possible that the UV irradiation damaged other regions of the viral RNA and not the region that our primers amplified, or that the amount of damage required to prevent RT-PCR amplification is greater than the amount of damage sufficient to prevent infection.']),\n",
       " ('RT-PCR',\n",
       "  'use',\n",
       "  'detection',\n",
       "  ['This suggests possible future studies to investigate the relationship between UV damage, infectiousness, and detection by RT-PCR of various regions of the rhinovirus genome.',\n",
       "   'The limit of detection of our assay must be put into context with the amount of virus shed by an infectious individual.']),\n",
       " ('infectious',\n",
       "  'has cause',\n",
       "  'virus',\n",
       "  ['This suggests possible future studies to investigate the relationship between UV damage, infectiousness, and detection by RT-PCR of various regions of the rhinovirus genome.',\n",
       "   'The limit of detection of our assay must be put into context with the amount of virus shed by an infectious individual.']),\n",
       " ('nasopharngeal wash',\n",
       "  'has part',\n",
       "  'TCID',\n",
       "  ['A study of rhinoviral shedding detected a titer of at least 1,600 TCID 50 per ml of nasopharngeal wash in 17 out of 26 infected volunteers [35] .',\n",
       "   'So it is clear that infected people can produce a large amount of virus, but it is unknown how much of that virus becomes airborne through coughs, sneezes, respiration and talking.']),\n",
       " ('cough',\n",
       "  'has effect',\n",
       "  'sneezes',\n",
       "  ['A study of rhinoviral shedding detected a titer of at least 1,600 TCID 50 per ml of nasopharngeal wash in 17 out of 26 infected volunteers [35] .',\n",
       "   'So it is clear that infected people can produce a large amount of virus, but it is unknown how much of that virus becomes airborne through coughs, sneezes, respiration and talking.']),\n",
       " ('infectious',\n",
       "  'has part',\n",
       "  'dose',\n",
       "  ['Two studies have attempted to determine the infectious dose 50% (ID 50 ) of rhinovirus for humans [36, 37] .',\n",
       "   'The ID 50 for microorganisms depend on at least four criteria: (1) the strain of the organism, (2) the route of transmission, (3) the method of inoculation, and (4) host immunity.']),\n",
       " ('microorganism',\n",
       "  'subclass of',\n",
       "  'organism',\n",
       "  ['Two studies have attempted to determine the infectious dose 50% (ID 50 ) of rhinovirus for humans [36, 37] .',\n",
       "   'The ID 50 for microorganisms depend on at least four criteria: (1) the strain of the organism, (2) the route of transmission, (3) the method of inoculation, and (4) host immunity.']),\n",
       " ('1966',\n",
       "  'point in time',\n",
       "  '1966',\n",
       "  ['In a study published in 1966, investigators from the U.S.',\n",
       "   'Army Biological Laboratories and the National Institute of Allergy and Infectious Disease, conducted a series of tests designed to determine if respiratory viral agents were transmitted via the aerosol route.']),\n",
       " ('National Institute of Allergy and Infectious Disease',\n",
       "  'subsidiary',\n",
       "  'Army Biological Laboratories',\n",
       "  ['In a study published in 1966, investigators from the U.S.',\n",
       "   'Army Biological Laboratories and the National Institute of Allergy and Infectious Disease, conducted a series of tests designed to determine if respiratory viral agents were transmitted via the aerosol route.']),\n",
       " ('nasal drops',\n",
       "  'subclass of',\n",
       "  'aerosol',\n",
       "  ['Using rhinovirus type 15, they inoculated volunteers in the nose with both nasal drops and an aerosol (particle size 0.3 to 2.5 µm).',\n",
       "   'The authors estimated a human ID 50 of 0.032 TCID 50 for nasal drops and 0.68 TCID 50 for aerosols [36] .']),\n",
       " ('nasal drops',\n",
       "  'subclass of',\n",
       "  'aerosols',\n",
       "  ['Using rhinovirus type 15, they inoculated volunteers in the nose with both nasal drops and an aerosol (particle size 0.3 to 2.5 µm).',\n",
       "   'The authors estimated a human ID 50 of 0.032 TCID 50 for nasal drops and 0.68 TCID 50 for aerosols [36] .']),\n",
       " ('rhinovirus type 16',\n",
       "  'subclass of',\n",
       "  'viral',\n",
       "  ['In a study conducted in 1984, researchers inoculated volunteers with rhinovirus type 16 with droplets of viral suspension in several locations.',\n",
       "   'The authors determined the ID 50 for nasal drops was 0.28 TCID 50 [37] .']),\n",
       " ('TCID 50',\n",
       "  'instance of',\n",
       "  'ID 50',\n",
       "  ['In a study conducted in 1984, researchers inoculated volunteers with rhinovirus type 16 with droplets of viral suspension in several locations.',\n",
       "   'The authors determined the ID 50 for nasal drops was 0.28 TCID 50 [37] .']),\n",
       " ('aerosol data from 1966',\n",
       "  'point in time',\n",
       "  '1966',\n",
       "  ['If we assume that the infectious dose is acquired through an aerosol, the aerosol data from 1966 suggests that our assay can detect an amount approximately equal to twice the ID 50 in humans.',\n",
       "   'But, if the spiking experiment is a better estimate of actual detection limit due to the inactivation by the nebulizer, then our detection limit is approximately equal to the ID 50 .']),\n",
       " ('spiking experiment',\n",
       "  'uses',\n",
       "  'nebulizer',\n",
       "  ['If we assume that the infectious dose is acquired through an aerosol, the aerosol data from 1966 suggests that our assay can detect an amount approximately equal to twice the ID 50 in humans.',\n",
       "   'But, if the spiking experiment is a better estimate of actual detection limit due to the inactivation by the nebulizer, then our detection limit is approximately equal to the ID 50 .']),\n",
       " ('PCR',\n",
       "  'use',\n",
       "  'air samples',\n",
       "  ['Caution must be taken when interpreting results from PCR-based air samples.',\n",
       "   'Nucleic acid detection methods, such as the one presented here, are capable of detecting very small quantities of microorganisms, but these techniques cannot determine if the microorganism is infectious at the time of detection.']),\n",
       " ('infectious',\n",
       "  'subclass of',\n",
       "  'microorganism',\n",
       "  ['Caution must be taken when interpreting results from PCR-based air samples.',\n",
       "   'Nucleic acid detection methods, such as the one presented here, are capable of detecting very small quantities of microorganisms, but these techniques cannot determine if the microorganism is infectious at the time of detection.']),\n",
       " ('PCR',\n",
       "  'use',\n",
       "  'detection',\n",
       "  ['The filter collection and PCR-based detection methods developed here offers some clear advantages over the use of more traditional methods such as impinger collection and cell culture based detection.',\n",
       "   'The use of filter collection allows for a much longer sample time, while impingers cannot be used for longer than 15-20 minutes.']),\n",
       " ('filter collection',\n",
       "  'opposite of',\n",
       "  'impingers',\n",
       "  ['The filter collection and PCR-based detection methods developed here offers some clear advantages over the use of more traditional methods such as impinger collection and cell culture based detection.',\n",
       "   'The use of filter collection allows for a much longer sample time, while impingers cannot be used for longer than 15-20 minutes.']),\n",
       " ('virus',\n",
       "  'has effect',\n",
       "  'detect',\n",
       "  ['As was demonstrated here, filters can be used to collect at least 40-hour samples without reduced ability to detect virus on the filter.',\n",
       "   'Cell culture methods, while useful in determining the amount of infectious virus, could not be used to detect the very low levels of infectious virus present in room air because collection of viable virus over long sampling times is not currently feasible.']),\n",
       " ('Cell culture',\n",
       "  'subclass of',\n",
       "  'sampling',\n",
       "  ['As was demonstrated here, filters can be used to collect at least 40-hour samples without reduced ability to detect virus on the filter.',\n",
       "   'Cell culture methods, while useful in determining the amount of infectious virus, could not be used to detect the very low levels of infectious virus present in room air because collection of viable virus over long sampling times is not currently feasible.']),\n",
       " ('PCR',\n",
       "  'use',\n",
       "  'detection',\n",
       "  ['Therefore, the use of filter collection and PCR based detection is the only method presently available to detect low concentrations of respiratory virus in air.',\n",
       "   'However, at UV doses much higher than typically used in upper room UV applications, this method was unable to distinguish between UV irradiated aerosols and nonexposed aerosols.']),\n",
       " ('UV irradiated aerosols',\n",
       "  'opposite of',\n",
       "  'nonexposed aerosols',\n",
       "  ['Therefore, the use of filter collection and PCR based detection is the only method presently available to detect low concentrations of respiratory virus in air.',\n",
       "   'However, at UV doses much higher than typically used in upper room UV applications, this method was unable to distinguish between UV irradiated aerosols and nonexposed aerosols.']),\n",
       " ('nonexposed aerosols',\n",
       "  'opposite of',\n",
       "  'UV irradiated aerosols',\n",
       "  ['Therefore, the use of filter collection and PCR based detection is the only method presently available to detect low concentrations of respiratory virus in air.',\n",
       "   'However, at UV doses much higher than typically used in upper room UV applications, this method was unable to distinguish between UV irradiated aerosols and nonexposed aerosols.']),\n",
       " ('hypothesized pattern',\n",
       "  'facet of',\n",
       "  'human population migrations',\n",
       "  ['Recent analyses of human pathogens have revealed that their evolutionary histories are congruent with the hypothesized pattern of ancient and modern human population migrations.',\n",
       "   'Phylogenetic trees of strains of the bacterium Helicobacter pylori and the polyoma JC virus taken from geographically diverse groups of human beings correlate closely with relationships of the populations in which they are found.']),\n",
       " ('Helicobacter pylori',\n",
       "  'instance of',\n",
       "  'bacterium',\n",
       "  ['Recent analyses of human pathogens have revealed that their evolutionary histories are congruent with the hypothesized pattern of ancient and modern human population migrations.',\n",
       "   'Phylogenetic trees of strains of the bacterium Helicobacter pylori and the polyoma JC virus taken from geographically diverse groups of human beings correlate closely with relationships of the populations in which they are found.']),\n",
       " ('Charles Darwin',\n",
       "  'notable work',\n",
       "  'evolutionarily significant',\n",
       "  ['Charles Darwin recognized that the distribution and form of parasites was evolutionarily significant.(?<=[.?!])\\\\s+He noted, for instance, that \"...',\n",
       "   'the Pediculi [lice] collected in different countries from the different races of man ...(?<=[.?!])\\\\s+differ, not only in colour, but in the structure of their claws and limbs.']),\n",
       " ('evolutionarily significant',\n",
       "  'discoverer or inventor',\n",
       "  'Charles Darwin',\n",
       "  ['Charles Darwin recognized that the distribution and form of parasites was evolutionarily significant.(?<=[.?!])\\\\s+He noted, for instance, that \"...',\n",
       "   'the Pediculi [lice] collected in different countries from the different races of man ...(?<=[.?!])\\\\s+differ, not only in colour, but in the structure of their claws and limbs.']),\n",
       " ('Claw',\n",
       "  'part of',\n",
       "  'limb',\n",
       "  ['Charles Darwin recognized that the distribution and form of parasites was evolutionarily significant.(?<=[.?!])\\\\s+He noted, for instance, that \"...',\n",
       "   'the Pediculi [lice] collected in different countries from the different races of man ...(?<=[.?!])\\\\s+differ, not only in colour, but in the structure of their claws and limbs.']),\n",
       " ('University of Oxford',\n",
       "  'country',\n",
       "  'United Kingdom',\n",
       "  ['In every case in which many specimens were obtained the differences were constant\" [1] .',\n",
       "   'More recently, several research groups [2] [3] [4] [5] [6] [7] have found interesting correlations between the evolutionary relationships among various bacterial and viral strains hosted by humans and the pattern of migrations of modern humans throughout the world.']),\n",
       " ('viral',\n",
       "  'subclass of',\n",
       "  'bacteria',\n",
       "  ['In every case in which many specimens were obtained the differences were constant\" [1] .',\n",
       "   'More recently, several research groups [2] [3] [4] [5] [6] [7] have found interesting correlations between the evolutionary relationships among various bacterial and viral strains hosted by humans and the pattern of migrations of modern humans throughout the world.']),\n",
       " ('Helicobacter pylori',\n",
       "  'Gram staining',\n",
       "  'Gram-negative',\n",
       "  ['A particularly interesting case is that of Helicobacter pylori, a Gram-negative bacterium associated with gastritis, peptic ulcers, and gastric cancer that may infect up to half of all humans [8] .',\n",
       "   'The discovery that a bacterial infection could lead to what were considered chronic diseases [8] was a striking example of the fact that infectious diseases have not yet been conquered.']),\n",
       " ('chronic disease',\n",
       "  'subclass of',\n",
       "  'infectious disease',\n",
       "  ['A particularly interesting case is that of Helicobacter pylori, a Gram-negative bacterium associated with gastritis, peptic ulcers, and gastric cancer that may infect up to half of all humans [8] .',\n",
       "   'The discovery that a bacterial infection could lead to what were considered chronic diseases [8] was a striking example of the fact that infectious diseases have not yet been conquered.']),\n",
       " ('West Nile Virus',\n",
       "  'country',\n",
       "  'United States',\n",
       "  ['The continuing acquired immune deficiency syndrome (AIDS) epidemic, outbreaks of Ebola in Central Africa, and the current spread of West Nile Virus in the United States and of severe acute respiratory syndrome (SARS) from Asia provide evidence of the pervasiveness and health consequences of infectious agents even in the age of vaccination and antimicrobial and antiviral therapies.',\n",
       "   'Many infectious diseases are thought to have arisen concurrently with the development of agriculture and the rise of urban living.']),\n",
       " ('urban living',\n",
       "  'subclass of',\n",
       "  'agriculture',\n",
       "  ['The continuing acquired immune deficiency syndrome (AIDS) epidemic, outbreaks of Ebola in Central Africa, and the current spread of West Nile Virus in the United States and of severe acute respiratory syndrome (SARS) from Asia provide evidence of the pervasiveness and health consequences of infectious agents even in the age of vaccination and antimicrobial and antiviral therapies.',\n",
       "   'Many infectious diseases are thought to have arisen concurrently with the development of agriculture and the rise of urban living.']),\n",
       " ('viral',\n",
       "  'subclass of',\n",
       "  'pathogen',\n",
       "  [\"If, instead, many pathogens' relationships with humans are much older, it would not be surprising to find deeper evolutionary associations between humans and their microbial and viral invaders.\",\n",
       "   'The evolutionary history of H.(?<=[.?!])\\\\s+pylori may provide an example of the coevolution of a bacterium and its only known host.(?<=[.?!])\\\\s+The H.']),\n",
       " ('host',\n",
       "  'subclass of',\n",
       "  'bacterium',\n",
       "  [\"If, instead, many pathogens' relationships with humans are much older, it would not be surprising to find deeper evolutionary associations between humans and their microbial and viral invaders.\",\n",
       "   'The evolutionary history of H.(?<=[.?!])\\\\s+pylori may provide an example of the coevolution of a bacterium and its only known host.(?<=[.?!])\\\\s+The H.']),\n",
       " ('genes',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['The H.(?<=[.?!])\\\\s+pylori genome is relatively small at 1.67 megabases, with a minimal complement of metabolic genes [9] .(?<=[.?!])\\\\s+Variation between H.',\n",
       "   'Variation between H.(?<=[.?!])\\\\s+pylori isolates from different people or even from one person is large, leading to unique fingerprints for nearly every isolate so far typed.']),\n",
       " ('unique',\n",
       "  'subclass of',\n",
       "  'fingerprint',\n",
       "  ['The H.(?<=[.?!])\\\\s+pylori genome is relatively small at 1.67 megabases, with a minimal complement of metabolic genes [9] .(?<=[.?!])\\\\s+Variation between H.',\n",
       "   'Variation between H.(?<=[.?!])\\\\s+pylori isolates from different people or even from one person is large, leading to unique fingerprints for nearly every isolate so far typed.']),\n",
       " ('codon',\n",
       "  'part of',\n",
       "  'amino-acid sequence',\n",
       "  ['The coding genes are not very diverse, however: most of the variation occurs in the third base position within codons or through inversions or translocations, leaving the encoded amino-acid sequences relatively similar [2] .',\n",
       "   'This amino-acid sequence conservation is fortunate for vaccine research, as one vaccine is likely to be effective on many strains.(?<=[.?!])\\\\s+More interesting is the fact that H.']),\n",
       " ('vaccine research',\n",
       "  'studies',\n",
       "  'vaccine',\n",
       "  ['The coding genes are not very diverse, however: most of the variation occurs in the third base position within codons or through inversions or translocations, leaving the encoded amino-acid sequences relatively similar [2] .',\n",
       "   'This amino-acid sequence conservation is fortunate for vaccine research, as one vaccine is likely to be effective on many strains.(?<=[.?!])\\\\s+More interesting is the fact that H.']),\n",
       " ('vaccine',\n",
       "  'studied by',\n",
       "  'vaccine research',\n",
       "  ['The coding genes are not very diverse, however: most of the variation occurs in the third base position within codons or through inversions or translocations, leaving the encoded amino-acid sequences relatively similar [2] .',\n",
       "   'This amino-acid sequence conservation is fortunate for vaccine research, as one vaccine is likely to be effective on many strains.(?<=[.?!])\\\\s+More interesting is the fact that H.']),\n",
       " ('recombination',\n",
       "  'participant',\n",
       "  'organism',\n",
       "  ['pylori has an extremely high rate of recombination, higher in fact than that of any other organism characterized to date [3, 10] .',\n",
       "   \"Normally, such a high rate of recombination would make inferring an organism's evolutionary history very difficult, as information about the origin of each mutation would be lost as it spreads throughout a population.\"]),\n",
       " ('mutation',\n",
       "  'part of',\n",
       "  'evolutionary history',\n",
       "  ['pylori has an extremely high rate of recombination, higher in fact than that of any other organism characterized to date [3, 10] .',\n",
       "   \"Normally, such a high rate of recombination would make inferring an organism's evolutionary history very difficult, as information about the origin of each mutation would be lost as it spreads throughout a population.\"]),\n",
       " ('mode of transmission',\n",
       "  'subclass of',\n",
       "  'recombination',\n",
       "  ['But coupled with the mode of transmission of H.(?<=[.?!])\\\\s+pylori, this extremely high rate of recombination may in fact make evolutionary inferences easier.',\n",
       "   'Several studies strongly suggest that H.(?<=[.?!])\\\\s+pylori is usually transmitted within families, generally from mother to child [11, 12] .(?<=[.?!])\\\\s+Thus, the transmission of H.']),\n",
       " ('mother',\n",
       "  'child',\n",
       "  'child',\n",
       "  ['But coupled with the mode of transmission of H.(?<=[.?!])\\\\s+pylori, this extremely high rate of recombination may in fact make evolutionary inferences easier.',\n",
       "   'Several studies strongly suggest that H.(?<=[.?!])\\\\s+pylori is usually transmitted within families, generally from mother to child [11, 12] .(?<=[.?!])\\\\s+Thus, the transmission of H.']),\n",
       " ('child',\n",
       "  'mother',\n",
       "  'mother',\n",
       "  ['But coupled with the mode of transmission of H.(?<=[.?!])\\\\s+pylori, this extremely high rate of recombination may in fact make evolutionary inferences easier.',\n",
       "   'Several studies strongly suggest that H.(?<=[.?!])\\\\s+pylori is usually transmitted within families, generally from mother to child [11, 12] .(?<=[.?!])\\\\s+Thus, the transmission of H.']),\n",
       " ('mitochondrial DNA',\n",
       "  'part of',\n",
       "  'maternally transmitted',\n",
       "  ['pylori in some ways mimics that of maternally transmitted mitochondrial DNA [13] .',\n",
       "   'Because mitochondrial DNA is transmitted solely from one parent (the mother) and does not recombine, it has proved to be an ideal genetic system for inferring human evolutionary history (see below) [14, 15] .']),\n",
       " ('mitochondrial DNA',\n",
       "  'instance of',\n",
       "  'genetic system',\n",
       "  ['pylori in some ways mimics that of maternally transmitted mitochondrial DNA [13] .',\n",
       "   'Because mitochondrial DNA is transmitted solely from one parent (the mother) and does not recombine, it has proved to be an ideal genetic system for inferring human evolutionary history (see below) [14, 15] .']),\n",
       " ('If H.',\n",
       "  'instance of',\n",
       "  'If',\n",
       "  ['If H.',\n",
       "   \"pylori is indeed predominantly maternally transmitted, new strains will generally not infect a person during their lifetime; together with the high rate of recombination, this would mean that the mutations that accumulate within the population of bacteria in an individual's stomach will be relatively homogeneous.\"]),\n",
       " ('recombination',\n",
       "  'has effect',\n",
       "  'mutation',\n",
       "  ['If H.',\n",
       "   \"pylori is indeed predominantly maternally transmitted, new strains will generally not infect a person during their lifetime; together with the high rate of recombination, this would mean that the mutations that accumulate within the population of bacteria in an individual's stomach will be relatively homogeneous.\"]),\n",
       " ('strains',\n",
       "  'subclass of',\n",
       "  'bacteria',\n",
       "  ['This should result in a swarm of strains that are very closely related to each other, containing many of the mutations that have occurred in individual bacteria.',\n",
       "   'Swarms found in different people will thus be more different from each other than if there was less recombination.']),\n",
       " ('different from each other',\n",
       "  'subclass of',\n",
       "  'Swarm',\n",
       "  ['This should result in a swarm of strains that are very closely related to each other, containing many of the mutations that have occurred in individual bacteria.',\n",
       "   'Swarms found in different people will thus be more different from each other than if there was less recombination.']),\n",
       " ('spread',\n",
       "  'has cause',\n",
       "  'infectious diseases',\n",
       "  ['Most infectious diseases spread quickly throughout the world and strains from different regions are relatively similar, but initial sampling of H.',\n",
       "   'pylori from people from different regions of the world revealed fairly strong geographic partitioning into European and Asian H.(?<=[.?!])\\\\s+pylori types [2] [3] [4] .']),\n",
       " ('Europe',\n",
       "  'shares border with',\n",
       "  'Asia',\n",
       "  ['Most infectious diseases spread quickly throughout the world and strains from different regions are relatively similar, but initial sampling of H.',\n",
       "   'pylori from people from different regions of the world revealed fairly strong geographic partitioning into European and Asian H.(?<=[.?!])\\\\s+pylori types [2] [3] [4] .']),\n",
       " ('Asia',\n",
       "  'shares border with',\n",
       "  'Europe',\n",
       "  ['Most infectious diseases spread quickly throughout the world and strains from different regions are relatively similar, but initial sampling of H.',\n",
       "   'pylori from people from different regions of the world revealed fairly strong geographic partitioning into European and Asian H.(?<=[.?!])\\\\s+pylori types [2] [3] [4] .']),\n",
       " ('Falush',\n",
       "  'member of',\n",
       "  'colleagues',\n",
       "  ['Recently, Falush and colleagues [5] have examined this partitioning in greater detail.',\n",
       "   'After sequencing eight genes -a total of 3,850 nucleotides -in 370 strains derived from 27 human populations, they found 1,418 polymorphic nucleotide positions.']),\n",
       " ('gene',\n",
       "  'encodes',\n",
       "  'nucleotide',\n",
       "  ['Recently, Falush and colleagues [5] have examined this partitioning in greater detail.',\n",
       "   'After sequencing eight genes -a total of 3,850 nucleotides -in 370 strains derived from 27 human populations, they found 1,418 polymorphic nucleotide positions.']),\n",
       " ('genotype',\n",
       "  'has effect',\n",
       "  'genetic structure',\n",
       "  ['They then applied a new analytic tool, STRUCTURE [16] , that was developed to infer human genetic structure from multilocus genotype data.',\n",
       "   'The program uses Bayesian methods to identify subgroups with distinctive allele frequencies, and clusters of the subgroups, even in the presence of recombination [16] .']),\n",
       " ('clusters',\n",
       "  'facet of',\n",
       "  'Bayesian',\n",
       "  ['They then applied a new analytic tool, STRUCTURE [16] , that was developed to infer human genetic structure from multilocus genotype data.',\n",
       "   'The program uses Bayesian methods to identify subgroups with distinctive allele frequencies, and clusters of the subgroups, even in the presence of recombination [16] .']),\n",
       " ('Europe',\n",
       "  'shares border with',\n",
       "  'Asia',\n",
       "  ['When this technique was applied to the H.(?<=[.?!])\\\\s+pylori sequences, four main clusters were found -two from Africa and one each from Europe and Asia (Figure 1a ) [5] .',\n",
       "   \"Each cluster found by Falush and colleagues [5] could be divided into subgroups; for instance, the 'Africa 1' cluster could be subdivided further into West and South African subclusters, and the East Asia cluster could be split into East Asian, Amerind, and Maori subclusters.\"]),\n",
       " ('Amerind',\n",
       "  'part of',\n",
       "  'East Asia',\n",
       "  ['When this technique was applied to the H.(?<=[.?!])\\\\s+pylori sequences, four main clusters were found -two from Africa and one each from Europe and Asia (Figure 1a ) [5] .',\n",
       "   \"Each cluster found by Falush and colleagues [5] could be divided into subgroups; for instance, the 'Africa 1' cluster could be subdivided further into West and South African subclusters, and the East Asia cluster could be split into East Asian, Amerind, and Maori subclusters.\"]),\n",
       " ('swept back and forth',\n",
       "  'location',\n",
       "  'Europe',\n",
       "  ['The geographic partitioning within the 200 European strains was particularly complicated, presumably because numerous groups have swept back and forth across Europe over the past several millennia.',\n",
       "   'European strains also occasionally appeared in the Americas, Australia, and among South Africans, presumably reflecting colonial conquest.']),\n",
       " ('colonial conquest',\n",
       "  'country',\n",
       "  'South Africa',\n",
       "  ['The geographic partitioning within the 200 European strains was particularly complicated, presumably because numerous groups have swept back and forth across Europe over the past several millennia.',\n",
       "   'European strains also occasionally appeared in the Americas, Australia, and among South Africans, presumably reflecting colonial conquest.']),\n",
       " ('The relationships between human populations',\n",
       "  'instance of',\n",
       "  'phylogenetic',\n",
       "  ['The phylogenetic relationships of these clusters (Figure 1b) and their subdivisions [5] show a pattern similar to that The relationships between human populations, as calculated from H.',\n",
       "   'pylori found in stomachs and from mitochondrial DNA data.(?<=[.?!])\\\\s+(a) Relationships between modern subpopulations of H.(?<=[.?!])\\\\s+pylori [5] .']),\n",
       " ('mitochondrial',\n",
       "  'has part',\n",
       "  'DNA',\n",
       "  ['The phylogenetic relationships of these clusters (Figure 1b) and their subdivisions [5] show a pattern similar to that The relationships between human populations, as calculated from H.',\n",
       "   'pylori found in stomachs and from mitochondrial DNA data.(?<=[.?!])\\\\s+(a) Relationships between modern subpopulations of H.(?<=[.?!])\\\\s+pylori [5] .']),\n",
       " ('diameter',\n",
       "  'part of',\n",
       "  'subpopulation',\n",
       "  ['pylori [5] .(?<=[.?!])\\\\s+Each subpopulation is represented by a circle with a diameter proportional to the genetic diversity within it.',\n",
       "   'The centres of the circles are joined by a phylogenetic tree showing the relationships between the four subpopulations.']),\n",
       " ('tree',\n",
       "  'studied by',\n",
       "  'phylogenetic',\n",
       "  ['pylori [5] .(?<=[.?!])\\\\s+Each subpopulation is represented by a circle with a diameter proportional to the genetic diversity within it.',\n",
       "   'The centres of the circles are joined by a phylogenetic tree showing the relationships between the four subpopulations.']),\n",
       " ('phylogenetic tree',\n",
       "  'subclass of',\n",
       "  'phylogenetic',\n",
       "  ['Bacteria in each subpopulation are found predominantly in people who originate from the regions shown.(?<=[.?!])\\\\s+(b) A population-level phylogenetic tree of the H.',\n",
       "   'pylori geographic subpopulations shown in (a).(?<=[.?!])\\\\s+(c) A median-joining network of human populations derived from mitochondrial DNA [14] .']),\n",
       " ('mitochondrial DNA',\n",
       "  'part of',\n",
       "  'human populations',\n",
       "  ['Bacteria in each subpopulation are found predominantly in people who originate from the regions shown.(?<=[.?!])\\\\s+(b) A population-level phylogenetic tree of the H.',\n",
       "   'pylori geographic subpopulations shown in (a).(?<=[.?!])\\\\s+(c) A median-joining network of human populations derived from mitochondrial DNA [14] .']),\n",
       " ('clusters',\n",
       "  'instance of',\n",
       "  'evolutionary relationships',\n",
       "  ['Such a network shows alternative potential evolutionary relationships between clusters.',\n",
       "   'Each circle represents a cluster of mitochondrial types with a diameter proportional to the frequency of that type within the subpopulations.']),\n",
       " ('cluster',\n",
       "  'subclass of',\n",
       "  'diameter',\n",
       "  ['Such a network shows alternative potential evolutionary relationships between clusters.',\n",
       "   'Each circle represents a cluster of mitochondrial types with a diameter proportional to the frequency of that type within the subpopulations.']),\n",
       " ('network of relationships',\n",
       "  'subclass of',\n",
       "  'lineage',\n",
       "  ['All non-African populations are derived from one African lineage; the network of relationships within this lineage is magnified (top).(?<=[.?!])\\\\s+(a,b) Adapted from [5] ; (c) adapted from [14] .',\n",
       "   'obtained using mitochondrial DNA variation (Figure 1c ) [14, 15] .']),\n",
       " ('mitochondrial DNA',\n",
       "  'subclass of',\n",
       "  'DNA',\n",
       "  ['All non-African populations are derived from one African lineage; the network of relationships within this lineage is magnified (top).(?<=[.?!])\\\\s+(a,b) Adapted from [5] ; (c) adapted from [14] .',\n",
       "   'obtained using mitochondrial DNA variation (Figure 1c ) [14, 15] .']),\n",
       " ('mitochondrial DNA',\n",
       "  'part of',\n",
       "  'Y chromosome',\n",
       "  ['The modern human gene pool, as inferred from mitochondrial DNA and corroborated by Y chromosome studies, is thought to have had an African origin approximately 150,000-200,000 years ago [14, 15, 17, 18] .',\n",
       "   'The original human population then spread and diversified throughout Africa for nearly 100,000 years, before expanding into Western Asia and Europe and into Southern and Eastern Asia approximately 50,000-60,000 years ago, replacing the existing archaic populations of humans in these regions.']),\n",
       " ('Western Asia',\n",
       "  'shares border with',\n",
       "  'Eastern Asia',\n",
       "  ['The modern human gene pool, as inferred from mitochondrial DNA and corroborated by Y chromosome studies, is thought to have had an African origin approximately 150,000-200,000 years ago [14, 15, 17, 18] .',\n",
       "   'The original human population then spread and diversified throughout Africa for nearly 100,000 years, before expanding into Western Asia and Europe and into Southern and Eastern Asia approximately 50,000-60,000 years ago, replacing the existing archaic populations of humans in these regions.']),\n",
       " ('Pacific Islands',\n",
       "  'part of',\n",
       "  'Australasia',\n",
       "  ['Subsequent migrations spread into Australasia by 40,000 years ago, then to the Pacific Islands, and later into North America, approximately 15,000 years ago ( Figure 2 ) [14, 15, 17, 18] .',\n",
       "   'The remarkable similarity between this view of human history and the results from studies of H.(?<=[.?!])\\\\s+pylori have led Falush and colleagues [5] as well as others [2] to conclude that H.']),\n",
       " ('World Heritage Site',\n",
       "  'maintained by',\n",
       "  'UNESCO',\n",
       "  ['Subsequent migrations spread into Australasia by 40,000 years ago, then to the Pacific Islands, and later into North America, approximately 15,000 years ago ( Figure 2 ) [14, 15, 17, 18] .',\n",
       "   'The remarkable similarity between this view of human history and the results from studies of H.(?<=[.?!])\\\\s+pylori have led Falush and colleagues [5] as well as others [2] to conclude that H.']),\n",
       " ('expansion',\n",
       "  'has part',\n",
       "  'migration',\n",
       "  ['pylori evolution has followed the path of modern human expansion and migration.',\n",
       "   'This work thus provides another type of data for analysis of human evolution and migration, independent of mitochondrial DNA and Y chromosomes, that will be valuable in further studies.']),\n",
       " ('Y chromosome',\n",
       "  'has part',\n",
       "  'mitochondrial DNA',\n",
       "  ['pylori evolution has followed the path of modern human expansion and migration.',\n",
       "   'This work thus provides another type of data for analysis of human evolution and migration, independent of mitochondrial DNA and Y chromosomes, that will be valuable in further studies.']),\n",
       " ('recombination',\n",
       "  'has effect',\n",
       "  'divergence',\n",
       "  ['Unfortunately, however, estimating the divergence dates from H.(?<=[.?!])\\\\s+pylori is particularly difficult, owing to the extremely high rate of recombination [10] .',\n",
       "   'Additional sampling and analytical techniques may be required to further test the migratory hypothesis.']),\n",
       " ('sampling',\n",
       "  'subclass of',\n",
       "  'analytical techniques',\n",
       "  ['Unfortunately, however, estimating the divergence dates from H.(?<=[.?!])\\\\s+pylori is particularly difficult, owing to the extremely high rate of recombination [10] .',\n",
       "   'Additional sampling and analytical techniques may be required to further test the migratory hypothesis.']),\n",
       " ('hosts',\n",
       "  'subclass of',\n",
       "  'pathogens',\n",
       "  ['Other pathogens have also been proposed to follow evolutionary histories similar to their hosts.',\n",
       "   'One of the most interesting examples, though with a non-human host, is that of aphids, a bacterium found within them, and two plasmids associated with the bacterium.(?<=[.?!])\\\\s+Funk et al.']),\n",
       " ('aphid',\n",
       "  'subclass of',\n",
       "  'bacterium',\n",
       "  ['Other pathogens have also been proposed to follow evolutionary histories similar to their hosts.',\n",
       "   'One of the most interesting examples, though with a non-human host, is that of aphids, a bacterium found within them, and two plasmids associated with the bacterium.(?<=[.?!])\\\\s+Funk et al.']),\n",
       " ('intraspecific',\n",
       "  'subclass of',\n",
       "  'phylogenies',\n",
       "  ['Funk et al.(?<=[.?!])\\\\s+[19] found that the inferred intraspecific phylogenies of these four genomes were completely congruent.',\n",
       "   'Returning to human pathogens, the human polyomavirus JC virus (JCV) can be divided into genotypes that correspond to the major continental land masses [6] .(?<=[.?!])\\\\s+Like H.']),\n",
       " ('JC virus',\n",
       "  'parent taxon',\n",
       "  'polyomavirus',\n",
       "  ['Funk et al.(?<=[.?!])\\\\s+[19] found that the inferred intraspecific phylogenies of these four genomes were completely congruent.',\n",
       "   'Returning to human pathogens, the human polyomavirus JC virus (JCV) can be divided into genotypes that correspond to the major continental land masses [6] .(?<=[.?!])\\\\s+Like H.']),\n",
       " ('multifocal leukoencephalopathy',\n",
       "  'has cause',\n",
       "  'myelination',\n",
       "  ['pylori, JCV -which can cause progressive multifocal leukoencephalopathy (loss of myelination in the central nervous system) -is very widespread amongst humans as a result of familial transmission.',\n",
       "   'A total of 12 known subtypes have been defined, with European, African, and Asian distributions [7] .']),\n",
       " ('subtypes',\n",
       "  'part of',\n",
       "  'distributions',\n",
       "  ['pylori, JCV -which can cause progressive multifocal leukoencephalopathy (loss of myelination in the central nervous system) -is very widespread amongst humans as a result of familial transmission.',\n",
       "   'A total of 12 known subtypes have been defined, with European, African, and Asian distributions [7] .']),\n",
       " ('phylogenetic tree',\n",
       "  'part of',\n",
       "  'evolutionary history',\n",
       "  ['Although direct inferences of an African origin for JCV are problematic because there is no suitable outgroup with which to root the phylogenetic tree, when an African origin is assumed, a reasonable evolutionary history can be hypothesized ( Figure 3 ) [7] .',\n",
       "   'As with H.(?<=[.?!])\\\\s+pylori, inferring molecular divergence dates is currently problematic for JCV.']),\n",
       " ('JCV',\n",
       "  'use',\n",
       "  'molecular divergence',\n",
       "  ['Although direct inferences of an African origin for JCV are problematic because there is no suitable outgroup with which to root the phylogenetic tree, when an African origin is assumed, a reasonable evolutionary history can be hypothesized ( Figure 3 ) [7] .',\n",
       "   'As with H.(?<=[.?!])\\\\s+pylori, inferring molecular divergence dates is currently problematic for JCV.']),\n",
       " ('evolutionary history',\n",
       "  'facet of',\n",
       "  'human pathogens',\n",
       "  ['Further investigation into the evolutionary history of these human pathogens is therefore necessary.',\n",
       "   'Elucidating the patterns of evolution of human pathogens may ultimately provide additional evidence not only about their history but also about human evolution and history.']),\n",
       " ('human evolution and history',\n",
       "  'facet of',\n",
       "  'evolution',\n",
       "  ['Further investigation into the evolutionary history of these human pathogens is therefore necessary.',\n",
       "   'Elucidating the patterns of evolution of human pathogens may ultimately provide additional evidence not only about their history but also about human evolution and history.']),\n",
       " ('mother-child mode of transmission',\n",
       "  'subclass of',\n",
       "  'mitochondrial DNA evolution',\n",
       "  ['This will be especially true for pathogens such as H.(?<=[.?!])\\\\s+pylori that have a predominantly mother-child mode of transmission, mimicking mitochondrial DNA evolution.(?<=[.?!])\\\\s+H.',\n",
       "   \"pylori's causative role in several chronic stomach conditions is just one of many current examples of the known or suspected  A map of the pattern of expansion and migration of modern humans throughout the world, derived from studies of mitochondrial DNA and Y chromosomes [14, 15, 17, 18] .\"]),\n",
       " ('mitochondrial DNA',\n",
       "  'part of',\n",
       "  'Y chromosome',\n",
       "  ['This will be especially true for pathogens such as H.(?<=[.?!])\\\\s+pylori that have a predominantly mother-child mode of transmission, mimicking mitochondrial DNA evolution.(?<=[.?!])\\\\s+H.',\n",
       "   \"pylori's causative role in several chronic stomach conditions is just one of many current examples of the known or suspected  A map of the pattern of expansion and migration of modern humans throughout the world, derived from studies of mitochondrial DNA and Y chromosomes [14, 15, 17, 18] .\"]),\n",
       " ('infectious agent',\n",
       "  'has effect',\n",
       "  'chronic diseases',\n",
       "  ['Numbers indicate the approximate time (in years before the present) when modern humans first appeared in the indicated region.(?<=[.?!])\\\\s+role of an infectious agent leading to chronic diseases.',\n",
       "   \"Bacteria are suspected to be involved in the development of arteriosclerosis, stroke, and Crohn's disease, whereas viruses are known to lead to AIDS and the various forms of chronic hepatitis.\"]),\n",
       " ('chronic diseases',\n",
       "  'has cause',\n",
       "  'infectious agent',\n",
       "  ['Numbers indicate the approximate time (in years before the present) when modern humans first appeared in the indicated region.(?<=[.?!])\\\\s+role of an infectious agent leading to chronic diseases.',\n",
       "   \"Bacteria are suspected to be involved in the development of arteriosclerosis, stroke, and Crohn's disease, whereas viruses are known to lead to AIDS and the various forms of chronic hepatitis.\"]),\n",
       " ('hepatitis',\n",
       "  'has cause',\n",
       "  'virus',\n",
       "  ['Numbers indicate the approximate time (in years before the present) when modern humans first appeared in the indicated region.(?<=[.?!])\\\\s+role of an infectious agent leading to chronic diseases.',\n",
       "   \"Bacteria are suspected to be involved in the development of arteriosclerosis, stroke, and Crohn's disease, whereas viruses are known to lead to AIDS and the various forms of chronic hepatitis.\"]),\n",
       " ('hepatocellular carcinoma',\n",
       "  'subclass of',\n",
       "  'Cervical cancer',\n",
       "  [\"Cervical cancer, hepatocellular carcinoma, Burkitt's lymphoma, Kaposi's sarcoma, and perhaps diabetes mellitus are also either known or suspected to be of viral origin.\",\n",
       "   'If some of the ubiquitous chronic diseases prove to be of bacterial or viral origin, worldwide surveys of these pathogens in human populations should be carried out immediately, so that knowledge about the evolution and diversity of these pathogens can be incorporated into the research programs designed to ameliorate the conditions that they cause.']),\n",
       " ('viral',\n",
       "  'subclass of',\n",
       "  'pathogen',\n",
       "  [\"Cervical cancer, hepatocellular carcinoma, Burkitt's lymphoma, Kaposi's sarcoma, and perhaps diabetes mellitus are also either known or suspected to be of viral origin.\",\n",
       "   'If some of the ubiquitous chronic diseases prove to be of bacterial or viral origin, worldwide surveys of these pathogens in human populations should be carried out immediately, so that knowledge about the evolution and diversity of these pathogens can be incorporated into the research programs designed to ameliorate the conditions that they cause.']),\n",
       " ('polyoma',\n",
       "  'has cause',\n",
       "  'JC virus',\n",
       "  ['Relationships of human polyoma JC virus (JCV) subtypes found in humans from different parts of the world [7] .(?<=[.?!])\\\\s+Letters refer to individual subtypes.',\n",
       "   '(a) The hypothesized pattern of spread of JCV subtypes through the world (excluding the Americas); (b) an inferred phylogeny of JCV subtypes, assuming an African origin for the virus.']),\n",
       " ('JCV subtypes',\n",
       "  'instance of',\n",
       "  'phylogeny',\n",
       "  ['Relationships of human polyoma JC virus (JCV) subtypes found in humans from different parts of the world [7] .(?<=[.?!])\\\\s+Letters refer to individual subtypes.',\n",
       "   '(a) The hypothesized pattern of spread of JCV subtypes through the world (excluding the Americas); (b) an inferred phylogeny of JCV subtypes, assuming an African origin for the virus.']),\n",
       " ('JCV subtypes',\n",
       "  'instance of',\n",
       "  'phylogeny',\n",
       "  ['Relationships of human polyoma JC virus (JCV) subtypes found in humans from different parts of the world [7] .(?<=[.?!])\\\\s+Letters refer to individual subtypes.',\n",
       "   '(a) The hypothesized pattern of spread of JCV subtypes through the world (excluding the Americas); (b) an inferred phylogeny of JCV subtypes, assuming an African origin for the virus.']),\n",
       " ('[7',\n",
       "  'instance of',\n",
       "  'adapted from',\n",
       "  ['Adapted from [7] .',\n",
       "   'BACKGROUND: The human leukocyte antigen (HLA) system is widely used as a strategy in the search for the etiology of infectious diseases and autoimmune disorders.']),\n",
       " ('autoimmune disorders',\n",
       "  'subclass of',\n",
       "  'infectious diseases',\n",
       "  ['Adapted from [7] .',\n",
       "   'BACKGROUND: The human leukocyte antigen (HLA) system is widely used as a strategy in the search for the etiology of infectious diseases and autoimmune disorders.']),\n",
       " ('Taiwan epidemic',\n",
       "  'has cause',\n",
       "  'severe acute respiratory syndrome',\n",
       "  ['During the Taiwan epidemic of severe acute respiratory syndrome (SARS), many health care workers were infected.',\n",
       "   'In an effort to establish a screening program for high risk personal, the distribution of HLA class I and II alleles in case and control groups was examined for the presence of an association to a genetic susceptibly or resistance to SARS coronavirus infection.']),\n",
       " ('severe acute respiratory syndrome',\n",
       "  'has effect',\n",
       "  'Taiwan epidemic',\n",
       "  ['During the Taiwan epidemic of severe acute respiratory syndrome (SARS), many health care workers were infected.',\n",
       "   'In an effort to establish a screening program for high risk personal, the distribution of HLA class I and II alleles in case and control groups was examined for the presence of an association to a genetic susceptibly or resistance to SARS coronavirus infection.']),\n",
       " ('HLA class I',\n",
       "  'subclass of',\n",
       "  'alleles',\n",
       "  ['During the Taiwan epidemic of severe acute respiratory syndrome (SARS), many health care workers were infected.',\n",
       "   'In an effort to establish a screening program for high risk personal, the distribution of HLA class I and II alleles in case and control groups was examined for the presence of an association to a genetic susceptibly or resistance to SARS coronavirus infection.']),\n",
       " ('SARS coronavirus',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['METHODS: HLA-class I and II allele typing by PCR-SSOP was performed on 37 cases of probable SARS, 28 fever patients excluded later as probable SARS, and 101 non-infected health care workers who were exposed or possibly exposed to SARS coronavirus.',\n",
       "   'An additional control set of 190 normal healthy unrelated Taiwanese was also used in the analysis.']),\n",
       " ('normal',\n",
       "  'subclass of',\n",
       "  'control set',\n",
       "  ['METHODS: HLA-class I and II allele typing by PCR-SSOP was performed on 37 cases of probable SARS, 28 fever patients excluded later as probable SARS, and 101 non-infected health care workers who were exposed or possibly exposed to SARS coronavirus.',\n",
       "   'An additional control set of 190 normal healthy unrelated Taiwanese was also used in the analysis.']),\n",
       " ('HLA',\n",
       "  'subclass of',\n",
       "  'alleles',\n",
       "  ['RESULTS: Woolf and Haldane Odds ratio (OR) and corrected P-value (Pc) obtained from two tails Fisher exact test were used to show susceptibility of HLA class I or class II alleles with coronavirus infection.',\n",
       "   'At first, when analyzing infected SARS patients and high risk health care workers groups, HLA-B*4601 (OR = 2.08, P = 0.04, Pc = n.s.) and HLA-B*5401 (OR = 5.44, P = 0.02, Pc = n.s.) appeared as the most probable elements that may be favoring SARS coronavirus infection.']),\n",
       " ('HLA-B*5401',\n",
       "  'has effect',\n",
       "  'SARS coronavirus infection',\n",
       "  ['RESULTS: Woolf and Haldane Odds ratio (OR) and corrected P-value (Pc) obtained from two tails Fisher exact test were used to show susceptibility of HLA class I or class II alleles with coronavirus infection.',\n",
       "   'At first, when analyzing infected SARS patients and high risk health care workers groups, HLA-B*4601 (OR = 2.08, P = 0.04, Pc = n.s.) and HLA-B*5401 (OR = 5.44, P = 0.02, Pc = n.s.) appeared as the most probable elements that may be favoring SARS coronavirus infection.']),\n",
       " ('HLA-B*4601',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['After selecting only a \"severe cases\" patient group from the infected \"probable SARS\" patient group and comparing them with the high risk health care workers group, the severity of SARS was shown to be significantly associated with HLA-B*4601 (P = 0.0008 or Pc = 0.0279).',\n",
       "   'CONCLUSIONS: Densely populated regions with genetically related southern Asian populations appear to be more affected by the spreading of SARS infection.']),\n",
       " ('SARS',\n",
       "  'location',\n",
       "  'southern Asia',\n",
       "  ['After selecting only a \"severe cases\" patient group from the infected \"probable SARS\" patient group and comparing them with the high risk health care workers group, the severity of SARS was shown to be significantly associated with HLA-B*4601 (P = 0.0008 or Pc = 0.0279).',\n",
       "   'CONCLUSIONS: Densely populated regions with genetically related southern Asian populations appear to be more affected by the spreading of SARS infection.']),\n",
       " ('Taiwan indigenous peoples',\n",
       "  'different from',\n",
       "  'Taiwanese general population',\n",
       "  ['Up until recently, no probable SARS patients were reported among Taiwan indigenous peoples who are genetically distinct from the Taiwanese general population, have no HLA-B* 4601 and have high frequency of HLA-B* 1301.',\n",
       "   'While increase of HLA-B* 4601 allele frequency was observed in the \"Probable SARS infected\" patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group.']),\n",
       " ('Taiwanese general population',\n",
       "  'different from',\n",
       "  'Taiwan indigenous peoples',\n",
       "  ['Up until recently, no probable SARS patients were reported among Taiwan indigenous peoples who are genetically distinct from the Taiwanese general population, have no HLA-B* 4601 and have high frequency of HLA-B* 1301.',\n",
       "   'While increase of HLA-B* 4601 allele frequency was observed in the \"Probable SARS infected\" patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group.']),\n",
       " ('HLA-B* 4601',\n",
       "  'instance of',\n",
       "  'allele',\n",
       "  ['Up until recently, no probable SARS patients were reported among Taiwan indigenous peoples who are genetically distinct from the Taiwanese general population, have no HLA-B* 4601 and have high frequency of HLA-B* 1301.',\n",
       "   'While increase of HLA-B* 4601 allele frequency was observed in the \"Probable SARS infected\" patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group.']),\n",
       " ('HLA-B* 4601',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations.(?<=[.?!])\\\\s+Independent studies are needed to test these results.',\n",
       "   'patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group.']),\n",
       " ('Severe cases',\n",
       "  'instance of',\n",
       "  'patient group',\n",
       "  ['These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations.(?<=[.?!])\\\\s+Independent studies are needed to test these results.',\n",
       "   'patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group.']),\n",
       " ('HLA-B* 4601',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations.(?<=[.?!])\\\\s+Independent studies are needed to test these results.',\n",
       "   'In late April 2003, Taiwan was stroked by an epidemic of severe acute respiratory syndrome (SARS).']),\n",
       " ('Taiwan was stroked',\n",
       "  'has cause',\n",
       "  'severe acute respiratory syndrome',\n",
       "  ['These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations.(?<=[.?!])\\\\s+Independent studies are needed to test these results.',\n",
       "   'In late April 2003, Taiwan was stroked by an epidemic of severe acute respiratory syndrome (SARS).']),\n",
       " ('severe acute respiratory syndrome',\n",
       "  'has effect',\n",
       "  'Taiwan was stroked',\n",
       "  ['These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations.(?<=[.?!])\\\\s+Independent studies are needed to test these results.',\n",
       "   'In late April 2003, Taiwan was stroked by an epidemic of severe acute respiratory syndrome (SARS).']),\n",
       " ('Taipei Municipal Hoping hospital',\n",
       "  'significant event',\n",
       "  'SARS panic',\n",
       "  ['A general SARS panic spread over the country when the Taipei Municipal Hoping hospital announced its shutdown as a preventive measure following heavy hospital infection.',\n",
       "   'Further closedown of medical services, including emergency services, appeared sporadically in other hospitals and clinics all over the country.']),\n",
       " ('emergency services',\n",
       "  'subclass of',\n",
       "  'medical services',\n",
       "  ['A general SARS panic spread over the country when the Taipei Municipal Hoping hospital announced its shutdown as a preventive measure following heavy hospital infection.',\n",
       "   'Further closedown of medical services, including emergency services, appeared sporadically in other hospitals and clinics all over the country.']),\n",
       " ('health care workers',\n",
       "  'has effect',\n",
       "  'anxiety',\n",
       "  ['This caused great anxiety among all health care workers.',\n",
       "   'On May 4, 2003, a suspected SARS patient with respiratory failure was brought to the Emergency Service of the Mackay Memorial Hospital in Taipei.']),\n",
       " ('Mackay Memorial Hospital',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Taipei',\n",
       "  ['This caused great anxiety among all health care workers.',\n",
       "   'On May 4, 2003, a suspected SARS patient with respiratory failure was brought to the Emergency Service of the Mackay Memorial Hospital in Taipei.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  '24 hours later',\n",
       "  ['The patient was admitted into SARS wards and died 24 hours later.',\n",
       "   'However, during this short period the patient most likely infected 4 health care workers and 5 individuals who were either patients or visiting family members of other patients present at the same time.']),\n",
       " ('health care workers',\n",
       "  'subclass of',\n",
       "  'patients',\n",
       "  ['The patient was admitted into SARS wards and died 24 hours later.',\n",
       "   'However, during this short period the patient most likely infected 4 health care workers and 5 individuals who were either patients or visiting family members of other patients present at the same time.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'emergency procedures',\n",
       "  ['The hospital set up emergency procedures, and most manpower was put into caring of SARS patients.',\n",
       "   'Taiwan hospitals were facing an unpredictable and invisible enemy for the first time in recent 50 years.']),\n",
       " ('hospital',\n",
       "  'country',\n",
       "  'Taiwan',\n",
       "  ['The hospital set up emergency procedures, and most manpower was put into caring of SARS patients.',\n",
       "   'Taiwan hospitals were facing an unpredictable and invisible enemy for the first time in recent 50 years.']),\n",
       " ('autoimmune disorders',\n",
       "  'subclass of',\n",
       "  'infectious diseases',\n",
       "  ['The human leukocyte antigen (HLA) system is widely used as a strategy in the search for the etiology of infectious diseases and autoimmune disorders.',\n",
       "   'In this study we investigated the HLA-class I and II system for evidence of disease association.']),\n",
       " ('HLA-class I',\n",
       "  'followed by',\n",
       "  'II system',\n",
       "  ['The human leukocyte antigen (HLA) system is widely used as a strategy in the search for the etiology of infectious diseases and autoimmune disorders.',\n",
       "   'In this study we investigated the HLA-class I and II system for evidence of disease association.']),\n",
       " ('disease protection phenotype',\n",
       "  'subclass of',\n",
       "  'disease susceptibility',\n",
       "  ['It was hoped that the discovery of disease susceptibility or a disease protection phenotype in the Taiwan population would help into setting up a preventive screening program for health care workers at risk.',\n",
       "   'In the course of March 7, 13, 18, 19, 21, 22, 26, 27 , and 31) were infected from the same source (unfortunately this \"super-spreader\" probable SARS patient died 24 hours after admission in hospital and was not studied).']),\n",
       " ('probable SARS',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['It was hoped that the discovery of disease susceptibility or a disease protection phenotype in the Taiwan population would help into setting up a preventive screening program for health care workers at risk.',\n",
       "   'In the course of March 7, 13, 18, 19, 21, 22, 26, 27 , and 31) were infected from the same source (unfortunately this \"super-spreader\" probable SARS patient died 24 hours after admission in hospital and was not studied).']),\n",
       " ('excluded fever patients',\n",
       "  'has cause',\n",
       "  'SARS',\n",
       "  ['Twenty eight cases of the 65 cases were not positive for SARS, and were referred as \"Excluded fever patients\" in tables 2 and 4.',\n",
       "   'Among the \"Excluded fever patients\" group, two patients showed positive serological tests for M.']),\n",
       " ('positive',\n",
       "  'subclass of',\n",
       "  'serological test',\n",
       "  ['Twenty eight cases of the 65 cases were not positive for SARS, and were referred as \"Excluded fever patients\" in tables 2 and 4.',\n",
       "   'Among the \"Excluded fever patients\" group, two patients showed positive serological tests for M.']),\n",
       " ('tuberculosis',\n",
       "  'subclass of',\n",
       "  'lung lesions',\n",
       "  ['pneumonias, one had tuberculosis, one patient presented lung edema, and the rest had no lung lesions.',\n",
       "   'Clinical symptoms, of the 37 patients diagnosed as probable SARS cases, were as follows: 92% (34 cases) showed diffuse infiltration of both lungs and 8% (3 cases) showed only solitary lung lesion; 65% (24 cases) showed at least one positive test result for SARS-CoV RNA by RT-PCR [2] or by real time PCR (Artus, Germany).']),\n",
       " ('Artus',\n",
       "  'country',\n",
       "  'Germany',\n",
       "  ['pneumonias, one had tuberculosis, one patient presented lung edema, and the rest had no lung lesions.',\n",
       "   'Clinical symptoms, of the 37 patients diagnosed as probable SARS cases, were as follows: 92% (34 cases) showed diffuse infiltration of both lungs and 8% (3 cases) showed only solitary lung lesion; 65% (24 cases) showed at least one positive test result for SARS-CoV RNA by RT-PCR [2] or by real time PCR (Artus, Germany).']),\n",
       " ('LDH',\n",
       "  'subclass of',\n",
       "  'AST',\n",
       "  ['In addition, the following laboratory data was obtained from the 35 probable SARS patients managed in our hospital: 27% (10 cases) with reduced platelet count (<150×10 -9 /liter), 59% (22 cases) with lymphocytopenia (<1×10 -9 /liter), 70% (26 cases) with elevated LDH, 5% (2 cases) with elevated creatine kinase, 54% (20 cases) with elevated AST and /or ALT and 68% (25 cases) with elevated C-reactive protein.',\n",
       "   'Fourteen cases were also tested for HBsAg, all showed negative results.(?<=[.?!])\\\\s+Serum samples from 27 probable SARS cases were tested for Chlamydia pneumoniae, C.(?<=[.?!])\\\\s+psittaci and C.']),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'Chlamydia pneumoniae',\n",
       "  ['In addition, the following laboratory data was obtained from the 35 probable SARS patients managed in our hospital: 27% (10 cases) with reduced platelet count (<150×10 -9 /liter), 59% (22 cases) with lymphocytopenia (<1×10 -9 /liter), 70% (26 cases) with elevated LDH, 5% (2 cases) with elevated creatine kinase, 54% (20 cases) with elevated AST and /or ALT and 68% (25 cases) with elevated C-reactive protein.',\n",
       "   'Fourteen cases were also tested for HBsAg, all showed negative results.(?<=[.?!])\\\\s+Serum samples from 27 probable SARS cases were tested for Chlamydia pneumoniae, C.(?<=[.?!])\\\\s+psittaci and C.']),\n",
       " ('Savyon Diagnostic',\n",
       "  'country',\n",
       "  'Israel',\n",
       "  ['psittaci and C.(?<=[.?!])\\\\s+trachomatis (Chlamydia IgM kit, Savyon Diagnostic, Israel), and 11 were further tested for IgG and IgA antibodies.(?<=[.?!])\\\\s+All showed negative results.',\n",
       "   'Among 24 probable SARS cases who had been tested for M.(?<=[.?!])\\\\s+pneumoniae (IgM, Savyon Diagnostic, Israel), only two cases tested positive.']),\n",
       " ('Savyon Diagnostic',\n",
       "  'country',\n",
       "  'Israel',\n",
       "  ['psittaci and C.(?<=[.?!])\\\\s+trachomatis (Chlamydia IgM kit, Savyon Diagnostic, Israel), and 11 were further tested for IgG and IgA antibodies.(?<=[.?!])\\\\s+All showed negative results.',\n",
       "   'Among 24 probable SARS cases who had been tested for M.(?<=[.?!])\\\\s+pneumoniae (IgM, Savyon Diagnostic, Israel), only two cases tested positive.']),\n",
       " ('SARS-CoV RNA test',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['These two cases also had diffuse bilateral lung lesions and positive SARS-CoV RNA test.(?<=[.?!])\\\\s+It is possible that 8% of probable SARS cases were co-infected with M.(?<=[.?!])\\\\s+pneumoniae.',\n",
       "   'pneumoniae.(?<=[.?!])\\\\s+Finally, 15 cases were also tested for Legionella pneumophilia serogroup 1 antigen (Binax, US), and showed negative results.']),\n",
       " ('Binax',\n",
       "  'instance of',\n",
       "  'antigen',\n",
       "  ['These two cases also had diffuse bilateral lung lesions and positive SARS-CoV RNA test.(?<=[.?!])\\\\s+It is possible that 8% of probable SARS cases were co-infected with M.(?<=[.?!])\\\\s+pneumoniae.',\n",
       "   'pneumoniae.(?<=[.?!])\\\\s+Finally, 15 cases were also tested for Legionella pneumophilia serogroup 1 antigen (Binax, US), and showed negative results.']),\n",
       " ('suspected probable case',\n",
       "  'subclass of',\n",
       "  'SARS',\n",
       "  ['All 37 probable cases revealed history of close or indirect contact with another suspected probable case of SARS.',\n",
       "   'Three patients with severe respiratory failure airways had been intubated and mechanical ventilation was applied during hospitalization.(?<=[.?!])\\\\s+Later, two patients died and one survived.']),\n",
       " ('intubated',\n",
       "  'has part',\n",
       "  'mechanical ventilation',\n",
       "  ['All 37 probable cases revealed history of close or indirect contact with another suspected probable case of SARS.',\n",
       "   'Three patients with severe respiratory failure airways had been intubated and mechanical ventilation was applied during hospitalization.(?<=[.?!])\\\\s+Later, two patients died and one survived.']),\n",
       " ('hospitalization',\n",
       "  'has effect',\n",
       "  'death',\n",
       "  ['Finally, among the 35 probable SARS cases of our hospital four patients died during hospitalization, giving an 11% mortality rate.',\n",
       "   'Except for two rapid severe/ fatal cases most probable SARS patients were admitted to the SARS wards for periods longer than two weeks.']),\n",
       " ('rapid severe',\n",
       "  'subclass of',\n",
       "  'fatal cases',\n",
       "  ['Finally, among the 35 probable SARS cases of our hospital four patients died during hospitalization, giving an 11% mortality rate.',\n",
       "   'Except for two rapid severe/ fatal cases most probable SARS patients were admitted to the SARS wards for periods longer than two weeks.']),\n",
       " ('probable SARS',\n",
       "  'instance of',\n",
       "  'diagnosis',\n",
       "  ['A final diagnosis of \"probable SARS\" was declared upon exclusion of all other possible causes.',\n",
       "   'In brief, 33 cases of \"probable SARS patients\" and 28 \"Excluded fever patients\" were retained for the analysis.']),\n",
       " ('Excluded fever patients',\n",
       "  'subclass of',\n",
       "  'SARS',\n",
       "  ['A final diagnosis of \"probable SARS\" was declared upon exclusion of all other possible causes.',\n",
       "   'In brief, 33 cases of \"probable SARS patients\" and 28 \"Excluded fever patients\" were retained for the analysis.']),\n",
       " ('Mackay Memorial Hospital',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Taipei',\n",
       "  ['This study was performed with approval of the ethics committee of the Mackay Memorial Hospital in Taipei.',\n",
       "   'Control A Blood samples were obtained at the end of May 2003 from 62 health care workers who had possibly been exposed to coronavirus and were not infected.']),\n",
       " ('coronavirus',\n",
       "  'has effect',\n",
       "  'Control A Blood',\n",
       "  ['This study was performed with approval of the ethics committee of the Mackay Memorial Hospital in Taipei.',\n",
       "   'Control A Blood samples were obtained at the end of May 2003 from 62 health care workers who had possibly been exposed to coronavirus and were not infected.']),\n",
       " ('nurses',\n",
       "  'subclass of',\n",
       "  'health care workers',\n",
       "  ['These 62 health care workers were either working at emergency services, SARS wards (doctors and nurses) or at the out patient clinic (phlebotomists) of our hospital in May 2003.',\n",
       "   'These 62 health care workers were included in control A (table 2) and had followed World Health Organization (WHO) vigorous measures against infection.']),\n",
       " ('vigorous measures against infection',\n",
       "  'creator',\n",
       "  'World Health Organization',\n",
       "  ['These 62 health care workers were either working at emergency services, SARS wards (doctors and nurses) or at the out patient clinic (phlebotomists) of our hospital in May 2003.',\n",
       "   'These 62 health care workers were included in control A (table 2) and had followed World Health Organization (WHO) vigorous measures against infection.']),\n",
       " ('Hoping Hospital',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Taipei',\n",
       "  ['Another 39 blood samples from non-infected health care workers from the Taipei Municipal Hoping Hospital were added to control A.',\n",
       "   'These 39 controls had been treating SARS patients in April 2003 without adequate protection from infection, and about 10% of health care workers from the same wards developed probable SARS.']),\n",
       " ('SARS',\n",
       "  'start time',\n",
       "  'April 2003',\n",
       "  ['Another 39 blood samples from non-infected health care workers from the Taipei Municipal Hoping Hospital were added to control A.',\n",
       "   'These 39 controls had been treating SARS patients in April 2003 without adequate protection from infection, and about 10% of health care workers from the same wards developed probable SARS.']),\n",
       " ('non-infected',\n",
       "  'subclass of',\n",
       "  'health care workers',\n",
       "  ['In brief, 101 high risk non-infected health care workers were retained as control A for the analysis.(?<=[.?!])\\\\s+Informed consent was obtained from all subjects.',\n",
       "   'HLA-A, B and DRB1 typing of 190 normal healthy, unrelated Taiwanese was obtained from data collected in our laboratory and used as control B.']),\n",
       " ('DRB1',\n",
       "  'subclass of',\n",
       "  'B',\n",
       "  ['In brief, 101 high risk non-infected health care workers were retained as control A for the analysis.(?<=[.?!])\\\\s+Informed consent was obtained from all subjects.',\n",
       "   'HLA-A, B and DRB1 typing of 190 normal healthy, unrelated Taiwanese was obtained from data collected in our laboratory and used as control B.']),\n",
       " ('allele typing',\n",
       "  'subclass of',\n",
       "  'serological methods',\n",
       "  ['For control B, HLA-A, B class I typing was performed by serological methods only [3] .(?<=[.?!])\\\\s+HLA-DRB1 typing was performed by allele typing as described below for the other groups.',\n",
       "   'Blood samples were collected in EDTA vacutainers.']),\n",
       " ('EDTA',\n",
       "  'subclass of',\n",
       "  'vacutainers',\n",
       "  ['For control B, HLA-A, B class I typing was performed by serological methods only [3] .(?<=[.?!])\\\\s+HLA-DRB1 typing was performed by allele typing as described below for the other groups.',\n",
       "   'Blood samples were collected in EDTA vacutainers.']),\n",
       " ('Hilden',\n",
       "  'country',\n",
       "  'Germany',\n",
       "  [\"Buffy coat were lysed and genomic DNA was isolated by using proteinase K treatment and Q1Aamp blood kit (Q1Agen, Hilden, Germany) according to the manufacturer's instructions.\",\n",
       "   'Suspected SARS samples were processed in a bio-safety level-3 environment.']),\n",
       " ('bio-safety level-3',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  [\"Buffy coat were lysed and genomic DNA was isolated by using proteinase K treatment and Q1Aamp blood kit (Q1Agen, Hilden, Germany) according to the manufacturer's instructions.\",\n",
       "   'Suspected SARS samples were processed in a bio-safety level-3 environment.']),\n",
       " ('Wirral',\n",
       "  'country',\n",
       "  'U.K.',\n",
       "  ['Medium resolution allele typing for HLA-A, B and DRB1 loci was performed using PCR amplification followed by sequence-specific oligonucleotide probing (PCR-SSOP), (Dynal Biotech Ltd; Wirral, U.K.).',\n",
       "   'Amplified sequences were hybridized to arrays of immobilized probes (35 probes for HLA-A, 56 for HLA-B, and 38 for HLA-DRB1).']),\n",
       " ('HLA-A',\n",
       "  'followed by',\n",
       "  'HLA-B',\n",
       "  ['Medium resolution allele typing for HLA-A, B and DRB1 loci was performed using PCR amplification followed by sequence-specific oligonucleotide probing (PCR-SSOP), (Dynal Biotech Ltd; Wirral, U.K.).',\n",
       "   'Amplified sequences were hybridized to arrays of immobilized probes (35 probes for HLA-A, 56 for HLA-B, and 38 for HLA-DRB1).']),\n",
       " ('class I',\n",
       "  'instance of',\n",
       "  'allele frequencies',\n",
       "  ['HLA class I and class II allele frequencies were estimated by direct counting assuming that there was no blanks.',\n",
       "   \"Odds ratios (OR) were obtained from standard contingency table analysis using Haldane's modification of Woolf's method [4, 5] .\"]),\n",
       " ('class II',\n",
       "  'instance of',\n",
       "  'allele frequencies',\n",
       "  ['HLA class I and class II allele frequencies were estimated by direct counting assuming that there was no blanks.',\n",
       "   \"Odds ratios (OR) were obtained from standard contingency table analysis using Haldane's modification of Woolf's method [4, 5] .\"]),\n",
       " (\"Woolf's method\",\n",
       "  'use',\n",
       "  'contingency table analysis',\n",
       "  ['HLA class I and class II allele frequencies were estimated by direct counting assuming that there was no blanks.',\n",
       "   \"Odds ratios (OR) were obtained from standard contingency table analysis using Haldane's modification of Woolf's method [4, 5] .\"]),\n",
       " (\"Fisher's exact test\",\n",
       "  'instance of',\n",
       "  'Statistical significance',\n",
       "  [\"Statistical significance was performed by a two tails Fisher's exact test [6] .\",\n",
       "   'Since small size samples were used in the analysis, the risk of introducing a bias in estimating the probability (P-value) and wrongly accepting association (type I error) was corrected using the method of Edwards (7) .']),\n",
       " ('type I error',\n",
       "  'subclass of',\n",
       "  'bias',\n",
       "  [\"Statistical significance was performed by a two tails Fisher's exact test [6] .\",\n",
       "   'Since small size samples were used in the analysis, the risk of introducing a bias in estimating the probability (P-value) and wrongly accepting association (type I error) was corrected using the method of Edwards (7) .']),\n",
       " ('corrected P-value',\n",
       "  'subclass of',\n",
       "  'Pvalue',\n",
       "  ['In this method the Pvalue is multiplied by the number of independent comparisons at every locus, thereby giving a more powerful corrected P-value (Pc-value) that may be interpreted with confidence.',\n",
       "   'Lastly, when analyzing serological data, data in the patient group were converted to serology typing using the World Health Organization nomenclature (WHO) table of correspondence [8] .']),\n",
       " ('table of correspondence',\n",
       "  'publisher',\n",
       "  'World Health Organization',\n",
       "  ['In this method the Pvalue is multiplied by the number of independent comparisons at every locus, thereby giving a more powerful corrected P-value (Pc-value) that may be interpreted with confidence.',\n",
       "   'Lastly, when analyzing serological data, data in the patient group were converted to serology typing using the World Health Organization nomenclature (WHO) table of correspondence [8] .']),\n",
       " ('HLA-B',\n",
       "  'instance of',\n",
       "  'gene',\n",
       "  [\"In this study we performed allele typing on 37 probable SARS cases, 28 fever patients of suspected SARS (these 28 patients were later excluded for being non SARS patients), and 101 health care workers (control A) exposed or possibly exposed to SARS coronavirus but non-infected health  HLA-B gene was found to be associated with susceptibility or protection from infection when analyzing descendants of Dutch colonists' survivors of 19 th Century yellow fever and typhoid epidemics [10] .\",\n",
       "   'Other studies also indicated that HLA variations were associated with susceptibility or resistance to malaria, tuberculosis, leprosy, HIV and hepatitis virus persistence [11, 12] .']),\n",
       " ('hepatitis virus',\n",
       "  'has effect',\n",
       "  'HIV',\n",
       "  [\"In this study we performed allele typing on 37 probable SARS cases, 28 fever patients of suspected SARS (these 28 patients were later excluded for being non SARS patients), and 101 health care workers (control A) exposed or possibly exposed to SARS coronavirus but non-infected health  HLA-B gene was found to be associated with susceptibility or protection from infection when analyzing descendants of Dutch colonists' survivors of 19 th Century yellow fever and typhoid epidemics [10] .\",\n",
       "   'Other studies also indicated that HLA variations were associated with susceptibility or resistance to malaria, tuberculosis, leprosy, HIV and hepatitis virus persistence [11, 12] .']),\n",
       " ('HLA-A',\n",
       "  'instance of',\n",
       "  'alleles',\n",
       "  ['In ethnic Thai, HLA-A and B alleles were confirmed to be associated with disease severity and clinical outcome of exposure to dengue virus in previously exposed immunologically primed individuals [13] .',\n",
       "   'Moreover, direct evidence of human coronavirus OC43 interaction with HLA class I molecules at the cell surface to establish infection was also described [14] .']),\n",
       " ('B',\n",
       "  'instance of',\n",
       "  'alleles',\n",
       "  ['In ethnic Thai, HLA-A and B alleles were confirmed to be associated with disease severity and clinical outcome of exposure to dengue virus in previously exposed immunologically primed individuals [13] .',\n",
       "   'Moreover, direct evidence of human coronavirus OC43 interaction with HLA class I molecules at the cell surface to establish infection was also described [14] .']),\n",
       " ('OC43',\n",
       "  'subclass of',\n",
       "  'coronavirus',\n",
       "  ['In ethnic Thai, HLA-A and B alleles were confirmed to be associated with disease severity and clinical outcome of exposure to dengue virus in previously exposed immunologically primed individuals [13] .',\n",
       "   'Moreover, direct evidence of human coronavirus OC43 interaction with HLA class I molecules at the cell surface to establish infection was also described [14] .']),\n",
       " ('SARS coronavirus infection',\n",
       "  'has cause',\n",
       "  'HLA class I',\n",
       "  ['Consequently we carried out a study to analyze the relation between HLA class I and class II alleles, in SARS coronavirus infection, and/or the severity of the disease.',\n",
       "   'In this study, no association between HLA class II allele and SARS coronavirus infection was seen.']),\n",
       " ('HLA class II',\n",
       "  'instance of',\n",
       "  'allele',\n",
       "  ['Consequently we carried out a study to analyze the relation between HLA class I and class II alleles, in SARS coronavirus infection, and/or the severity of the disease.',\n",
       "   'In this study, no association between HLA class II allele and SARS coronavirus infection was seen.']),\n",
       " ('HLA-B*4601',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['Although HLA-B* 4601 frequency in the 33 probable SARS patients (table 2) was only slightly higher than the frequencies seen in control A or control B (P = 0.04, Pc = n.s.',\n",
       "   'and P = 0.06, Pc = n.s., respectively).']),\n",
       " ('Pc',\n",
       "  'has part',\n",
       "  'n',\n",
       "  ['Although HLA-B* 4601 frequency in the 33 probable SARS patients (table 2) was only slightly higher than the frequencies seen in control A or control B (P = 0.04, Pc = n.s.',\n",
       "   'and P = 0.06, Pc = n.s., respectively).']),\n",
       " ('severe cases',\n",
       "  'subclass of',\n",
       "  'fever',\n",
       "  ['On the contrary, the frequency of HLA-B* 4601 in the five severe cases (table 4) was slightly increased when compared to the 28 excluded fever patients (P = 0.007, Pc = ns), but the difference was clearly significant when compared to control A (P = 0.0008, Pc = 0.03).',\n",
       "   'This increased HLA-B* 4601 difference was unique to the five severe cases patients and may indicate association with the severity of SARS.']),\n",
       " ('HLA-B* 4601',\n",
       "  'has cause',\n",
       "  'SARS',\n",
       "  ['On the contrary, the frequency of HLA-B* 4601 in the five severe cases (table 4) was slightly increased when compared to the 28 excluded fever patients (P = 0.007, Pc = ns), but the difference was clearly significant when compared to control A (P = 0.0008, Pc = 0.03).',\n",
       "   'This increased HLA-B* 4601 difference was unique to the five severe cases patients and may indicate association with the severity of SARS.']),\n",
       " ('B* 3901/B39',\n",
       "  'has part',\n",
       "  'HLA-B* 5401/B54',\n",
       "  ['A susceptibly of HLA-B* 5401/B54 or B* 3901/B39 to infection rather than to the severity of the disease (table 2) could not be concluded upon (Pc = n.s.) and requires further studies.',\n",
       "   'HLA-B13 is widely distributed throughout Asia [18] where it is mostly represented by HLA-B* 1301 (4-30%) and rarely by B* 1302 (0-2%) [Unpublished data].']),\n",
       " ('HLA-B13',\n",
       "  'instance of',\n",
       "  'HLA-B* 1301',\n",
       "  ['A susceptibly of HLA-B* 5401/B54 or B* 3901/B39 to infection rather than to the severity of the disease (table 2) could not be concluded upon (Pc = n.s.) and requires further studies.',\n",
       "   'HLA-B13 is widely distributed throughout Asia [18] where it is mostly represented by HLA-B* 1301 (4-30%) and rarely by B* 1302 (0-2%) [Unpublished data].']),\n",
       " ('P-value',\n",
       "  'subclass of',\n",
       "  'significant',\n",
       "  ['Furthermore, HLA-B13 is seen with frequencies greater than 16% among most n= count of alleles; P = P-value; n.s.(?<=[.?!])\\\\s+= not significant.',\n",
       "   '= not significant.(?<=[.?!])\\\\s+† The correction factor (number of different alleles or number of comparisons) is shown within parenthesis; Pc-value = P × Correction factor (Edward, 1974) .']),\n",
       " ('Pc-value',\n",
       "  'facet of',\n",
       "  'correction factor',\n",
       "  ['Furthermore, HLA-B13 is seen with frequencies greater than 16% among most n= count of alleles; P = P-value; n.s.(?<=[.?!])\\\\s+= not significant.',\n",
       "   '= not significant.(?<=[.?!])\\\\s+† The correction factor (number of different alleles or number of comparisons) is shown within parenthesis; Pc-value = P × Correction factor (Edward, 1974) .']),\n",
       " ('central mountain area',\n",
       "  'country',\n",
       "  'Taiwan',\n",
       "  ['indigenous tribes of the central mountain area in Taiwan [9] and in the Taiwanese (9.1%).',\n",
       "   'In this study, we found that presence of HLA-B* 1301/B13 was decreased in the 33 probable SARS patients when compared with the two control groups A (OR = 0.16, P = 0.03, Pc = n.s.) or B (OR = 0.16, P = 0.02, Pc = n.s.).']),\n",
       " ('B13',\n",
       "  'subclass of',\n",
       "  'HLA-B*',\n",
       "  ['indigenous tribes of the central mountain area in Taiwan [9] and in the Taiwanese (9.1%).',\n",
       "   'In this study, we found that presence of HLA-B* 1301/B13 was decreased in the 33 probable SARS patients when compared with the two control groups A (OR = 0.16, P = 0.03, Pc = n.s.) or B (OR = 0.16, P = 0.02, Pc = n.s.).']),\n",
       " ('coronavirus infection',\n",
       "  'has cause',\n",
       "  'HLA-B* 1301',\n",
       "  ['However, only the analysis of a larger case-control data set would confirm or refute association of HLA-B* 1301 with resistance to coronavirus infection.',\n",
       "   'Taiwan population is heterogeneous and comprises 91% Taiwanese, 7.5% Chinese who arrived from many different provinces of China after World War II, and 1.5% of indigenous peoples established there since several millenniums.']),\n",
       " ('Taiwan',\n",
       "  'ethnic group',\n",
       "  'Taiwanese',\n",
       "  ['However, only the analysis of a larger case-control data set would confirm or refute association of HLA-B* 1301 with resistance to coronavirus infection.',\n",
       "   'Taiwan population is heterogeneous and comprises 91% Taiwanese, 7.5% Chinese who arrived from many different provinces of China after World War II, and 1.5% of indigenous peoples established there since several millenniums.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Fujian',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Guangdong',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('Fujian',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('Guangdong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('China',\n",
       "  'ethnic group',\n",
       "  'Singapore Chinese',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('China',\n",
       "  'ethnic group',\n",
       "  'Thai Chinese',\n",
       "  ['The Taiwanese comprised the Minnan and Hakka people groups, and are the descendants of early settlers from the southeast coast of China (Fujian and Guangdong Provinces) during the last few centuries.',\n",
       "   'The genetic profile of the Taiwanese shows many affinities to southern Asian populations [3] and share the same origin with Singapore Chinese and Thai Chinese who also originated from the southeast coast of China.']),\n",
       " ('Hong River',\n",
       "  'country',\n",
       "  'Vietnam',\n",
       "  ['According to archeological evidences [15] , the populations in the delta of the Hong River in northern Vietnam are also likely related to these populations.',\n",
       "   'After the outbreak of SARS coronavirus infection in the Guangdong Province of China, it was surprising to observe that the spreading of the disease was mostly confined among southern Asian populations (Hong-Kong peoples, Vietnamese, Singaporeans and Taiwanese).']),\n",
       " ('Guangdong Province',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['According to archeological evidences [15] , the populations in the delta of the Hong River in northern Vietnam are also likely related to these populations.',\n",
       "   'After the outbreak of SARS coronavirus infection in the Guangdong Province of China, it was surprising to observe that the spreading of the disease was mostly confined among southern Asian populations (Hong-Kong peoples, Vietnamese, Singaporeans and Taiwanese).']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Guangdong Province',\n",
       "  ['According to archeological evidences [15] , the populations in the delta of the Hong River in northern Vietnam are also likely related to these populations.',\n",
       "   'After the outbreak of SARS coronavirus infection in the Guangdong Province of China, it was surprising to observe that the spreading of the disease was mostly confined among southern Asian populations (Hong-Kong peoples, Vietnamese, Singaporeans and Taiwanese).']),\n",
       " ('population',\n",
       "  'has part',\n",
       "  'density',\n",
       "  ['Density of populations possibly favored the growth of the epidemic.',\n",
       "   'The question aroused as to whether affected southern Asian populations were more susceptible to SARS infection than peoples of northern China (Beijing), Japan, or Korea.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Beijing',\n",
       "  ['Density of populations possibly favored the growth of the epidemic.',\n",
       "   'The question aroused as to whether affected southern Asian populations were more susceptible to SARS infection than peoples of northern China (Beijing), Japan, or Korea.']),\n",
       " ('Beijing',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Density of populations possibly favored the growth of the epidemic.',\n",
       "   'The question aroused as to whether affected southern Asian populations were more susceptible to SARS infection than peoples of northern China (Beijing), Japan, or Korea.']),\n",
       " ('Cultural Revolution',\n",
       "  'location',\n",
       "  'Beijing',\n",
       "  ['Many peoples in Beijing generally considered as northern Chinese [16] , were also infected, however, infection in Beijing could be attributed to heterogeneity of populations following important recent migration of southern Chinese and population resettlement during the Cultural Revolution.',\n",
       "   'The probable SARS cases in this report consist mainly of southern Chinese origin Taiwanese (non indigenous peoples).']),\n",
       " ('Taiwanese',\n",
       "  'subclass of',\n",
       "  'indigenous peoples',\n",
       "  ['Many peoples in Beijing generally considered as northern Chinese [16] , were also infected, however, infection in Beijing could be attributed to heterogeneity of populations following important recent migration of southern Chinese and population resettlement during the Cultural Revolution.',\n",
       "   'The probable SARS cases in this report consist mainly of southern Chinese origin Taiwanese (non indigenous peoples).']),\n",
       " ('Taiwan indigenous',\n",
       "  'country',\n",
       "  'Taiwan',\n",
       "  ['Contraction of SARS by Taiwan indigenous individuals from any of the nine tribes of Taiwan has not yet been reported to this day.',\n",
       "   'It is possible that the HLA make up of the Taiwan indigenous peoples has little in common to the Taiwanese [9] and may not contain any element favoring SARS infection.']),\n",
       " ('Taiwan indigenous peoples',\n",
       "  'different from',\n",
       "  'Taiwanese',\n",
       "  ['Contraction of SARS by Taiwan indigenous individuals from any of the nine tribes of Taiwan has not yet been reported to this day.',\n",
       "   'It is possible that the HLA make up of the Taiwan indigenous peoples has little in common to the Taiwanese [9] and may not contain any element favoring SARS infection.']),\n",
       " ('Taiwanese',\n",
       "  'different from',\n",
       "  'Taiwan indigenous peoples',\n",
       "  ['Contraction of SARS by Taiwan indigenous individuals from any of the nine tribes of Taiwan has not yet been reported to this day.',\n",
       "   'It is possible that the HLA make up of the Taiwan indigenous peoples has little in common to the Taiwanese [9] and may not contain any element favoring SARS infection.']),\n",
       " ('HLA-B* 4601/B46',\n",
       "  'instance of',\n",
       "  'allele',\n",
       "  ['Also, HLA-B* 4601/B46 displays higher frequencies in Southern Han (15.4%), in Singaporean (15.1%) and in Vietnamese (13.2%) than in Northern Han (2.8%) [18] , but the allele has rarely been seen among indigenous peoples [9] except in children of intermarriage between Taiwanese and indigenous peoples [9] .',\n",
       "   'Interestingly, HLA-B* 4601/B46 is also seldom seen in Europeans populations where very few cases of SARS infection of individuals of European origin were reported [17] .']),\n",
       " ('HLA-B* 4601',\n",
       "  'has part',\n",
       "  'B46',\n",
       "  ['Also, HLA-B* 4601/B46 displays higher frequencies in Southern Han (15.4%), in Singaporean (15.1%) and in Vietnamese (13.2%) than in Northern Han (2.8%) [18] , but the allele has rarely been seen among indigenous peoples [9] except in children of intermarriage between Taiwanese and indigenous peoples [9] .',\n",
       "   'Interestingly, HLA-B* 4601/B46 is also seldom seen in Europeans populations where very few cases of SARS infection of individuals of European origin were reported [17] .']),\n",
       " ('SARS epidemic',\n",
       "  'has cause',\n",
       "  'HLA-B* 4601',\n",
       "  ['The presence or absence of HLA-B* 4601 in a population appeared to be an important element that acted during the outbreak of the SARS epidemic.',\n",
       "   'Certainly, further independent studies are still needed to confirm this hypothesis.(?<=[.?!])\\\\s+However, Taiwan indigenous peoples have no HLA-B* 4601 and have very low population density.']),\n",
       " ('indigenous peoples',\n",
       "  'country',\n",
       "  'Taiwan',\n",
       "  ['The presence or absence of HLA-B* 4601 in a population appeared to be an important element that acted during the outbreak of the SARS epidemic.',\n",
       "   'Certainly, further independent studies are still needed to confirm this hypothesis.(?<=[.?!])\\\\s+However, Taiwan indigenous peoples have no HLA-B* 4601 and have very low population density.']),\n",
       " ('urbanized',\n",
       "  'country',\n",
       "  'Taiwan',\n",
       "  ['This does not exclude that they may have avoided infection only by chance, even though many indigenous peoples are now urbanized and would have run the same risk as Taiwanese.',\n",
       "   'Southern Asian peoples, not only live in highly densely populated regions, but also have less variation in their HLA related immune repertoires than European or African populations [Variation was inferred here from the heterozygosity (h = 1 -sum of squares of allele frequencies) seen in these populations, and h was obtained and calculated (data not shown) from published data [18] ], which as a whole creates a favorable factor for rapid settling of any epidemic.']),\n",
       " ('HLA',\n",
       "  'subclass of',\n",
       "  'immune repertoires',\n",
       "  ['This does not exclude that they may have avoided infection only by chance, even though many indigenous peoples are now urbanized and would have run the same risk as Taiwanese.',\n",
       "   'Southern Asian peoples, not only live in highly densely populated regions, but also have less variation in their HLA related immune repertoires than European or African populations [Variation was inferred here from the heterozygosity (h = 1 -sum of squares of allele frequencies) seen in these populations, and h was obtained and calculated (data not shown) from published data [18] ], which as a whole creates a favorable factor for rapid settling of any epidemic.']),\n",
       " ('SARS',\n",
       "  'subclass of',\n",
       "  'coronavirus',\n",
       "  ['It becomes evident that authorities must keep a state of disease awareness at all times.(?<=[.?!])\\\\s+Little is known on the routes of infection of SARS coronavirus [19] .',\n",
       "   'In this study, the effect of HLA alleles on the resistance or susceptibility to SARS coronavirus infection was shown to be associated to class I alleles.']),\n",
       " ('class I alleles',\n",
       "  'subclass of',\n",
       "  'HLA alleles',\n",
       "  ['It becomes evident that authorities must keep a state of disease awareness at all times.(?<=[.?!])\\\\s+Little is known on the routes of infection of SARS coronavirus [19] .',\n",
       "   'In this study, the effect of HLA alleles on the resistance or susceptibility to SARS coronavirus infection was shown to be associated to class I alleles.']),\n",
       " ('class II',\n",
       "  'subclass of',\n",
       "  'alleles',\n",
       "  ['No association with class II alleles was seen.',\n",
       "   'Quickly establishing a procedure of mass screening for health care workers at risk of exposure to SARS coronavirus by detecting HLA-B* 4601 and possibly HLA-B* 1301 alleles appears justified.']),\n",
       " ('HLA-B* 1301',\n",
       "  'has part',\n",
       "  'HLA-B* 4601',\n",
       "  ['No association with class II alleles was seen.',\n",
       "   'Quickly establishing a procedure of mass screening for health care workers at risk of exposure to SARS coronavirus by detecting HLA-B* 4601 and possibly HLA-B* 1301 alleles appears justified.']),\n",
       " ('infection',\n",
       "  'location',\n",
       "  'hospital',\n",
       "  ['The results of such screening would help hospitals to better guard their personals against infection, make the workplace safer, and help to ease the tension and anxiety seen in the hospital environment.',\n",
       "   'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior.']),\n",
       " ('Guangdong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['The results of such screening would help hospitals to better guard their personals against infection, make the workplace safer, and help to ease the tension and anxiety seen in the hospital environment.',\n",
       "   'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Guangdong',\n",
       "  ['The results of such screening would help hospitals to better guard their personals against infection, make the workplace safer, and help to ease the tension and anxiety seen in the hospital environment.',\n",
       "   'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior.']),\n",
       " ('prophylactic measures',\n",
       "  'facet of',\n",
       "  'outbreak',\n",
       "  ['It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior.',\n",
       "   'RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread.']),\n",
       " ('Susceptible-Infected-Removed',\n",
       "  'instance of',\n",
       "  'model',\n",
       "  ['It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior.',\n",
       "   'RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread.']),\n",
       " ('SARS',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region.',\n",
       "   'We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses.']),\n",
       " ('tropism',\n",
       "  'facet of',\n",
       "  'virus',\n",
       "  ['We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region.',\n",
       "   'We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses.']),\n",
       " ('SARS',\n",
       "  'instance of',\n",
       "  'epidemic',\n",
       "  ['This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS.',\n",
       "   'CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS.']),\n",
       " ('first milder outbreak',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS.',\n",
       "   'CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS.']),\n",
       " ('first epidemic',\n",
       "  'has cause',\n",
       "  'SARS',\n",
       "  ['Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected.',\n",
       "   'We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected.',\n",
       "   'We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients.']),\n",
       " ('epidemic',\n",
       "  'has effect',\n",
       "  'epidemic',\n",
       "  ['Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic.',\n",
       "   'The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.']),\n",
       " ('SARS',\n",
       "  'subclass of',\n",
       "  'precursor',\n",
       "  ['Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic.',\n",
       "   'The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.']),\n",
       " ('Guangdong Province',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Guangdong Province',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('Guangzhou',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('Kow-loon',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('SAR Hong Kong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'SAR Hong Kong',\n",
       "  ['Since November 2002 (and perhaps earlier) an outbreak of a very contagious atypical pneumonia (now named Severe Acute Respiratory Syndrome) initiated in the Guangdong Province of China.',\n",
       "   'This outbreak started a world-wide epidemic after a medical doctor from Guangzhou infected several persons at an hotel in Kow-loon around February 21 st , 2003 (SAR Hong Kong, China).']),\n",
       " ('estate',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['Although apparently classical in its onset, the pattern of the outbreak became puzzling after an estate in Hong Kong was affected, with a huge number of patients infected by the virus apparently causing the disease.',\n",
       "   'In particular it appeared that underlying this highly focused outbreak there remained a more or less constant background infection level, that remained present in Guangdong for a long time and witnessed a slow decline in Hong Kong, on which superimposes sudden local outbreaks.']),\n",
       " ('Guangdong',\n",
       "  'shares border with',\n",
       "  'Hong Kong',\n",
       "  ['Although apparently classical in its onset, the pattern of the outbreak became puzzling after an estate in Hong Kong was affected, with a huge number of patients infected by the virus apparently causing the disease.',\n",
       "   'In particular it appeared that underlying this highly focused outbreak there remained a more or less constant background infection level, that remained present in Guangdong for a long time and witnessed a slow decline in Hong Kong, on which superimposes sudden local outbreaks.']),\n",
       " ('Hong Kong',\n",
       "  'shares border with',\n",
       "  'Guangdong',\n",
       "  ['Although apparently classical in its onset, the pattern of the outbreak became puzzling after an estate in Hong Kong was affected, with a huge number of patients infected by the virus apparently causing the disease.',\n",
       "   'In particular it appeared that underlying this highly focused outbreak there remained a more or less constant background infection level, that remained present in Guangdong for a long time and witnessed a slow decline in Hong Kong, on which superimposes sudden local outbreaks.']),\n",
       " ('pattern',\n",
       "  'facet of',\n",
       "  'epidemic',\n",
       "  ['This pattern is difficult to reconcile with a standard epidemic pattern for reasons that have to be analyzed.(?<=[.?!])\\\\s+Many hypotheses can be put forward to account for this observation.',\n",
       "   'Learning from a set of coronavirus mediated epidemics that affected pigs in the 1983 -1985 [1] we chose to explore here the consequences of a situation where two overlapping epidemics interfere with each other.']),\n",
       " ('coronavirus',\n",
       "  'has effect',\n",
       "  'influenza',\n",
       "  ['This pattern is difficult to reconcile with a standard epidemic pattern for reasons that have to be analyzed.(?<=[.?!])\\\\s+Many hypotheses can be put forward to account for this observation.',\n",
       "   'Learning from a set of coronavirus mediated epidemics that affected pigs in the 1983 -1985 [1] we chose to explore here the consequences of a situation where two overlapping epidemics interfere with each other.']),\n",
       " ('hypothesis of the double epidemic model for SARS',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['The hypothesis of the double epidemic model for SARS we develop here is as follow.',\n",
       "   'It is based on the high mutation and recombination rate of coronaviruses [2] , and on the observation that tissue tropism can be changed by simple mutations [3] .']),\n",
       " ('recombination',\n",
       "  'subclass of',\n",
       "  'mutation',\n",
       "  ['The hypothesis of the double epidemic model for SARS we develop here is as follow.',\n",
       "   'It is based on the high mutation and recombination rate of coronaviruses [2] , and on the observation that tissue tropism can be changed by simple mutations [3] .']),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'coronavirus virus',\n",
       "  ['There are two epidemics, one epidemic is SARS caused by a coronavirus virus, call it virus A.',\n",
       "   'Another epidemic, which may have appeared before SARS, is assumed to be extremely contagious because of the nature of the virus and of its relative innocuousness, could be propagated by contaminated food and soiled surfaces.']),\n",
       " ('coronavirus virus',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['There are two epidemics, one epidemic is SARS caused by a coronavirus virus, call it virus A.',\n",
       "   'Another epidemic, which may have appeared before SARS, is assumed to be extremely contagious because of the nature of the virus and of its relative innocuousness, could be propagated by contaminated food and soiled surfaces.']),\n",
       " ('Another epidemic',\n",
       "  'has cause',\n",
       "  'SARS',\n",
       "  ['There are two epidemics, one epidemic is SARS caused by a coronavirus virus, call it virus A.',\n",
       "   'Another epidemic, which may have appeared before SARS, is assumed to be extremely contagious because of the nature of the virus and of its relative innocuousness, could be propagated by contaminated food and soiled surfaces.']),\n",
       " ('coronavirus',\n",
       "  'taxonomic type',\n",
       "  'virus B',\n",
       "  ['It could be caused by some coronavirus, call it virus B.',\n",
       "   'The most likely is that it would cause gastroenteritis (this fits with the observation that many people had diarrhoea for about one day, during this winter in Guangdong and in Hong Kong).']),\n",
       " ('virus B',\n",
       "  'parent taxon',\n",
       "  'coronavirus',\n",
       "  ['It could be caused by some coronavirus, call it virus B.',\n",
       "   'The most likely is that it would cause gastroenteritis (this fits with the observation that many people had diarrhoea for about one day, during this winter in Guangdong and in Hong Kong).']),\n",
       " ('diarrhoea',\n",
       "  'has cause',\n",
       "  'gastroenteritis',\n",
       "  ['It could be caused by some coronavirus, call it virus B.',\n",
       "   'The most likely is that it would cause gastroenteritis (this fits with the observation that many people had diarrhoea for about one day, during this winter in Guangdong and in Hong Kong).']),\n",
       " ('virus B',\n",
       "  'followed by',\n",
       "  'virus A',\n",
       "  ['The most likely origin of virus A is a more or less complicated mutation or recombination event from virus B [4] [5] [6] .',\n",
       "   'We also explore the possible situation where viruses A and B would be of totally different origin, but would cause an overlapping immune response of the host.']),\n",
       " ('immune response',\n",
       "  'has part',\n",
       "  'host',\n",
       "  ['The most likely origin of virus A is a more or less complicated mutation or recombination event from virus B [4] [5] [6] .',\n",
       "   'We also explore the possible situation where viruses A and B would be of totally different origin, but would cause an overlapping immune response of the host.']),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'virus B',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Mainland China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Beijing',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Mainland China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Shanghai',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Mainland China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Guangdong',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Mainland China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Beijing',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Mainland China',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Shanghai',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Mainland China',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Guangdong',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Mainland China',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('Hong Kong',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Mainland China',\n",
       "  ['Both epidemics would spread in parallel, and it can be expected that the epidemic caused by virus B which is rather innocuous, protects against SARS (so that naïve regions, not protected by the epidemic B can get SARS large outbreaks).',\n",
       "   'Various consequences of the spreading pattern of this double epidemic hypothesis are discussed in relation with the puzzling distribution of the disease in Mainland China, noticeably, in the huge difference witnessed between the situation in Beijing and in Shanghai (both regions have tight and frequent contacts with Guangdong and Hong Kong).']),\n",
       " ('data',\n",
       "  'facet of',\n",
       "  'epidemiological',\n",
       "  ['As a side prediction of the model we can infer conclusions on the accuracy of available epidemiological data.',\n",
       "   'In this paper, we introduce a new epidemic model to study the outbreak of SARS in China.']),\n",
       " ('SARS in China',\n",
       "  'instance of',\n",
       "  'epidemic',\n",
       "  ['As a side prediction of the model we can infer conclusions on the accuracy of available epidemiological data.',\n",
       "   'In this paper, we introduce a new epidemic model to study the outbreak of SARS in China.']),\n",
       " ('Susceptible-Infected-Removed',\n",
       "  'instance of',\n",
       "  'epidemic model',\n",
       "  ['We begin with a brief review on a classical epidemic model, the so called Susceptible-Infected-Removed (SIR) model which is simple and useful when one tries to understand the propagation of many real-life epidemics.',\n",
       "   'We then point out the shortcomings of the SIR model and explain our motivations of building a new epidemic model.']),\n",
       " ('SIR model',\n",
       "  'instance of',\n",
       "  'epidemic model',\n",
       "  ['We begin with a brief review on a classical epidemic model, the so called Susceptible-Infected-Removed (SIR) model which is simple and useful when one tries to understand the propagation of many real-life epidemics.',\n",
       "   'We then point out the shortcomings of the SIR model and explain our motivations of building a new epidemic model.']),\n",
       " ('outbreak of SARS',\n",
       "  'location',\n",
       "  'Hong Kong',\n",
       "  ['While this new model is still relatively simple, it has some nice properties which makes it possible to model better the outbreak of SARS in Hong Kong and other cities.',\n",
       "   'Consider a disease that, after recovery, confers immunity (which includes deaths: dead individuals are still counted).']),\n",
       " ('immunity',\n",
       "  'facet of',\n",
       "  'disease',\n",
       "  ['While this new model is still relatively simple, it has some nice properties which makes it possible to model better the outbreak of SARS in Hong Kong and other cities.',\n",
       "   'Consider a disease that, after recovery, confers immunity (which includes deaths: dead individuals are still counted).']),\n",
       " ('population',\n",
       "  'has part',\n",
       "  'entry',\n",
       "  ['We assume that there is no entry into or departure from the population.',\n",
       "   'The population can then be divided into three distinct classes; the susceptibles, S, who can catch the disease; the infectives, I, who have the disease and can transmit it; and the removed class, R, namely those who have either had the disease, or are recovered, immune or isolated until recovered.']),\n",
       " ('susceptible',\n",
       "  'opposite of',\n",
       "  'infectives',\n",
       "  ['We assume that there is no entry into or departure from the population.',\n",
       "   'The population can then be divided into three distinct classes; the susceptibles, S, who can catch the disease; the infectives, I, who have the disease and can transmit it; and the removed class, R, namely those who have either had the disease, or are recovered, immune or isolated until recovered.']),\n",
       " ('infectives',\n",
       "  'opposite of',\n",
       "  'susceptible',\n",
       "  ['We assume that there is no entry into or departure from the population.',\n",
       "   'The population can then be divided into three distinct classes; the susceptibles, S, who can catch the disease; the infectives, I, who have the disease and can transmit it; and the removed class, R, namely those who have either had the disease, or are recovered, immune or isolated until recovered.']),\n",
       " ('R',\n",
       "  'instance of',\n",
       "  'class',\n",
       "  ['Here we follow the definition of the class R given in [8] .',\n",
       "   \"However, we would like to draw the readers' attention that this definition of the class R is different from those given in [7] , [11] which do not include isolated infectives in the class R.\"]),\n",
       " ('isolated',\n",
       "  'subclass of',\n",
       "  'infectives',\n",
       "  ['Here we follow the definition of the class R given in [8] .',\n",
       "   \"However, we would like to draw the readers' attention that this definition of the class R is different from those given in [7] , [11] which do not include isolated infectives in the class R.\"]),\n",
       " ('class',\n",
       "  'has part',\n",
       "  'individuals',\n",
       "  ['The progress of individuals is schematically described by S → I → R.(?<=[.?!])\\\\s+Let S(t), I(t) and R(t) be the number of individuals in each of the corresponding class at time t.',\n",
       "   'Note that usually only R(t) can be known.(?<=[.?!])\\\\s+It is often considered that R(t) is the cumulative number of patients admitted to hospitals.']),\n",
       " ('admitted',\n",
       "  'facet of',\n",
       "  'hospital',\n",
       "  ['The progress of individuals is schematically described by S → I → R.(?<=[.?!])\\\\s+Let S(t), I(t) and R(t) be the number of individuals in each of the corresponding class at time t.',\n",
       "   'Note that usually only R(t) can be known.(?<=[.?!])\\\\s+It is often considered that R(t) is the cumulative number of patients admitted to hospitals.']),\n",
       " ('nonlinear',\n",
       "  'subclass of',\n",
       "  'ordinary differential equations',\n",
       "  ['With some reasonable assumptions (which will be explained in details in the next section), we can show that these three functions are governed by the following system of nonlinear ordinary differential equations (see next section for the derivation of these differential equations): where r is the infection rate and a the removal rate of infectives.',\n",
       "   'The parameters r and a characterize the propagation of the disease and can also be used as control parameters in order to stop the epidemic.']),\n",
       " ('control parameters',\n",
       "  'facet of',\n",
       "  'propagation of the disease',\n",
       "  ['With some reasonable assumptions (which will be explained in details in the next section), we can show that these three functions are governed by the following system of nonlinear ordinary differential equations (see next section for the derivation of these differential equations): where r is the infection rate and a the removal rate of infectives.',\n",
       "   'The parameters r and a characterize the propagation of the disease and can also be used as control parameters in order to stop the epidemic.']),\n",
       " ('S',\n",
       "  'instance of',\n",
       "  'functions',\n",
       "  ['In general, the functions S, I and R behave as the three curves in Figure 1 .(?<=[.?!])\\\\s+The characteristics of these curves are as follow.',\n",
       "   '-S decreases monotonically from its initial value to its final value, while R increases monotonically until it reaches the limiting value R ∞ .']),\n",
       " ('I',\n",
       "  'instance of',\n",
       "  'functions',\n",
       "  ['In general, the functions S, I and R behave as the three curves in Figure 1 .(?<=[.?!])\\\\s+The characteristics of these curves are as follow.',\n",
       "   '-S decreases monotonically from its initial value to its final value, while R increases monotonically until it reaches the limiting value R ∞ .']),\n",
       " ('R',\n",
       "  'instance of',\n",
       "  'functions',\n",
       "  ['In general, the functions S, I and R behave as the three curves in Figure 1 .(?<=[.?!])\\\\s+The characteristics of these curves are as follow.',\n",
       "   '-S decreases monotonically from its initial value to its final value, while R increases monotonically until it reaches the limiting value R ∞ .']),\n",
       " ('initial value',\n",
       "  'followed by',\n",
       "  'final value',\n",
       "  ['In general, the functions S, I and R behave as the three curves in Figure 1 .(?<=[.?!])\\\\s+The characteristics of these curves are as follow.',\n",
       "   '-S decreases monotonically from its initial value to its final value, while R increases monotonically until it reaches the limiting value R ∞ .']),\n",
       " ('final value',\n",
       "  'follows',\n",
       "  'initial value',\n",
       "  ['In general, the functions S, I and R behave as the three curves in Figure 1 .(?<=[.?!])\\\\s+The characteristics of these curves are as follow.',\n",
       "   '-S decreases monotonically from its initial value to its final value, while R increases monotonically until it reaches the limiting value R ∞ .']),\n",
       " ('exponentially slow',\n",
       "  'opposite of',\n",
       "  'exponentially fast',\n",
       "  ['Both curves have two distinct periods where the decrease (or increase) is exponentially slow or exponentially fast.',\n",
       "   '-The function I first increases exponentially and reaches the maximum value at time t 0 and then decreases to zero.']),\n",
       " ('exponentially fast',\n",
       "  'opposite of',\n",
       "  'exponentially slow',\n",
       "  ['Both curves have two distinct periods where the decrease (or increase) is exponentially slow or exponentially fast.',\n",
       "   '-The function I first increases exponentially and reaches the maximum value at time t 0 and then decreases to zero.']),\n",
       " ('I',\n",
       "  'instance of',\n",
       "  'function',\n",
       "  ['Both curves have two distinct periods where the decrease (or increase) is exponentially slow or exponentially fast.',\n",
       "   '-The function I first increases exponentially and reaches the maximum value at time t 0 and then decreases to zero.']),\n",
       " ('critical point',\n",
       "  'part of',\n",
       "  'graph of the function I',\n",
       "  ['The time t 0 is the critical point of the function I at which has several peaks and it therefore looks very different from the general shape of the graph of the function I.',\n",
       "   'In the SIR model once the I curve decreases, it will decrease to the zero value Therefore, the standard SIR model could not be used to model the outbreak of SARS in Hong Kong.']),\n",
       " ('SARS in Hong Kong',\n",
       "  'instance of',\n",
       "  'outbreak',\n",
       "  ['The time t 0 is the critical point of the function I at which has several peaks and it therefore looks very different from the general shape of the graph of the function I.',\n",
       "   'In the SIR model once the I curve decreases, it will decrease to the zero value Therefore, the standard SIR model could not be used to model the outbreak of SARS in Hong Kong.']),\n",
       " ('general shape of a graph',\n",
       "  'subclass of',\n",
       "  'graph',\n",
       "  ['If we take away those cases from the hospital staff and the Amoy Gardens Estate, that is if we only consider the cases from the community, then the graph looks closer to the general shape of a graph for a function I.',\n",
       "   'However, it is still difficult for a general SIR model to produce a I curve whose values are within a narrow range (say 15 to 25 cases) for a relatively long time interval.']),\n",
       " ('I curve',\n",
       "  'subclass of',\n",
       "  'SIR model',\n",
       "  ['If we take away those cases from the hospital staff and the Amoy Gardens Estate, that is if we only consider the cases from the community, then the graph looks closer to the general shape of a graph for a function I.',\n",
       "   'However, it is still difficult for a general SIR model to produce a I curve whose values are within a narrow range (say 15 to 25 cases) for a relatively long time interval.']),\n",
       " ('SIR model',\n",
       "  'replaced by',\n",
       "  'SEIR model',\n",
       "  ['This motivated us to replace the SIR model by the SEIR model.',\n",
       "   'In this model there is a fixed period between exposure and becoming infectious, called latent period (this is in fact the case for SARS and it is another reason why we consider the SEIR model).']),\n",
       " ('SEIR model',\n",
       "  'replaces',\n",
       "  'SIR model',\n",
       "  ['This motivated us to replace the SIR model by the SEIR model.',\n",
       "   'In this model there is a fixed period between exposure and becoming infectious, called latent period (this is in fact the case for SARS and it is another reason why we consider the SEIR model).']),\n",
       " ('SEIR model',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['This motivated us to replace the SIR model by the SEIR model.',\n",
       "   'In this model there is a fixed period between exposure and becoming infectious, called latent period (this is in fact the case for SARS and it is another reason why we consider the SEIR model).']),\n",
       " ('Typical dynamics for the SIR model',\n",
       "  'different from',\n",
       "  'Typical dynamics for the SIR model',\n",
       "  ['Thus, rather than an exposed susceptible Typical dynamics for the SIR model Figure 1 Typical dynamics for the SIR model.',\n",
       "   'becoming immediately infectious, it enters the Exposed class, labeled E, remaining there a fixed period of time.(?<=[.?!])\\\\s+More detailed description of this model will be given in the next section.']),\n",
       " ('Typical dynamics for the SIR model',\n",
       "  'different from',\n",
       "  'Typical dynamics for the SIR model',\n",
       "  ['Thus, rather than an exposed susceptible Typical dynamics for the SIR model Figure 1 Typical dynamics for the SIR model.',\n",
       "   'becoming immediately infectious, it enters the Exposed class, labeled E, remaining there a fixed period of time.(?<=[.?!])\\\\s+More detailed description of this model will be given in the next section.']),\n",
       " ('Exposed',\n",
       "  'instance of',\n",
       "  'class',\n",
       "  ['Thus, rather than an exposed susceptible Typical dynamics for the SIR model Figure 1 Typical dynamics for the SIR model.',\n",
       "   'becoming immediately infectious, it enters the Exposed class, labeled E, remaining there a fixed period of time.(?<=[.?!])\\\\s+More detailed description of this model will be given in the next section.']),\n",
       " ('curves',\n",
       "  'has part',\n",
       "  'I',\n",
       "  ['It is worth mentioning here that the SEIR model can produce much more interesting and complicated I, R and curves.',\n",
       "   'It is also possible to produce a I curve whose values are within a narrow range for a relatively long time interval and result thus in a relatively large R ∞ .']),\n",
       " ('I curve',\n",
       "  'has part',\n",
       "  'R ∞',\n",
       "  ['It is worth mentioning here that the SEIR model can produce much more interesting and complicated I, R and curves.',\n",
       "   'It is also possible to produce a I curve whose values are within a narrow range for a relatively long time interval and result thus in a relatively large R ∞ .']),\n",
       " ('Infective',\n",
       "  'subclass of',\n",
       "  'epidemic',\n",
       "  ['Note that the SIR model is the limiting case of the SEIR model when the time interval from the infection to onset is zero; for given parameters r and a the total size of the epidemic is the same in the two models; however the duration is longer for the SEIR model which will display lower admission per day curves (or the I curves); if we see this result from the reciprocal point of view, the same level of admissions per day will result in a higher total epi-demic size for the SEIR model than for the SIR model because of the Exposed class which is \"hidden\" in the sense that it has no effects on the other classes until the individuals move to the Infective class and thus contributing to the propagation of the epidemic.',\n",
       "   'A numerical illustration of this phenomena is given in Figure 3 that plots the evolution of the S,E, I and R classes for the two models with the same parameters r and a.']),\n",
       " ('S',\n",
       "  'instance of',\n",
       "  'classes',\n",
       "  ['Note that the SIR model is the limiting case of the SEIR model when the time interval from the infection to onset is zero; for given parameters r and a the total size of the epidemic is the same in the two models; however the duration is longer for the SEIR model which will display lower admission per day curves (or the I curves); if we see this result from the reciprocal point of view, the same level of admissions per day will result in a higher total epi-demic size for the SEIR model than for the SIR model because of the Exposed class which is \"hidden\" in the sense that it has no effects on the other classes until the individuals move to the Infective class and thus contributing to the propagation of the epidemic.',\n",
       "   'A numerical illustration of this phenomena is given in Figure 3 that plots the evolution of the S,E, I and R classes for the two models with the same parameters r and a.']),\n",
       " ('E',\n",
       "  'instance of',\n",
       "  'classes',\n",
       "  ['Note that the SIR model is the limiting case of the SEIR model when the time interval from the infection to onset is zero; for given parameters r and a the total size of the epidemic is the same in the two models; however the duration is longer for the SEIR model which will display lower admission per day curves (or the I curves); if we see this result from the reciprocal point of view, the same level of admissions per day will result in a higher total epi-demic size for the SEIR model than for the SIR model because of the Exposed class which is \"hidden\" in the sense that it has no effects on the other classes until the individuals move to the Infective class and thus contributing to the propagation of the epidemic.',\n",
       "   'A numerical illustration of this phenomena is given in Figure 3 that plots the evolution of the S,E, I and R classes for the two models with the same parameters r and a.']),\n",
       " ('I',\n",
       "  'instance of',\n",
       "  'classes',\n",
       "  ['Note that the SIR model is the limiting case of the SEIR model when the time interval from the infection to onset is zero; for given parameters r and a the total size of the epidemic is the same in the two models; however the duration is longer for the SEIR model which will display lower admission per day curves (or the I curves); if we see this result from the reciprocal point of view, the same level of admissions per day will result in a higher total epi-demic size for the SEIR model than for the SIR model because of the Exposed class which is \"hidden\" in the sense that it has no effects on the other classes until the individuals move to the Infective class and thus contributing to the propagation of the epidemic.',\n",
       "   'A numerical illustration of this phenomena is given in Figure 3 that plots the evolution of the S,E, I and R classes for the two models with the same parameters r and a.']),\n",
       " ('R',\n",
       "  'instance of',\n",
       "  'classes',\n",
       "  ['Note that the SIR model is the limiting case of the SEIR model when the time interval from the infection to onset is zero; for given parameters r and a the total size of the epidemic is the same in the two models; however the duration is longer for the SEIR model which will display lower admission per day curves (or the I curves); if we see this result from the reciprocal point of view, the same level of admissions per day will result in a higher total epi-demic size for the SEIR model than for the SIR model because of the Exposed class which is \"hidden\" in the sense that it has no effects on the other classes until the individuals move to the Infective class and thus contributing to the propagation of the epidemic.',\n",
       "   'A numerical illustration of this phenomena is given in Figure 3 that plots the evolution of the S,E, I and R classes for the two models with the same parameters r and a.']),\n",
       " ('parameters estimations',\n",
       "  'part of',\n",
       "  'SIR model',\n",
       "  ['The parameters estimations are very crucial when one applies the SIR models.',\n",
       "   'When the initial population of the susceptibles is very large, a small change in the parameters can result in a large difference in the values of R ∞ .']),\n",
       " ('population',\n",
       "  'has part',\n",
       "  'parameters',\n",
       "  ['The parameters estimations are very crucial when one applies the SIR models.',\n",
       "   'When the initial population of the susceptibles is very large, a small change in the parameters can result in a large difference in the values of R ∞ .']),\n",
       " ('population',\n",
       "  'has part',\n",
       "  'S(0)',\n",
       "  ['For example, if we take the total initial population, S(0) = 6.8 millions (e.g.',\n",
       "   'the case of Hong Kong) and an initial infected class of I(0) = 10 persons, together with a = 2/3 and r = 9.788 × 10 -8 , then R ∞ is about 5,000.']),\n",
       " ('case',\n",
       "  'country',\n",
       "  'Hong Kong',\n",
       "  ['For example, if we take the total initial population, S(0) = 6.8 millions (e.g.',\n",
       "   'the case of Hong Kong) and an initial infected class of I(0) = 10 persons, together with a = 2/3 and r = 9.788 × 10 -8 , then R ∞ is about 5,000.']),\n",
       " ('Amoy Gardens',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['Suppose we Daily new number of confirmed SARS cases from Hong Kong: hospital, community and the Amoy Gardens Amoy Gardens Community dR dt increase the r parameter by 1% to have r = 9.88588 × 10 -8 , then R ∞ will become 116464.',\n",
       "   'This sensitivity in the parameters can therefore give rise to unacceptable errors in predicting the size of the epidemic and invalidates thus the model.']),\n",
       " ('invalidates',\n",
       "  'subclass of',\n",
       "  'errors',\n",
       "  ['Suppose we Daily new number of confirmed SARS cases from Hong Kong: hospital, community and the Amoy Gardens Amoy Gardens Community dR dt increase the r parameter by 1% to have r = 9.88588 × 10 -8 , then R ∞ will become 116464.',\n",
       "   'This sensitivity in the parameters can therefore give rise to unacceptable errors in predicting the size of the epidemic and invalidates thus the model.']),\n",
       " ('population',\n",
       "  'part of',\n",
       "  'population',\n",
       "  ['Contrariwise, when we try to fit the data with respect to a hypothetical much smaller population, the sensitivity decreases.',\n",
       "   'Therefore, we are lead to consider a model for which the effective population, i.e.']),\n",
       " ('effective population',\n",
       "  'instance of',\n",
       "  'model',\n",
       "  ['Contrariwise, when we try to fit the data with respect to a hypothetical much smaller population, the sensitivity decreases.',\n",
       "   'Therefore, we are lead to consider a model for which the effective population, i.e.']),\n",
       " ('population',\n",
       "  'has part',\n",
       "  'population',\n",
       "  ['the total population that enters the susceptible class is, for some yet to define reason, much less than the total population.',\n",
       "   'In other words, a large portion of the total population is protected during the epidemic.(?<=[.?!])\\\\s+Note that this happens when a population undergoes vaccination against a disease, i.e.']),\n",
       " ('vaccination',\n",
       "  'facet of',\n",
       "  'disease',\n",
       "  ['the total population that enters the susceptible class is, for some yet to define reason, much less than the total population.',\n",
       "   'In other words, a large portion of the total population is protected during the epidemic.(?<=[.?!])\\\\s+Note that this happens when a population undergoes vaccination against a disease, i.e.']),\n",
       " ('immune',\n",
       "  'facet of',\n",
       "  'epidemic',\n",
       "  ['some people become not only immune to the epidemic but do not transmit it neither.',\n",
       "   'It is also possible that the vaccination against another disease can protect people from being infected by SARS.']),\n",
       " ('vaccination',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['some people become not only immune to the epidemic but do not transmit it neither.',\n",
       "   'It is also possible that the vaccination against another disease can protect people from being infected by SARS.']),\n",
       " ('different, but similar, disease that spread before',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['Another possibility is that a different, but similar, disease that spread before (or is spreading simultaneously with SARS) has immunized some of the population against SARS; this is our working hypothesis because it fits better with the relatively surprising SARS propagation which is less severe in places with low hygiene conditions.',\n",
       "   'Based on this hypothesis, we shall develop a double epidemic model, the SEIRP model to study the outbreak of SARS in the next section.']),\n",
       " ('SEIRP model',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['Another possibility is that a different, but similar, disease that spread before (or is spreading simultaneously with SARS) has immunized some of the population against SARS; this is our working hypothesis because it fits better with the relatively surprising SARS propagation which is less severe in places with low hygiene conditions.',\n",
       "   'Based on this hypothesis, we shall develop a double epidemic model, the SEIRP model to study the outbreak of SARS in the next section.']),\n",
       " ('outbreak of infections',\n",
       "  'has cause',\n",
       "  'coronavirus',\n",
       "  ['As will be discussed at the end of this paper an outbreak of infections caused by a coronavirus and its mutants [1, 10] provided hints that epidemics that superimpose on each other might account for the situation we are witnessing at several places in the world.',\n",
       "   'Motivated by the above considerations we shall introduce a new epidemic model here to describe a double epidemic interaction.']),\n",
       " ('double epidemic interaction',\n",
       "  'subclass of',\n",
       "  'epidemic model',\n",
       "  ['As will be discussed at the end of this paper an outbreak of infections caused by a coronavirus and its mutants [1, 10] provided hints that epidemics that superimpose on each other might account for the situation we are witnessing at several places in the world.',\n",
       "   'Motivated by the above considerations we shall introduce a new epidemic model here to describe a double epidemic interaction.']),\n",
       " ('SEIRP',\n",
       "  'subclass of',\n",
       "  'SIR',\n",
       "  ['This new model will be called SEIRP model (E stands for the Exposed class while P stands for protection) which can be considered as a variant of the standard SIR.',\n",
       "   'Other variants of the SIR model can be found in 17.']),\n",
       " ('SIR model',\n",
       "  'part of',\n",
       "  '17',\n",
       "  ['This new model will be called SEIRP model (E stands for the Exposed class while P stands for protection) which can be considered as a variant of the standard SIR.',\n",
       "   'Other variants of the SIR model can be found in 17.']),\n",
       " ('non-linear',\n",
       "  'subclass of',\n",
       "  'differential equation',\n",
       "  ['We shall show here that by choosing the parameters carefully, the solution of the system of non-linear differential equations governing the SEIRP model can decay extremely slowly and therefore remains almost constant for a long time interval.',\n",
       "   'This may explain the fact that a more or less constant background infection level can be maintained (eg in Hong Kong or in Guangdong), on which superimposes large local outbreaks.']),\n",
       " ('Guangdong',\n",
       "  'shares border with',\n",
       "  'Hong Kong',\n",
       "  ['We shall show here that by choosing the parameters carefully, the solution of the system of non-linear differential equations governing the SEIRP model can decay extremely slowly and therefore remains almost constant for a long time interval.',\n",
       "   'This may explain the fact that a more or less constant background infection level can be maintained (eg in Hong Kong or in Guangdong), on which superimposes large local outbreaks.']),\n",
       " ('virus A',\n",
       "  'opposite of',\n",
       "  'virus B',\n",
       "  ['We assume that two groups of infected individuals are introduced into a large population.(?<=[.?!])\\\\s+One group is infected by virus A and the other group by virus B.',\n",
       "   'One would like to describe the spread of the infection within the population as a function of time.']),\n",
       " ('virus B',\n",
       "  'opposite of',\n",
       "  'virus A',\n",
       "  ['We assume that two groups of infected individuals are introduced into a large population.(?<=[.?!])\\\\s+One group is infected by virus A and the other group by virus B.',\n",
       "   'One would like to describe the spread of the infection within the population as a function of time.']),\n",
       " ('spread',\n",
       "  'has cause',\n",
       "  'infection',\n",
       "  ['We assume that two groups of infected individuals are introduced into a large population.(?<=[.?!])\\\\s+One group is infected by virus A and the other group by virus B.',\n",
       "   'One would like to describe the spread of the infection within the population as a function of time.']),\n",
       " ('infection',\n",
       "  'has effect',\n",
       "  'spread',\n",
       "  ['We assume that two groups of infected individuals are introduced into a large population.(?<=[.?!])\\\\s+One group is infected by virus A and the other group by virus B.',\n",
       "   'One would like to describe the spread of the infection within the population as a function of time.']),\n",
       " ('directly transmitted diseases',\n",
       "  'subclass of',\n",
       "  'disease',\n",
       "  ['This will depend on a variety of circumstances, including the actual disease involved, but as a first attempt to model directly transmitted diseases we make some reasonable general assumptions.',\n",
       "   'Assume both diseases which, after recovery, confers immunity (which includes deaths: dead individuals are still counted).']),\n",
       " ('immunity',\n",
       "  'facet of',\n",
       "  'disease',\n",
       "  ['This will depend on a variety of circumstances, including the actual disease involved, but as a first attempt to model directly transmitted diseases we make some reasonable general assumptions.',\n",
       "   'Assume both diseases which, after recovery, confers immunity (which includes deaths: dead individuals are still counted).']),\n",
       " ('classes',\n",
       "  'part of',\n",
       "  'population',\n",
       "  ['We assume that there is no entry into or departure from the population.(?<=[.?!])\\\\s+The population can then be divided into six distinct classes.',\n",
       "   'The class of Susceptibles, S, are those who can catch the disease A or disease B.']),\n",
       " ('Susceptible',\n",
       "  'subclass of',\n",
       "  'Susceptible',\n",
       "  ['We assume that there is no entry into or departure from the population.(?<=[.?!])\\\\s+The population can then be divided into six distinct classes.',\n",
       "   'The class of Susceptibles, S, are those who can catch the disease A or disease B.']),\n",
       " ('Exposed class E',\n",
       "  'subclass of',\n",
       "  'latent period',\n",
       "  ['A susceptible who catches the disease A first will enter the Exposed class E which includes those in the latent period, who are infected by disease A but not yet infectious.',\n",
       "   'After the latent period ends, the individual enters the class I of Comparison of the SEIR (red) and SIR (green) models with the same parameters \"r\" and \"a\" Figure 3 Comparison of the SEIR (red) and SIR (green) models with the same parameters \"r\" and \"a\".']),\n",
       " ('Figure 3',\n",
       "  'depicts',\n",
       "  'Comparison of the SEIR (red) and SIR (green) models',\n",
       "  ['A susceptible who catches the disease A first will enter the Exposed class E which includes those in the latent period, who are infected by disease A but not yet infectious.',\n",
       "   'After the latent period ends, the individual enters the class I of Comparison of the SEIR (red) and SIR (green) models with the same parameters \"r\" and \"a\" Figure 3 Comparison of the SEIR (red) and SIR (green) models with the same parameters \"r\" and \"a\".']),\n",
       " ('SEIR model',\n",
       "  'instance of',\n",
       "  'model evolution',\n",
       "  ['SEIR model evolution (in red) is much slower (and thus may seam less severe) but reaches the same final size of epidemic.',\n",
       "   'infectives, which includes those who have the disease A and can transmit it.']),\n",
       " ('transmit it',\n",
       "  'has cause',\n",
       "  'disease A',\n",
       "  ['SEIR model evolution (in red) is much slower (and thus may seam less severe) but reaches the same final size of epidemic.',\n",
       "   'infectives, which includes those who have the disease A and can transmit it.']),\n",
       " ('Removed class R',\n",
       "  'subclass of',\n",
       "  'immune',\n",
       "  ['When the infectious period ends, the individual enters the Removed class R, namely those who are either recovered from disease A, immune or isolated until recovered from disease.',\n",
       "   'A susceptible who catches the disease B first will enter the class I P of infectives and then the Removed class R P .']),\n",
       " ('class I P',\n",
       "  'followed by',\n",
       "  'class R P',\n",
       "  ['When the infectious period ends, the individual enters the Removed class R, namely those who are either recovered from disease A, immune or isolated until recovered from disease.',\n",
       "   'A susceptible who catches the disease B first will enter the class I P of infectives and then the Removed class R P .']),\n",
       " ('class R P',\n",
       "  'follows',\n",
       "  'class I P',\n",
       "  ['When the infectious period ends, the individual enters the Removed class R, namely those who are either recovered from disease A, immune or isolated until recovered from disease.',\n",
       "   'A susceptible who catches the disease B first will enter the class I P of infectives and then the Removed class R P .']),\n",
       " ('disease B',\n",
       "  'opposite of',\n",
       "  'disease A',\n",
       "  ['It is assumed that catching disease B first will protect the individual from catching disease A.(?<=[.?!])\\\\s+The progress of individuals is schematically described by the following diagram.',\n",
       "   'With, S(t), E(t), I(t), R(t) I P (t) and R P (t) as the number of individuals in each class at time t, we assume here that: (i) There is no entry into or departure from the population, except possibly through death from the disease.']),\n",
       " ('disease A',\n",
       "  'opposite of',\n",
       "  'disease B',\n",
       "  ['It is assumed that catching disease B first will protect the individual from catching disease A.(?<=[.?!])\\\\s+The progress of individuals is schematically described by the following diagram.',\n",
       "   'With, S(t), E(t), I(t), R(t) I P (t) and R P (t) as the number of individuals in each class at time t, we assume here that: (i) There is no entry into or departure from the population, except possibly through death from the disease.']),\n",
       " ('death',\n",
       "  'has cause',\n",
       "  'disease',\n",
       "  ['It is assumed that catching disease B first will protect the individual from catching disease A.(?<=[.?!])\\\\s+The progress of individuals is schematically described by the following diagram.',\n",
       "   'With, S(t), E(t), I(t), R(t) I P (t) and R P (t) as the number of individuals in each class at time t, we assume here that: (i) There is no entry into or departure from the population, except possibly through death from the disease.']),\n",
       " ('infective',\n",
       "  'opposite of',\n",
       "  'susceptible',\n",
       "  ['(ii) The gain in the exposed class E is at a rate proportional to the number of people in the infective class I and that of the susceptibles S, that is rS(t)I(t), where r > 0 is a constant.',\n",
       "   '(iii) The gain in the infective class I P is at a rate proportional to the number of people in the infective class I P and that of the susceptibles S, that is r P S(t)I P (t), where r P > 0 is a constant.']),\n",
       " ('susceptible',\n",
       "  'opposite of',\n",
       "  'infective',\n",
       "  ['(ii) The gain in the exposed class E is at a rate proportional to the number of people in the infective class I and that of the susceptibles S, that is rS(t)I(t), where r > 0 is a constant.',\n",
       "   '(iii) The gain in the infective class I P is at a rate proportional to the number of people in the infective class I P and that of the susceptibles S, that is r P S(t)I P (t), where r P > 0 is a constant.']),\n",
       " ('infective',\n",
       "  'opposite of',\n",
       "  'susceptible',\n",
       "  ['(ii) The gain in the exposed class E is at a rate proportional to the number of people in the infective class I and that of the susceptibles S, that is rS(t)I(t), where r > 0 is a constant.',\n",
       "   '(iii) The gain in the infective class I P is at a rate proportional to the number of people in the infective class I P and that of the susceptibles S, that is r P S(t)I P (t), where r P > 0 is a constant.']),\n",
       " ('susceptible',\n",
       "  'opposite of',\n",
       "  'infective',\n",
       "  ['(ii) The gain in the exposed class E is at a rate proportional to the number of people in the infective class I and that of the susceptibles S, that is rS(t)I(t), where r > 0 is a constant.',\n",
       "   '(iii) The gain in the infective class I P is at a rate proportional to the number of people in the infective class I P and that of the susceptibles S, that is r P S(t)I P (t), where r P > 0 is a constant.']),\n",
       " ('P',\n",
       "  'has part',\n",
       "  'I',\n",
       "  ['(iv) The susceptibles are therefore lost at the rate rS(t)I(t) + r P S(t)I P (t).',\n",
       "   '(v) The rate of removal of the people in class E to the infective class I is proportional to the number of people in class E, that is bE(t), where b is a positive number.']),\n",
       " ('infective',\n",
       "  'opposite of',\n",
       "  'class I',\n",
       "  ['(iv) The susceptibles are therefore lost at the rate rS(t)I(t) + r P S(t)I P (t).',\n",
       "   '(v) The rate of removal of the people in class E to the infective class I is proportional to the number of people in class E, that is bE(t), where b is a positive number.']),\n",
       " ('class I',\n",
       "  'opposite of',\n",
       "  'infective',\n",
       "  ['(iv) The susceptibles are therefore lost at the rate rS(t)I(t) + r P S(t)I P (t).',\n",
       "   '(v) The rate of removal of the people in class E to the infective class I is proportional to the number of people in class E, that is bE(t), where b is a positive number.']),\n",
       " ('class E',\n",
       "  'instance of',\n",
       "  'time unit',\n",
       "  ['It can be shown that the fraction of people remaining in the exposed class E s time unit after entering class E is e -bs , so the length of the latent period is distributed exponentially with mean equals to e -bs ds = 1/b (c.f.',\n",
       "   '[11] ).(?<=[.?!])\\\\s+(vi) The rate of removal of infectives in class I to the removed class R is proportional to the number of infectives in class I, that is aI(t), where a is a positive number.']),\n",
       " ('I(t)',\n",
       "  'instance of',\n",
       "  'positive number',\n",
       "  ['It can be shown that the fraction of people remaining in the exposed class E s time unit after entering class E is e -bs , so the length of the latent period is distributed exponentially with mean equals to e -bs ds = 1/b (c.f.',\n",
       "   '[11] ).(?<=[.?!])\\\\s+(vi) The rate of removal of infectives in class I to the removed class R is proportional to the number of infectives in class I, that is aI(t), where a is a positive number.']),\n",
       " ('infectious period',\n",
       "  'subclass of',\n",
       "  'time unit',\n",
       "  ['It can be shown that the fraction of people remaining in the infective class I s time unit after entering class I is e -as , so the length of the infectious period is distributed exponentially with mean equals to e -as ds = 1/a.',\n",
       "   '(vii) The rate of removal of people in infective class I P to the removal class R P is proportional to the number of people in class I P , that is a P I P (t), where a P is a positive number.']),\n",
       " ('P',\n",
       "  'instance of',\n",
       "  'positive number',\n",
       "  ['It can be shown that the fraction of people remaining in the infective class I s time unit after entering class I is e -as , so the length of the infectious period is distributed exponentially with mean equals to e -as ds = 1/a.',\n",
       "   '(vii) The rate of removal of people in infective class I P to the removal class R P is proportional to the number of people in class I P , that is a P I P (t), where a P is a positive number.']),\n",
       " ('symptoms',\n",
       "  'has cause',\n",
       "  'infection',\n",
       "  ['(viii) The incubation period (the time from first infection to the appearances of symptoms) plus the onset to admission interval is equal to the sum of the latent period and the infectious period and is therefore equal to We now assume the various classes as uniformly mixed: that is each pair of individuals has equal probability of coming into contact with one another.',\n",
       "   'The model mechanism is then governed by the following system of nonlinear ordinary differential equations: The above system of differential equations are derived from the so-called simple mass action incidence approach which is described in [11] .']),\n",
       " ('nonlinear',\n",
       "  'subclass of',\n",
       "  'ordinary differential equations',\n",
       "  ['(viii) The incubation period (the time from first infection to the appearances of symptoms) plus the onset to admission interval is equal to the sum of the latent period and the infectious period and is therefore equal to We now assume the various classes as uniformly mixed: that is each pair of individuals has equal probability of coming into contact with one another.',\n",
       "   'The model mechanism is then governed by the following system of nonlinear ordinary differential equations: The above system of differential equations are derived from the so-called simple mass action incidence approach which is described in [11] .']),\n",
       " ('r',\n",
       "  'instance of',\n",
       "  'constant',\n",
       "  ['The interpretation of the constant r is as follows.',\n",
       "   'An average infective of disease A makes contact sufficient to transmit infection with rN others per unit time, where N is the total population.']),\n",
       " ('infective',\n",
       "  'subclass of',\n",
       "  'transmission',\n",
       "  ['The interpretation of the constant r is as follows.',\n",
       "   'An average infective of disease A makes contact sufficient to transmit infection with rN others per unit time, where N is the total population.']),\n",
       " ('infective',\n",
       "  'has effect',\n",
       "  'transmit infection',\n",
       "  ['Note that the probability that a random contact by an infective with a susceptible, who can then transmit infection, is S/ N, therefore the number of new infections in unit time is (rN)(S/N)I = rSI.',\n",
       "   'One can also interpret r P in a similar way.']),\n",
       " ('transmit infection',\n",
       "  'has cause',\n",
       "  'infective',\n",
       "  ['Note that the probability that a random contact by an infective with a susceptible, who can then transmit infection, is S/ N, therefore the number of new infections in unit time is (rN)(S/N)I = rSI.',\n",
       "   'One can also interpret r P in a similar way.']),\n",
       " ('P',\n",
       "  'has part',\n",
       "  'r',\n",
       "  ['Note that the probability that a random contact by an infective with a susceptible, who can then transmit infection, is S/ N, therefore the number of new infections in unit time is (rN)(S/N)I = rSI.',\n",
       "   'One can also interpret r P in a similar way.']),\n",
       " ('infection',\n",
       "  'has effect',\n",
       "  'disease',\n",
       "  ['Here r and r P are related to the infection rate of disease A and B respectively, while a, a P and b are the removal rate of individuals in class I, I P and E respectively.',\n",
       "   'The last two equations follow from the assumption that the population size N is constant.(?<=[.?!])\\\\s+We are, of course, only interested in non-negative solutions for S, E, I,R, I P and R P .']),\n",
       " ('non-negative',\n",
       "  'subclass of',\n",
       "  'population size',\n",
       "  ['Here r and r P are related to the infection rate of disease A and B respectively, while a, a P and b are the removal rate of individuals in class I, I P and E respectively.',\n",
       "   'The last two equations follow from the assumption that the population size N is constant.(?<=[.?!])\\\\s+We are, of course, only interested in non-negative solutions for S, E, I,R, I P and R P .']),\n",
       " ('latency',\n",
       "  'facet of',\n",
       "  'disease',\n",
       "  ['In the SEIRP model, there is no latency for the disease B.',\n",
       "   'The inclusion of the latency for the disease B will imply the introduction of one additional parameter b P and will further complicate the model.']),\n",
       " ('latency',\n",
       "  'subclass of',\n",
       "  'parameter',\n",
       "  ['In the SEIRP model, there is no latency for the disease B.',\n",
       "   'The inclusion of the latency for the disease B will imply the introduction of one additional parameter b P and will further complicate the model.']),\n",
       " ('indirect evidence',\n",
       "  'subclass of',\n",
       "  'evidence',\n",
       "  ['Since the main purpose of the paper is to provide some indirect evidence that a double epidemic may exist, our simpler model may already be good enough for this purpose because of the following reasons.',\n",
       "   'Firstly, even if the diseases A and B are generated from similar viruses, they can still be quite different (for instance at the level of the physical location of the infection) and therefore they need not share similar models.']),\n",
       " ('virus',\n",
       "  'has effect',\n",
       "  'disease',\n",
       "  ['Since the main purpose of the paper is to provide some indirect evidence that a double epidemic may exist, our simpler model may already be good enough for this purpose because of the following reasons.',\n",
       "   'Firstly, even if the diseases A and B are generated from similar viruses, they can still be quite different (for instance at the level of the physical location of the infection) and therefore they need not share similar models.']),\n",
       " ('B',\n",
       "  'opposite of',\n",
       "  'epidemic A',\n",
       "  ['Secondly, since we only want to study phenomena that immunize people from the initial S class, the details of the (SIR or SEIR-like) spread of the protective disease B are less important that those of the propagation of the epidemic A.',\n",
       "   'Of course, future studies may try to analyze at a finer level the spread of the protective epidemic once its existence and characteristics have been assessed in practice.']),\n",
       " ('epidemic A',\n",
       "  'opposite of',\n",
       "  'B',\n",
       "  ['Secondly, since we only want to study phenomena that immunize people from the initial S class, the details of the (SIR or SEIR-like) spread of the protective disease B are less important that those of the propagation of the epidemic A.',\n",
       "   'Of course, future studies may try to analyze at a finer level the spread of the protective epidemic once its existence and characteristics have been assessed in practice.']),\n",
       " ('protective epidemic',\n",
       "  'instance of',\n",
       "  'epidemic',\n",
       "  ['Secondly, since we only want to study phenomena that immunize people from the initial S class, the details of the (SIR or SEIR-like) spread of the protective disease B are less important that those of the propagation of the epidemic A.',\n",
       "   'Of course, future studies may try to analyze at a finer level the spread of the protective epidemic once its existence and characteristics have been assessed in practice.']),\n",
       " ('Susceptibles',\n",
       "  'instance of',\n",
       "  'class',\n",
       "  ['Note that two types of protections \"static\" and \"dynamic\" can be seen as particular cases of a double epidemic model: -the \"static\" protection is given by a protective epidemic (that has already finished its spreading) or a vaccine leaving some of the population immune; this amounts to only consider a fraction of the initial population in the Susceptibles class S (and to set I P (0) = 0 because the spread has already taken place); equivalently this can be implemented by putting at the initial time some of the population in the Removed class R (i.e., R(0) > 0).',\n",
       "   '-the \"dynamic\" protection is when the protective epidemic spreads at the same time with the SARS giving rise to various interference patterns.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'epidemic',\n",
       "  ['Note that two types of protections \"static\" and \"dynamic\" can be seen as particular cases of a double epidemic model: -the \"static\" protection is given by a protective epidemic (that has already finished its spreading) or a vaccine leaving some of the population immune; this amounts to only consider a fraction of the initial population in the Susceptibles class S (and to set I P (0) = 0 because the spread has already taken place); equivalently this can be implemented by putting at the initial time some of the population in the Removed class R (i.e., R(0) > 0).',\n",
       "   '-the \"dynamic\" protection is when the protective epidemic spreads at the same time with the SARS giving rise to various interference patterns.']),\n",
       " ('Dynamic',\n",
       "  'instance of',\n",
       "  'protection assumption',\n",
       "  ['One may wonder if the \"dynamic\" protection assumption is really necessary to get a good fit of the data.',\n",
       "   'We have tested the \"static\" protection assumption by setting the total population to several initial values less than the real population (see also the case a) of Hong-Kong simulations below) but the fit quality was generally less satisfactory than in the case of the co-existing epidemics (i.e.']),\n",
       " ('case a',\n",
       "  'instance of',\n",
       "  'Hong-Kong simulations',\n",
       "  ['One may wonder if the \"dynamic\" protection assumption is really necessary to get a good fit of the data.',\n",
       "   'We have tested the \"static\" protection assumption by setting the total population to several initial values less than the real population (see also the case a) of Hong-Kong simulations below) but the fit quality was generally less satisfactory than in the case of the co-existing epidemics (i.e.']),\n",
       " ('outbreak of SARS',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['\"dynamic\" effect) ; more testing is needed to fully certify this conclusion.(?<=[.?!])\\\\s+We shall apply the SEIRP model to study the outbreak of SARS in Hong Kong.',\n",
       "   'In particular, we would like to explain the fact that a more or less constant background infection level remained present in Hong Kong for a long period of time, on which superimposes large local outbreaks at the several hospitals and the Amoy Gardens, a housing estate in Hong Kong.']),\n",
       " ('Amoy Gardens',\n",
       "  'instance of',\n",
       "  'housing estate',\n",
       "  ['\"dynamic\" effect) ; more testing is needed to fully certify this conclusion.(?<=[.?!])\\\\s+We shall apply the SEIRP model to study the outbreak of SARS in Hong Kong.',\n",
       "   'In particular, we would like to explain the fact that a more or less constant background infection level remained present in Hong Kong for a long period of time, on which superimposes large local outbreaks at the several hospitals and the Amoy Gardens, a housing estate in Hong Kong.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['If we only consider those SARS cases from the community, i.e.',\n",
       "   'we take away the number of confirmed SARS cases from the Amoy Gardens and those from the local hospitals, then most of the daily new number of confirmed cases from the community stayed around 15 to 25 (of course this level went down after strong confinement directives were implemented by the Hong Kong government).']),\n",
       " ('Amoy Gardens',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['If we only consider those SARS cases from the community, i.e.',\n",
       "   'we take away the number of confirmed SARS cases from the Amoy Gardens and those from the local hospitals, then most of the daily new number of confirmed cases from the community stayed around 15 to 25 (of course this level went down after strong confinement directives were implemented by the Hong Kong government).']),\n",
       " ('outbreak of SARS in Hong Kong',\n",
       "  'has cause',\n",
       "  'SARS virus',\n",
       "  ['To apply our model to study the outbreak of SARS in Hong Kong, we assume that disease A is caused by SARS virus, while disease B is caused by some unknown virus and disease B \"protects\" people from being infected by SARS provided that they are infected by disease B first.',\n",
       "   'We accept that A can be derived from B, after a mutation and/ or recombination event.']),\n",
       " ('SARS virus',\n",
       "  'has effect',\n",
       "  'outbreak of SARS in Hong Kong',\n",
       "  ['To apply our model to study the outbreak of SARS in Hong Kong, we assume that disease A is caused by SARS virus, while disease B is caused by some unknown virus and disease B \"protects\" people from being infected by SARS provided that they are infected by disease B first.',\n",
       "   'We accept that A can be derived from B, after a mutation and/ or recombination event.']),\n",
       " ('recombination',\n",
       "  'has effect',\n",
       "  'mutation',\n",
       "  ['To apply our model to study the outbreak of SARS in Hong Kong, we assume that disease A is caused by SARS virus, while disease B is caused by some unknown virus and disease B \"protects\" people from being infected by SARS provided that they are infected by disease B first.',\n",
       "   'We accept that A can be derived from B, after a mutation and/ or recombination event.']),\n",
       " ('Hong Kong',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['Since we do not know how many Hong Kong people are infected by virus B, we shall consider the following two scenarios: a) 0.5 million Hong Kong people are infected by disease B.',\n",
       "   'b) only 10 Hong Kong people are infected by disease B.(?<=[.?!])\\\\s+First, we assume that I P (0) = 0.5 million, that is 0.5 million Hong Kong people are infected by disease B.']),\n",
       " ('Hong Kong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['Since we do not know how many Hong Kong people are infected by virus B, we shall consider the following two scenarios: a) 0.5 million Hong Kong people are infected by disease B.',\n",
       "   'b) only 10 Hong Kong people are infected by disease B.(?<=[.?!])\\\\s+First, we assume that I P (0) = 0.5 million, that is 0.5 million Hong Kong people are infected by disease B.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['Since we do not know how many Hong Kong people are infected by virus B, we shall consider the following two scenarios: a) 0.5 million Hong Kong people are infected by disease B.',\n",
       "   'b) only 10 Hong Kong people are infected by disease B.(?<=[.?!])\\\\s+First, we assume that I P (0) = 0.5 million, that is 0.5 million Hong Kong people are infected by disease B.']),\n",
       " ('disease B',\n",
       "  'has cause',\n",
       "  'gastro-enteritis virus',\n",
       "  ['This assumption is quite reasonable because we believe that disease B is caused by a gastro-enteritis virus which is usually highly contagious.',\n",
       "   'Since the population of Hong Kong is about 6.8 million, we therefore assume that S(0) = 6.8 -0.5 = 6.3 million.(?<=[.?!])\\\\s+We then assume rather arbitrarily that E(0) = 100 and I(0) = 50.']),\n",
       " ('gastro-enteritis virus',\n",
       "  'has effect',\n",
       "  'disease B',\n",
       "  ['This assumption is quite reasonable because we believe that disease B is caused by a gastro-enteritis virus which is usually highly contagious.',\n",
       "   'Since the population of Hong Kong is about 6.8 million, we therefore assume that S(0) = 6.8 -0.5 = 6.3 million.(?<=[.?!])\\\\s+We then assume rather arbitrarily that E(0) = 100 and I(0) = 50.']),\n",
       " ('population of Hong Kong',\n",
       "  'located in the administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['This assumption is quite reasonable because we believe that disease B is caused by a gastro-enteritis virus which is usually highly contagious.',\n",
       "   'Since the population of Hong Kong is about 6.8 million, we therefore assume that S(0) = 6.8 -0.5 = 6.3 million.(?<=[.?!])\\\\s+We then assume rather arbitrarily that E(0) = 100 and I(0) = 50.']),\n",
       " ('gradient-based optimization',\n",
       "  'subclass of',\n",
       "  'algorithm',\n",
       "  ['With these initial conditions, we tried to search the parameter values by a gradient-based optimization algorithm.',\n",
       "   'The result is if we use the parameter values r = 10.19 × 10 -8 , r P = 7.079 × 10 -8 , a = 0.47, a P = 0.461 and b = 0.103, then the resulting curve for R fits very well with the observed total number of confirmed cases of SARS from the community.']),\n",
       " ('confirmed cases',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['With these initial conditions, we tried to search the parameter values by a gradient-based optimization algorithm.',\n",
       "   'The result is if we use the parameter values r = 10.19 × 10 -8 , r P = 7.079 × 10 -8 , a = 0.47, a P = 0.461 and b = 0.103, then the resulting curve for R fits very well with the observed total number of confirmed cases of SARS from the community.']),\n",
       " ('2003',\n",
       "  'point in time',\n",
       "  '2003',\n",
       "  ['We fit the model with the total number of confirmed cases from 17 March, 2003 to 10 May, 2003 (totally 55 days).',\n",
       "   'Figure  4 is the curve for R, while figure 5 shows the expected and observed daily new cases.(?<=[.?!])\\\\s+The limiting value of the curve R is 1011.']),\n",
       " ('1011',\n",
       "  'instance of',\n",
       "  'limit',\n",
       "  ['We fit the model with the total number of confirmed cases from 17 March, 2003 to 10 May, 2003 (totally 55 days).',\n",
       "   'Figure  4 is the curve for R, while figure 5 shows the expected and observed daily new cases.(?<=[.?!])\\\\s+The limiting value of the curve R is 1011.']),\n",
       " ('onset',\n",
       "  'followed by',\n",
       "  'admission',\n",
       "  ['Recall that the incubation period plus the time from onset to admission can be estimated by 1/a + 1/b.(?<=[.?!])\\\\s+If we take a = 0.47 and b = 0.103, then 1/a + 1/b = 11.83.',\n",
       "   'On the other hand, from the statistics obtained in, the observed mean of the incubation period for SARS is 6.37.(?<=[.?!])\\\\s+The observed mean of the time from onset to admission is about 3.75.']),\n",
       " ('mean',\n",
       "  'facet of',\n",
       "  'statistics',\n",
       "  ['Recall that the incubation period plus the time from onset to admission can be estimated by 1/a + 1/b.(?<=[.?!])\\\\s+If we take a = 0.47 and b = 0.103, then 1/a + 1/b = 11.83.',\n",
       "   'On the other hand, from the statistics obtained in, the observed mean of the incubation period for SARS is 6.37.(?<=[.?!])\\\\s+The observed mean of the time from onset to admission is about 3.75.']),\n",
       " ('11.83',\n",
       "  'instance of',\n",
       "  'sum',\n",
       "  ['Therefore, the observed sum 10.12 = 6.37+3.75 which is close to the estimation 11.83.',\n",
       "   'We consider here the extreme case that I P (0) = 10, that is only 10 Hong Kong people are infected by disease B.']),\n",
       " ('Hong Kong',\n",
       "  'population',\n",
       "  '10',\n",
       "  ['Therefore, the observed sum 10.12 = 6.37+3.75 which is close to the estimation 11.83.',\n",
       "   'We consider here the extreme case that I P (0) = 10, that is only 10 Hong Kong people are infected by disease B.']),\n",
       " ('S(0)',\n",
       "  'has part',\n",
       "  'E(0)',\n",
       "  ['S(0) is assumed to be 6.8 million and we still assume that E(0) = 100 and I(0) = 50.',\n",
       "   'The parameters obtained by the gradient-based optimization algorithm are as follows: r = 10.08 × 10 -8 , r P = 7.94 × 10 -8 , a = 0.52, a P = 0.12 and b = 0.105.']),\n",
       " ('gradient-based optimization',\n",
       "  'subclass of',\n",
       "  'algorithm',\n",
       "  ['S(0) is assumed to be 6.8 million and we still assume that E(0) = 100 and I(0) = 50.',\n",
       "   'The parameters obtained by the gradient-based optimization algorithm are as follows: r = 10.08 × 10 -8 , r P = 7.94 × 10 -8 , a = 0.52, a P = 0.12 and b = 0.105.']),\n",
       " ('confirmed cases of SARS',\n",
       "  'start time',\n",
       "  '17 March, 2003',\n",
       "  ['The resulting curve for R fits well with the observed total number of confirmed cases of SARS from the community from 17 March, 2003 to 10 May, 2003 (totally 55 days).',\n",
       "   'Figure 6 is the curve for R, while Figure 7 shows the expected and observed daily new cases.(?<=[.?!])\\\\s+The limiting value of the curve R is 1033.']),\n",
       " ('1033',\n",
       "  'instance of',\n",
       "  'limiting value',\n",
       "  ['The resulting curve for R fits well with the observed total number of confirmed cases of SARS from the community from 17 March, 2003 to 10 May, 2003 (totally 55 days).',\n",
       "   'Figure 6 is the curve for R, while Figure 7 shows the expected and observed daily new cases.(?<=[.?!])\\\\s+The limiting value of the curve R is 1033.']),\n",
       " ('figure 10',\n",
       "  'point in time',\n",
       "  '10',\n",
       "  ['Note that a = 0.52 and b = 0.105 and therefore 1/a + 1/b = 11.44 which is close to the observed figure 10.12.',\n",
       "   'Both the parameters r P and a are greater than the corresponding parameters in case a.(?<=[.?!])\\\\s+This is reasonable as the initial population I P (0) = 10 is much smaller than the previous case.']),\n",
       " ('population',\n",
       "  'different from',\n",
       "  'initial population',\n",
       "  ['Note that a = 0.52 and b = 0.105 and therefore 1/a + 1/b = 11.44 which is close to the observed figure 10.12.',\n",
       "   'Both the parameters r P and a are greater than the corresponding parameters in case a.(?<=[.?!])\\\\s+This is reasonable as the initial population I P (0) = 10 is much smaller than the previous case.']),\n",
       " ('outbreak of SARS',\n",
       "  'location',\n",
       "  'Beijing',\n",
       "  ['Therefore disease B needs to spread much faster in order to \"protect\" people from being infected by SARS.(?<=[.?!])\\\\s+We have also applied the SEIRP model to study the outbreak of SARS in Beijing.',\n",
       "   'The statistics on the numbers of SARS cases in Beijing are obtained from http:// www.moh.gov.cn/zhgl/yqfb/index.htm.']),\n",
       " ('SARS cases in Beijing',\n",
       "  'location',\n",
       "  'Beijing',\n",
       "  ['Therefore disease B needs to spread much faster in order to \"protect\" people from being infected by SARS.(?<=[.?!])\\\\s+We have also applied the SEIRP model to study the outbreak of SARS in Beijing.',\n",
       "   'The statistics on the numbers of SARS cases in Beijing are obtained from http:// www.moh.gov.cn/zhgl/yqfb/index.htm.']),\n",
       " ('E(0)',\n",
       "  'subclass of',\n",
       "  'population',\n",
       "  ['The initial population S(0) was set to be 13.82 millions; the other parameters (including the initial conditions) that characterise the model were optimised to obtain the best fit; we obtained thus E(0) = 400; I P (0) = 35000; I(0) = 340; R(0) = 0; R P (0) = 0; r = 7.69 × 10 -8 ; r P = 10.79 × 10 -8 ; a = 0.488; a P = 0.618; b = 0.103.',\n",
       "   'Figure 8 and 9 show that the model fit well with the observation in both the numbers cumulative cases and the daily new cases.']),\n",
       " ('cumulative case',\n",
       "  'followed by',\n",
       "  'daily new case',\n",
       "  ['The initial population S(0) was set to be 13.82 millions; the other parameters (including the initial conditions) that characterise the model were optimised to obtain the best fit; we obtained thus E(0) = 400; I P (0) = 35000; I(0) = 340; R(0) = 0; R P (0) = 0; r = 7.69 × 10 -8 ; r P = 10.79 × 10 -8 ; a = 0.488; a P = 0.618; b = 0.103.',\n",
       "   'Figure 8 and 9 show that the model fit well with the observation in both the numbers cumulative cases and the daily new cases.']),\n",
       " ('P',\n",
       "  'part of',\n",
       "  'R',\n",
       "  ['The total number of people infected with the protective virus is R P = 12 millions.',\n",
       "   'If the values of the parameters defining the model remain the same this simulation results in a final value of the total people infected with the SARS virus (that is R ∞ ) at the level of 2700.']),\n",
       " ('SARS',\n",
       "  'instance of',\n",
       "  'virus',\n",
       "  ['The total number of people infected with the protective virus is R P = 12 millions.',\n",
       "   'If the values of the parameters defining the model remain the same this simulation results in a final value of the total people infected with the SARS virus (that is R ∞ ) at the level of 2700.']),\n",
       " ('outbreak of SARS in Inner Mongolia',\n",
       "  'location',\n",
       "  'Inner Mongolia',\n",
       "  ['Here we consider the outbreak of SARS in Inner Mongolia.(?<=[.?!])\\\\s+The statistics on the numbers of SARS cases in Inner Mongolia are obtained from http://www.moh.gov.cn/zhgl/ yqfb/index.htm.',\n",
       "   'The initial population S(0) was The total population was taken to be S(0) = 23.67 millions ; the other parameters (including the initial conditions) were optimised to ensure a good agreement with the data and were obtained to be E(0) = 2; I P (0) = 137638; I(0) = 32; R(0) = 0; R P (0) = 0; r = 1.62 × 10 -8 ; r P = 3.87 × 10 -8 ; a = 0.120, a P = 0.272; b = 7.644.']),\n",
       " ('E(0)',\n",
       "  'facet of',\n",
       "  'population',\n",
       "  ['Here we consider the outbreak of SARS in Inner Mongolia.(?<=[.?!])\\\\s+The statistics on the numbers of SARS cases in Inner Mongolia are obtained from http://www.moh.gov.cn/zhgl/ yqfb/index.htm.',\n",
       "   'The initial population S(0) was The total population was taken to be S(0) = 23.67 millions ; the other parameters (including the initial conditions) were optimised to ensure a good agreement with the data and were obtained to be E(0) = 2; I P (0) = 137638; I(0) = 32; R(0) = 0; R P (0) = 0; r = 1.62 × 10 -8 ; r P = 3.87 × 10 -8 ; a = 0.120, a P = 0.272; b = 7.644.']),\n",
       " ('figure 10',\n",
       "  'followed by',\n",
       "  'figure 11',\n",
       "  ['The graphical representation of the simulation is given in figure 10 and figure 11 .',\n",
       "   'If the parameters above are used to simulate the future spread of epidemic we obtain the value of R ∞ to be 350.']),\n",
       " ('epidemic',\n",
       "  'subclass of',\n",
       "  'spread',\n",
       "  ['The graphical representation of the simulation is given in figure 10 and figure 11 .',\n",
       "   'If the parameters above are used to simulate the future spread of epidemic we obtain the value of R ∞ to be 350.']),\n",
       " ('P',\n",
       "  'subclass of',\n",
       "  'parameters',\n",
       "  ['Observed Expected values of a and b for all the different scenarios before the parameters r, r P and a P are chosen to fit the data.',\n",
       "   'The second approach is simply choosing all the parameters to fit the data.']),\n",
       " ('parameters',\n",
       "  'part of',\n",
       "  'data',\n",
       "  ['Observed Expected values of a and b for all the different scenarios before the parameters r, r P and a P are chosen to fit the data.',\n",
       "   'The second approach is simply choosing all the parameters to fit the data.']),\n",
       " ('a',\n",
       "  'instance of',\n",
       "  'parameters',\n",
       "  ['We chose the second approach because we do believe that there are also some good reasons for the variations of the parameters a and b in different cases.',\n",
       "   'The parameters a and a P describe the removal from the classes I and I P to the classes R and R P respectively.']),\n",
       " ('I',\n",
       "  'has part',\n",
       "  'I P',\n",
       "  ['We chose the second approach because we do believe that there are also some good reasons for the variations of the parameters a and b in different cases.',\n",
       "   'The parameters a and a P describe the removal from the classes I and I P to the classes R and R P respectively.']),\n",
       " ('I P',\n",
       "  'part of',\n",
       "  'I',\n",
       "  ['We chose the second approach because we do believe that there are also some good reasons for the variations of the parameters a and b in different cases.',\n",
       "   'The parameters a and a P describe the removal from the classes I and I P to the classes R and R P respectively.']),\n",
       " ('infection',\n",
       "  'subclass of',\n",
       "  'disease',\n",
       "  ['Since the removed classes R are considered to contain the individuals with infections the parameters a and a P characterize the identification rate of potential cases; it is then rather related to the health policy than to the disease itself (e.g.',\n",
       "   'a is not the mortality rate!).']),\n",
       " ('mortality rate',\n",
       "  'different from',\n",
       "  'mortality',\n",
       "  ['Since the removed classes R are considered to contain the individuals with infections the parameters a and a P characterize the identification rate of potential cases; it is then rather related to the health policy than to the disease itself (e.g.',\n",
       "   'a is not the mortality rate!).']),\n",
       " ('mortality',\n",
       "  'different from',\n",
       "  'mortality rate',\n",
       "  ['Since the removed classes R are considered to contain the individuals with infections the parameters a and a P characterize the identification rate of potential cases; it is then rather related to the health policy than to the disease itself (e.g.',\n",
       "   'a is not the mortality rate!).']),\n",
       " ('simulation',\n",
       "  'subclass of',\n",
       "  'simulation',\n",
       "  ['Since health policies vary from places to places and the parameters a and a P are not directly related to the disease itself these parameters are not to be considered known but will rather be computed by fitting the simulation results to the data.',\n",
       "   'The parameter b describes the evolution from the exposed class E to the infectives class I; its reciprocal 1/b is related to the latent period of the disease; as such it is indeed more characteristic of the disease itself than the parameter a but it may still be affected by the geographic region.']),\n",
       " ('class E',\n",
       "  'followed by',\n",
       "  'class I',\n",
       "  ['Since health policies vary from places to places and the parameters a and a P are not directly related to the disease itself these parameters are not to be considered known but will rather be computed by fitting the simulation results to the data.',\n",
       "   'The parameter b describes the evolution from the exposed class E to the infectives class I; its reciprocal 1/b is related to the latent period of the disease; as such it is indeed more characteristic of the disease itself than the parameter a but it may still be affected by the geographic region.']),\n",
       " ('class I',\n",
       "  'follows',\n",
       "  'class E',\n",
       "  ['Since health policies vary from places to places and the parameters a and a P are not directly related to the disease itself these parameters are not to be considered known but will rather be computed by fitting the simulation results to the data.',\n",
       "   'The parameter b describes the evolution from the exposed class E to the infectives class I; its reciprocal 1/b is related to the latent period of the disease; as such it is indeed more characteristic of the disease itself than the parameter a but it may still be affected by the geographic region.']),\n",
       " ('Beijing',\n",
       "  'shares border with',\n",
       "  'Inner Mongolia',\n",
       "  ['For instance, the lifestyles of the people in Beijing and Inner Mongolia are very different because Beijing is the capital, and hence much richer (with better hygiene, but with more important air pollution).',\n",
       "   'The value of the b parameter will also be obtained from the fitting procedure.']),\n",
       " ('Inner Mongolia',\n",
       "  'shares border with',\n",
       "  'Beijing',\n",
       "  ['For instance, the lifestyles of the people in Beijing and Inner Mongolia are very different because Beijing is the capital, and hence much richer (with better hygiene, but with more important air pollution).',\n",
       "   'The value of the b parameter will also be obtained from the fitting procedure.']),\n",
       " ('b parameter',\n",
       "  'facet of',\n",
       "  'fitting procedure',\n",
       "  ['For instance, the lifestyles of the people in Beijing and Inner Mongolia are very different because Beijing is the capital, and hence much richer (with better hygiene, but with more important air pollution).',\n",
       "   'The value of the b parameter will also be obtained from the fitting procedure.']),\n",
       " ('double epidemic',\n",
       "  'instance of',\n",
       "  'hypothesis',\n",
       "  ['Finally, our approach that not fixing the parameters a and b at the very beginning but rather than estimating them from the data can let us to indirectly verify the double epidemic hypothesis.',\n",
       "   'In fact, as already pointed out in the paper, 1/a+1/b can be considered as the incubation period plus the onset to admission interval.']),\n",
       " ('incubation period',\n",
       "  'followed by',\n",
       "  'onset to admission interval',\n",
       "  ['Finally, our approach that not fixing the parameters a and b at the very beginning but rather than estimating them from the data can let us to indirectly verify the double epidemic hypothesis.',\n",
       "   'In fact, as already pointed out in the paper, 1/a+1/b can be considered as the incubation period plus the onset to admission interval.']),\n",
       " ('Hong Kong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['For the two Hong Kong scenarios we considered, our estimations are quite close to the observed figures.',\n",
       "   'Note that in Hong Kong, r >r P but this relation is reversed in Beijing and Inner Mongolia.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['For the two Hong Kong scenarios we considered, our estimations are quite close to the observed figures.',\n",
       "   'Note that in Hong Kong, r >r P but this relation is reversed in Beijing and Inner Mongolia.']),\n",
       " ('Beijing',\n",
       "  'shares border with',\n",
       "  'Inner Mongolia',\n",
       "  ['For the two Hong Kong scenarios we considered, our estimations are quite close to the observed figures.',\n",
       "   'Note that in Hong Kong, r >r P but this relation is reversed in Beijing and Inner Mongolia.']),\n",
       " ('Inner Mongolia',\n",
       "  'shares border with',\n",
       "  'Beijing',\n",
       "  ['For the two Hong Kong scenarios we considered, our estimations are quite close to the observed figures.',\n",
       "   'Note that in Hong Kong, r >r P but this relation is reversed in Beijing and Inner Mongolia.']),\n",
       " ('SARS',\n",
       "  'country of origin',\n",
       "  'Hong Kong',\n",
       "  ['One possible reason for these differences is that the values of r and r P depend on Number of SARS cases in Hong Kong community (and the simulated case \"a\") per three days While parameters r and r P are related to the infective rates that may depend on the hygienic situation, the a\\'s and b are closer to clinical properties of the infection.',\n",
       "   'That they differ quite a lot in different scenarios is surprising.']),\n",
       " ('different scenarios',\n",
       "  'subclass of',\n",
       "  'scenarios',\n",
       "  ['One possible reason for these differences is that the values of r and r P depend on Number of SARS cases in Hong Kong community (and the simulated case \"a\") per three days While parameters r and r P are related to the infective rates that may depend on the hygienic situation, the a\\'s and b are closer to clinical properties of the infection.',\n",
       "   'That they differ quite a lot in different scenarios is surprising.']),\n",
       " ('Inner Mongolia',\n",
       "  'shares border with',\n",
       "  'Beijing',\n",
       "  ['In particular, the b value for Inner Mongolia is 76 times that of Beijing, implying that latency lasts so much longer in Beijing.',\n",
       "   'One explanation is there may be a difference in the spread of the \"protective\" epidemic, that might have started earlier at one of those places.']),\n",
       " ('Beijing',\n",
       "  'shares border with',\n",
       "  'Inner Mongolia',\n",
       "  ['In particular, the b value for Inner Mongolia is 76 times that of Beijing, implying that latency lasts so much longer in Beijing.',\n",
       "   'One explanation is there may be a difference in the spread of the \"protective\" epidemic, that might have started earlier at one of those places.']),\n",
       " ('\"protective\" epidemic',\n",
       "  'instance of',\n",
       "  'spread',\n",
       "  ['In particular, the b value for Inner Mongolia is 76 times that of Beijing, implying that latency lasts so much longer in Beijing.',\n",
       "   'One explanation is there may be a difference in the spread of the \"protective\" epidemic, that might have started earlier at one of those places.']),\n",
       " ('immunodepression',\n",
       "  'subclass of',\n",
       "  'latent',\n",
       "  ['Another explanation is due to the latent immunodepression of the people there.',\n",
       "   'This means that people do not build up a strong immune response when infected, because they are either infected by other viruses (this is the case of AIDS) or because they are generally in poor health.']),\n",
       " ('AIDS',\n",
       "  'has cause',\n",
       "  'virus',\n",
       "  ['Another explanation is due to the latent immunodepression of the people there.',\n",
       "   'This means that people do not build up a strong immune response when infected, because they are either infected by other viruses (this is the case of AIDS) or because they are generally in poor health.']),\n",
       " ('severe draught',\n",
       "  'has part',\n",
       "  'recently more or less submitted to a severe draught',\n",
       "  ['Note that they were recently more or less submitted to a severe draught, which meant that they were probably undernourished.',\n",
       "   'The Severe Acute Respiratory Syndrome outbreak is the first epidemic of the XXIst century.']),\n",
       " ('Severe Acute Respiratory Syndrome outbreak',\n",
       "  'instance of',\n",
       "  'epidemic',\n",
       "  ['Note that they were recently more or less submitted to a severe draught, which meant that they were probably undernourished.',\n",
       "   'The Severe Acute Respiratory Syndrome outbreak is the first epidemic of the XXIst century.']),\n",
       " ('next autumn',\n",
       "  'follows',\n",
       "  'next year',\n",
       "  ['We do not know whether it will start again next autumn, and we need to combine a variety of approaches to tackle the epidemic, making it as short as possible, and preparing for future similar, or less similar outbreaks.',\n",
       "   'In this context, mathematical models can have several functions.(?<=[.?!])\\\\s+They may be phenomenological and descriptive, to predict the immediate future [7] .']),\n",
       " ('next year',\n",
       "  'followed by',\n",
       "  'next autumn',\n",
       "  ['We do not know whether it will start again next autumn, and we need to combine a variety of approaches to tackle the epidemic, making it as short as possible, and preparing for future similar, or less similar outbreaks.',\n",
       "   'In this context, mathematical models can have several functions.(?<=[.?!])\\\\s+They may be phenomenological and descriptive, to predict the immediate future [7] .']),\n",
       " ('descriptive',\n",
       "  'subclass of',\n",
       "  'phenomenological',\n",
       "  ['We do not know whether it will start again next autumn, and we need to combine a variety of approaches to tackle the epidemic, making it as short as possible, and preparing for future similar, or less similar outbreaks.',\n",
       "   'In this context, mathematical models can have several functions.(?<=[.?!])\\\\s+They may be phenomenological and descriptive, to predict the immediate future [7] .']),\n",
       " ('hypotheses',\n",
       "  'has effect',\n",
       "  'consequences',\n",
       "  ['They may also propose hypotheses and explore their consequences in order to provide political and medical authorities with orientations that may help them to take decisions, at a time when a situation is rapidly evolving.',\n",
       "   'The model we explore here is meant to that purpose.']),\n",
       " ('The Model',\n",
       "  'instance of',\n",
       "  'model',\n",
       "  ['They may also propose hypotheses and explore their consequences in order to provide political and medical authorities with orientations that may help them to take decisions, at a time when a situation is rapidly evolving.',\n",
       "   'The model we explore here is meant to that purpose.']),\n",
       " ('Susceptible-Infected-Removed (SIR) epidemic model',\n",
       "  'instance of',\n",
       "  'epidemiological model',\n",
       "  ['We started to try to account for the data collected on the disease using the standard epidemiological model, the Susceptible-Infected-Removed (SIR) epidemic model [12, 13, 17] .',\n",
       "   'As shown in the first section of our work, data do not fit with the model.']),\n",
       " ('data',\n",
       "  'part of',\n",
       "  'model',\n",
       "  ['We started to try to account for the data collected on the disease using the standard epidemiological model, the Susceptible-Infected-Removed (SIR) epidemic model [12, 13, 17] .',\n",
       "   'As shown in the first section of our work, data do not fit with the model.']),\n",
       " ('trailing',\n",
       "  'subclass of',\n",
       "  'cases',\n",
       "  ['In particular, the long \"tail\" of the number of cases that appears to be trailing for a long time cannot be easily accounted for in this framework.',\n",
       "   'In addition there are huge discrepancies, within the same The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\") The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\").']),\n",
       " ('Hong Kong',\n",
       "  'country',\n",
       "  'China',\n",
       "  ['In particular, the long \"tail\" of the number of cases that appears to be trailing for a long time cannot be easily accounted for in this framework.',\n",
       "   'In addition there are huge discrepancies, within the same The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\") The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\").']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Hong Kong',\n",
       "  ['In particular, the long \"tail\" of the number of cases that appears to be trailing for a long time cannot be easily accounted for in this framework.',\n",
       "   'In addition there are huge discrepancies, within the same The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\") The cumulative number of SARS cases in Hong Kong community (and the simulated case \"b\").']),\n",
       " ('World Heritage Site',\n",
       "  'maintained by',\n",
       "  'UNESCO',\n",
       "  ['Of course one obvious explanation might be a difference in reporting cases (see below), but this looks unlikely in these two cities which are so tightly connected to the rest of the world that it would be difficult to hide such a severe disease.',\n",
       "   'In a situation where the accuracy of data is disputable, it would be counterproductive (and even unscientific) to try to incorporate the many fine details of causes of an epidemic.']),\n",
       " ('unscientific',\n",
       "  'subclass of',\n",
       "  'data',\n",
       "  ['Of course one obvious explanation might be a difference in reporting cases (see below), but this looks unlikely in these two cities which are so tightly connected to the rest of the world that it would be difficult to hide such a severe disease.',\n",
       "   'In a situation where the accuracy of data is disputable, it would be counterproductive (and even unscientific) to try to incorporate the many fine details of causes of an epidemic.']),\n",
       " ('causes',\n",
       "  'facet of',\n",
       "  'epidemics',\n",
       "  ['It is important to extract an outline of what is happening with models that are progressively made more complex, in order to extract the relevant parameters and try to draw conclusions on the many possible causes and controls of epidemics.',\n",
       "   'When finding that the SIR model could not account for the available data we faced many possibilities based on virus behavior, population dynamics, technical difficulties in identifying the disease (many atypical to the respiratory tract.']),\n",
       " ('virus',\n",
       "  'has effect',\n",
       "  'population dynamics',\n",
       "  ['It is important to extract an outline of what is happening with models that are progressively made more complex, in order to extract the relevant parameters and try to draw conclusions on the many possible causes and controls of epidemics.',\n",
       "   'When finding that the SIR model could not account for the available data we faced many possibilities based on virus behavior, population dynamics, technical difficulties in identifying the disease (many atypical to the respiratory tract.']),\n",
       " ('pneumonia',\n",
       "  'subclass of',\n",
       "  'epidemics',\n",
       "  ['We explored two major situations.(?<=[.?!])\\\\s+One when the epidemics pneumonia types exist) and socio-political constraints that cast significant doubt on the accuracy of data collection.',\n",
       "   'Knowing the nature of the virus that causes SARS [14] , a coronavirus, we chose here to start by exploring the consequences of a situation that prevailed twenty years ago in the population of pigs in Europe [10, 15] , where a virus and its variant caused a double epidemic when it changed its tropism from the small intestine are subsequent to each other, in a way allowing the first one to provide some protection to part of the exposed population.']),\n",
       " ('SARS',\n",
       "  'parent taxon',\n",
       "  'coronavirus',\n",
       "  ['We explored two major situations.(?<=[.?!])\\\\s+One when the epidemics pneumonia types exist) and socio-political constraints that cast significant doubt on the accuracy of data collection.',\n",
       "   'Knowing the nature of the virus that causes SARS [14] , a coronavirus, we chose here to start by exploring the consequences of a situation that prevailed twenty years ago in the population of pigs in Europe [10, 15] , where a virus and its variant caused a double epidemic when it changed its tropism from the small intestine are subsequent to each other, in a way allowing the first one to provide some protection to part of the exposed population.']),\n",
       " ('population',\n",
       "  'different from',\n",
       "  'prevalence',\n",
       "  ['This situation is not fundamentally different from the situation explored by the SIR model.(?<=[.?!])\\\\s+It may account for places with a large population and a low level of prevalence.',\n",
       "   'In contrast, a second situation where both epidemics overlap, the first one being highly contagious and under no control because it is understood as a mild frequent ailment and the second one causing SARS, will give rise to patterns that appear to be similar to what is observed in places such as  New cases per day Hong Kong and the Guangdong province.']),\n",
       " ('Hong Kong',\n",
       "  'shares border with',\n",
       "  'Guangdong',\n",
       "  ['This situation is not fundamentally different from the situation explored by the SIR model.(?<=[.?!])\\\\s+It may account for places with a large population and a low level of prevalence.',\n",
       "   'In contrast, a second situation where both epidemics overlap, the first one being highly contagious and under no control because it is understood as a mild frequent ailment and the second one causing SARS, will give rise to patterns that appear to be similar to what is observed in places such as  New cases per day Hong Kong and the Guangdong province.']),\n",
       " ('Guangdong',\n",
       "  'shares border with',\n",
       "  'Hong Kong',\n",
       "  ['This situation is not fundamentally different from the situation explored by the SIR model.(?<=[.?!])\\\\s+It may account for places with a large population and a low level of prevalence.',\n",
       "   'In contrast, a second situation where both epidemics overlap, the first one being highly contagious and under no control because it is understood as a mild frequent ailment and the second one causing SARS, will give rise to patterns that appear to be similar to what is observed in places such as  New cases per day Hong Kong and the Guangdong province.']),\n",
       " ('transmission',\n",
       "  'studied by',\n",
       "  'epidemiological',\n",
       "  ['It is worth stressing that we did not explore the intricacies of the mathematical solutions of this new epidemiological model, but, rather, tried to test with very crude hypotheses whether a new mode of transmission might not account for surprising aspects of some epidemics.',\n",
       "   'The central reason for our approach is that the more parameters introduced, the easier it is to represent any type of reality (this was the basis of the epicycle representation of the movement of planets in the Ptolemean system).']),\n",
       " ('Ptolemean system',\n",
       "  'depicts',\n",
       "  'planets',\n",
       "  ['It is worth stressing that we did not explore the intricacies of the mathematical solutions of this new epidemiological model, but, rather, tried to test with very crude hypotheses whether a new mode of transmission might not account for surprising aspects of some epidemics.',\n",
       "   'The central reason for our approach is that the more parameters introduced, the easier it is to represent any type of reality (this was the basis of the epicycle representation of the movement of planets in the Ptolemean system).']),\n",
       " ('World Health Organization',\n",
       "  'parent organization',\n",
       "  'WHO',\n",
       "  ['Because data may be inaccurate (and at the time of writing this is a major concern for the WHO authorities) it is convenient to smooth them out.',\n",
       "   'This is convenient in a continuous model represented by a set of simple differential equations.']),\n",
       " ('continuous model',\n",
       "  'has part',\n",
       "  'differential equation',\n",
       "  ['Because data may be inaccurate (and at the time of writing this is a major concern for the WHO authorities) it is convenient to smooth them out.',\n",
       "   'This is convenient in a continuous model represented by a set of simple differential equations.']),\n",
       " ('stochastic parameters',\n",
       "  'part of',\n",
       "  'discrete model',\n",
       "  ['A drawback of this approach appears when the number of cases becomes small: a discrete model involving stochastic parameters would be more adapted.',\n",
       "   'However this would probably require even more accuracy in the data, a feat that is impossible to achieve.']),\n",
       " ('John F. Kennedy School of Government',\n",
       "  'part of',\n",
       "  'Harvard University',\n",
       "  ['A drawback of this approach appears when the number of cases becomes small: a discrete model involving stochastic parameters would be more adapted.',\n",
       "   'However this would probably require even more accuracy in the data, a feat that is impossible to achieve.']),\n",
       " ('parameters',\n",
       "  'part of',\n",
       "  'data set',\n",
       "  ['A second constraint introduced by inaccuracy in the data set is that the behavior of the epidemic is so complex that it would require many parameters to be represented, implying, in fact that a very large set of causes would account for the observations.',\n",
       "   'We tried to overcome these difficulties by restricting as much as possible the number of fitting parameters in the model.']),\n",
       " ('Füchseldorf',\n",
       "  'country',\n",
       "  'Germany',\n",
       "  ['A second constraint introduced by inaccuracy in the data set is that the behavior of the epidemic is so complex that it would require many parameters to be represented, implying, in fact that a very large set of causes would account for the observations.',\n",
       "   'We tried to overcome these difficulties by restricting as much as possible the number of fitting parameters in the model.']),\n",
       " ('Donnelly',\n",
       "  'field of work',\n",
       "  'phenomenological',\n",
       "  ['Furthermore we checked that the parameters we introduced were consistent with a purely phenomenological description of the disease.(?<=[.?!])\\\\s+In fact, from the work of Donnelly C.(?<=[.?!])\\\\s+et al.',\n",
       "   'et al.(?<=[.?!])\\\\s+[18] on the statistics about the outbreak of SARS in Hong Kong, the incubation period plus the time from onset to admission is about 10.12 days.']),\n",
       " ('outbreak of SARS in Hong Kong',\n",
       "  'number of participants',\n",
       "  '10',\n",
       "  ['Furthermore we checked that the parameters we introduced were consistent with a purely phenomenological description of the disease.(?<=[.?!])\\\\s+In fact, from the work of Donnelly C.(?<=[.?!])\\\\s+et al.',\n",
       "   'et al.(?<=[.?!])\\\\s+[18] on the statistics about the outbreak of SARS in Hong Kong, the incubation period plus the time from onset to admission is about 10.12 days.']),\n",
       " ('onset',\n",
       "  'followed by',\n",
       "  'admission',\n",
       "  ['On the other hand, the incubation period plus the time from onset to admission is about 1/ a + 1/b days and from our parameters estimations on a and b, we find that 1/a + 1/b is about 11.8 and 11.4 for two different scenarios.',\n",
       "   'With these constraints we were able to simulate the epidemic at several places (Hong Kong, Beijing, Inner Mongolia) while other places failed to be represented faithfully (for instance for the Guangdong region where the size of the epidemic -with only 2 or 3 new cases for some days -is too small when compared with the total population of over 80 millions : in this case either the data does not reflect the real situation or the differential model is not relevant for this setting).']),\n",
       " ('Guangdong',\n",
       "  'shares border with',\n",
       "  'Beijing',\n",
       "  ['On the other hand, the incubation period plus the time from onset to admission is about 1/ a + 1/b days and from our parameters estimations on a and b, we find that 1/a + 1/b is about 11.8 and 11.4 for two different scenarios.',\n",
       "   'With these constraints we were able to simulate the epidemic at several places (Hong Kong, Beijing, Inner Mongolia) while other places failed to be represented faithfully (for instance for the Guangdong region where the size of the epidemic -with only 2 or 3 new cases for some days -is too small when compared with the total population of over 80 millions : in this case either the data does not reflect the real situation or the differential model is not relevant for this setting).']),\n",
       " ('control',\n",
       "  'facet of',\n",
       "  'epidemic',\n",
       "  ['Furthermore we could use the set of parameters measured from the past unfolding of the epidemic, to construct scenarios for its future trends and monitor the impact of political decisions to control its spread and development.',\n",
       "   'Note that the above results are to be treated with care when trying to validate provisional hypothesis.']),\n",
       " ('provisional hypothesis',\n",
       "  'subclass of',\n",
       "  'validate',\n",
       "  ['Furthermore we could use the set of parameters measured from the past unfolding of the epidemic, to construct scenarios for its future trends and monitor the impact of political decisions to control its spread and development.',\n",
       "   'Note that the above results are to be treated with care when trying to validate provisional hypothesis.']),\n",
       " ('Observed cummulative cases per day Predicted cummulative cases data',\n",
       "  'instance of',\n",
       "  'time series',\n",
       "  ['Indeed, we noticed that due to the imprecision inherent in the  Observed cummulative cases per day Predicted cummulative cases data many sets of parameters display good fit with the observed time series; however the difference of behaviour between e.g.',\n",
       "   'the final size of the epidemic as given by this simulations is sometimes important (error bars too large) and cannot be reliably asserted in the absence of further investigations.']),\n",
       " ('error bars',\n",
       "  'facet of',\n",
       "  'size',\n",
       "  ['Indeed, we noticed that due to the imprecision inherent in the  Observed cummulative cases per day Predicted cummulative cases data many sets of parameters display good fit with the observed time series; however the difference of behaviour between e.g.',\n",
       "   'the final size of the epidemic as given by this simulations is sometimes important (error bars too large) and cannot be reliably asserted in the absence of further investigations.']),\n",
       " ('qualitative',\n",
       "  'subclass of',\n",
       "  'dynamics',\n",
       "  ['However some qualitative properties of the dynamics of the system under consideration can prove to be important when analysing this epidemic.',\n",
       "   'For instance one important problem is the question of whether the epidemic is under control or not.']),\n",
       " ('under control',\n",
       "  'facet of',\n",
       "  'epidemic',\n",
       "  ['However some qualitative properties of the dynamics of the system under consideration can prove to be important when analysing this epidemic.',\n",
       "   'For instance one important problem is the question of whether the epidemic is under control or not.']),\n",
       " ('SIR model',\n",
       "  'different from',\n",
       "  'SEIR model',\n",
       "  ['Contrary to the standard SIR model where this question has a simple answer, here we cannot say that the epidemic is under control when the number of admission per day decreases; indeed in the SEIR models, it may happen that momentarily the number of people in the Infective class is low while the Exposed class is still high (they have not yet been infectious); thus the epidemic may seem stopped but will then be out of control again when in people in the Exposed class migrate to the Infected class and will start contaminating other people (especially if sanitary security policy has been relaxed).',\n",
       "   'Thus an effective policy necessarily takes into account the time required for the Exposed (E) class to become infectious and will require zero new cases during all the period.']),\n",
       " ('SEIR model',\n",
       "  'different from',\n",
       "  'SIR model',\n",
       "  ['Contrary to the standard SIR model where this question has a simple answer, here we cannot say that the epidemic is under control when the number of admission per day decreases; indeed in the SEIR models, it may happen that momentarily the number of people in the Infective class is low while the Exposed class is still high (they have not yet been infectious); thus the epidemic may seem stopped but will then be out of control again when in people in the Exposed class migrate to the Infected class and will start contaminating other people (especially if sanitary security policy has been relaxed).',\n",
       "   'Thus an effective policy necessarily takes into account the time required for the Exposed (E) class to become infectious and will require zero new cases during all the period.']),\n",
       " ('Exposed(E)',\n",
       "  'has effect',\n",
       "  'infectious',\n",
       "  ['Contrary to the standard SIR model where this question has a simple answer, here we cannot say that the epidemic is under control when the number of admission per day decreases; indeed in the SEIR models, it may happen that momentarily the number of people in the Infective class is low while the Exposed class is still high (they have not yet been infectious); thus the epidemic may seem stopped but will then be out of control again when in people in the Exposed class migrate to the Infected class and will start contaminating other people (especially if sanitary security policy has been relaxed).',\n",
       "   'Thus an effective policy necessarily takes into account the time required for the Exposed (E) class to become infectious and will require zero new cases during all the period.']),\n",
       " ('tail',\n",
       "  'opposite of',\n",
       "  'peak',\n",
       "  ['The double epidemic can have a flat, extended peak and short tail compared to a single epidemic, and it may have more than one peak because of the latency so that claims of success may be premature.',\n",
       "   'An interesting outcome of the present model is that it can point out places where either the model does not hold (indicating that there are other causes interfering with the disease) or that data collection is not accurate (providing a way to political authorities to enforce a new policy for identifying patients and collecting data).']),\n",
       " ('identifying patients',\n",
       "  'subclass of',\n",
       "  'collecting data',\n",
       "  ['The double epidemic can have a flat, extended peak and short tail compared to a single epidemic, and it may have more than one peak because of the latency so that claims of success may be premature.',\n",
       "   'An interesting outcome of the present model is that it can point out places where either the model does not hold (indicating that there are other causes interfering with the disease) or that data collection is not accurate (providing a way to political authorities to enforce a new policy for identifying patients and collecting data).']),\n",
       " ('cross immunity',\n",
       "  'facet of',\n",
       "  'vaccine',\n",
       "  ['A second, perhaps more comforting consequence is that it suggests that a vaccine will be possible: indeed, in our model one form of the virus (or, alternatively, other viruses displaying cross immunity) is protecting against SARS.',\n",
       "   'This is typically what is expected from a vaccine and this is perhaps what is witnessed with the apparent low infection rate or, at least mild infection, in children and young adults (an observation that needs to be fully documented).']),\n",
       " ('mild infection',\n",
       "  'subclass of',\n",
       "  'infection',\n",
       "  ['A second, perhaps more comforting consequence is that it suggests that a vaccine will be possible: indeed, in our model one form of the virus (or, alternatively, other viruses displaying cross immunity) is protecting against SARS.',\n",
       "   'This is typically what is expected from a vaccine and this is perhaps what is witnessed with the apparent low infection rate or, at least mild infection, in children and young adults (an observation that needs to be fully documented).']),\n",
       " ('vaccination against coronavirus',\n",
       "  'facet of',\n",
       "  'coronavirus',\n",
       "  ['We should however remain cautious, since vaccination against coronaviruses is not always straightforward [16] .',\n",
       "   'If our model is right, then it is likely that virus B gave several different types of A variants, causing SARS with a variety of virulence.']),\n",
       " ('variety of virulence',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['We should however remain cautious, since vaccination against coronaviruses is not always straightforward [16] .',\n",
       "   'If our model is right, then it is likely that virus B gave several different types of A variants, causing SARS with a variety of virulence.']),\n",
       " ('sequencing',\n",
       "  'uses',\n",
       "  'genomes',\n",
       "  ['This may also account for some of the variability observed at different places.(?<=[.?!])\\\\s+This might be substantiated by sequencing CoV genomes isolated from many patients.',\n",
       "   'If this were true, then the reservoir might be  Observed three-day mean cases per day Predicted cases (I*a) much larger than suspected and SARS may happen again when the weather cools down in autumn.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'Predicted cases',\n",
       "  ['This may also account for some of the variability observed at different places.(?<=[.?!])\\\\s+This might be substantiated by sequencing CoV genomes isolated from many patients.',\n",
       "   'If this were true, then the reservoir might be  Observed three-day mean cases per day Predicted cases (I*a) much larger than suspected and SARS may happen again when the weather cools down in autumn.']),\n",
       " ('Our model',\n",
       "  'use',\n",
       "  'precautionary measures',\n",
       "  ['Our model would then be useful to enforce proper precautionary measures immediately.',\n",
       "   'Finally, there are many other possible scenarios, and as stated in the conclusion of the WHO panel held on 15-17 May 2003 in Geneva: \"Participants from the main outbreak sites noted the striking similarity of the pattern of outbreaks in different countries and the consistent effectiveness of specific control measures, including early identification and isolation of patients, vigorous contact tracing, management of close contacts by home confinement or quarantine, and public information and education to encourage prompt reporting of symptoms.']),\n",
       " ('WHO',\n",
       "  'headquarters location',\n",
       "  'Geneva',\n",
       "  ['Our model would then be useful to enforce proper precautionary measures immediately.',\n",
       "   'Finally, there are many other possible scenarios, and as stated in the conclusion of the WHO panel held on 15-17 May 2003 in Geneva: \"Participants from the main outbreak sites noted the striking similarity of the pattern of outbreaks in different countries and the consistent effectiveness of specific control measures, including early identification and isolation of patients, vigorous contact tracing, management of close contacts by home confinement or quarantine, and public information and education to encourage prompt reporting of symptoms.']),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'human host',\n",
       "  ['The effectiveness of these measures was observed in all outbreak sites under widely varying conditions, supporting the overall WHO view that SARS can be contained and driven back out of its new human host.\" The present work aims at helping exploring one possible route accounting for at least some of the features of the epidemic.',\n",
       "   'The first conclusion of the present work is that except at very restricted places recording a simple event, such as the Amoy Gardens Estate outbreak, the SIR model cannot represent the unfolding of the SARS epidemic, even at its first steps.']),\n",
       " ('human host',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  ['The effectiveness of these measures was observed in all outbreak sites under widely varying conditions, supporting the overall WHO view that SARS can be contained and driven back out of its new human host.\" The present work aims at helping exploring one possible route accounting for at least some of the features of the epidemic.',\n",
       "   'The first conclusion of the present work is that except at very restricted places recording a simple event, such as the Amoy Gardens Estate outbreak, the SIR model cannot represent the unfolding of the SARS epidemic, even at its first steps.']),\n",
       " ('Amoy Gardens Estate outbreak',\n",
       "  'has effect',\n",
       "  'SARS epidemic',\n",
       "  ['The effectiveness of these measures was observed in all outbreak sites under widely varying conditions, supporting the overall WHO view that SARS can be contained and driven back out of its new human host.\" The present work aims at helping exploring one possible route accounting for at least some of the features of the epidemic.',\n",
       "   'The first conclusion of the present work is that except at very restricted places recording a simple event, such as the Amoy Gardens Estate outbreak, the SIR model cannot represent the unfolding of the SARS epidemic, even at its first steps.']),\n",
       " ('simple model',\n",
       "  'facet of',\n",
       "  'contagion',\n",
       "  ['This indicates that the simple model of contagion by a single stable virus does not hold.',\n",
       "   'The second conclusion is that, at least in some regions, data may be inaccurate to account for the reality of the epidemic.']),\n",
       " ('data',\n",
       "  'part of',\n",
       "  'reality',\n",
       "  ['This indicates that the simple model of contagion by a single stable virus does not hold.',\n",
       "   'The second conclusion is that, at least in some regions, data may be inaccurate to account for the reality of the epidemic.']),\n",
       " ('China',\n",
       "  'contains administrative territorial entity',\n",
       "  'Beijing',\n",
       "  ['This is true in various parts of mainland China and can be accounted for by the large variation in SARS symptoms, and perhaps in provinces next to Beijing, by underreporting.',\n",
       "   'Finally, this study sends a message of hope: lack of fitting of predicted curves with actual recent data indicate that the containment measures decided by the WHO and relayed by governments are effective.']),\n",
       " ('Beijing',\n",
       "  'located in the administrative territorial entity',\n",
       "  'China',\n",
       "  ['This is true in various parts of mainland China and can be accounted for by the large variation in SARS symptoms, and perhaps in provinces next to Beijing, by underreporting.',\n",
       "   'Finally, this study sends a message of hope: lack of fitting of predicted curves with actual recent data indicate that the containment measures decided by the WHO and relayed by governments are effective.']),\n",
       " ('WHO',\n",
       "  'field of work',\n",
       "  'containment measures',\n",
       "  ['This is true in various parts of mainland China and can be accounted for by the large variation in SARS symptoms, and perhaps in provinces next to Beijing, by underreporting.',\n",
       "   'Finally, this study sends a message of hope: lack of fitting of predicted curves with actual recent data indicate that the containment measures decided by the WHO and relayed by governments are effective.']),\n",
       " ('vaccine',\n",
       "  'facet of',\n",
       "  'SARS',\n",
       "  ['Furthermore, this model, that assumes that a mild epidemic protects against SARS would predict that a vaccine is possible, and may soon be created.',\n",
       "   'It also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.']),\n",
       " ('SARS',\n",
       "  'subclass of',\n",
       "  'precursor',\n",
       "  ['Furthermore, this model, that assumes that a mild epidemic protects against SARS would predict that a vaccine is possible, and may soon be created.',\n",
       "   'It also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.']),\n",
       " ('febrile',\n",
       "  'subclass of',\n",
       "  'respiratory illness',\n",
       "  ['BACKGROUND: Severe acute respiratory syndrome (SARS) is a febrile respiratory illness.',\n",
       "   'The disease has been etiologically linked to a novel coronavirus that has been named the SARS-associated coronavirus (SARS-CoV), whose genome was recently sequenced.']),\n",
       " ('SARS-associated coronavirus',\n",
       "  'parent taxon',\n",
       "  'coronavirus',\n",
       "  ['BACKGROUND: Severe acute respiratory syndrome (SARS) is a febrile respiratory illness.',\n",
       "   'The disease has been etiologically linked to a novel coronavirus that has been named the SARS-associated coronavirus (SARS-CoV), whose genome was recently sequenced.']),\n",
       " ('spike protein',\n",
       "  'part of',\n",
       "  'Coronaviridae',\n",
       "  ['Since it is a member of the Coronaviridae, its spike protein (S2) is believed to play a central role in viral entry by facilitating fusion between the viral and host cell membranes.',\n",
       "   'The protein responsible for viral-induced membrane fusion of HIV-1 (gp41) differs in length, and has no sequence homology with S2.']),\n",
       " ('gp41',\n",
       "  'instance of',\n",
       "  'protein',\n",
       "  ['Since it is a member of the Coronaviridae, its spike protein (S2) is believed to play a central role in viral entry by facilitating fusion between the viral and host cell membranes.',\n",
       "   'The protein responsible for viral-induced membrane fusion of HIV-1 (gp41) differs in length, and has no sequence homology with S2.']),\n",
       " ('active conformation',\n",
       "  'subclass of',\n",
       "  'protein',\n",
       "  ['RESULTS: Sequence analysis reveals that the two viral proteins share the sequence motifs that construct their active conformation.',\n",
       "   'These include (1) an N-terminal leucine/isoleucine zipper-like sequence, and (2) a C-terminal heptad repeat located upstream of (3) an aromatic residue-rich region juxtaposed to the (4) transmembrane segment.']),\n",
       " ('heptad repeat',\n",
       "  'subclass of',\n",
       "  'N-terminal',\n",
       "  ['RESULTS: Sequence analysis reveals that the two viral proteins share the sequence motifs that construct their active conformation.',\n",
       "   'These include (1) an N-terminal leucine/isoleucine zipper-like sequence, and (2) a C-terminal heptad repeat located upstream of (3) an aromatic residue-rich region juxtaposed to the (4) transmembrane segment.']),\n",
       " ('SARS-CoV',\n",
       "  'has cause',\n",
       "  'HIV-1',\n",
       "  ['CONCLUSIONS: This study points to a similar mode of action for the two viral proteins, suggesting that anti-viral strategy that targets the viral-induced membrane fusion step can be adopted from HIV-1 to SARS-CoV.',\n",
       "   'Recently the FDA approved Enfuvirtide, a synthetic peptide corresponding to the C-terminal heptad repeat of HIV-1 gp41, as an anti-AIDS agent.']),\n",
       " ('Enfuvirtide',\n",
       "  'subject has role',\n",
       "  'anti-AIDS',\n",
       "  ['CONCLUSIONS: This study points to a similar mode of action for the two viral proteins, suggesting that anti-viral strategy that targets the viral-induced membrane fusion step can be adopted from HIV-1 to SARS-CoV.',\n",
       "   'Recently the FDA approved Enfuvirtide, a synthetic peptide corresponding to the C-terminal heptad repeat of HIV-1 gp41, as an anti-AIDS agent.']),\n",
       " ('Enfuvirtide',\n",
       "  'facet of',\n",
       "  'HIV-1',\n",
       "  ['Enfuvirtide and C34, another anti HIV-1 peptide, exert their inhibitory activity by binding to a leucine/isoleucine zipper-like sequence in gp41, thus inhibiting a conformational change of gp41 required for its activation.',\n",
       "   'We suggest that peptides corresponding to the C-terminal heptad repeat of the S2 protein may serve as inhibitors for SARS-CoV entry.']),\n",
       " ('C-terminal',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['Enfuvirtide and C34, another anti HIV-1 peptide, exert their inhibitory activity by binding to a leucine/isoleucine zipper-like sequence in gp41, thus inhibiting a conformational change of gp41 required for its activation.',\n",
       "   'We suggest that peptides corresponding to the C-terminal heptad repeat of the S2 protein may serve as inhibitors for SARS-CoV entry.']),\n",
       " ('viral',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['Infection by many enveloped viruses requires fusion of the viral and cellular membranes.(?<=[.?!])\\\\s+A viral envelope protein mediates this membrane fusion process.',\n",
       "   'These proteins are synthesized as precursors (ENV in Retroviridae, and E2 in Coronaviridae) that are later processed into a transmembrane subunit (gp41 in the retrovirus HIV-1, and S2 in the coronavirus SARS-CoV) that is responsible for viralinduced membrane fusion, and a surface subunit that is responsible for the interaction with the cellular receptor/s.']),\n",
       " ('transmembrane subunit',\n",
       "  'subclass of',\n",
       "  'proteins',\n",
       "  ['Infection by many enveloped viruses requires fusion of the viral and cellular membranes.(?<=[.?!])\\\\s+A viral envelope protein mediates this membrane fusion process.',\n",
       "   'These proteins are synthesized as precursors (ENV in Retroviridae, and E2 in Coronaviridae) that are later processed into a transmembrane subunit (gp41 in the retrovirus HIV-1, and S2 in the coronavirus SARS-CoV) that is responsible for viralinduced membrane fusion, and a surface subunit that is responsible for the interaction with the cellular receptor/s.']),\n",
       " ('N-terminus',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['HIV-1 gp41, which is a well-characterized protein [1, 2] contains two heptad repeat (HR) regions, a leucine/isoleucine HR adjacent to its N-terminus (N-HR), and C-HR proximal to the transmembrane domain (see Figure 1 ).',\n",
       "   'Heptad repeats are characterized by hydrophobic amino acids in the \"a\" and \"d\" positions of the helix.(?<=[.?!])\\\\s+In the N-HR of gp41, all but one of the \"a\" positions are Leucines or Isoleucines.']),\n",
       " ('transmembrane domain',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['HIV-1 gp41, which is a well-characterized protein [1, 2] contains two heptad repeat (HR) regions, a leucine/isoleucine HR adjacent to its N-terminus (N-HR), and C-HR proximal to the transmembrane domain (see Figure 1 ).',\n",
       "   'Heptad repeats are characterized by hydrophobic amino acids in the \"a\" and \"d\" positions of the helix.(?<=[.?!])\\\\s+In the N-HR of gp41, all but one of the \"a\" positions are Leucines or Isoleucines.']),\n",
       " ('Leucine',\n",
       "  'subclass of',\n",
       "  'amino acid',\n",
       "  ['HIV-1 gp41, which is a well-characterized protein [1, 2] contains two heptad repeat (HR) regions, a leucine/isoleucine HR adjacent to its N-terminus (N-HR), and C-HR proximal to the transmembrane domain (see Figure 1 ).',\n",
       "   'Heptad repeats are characterized by hydrophobic amino acids in the \"a\" and \"d\" positions of the helix.(?<=[.?!])\\\\s+In the N-HR of gp41, all but one of the \"a\" positions are Leucines or Isoleucines.']),\n",
       " ('Isoleucine',\n",
       "  'subclass of',\n",
       "  'amino acid',\n",
       "  ['HIV-1 gp41, which is a well-characterized protein [1, 2] contains two heptad repeat (HR) regions, a leucine/isoleucine HR adjacent to its N-terminus (N-HR), and C-HR proximal to the transmembrane domain (see Figure 1 ).',\n",
       "   'Heptad repeats are characterized by hydrophobic amino acids in the \"a\" and \"d\" positions of the helix.(?<=[.?!])\\\\s+In the N-HR of gp41, all but one of the \"a\" positions are Leucines or Isoleucines.']),\n",
       " ('N-HR',\n",
       "  'opposite of',\n",
       "  'C-HR',\n",
       "  ['This feature is less strict in the \"d\" positions of N-HR, and in the \"a\" and \"d\" positions of the C-HR.',\n",
       "   'Peptides corresponding to these heptad repeat regions form the \"trimer-of-hairpins\" core structure of gp41 [3] as confirmed by the solution of the crystal structures [1, 2] .']),\n",
       " ('C-HR',\n",
       "  'opposite of',\n",
       "  'N-HR',\n",
       "  ['This feature is less strict in the \"d\" positions of N-HR, and in the \"a\" and \"d\" positions of the C-HR.',\n",
       "   'Peptides corresponding to these heptad repeat regions form the \"trimer-of-hairpins\" core structure of gp41 [3] as confirmed by the solution of the crystal structures [1, 2] .']),\n",
       " ('core structure',\n",
       "  'subclass of',\n",
       "  'crystal structure',\n",
       "  ['This feature is less strict in the \"d\" positions of N-HR, and in the \"a\" and \"d\" positions of the C-HR.',\n",
       "   'Peptides corresponding to these heptad repeat regions form the \"trimer-of-hairpins\" core structure of gp41 [3] as confirmed by the solution of the crystal structures [1, 2] .']),\n",
       " ('Proline',\n",
       "  'subclass of',\n",
       "  'Cysteine',\n",
       "  ['Two Cysteine residues and one Proline residue, located between these two HRs, confine a hairpin conformation (Figure 2a) .',\n",
       "   'A tryptophan-rich motif, located between the C-HR and the transmembrane domain, was shown to play a crucial role in gp41-mediated membrane fusion [4] (Figure 2a) .']),\n",
       " ('C-HR',\n",
       "  'part of',\n",
       "  'transmembrane domain',\n",
       "  ['Two Cysteine residues and one Proline residue, located between these two HRs, confine a hairpin conformation (Figure 2a) .',\n",
       "   'A tryptophan-rich motif, located between the C-HR and the transmembrane domain, was shown to play a crucial role in gp41-mediated membrane fusion [4] (Figure 2a) .']),\n",
       " ('oligomer',\n",
       "  'subclass of',\n",
       "  'conformational change',\n",
       "  ['In order to exert their role in membrane fusion, viral spike proteins act as oligomers and go through a significant conformational change resulting in the \"trimer-of-hairpin\" conformation.',\n",
       "   'The oligomerization and the change in conformation of viral spike proteins involve interactions between proteins segments.']),\n",
       " ('conformation',\n",
       "  'facet of',\n",
       "  'protein',\n",
       "  ['In order to exert their role in membrane fusion, viral spike proteins act as oligomers and go through a significant conformational change resulting in the \"trimer-of-hairpin\" conformation.',\n",
       "   'The oligomerization and the change in conformation of viral spike proteins involve interactions between proteins segments.']),\n",
       " ('Peptide',\n",
       "  'subclass of',\n",
       "  'protein',\n",
       "  ['Peptides derived from a segment of this protein might therefore interfere with one of these processes, and inhibit viral infection.',\n",
       "   'Indeed, Wheel projection of the N-HR (a) and C-HR (b) of HIV-1 gp41 (gi|9629363) Figure 1 Wheel projection of the N-HR (a) and C-HR (b) of HIV-1 gp41 (gi|9629363).']),\n",
       " ('Figure 1 Wheel projection of the N-HR',\n",
       "  'depicts',\n",
       "  'Wheel projection of the N-HR',\n",
       "  ['Peptides derived from a segment of this protein might therefore interfere with one of these processes, and inhibit viral infection.',\n",
       "   'Indeed, Wheel projection of the N-HR (a) and C-HR (b) of HIV-1 gp41 (gi|9629363) Figure 1 Wheel projection of the N-HR (a) and C-HR (b) of HIV-1 gp41 (gi|9629363).']),\n",
       " ('amino acid sequence',\n",
       "  'has part',\n",
       "  'amino acid',\n",
       "  ['The amino acid sequence is displayed end-to-end down the axis of a schematic helix.(?<=[.?!])\\\\s+The angle between every two consecutive amino acids is 102.9°.',\n",
       "   'The helical wheel consists of seven corners, corresponding to the fit of seven amino acid residues into every two helical turns.']),\n",
       " ('amino acid',\n",
       "  'subclass of',\n",
       "  'residues',\n",
       "  ['The amino acid sequence is displayed end-to-end down the axis of a schematic helix.(?<=[.?!])\\\\s+The angle between every two consecutive amino acids is 102.9°.',\n",
       "   'The helical wheel consists of seven corners, corresponding to the fit of seven amino acid residues into every two helical turns.']),\n",
       " ('HIV-1',\n",
       "  'has effect',\n",
       "  'SARS-CoV',\n",
       "  ['Similarity between the fusion proteins of HIV-1 and SARS-CoV Figure 2 Similarity between the fusion proteins of HIV-1 and SARS-CoV.',\n",
       "   'The HIV-1 gp41 (a) and the equivalent S2 protein from the SARS-CoV (b) are shown.(?<=[.?!])\\\\s+A Leucine/Isoleucine heptad repeat adjacent to the N-terminus of both proteins appears in red.']),\n",
       " ('N-terminus',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['Similarity between the fusion proteins of HIV-1 and SARS-CoV Figure 2 Similarity between the fusion proteins of HIV-1 and SARS-CoV.',\n",
       "   'The HIV-1 gp41 (a) and the equivalent S2 protein from the SARS-CoV (b) are shown.(?<=[.?!])\\\\s+A Leucine/Isoleucine heptad repeat adjacent to the N-terminus of both proteins appears in red.']),\n",
       " ('loop structure',\n",
       "  'has part',\n",
       "  'heptad repeat',\n",
       "  ['The C-terminal heptad repeat is in green.(?<=[.?!])\\\\s+Cysteine residues (purple) confining a loop structure are located between the two heptad repeats.',\n",
       "   'An aromatic residues-rich motif is marked blue, and the transmembrane segment is in orange.']),\n",
       " ('transmembrane segment',\n",
       "  'has part',\n",
       "  'aromatic residues',\n",
       "  ['The C-terminal heptad repeat is in green.(?<=[.?!])\\\\s+Cysteine residues (purple) confining a loop structure are located between the two heptad repeats.',\n",
       "   'An aromatic residues-rich motif is marked blue, and the transmembrane segment is in orange.']),\n",
       " ('inhibitor',\n",
       "  'subclass of',\n",
       "  'peptide',\n",
       "  ['A peptide corresponding to the C-terminal heptad repeat, which acts as potent inhibitor of HIV-1 entry into the cell, appears in yellow.',\n",
       "   'peptides corresponding to the C-HR of gp41 are potent inhibitors of HIV-1 entry into cells, one of them, Enfuvirtide (Fuzeon), was recently approved by the FDA as an addition to the cocktail currently given to AIDS patients [5] , and C34, a peptide corresponding to the C-HR of the gp41 core complex is promising in-vitro [1] .']),\n",
       " ('Enfuvirtide',\n",
       "  'subject has role',\n",
       "  'inhibitors of HIV-1 entry into cells',\n",
       "  ['A peptide corresponding to the C-terminal heptad repeat, which acts as potent inhibitor of HIV-1 entry into the cell, appears in yellow.',\n",
       "   'peptides corresponding to the C-HR of gp41 are potent inhibitors of HIV-1 entry into cells, one of them, Enfuvirtide (Fuzeon), was recently approved by the FDA as an addition to the cocktail currently given to AIDS patients [5] , and C34, a peptide corresponding to the C-HR of the gp41 core complex is promising in-vitro [1] .']),\n",
       " ('gp41',\n",
       "  'has part',\n",
       "  'N-HR segment',\n",
       "  ['It is believed that these peptides exert their anti-viral activity via a dominant negative mechanism by interacting with the central N-HR segment of gp41 [6] .',\n",
       "   'This is a promising approach also in developing anti-viral peptides against several paramyxoviruses [7] .(?<=[.?!])\\\\s+The genome of the SARS-CoV was recently sequenced [8, 9] .']),\n",
       " ('genome',\n",
       "  'studied by',\n",
       "  'sequenced',\n",
       "  ['It is believed that these peptides exert their anti-viral activity via a dominant negative mechanism by interacting with the central N-HR segment of gp41 [6] .',\n",
       "   'This is a promising approach also in developing anti-viral peptides against several paramyxoviruses [7] .(?<=[.?!])\\\\s+The genome of the SARS-CoV was recently sequenced [8, 9] .']),\n",
       " ('S2',\n",
       "  'part of',\n",
       "  'Coronaviridae',\n",
       "  ['Since it is a member of the Coronaviridae [10] , the S2 protein is believed to play a central role in viral entry.',\n",
       "   'Although we found no sequence homology between the SARS-CoV S2 and HIV-1 gp41, a comprehensive sequence analysis reveals that all the above-mentioned elements of gp41 are present also in S2.']),\n",
       " ('SARS-CoV S2',\n",
       "  'has part',\n",
       "  'gp41',\n",
       "  ['Since it is a member of the Coronaviridae [10] , the S2 protein is believed to play a central role in viral entry.',\n",
       "   'Although we found no sequence homology between the SARS-CoV S2 and HIV-1 gp41, a comprehensive sequence analysis reveals that all the above-mentioned elements of gp41 are present also in S2.']),\n",
       " ('HIV-1',\n",
       "  'has part',\n",
       "  'gp41',\n",
       "  ['In analogy to HIV-1 gp41, N-HR and an aromatic-rich region in SARS-CoV S2 protein were identified by Gallaher & Garry [11] .',\n",
       "   'Whereas these discoveries have structural importance, peptides corresponding to the N-HR of HIV-1 [12] and Sendai virus [13] , and to the aromatic-rich region of feline immunodeficiency virus [14] were found to have only low anti-viral activity, compared to the highly active C-HR corresponding peptides.']),\n",
       " ('N-HR',\n",
       "  'part of',\n",
       "  'HIV-1',\n",
       "  ['In analogy to HIV-1 gp41, N-HR and an aromatic-rich region in SARS-CoV S2 protein were identified by Gallaher & Garry [11] .',\n",
       "   'Whereas these discoveries have structural importance, peptides corresponding to the N-HR of HIV-1 [12] and Sendai virus [13] , and to the aromatic-rich region of feline immunodeficiency virus [14] were found to have only low anti-viral activity, compared to the highly active C-HR corresponding peptides.']),\n",
       " ('HIV-1',\n",
       "  'has part',\n",
       "  'gp41',\n",
       "  ['Herein, we report the identification of the sequence in SARS-CoV S2 protein that is analogous to the C-HR of HIV-1 gp41.',\n",
       "   'This led us to the suggestion that the viral entry mechanisms are analogous and therefore a therapeutic strategy that is being applied against the HIV-1 can be adopted to fight SARS-CoV.']),\n",
       " ('HIV-1',\n",
       "  'has effect',\n",
       "  'SARS-CoV',\n",
       "  ['Herein, we report the identification of the sequence in SARS-CoV S2 protein that is analogous to the C-HR of HIV-1 gp41.',\n",
       "   'This led us to the suggestion that the viral entry mechanisms are analogous and therefore a therapeutic strategy that is being applied against the HIV-1 can be adopted to fight SARS-CoV.']),\n",
       " ('transmembrane subunits',\n",
       "  'subclass of',\n",
       "  'proteins',\n",
       "  ['Coronaviridae S2 proteins are believed to be functionally equivalent to the transmembrane subunits of Retroviral ENV.',\n",
       "   'However, using well-known sequence comparison algorithms [15, 16] , we found that there is no sequence homology between the S2 protein of SARS-CoV and HIV-1 gp41.']),\n",
       " ('HIV-1',\n",
       "  'has part',\n",
       "  'gp41',\n",
       "  ['Coronaviridae S2 proteins are believed to be functionally equivalent to the transmembrane subunits of Retroviral ENV.',\n",
       "   'However, using well-known sequence comparison algorithms [15, 16] , we found that there is no sequence homology between the S2 protein of SARS-CoV and HIV-1 gp41.']),\n",
       " ('C-terminus',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['Markedly, LearnCoil-VMF [17] , helical wheel analysis and protein topology prediction [18] reveal that the SARS-CoV S2 protein consists of the same elements that were characterized in HIV-1 gp41 (Figure 2b ): (I) N-HR, a Leucine/ Isoleucine heptad repeat appears on residues 913-1000 of the SARS-CoV CUHK-W1 isolate (Figure 3a) .; (II) C-HR, a Leucine/Isoleucine heptad repeat appears on residues 1151-1185 ( Figure 3b ); (III) the loop between these two HRs is longer than that of gp41, and is confined by four Cysteine residues and nine Proline residues that might conform a double loop structure; (IV) a transmembrane region is predicted adjacent to the C-terminus of the protein [18] ; (V) a Tryptophan/Tyrosine-rich motif is located between the C-HR and the transmembrane domain (Figure 4) .',\n",
       "   'The results reveal similar structural motifs in HIV-1 gp41 and SARS-CoV S2 proteins, suggesting an analogous membrane fusion mechanism induced by the two viruses.']),\n",
       " ('structural motifs',\n",
       "  'part of',\n",
       "  'proteins',\n",
       "  ['Markedly, LearnCoil-VMF [17] , helical wheel analysis and protein topology prediction [18] reveal that the SARS-CoV S2 protein consists of the same elements that were characterized in HIV-1 gp41 (Figure 2b ): (I) N-HR, a Leucine/ Isoleucine heptad repeat appears on residues 913-1000 of the SARS-CoV CUHK-W1 isolate (Figure 3a) .; (II) C-HR, a Leucine/Isoleucine heptad repeat appears on residues 1151-1185 ( Figure 3b ); (III) the loop between these two HRs is longer than that of gp41, and is confined by four Cysteine residues and nine Proline residues that might conform a double loop structure; (IV) a transmembrane region is predicted adjacent to the C-terminus of the protein [18] ; (V) a Tryptophan/Tyrosine-rich motif is located between the C-HR and the transmembrane domain (Figure 4) .',\n",
       "   'The results reveal similar structural motifs in HIV-1 gp41 and SARS-CoV S2 proteins, suggesting an analogous membrane fusion mechanism induced by the two viruses.']),\n",
       " ('Spartan scytale',\n",
       "  'instance of',\n",
       "  'cryptographic device',\n",
       "  ['The helical wheel diagram serves as an equivalent to the Spartan scytale, the first military cryptographic device, consisting of a cylinder with a strip of paper wound around it.',\n",
       "   'The recipient has a rod of the same diameter, on which he wraps the paper to read the message.']),\n",
       " ('letter',\n",
       "  'subclass of',\n",
       "  'paper',\n",
       "  ['The helical wheel diagram serves as an equivalent to the Spartan scytale, the first military cryptographic device, consisting of a cylinder with a strip of paper wound around it.',\n",
       "   'The recipient has a rod of the same diameter, on which he wraps the paper to read the message.']),\n",
       " ('HIV-1',\n",
       "  'has effect',\n",
       "  'SARS-CoV',\n",
       "  ['Herein, the prior knowledge of the 3.5 periodicity learned from the HIV-1 gp41 structural studies, is used to decipher the structural features of the SARS-CoV S2 protein.',\n",
       "   'In general, SARS-CoV S2 and HIV-1 gp41 share the hairpin structure.(?<=[.?!])\\\\s+However, it is worth noting that the N-HR of S2 is longer than that of gp41 and contains more Leucines and Isoleucines.']),\n",
       " ('SARS-CoV',\n",
       "  'has cause',\n",
       "  'HIV-1',\n",
       "  ['Herein, the prior knowledge of the 3.5 periodicity learned from the HIV-1 gp41 structural studies, is used to decipher the structural features of the SARS-CoV S2 protein.',\n",
       "   'In general, SARS-CoV S2 and HIV-1 gp41 share the hairpin structure.(?<=[.?!])\\\\s+However, it is worth noting that the N-HR of S2 is longer than that of gp41 and contains more Leucines and Isoleucines.']),\n",
       " ('SARS-CoV S2',\n",
       "  'has cause',\n",
       "  'HIV-1',\n",
       "  ['Herein, the prior knowledge of the 3.5 periodicity learned from the HIV-1 gp41 structural studies, is used to decipher the structural features of the SARS-CoV S2 protein.',\n",
       "   'In general, SARS-CoV S2 and HIV-1 gp41 share the hairpin structure.(?<=[.?!])\\\\s+However, it is worth noting that the N-HR of S2 is longer than that of gp41 and contains more Leucines and Isoleucines.']),\n",
       " ('Isoleucine',\n",
       "  'subclass of',\n",
       "  'Leucine',\n",
       "  ['Furthermore, while the C-HR of gp41 barely shows any heptad repeat signal, the C-HR of S2 has a perfect Leucine/Isoleucine heptad repeat in its \"d\" positions.',\n",
       "   'The antiviral activity of Enfuvirtide was reported as early as 1993 [19] , while it took few years until the fold of gp41 core complex was discovered [3] and its crystal structure being solved [1, 2] .']),\n",
       " ('Enfuvirtide',\n",
       "  'subject has role',\n",
       "  'antiviral',\n",
       "  ['Furthermore, while the C-HR of gp41 barely shows any heptad repeat signal, the C-HR of S2 has a perfect Leucine/Isoleucine heptad repeat in its \"d\" positions.',\n",
       "   'The antiviral activity of Enfuvirtide was reported as early as 1993 [19] , while it took few years until the fold of gp41 core complex was discovered [3] and its crystal structure being solved [1, 2] .']),\n",
       " ('C34',\n",
       "  'instance of',\n",
       "  'peptide',\n",
       "  ['Based on these structural studies, peptides corresponding to the C-HR of gp41 core complex, such as C34, were synthesized and found to efficiently inhibit HIV-1 induced membrane fusion [1] .',\n",
       "   'Interestingly, Enfuvirtide, which is a shifted version of these Cpeptides, does not comprise the residues that were shown to be essential for their inhibitory activity [20] .']),\n",
       " ('Enfuvirtide',\n",
       "  'instance of',\n",
       "  'Cpeptide',\n",
       "  ['Based on these structural studies, peptides corresponding to the C-HR of gp41 core complex, such as C34, were synthesized and found to efficiently inhibit HIV-1 induced membrane fusion [1] .',\n",
       "   'Interestingly, Enfuvirtide, which is a shifted version of these Cpeptides, does not comprise the residues that were shown to be essential for their inhibitory activity [20] .']),\n",
       " ('HIV',\n",
       "  'has part',\n",
       "  'gp41',\n",
       "  ['Indeed, others and myself reported that in some cases C34 is more potent in inhibiting HIV-1 gp41 induced membrane fusion than Enfuvirtide [21, 22] .',\n",
       "   'Thus, the SARS-CoV sequence corresponding to C34 has higher chances to block SARS-CoV entry.']),\n",
       " ('C34',\n",
       "  'instance of',\n",
       "  'SARS-CoV sequence',\n",
       "  ['Indeed, others and myself reported that in some cases C34 is more potent in inhibiting HIV-1 gp41 induced membrane fusion than Enfuvirtide [21, 22] .',\n",
       "   'Thus, the SARS-CoV sequence corresponding to C34 has higher chances to block SARS-CoV entry.']),\n",
       " ('anti-fusion therapeutic approach',\n",
       "  'facet of',\n",
       "  'HIV-1',\n",
       "  ['Based on the similarity in the mechanisms in which the two viruses induce fusion between their membrane and their host cell membrane, we propose adapting the successful anti-fusion therapeutic approach used against HIV-1 to the SARS-CoV case.',\n",
       "   'Peptides derived from the C-HR segment of SARS-CoV S2 protein (ISGINASVV-NIQKEIDRLNEVAKNLNESLIDLQEL) might inhibit viral induced membrane fusion, thereby blocking SARS-CoV infection.']),\n",
       " ('SARS-CoV',\n",
       "  'has part',\n",
       "  'SARS-CoV S2 protein',\n",
       "  ['Based on the similarity in the mechanisms in which the two viruses induce fusion between their membrane and their host cell membrane, we propose adapting the successful anti-fusion therapeutic approach used against HIV-1 to the SARS-CoV case.',\n",
       "   'Peptides derived from the C-HR segment of SARS-CoV S2 protein (ISGINASVV-NIQKEIDRLNEVAKNLNESLIDLQEL) might inhibit viral induced membrane fusion, thereby blocking SARS-CoV infection.']),\n",
       " ('intranasal delivery',\n",
       "  'subclass of',\n",
       "  'administration',\n",
       "  ['As the SARS-CoV infects respiratory tissues, the classical disadvantages of peptide therapeutics administration may be overcome using intranasal delivery [23] .',\n",
       "   'Wheel projection of the N-HR (a) and C-HR (b) of SARS-CoV S2 protein (gi|30023954) Figure 3 Wheel projection of the N-HR (a) and C-HR (b) of SARS-CoV S2 protein (gi|30023954).']),\n",
       " ('Figure 3',\n",
       "  'depicts',\n",
       "  'Wheel projection',\n",
       "  ['As the SARS-CoV infects respiratory tissues, the classical disadvantages of peptide therapeutics administration may be overcome using intranasal delivery [23] .',\n",
       "   'Wheel projection of the N-HR (a) and C-HR (b) of SARS-CoV S2 protein (gi|30023954) Figure 3 Wheel projection of the N-HR (a) and C-HR (b) of SARS-CoV S2 protein (gi|30023954).']),\n",
       " ('amino acid sequence',\n",
       "  'has part',\n",
       "  'amino acid',\n",
       "  ['The amino acid sequence is displayed end-to-end down the axis of a schematic helix.(?<=[.?!])\\\\s+The angle between every two consecutive amino acids is 102.9°.',\n",
       "   'The helical wheel consists of seven corners, corresponding to the fit of seven amino acid residues into every two helical turns.']),\n",
       " ('amino acid',\n",
       "  'subclass of',\n",
       "  'residues',\n",
       "  ['The amino acid sequence is displayed end-to-end down the axis of a schematic helix.(?<=[.?!])\\\\s+The angle between every two consecutive amino acids is 102.9°.',\n",
       "   'The helical wheel consists of seven corners, corresponding to the fit of seven amino acid residues into every two helical turns.']),\n",
       " ('protein',\n",
       "  'has part',\n",
       "  'amino acid',\n",
       "  ['LearnCoil-VMF program [17] and helical wheel diagrams, with 3.5 amino acid per residue, were used to detect coiled-coil regions in the SARS-CoV S2 protein.',\n",
       "   'HMMTOP program [18] was used to predict the location of transmembrane regions in the SARS-CoV S2 protein.']),\n",
       " ('transmembrane regions',\n",
       "  'part of',\n",
       "  'protein',\n",
       "  ['LearnCoil-VMF program [17] and helical wheel diagrams, with 3.5 amino acid per residue, were used to detect coiled-coil regions in the SARS-CoV S2 protein.',\n",
       "   'HMMTOP program [18] was used to predict the location of transmembrane regions in the SARS-CoV S2 protein.']),\n",
       " ('GenBank',\n",
       "  'parent organization',\n",
       "  'National Center for Biotechnology Information',\n",
       "  ['SARS-CoV genomic sequence information (NC_004718) was retrieved from GenBank at the National Center for Biotechnology Information (NCBI).',\n",
       "   'The SARS-CoV E2 precursor protein accession number is gi|30023954.']),\n",
       " ('National Center for Biotechnology Information',\n",
       "  'subsidiary',\n",
       "  'GenBank',\n",
       "  ['SARS-CoV genomic sequence information (NC_004718) was retrieved from GenBank at the National Center for Biotechnology Information (NCBI).',\n",
       "   'The SARS-CoV E2 precursor protein accession number is gi|30023954.']),\n",
       " ('SARS-CoV E2 precursor protein accession number',\n",
       "  'instance of',\n",
       "  'gi|30023954',\n",
       "  ['SARS-CoV genomic sequence information (NC_004718) was retrieved from GenBank at the National Center for Biotechnology Information (NCBI).',\n",
       "   'The SARS-CoV E2 precursor protein accession number is gi|30023954.']),\n",
       " ('S1',\n",
       "  'followed by',\n",
       "  'S2',\n",
       "  ['This precursor is believed to be post-translationally processed to reveal S1 and S2, as confirmed in other Coronaviridae viruses.',\n",
       "   \"The HIV-1 gp160 precursor protein sequence used here is gi|9629363.(?<=[.?!])\\\\s+Gp41 starts at residue 512, after the basic cleavage site.(?<=[.?!])\\\\s+Authors' contributions Y.K.\"]),\n",
       " ('Gp41',\n",
       "  'subclass of',\n",
       "  'precursor protein',\n",
       "  ['This precursor is believed to be post-translationally processed to reveal S1 and S2, as confirmed in other Coronaviridae viruses.',\n",
       "   \"The HIV-1 gp160 precursor protein sequence used here is gi|9629363.(?<=[.?!])\\\\s+Gp41 starts at residue 512, after the basic cleavage site.(?<=[.?!])\\\\s+Authors' contributions Y.K.\"]),\n",
       " ('E.K.',\n",
       "  'member of',\n",
       "  'Y.Y.',\n",
       "  ['was responsible for the initiation of this project.(?<=[.?!])\\\\s+E.Y.L.(?<=[.?!])\\\\s+& Y.K.',\n",
       "   '& Y.K.(?<=[.?!])\\\\s+carried out the technical work, analyzed the data, drafted the manuscript and approved the final manuscript.']),\n",
       " ('drafted',\n",
       "  'product or material produced',\n",
       "  'manuscript',\n",
       "  ['was responsible for the initiation of this project.(?<=[.?!])\\\\s+E.Y.L.(?<=[.?!])\\\\s+& Y.K.',\n",
       "   '& Y.K.(?<=[.?!])\\\\s+carried out the technical work, analyzed the data, drafted the manuscript and approved the final manuscript.']),\n",
       " ('SARS',\n",
       "  'has effect',\n",
       "  'Severe Acute Respiratory Syndrome',\n",
       "  ['BACKGROUND: The exact origin of the cause of the Severe Acute Respiratory Syndrome (SARS) is still an open question.',\n",
       "   'The genomic sequence relationship of SARS-CoV with 30 different single-stranded RNA (ssRNA) viruses of various families was studied using two non-standard approaches.']),\n",
       " ('single-stranded RNA',\n",
       "  'part of',\n",
       "  'viruses',\n",
       "  ['BACKGROUND: The exact origin of the cause of the Severe Acute Respiratory Syndrome (SARS) is still an open question.',\n",
       "   'The genomic sequence relationship of SARS-CoV with 30 different single-stranded RNA (ssRNA) viruses of various families was studied using two non-standard approaches.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['Both approaches began with the vectorial profiling of the tetra-nucleotide usage pattern V for each virus.',\n",
       "   'In approach one, a distance measure of a vector V, based on correlation coefficient was devised to construct a relationship tree by the neighbor-joining algorithm.']),\n",
       " ('neighbor-joining algorithm',\n",
       "  'use',\n",
       "  'relationship tree',\n",
       "  ['Both approaches began with the vectorial profiling of the tetra-nucleotide usage pattern V for each virus.',\n",
       "   'In approach one, a distance measure of a vector V, based on correlation coefficient was devised to construct a relationship tree by the neighbor-joining algorithm.']),\n",
       " ('multivariate factor analysis',\n",
       "  'studies',\n",
       "  'embedded tetra-nucleotide usage',\n",
       "  ['In approach two, a multivariate factor analysis was performed to derive the embedded tetra-nucleotide usage patterns.',\n",
       "   'These patterns were subsequently used to classify the selected viruses.(?<=[.?!])\\\\s+RESULTS: Both approaches yielded relationship outcomes that are consistent with the known virus classification.']),\n",
       " ('embedded tetra-nucleotide usage',\n",
       "  'studied by',\n",
       "  'multivariate factor analysis',\n",
       "  ['In approach two, a multivariate factor analysis was performed to derive the embedded tetra-nucleotide usage patterns.',\n",
       "   'These patterns were subsequently used to classify the selected viruses.(?<=[.?!])\\\\s+RESULTS: Both approaches yielded relationship outcomes that are consistent with the known virus classification.']),\n",
       " ('virus classification',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['In approach two, a multivariate factor analysis was performed to derive the embedded tetra-nucleotide usage patterns.',\n",
       "   'These patterns were subsequently used to classify the selected viruses.(?<=[.?!])\\\\s+RESULTS: Both approaches yielded relationship outcomes that are consistent with the known virus classification.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'RNA virus',\n",
       "  ['They also indicated that the genome of RNA viruses from the same family conform to a specific pattern of word usage.',\n",
       "   'Based on the correlation of the overall tetra-nucleotide usage patterns, the Transmissible Gastroenteritis Virus (TGV) and the Feline CoronaVirus (FCoV) are closest to SARS-CoV.']),\n",
       " ('Transmissible Gastroenteritis Virus',\n",
       "  'parent taxon',\n",
       "  'SARS-CoV',\n",
       "  ['They also indicated that the genome of RNA viruses from the same family conform to a specific pattern of word usage.',\n",
       "   'Based on the correlation of the overall tetra-nucleotide usage patterns, the Transmissible Gastroenteritis Virus (TGV) and the Feline CoronaVirus (FCoV) are closest to SARS-CoV.']),\n",
       " ('DNA',\n",
       "  'has part',\n",
       "  'di-nucleotide',\n",
       "  ['Surprisingly also, the RNA viruses that do not go through a DNA stage displayed a remarkable discrimination against the CpG and UpA di-nucleotide (z = -77.31, -52.48 respectively) and selection for UpG and CpA (z = 65.79,49.99 respectively).',\n",
       "   'Potential factors influencing these biases are discussed.(?<=[.?!])\\\\s+CONCLUSION: The study of genomic word usage is a powerful method to classify RNA viruses.']),\n",
       " ('RNA virus',\n",
       "  'part of',\n",
       "  'genomic',\n",
       "  ['Surprisingly also, the RNA viruses that do not go through a DNA stage displayed a remarkable discrimination against the CpG and UpA di-nucleotide (z = -77.31, -52.48 respectively) and selection for UpG and CpA (z = 65.79,49.99 respectively).',\n",
       "   'Potential factors influencing these biases are discussed.(?<=[.?!])\\\\s+CONCLUSION: The study of genomic word usage is a powerful method to classify RNA viruses.']),\n",
       " ('replicase',\n",
       "  'subclass of',\n",
       "  'open reading frame',\n",
       "  ['The congruence of the relationship outcomes with the known classification indicates that there exist phylogenetic signals in the tetra-nucleotide usage patterns, that is most prominent in the replicase open reading frames.',\n",
       "   'Severe Acute Respiratory Syndrome (SARS), a newly identified infectious disease, has imperilled the health of human population in more than 30 nations.']),\n",
       " ('Severe Acute Respiratory Syndrome',\n",
       "  'instance of',\n",
       "  'newly identified infectious disease',\n",
       "  ['The congruence of the relationship outcomes with the known classification indicates that there exist phylogenetic signals in the tetra-nucleotide usage patterns, that is most prominent in the replicase open reading frames.',\n",
       "   'Severe Acute Respiratory Syndrome (SARS), a newly identified infectious disease, has imperilled the health of human population in more than 30 nations.']),\n",
       " ('GuangDong',\n",
       "  'country',\n",
       "  \"People's Republic of China\",\n",
       "  [\"It has claimed over 812 lives and infected more than 8442 (9.61% death rate) by July 2, 2003 [1] since its outbreak in November 2002 in the province of GuangDong, People's Republic of China.\",\n",
       "   \"By May 15, 2003 , the primary etiological agent for SARS was found to fulfil Koch's postulate through experimental infection of cynomolgus macaques (Macaca fascicularis) [2] .\"]),\n",
       " (\"People's Republic of China\",\n",
       "  'contains administrative territorial entity',\n",
       "  'GuangDong',\n",
       "  [\"It has claimed over 812 lives and infected more than 8442 (9.61% death rate) by July 2, 2003 [1] since its outbreak in November 2002 in the province of GuangDong, People's Republic of China.\",\n",
       "   \"By May 15, 2003 , the primary etiological agent for SARS was found to fulfil Koch's postulate through experimental infection of cynomolgus macaques (Macaca fascicularis) [2] .\"]),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'cynomolgus macaques',\n",
       "  [\"It has claimed over 812 lives and infected more than 8442 (9.61% death rate) by July 2, 2003 [1] since its outbreak in November 2002 in the province of GuangDong, People's Republic of China.\",\n",
       "   \"By May 15, 2003 , the primary etiological agent for SARS was found to fulfil Koch's postulate through experimental infection of cynomolgus macaques (Macaca fascicularis) [2] .\"]),\n",
       " ('cynomolgus macaques',\n",
       "  'has effect',\n",
       "  'SARS',\n",
       "  [\"It has claimed over 812 lives and infected more than 8442 (9.61% death rate) by July 2, 2003 [1] since its outbreak in November 2002 in the province of GuangDong, People's Republic of China.\",\n",
       "   \"By May 15, 2003 , the primary etiological agent for SARS was found to fulfil Koch's postulate through experimental infection of cynomolgus macaques (Macaca fascicularis) [2] .\"]),\n",
       " ('SARS CoronaVirus',\n",
       "  'subclass of',\n",
       "  'SARS-CoV',\n",
       "  ['Chronicles for the discovery of SARS CoronaVirus (SARS-CoV) can be found in articles [e.g.(?<=[.?!])\\\\s+[3, 4] ] and websites [e.g.(?<=[.?!])\\\\s+[5] ].',\n",
       "   'A common question is often asked when investigating viral evolution: what hallmark, in term of genome sequence or RNA word usage, could be used to trace back the emergence of a new pathogen in humans/animals?']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Chronicles for the discovery of SARS CoronaVirus (SARS-CoV) can be found in articles [e.g.(?<=[.?!])\\\\s+[3, 4] ] and websites [e.g.(?<=[.?!])\\\\s+[5] ].',\n",
       "   'A common question is often asked when investigating viral evolution: what hallmark, in term of genome sequence or RNA word usage, could be used to trace back the emergence of a new pathogen in humans/animals?']),\n",
       " ('recombination',\n",
       "  'subclass of',\n",
       "  'mutate',\n",
       "  ['In particular, CoronaViruses are prone to recombination [6, 7] and like all other viruses they mutate at a high frequency [8] .',\n",
       "   'This makes extremely hazardous to try to trace the origin of the virus.']),\n",
       " ('virus',\n",
       "  'has cause',\n",
       "  'origin',\n",
       "  ['In particular, CoronaViruses are prone to recombination [6, 7] and like all other viruses they mutate at a high frequency [8] .',\n",
       "   'This makes extremely hazardous to try to trace the origin of the virus.']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Nevertheless, this prompted us to investigate their relationships using the RNA word usage hoping to identify some RNA viruses that display similar word usage pattern.',\n",
       "   'Such RNA viruses might hint about the origin of SARS-CoV.(?<=[.?!])\\\\s+This study will contribute to our understanding of the RNA word usage of SARS-CoV and some other pathogenic RNA viruses.']),\n",
       " ('SARS-CoV',\n",
       "  'instance of',\n",
       "  'pathogen',\n",
       "  ['Nevertheless, this prompted us to investigate their relationships using the RNA word usage hoping to identify some RNA viruses that display similar word usage pattern.',\n",
       "   'Such RNA viruses might hint about the origin of SARS-CoV.(?<=[.?!])\\\\s+This study will contribute to our understanding of the RNA word usage of SARS-CoV and some other pathogenic RNA viruses.']),\n",
       " ('SARS-CoV',\n",
       "  'subclass of',\n",
       "  'RNA virus',\n",
       "  ['In the present study, we explored the relationships of 31 RNA viruses, which are known to cause diseases to their corresponding hosts with either similar symptoms or infectiousness, including SARS-CoV, based on their global tetra-nucleotide usage pattern.',\n",
       "   'Preliminary analysis of the sequence data indicated that there are [11] [12] [13] [14] open reading frames in the SARS-CoV genome [9] [10] [11] .']),\n",
       " ('open reading frame',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['In the present study, we explored the relationships of 31 RNA viruses, which are known to cause diseases to their corresponding hosts with either similar symptoms or infectiousness, including SARS-CoV, based on their global tetra-nucleotide usage pattern.',\n",
       "   'Preliminary analysis of the sequence data indicated that there are [11] [12] [13] [14] open reading frames in the SARS-CoV genome [9] [10] [11] .']),\n",
       " ('CoronaVirus',\n",
       "  'parent taxon',\n",
       "  'Coronaviridae',\n",
       "  ['The overall gene order for this novel pathogen supported its placement in the family of Coronaviridae which includes the animal/human CoronaViruses.',\n",
       "   'It should be emphasized that the sequence similarity shown is attributed mainly to the large RNAdependent RNA polymerase (replication enzyme or RdRp) residing in the first two open reading frames (ORFs).']),\n",
       " ('Replication enzyme',\n",
       "  'subclass of',\n",
       "  'RNA polymerase',\n",
       "  ['The overall gene order for this novel pathogen supported its placement in the family of Coronaviridae which includes the animal/human CoronaViruses.',\n",
       "   'It should be emphasized that the sequence similarity shown is attributed mainly to the large RNAdependent RNA polymerase (replication enzyme or RdRp) residing in the first two open reading frames (ORFs).']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['These two ORFs constitute more than 65% (>20 kb) of the total genome size and these regions are more conserved in their nucleotide sequences due to their specialized role for viral RNA replication.',\n",
       "   'Therefore, the possible relationship based on the sequence of the replication enzyme alone was also investigated.']),\n",
       " ('RNA',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['These two ORFs constitute more than 65% (>20 kb) of the total genome size and these regions are more conserved in their nucleotide sequences due to their specialized role for viral RNA replication.',\n",
       "   'Therefore, the possible relationship based on the sequence of the replication enzyme alone was also investigated.']),\n",
       " ('sequence',\n",
       "  'facet of',\n",
       "  'replication enzyme',\n",
       "  ['These two ORFs constitute more than 65% (>20 kb) of the total genome size and these regions are more conserved in their nucleotide sequences due to their specialized role for viral RNA replication.',\n",
       "   'Therefore, the possible relationship based on the sequence of the replication enzyme alone was also investigated.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Table 1 presents the breakdown of the RNA sequence into mononucleotide frequencies for the 31 viral genomes in our dataset.',\n",
       "   'Except for the Rabbit Hemorrhagic disease Virus (RHV) that shows a fair usage of the four nucleotides in approximately equal number, the other RNA viruses have a biased genome composition.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['Table 1 presents the breakdown of the RNA sequence into mononucleotide frequencies for the 31 viral genomes in our dataset.',\n",
       "   'Except for the Rabbit Hemorrhagic disease Virus (RHV) that shows a fair usage of the four nucleotides in approximately equal number, the other RNA viruses have a biased genome composition.']),\n",
       " ('U nucleotide',\n",
       "  'opposite of',\n",
       "  'C nucleotide',\n",
       "  ['Bovine CoronaVirus (BCoV) and Human CoronaVirus 229E (HCoV) favor the U nucleotide (35.5% and 34.6%) at the expense of the C nucleotide (15.3% and 16.7%).',\n",
       "   'Relatively strong nucleotide biases are visible in the other genomes and we will mention a few of the extremes.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['Bovine CoronaVirus (BCoV) and Human CoronaVirus 229E (HCoV) favor the U nucleotide (35.5% and 34.6%) at the expense of the C nucleotide (15.3% and 16.7%).',\n",
       "   'Relatively strong nucleotide biases are visible in the other genomes and we will mention a few of the extremes.']),\n",
       " ('Bovine CoronaVirus',\n",
       "  'parent taxon',\n",
       "  'Yellow Fever Virus',\n",
       "  ['The highest base count is 28.4% G in the Yellow Fever Virus (YFV), 38.9% A in the Respiratory Syncytial Virus (RSV), 35.5% U in the Bovine CoronaVirus (BCoV) and 28 .5% C count in the Foot-and-Mouth disease Virus (FMV).',\n",
       "   'The lowest base counts are 15.8% G in the Human Respiratory syncytial Virus (HRV), 21 .2% A in the Equine arteritis Virus (EV1), 20.9% U in the Igbo Ora Virus (IOV) and 13.6% C in the Bovine ephemeral Fever Virus (BFV).']),\n",
       " ('G',\n",
       "  'has part',\n",
       "  'C',\n",
       "  ['The highest base count is 28.4% G in the Yellow Fever Virus (YFV), 38.9% A in the Respiratory Syncytial Virus (RSV), 35.5% U in the Bovine CoronaVirus (BCoV) and 28 .5% C count in the Foot-and-Mouth disease Virus (FMV).',\n",
       "   'The lowest base counts are 15.8% G in the Human Respiratory syncytial Virus (HRV), 21 .2% A in the Equine arteritis Virus (EV1), 20.9% U in the Igbo Ora Virus (IOV) and 13.6% C in the Bovine ephemeral Fever Virus (BFV).']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['The A nucleotide is the most popular base among RNA viruses (ranging from 21.2% to 38.9%), and C is the most variable nucleotide (ranging from 13.6% to 33.1%).',\n",
       "   'From the standpoint of the overall genomic composition analysis, the G+C content is an interesting property for a genome, in that the overall content often correlates with the organism pathogenicity [12] .']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'organism',\n",
       "  ['The A nucleotide is the most popular base among RNA viruses (ranging from 21.2% to 38.9%), and C is the most variable nucleotide (ranging from 13.6% to 33.1%).',\n",
       "   'From the standpoint of the overall genomic composition analysis, the G+C content is an interesting property for a genome, in that the overall content often correlates with the organism pathogenicity [12] .']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'pathogens',\n",
       "  ['Most of the pathogens genomes have a low G+C content, while some such as Mycobacterium tuberculosis has a relatively high G+C content.',\n",
       "   'Therefore, as expected in Table 1 , we noted that most of the pathogenic viruses are A+U-rich (>50%), except for Porcine reproductive and Respiratory syndrome Virus (PRV), Equine arteritis virus (EV1), Rabbit hemorrhagic disease virus (RHV), Simian hemorrhagic Fever Virus (SFV) and Foot-and-Mouth disease Virus C (FMV).']),\n",
       " ('Porcine reproductive and Respiratory syndrome Virus',\n",
       "  'instance of',\n",
       "  'pathogen',\n",
       "  ['Most of the pathogens genomes have a low G+C content, while some such as Mycobacterium tuberculosis has a relatively high G+C content.',\n",
       "   'Therefore, as expected in Table 1 , we noted that most of the pathogenic viruses are A+U-rich (>50%), except for Porcine reproductive and Respiratory syndrome Virus (PRV), Equine arteritis virus (EV1), Rabbit hemorrhagic disease virus (RHV), Simian hemorrhagic Fever Virus (SFV) and Foot-and-Mouth disease Virus C (FMV).']),\n",
       " ('RNA',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['The frequencies of occurrence for di-nucleotides were compared to the random RNA counterparts having the same base proportion in order to compute the z value that reflected their di-nucleotide bias ( Table 2) .',\n",
       "   'Among the 31 virus sequences examined, the frequencies of occurrence for di-nucleotide were not randomly distributed, with only a few exceptional di-nucleotides starting with a purine residue present at the expected frequencies (ApC, ApG, GpC, |z| < 3).']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'di-nucleotide',\n",
       "  ['The frequencies of occurrence for di-nucleotides were compared to the random RNA counterparts having the same base proportion in order to compute the z value that reflected their di-nucleotide bias ( Table 2) .',\n",
       "   'Among the 31 virus sequences examined, the frequencies of occurrence for di-nucleotide were not randomly distributed, with only a few exceptional di-nucleotides starting with a purine residue present at the expected frequencies (ApC, ApG, GpC, |z| < 3).']),\n",
       " ('UpA',\n",
       "  'subclass of',\n",
       "  'di-nucleotide',\n",
       "  ['A remarkable deviation from the expected frequencies occurs for the di-nucleotide pairs CpG and UpA (suppression or under-representation, z < -50) as well as di-nucleotides pairs CpA and UpG (enhancement or over-representation, z > 40).',\n",
       "   'These dinucleotide biases, together with mono-nucleotide bias [13] , have a direct impact on the codon usage of viruses.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'codon',\n",
       "  ['A remarkable deviation from the expected frequencies occurs for the di-nucleotide pairs CpG and UpA (suppression or under-representation, z < -50) as well as di-nucleotides pairs CpA and UpG (enhancement or over-representation, z > 40).',\n",
       "   'These dinucleotide biases, together with mono-nucleotide bias [13] , have a direct impact on the codon usage of viruses.']),\n",
       " ('codon',\n",
       "  'part of',\n",
       "  'protein coding sequences',\n",
       "  ['For example, in the codon usage for the 24 protein coding sequences in human CoronaVirus 229E (Table 3) , only 2.85% of codons contain the under-represented subword CpG di-nucleotide whereas 11.26% of the codons contain the over-represented CpA di-nucleotide (the aggregate codon usage containing each di-nucleotide subword without mono-and di-nucleotide bias is close to 6.25%).',\n",
       "   'In double stranded DNA genomes the deficiency in dinucleotide CpG is often supposed to be due to the fact that they are the targets for methyltransferase activity that leads to cytosine deamination [14, 15] .']),\n",
       " ('DNA',\n",
       "  'has part',\n",
       "  'dinucleotide',\n",
       "  ['For example, in the codon usage for the 24 protein coding sequences in human CoronaVirus 229E (Table 3) , only 2.85% of codons contain the under-represented subword CpG di-nucleotide whereas 11.26% of the codons contain the over-represented CpA di-nucleotide (the aggregate codon usage containing each di-nucleotide subword without mono-and di-nucleotide bias is close to 6.25%).',\n",
       "   'In double stranded DNA genomes the deficiency in dinucleotide CpG is often supposed to be due to the fact that they are the targets for methyltransferase activity that leads to cytosine deamination [14, 15] .']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'viral RNA',\n",
       "  ['It is however unlikely that the mechanism of deamination that alters the genetic contents at the DNA level would affect the viral RNA content of most RNA viruses without a DNA stage.',\n",
       "   'There might exist specific cytosine RNA methylases that could be responsible for this effect [16] .']),\n",
       " ('cytosine',\n",
       "  'subclass of',\n",
       "  'RNA',\n",
       "  ['It is however unlikely that the mechanism of deamination that alters the genetic contents at the DNA level would affect the viral RNA content of most RNA viruses without a DNA stage.',\n",
       "   'There might exist specific cytosine RNA methylases that could be responsible for this effect [16] .']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'cytosine',\n",
       "  ['However it is more consistent to propose that, unlike the mechanism of cytosine deamination in the DNA realm, the dominating process is cytosine deamination in RNA viruses, converting cytosine to uracil (C ♦ U) instead of thymine (T).',\n",
       "   'As a consequence of this mechanism, di-nucleotide CpG changes to either di-nucleotide UpG or CpA in the direct/ complementary strands of RNA viruses and causes the over-representation in di-nucleotide UpG and CpA (z > 19) .']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'complementary strands',\n",
       "  ['However it is more consistent to propose that, unlike the mechanism of cytosine deamination in the DNA realm, the dominating process is cytosine deamination in RNA viruses, converting cytosine to uracil (C ♦ U) instead of thymine (T).',\n",
       "   'As a consequence of this mechanism, di-nucleotide CpG changes to either di-nucleotide UpG or CpA in the direct/ complementary strands of RNA viruses and causes the over-representation in di-nucleotide UpG and CpA (z > 19) .']),\n",
       " ('single stranded',\n",
       "  'opposite of',\n",
       "  'double-stranded',\n",
       "  ['Interestingly, there is experimental evidence in vitro that the rate of cytosine deamination is faster (>100 times) in the single stranded than in double-stranded state [17] .',\n",
       "   'Apart from the under-representation in dinucleotide CpG and over-representation in di-nucleotide CpA and UpG, the reason for the observed di-nucleotide UpA scarcity in RNA may be explained by its chemical lability [18] .']),\n",
       " ('RNA',\n",
       "  'has part',\n",
       "  'dinucleotide',\n",
       "  ['Interestingly, there is experimental evidence in vitro that the rate of cytosine deamination is faster (>100 times) in the single stranded than in double-stranded state [17] .',\n",
       "   'Apart from the under-representation in dinucleotide CpG and over-representation in di-nucleotide CpA and UpG, the reason for the observed di-nucleotide UpA scarcity in RNA may be explained by its chemical lability [18] .']),\n",
       " ('RNA',\n",
       "  'has part',\n",
       "  'dinucleotide',\n",
       "  ['The UpA dinucleotide is chemically the most unstable among the 16 dinucleotides.(?<=[.?!])\\\\s+Furthermore, UpA The information about 31 RNA viruses being investigated in this study.',\n",
       "   'Their accession number, abbreviation, genome size, number of segments and whether they undergo DNA stage are tabulated.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'DNA',\n",
       "  ['The UpA dinucleotide is chemically the most unstable among the 16 dinucleotides.(?<=[.?!])\\\\s+Furthermore, UpA The information about 31 RNA viruses being investigated in this study.',\n",
       "   'Their accession number, abbreviation, genome size, number of segments and whether they undergo DNA stage are tabulated.']),\n",
       " ('DNA',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['The UpA dinucleotide is chemically the most unstable among the 16 dinucleotides.(?<=[.?!])\\\\s+Furthermore, UpA The information about 31 RNA viruses being investigated in this study.',\n",
       "   'Their accession number, abbreviation, genome size, number of segments and whether they undergo DNA stage are tabulated.']),\n",
       " ('RNA',\n",
       "  'subclass of',\n",
       "  'nucleic acid',\n",
       "  ['The breakdown of the RNA nucleic acids and A+U contents are also shown.(?<=[.?!])\\\\s+appears to be a preferential target for ribonucleases [19] .',\n",
       "   'This lability would create a selection pressure against dinucleotide UpA in RNA viruses.']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'dinucleotide UpA',\n",
       "  ['The breakdown of the RNA nucleic acids and A+U contents are also shown.(?<=[.?!])\\\\s+appears to be a preferential target for ribonucleases [19] .',\n",
       "   'This lability would create a selection pressure against dinucleotide UpA in RNA viruses.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['If we choose a critical value for z (|z| = 3.29) that only allows a chance of 1 in 1000 error for classifying a word as biased (over/under-represented), all di-nucleotides show some kind of bias in their usage pattern across 31 different viruses (Table 4 , derived from the complete form of Table  2 provided as the additional file 1).',\n",
       "   'The causes for these biases await further investigation.(?<=[.?!])\\\\s+Inspection of the tetra-nucleotide usage pattern for RNA viruses (additional file 2) reveals considerable differences.']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['If we choose a critical value for z (|z| = 3.29) that only allows a chance of 1 in 1000 error for classifying a word as biased (over/under-represented), all di-nucleotides show some kind of bias in their usage pattern across 31 different viruses (Table 4 , derived from the complete form of Table  2 provided as the additional file 1).',\n",
       "   'The causes for these biases await further investigation.(?<=[.?!])\\\\s+Inspection of the tetra-nucleotide usage pattern for RNA viruses (additional file 2) reveals considerable differences.']),\n",
       " ('RNA',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['The frequencies of occurrence for tetra-nucleotides were compared to artificial chromosomes constructed as random RNA sequences having the same nucleotide succession up to order three to compute the z values that reflect their tetra-nucleotide bias in the corresponding virus (Table 5 ).',\n",
       "   'If we choose a critical value for z (|z| = 3.29) that only allows a chance of 1 in 1000 error for classifying a word as over/under-represented, 96% of the tetra-nucleotides show a strong bias in their usage pattern across 31 viruses (shown in Table 4 , derived from the complete form of Table 5 provided as the additional file 1).']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['The frequencies of occurrence for tetra-nucleotides were compared to artificial chromosomes constructed as random RNA sequences having the same nucleotide succession up to order three to compute the z values that reflect their tetra-nucleotide bias in the corresponding virus (Table 5 ).',\n",
       "   'If we choose a critical value for z (|z| = 3.29) that only allows a chance of 1 in 1000 error for classifying a word as over/under-represented, 96% of the tetra-nucleotides show a strong bias in their usage pattern across 31 viruses (shown in Table 4 , derived from the complete form of Table 5 provided as the additional file 1).']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'genomes',\n",
       "  ['This indicated strongly that tetra-nucleotides are being used in a different manner between different viruses, providing us with a tool to study the relationships between viruses based on the tetra-nucleotide bias exhibited in their genomes.',\n",
       "   'The di-nucleotide bias in six RNA viruses.(?<=[.?!])\\\\s+The z value quantifies the di-nucleotide bias as defined in equation 1.']),\n",
       " ('genomes',\n",
       "  'part of',\n",
       "  'viruses',\n",
       "  ['This indicated strongly that tetra-nucleotides are being used in a different manner between different viruses, providing us with a tool to study the relationships between viruses based on the tetra-nucleotide bias exhibited in their genomes.',\n",
       "   'The di-nucleotide bias in six RNA viruses.(?<=[.?!])\\\\s+The z value quantifies the di-nucleotide bias as defined in equation 1.']),\n",
       " ('di-nucleotide bias in six RNA viruses',\n",
       "  'instance of',\n",
       "  'di-nucleotide bias',\n",
       "  ['This indicated strongly that tetra-nucleotides are being used in a different manner between different viruses, providing us with a tool to study the relationships between viruses based on the tetra-nucleotide bias exhibited in their genomes.',\n",
       "   'The di-nucleotide bias in six RNA viruses.(?<=[.?!])\\\\s+The z value quantifies the di-nucleotide bias as defined in equation 1.']),\n",
       " ('N(w) and E(w)',\n",
       "  'instance of',\n",
       "  'frequency of occurrence',\n",
       "  ['N (w) and E (w) are actual and expected frequency of occurrence for word w.(?<=[.?!])\\\\s+The last column is the average z value across 31 RNA viruses.',\n",
       "   'The percentage of biased di-nucleotides and tetra-nucleotides that shows strong biases (lzl > 3.29) in 31 RNA viruses (right).']),\n",
       " ('di-nucleotide',\n",
       "  'followed by',\n",
       "  'tetra-nucleotide',\n",
       "  ['N (w) and E (w) are actual and expected frequency of occurrence for word w.(?<=[.?!])\\\\s+The last column is the average z value across 31 RNA viruses.',\n",
       "   'The percentage of biased di-nucleotides and tetra-nucleotides that shows strong biases (lzl > 3.29) in 31 RNA viruses (right).']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'di-nucleotide',\n",
       "  ['For di-nucleotides, all 16 (100%) of them show strong biases in part of or all 31 RNA viruses.',\n",
       "   'For tetra-nucleotides, 246 (96%) of the tetra-nucleotides show strong biases in part of or all 31 RNA viruses.(?<=[.?!])\\\\s+Table 5 : Tetra-nucleotide bias for three RNA viruses.']),\n",
       " ('Tetra-nucleotide bias',\n",
       "  'facet of',\n",
       "  'Tetra-nucleotide',\n",
       "  ['For di-nucleotides, all 16 (100%) of them show strong biases in part of or all 31 RNA viruses.',\n",
       "   'For tetra-nucleotides, 246 (96%) of the tetra-nucleotides show strong biases in part of or all 31 RNA viruses.(?<=[.?!])\\\\s+Table 5 : Tetra-nucleotide bias for three RNA viruses.']),\n",
       " ('tetra-nucleotide bias',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['The tetra-nucleotide bias in three viruses.(?<=[.?!])\\\\s+z value quantifies the tetra-nucleotide bias, as defined in equation (1) .',\n",
       "   'N (w) and E (w) are actual and expected frequency of occurrence for word w.']),\n",
       " ('N(w) and E(w)',\n",
       "  'instance of',\n",
       "  'frequency of occurrence',\n",
       "  ['The tetra-nucleotide bias in three viruses.(?<=[.?!])\\\\s+z value quantifies the tetra-nucleotide bias, as defined in equation (1) .',\n",
       "   'N (w) and E (w) are actual and expected frequency of occurrence for word w.']),\n",
       " ('replication enzyme',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['Two relationship trees were derived, one from the entire genome and the other from the replication enzyme ( Figure 1 ).',\n",
       "   'The result based on the replication enzyme sequence was included because these regions in RNA viruses are submitted to a strong selective pressure to ensure successful replication of their own RNA in the host cell.']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Two relationship trees were derived, one from the entire genome and the other from the replication enzyme ( Figure 1 ).',\n",
       "   'The result based on the replication enzyme sequence was included because these regions in RNA viruses are submitted to a strong selective pressure to ensure successful replication of their own RNA in the host cell.']),\n",
       " ('distance tree',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['The two distance trees can be clustered distinctly into two major groups of viruses.',\n",
       "   'Interestingly, this clustering validates our approach, since these clusters are consistent with biological properties of the viruses: Group #1 corresponds to all positive strand ssRNA viruses while Group #2 corresponds to negative strand ssRNA viruses.']),\n",
       " ('positive strand',\n",
       "  'subclass of',\n",
       "  'virus',\n",
       "  ['The two distance trees can be clustered distinctly into two major groups of viruses.',\n",
       "   'Interestingly, this clustering validates our approach, since these clusters are consistent with biological properties of the viruses: Group #1 corresponds to all positive strand ssRNA viruses while Group #2 corresponds to negative strand ssRNA viruses.']),\n",
       " ('tetra-nucleotide',\n",
       "  'instance of',\n",
       "  'evolutionary paths',\n",
       "  ['Each group must undergo different evolutionary paths which lead to their distinct pattern in tetra-nucleotide usage.',\n",
       "   'The classification for the two main groups of viruses (positive/ negative strand ssRNA viruses) demonstrate a level of congruence with the taxonomy of the viruses [20] and indicated that there exists a relationship signal in tetranucleotide usage patterns.']),\n",
       " ('ssRNA',\n",
       "  'has part',\n",
       "  'tetranucleotide',\n",
       "  ['Each group must undergo different evolutionary paths which lead to their distinct pattern in tetra-nucleotide usage.',\n",
       "   'The classification for the two main groups of viruses (positive/ negative strand ssRNA viruses) demonstrate a level of congruence with the taxonomy of the viruses [20] and indicated that there exists a relationship signal in tetranucleotide usage patterns.']),\n",
       " ('eigenvectors',\n",
       "  'part of',\n",
       "  'Relationship tree',\n",
       "  ['Two Relationship trees based on the correlation coefficients of tetra-nucleotide usage bias  The overall tetra-nucleotide usage pattern (additional file 2) was decomposed into several eigen-vectors using a factor analysis algorithm.',\n",
       "   'They are the uncorrelated components of the original usage pattern embedded within the overall tetra-nucleotide usage pattern.']),\n",
       " ('tetra-nucleotide usage pattern',\n",
       "  'instance of',\n",
       "  'usage pattern',\n",
       "  ['Two Relationship trees based on the correlation coefficients of tetra-nucleotide usage bias  The overall tetra-nucleotide usage pattern (additional file 2) was decomposed into several eigen-vectors using a factor analysis algorithm.',\n",
       "   'They are the uncorrelated components of the original usage pattern embedded within the overall tetra-nucleotide usage pattern.']),\n",
       " ('eigen-vectors',\n",
       "  'subclass of',\n",
       "  'variance',\n",
       "  ['Three eigen-vectors, which carry 83.3% of the variance for the viral tetra-nucleotide usage patterns, were retained ( Figure 2 ).',\n",
       "   'From the three dimensional figures (Figure 3, Figure 4 , Figure 5 and Figure 6 ) plotted against these retained eigen-vectors, the negative strand ssRNA viruses stemmed clearly out from the positive strand ssRNA viruses.']),\n",
       " ('negative strand',\n",
       "  'opposite of',\n",
       "  'positive strand',\n",
       "  ['Three eigen-vectors, which carry 83.3% of the variance for the viral tetra-nucleotide usage patterns, were retained ( Figure 2 ).',\n",
       "   'From the three dimensional figures (Figure 3, Figure 4 , Figure 5 and Figure 6 ) plotted against these retained eigen-vectors, the negative strand ssRNA viruses stemmed clearly out from the positive strand ssRNA viruses.']),\n",
       " ('positive strand',\n",
       "  'opposite of',\n",
       "  'negative strand',\n",
       "  ['Three eigen-vectors, which carry 83.3% of the variance for the viral tetra-nucleotide usage patterns, were retained ( Figure 2 ).',\n",
       "   'From the three dimensional figures (Figure 3, Figure 4 , Figure 5 and Figure 6 ) plotted against these retained eigen-vectors, the negative strand ssRNA viruses stemmed clearly out from the positive strand ssRNA viruses.']),\n",
       " ('3 rd',\n",
       "  'subclass of',\n",
       "  'eigen-vectors',\n",
       "  ['This is most obvious when the axes of projection were the 1 st and 3 rd eigen-vectors.',\n",
       "   'This indicated that both types of viruses have a complex component of tetra-nucleotide usage patterns and that these patterns changes with different family of viruses.']),\n",
       " ('tetra-nucleotide usage pattern',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['This is most obvious when the axes of projection were the 1 st and 3 rd eigen-vectors.',\n",
       "   'This indicated that both types of viruses have a complex component of tetra-nucleotide usage patterns and that these patterns changes with different family of viruses.']),\n",
       " ('ssRNA virus',\n",
       "  'subclass of',\n",
       "  'RNA virus',\n",
       "  ['In the result based on replication enzyme sequence (Figure 3 and Figure 4 ), we observed a clear splitting between two main families of RNA viruses (positive/negative strand ssRNA virus).',\n",
       "   'All viruses that belong to a specific family were clustered together closely.']),\n",
       " ('family',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['In the result based on replication enzyme sequence (Figure 3 and Figure 4 ), we observed a clear splitting between two main families of RNA viruses (positive/negative strand ssRNA virus).',\n",
       "   'All viruses that belong to a specific family were clustered together closely.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['This pointed to an interesting hypothesis that the replication enzyme sequence between closely related RNA viruses adopt a common word usage pattern that are closely linked.',\n",
       "   'In addition, it is clear that the viruses from different family groups adopt different strategy of word usage.']),\n",
       " ('family',\n",
       "  'has part',\n",
       "  'viruses',\n",
       "  ['This pointed to an interesting hypothesis that the replication enzyme sequence between closely related RNA viruses adopt a common word usage pattern that are closely linked.',\n",
       "   'In addition, it is clear that the viruses from different family groups adopt different strategy of word usage.']),\n",
       " ('tetra-nucleotide',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['However in Figure 5 and Figure 6 , when we project the tetra-nucleotide usage patterns (entire genome) for each virus on the 1 st , 2 nd and 3 rd eigen-vector axes, the separation between viruses showed a different outcome when V was derived from the entire genome.',\n",
       "   'The two main families of viruses were grouped into three clusters, two being allocated to the positive strand ssRNA viruses.']),\n",
       " ('ssRNA virus',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['However in Figure 5 and Figure 6 , when we project the tetra-nucleotide usage patterns (entire genome) for each virus on the 1 st , 2 nd and 3 rd eigen-vector axes, the separation between viruses showed a different outcome when V was derived from the entire genome.',\n",
       "   'The two main families of viruses were grouped into three clusters, two being allocated to the positive strand ssRNA viruses.']),\n",
       " ('Corona-Virus',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['It is particularly interesting that all viruses in the upper left corner corresponded to the viruses originating from the Corona-Virus family.',\n",
       "   'Unexpectedly, the Hantaan Virus (HV1) is the only negative strand ssRNA virus to have a high loading on the eigen-vector that corresponded to the tetranucleotide usage pattern for the positive strand ssRNA viruses.']),\n",
       " ('Hantaan Virus',\n",
       "  'instance of',\n",
       "  'eigen-vector',\n",
       "  ['It is particularly interesting that all viruses in the upper left corner corresponded to the viruses originating from the Corona-Virus family.',\n",
       "   'Unexpectedly, the Hantaan Virus (HV1) is the only negative strand ssRNA virus to have a high loading on the eigen-vector that corresponded to the tetranucleotide usage pattern for the positive strand ssRNA viruses.']),\n",
       " ('correlation coefficient',\n",
       "  'part of',\n",
       "  'relationship tree',\n",
       "  ['It is important to realize what factor analysis will provide and how this analysis is different from the previous method of relationship tree generation using correlation coefficient.',\n",
       "   'For the previous method that is based on correlation coefficient of word usage patterns, it treats the vectorial profiling V for each virus as a whole entity, However, the factor analysis considered the vectorial profiling V as a superposition of many patterns which can be separated into mutually uncorrelated patterns of word usage.']),\n",
       " ('correlation coefficient',\n",
       "  'facet of',\n",
       "  'word usage',\n",
       "  ['It is important to realize what factor analysis will provide and how this analysis is different from the previous method of relationship tree generation using correlation coefficient.',\n",
       "   'For the previous method that is based on correlation coefficient of word usage patterns, it treats the vectorial profiling V for each virus as a whole entity, However, the factor analysis considered the vectorial profiling V as a superposition of many patterns which can be separated into mutually uncorrelated patterns of word usage.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['Each eigen-vector represents the embedded component of RNA word usage patterns communalised by a group of viruses presumably under the same selection pressures.',\n",
       "   'By projecting the overall usage patterns on these eigen-vectors, it is possible to determine a group of viruses that adopt a common strategy of word usage.']),\n",
       " ('word usage',\n",
       "  'facet of',\n",
       "  'viruses',\n",
       "  ['Each eigen-vector represents the embedded component of RNA word usage patterns communalised by a group of viruses presumably under the same selection pressures.',\n",
       "   'By projecting the overall usage patterns on these eigen-vectors, it is possible to determine a group of viruses that adopt a common strategy of word usage.']),\n",
       " ('RNA virus',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['Using the two approaches to study the tetra-nucleotide usage pattern in RNA viruses, we reached the following conclusions: 1.',\n",
       "   'Based on the correlation of the overall tetra-nucleotide usage patterns, the Transmissible Gastroenteritis Virus (TGV) and the Feline CoronaVirus (FCoV) are closest to SARS-CoV.']),\n",
       " ('Transmissible Gastroenteritis Virus',\n",
       "  'parent taxon',\n",
       "  'SARS-CoV',\n",
       "  ['Using the two approaches to study the tetra-nucleotide usage pattern in RNA viruses, we reached the following conclusions: 1.',\n",
       "   'Based on the correlation of the overall tetra-nucleotide usage patterns, the Transmissible Gastroenteritis Virus (TGV) and the Feline CoronaVirus (FCoV) are closest to SARS-CoV.']),\n",
       " ('Figure 2',\n",
       "  'depicts',\n",
       "  'eigen-vectors',\n",
       "  ['Relationship between the number of eigen-vectors retained and the percentage of the variance they represent in the entire usage patterns for 31 viruses Figure 2 Relationship between the number of eigen-vectors retained and the percentage of the variance they represent in the entire usage patterns for 31 viruses.',\n",
       "   'As each consecutive factor is defined to identify a usage pattern that is not captured by the preceding eigen-vectors, each consecutive factors are therefore independent of each other.']),\n",
       " ('usage pattern',\n",
       "  'subclass of',\n",
       "  'eigen-vectors',\n",
       "  ['Relationship between the number of eigen-vectors retained and the percentage of the variance they represent in the entire usage patterns for 31 viruses Figure 2 Relationship between the number of eigen-vectors retained and the percentage of the variance they represent in the entire usage patterns for 31 viruses.',\n",
       "   'As each consecutive factor is defined to identify a usage pattern that is not captured by the preceding eigen-vectors, each consecutive factors are therefore independent of each other.']),\n",
       " ('order',\n",
       "  'facet of',\n",
       "  'eigen-vectors',\n",
       "  ['In addition, the order for the consecutive eigen-vectors is extracted with diminishing importance.(?<=[.?!])\\\\s+2.',\n",
       "   'Based on the three most significant eigen-vectors, the genomes of the viruses from the same family conform to a similar tetra-nucleotide usage pattern, irrespective of their genome size.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'genome',\n",
       "  ['In addition, the order for the consecutive eigen-vectors is extracted with diminishing importance.(?<=[.?!])\\\\s+2.',\n",
       "   'Based on the three most significant eigen-vectors, the genomes of the viruses from the same family conform to a similar tetra-nucleotide usage pattern, irrespective of their genome size.']),\n",
       " ('RNA virus',\n",
       "  'subclass of',\n",
       "  'word usage',\n",
       "  ['3.(?<=[.?!])\\\\s+The study of word usage is a powerful method to classify RNA viruses.',\n",
       "   'The congruence of the relationship trees with the known classification indicates that there exist phylogenetic signals in tetra-nucleotide usage patterns, and this signal is most prominent in the replicase open reading frames.']),\n",
       " ('replicase',\n",
       "  'subclass of',\n",
       "  'open reading frame',\n",
       "  ['3.(?<=[.?!])\\\\s+The study of word usage is a powerful method to classify RNA viruses.',\n",
       "   'The congruence of the relationship trees with the known classification indicates that there exist phylogenetic signals in tetra-nucleotide usage patterns, and this signal is most prominent in the replicase open reading frames.']),\n",
       " ('positive strand',\n",
       "  'subclass of',\n",
       "  'genomic',\n",
       "  ['We focused our study on the genomic sequences (their translated strand) of ssRNA viruses (Table 1) , which incorporated 20 species from the family of positive strand ssRNA viruses and 11 species from the family of negative strand ssRNA viruses.',\n",
       "   'We are aware of the fact that these viruses constitute completely different species, most probably unrelated to one another.']),\n",
       " ('different species',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['We focused our study on the genomic sequences (their translated strand) of ssRNA viruses (Table 1) , which incorporated 20 species from the family of positive strand ssRNA viruses and 11 species from the family of negative strand ssRNA viruses.',\n",
       "   'We are aware of the fact that these viruses constitute completely different species, most probably unrelated to one another.']),\n",
       " ('background properties',\n",
       "  'facet of',\n",
       "  'statistical',\n",
       "  ['They are included in a common study in order to try to have means to identify relevant features from purely statistical background properties.',\n",
       "   'The coverage included the viruses that are known to cause diseases to their corresponding hosts.(?<=[.?!])\\\\s+The acronym for each virus is shown in the table and is referred to throughout this study.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'hosts',\n",
       "  ['They are included in a common study in order to try to have means to identify relevant features from purely statistical background properties.',\n",
       "   'The coverage included the viruses that are known to cause diseases to their corresponding hosts.(?<=[.?!])\\\\s+The acronym for each virus is shown in the table and is referred to throughout this study.']),\n",
       " ('eigen-vectors',\n",
       "  'part of',\n",
       "  '3-D plot',\n",
       "  [\"All sequences corresponding to 3-D plot for the vectorial profiling of each virus onto the three eigen-vectors [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7 ,3,10,6,2,5,1]* (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\",\n",
       "   \"For the middle figure, the order for 'o' is [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7 ,3,10,6,2,5,1]* (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\"]),\n",
       " ('middle figure',\n",
       "  'instance of',\n",
       "  'order',\n",
       "  [\"All sequences corresponding to 3-D plot for the vectorial profiling of each virus onto the three eigen-vectors [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7 ,3,10,6,2,5,1]* (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\",\n",
       "   \"For the middle figure, the order for 'o' is [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7 ,3,10,6,2,5,1]* (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\"]),\n",
       " ('top figure',\n",
       "  'opposite of',\n",
       "  'bottom figure',\n",
       "  [\"For the bottom figure, the order for 'o' is [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7, 3, 10, 6, 2, 5, 1] * (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\",\n",
       "   '*The corresponded virus for each number follows Figure 3 .(?<=[.?!])\\\\s+Sun Fire 6800 Server with 24 CPUs (each running with a clock speed of 900 MHz) was employed throughout this study.']),\n",
       " ('Sun Fire',\n",
       "  'instance of',\n",
       "  'Server',\n",
       "  [\"For the bottom figure, the order for 'o' is [15, 17, 12, 16, 8, 14, 9, 11, 13, 4, 7, 3, 10, 6, 2, 5, 1] * (left to right), whereas 'x' is [24, 27, 25, 28, 22, 26, 18, 23, 20, 19, 21] * (left to right).\",\n",
       "   '*The corresponded virus for each number follows Figure 3 .(?<=[.?!])\\\\s+Sun Fire 6800 Server with 24 CPUs (each running with a clock speed of 900 MHz) was employed throughout this study.']),\n",
       " ('Matlab',\n",
       "  'instance of',\n",
       "  'Technical Programming language',\n",
       "  ['The computation of correlation coefficient and factor analysis algorithm were implemented using Matlab Technical Programming language.',\n",
       "   \"It is necessary to address the question of how we counted the number of time each tetra-nucleotide (for example 'GAGA' or any other tetra-nucleotide), appeared in a given genome.\"]),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['The computation of correlation coefficient and factor analysis algorithm were implemented using Matlab Technical Programming language.',\n",
       "   \"It is necessary to address the question of how we counted the number of time each tetra-nucleotide (for example 'GAGA' or any other tetra-nucleotide), appeared in a given genome.\"]),\n",
       " ('UAU-GAGAGAUCCGAGA',\n",
       "  'instance of',\n",
       "  'sequence',\n",
       "  ['For this study, we adopted the convention of not counting overlapping words [24] .(?<=[.?!])\\\\s+Take a sequence \"UAU-GAGAGAUCCGAGA\\' as example.',\n",
       "   \"With second or higher overlapping words not counted, the tetra-nucleotide 'GAGA' is counted as occurring only twice, namely in position 4-7 and 13-16.\"]),\n",
       " ('GAGA',\n",
       "  'subclass of',\n",
       "  'tetra-nucleotide',\n",
       "  ['For this study, we adopted the convention of not counting overlapping words [24] .(?<=[.?!])\\\\s+Take a sequence \"UAU-GAGAGAUCCGAGA\\' as example.',\n",
       "   \"With second or higher overlapping words not counted, the tetra-nucleotide 'GAGA' is counted as occurring only twice, namely in position 4-7 and 13-16.\"]),\n",
       " ('GAGA',\n",
       "  'length',\n",
       "  '6',\n",
       "  [\"Positions 6-9 are omitted because they overlap with 'GAGA' at position 4-7.\",\n",
       "   \"However, when we counted tetra-nucleotide 'UGAG', position 3-6 would also be registered as position 4-6 already recorded when counting tetra-nucleotide 'GAGA'.\"]),\n",
       " ('GAGA',\n",
       "  'instance of',\n",
       "  'tetra-nucleotide',\n",
       "  [\"Positions 6-9 are omitted because they overlap with 'GAGA' at position 4-7.\",\n",
       "   \"However, when we counted tetra-nucleotide 'UGAG', position 3-6 would also be registered as position 4-6 already recorded when counting tetra-nucleotide 'GAGA'.\"]),\n",
       " ('UAGA',\n",
       "  'followed by',\n",
       "  'GACA',\n",
       "  [\"In short, all frequency counting of tetra-nucleotide were started anew when we changed from counting the frequency of one tetra-nucleotide to another; this was to preserve the correlation of tetra-nucleotides which have overlapping subword (e.g: 'UAGA' and 'GACA').\",\n",
       "   'A table showing the frequencies of tetra-nucleotides is shown in the additional file 2.']),\n",
       " ('tetra-nucleotide',\n",
       "  'subclass of',\n",
       "  'nucleotide',\n",
       "  [\"In short, all frequency counting of tetra-nucleotide were started anew when we changed from counting the frequency of one tetra-nucleotide to another; this was to preserve the correlation of tetra-nucleotides which have overlapping subword (e.g: 'UAGA' and 'GACA').\",\n",
       "   'A table showing the frequencies of tetra-nucleotides is shown in the additional file 2.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['The nucleotide composition has being suggested to be a specific characteristic in different virus phylogeny [25] .',\n",
       "   'Because most viral genomes are short, and because we lack a prior information on the tempo and modes of evolution of RNA viruses, we proceeded as follows.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'RNA virus',\n",
       "  ['The nucleotide composition has being suggested to be a specific characteristic in different virus phylogeny [25] .',\n",
       "   'Because most viral genomes are short, and because we lack a prior information on the tempo and modes of evolution of RNA viruses, we proceeded as follows.']),\n",
       " ('word',\n",
       "  'subclass of',\n",
       "  'vector',\n",
       "  ['We created a vector, V = [C 1 ,C 2 , ...(?<=[.?!])\\\\s+C i , ...(?<=[.?!])\\\\s+C k ], with each element representing the frequency for a specific RNA word of length n.',\n",
       "   'The number of components (k) in V increases exponentially with word size (n) -k = 4 n .(?<=[.?!])\\\\s+In order to use V for discrimination between viruses, two criteria must be met.']),\n",
       " ('V',\n",
       "  'instance of',\n",
       "  'word size',\n",
       "  ['We created a vector, V = [C 1 ,C 2 , ...(?<=[.?!])\\\\s+C i , ...(?<=[.?!])\\\\s+C k ], with each element representing the frequency for a specific RNA word of length n.',\n",
       "   'The number of components (k) in V increases exponentially with word size (n) -k = 4 n .(?<=[.?!])\\\\s+In order to use V for discrimination between viruses, two criteria must be met.']),\n",
       " ('V',\n",
       "  'instance of',\n",
       "  'word size',\n",
       "  ['We created a vector, V = [C 1 ,C 2 , ...(?<=[.?!])\\\\s+C i , ...(?<=[.?!])\\\\s+C k ], with each element representing the frequency for a specific RNA word of length n.',\n",
       "   'The number of components (k) in V increases exponentially with word size (n) -k = 4 n .(?<=[.?!])\\\\s+In order to use V for discrimination between viruses, two criteria must be met.']),\n",
       " ('tri-nucleotide',\n",
       "  'followed by',\n",
       "  'tetra-nucleotide',\n",
       "  ['First, V must contain sufficient components (di-nucleotide k = 16; tri-nucleotide k = 64; tetra-nucleotide k = 256); second, the frequencies for tetra-nucleotides must show a prominent bias (over/under-representation) that is unique for a family of viruses.',\n",
       "   'For the first criteria, there are pros and cons for choosing either longer or shorter words.']),\n",
       " ('pros',\n",
       "  'opposite of',\n",
       "  'cons',\n",
       "  ['First, V must contain sufficient components (di-nucleotide k = 16; tri-nucleotide k = 64; tetra-nucleotide k = 256); second, the frequencies for tetra-nucleotides must show a prominent bias (over/under-representation) that is unique for a family of viruses.',\n",
       "   'For the first criteria, there are pros and cons for choosing either longer or shorter words.']),\n",
       " ('Figure 5',\n",
       "  'followed by',\n",
       "  'Figure 6',\n",
       "  ['When the shorter words are used, they inherit the problem of inadequate representation of the viral genome because the long motifs will be 2-D plots for Figure 5 with different viewpoint specifications Figure 6 2-D plots for Figure 5 with different viewpoint specifications.',\n",
       "   'The tetra-nucleotide usage patterns Figure 5 .(?<=[.?!])\\\\s+neglected.(?<=[.?!])\\\\s+But the shorter words have an advantage of saving computational time.']),\n",
       " ('Figure 5',\n",
       "  'depicts',\n",
       "  'tetra-nucleotide usage patterns',\n",
       "  ['When the shorter words are used, they inherit the problem of inadequate representation of the viral genome because the long motifs will be 2-D plots for Figure 5 with different viewpoint specifications Figure 6 2-D plots for Figure 5 with different viewpoint specifications.',\n",
       "   'The tetra-nucleotide usage patterns Figure 5 .(?<=[.?!])\\\\s+neglected.(?<=[.?!])\\\\s+But the shorter words have an advantage of saving computational time.']),\n",
       " ('word set',\n",
       "  'facet of',\n",
       "  'computer tractability',\n",
       "  ['On the other hand, when the longer words are used, they cause a problem of computer tractability due to a larger word set to explore (k = 4 n ).',\n",
       "   'However, the larger words have an advantage of accounting for the correlation of their sub-words.']),\n",
       " ('sub-word',\n",
       "  'subclass of',\n",
       "  'word',\n",
       "  ['On the other hand, when the longer words are used, they cause a problem of computer tractability due to a larger word set to explore (k = 4 n ).',\n",
       "   'However, the larger words have an advantage of accounting for the correlation of their sub-words.']),\n",
       " ('statistical analysis',\n",
       "  'studies',\n",
       "  'occurrences',\n",
       "  ['In contrast the number of their occurrences falls down rapidly, preventing accurate statistical analysis.',\n",
       "   'We chose tetra-nucleotides for our study because they provide 256 vector components (additional file 2) and account for correlation of sub-words up to the order three.']),\n",
       " ('tetra-nucleotide',\n",
       "  'subclass of',\n",
       "  'vector components',\n",
       "  ['In contrast the number of their occurrences falls down rapidly, preventing accurate statistical analysis.',\n",
       "   'We chose tetra-nucleotides for our study because they provide 256 vector components (additional file 2) and account for correlation of sub-words up to the order three.']),\n",
       " ('bias of word with sizes less than n',\n",
       "  'subclass of',\n",
       "  'bias',\n",
       "  ['For the second criteria, the bias in RNA word usage was examined.(?<=[.?!])\\\\s+The bias in word usage (of size n) is influenced by the bias of word with sizes less than n [26] .',\n",
       "   'Therefore, in order to evaluate the true bias of word size m, it is required to compare the frequencies of word usage in the original sequence to that of model chromosomes that take into account the biases of word size m -1, m -2 ...']),\n",
       " ('word size m -1',\n",
       "  'instance of',\n",
       "  'model chromosome',\n",
       "  ['For the second criteria, the bias in RNA word usage was examined.(?<=[.?!])\\\\s+The bias in word usage (of size n) is influenced by the bias of word with sizes less than n [26] .',\n",
       "   'Therefore, in order to evaluate the true bias of word size m, it is required to compare the frequencies of word usage in the original sequence to that of model chromosomes that take into account the biases of word size m -1, m -2 ...']),\n",
       " ('Markov model',\n",
       "  'facet of',\n",
       "  'chromosomes',\n",
       "  ['1.(?<=[.?!])\\\\s+These model chromosomes were generated by obeying the Markov model of the order (m -1) th .',\n",
       "   'This can be achieved by shuffling m -1 viral nucleotides as one whole unit so that the nucleotide successions up to order (m -1) th were being preserved.']),\n",
       " ('viral',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['1.(?<=[.?!])\\\\s+These model chromosomes were generated by obeying the Markov model of the order (m -1) th .',\n",
       "   'This can be achieved by shuffling m -1 viral nucleotides as one whole unit so that the nucleotide successions up to order (m -1) th were being preserved.']),\n",
       " ('quantifying',\n",
       "  'part of',\n",
       "  'statistical',\n",
       "  ['Several statistical approaches have been proposed for quantifying word biases [27, 28] .',\n",
       "   'In this study, we employed the z statistics (Equation 1) for dinucleotide and tetra-nucleotide biases [27, 28] .']),\n",
       " ('tetra-nucleotide bias',\n",
       "  'facet of',\n",
       "  'z statistics',\n",
       "  ['Several statistical approaches have been proposed for quantifying word biases [27, 28] .',\n",
       "   'In this study, we employed the z statistics (Equation 1) for dinucleotide and tetra-nucleotide biases [27, 28] .']),\n",
       " ('bias',\n",
       "  'subclass of',\n",
       "  'measure',\n",
       "  ['The z value is a measure of the bias of a word, with values close to zero meaning no bias, negative values meaning under-representation and positive values meaning over-representation of the word w in the RNA text.',\n",
       "   'where w is a word of size m; N(w) is observed count in actual viral RNA; E(w) and Var(w) are expected count and variance for w derived from the 100 artificial chromosomes that preserved the nucleotide succession up to order m -1.']),\n",
       " ('RNA',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['The z value is a measure of the bias of a word, with values close to zero meaning no bias, negative values meaning under-representation and positive values meaning over-representation of the word w in the RNA text.',\n",
       "   'where w is a word of size m; N(w) is observed count in actual viral RNA; E(w) and Var(w) are expected count and variance for w derived from the 100 artificial chromosomes that preserved the nucleotide succession up to order m -1.']),\n",
       " ('Flowchart for studying the tetra-nucleotide usage pattern',\n",
       "  'instance of',\n",
       "  'Flowchart',\n",
       "  ['Flowchart for studying the tetra-nucleotide usage pattern Figure 7 Flowchart for studying the tetra-nucleotide usage pattern.',\n",
       "   'The FA and NJ algorithms stand for factor analysis [21] [22] [23] and neighbor joining [29] algorithm.']),\n",
       " ('Flowchart for studying the tetra-nucleotide usage pattern',\n",
       "  'instance of',\n",
       "  'Flowchart',\n",
       "  ['Flowchart for studying the tetra-nucleotide usage pattern Figure 7 Flowchart for studying the tetra-nucleotide usage pattern.',\n",
       "   'The FA and NJ algorithms stand for factor analysis [21] [22] [23] and neighbor joining [29] algorithm.']),\n",
       " ('neighbor joining',\n",
       "  'instance of',\n",
       "  'algorithm',\n",
       "  ['Flowchart for studying the tetra-nucleotide usage pattern Figure 7 Flowchart for studying the tetra-nucleotide usage pattern.',\n",
       "   'The FA and NJ algorithms stand for factor analysis [21] [22] [23] and neighbor joining [29] algorithm.']),\n",
       " ('r',\n",
       "  'instance of',\n",
       "  'correlation coefficient',\n",
       "  ['A scale-invariant parameter, the correlation coefficient r, was employed to compare between word usage patterns of viruses.',\n",
       "   'The correlation coefficient r measures the degree of linear relationship between two vectors.']),\n",
       " ('linear relationship',\n",
       "  'has part',\n",
       "  'vector',\n",
       "  ['A scale-invariant parameter, the correlation coefficient r, was employed to compare between word usage patterns of viruses.',\n",
       "   'The correlation coefficient r measures the degree of linear relationship between two vectors.']),\n",
       " ('tetra-nucleotide',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['Here, the two vectors are the tetra-nucleotide word usage pattern V corresponding to each viral genome.',\n",
       "   'The magnitude of r would indicate how much of the change of pattern in the tetra-nucleotide word usage in one virus is explained by the change in another.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['Here, the two vectors are the tetra-nucleotide word usage pattern V corresponding to each viral genome.',\n",
       "   'The magnitude of r would indicate how much of the change of pattern in the tetra-nucleotide word usage in one virus is explained by the change in another.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['Here, the two vectors are the tetra-nucleotide word usage pattern V corresponding to each viral genome.',\n",
       "   'The magnitude of r would indicate how much of the change of pattern in the tetra-nucleotide word usage in one virus is explained by the change in another.']),\n",
       " ('perfect linearity',\n",
       "  'subclass of',\n",
       "  'magnitude',\n",
       "  ['The magnitude of r is always between -1 and +1 and the relationship between the two variables will approach perfect linearity as the magnitude of correlation coefficient approaches to extreme values (+/ -1).',\n",
       "   'However, perfect positive correlation (r = 1) does not mean identity of the paired V i , but, rather, identity up to positive linearity, that is, identity between the paired standardized values.']),\n",
       " ('positive linearity',\n",
       "  'subclass of',\n",
       "  'identity',\n",
       "  ['The magnitude of r is always between -1 and +1 and the relationship between the two variables will approach perfect linearity as the magnitude of correlation coefficient approaches to extreme values (+/ -1).',\n",
       "   'However, perfect positive correlation (r = 1) does not mean identity of the paired V i , but, rather, identity up to positive linearity, that is, identity between the paired standardized values.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'genomic',\n",
       "  ['This is a crucial property of r (scaleinvariant) that enables the comparison of viral genome despite their differences in genomic sizes.',\n",
       "   'Positive magnitude of r indicates positive association whereas negative magnitude of r indicates negative association between two usage patterns.']),\n",
       " ('Positive magnitude of r',\n",
       "  'opposite of',\n",
       "  'negative magnitude of r',\n",
       "  ['This is a crucial property of r (scaleinvariant) that enables the comparison of viral genome despite their differences in genomic sizes.',\n",
       "   'Positive magnitude of r indicates positive association whereas negative magnitude of r indicates negative association between two usage patterns.']),\n",
       " ('negative magnitude of r',\n",
       "  'opposite of',\n",
       "  'Positive magnitude of r',\n",
       "  ['This is a crucial property of r (scaleinvariant) that enables the comparison of viral genome despite their differences in genomic sizes.',\n",
       "   'Positive magnitude of r indicates positive association whereas negative magnitude of r indicates negative association between two usage patterns.']),\n",
       " ('standard deviation',\n",
       "  'opposite of',\n",
       "  'mean',\n",
       "  ['For this study, correlation coefficient, r, for let say virus 1 and virus 2, is defined as follow: where V 1 , V 2 are vector representing the tetra-nucleotide usage pattern; S V1 and S V2 standard deviation of V 1 , V 2 ; are the mean of V 1 , V 2 .',\n",
       "   'Then, the distance between the tetra-nucleotide usage patterns of two viruses is defined as follows: Distance D ij = 1 -r ij ; (3) where D ij is the distance between the tetra-nucleotide usage patterns of virus i and virus j; r ij is the correlation coefficient between the tetra-nucleotide usage patterns of virus i and virus j Prior to the construction of a relationship tree, the pairwise distance matrix M of size 31 by 31 was constructed (see additional file 3).']),\n",
       " ('virus i',\n",
       "  'followed by',\n",
       "  'virus j',\n",
       "  ['For this study, correlation coefficient, r, for let say virus 1 and virus 2, is defined as follow: where V 1 , V 2 are vector representing the tetra-nucleotide usage pattern; S V1 and S V2 standard deviation of V 1 , V 2 ; are the mean of V 1 , V 2 .',\n",
       "   'Then, the distance between the tetra-nucleotide usage patterns of two viruses is defined as follows: Distance D ij = 1 -r ij ; (3) where D ij is the distance between the tetra-nucleotide usage patterns of virus i and virus j; r ij is the correlation coefficient between the tetra-nucleotide usage patterns of virus i and virus j Prior to the construction of a relationship tree, the pairwise distance matrix M of size 31 by 31 was constructed (see additional file 3).']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'viral',\n",
       "  ['Pair-wise distance between two viral genomes is measured by the value of (1 -r).',\n",
       "   'Each row/column corresponds to a specific virus and an entry at the intersection of row X and column Y corresponds to the distance between virus X and virus Y.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'virus Y',\n",
       "  ['Pair-wise distance between two viral genomes is measured by the value of (1 -r).',\n",
       "   'Each row/column corresponds to a specific virus and an entry at the intersection of row X and column Y corresponds to the distance between virus X and virus Y.']),\n",
       " ('triangular matrix',\n",
       "  'subclass of',\n",
       "  'matrix',\n",
       "  ['Such matrix has a diagonal entry of value 0.(?<=[.?!])\\\\s+For the purpose of constructing a relationship tree, only the lower/upper triangular matrix of M is required.',\n",
       "   'After obtaining lower/upper trian-gular matrix of M, the neighbor-joining method (NJ) algorithm was used to construct the relationship tree (Figure 1) .']),\n",
       " ('neighbor-joining method',\n",
       "  'instance of',\n",
       "  'algorithm',\n",
       "  ['Such matrix has a diagonal entry of value 0.(?<=[.?!])\\\\s+For the purpose of constructing a relationship tree, only the lower/upper triangular matrix of M is required.',\n",
       "   'After obtaining lower/upper trian-gular matrix of M, the neighbor-joining method (NJ) algorithm was used to construct the relationship tree (Figure 1) .']),\n",
       " ('NJ algorithm',\n",
       "  'based on',\n",
       "  'minimum-distance principle',\n",
       "  ['The neighbor-joining method is based on minimum-distance principle.(?<=[.?!])\\\\s+Details of the NJ algorithm are available in [29] .',\n",
       "   'The factor analysis is a statistical method that reveals simpler patterns within a complex set of tetra-nucleotide usage patterns V (additional file 2).']),\n",
       " ('statistical',\n",
       "  'subclass of',\n",
       "  'method',\n",
       "  ['The neighbor-joining method is based on minimum-distance principle.(?<=[.?!])\\\\s+Details of the NJ algorithm are available in [29] .',\n",
       "   'The factor analysis is a statistical method that reveals simpler patterns within a complex set of tetra-nucleotide usage patterns V (additional file 2).']),\n",
       " ('factors',\n",
       "  'subclass of',\n",
       "  'eigen-vectors',\n",
       "  ['It seeks to discover if the observed usage patterns can be explained in terms of a much smaller number of un-correlated pattern sets called factors (eigen-vectors).',\n",
       "   'Suppose we take a simple case where there are 31 viruses each represented by two components (x,y) in vector V (x,y represent the frequencies of occurrence for two specific tetra-nucleotides).']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'tetra-nucleotide',\n",
       "  ['It seeks to discover if the observed usage patterns can be explained in terms of a much smaller number of un-correlated pattern sets called factors (eigen-vectors).',\n",
       "   'Suppose we take a simple case where there are 31 viruses each represented by two components (x,y) in vector V (x,y represent the frequencies of occurrence for two specific tetra-nucleotides).']),\n",
       " ('scatter-plot',\n",
       "  'has part',\n",
       "  'X-axis',\n",
       "  ['Then, in a scatter-plot we can think of the regression line as the original X-axis, rotated so that it approximates the regression line.',\n",
       "   'This type of rotation maximize the variance of the variables (x,y) on the eigen-vector.']),\n",
       " ('X-axis',\n",
       "  'part of',\n",
       "  'scatter-plot',\n",
       "  ['Then, in a scatter-plot we can think of the regression line as the original X-axis, rotated so that it approximates the regression line.',\n",
       "   'This type of rotation maximize the variance of the variables (x,y) on the eigen-vector.']),\n",
       " ('eigen-vector',\n",
       "  'subclass of',\n",
       "  'variables',\n",
       "  ['Then, in a scatter-plot we can think of the regression line as the original X-axis, rotated so that it approximates the regression line.',\n",
       "   'This type of rotation maximize the variance of the variables (x,y) on the eigen-vector.']),\n",
       " ('first eigen-vector',\n",
       "  'followed by',\n",
       "  'subsequent eigen-vectors',\n",
       "  ['The remaining variability around this the first eigen-vector was captured in the subsequent eigen-vectors.',\n",
       "   'In this manner, consecutive eigen-vectors are extracted but with a diminishing importance.']),\n",
       " ('subsequent eigen-vectors',\n",
       "  'follows',\n",
       "  'first eigen-vector',\n",
       "  ['The remaining variability around this the first eigen-vector was captured in the subsequent eigen-vectors.',\n",
       "   'In this manner, consecutive eigen-vectors are extracted but with a diminishing importance.']),\n",
       " ('eigen-vectors',\n",
       "  'subclass of',\n",
       "  'extracted',\n",
       "  ['The remaining variability around this the first eigen-vector was captured in the subsequent eigen-vectors.',\n",
       "   'In this manner, consecutive eigen-vectors are extracted but with a diminishing importance.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['What each eigen-vector represents is the embedded RNA word usage patterns communalised by a group of viruses presumably under the same selection pressures.',\n",
       "   'We implemented the factor analysis algorithm [21] [22] [23] in Matlab Technical Programming Language and computed a set of eigen-vectors.']),\n",
       " ('Matlab',\n",
       "  'instance of',\n",
       "  'Technical Programming Language',\n",
       "  ['What each eigen-vector represents is the embedded RNA word usage patterns communalised by a group of viruses presumably under the same selection pressures.',\n",
       "   'We implemented the factor analysis algorithm [21] [22] [23] in Matlab Technical Programming Language and computed a set of eigen-vectors.']),\n",
       " ('vector',\n",
       "  'subclass of',\n",
       "  'eigen-vectors',\n",
       "  ['Then, the original usage pattern V was re-mapped for each virus onto the new coordinate system based on these derived eigen-vectors.',\n",
       "   'The difference between approach two and approach one is discussed in the results and discussion section.(?<=[.?!])\\\\s+YLY participated in the design and performed the statistical analysis.']),\n",
       " ('design',\n",
       "  'facet of',\n",
       "  'statistical analysis',\n",
       "  ['Then, the original usage pattern V was re-mapped for each virus onto the new coordinate system based on these derived eigen-vectors.',\n",
       "   'The difference between approach two and approach one is discussed in the results and discussion section.(?<=[.?!])\\\\s+YLY participated in the design and performed the statistical analysis.']),\n",
       " ('AD',\n",
       "  'has part',\n",
       "  'XWZ',\n",
       "  ['AD participated in the design and overall coordination of this study.(?<=[.?!])\\\\s+XWZ participated in the design of the study.(?<=[.?!])\\\\s+All authors read and approved the final manuscript.',\n",
       "   'Because of the constant threat posed by emerging infectious diseases and the limitations of existing approaches used to identify new pathogens, there is a great demand for new technological methods for viral discovery.']),\n",
       " ('XWZ',\n",
       "  'part of',\n",
       "  'AD',\n",
       "  ['AD participated in the design and overall coordination of this study.(?<=[.?!])\\\\s+XWZ participated in the design of the study.(?<=[.?!])\\\\s+All authors read and approved the final manuscript.',\n",
       "   'Because of the constant threat posed by emerging infectious diseases and the limitations of existing approaches used to identify new pathogens, there is a great demand for new technological methods for viral discovery.']),\n",
       " ('infectious diseases',\n",
       "  'has cause',\n",
       "  'pathogens',\n",
       "  ['AD participated in the design and overall coordination of this study.(?<=[.?!])\\\\s+XWZ participated in the design of the study.(?<=[.?!])\\\\s+All authors read and approved the final manuscript.',\n",
       "   'Because of the constant threat posed by emerging infectious diseases and the limitations of existing approaches used to identify new pathogens, there is a great demand for new technological methods for viral discovery.']),\n",
       " ('pathogens',\n",
       "  'has effect',\n",
       "  'infectious diseases',\n",
       "  ['AD participated in the design and overall coordination of this study.(?<=[.?!])\\\\s+XWZ participated in the design of the study.(?<=[.?!])\\\\s+All authors read and approved the final manuscript.',\n",
       "   'Because of the constant threat posed by emerging infectious diseases and the limitations of existing approaches used to identify new pathogens, there is a great demand for new technological methods for viral discovery.']),\n",
       " ('microarray',\n",
       "  'uses',\n",
       "  'DNA',\n",
       "  ['We describe herein a DNA microarray-based platform for novel virus identification and characterization.',\n",
       "   'Central to this approach was a DNA microarray designed to detect a wide range of known viruses as well as novel members of existing viral families; this microarray contained the most highly conserved 70mer sequences from every fully sequenced reference viral genome in GenBank.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['We describe herein a DNA microarray-based platform for novel virus identification and characterization.',\n",
       "   'Central to this approach was a DNA microarray designed to detect a wide range of known viruses as well as novel members of existing viral families; this microarray contained the most highly conserved 70mer sequences from every fully sequenced reference viral genome in GenBank.']),\n",
       " ('severe acute respiratory syndrome',\n",
       "  'has cause',\n",
       "  'coronavirus',\n",
       "  ['During an outbreak of severe acute respiratory syndrome (SARS) in March 2003, hybridization to this microarray revealed the presence of a previously uncharacterized coronavirus in a viral isolate cultivated from a SARS patient.',\n",
       "   'To further characterize this new virus, approximately 1 kb of the unknown virus genome was cloned by physically recovering viral sequences hybridized to individual array elements.']),\n",
       " ('genome',\n",
       "  'part of',\n",
       "  'virus',\n",
       "  ['During an outbreak of severe acute respiratory syndrome (SARS) in March 2003, hybridization to this microarray revealed the presence of a previously uncharacterized coronavirus in a viral isolate cultivated from a SARS patient.',\n",
       "   'To further characterize this new virus, approximately 1 kb of the unknown virus genome was cloned by physically recovering viral sequences hybridized to individual array elements.']),\n",
       " ('coronavirus',\n",
       "  'parent taxon',\n",
       "  'virus',\n",
       "  ['Sequencing of these fragments confirmed that the virus was indeed a new member of the coronavirus family.',\n",
       "   'This combination of array hybridization followed by direct viral sequence recovery should prove to be a general strategy for the rapid identification and characterization of novel viruses and emerging infectious disease.']),\n",
       " ('emerging infectious disease',\n",
       "  'subclass of',\n",
       "  'novel viruses',\n",
       "  ['Sequencing of these fragments confirmed that the virus was indeed a new member of the coronavirus family.',\n",
       "   'This combination of array hybridization followed by direct viral sequence recovery should prove to be a general strategy for the rapid identification and characterization of novel viruses and emerging infectious disease.']),\n",
       " ('pathogens',\n",
       "  'has effect',\n",
       "  'disease',\n",
       "  ['Over the past two decades, technological advances in molecular biology have fuelled progress in the discovery of new pathogens associated with human diseases.',\n",
       "   \"The identification of novel viruses such as hepatitis C virus (Choo et al.(?<=[.?!])\\\\s+1989 ), sin nombre virus (Nichol et al.(?<=[.?!])\\\\s+1993) , and Kaposi's sarcoma herpesvirus (Chang et al.\"]),\n",
       " ('disease',\n",
       "  'has cause',\n",
       "  'pathogens',\n",
       "  ['Over the past two decades, technological advances in molecular biology have fuelled progress in the discovery of new pathogens associated with human diseases.',\n",
       "   \"The identification of novel viruses such as hepatitis C virus (Choo et al.(?<=[.?!])\\\\s+1989 ), sin nombre virus (Nichol et al.(?<=[.?!])\\\\s+1993) , and Kaposi's sarcoma herpesvirus (Chang et al.\"]),\n",
       " ('hepatitis C virus',\n",
       "  'instance of',\n",
       "  'novel viruses',\n",
       "  ['Over the past two decades, technological advances in molecular biology have fuelled progress in the discovery of new pathogens associated with human diseases.',\n",
       "   \"The identification of novel viruses such as hepatitis C virus (Choo et al.(?<=[.?!])\\\\s+1989 ), sin nombre virus (Nichol et al.(?<=[.?!])\\\\s+1993) , and Kaposi's sarcoma herpesvirus (Chang et al.\"]),\n",
       " ('immunoscreening',\n",
       "  'subclass of',\n",
       "  'molecular methods',\n",
       "  ['1994 ) has relied upon a diverse range of modern molecular methods such as immunoscreening of cDNA libraries, degenerate PCR, and representational difference analysis, respectively.',\n",
       "   'In spite of these successes, there remain numerous syndromes with suspected infectious etiologies that continue to escape identification efforts, in part due to limitations of existing methodologies for viral discovery (Muerhoff et al.']),\n",
       " ('infectious etiologies',\n",
       "  'subclass of',\n",
       "  'syndromes',\n",
       "  ['1994 ) has relied upon a diverse range of modern molecular methods such as immunoscreening of cDNA libraries, degenerate PCR, and representational difference analysis, respectively.',\n",
       "   'In spite of these successes, there remain numerous syndromes with suspected infectious etiologies that continue to escape identification efforts, in part due to limitations of existing methodologies for viral discovery (Muerhoff et al.']),\n",
       " ('Kellam',\n",
       "  'publication date',\n",
       "  '1998',\n",
       "  ['1997; Kellam 1998) .',\n",
       "   'These limitations, coupled with the constant threat posed by newly emerging infectious diseases of unknown origin, necessitate that new approaches be developed to augment the repertoire of available tools for pathogen discovery.']),\n",
       " ('pathogen discovery',\n",
       "  'facet of',\n",
       "  'infectious diseases',\n",
       "  ['1997; Kellam 1998) .',\n",
       "   'These limitations, coupled with the constant threat posed by newly emerging infectious diseases of unknown origin, necessitate that new approaches be developed to augment the repertoire of available tools for pathogen discovery.']),\n",
       " ('DNA microarray',\n",
       "  'use',\n",
       "  'viral detection',\n",
       "  ['We have previously described a prototype DNA microarray designed for highly parallel viral detection with the potential to detect novel members of known viral families (Wang et al.(?<=[.?!])\\\\s+2002) .',\n",
       "   '2002) .(?<=[.?!])\\\\s+This microarray contained approximately 1600 oligonucleotides representing 140 viruses.']),\n",
       " ('viruses',\n",
       "  'has part',\n",
       "  'oligonucleotide',\n",
       "  ['We have previously described a prototype DNA microarray designed for highly parallel viral detection with the potential to detect novel members of known viral families (Wang et al.(?<=[.?!])\\\\s+2002) .',\n",
       "   '2002) .(?<=[.?!])\\\\s+This microarray contained approximately 1600 oligonucleotides representing 140 viruses.']),\n",
       " ('DNA',\n",
       "  'has part',\n",
       "  'oligonucleotide',\n",
       "  ['Building upon this foundation, a more comprehensive second-generation DNA microarray consisting of 70mer oligonucleotides derived from every fully sequenced reference viral genome in GenBank (as of August 15, 2002) was constructed.',\n",
       "   'The most highly conserved 70mers from each virus were selected as described by Wang et al.']),\n",
       " ('70mers',\n",
       "  'subclass of',\n",
       "  'virus',\n",
       "  ['Building upon this foundation, a more comprehensive second-generation DNA microarray consisting of 70mer oligonucleotides derived from every fully sequenced reference viral genome in GenBank (as of August 15, 2002) was constructed.',\n",
       "   'The most highly conserved 70mers from each virus were selected as described by Wang et al.']),\n",
       " ('unsequenced',\n",
       "  'subclass of',\n",
       "  'unknown',\n",
       "  ['(2002) to maximize the probability of detecting unknown and unsequenced members of existing families by cross-hybridization to these array elements.',\n",
       "   'On average, ten 70mers were selected for each virus, totaling approximately 10,000 oligonucleotides from approximately 1,000 viruses.']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'oligonucleotide',\n",
       "  ['(2002) to maximize the probability of detecting unknown and unsequenced members of existing families by cross-hybridization to these array elements.',\n",
       "   'On average, ten 70mers were selected for each virus, totaling approximately 10,000 oligonucleotides from approximately 1,000 viruses.']),\n",
       " ('known',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['The objective was to create a microarray with the capability of detecting the widest possible range of both known and unknown viruses.',\n",
       "   'This pan-viral microarray was used as part of the global effort to identify a novel virus associated with severe acute respiratory syndrome (SARS) in March 2003, as reported by Ksiazek et al.']),\n",
       " ('severe acute respiratory syndrome',\n",
       "  'has cause',\n",
       "  'novel virus',\n",
       "  ['The objective was to create a microarray with the capability of detecting the widest possible range of both known and unknown viruses.',\n",
       "   'This pan-viral microarray was used as part of the global effort to identify a novel virus associated with severe acute respiratory syndrome (SARS) in March 2003, as reported by Ksiazek et al.']),\n",
       " ('SARS outbreak',\n",
       "  'has cause',\n",
       "  'virus',\n",
       "  ['(2003) .(?<=[.?!])\\\\s+We describe here the experimental details of the microarray methodology for novel virus identification, using the SARS outbreak as an example.',\n",
       "   'During the initial phase of research into the etiology of SARS, an unknown virus was cultured in Vero cells from a patient suffering from SARS (Ksiazek et al.(?<=[.?!])\\\\s+2003) .']),\n",
       " ('SARS',\n",
       "  'has cause',\n",
       "  'unknown virus',\n",
       "  ['(2003) .(?<=[.?!])\\\\s+We describe here the experimental details of the microarray methodology for novel virus identification, using the SARS outbreak as an example.',\n",
       "   'During the initial phase of research into the etiology of SARS, an unknown virus was cultured in Vero cells from a patient suffering from SARS (Ksiazek et al.(?<=[.?!])\\\\s+2003) .']),\n",
       " ('Centers for Disease Control and Prevention',\n",
       "  'product or material produced',\n",
       "  'nucleic acid',\n",
       "  ['2003) .(?<=[.?!])\\\\s+Total nucleic acid purified from this viral culture, as well as a control culture, was obtained from the Centers for Disease Control and Prevention on March 22, 2003.',\n",
       "   'These two samples, along with additional controls (HeLa cell RNA and water alone), were amplified and hybridized within 24 h to the virus DNA microarray.']),\n",
       " ('DNA',\n",
       "  'has part',\n",
       "  'RNA',\n",
       "  ['2003) .(?<=[.?!])\\\\s+Total nucleic acid purified from this viral culture, as well as a control culture, was obtained from the Centers for Disease Control and Prevention on March 22, 2003.',\n",
       "   'These two samples, along with additional controls (HeLa cell RNA and water alone), were amplified and hybridized within 24 h to the virus DNA microarray.']),\n",
       " ('coronaviridae',\n",
       "  'parent taxon',\n",
       "  'astroviridae',\n",
       "  ['The strongest hybridizing array elements from the infected culture were derived from two families: astroviridae and coronaviridae.',\n",
       "   'Table 1 lists the oligonucleotides from these families with the greatest hybridization intensity.']),\n",
       " ('oligonucleotide',\n",
       "  'instance of',\n",
       "  'families',\n",
       "  ['The strongest hybridizing array elements from the infected culture were derived from two families: astroviridae and coronaviridae.',\n",
       "   'Table 1 lists the oligonucleotides from these families with the greatest hybridization intensity.']),\n",
       " ('oligonucleotide',\n",
       "  'subclass of',\n",
       "  'hybridization',\n",
       "  ['By comparison, these oligonucleotides yielded essentially background levels of hybridization in the various control arrays performed in parallel.',\n",
       "   'The initial suggestion from this hybridization pattern was that members of both of these viral families might be present.']),\n",
       " ('hybridization',\n",
       "  'facet of',\n",
       "  'viral',\n",
       "  ['By comparison, these oligonucleotides yielded essentially background levels of hybridization in the various control arrays performed in parallel.',\n",
       "   'The initial suggestion from this hybridization pattern was that members of both of these viral families might be present.']),\n",
       " ('nucleotide',\n",
       "  'part of',\n",
       "  'oligonucleotide',\n",
       "  ['However, alignment of the oligonucleotides using ClustalX revealed that all four hybridizing oligonucleotides from the astroviridae and one oligonucleotide from avian infectious bronchitis virus (IBV) (GenBank NC_001451), an avian coronavirus, shared a core consensus motif spanning 33 nucleotides (data not shown); thus, these five oligonucleotides behaved essentially as multiple redundant probes for the same sequence.',\n",
       "   'This motif is known to be present in the 39 UTR of all astroviruses and the avian coronaviruses (Jonassen et al.']),\n",
       " ('coronavirus',\n",
       "  'parent taxon',\n",
       "  'astrovirus',\n",
       "  ['However, alignment of the oligonucleotides using ClustalX revealed that all four hybridizing oligonucleotides from the astroviridae and one oligonucleotide from avian infectious bronchitis virus (IBV) (GenBank NC_001451), an avian coronavirus, shared a core consensus motif spanning 33 nucleotides (data not shown); thus, these five oligonucleotides behaved essentially as multiple redundant probes for the same sequence.',\n",
       "   'This motif is known to be present in the 39 UTR of all astroviruses and the avian coronaviruses (Jonassen et al.']),\n",
       " ('transmissible gastroenteritis virus',\n",
       "  'parent taxon',\n",
       "  'coronavirus',\n",
       "  ['1998 ), but appears to be absent in the available sequenced mammalian coronaviruses (bovine coronavirus, murine hepatitis virus [MHV], human coronavirus 229E, porcine epidemic diarrhea virus, and transmissible gastroenteritis virus).',\n",
       "   'The other three hybridizing oligonucleotides were derived from three conserved regions within the ORF1AB polyprotein common to all coronaviruses ( Figure 1 ).']),\n",
       " ('ORF1AB',\n",
       "  'instance of',\n",
       "  'polyprotein',\n",
       "  ['1998 ), but appears to be absent in the available sequenced mammalian coronaviruses (bovine coronavirus, murine hepatitis virus [MHV], human coronavirus 229E, porcine epidemic diarrhea virus, and transmissible gastroenteritis virus).',\n",
       "   'The other three hybridizing oligonucleotides were derived from three conserved regions within the ORF1AB polyprotein common to all coronaviruses ( Figure 1 ).']),\n",
       " ('coronavirus',\n",
       "  'parent taxon',\n",
       "  'virus',\n",
       "  ['Based on the aggregate hybridization pattern, the virus appeared to be a novel member of the coronavirus family.',\n",
       "   'To further characterize this virus, we sequenced fragments of the viral genome using two complementary approaches.']),\n",
       " ('genome',\n",
       "  'subclass of',\n",
       "  'sequenced',\n",
       "  ['Based on the aggregate hybridization pattern, the virus appeared to be a novel member of the coronavirus family.',\n",
       "   'To further characterize this virus, we sequenced fragments of the viral genome using two complementary approaches.']),\n",
       " ('NSP11',\n",
       "  'instance of',\n",
       "  'gene',\n",
       "  ['First, BLAST alignment of two of the hybridizing viral oligonucleotides, one each from bovine coronavirus and human coronavirus 229E, to the IBV genome indicated that the oligonucleotides possessed homology to distinct con- served regions within the NSP11 gene (BLAST identity matches of 42/47 and 26/27, respectively).',\n",
       "   'A pair of PCR primers was designed to amplify the intervening sequences between the two conserved regions, and a fragment that possessed 89% identity over 37 amino acids to MHV, a murine coronavirus, was obtained ( Figure 1 ; sequence available as Data S1, found at http://dx.doi.org/10.1371/ journal.pbio.0000002.sd001.).']),\n",
       " ('MHV',\n",
       "  'parent taxon',\n",
       "  'coronavirus',\n",
       "  ['First, BLAST alignment of two of the hybridizing viral oligonucleotides, one each from bovine coronavirus and human coronavirus 229E, to the IBV genome indicated that the oligonucleotides possessed homology to distinct con- served regions within the NSP11 gene (BLAST identity matches of 42/47 and 26/27, respectively).',\n",
       "   'A pair of PCR primers was designed to amplify the intervening sequences between the two conserved regions, and a fragment that possessed 89% identity over 37 amino acids to MHV, a murine coronavirus, was obtained ( Figure 1 ; sequence available as Data S1, found at http://dx.doi.org/10.1371/ journal.pbio.0000002.sd001.).']),\n",
       " ('hybridized',\n",
       "  'subclass of',\n",
       "  'viral',\n",
       "  ['In a parallel approach, we directly recovered hybridized viral sequences from the surface of the microarray.',\n",
       "   'This procedure took advantage of the physical separation achieved during microarray hybridization, which effectively purified the viral nucleic acid from other nucleic acid species present in the sample.']),\n",
       " ('viral nucleic acid',\n",
       "  'subclass of',\n",
       "  'nucleic acid',\n",
       "  ['In a parallel approach, we directly recovered hybridized viral sequences from the surface of the microarray.',\n",
       "   'This procedure took advantage of the physical separation achieved during microarray hybridization, which effectively purified the viral nucleic acid from other nucleic acid species present in the sample.']),\n",
       " ('DNA',\n",
       "  'subclass of',\n",
       "  'nucleic acid',\n",
       "  ['Using a tungsten needle, the DNA microarray spot corresponding to the conserved 39 UTR motif was repeatedly scraped and the hybridized nucleic acid was recovered.',\n",
       "   'This material was subsequently amplified, cloned, and sequenced ( Figure 2 ).']),\n",
       " ('cloned',\n",
       "  'followed by',\n",
       "  'sequenced',\n",
       "  ['Using a tungsten needle, the DNA microarray spot corresponding to the conserved 39 UTR motif was repeatedly scraped and the hybridized nucleic acid was recovered.',\n",
       "   'This material was subsequently amplified, cloned, and sequenced ( Figure 2 ).']),\n",
       " ('UTR',\n",
       "  'subclass of',\n",
       "  'coding region',\n",
       "  ['The largest clone spanned almost 1.1 kb; this fragment encompassed the 39 UTR conserved motif and extended into the most 39 coding region of the viral genome.',\n",
       "   'BLAST analysis revealed 33% identity over 157 amino acids to MHV nucleocapsid, thus confirming the presence of a novel coronavirus (see Figure 1 ; see Data S1).']),\n",
       " ('nucleocapsid',\n",
       "  'has part',\n",
       "  'amino acid',\n",
       "  ['The largest clone spanned almost 1.1 kb; this fragment encompassed the 39 UTR conserved motif and extended into the most 39 coding region of the viral genome.',\n",
       "   'BLAST analysis revealed 33% identity over 157 amino acids to MHV nucleocapsid, thus confirming the presence of a novel coronavirus (see Figure 1 ; see Data S1).']),\n",
       " ('shotgun sequencing',\n",
       "  'subclass of',\n",
       "  'RT-PCR',\n",
       "  ['We subsequently confirmed results obtained from both strategies described above by using a random-primed RT-PCR shotgun sequencing approach that generated contigs totaling approximately 25 kb of viral genome sequence (see Data S1).',\n",
       "   'In this report, we have demonstrated the viability of detecting novel pathogens via cross-hybridization to highly conserved sequence motifs.']),\n",
       " ('cross-hybridization',\n",
       "  'subclass of',\n",
       "  'pathogens',\n",
       "  ['We subsequently confirmed results obtained from both strategies described above by using a random-primed RT-PCR shotgun sequencing approach that generated contigs totaling approximately 25 kb of viral genome sequence (see Data S1).',\n",
       "   'In this report, we have demonstrated the viability of detecting novel pathogens via cross-hybridization to highly conserved sequence motifs.']),\n",
       " ('genome',\n",
       "  'subclass of',\n",
       "  'sequencing',\n",
       "  ['With the recent sequencing of the complete SARS coronavirus genome (GenBank NC_004718) (Marra et al.(?<=[.?!])\\\\s+2003; Rota et al.',\n",
       "   '2003) , we were able to retrospectively determine the degree of nucleotide identity shared between the hybridizing oligonucleotides and the new coronavirus genome (see Table 1 ).']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['With the recent sequencing of the complete SARS coronavirus genome (GenBank NC_004718) (Marra et al.(?<=[.?!])\\\\s+2003; Rota et al.',\n",
       "   '2003) , we were able to retrospectively determine the degree of nucleotide identity shared between the hybridizing oligonucleotides and the new coronavirus genome (see Table 1 ).']),\n",
       " ('virus',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['Stretches of relatively uninterrupted nucleotide identity as short as 25 nucleotides yielded clearly detectable hybridization signal, confirming that novel viruses with only limited homology to known viruses can be successfully detected by this strategy.',\n",
       "   'A key feature of this approach is that direct recovery of hybridized material from the microarray provides a rapid route for obtaining sequences of novel viruses.']),\n",
       " ('hybridized',\n",
       "  'subclass of',\n",
       "  'viruses',\n",
       "  ['Stretches of relatively uninterrupted nucleotide identity as short as 25 nucleotides yielded clearly detectable hybridization signal, confirming that novel viruses with only limited homology to known viruses can be successfully detected by this strategy.',\n",
       "   'A key feature of this approach is that direct recovery of hybridized material from the microarray provides a rapid route for obtaining sequences of novel viruses.']),\n",
       " ('primer',\n",
       "  'part of',\n",
       "  'PCR',\n",
       "  ['By contrast, conventional strategies for subsequent sequence identifica-tion would require time-consuming steps such as library screening or additional rounds of PCR primer design and synthesis.',\n",
       "   'In the case of SARS, we were able to ascertain within 24 h that a novel coronavirus was present in the unknown sample, and partial genome sequences of this virus were obtained over the next few days without the need for specific primer design.']),\n",
       " ('SARS',\n",
       "  'parent taxon',\n",
       "  'novel coronavirus',\n",
       "  ['By contrast, conventional strategies for subsequent sequence identifica-tion would require time-consuming steps such as library screening or additional rounds of PCR primer design and synthesis.',\n",
       "   'In the case of SARS, we were able to ascertain within 24 h that a novel coronavirus was present in the unknown sample, and partial genome sequences of this virus were obtained over the next few days without the need for specific primer design.']),\n",
       " ('DNA',\n",
       "  'subclass of',\n",
       "  'nucleic acid',\n",
       "  ['To our knowledge, this is the first demonstration of the feasibility and utility of directly recovering nucleic acid sequences from a hybridized DNA microarray.',\n",
       "   'In light of the continuous threat of emerging infectious diseases, this overall approach will greatly facilitate the rapid identification and characterization of novel viruses.']),\n",
       " ('emerging infectious diseases',\n",
       "  'has cause',\n",
       "  'viruses',\n",
       "  ['To our knowledge, this is the first demonstration of the feasibility and utility of directly recovering nucleic acid sequences from a hybridized DNA microarray.',\n",
       "   'In light of the continuous threat of emerging infectious diseases, this overall approach will greatly facilitate the rapid identification and characterization of novel viruses.']),\n",
       " ('viruses',\n",
       "  'has effect',\n",
       "  'emerging infectious diseases',\n",
       "  ['To our knowledge, this is the first demonstration of the feasibility and utility of directly recovering nucleic acid sequences from a hybridized DNA microarray.',\n",
       "   'In light of the continuous threat of emerging infectious diseases, this overall approach will greatly facilitate the rapid identification and characterization of novel viruses.']),\n",
       " ('BioMerieux',\n",
       "  'headquarters location',\n",
       "  'Durham, North Carolina',\n",
       "  ['Nucleic acid isolation.(?<=[.?!])\\\\s+Total nucleic acid was purified using the automated NucliSens extraction system (BioMerieux, Durham, North Carolina).',\n",
       "   \"Following the manufacturer's instructions, 100 ll of each specimen was added to tubes containing 900 ll of prewarmed NucliSens lysis buffer and incubated at 378C for 30 min with intermittent mixing.\"]),\n",
       " ('NucliSens',\n",
       "  'instance of',\n",
       "  'lysis buffer',\n",
       "  ['Nucleic acid isolation.(?<=[.?!])\\\\s+Total nucleic acid was purified using the automated NucliSens extraction system (BioMerieux, Durham, North Carolina).',\n",
       "   \"Following the manufacturer's instructions, 100 ll of each specimen was added to tubes containing 900 ll of prewarmed NucliSens lysis buffer and incubated at 378C for 30 min with intermittent mixing.\"]),\n",
       " ('extraction kit',\n",
       "  'product or material produced',\n",
       "  'silica',\n",
       "  ['Fifty microliters of silica suspension provided in the extraction kit was added to each tube and mixed.',\n",
       "   'The mixtures were then transferred to a nucleic acid extraction cartridge and loaded onto the extractor workstation for processing.']),\n",
       " ('extractor workstation',\n",
       "  'use',\n",
       "  'nucleic acid extraction',\n",
       "  ['Fifty microliters of silica suspension provided in the extraction kit was added to each tube and mixed.',\n",
       "   'The mixtures were then transferred to a nucleic acid extraction cartridge and loaded onto the extractor workstation for processing.']),\n",
       " ('eluate',\n",
       "  'subclass of',\n",
       "  'nucleic acid',\n",
       "  ['Approximately 50 ll of total nucleic acid eluate was recovered.(?<=[.?!])\\\\s+Amplification.',\n",
       "   'Amplification.(?<=[.?!])\\\\s+For the culture supernatants, 450 ng of nucleic acid was used as input for the amplification protocol.']),\n",
       " ('supernatant',\n",
       "  'part of',\n",
       "  'culture',\n",
       "  ['Approximately 50 ll of total nucleic acid eluate was recovered.(?<=[.?!])\\\\s+Amplification.',\n",
       "   'Amplification.(?<=[.?!])\\\\s+For the culture supernatants, 450 ng of nucleic acid was used as input for the amplification protocol.']),\n",
       " ('positive amplification control',\n",
       "  'opposite of',\n",
       "  'negative control',\n",
       "  ['In parallel, 50 ng of HeLa cell RNA was used as a positive amplification control and water was used for a negative control.',\n",
       "   'Samples were amplified using a random-primer protocol as described by Wang et al.']),\n",
       " ('negative control',\n",
       "  'opposite of',\n",
       "  'positive amplification control',\n",
       "  ['In parallel, 50 ng of HeLa cell RNA was used as a positive amplification control and water was used for a negative control.',\n",
       "   'Samples were amplified using a random-primer protocol as described by Wang et al.']),\n",
       " ('random-primer protocol',\n",
       "  'subclass of',\n",
       "  'amplified',\n",
       "  ['In parallel, 50 ng of HeLa cell RNA was used as a positive amplification control and water was used for a negative control.',\n",
       "   'Samples were amplified using a random-primer protocol as described by Wang et al.']),\n",
       " ('primer-A',\n",
       "  'use',\n",
       "  'PCR amplification',\n",
       "  ['(2002) , with the following modifications: first-and second-strand synthesis were primed using primer-A (59-GTTTCCCAGTCACGATCNNNNNNN-NN) followed by PCR amplification using primer-B (59-GTTTCCC-AGTCACGATC) for 40 cycles.',\n",
       "   'Aminoallyl-dUTP was incorporated into the PCR product using an additional 20 cycles of thermocycling.']),\n",
       " ('thermocycling',\n",
       "  'subclass of',\n",
       "  'PCR',\n",
       "  ['(2002) , with the following modifications: first-and second-strand synthesis were primed using primer-A (59-GTTTCCCAGTCACGATCNNNNNNN-NN) followed by PCR amplification using primer-B (59-GTTTCCC-AGTCACGATC) for 40 cycles.',\n",
       "   'Aminoallyl-dUTP was incorporated into the PCR product using an additional 20 cycles of thermocycling.']),\n",
       " ('Protocol S1',\n",
       "  'use',\n",
       "  'Microarray hybridization and analysis',\n",
       "  ['A detailed protocol is available as Protocol S1, found at http:// dx.doi.org/10.1371/journal.pbio.0000002.sd002.(?<=[.?!])\\\\s+Microarray hybridization and analysis.',\n",
       "   'DNA microarrays were printed and hybridized essentially as described by Wang et al.']),\n",
       " ('hybridized',\n",
       "  'subclass of',\n",
       "  'DNA microarray',\n",
       "  ['A detailed protocol is available as Protocol S1, found at http:// dx.doi.org/10.1371/journal.pbio.0000002.sd002.(?<=[.?!])\\\\s+Microarray hybridization and analysis.',\n",
       "   'DNA microarrays were printed and hybridized essentially as described by Wang et al.']),\n",
       " ('70mer',\n",
       "  'subclass of',\n",
       "  'oligonucleotide',\n",
       "  ['(2002) , with the following modifications: for array printing, a single-defined 70mer (spike-70) was mixed with each viral oligonucleotide in a 1:50 ratio.',\n",
       "   'Array hybridizations used Cy5-labeled amplified probe from either virally infected cultures or controls (mock-infected culture, HeLa RNA, or water); a reference signal for every spot on each array was generated by using a Cy3-labeled version of the reverse complement of spike-70.']),\n",
       " ('spike-70',\n",
       "  'use',\n",
       "  'Array hybridization',\n",
       "  ['(2002) , with the following modifications: for array printing, a single-defined 70mer (spike-70) was mixed with each viral oligonucleotide in a 1:50 ratio.',\n",
       "   'Array hybridizations used Cy5-labeled amplified probe from either virally infected cultures or controls (mock-infected culture, HeLa RNA, or water); a reference signal for every spot on each array was generated by using a Cy3-labeled version of the reverse complement of spike-70.']),\n",
       " ('Cy5',\n",
       "  'subclass of',\n",
       "  'Oligonucleotide',\n",
       "  ['Oligonucleotides were assessed by Cy5 intensity.',\n",
       "   'Oligonucleotides from the astrovirus and coronavirus families that passed a conservative, arbitrarily set cutoff of (Cy5 infection -Cy5 mock ) .']),\n",
       " ('coronavirus',\n",
       "  'parent taxon',\n",
       "  'astrovirus',\n",
       "  ['Oligonucleotides were assessed by Cy5 intensity.',\n",
       "   'Oligonucleotides from the astrovirus and coronavirus families that passed a conservative, arbitrarily set cutoff of (Cy5 infection -Cy5 mock ) .']),\n",
       " ('1500 intensity units',\n",
       "  'point in time',\n",
       "  '1500',\n",
       "  ['1500 intensity units are listed in Table 1 .',\n",
       "   'Additional oligonucleotides from these families and their homology to the SARS coronavirus are listed in Table S1 , found at http:// dx.doi.org/10.1371/journal.pbio.0000002.st001.']),\n",
       " ('SARS coronavirus',\n",
       "  'has part',\n",
       "  'oligonucleotide',\n",
       "  ['1500 intensity units are listed in Table 1 .',\n",
       "   'Additional oligonucleotides from these families and their homology to the SARS coronavirus are listed in Table S1 , found at http:// dx.doi.org/10.1371/journal.pbio.0000002.st001.']),\n",
       " ('Gene Expression Omnibus',\n",
       "  'instance of',\n",
       "  'database',\n",
       "  ['Array data has been deposited in the Gene Expression Omnibus (GEO) database (accession number GSE546).',\n",
       "   'A complete list of the viral oligonucleotide sequences on the microarray is also available as Table S2 , found at http://dx.doi.org/10.1371/journal.pbio.0000002.st002.']),\n",
       " ('oligonucleotide',\n",
       "  'subclass of',\n",
       "  'sequences',\n",
       "  ['Array data has been deposited in the Gene Expression Omnibus (GEO) database (accession number GSE546).',\n",
       "   'A complete list of the viral oligonucleotide sequences on the microarray is also available as Table S2 , found at http://dx.doi.org/10.1371/journal.pbio.0000002.st002.']),\n",
       " ('array element',\n",
       "  'use',\n",
       "  'PCR',\n",
       "  ['Conventional PCR using array element sequences.',\n",
       "   'PCR primers were designed by aligning the hybridizing oligonucleotides (Oligo IDs 15081544_766 and 12175745_728) to the IBV genome (Fwd: 59-TGTTTTGGAATTGTAATGTGGAT; Rev: 59-TACAAACTACCTCCA-TTACAGCC) and selecting stretches of near-identity.']),\n",
       " ('genome',\n",
       "  'has part',\n",
       "  'oligonucleotide',\n",
       "  ['Conventional PCR using array element sequences.',\n",
       "   'PCR primers were designed by aligning the hybridizing oligonucleotides (Oligo IDs 15081544_766 and 12175745_728) to the IBV genome (Fwd: 59-TGTTTTGGAATTGTAATGTGGAT; Rev: 59-TACAAACTACCTCCA-TTACAGCC) and selecting stretches of near-identity.']),\n",
       " ('Primer-Bamplified',\n",
       "  'use',\n",
       "  'thermocycling',\n",
       "  ['Primer-Bamplified material was used as the template for 35 cycles of thermocycling using the following program: 948C for 30 s, 568C for 30 s, and 728C for 60 s.',\n",
       "   'Direct sequence recovery from the microarray.']),\n",
       " ('microarray',\n",
       "  'subclass of',\n",
       "  'Direct sequence recovery',\n",
       "  ['Primer-Bamplified material was used as the template for 35 cycles of thermocycling using the following program: 948C for 30 s, 568C for 30 s, and 728C for 60 s.',\n",
       "   'Direct sequence recovery from the microarray.']),\n",
       " ('Nikon TE300',\n",
       "  'use',\n",
       "  'fluorescence microscopy',\n",
       "  ['Amplified viral sequences hybridized to individual microarray spots were recovered by scraping a 100 lm area of the microarray using a tungsten wire probe (Omega Engineering, Inc.) mounted on a micromanipulator while visualized by fluorescence microscopy (Nikon TE300).',\n",
       "   'Recovered material was PCR amplified using primer-B, cloned into pCR2.1TOPO (Invitrogen), and sequenced.']),\n",
       " ('pCR2',\n",
       "  'developer',\n",
       "  'Invitrogen',\n",
       "  ['Amplified viral sequences hybridized to individual microarray spots were recovered by scraping a 100 lm area of the microarray using a tungsten wire probe (Omega Engineering, Inc.) mounted on a micromanipulator while visualized by fluorescence microscopy (Nikon TE300).',\n",
       "   'Recovered material was PCR amplified using primer-B, cloned into pCR2.1TOPO (Invitrogen), and sequenced.']),\n",
       " ('Protocol S2',\n",
       "  'use',\n",
       "  'Shotgun sequencing',\n",
       "  ['A detailed protocol is available as Protocol S2, found at http://dx.doi.org/10.1371/journal.pbio.0000002.sd3.(?<=[.?!])\\\\s+Shotgun sequencing.',\n",
       "   'Shotgun sequencing.(?<=[.?!])\\\\s+Primer-B-amplified nucleic acid (see above) was cloned in pCR2.1TOPO, plated on 2xYT/kan plates, and grown overnight at 378C.']),\n",
       " ('pCR2',\n",
       "  'use',\n",
       "  'Shotgun sequencing',\n",
       "  ['A detailed protocol is available as Protocol S2, found at http://dx.doi.org/10.1371/journal.pbio.0000002.sd3.(?<=[.?!])\\\\s+Shotgun sequencing.',\n",
       "   'Shotgun sequencing.(?<=[.?!])\\\\s+Primer-B-amplified nucleic acid (see above) was cloned in pCR2.1TOPO, plated on 2xYT/kan plates, and grown overnight at 378C.']),\n",
       " ('magnetic bead isolation',\n",
       "  'subclass of',\n",
       "  'purified',\n",
       "  ['White colonies were picked into 384-well plates containing 2xYT/kan plus 8% glycerol and incubated overnight at 378C.(?<=[.?!])\\\\s+DNA was purified by magnetic bead isolation.',\n",
       "   'DNA sequencing involved adding 3 ll of water to each bead pellet, followed by 3 ll of Big Dye terminator (v3.1) sequencing cocktail, and incubation for 35 cycles of 958C for 5 s, 508C for 5 s, and 608C for 2 min.']),\n",
       " ('sequencing cocktail',\n",
       "  'use',\n",
       "  'DNA sequencing',\n",
       "  ['White colonies were picked into 384-well plates containing 2xYT/kan plus 8% glycerol and incubated overnight at 378C.(?<=[.?!])\\\\s+DNA was purified by magnetic bead isolation.',\n",
       "   'DNA sequencing involved adding 3 ll of water to each bead pellet, followed by 3 ll of Big Dye terminator (v3.1) sequencing cocktail, and incubation for 35 cycles of 958C for 5 s, 508C for 5 s, and 608C for 2 min.']),\n",
       " ('ABI 3730xl',\n",
       "  'subclass of',\n",
       "  'sequencer',\n",
       "  ['Reaction products were ethanol precipitated, resuspended in 25 ll of water, and loaded onto the ABI 3730xl sequencer.',\n",
       "   'The resulting sequence reads were trimmed to remove primer sequences from the RT-PCR step and then assembled by Phrap (P.(?<=[.?!])\\\\s+Green, unpublished data).']),\n",
       " ('RT-PCR',\n",
       "  'uses',\n",
       "  'sequence read',\n",
       "  ['Reaction products were ethanol precipitated, resuspended in 25 ll of water, and loaded onto the ABI 3730xl sequencer.',\n",
       "   'The resulting sequence reads were trimmed to remove primer sequences from the RT-PCR step and then assembled by Phrap (P.(?<=[.?!])\\\\s+Green, unpublished data).']),\n",
       " ('human',\n",
       "  'different from',\n",
       "  'monkey',\n",
       "  ['Resulting contigs were screened by blast to remove any contigs with high human or monkey sequence similarity.(?<=[.?!])\\\\s+The remaining contigs were edited to high quality, making any obvious joins.',\n",
       "   '(Sequences are available as Data S1.) Data S1.']),\n",
       " ('monkey',\n",
       "  'different from',\n",
       "  'human',\n",
       "  ['Resulting contigs were screened by blast to remove any contigs with high human or monkey sequence similarity.(?<=[.?!])\\\\s+The remaining contigs were edited to high quality, making any obvious joins.',\n",
       "   '(Sequences are available as Data S1.) Data S1.']),\n",
       " ('Data S1',\n",
       "  'subclass of',\n",
       "  'Sequences',\n",
       "  ['Resulting contigs were screened by blast to remove any contigs with high human or monkey sequence similarity.(?<=[.?!])\\\\s+The remaining contigs were edited to high quality, making any obvious joins.',\n",
       "   '(Sequences are available as Data S1.) Data S1.']),\n",
       " ('Data S1',\n",
       "  'subclass of',\n",
       "  'Sequences',\n",
       "  ['Resulting contigs were screened by blast to remove any contigs with high human or monkey sequence similarity.(?<=[.?!])\\\\s+The remaining contigs were edited to high quality, making any obvious joins.',\n",
       "   '(Sequences are available as Data S1.) Data S1.']),\n",
       " ('Tripeptidyl-peptidase I',\n",
       "  'subclass of',\n",
       "  'sedolisin',\n",
       "  ['BACKGROUND: Tripeptidyl-peptidase I, also known as CLN2, is a member of the family of sedolisins (serine-carboxyl peptidases).',\n",
       "   'In humans, defects in expression of this enzyme lead to a fatal neurodegenerative disease, classical late-infantile neuronal ceroid lipofuscinosis.']),\n",
       " ('classical late-infantile neuronal ceroid lipofuscinosis',\n",
       "  'subclass of',\n",
       "  'neurodegenerative disease',\n",
       "  ['BACKGROUND: Tripeptidyl-peptidase I, also known as CLN2, is a member of the family of sedolisins (serine-carboxyl peptidases).',\n",
       "   'In humans, defects in expression of this enzyme lead to a fatal neurodegenerative disease, classical late-infantile neuronal ceroid lipofuscinosis.']),\n",
       " ('genomic',\n",
       "  'subclass of',\n",
       "  'sequences',\n",
       "  ['Similar enzymes have been found in the genomic sequences of several species, but neither systematic analyses of their distribution nor modeling of their structures have been previously attempted.',\n",
       "   'RESULTS: We have analyzed the presence of orthologs of human CLN2 in the genomic sequences of a number of eukaryotic species.']),\n",
       " ('eukaryotic',\n",
       "  'has parts of the class',\n",
       "  'genomic',\n",
       "  ['Similar enzymes have been found in the genomic sequences of several species, but neither systematic analyses of their distribution nor modeling of their structures have been previously attempted.',\n",
       "   'RESULTS: We have analyzed the presence of orthologs of human CLN2 in the genomic sequences of a number of eukaryotic species.']),\n",
       " ('Enzyme',\n",
       "  'part of',\n",
       "  'genome',\n",
       "  ['Enzymes with sequences sharing over 80% identity have been found in the genomes of macaque, mouse, rat, dog, and cow.',\n",
       "   'Closely related, although clearly distinct, enzymes are present in fish (fugu and zebra), as well as in frogs (Xenopus tropicalis).']),\n",
       " ('Fenopus tropicalis',\n",
       "  'parent taxon',\n",
       "  'frog',\n",
       "  ['Enzymes with sequences sharing over 80% identity have been found in the genomes of macaque, mouse, rat, dog, and cow.',\n",
       "   'Closely related, although clearly distinct, enzymes are present in fish (fugu and zebra), as well as in frogs (Xenopus tropicalis).']),\n",
       " ('three-dimensional',\n",
       "  'subclass of',\n",
       "  'model',\n",
       "  ['A three-dimensional model of human CLN2 was built based mainly on the homology with Pseudomonas sp.(?<=[.?!])\\\\s+101 sedolisin.',\n",
       "   '101 sedolisin.(?<=[.?!])\\\\s+CONCLUSION: CLN2 is very highly conserved and widely distributed among higher organisms and may play an important role in their life cycles.']),\n",
       " ('life cycle',\n",
       "  'facet of',\n",
       "  'organism',\n",
       "  ['A three-dimensional model of human CLN2 was built based mainly on the homology with Pseudomonas sp.(?<=[.?!])\\\\s+101 sedolisin.',\n",
       "   '101 sedolisin.(?<=[.?!])\\\\s+CONCLUSION: CLN2 is very highly conserved and widely distributed among higher organisms and may play an important role in their life cycles.']),\n",
       " ('active site',\n",
       "  'part of',\n",
       "  'enzyme',\n",
       "  ['The model presented here indicates a very open and accessible active site that is almost completely conserved among all known CLN2 enzymes.',\n",
       "   'This result is somehow surprising for a tripeptidase where the presence of a more constrained binding pocket was anticipated.']),\n",
       " ('binding pocket',\n",
       "  'subclass of',\n",
       "  'pocket',\n",
       "  ['The model presented here indicates a very open and accessible active site that is almost completely conserved among all known CLN2 enzymes.',\n",
       "   'This result is somehow surprising for a tripeptidase where the presence of a more constrained binding pocket was anticipated.']),\n",
       " ('inhibitors',\n",
       "  'subclass of',\n",
       "  'enzymes',\n",
       "  ['This structural model should be useful in the search for the physiological substrates of these enzymes and in the design of more specific inhibitors of CLN2.',\n",
       "   'Although the existence of tripeptidyl-peptidase I (TPP-I) was first noted over 40 years ago [1] , the structural and mechanistic basis of its activity has been largely misunder-stood until quite recently.']),\n",
       " ('TPP-I',\n",
       "  'subclass of',\n",
       "  'tripeptidyl-peptidase I',\n",
       "  ['This structural model should be useful in the search for the physiological substrates of these enzymes and in the design of more specific inhibitors of CLN2.',\n",
       "   'Although the existence of tripeptidyl-peptidase I (TPP-I) was first noted over 40 years ago [1] , the structural and mechanistic basis of its activity has been largely misunder-stood until quite recently.']),\n",
       " ('CLN2',\n",
       "  'instance of',\n",
       "  'enzyme',\n",
       "  ['The situation changed after it was shown that TPP-I is identical to an independently characterized enzyme named CLN2.',\n",
       "   'It was also demonstrated that mutations leading to abolishment of the enzymatic activity of CLN2 were the direct cause of a fatal inherited neurodegenerative disease, classical late-infantile neuronal ceroid lipofuscinosis [2] .']),\n",
       " ('classical late-infantile neuronal ceroid lipofuscinosis',\n",
       "  'subclass of',\n",
       "  'inherited neurodegenerative disease',\n",
       "  ['The situation changed after it was shown that TPP-I is identical to an independently characterized enzyme named CLN2.',\n",
       "   'It was also demonstrated that mutations leading to abolishment of the enzymatic activity of CLN2 were the direct cause of a fatal inherited neurodegenerative disease, classical late-infantile neuronal ceroid lipofuscinosis [2] .']),\n",
       " ('CLN2',\n",
       "  'subclass of',\n",
       "  'serine peptidase',\n",
       "  ['This important observation was followed by the identification of CLN2 as a serine peptidase [3, 4] , without, however, specifying its structural fold and the details of the catalytic site.',\n",
       "   'More accurate placement of CLN2 within the context of a family of related enzymes became possible only after high-resolution crystal structures of two bacterial enzymes with a limited sequence similarity to CLN2, sedolisin and kumamolisin, became available [5] [6] [7] .']),\n",
       " ('kumamolisin',\n",
       "  'subclass of',\n",
       "  'enzyme',\n",
       "  ['This important observation was followed by the identification of CLN2 as a serine peptidase [3, 4] , without, however, specifying its structural fold and the details of the catalytic site.',\n",
       "   'More accurate placement of CLN2 within the context of a family of related enzymes became possible only after high-resolution crystal structures of two bacterial enzymes with a limited sequence similarity to CLN2, sedolisin and kumamolisin, became available [5] [6] [7] .']),\n",
       " ('oxyanion hole',\n",
       "  'part of',\n",
       "  'catalytic triad',\n",
       "  ['These structures defined a novel family of enzymes, now called sedolisins or serine-carboxyl peptidases, that is characterized by the utilization of a fully conserved catalytic triad (Ser, Glu, Asp) and by the presence of an Asp in the oxyanion hole [8] .',\n",
       "   'Sedolisin and its several variants (e.g., kumamolisin, aorsin [9] , and physarolisin [10] ) have now been found in archaea, bacteria, fungi and amoebae, whereas the higher organisms seem to contain only variants of CLN2 [8] .']),\n",
       " ('kumamolisin',\n",
       "  'subclass of',\n",
       "  'Sedolisin',\n",
       "  ['These structures defined a novel family of enzymes, now called sedolisins or serine-carboxyl peptidases, that is characterized by the utilization of a fully conserved catalytic triad (Ser, Glu, Asp) and by the presence of an Asp in the oxyanion hole [8] .',\n",
       "   'Sedolisin and its several variants (e.g., kumamolisin, aorsin [9] , and physarolisin [10] ) have now been found in archaea, bacteria, fungi and amoebae, whereas the higher organisms seem to contain only variants of CLN2 [8] .']),\n",
       " ('physiological role',\n",
       "  'instance of',\n",
       "  'role',\n",
       "  ['The physiological role of sedolisins in the lower organisms has not yet been elucidated.',\n",
       "   'Despite the potential medical importance of CLN2 and related enzymes, no systematic studies of their genomic distribution have been published to date.']),\n",
       " ('CLN2',\n",
       "  'subclass of',\n",
       "  'enzymes',\n",
       "  ['The physiological role of sedolisins in the lower organisms has not yet been elucidated.',\n",
       "   'Despite the potential medical importance of CLN2 and related enzymes, no systematic studies of their genomic distribution have been published to date.']),\n",
       " ('crystallization',\n",
       "  'subclass of',\n",
       "  'enzyme',\n",
       "  ['There are also no published reports of the crystallization of this enzyme.',\n",
       "   'In the absence of an experimental structure obtained by crystallography or NMR it is sometimes necessary to resort to molecular modeling in order to provide a structural basis for the explanation of the biological properties of an enzyme, and, in particular, to initiate design of its inhibitors.']),\n",
       " ('molecular modeling',\n",
       "  'subclass of',\n",
       "  'structural basis',\n",
       "  ['There are also no published reports of the crystallization of this enzyme.',\n",
       "   'In the absence of an experimental structure obtained by crystallography or NMR it is sometimes necessary to resort to molecular modeling in order to provide a structural basis for the explanation of the biological properties of an enzyme, and, in particular, to initiate design of its inhibitors.']),\n",
       " ('coronavirus',\n",
       "  'has effect',\n",
       "  'severe acute respiratory syndrome',\n",
       "  ['Examples of such very successful and useful modeling efforts are provided by HIV protease [11] , or very recently by the peptidase from a coronavirus involved in the severe acute respiratory syndrome [12] , among others.',\n",
       "   'We have now applied the tools of molecular homology modeling to predicting a structure of CLN2 that could be used as a basis for a search for the biological substrates of this family of enzymes and for the design of specific inhibitors.']),\n",
       " ('molecular homology modeling',\n",
       "  'subclass of',\n",
       "  'structure',\n",
       "  ['Examples of such very successful and useful modeling efforts are provided by HIV protease [11] , or very recently by the peptidase from a coronavirus involved in the severe acute respiratory syndrome [12] , among others.',\n",
       "   'We have now applied the tools of molecular homology modeling to predicting a structure of CLN2 that could be used as a basis for a search for the biological substrates of this family of enzymes and for the design of specific inhibitors.']),\n",
       " ('CLN2',\n",
       "  'subclass of',\n",
       "  'sedolisin',\n",
       "  ['Mammalian enzymes homologous to human CLN2 [2, 4] form a subfamily of sedolisins with highly conserved sequences ( Figure 1 ).',\n",
       "   'These enzymes are expressed with a prosegment consisting of 195 residues that is cleaved off during maturation, yielding the active catalytic domain.']),\n",
       " ('catalytic domain',\n",
       "  'part of',\n",
       "  'enzyme',\n",
       "  ['Mammalian enzymes homologous to human CLN2 [2, 4] form a subfamily of sedolisins with highly conserved sequences ( Figure 1 ).',\n",
       "   'These enzymes are expressed with a prosegment consisting of 195 residues that is cleaved off during maturation, yielding the active catalytic domain.']),\n",
       " ('macaque',\n",
       "  'subclass of',\n",
       "  'human',\n",
       "  ['Complete sequences are available for CLN2 from six species in which it has been found so far (human, macaque, dog, mouse, rat, and cow).',\n",
       "   'The full-length enzymes consist of 563 amino acids arranged in a single polypeptide chain containing both the prosegment and the catalytic domain, with the exception of mouse CLN2 that has a single deletion in the prosegment.']),\n",
       " ('catalytic domain',\n",
       "  'part of',\n",
       "  'polypeptide chain',\n",
       "  ['Complete sequences are available for CLN2 from six species in which it has been found so far (human, macaque, dog, mouse, rat, and cow).',\n",
       "   'The full-length enzymes consist of 563 amino acids arranged in a single polypeptide chain containing both the prosegment and the catalytic domain, with the exception of mouse CLN2 that has a single deletion in the prosegment.']),\n",
       " ('similarity',\n",
       "  'subclass of',\n",
       "  'iden-tity',\n",
       "  ['The overall sequence iden-tity for these enzymes is 81%, whereas the similarity is 92%.',\n",
       "   'A pairwise comparison of the human and mouse enzymes yields 88% identity and 94% similarity, considerably higher than the median 78.5% identity reported for all identified mouse-human orthologs [13] .']),\n",
       " ('orthologs',\n",
       "  'subclass of',\n",
       "  'identity',\n",
       "  ['The overall sequence iden-tity for these enzymes is 81%, whereas the similarity is 92%.',\n",
       "   'A pairwise comparison of the human and mouse enzymes yields 88% identity and 94% similarity, considerably higher than the median 78.5% identity reported for all identified mouse-human orthologs [13] .']),\n",
       " ('CLN2',\n",
       "  'subclass of',\n",
       "  'enzyme',\n",
       "  ['Thus, mammalian CLN2 appears to be a highly conserved enzyme.',\n",
       "   'In addition to the mammals, CLN2-like enzymes are also found in fugu (puffer fish -unannotated record SINFRUP00000077297 in the NCBI fugu sequence database, http://www.ncbi.nlm.nih.gov/BLAST/Genome/ fugu.html) and zebrafish (contig wz4596.2 in the zebrafish EST database, http://fisher.wustl.edu/fish_lab).']),\n",
       " ('fugu sequence database',\n",
       "  'maintained by',\n",
       "  'NCBI',\n",
       "  ['Thus, mammalian CLN2 appears to be a highly conserved enzyme.',\n",
       "   'In addition to the mammals, CLN2-like enzymes are also found in fugu (puffer fish -unannotated record SINFRUP00000077297 in the NCBI fugu sequence database, http://www.ncbi.nlm.nih.gov/BLAST/Genome/ fugu.html) and zebrafish (contig wz4596.2 in the zebrafish EST database, http://fisher.wustl.edu/fish_lab).']),\n",
       " ('sequence',\n",
       "  'facet of',\n",
       "  'enzyme',\n",
       "  ['Only a fragment of the sequence of the latter enzyme agreed with the former.',\n",
       "   'However, a few comparatively minor modifications can bring the zebrafish sequence into good agreement with that of the fugu CLN2.']),\n",
       " ('fugu',\n",
       "  'taxonomic type',\n",
       "  'CLN2',\n",
       "  ['Only a fragment of the sequence of the latter enzyme agreed with the former.',\n",
       "   'However, a few comparatively minor modifications can bring the zebrafish sequence into good agreement with that of the fugu CLN2.']),\n",
       " ('CLN2',\n",
       "  'parent taxon',\n",
       "  'fugu',\n",
       "  ['Only a fragment of the sequence of the latter enzyme agreed with the former.',\n",
       "   'However, a few comparatively minor modifications can bring the zebrafish sequence into good agreement with that of the fugu CLN2.']),\n",
       " ('nucleotide',\n",
       "  'part of',\n",
       "  'nucleotide pair',\n",
       "  ['These modifications include a deletion of a single nucleotide from a run of three, as well as three insertions of repeated nucleotide pairs ( Figure 2 ).',\n",
       "   'It must be stressed that these modifications are speculative and may lead to prediction of several incorrect amino acids; however, they bring the two sequences into good global agreement (69% identities and 83% similarities).']),\n",
       " ('nucleotide pair',\n",
       "  'has part',\n",
       "  'nucleotide',\n",
       "  ['These modifications include a deletion of a single nucleotide from a run of three, as well as three insertions of repeated nucleotide pairs ( Figure 2 ).',\n",
       "   'It must be stressed that these modifications are speculative and may lead to prediction of several incorrect amino acids; however, they bring the two sequences into good global agreement (69% identities and 83% similarities).']),\n",
       " ('amino acid',\n",
       "  'part of',\n",
       "  'sequences',\n",
       "  ['These modifications include a deletion of a single nucleotide from a run of three, as well as three insertions of repeated nucleotide pairs ( Figure 2 ).',\n",
       "   'It must be stressed that these modifications are speculative and may lead to prediction of several incorrect amino acids; however, they bring the two sequences into good global agreement (69% identities and 83% similarities).']),\n",
       " ('fugu',\n",
       "  'subclass of',\n",
       "  'amino acid',\n",
       "  ['The available amino acid sequence of the fugu CLN2 analog, named by us sedolisin-TPP [8] , is also in good agreement with the sequences of the mammalian orthologs ( Figure 3 ).',\n",
       "   'The only major difference in the translated amino acid sequence compared to the mammalian and zebrafish enzymes is in the amino terminus of the propeptide region that is shorter by 30 amino acids (not shown).']),\n",
       " ('amino terminus',\n",
       "  'subclass of',\n",
       "  'amino acid',\n",
       "  ['The available amino acid sequence of the fugu CLN2 analog, named by us sedolisin-TPP [8] , is also in good agreement with the sequences of the mammalian orthologs ( Figure 3 ).',\n",
       "   'The only major difference in the translated amino acid sequence compared to the mammalian and zebrafish enzymes is in the amino terminus of the propeptide region that is shorter by 30 amino acids (not shown).']),\n",
       " ('fugu6',\n",
       "  'instance of',\n",
       "  'genomic sequence',\n",
       "  ['It is very likely that this represents a fault in the assembled sequence rather than a real variation, since the current coding frame is not initiated with a methionine, and a few extra residues are present in the full genomic sequence available from the fugu sequencing consortium http://genome.jgi-psf.org/fugu6/fugu6.home.html.',\n",
       "   'CLN2 is present not only in fish, but also in amphibians, in particular in Xenopus tropicalis (a species of frog).']),\n",
       " ('Xenopus tropicalis',\n",
       "  'parent taxon',\n",
       "  'frog',\n",
       "  ['It is very likely that this represents a fault in the assembled sequence rather than a real variation, since the current coding frame is not initiated with a methionine, and a few extra residues are present in the full genomic sequence available from the fugu sequencing consortium http://genome.jgi-psf.org/fugu6/fugu6.home.html.',\n",
       "   'CLN2 is present not only in fish, but also in amphibians, in particular in Xenopus tropicalis (a species of frog).']),\n",
       " ('catalytic domain',\n",
       "  'has part',\n",
       "  'active site',\n",
       "  ['A partial sequence of its sedolisin-TPP (AL594774) found in the EST database http://www.sanger.ac.uk/Projects/ X_tropicalis/blast_server.shtml spans the middle part of the catalytic domain, without reaching the part of the active site closer to the N terminus that contains the aspartic and glutamic acids that belong to the catalytic triad.',\n",
       "   'However, the sequenced part of the enzyme shows 75% identity with the fugu sedolisin-TPP, and 69% identity with human CLN2 (Figure 3 ).']),\n",
       " ('fugu',\n",
       "  'product or material produced',\n",
       "  'sedolisin',\n",
       "  ['A partial sequence of its sedolisin-TPP (AL594774) found in the EST database http://www.sanger.ac.uk/Projects/ X_tropicalis/blast_server.shtml spans the middle part of the catalytic domain, without reaching the part of the active site closer to the N terminus that contains the aspartic and glutamic acids that belong to the catalytic triad.',\n",
       "   'However, the sequenced part of the enzyme shows 75% identity with the fugu sedolisin-TPP, and 69% identity with human CLN2 (Figure 3 ).']),\n",
       " ('fungal',\n",
       "  'subclass of',\n",
       "  'sedolisin',\n",
       "  ['Sequence similarity to bacterial or fungal sedolisins is much lower, indicating that the enzyme found in frogs might also share the functional properties of the CLN2 subfamily.',\n",
       "   'Sequence comparisons of mammalian CLN2-like enzymes Figure 1 Sequence comparisons of mammalian CLN2-like enzymes.These sequences correspond to the complete enzymes, including the prosegment.']),\n",
       " ('prosegment',\n",
       "  'part of',\n",
       "  'enzyme',\n",
       "  ['Sequence similarity to bacterial or fungal sedolisins is much lower, indicating that the enzyme found in frogs might also share the functional properties of the CLN2 subfamily.',\n",
       "   'Sequence comparisons of mammalian CLN2-like enzymes Figure 1 Sequence comparisons of mammalian CLN2-like enzymes.These sequences correspond to the complete enzymes, including the prosegment.']),\n",
       " ('active site',\n",
       "  'part of',\n",
       "  'enzyme',\n",
       "  ['Residues forming the active site are shown in yellow on red background, other conserved residues identified as important for the stability of the enzyme are marked with yellow background, residues identical in at least 5 of the structures are green, and residues similar in their character are shown in magenta.',\n",
       "   'The maturation cleavage point generating the N terminus of the active enzyme is marked with black triangles.']),\n",
       " ('N terminus',\n",
       "  'part of',\n",
       "  'active enzyme',\n",
       "  ['Residues forming the active site are shown in yellow on red background, other conserved residues identified as important for the stability of the enzyme are marked with yellow background, residues identical in at least 5 of the structures are green, and residues similar in their character are shown in magenta.',\n",
       "   'The maturation cleavage point generating the N terminus of the active enzyme is marked with black triangles.']),\n",
       " ('fish',\n",
       "  'subclass of',\n",
       "  'vertebrates',\n",
       "  ['The discovery of highly conserved CLN2-like enzymes not only in mammals but also in two fish species and one of frogs may indicate that these peptidases are universally present in the vertebrates, and that their important role identified in humans [2] and mice [14] might be a more general feature.',\n",
       "   'The medical importance of CLN2 and the lack of a crystal structure inspired attempts at protein modeling.']),\n",
       " ('protein modeling',\n",
       "  'uses',\n",
       "  'crystal structure',\n",
       "  ['The discovery of highly conserved CLN2-like enzymes not only in mammals but also in two fish species and one of frogs may indicate that these peptidases are universally present in the vertebrates, and that their important role identified in humans [2] and mice [14] might be a more general feature.',\n",
       "   'The medical importance of CLN2 and the lack of a crystal structure inspired attempts at protein modeling.']),\n",
       " ('transmembrane anchor',\n",
       "  'part of',\n",
       "  'membranebound',\n",
       "  ['The first such model assumed that this enzyme is membranebound, with the sequence 271-294 (numbering corresponding to the mature enzyme, Figure 1 ) forming the putative transmembrane anchor [15] .',\n",
       "   'However, in view of the subsequently obtained structures of the fully watersoluble sedolisins, this model was clearly incorrect.']),\n",
       " ('sedolisin',\n",
       "  'instance of',\n",
       "  'watersoluble',\n",
       "  ['The first such model assumed that this enzyme is membranebound, with the sequence 271-294 (numbering corresponding to the mature enzyme, Figure 1 ) forming the putative transmembrane anchor [15] .',\n",
       "   'However, in view of the subsequently obtained structures of the fully watersoluble sedolisins, this model was clearly incorrect.']),\n",
       " ('kumamolisin',\n",
       "  'instance of',\n",
       "  'enzyme',\n",
       "  ['Exploiting the sequence similarity between CLN2, sedolisin, and kumamolisin ( Figure 4 ), we have now used the experimentally obtained structures of the latter two enzymes to form a new, homology-derived model of human CLN2.',\n",
       "   'The primary basis of the homology model was the structure of a complex of sedolisin with a covalently-bound inhibitor, pseudo-iodotyrostatin.']),\n",
       " ('pseudo-iodotyrostatin',\n",
       "  'subject has role',\n",
       "  'inhibitor',\n",
       "  ['Exploiting the sequence similarity between CLN2, sedolisin, and kumamolisin ( Figure 4 ), we have now used the experimentally obtained structures of the latter two enzymes to form a new, homology-derived model of human CLN2.',\n",
       "   'The primary basis of the homology model was the structure of a complex of sedolisin with a covalently-bound inhibitor, pseudo-iodotyrostatin.']),\n",
       " ...]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "triples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Triples saved to 'rebel_triples.csv' with duplicates and NaN values removed.\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Convert the triples list to a DataFrame\n",
    "triples_df = pd.DataFrame(triples, columns=['subject', 'relationship', 'object', 'chunk'])\n",
    "\n",
    "# Convert the 'chunk' column to strings to make it hashable\n",
    "triples_df['chunk'] = triples_df['chunk'].apply(lambda x: str(x))\n",
    "\n",
    "# Drop rows with NaN values\n",
    "triples_df = triples_df.dropna()\n",
    "\n",
    "# Drop duplicates\n",
    "triples_df_cleaned = triples_df.drop_duplicates()\n",
    "\n",
    "# Save the cleaned DataFrame to a CSV file\n",
    "triples_df_cleaned.to_csv('output_from_using_REBEL/rebel_triples.csv', index=False)\n",
    "\n",
    "print(\"Triples saved to 'rebel_triples.csv' with duplicates and NaN values removed.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 15866 entries, 0 to 15865\n",
      "Data columns (total 4 columns):\n",
      " #   Column        Non-Null Count  Dtype \n",
      "---  ------        --------------  ----- \n",
      " 0   subject       15866 non-null  object\n",
      " 1   relationship  15866 non-null  object\n",
      " 2   object        15864 non-null  object\n",
      " 3   chunk         15866 non-null  object\n",
      "dtypes: object(4)\n",
      "memory usage: 495.9+ KB\n"
     ]
    }
   ],
   "source": [
    "dsads = pd.read_csv('output_from_using_REBEL/rebel_triples.csv')\n",
    "dsads.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Json to CSV"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processing chunks for document 0: 100%|██████████| 132/132 [00:00<00:00, 1693113.54it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chunks for document 1 saved to document_1_chunks.json\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 132/132 [00:00<00:00, 4266.31it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output saved to all_triples_with_metadata_0.json\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processing chunks for document 1: 100%|██████████| 73/73 [00:00<00:00, 1515763.33it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chunks for document 2 saved to document_2_chunks.json\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 73/73 [00:00<00:00, 3021.53it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output saved to all_triples_with_metadata_1.json\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import hashlib\n",
    "from transformers import AutoTokenizer\n",
    "from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "from tqdm import tqdm\n",
    "import pandas as pd\n",
    "import time\n",
    "import os\n",
    "\n",
    "# Initialize tokenizer and text splitter\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"bert-base-uncased\")\n",
    "text_splitter = RecursiveCharacterTextSplitter(\n",
    "    separators=[\"\\n\\n\", \"\\n\", r\"(?<=[.?!])\\s+\"],\n",
    "    keep_separator=False, is_separator_regex=True,\n",
    "    chunk_size=200, chunk_overlap=20\n",
    ")\n",
    "\n",
    "# Load metadata\n",
    "metadata_df = pd.read_csv(\"cleaned_data_with_document.csv\")\n",
    "metadata_map = metadata_df.to_dict(\"index\")\n",
    "\n",
    "# Initialize hashing object\n",
    "m = hashlib.md5()\n",
    "\n",
    "# Function to extract triples (placeholder for your actual logic)\n",
    "def extract_triples(text):\n",
    "    file_path = \"rebel_triples.json\"\n",
    "    if not os.path.exists(file_path):\n",
    "        print(f\"Error: {file_path} not found.\")\n",
    "        return []\n",
    "\n",
    "    # Load the source JSON file\n",
    "    with open(file_path, \"r\") as f:\n",
    "        source_data = json.load(f)\n",
    "\n",
    "    # Process the data into the target format\n",
    "    converted_data = []\n",
    "    for triple in source_data:\n",
    "        # Ensure the triple is a list with three elements\n",
    "        if len(triple) == 3:\n",
    "            subject, relationship, obj = triple\n",
    "            converted_data.append({\n",
    "                \"subject\": subject,\n",
    "                \"relationship\": relationship,\n",
    "                \"object\": obj\n",
    "            })\n",
    "    return converted_data\n",
    "\n",
    "# Process each document\n",
    "for doc_index, document in enumerate(documents[:2]):  # Iterate over the first two documents\n",
    "    m.update(document.encode('utf-8'))\n",
    "    uid = m.hexdigest()[:12]\n",
    "    chunks = text_splitter.split_text(document)  # Split the document into chunks\n",
    "    \n",
    "    # Initialize storage for chunks\n",
    "    doc_chunks = []\n",
    "    \n",
    "    # Process each chunk\n",
    "    for i, chunk in enumerate(tqdm(chunks, desc=f\"Processing chunks for document {doc_index}\")):\n",
    "        doc_chunks.append({\n",
    "            'id': f'{uid}-{i}',\n",
    "            'text': chunk\n",
    "        })\n",
    "\n",
    "    # Save results for the current document\n",
    "    output_file = f\"document_{doc_index + 1}_chunks.json\"\n",
    "    result = {\n",
    "        \"document_index\": doc_index,\n",
    "        \"document_id\": uid,\n",
    "        \"chunks\": doc_chunks\n",
    "    }\n",
    "    with open(output_file, \"w\") as f:\n",
    "        json.dump(result, f, indent=4)\n",
    "\n",
    "    print(f\"Chunks for document {doc_index + 1} saved to {output_file}\")\n",
    "\n",
    "    # Extract triples and enrich with metadata\n",
    "    errors = []\n",
    "    all_triples = []\n",
    "    for i in tqdm(range(len(doc_chunks))):\n",
    "        try:\n",
    "            text = doc_chunks[i]['text']\n",
    "            chunk_id = doc_chunks[i]['id']\n",
    "            triples = extract_triples(text)  # Extract triples\n",
    "\n",
    "            # Get the metadata for the current document\n",
    "            document_metadata = metadata_map[doc_index]\n",
    "\n",
    "            # Enrich each triple with the corresponding metadata and chunk information\n",
    "            for triple in triples:\n",
    "                enriched_triple = {\n",
    "                    \"subject\": triple[\"subject\"],\n",
    "                    \"relationship\": triple[\"relationship\"],\n",
    "                    \"object\": triple[\"object\"],\n",
    "                    # Metadata\n",
    "                    # \"cord_uid\": document_metadata[\"cord_uid\"],\n",
    "                    # \"sha\": document_metadata[\"sha\"],\n",
    "                    # \"title\": document_metadata[\"title\"],\n",
    "                    # \"abstract\": document_metadata[\"abstract\"],\n",
    "                    # \"publish_time\": document_metadata[\"publish_time\"],\n",
    "                    # \"authors\": document_metadata[\"authors\"],\n",
    "                    # Chunk information\n",
    "                    \"chunk_id\": chunk_id,\n",
    "                    \"chunk_text\": text\n",
    "                }\n",
    "                all_triples.append(enriched_triple)\n",
    "            # time.sleep(3)  # Avoid rate-limiting by adding a delay\n",
    "        except Exception as e:\n",
    "            print(f\"Error for chunk {i}: {e}\")\n",
    "            errors.append({\"chunk\": i, \"error\": str(e)})\n",
    "            all_triples.append([])  # Append an empty list for failed chunks\n",
    "\n",
    "    # Save the enriched triples to a JSON file\n",
    "    output_file = f\"all_triples_with_metadata_{doc_index}.json\"\n",
    "    with open(output_file, \"w\") as f:\n",
    "        json.dump(all_triples, f, indent=4)\n",
    "    print(f\"Output saved to {output_file}\")\n",
    "\n",
    "# Optionally save errors to a separate file\n",
    "if errors:\n",
    "    with open(\"errors.json\", \"w\") as f:\n",
    "        json.dump(errors, f, indent=4)\n",
    "    print(\"Errors saved to 'errors.json'.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Combined data saved to all_triples_with_metadata_combined.csv\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import pandas as pd\n",
    "\n",
    "# Function to convert JSON to a list of dictionaries\n",
    "def json_to_list(json_file):\n",
    "    with open(json_file, 'r') as f:\n",
    "        data = json.load(f)\n",
    "    \n",
    "    # Flatten the JSON structure\n",
    "    flattened_data = []\n",
    "    for doc in data:\n",
    "        for triple in doc:\n",
    "            flattened_data.append(triple)\n",
    "    \n",
    "    return flattened_data\n",
    "\n",
    "# Initialize an empty list to store all data\n",
    "all_data = []\n",
    "\n",
    "# Convert each JSON file to a list of dictionaries and append to all_data\n",
    "for doc_index in range(2):  # Assuming you have two documents\n",
    "    json_file = f\"output_from_using_REBEL/all_triples_with_metadata_{doc_index}.json\"\n",
    "    all_data.extend(json_to_list(json_file))\n",
    "\n",
    "# Convert the combined data to a DataFrame\n",
    "df = pd.DataFrame(all_data)\n",
    "\n",
    "# Save the DataFrame to a single CSV file\n",
    "csv_file = \"all_triples_with_metadata_combined.csv\"\n",
    "df.to_csv(csv_file, index=False)\n",
    "print(f\"Combined data saved to {csv_file}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>subject</th>\n",
       "      <th>relationship</th>\n",
       "      <th>object</th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "      <td>128535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unique</th>\n",
       "      <td>466</td>\n",
       "      <td>52</td>\n",
       "      <td>402</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>top</th>\n",
       "      <td>SARS</td>\n",
       "      <td>subclass of</td>\n",
       "      <td>SARS</td>\n",
       "      <td>xqhn0vbp</td>\n",
       "      <td>1e1286db212100993d03cc22374b624f7caee956</td>\n",
       "      <td>Airborne rhinovirus detection and effect of ul...</td>\n",
       "      <td>BACKGROUND: Rhinovirus, the most common cause ...</td>\n",
       "      <td>2003-01-13</td>\n",
       "      <td>Myatt, Theodore A; Johnston, Sebastian L; Rudn...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freq</th>\n",
       "      <td>3690</td>\n",
       "      <td>27470</td>\n",
       "      <td>4920</td>\n",
       "      <td>82764</td>\n",
       "      <td>82764</td>\n",
       "      <td>82764</td>\n",
       "      <td>82764</td>\n",
       "      <td>82764</td>\n",
       "      <td>82764</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       subject relationship  object  cord_uid  \\\n",
       "count   128535       128535  128535    128535   \n",
       "unique     466           52     402         2   \n",
       "top       SARS  subclass of    SARS  xqhn0vbp   \n",
       "freq      3690        27470    4920     82764   \n",
       "\n",
       "                                             sha  \\\n",
       "count                                     128535   \n",
       "unique                                         2   \n",
       "top     1e1286db212100993d03cc22374b624f7caee956   \n",
       "freq                                       82764   \n",
       "\n",
       "                                                    title  \\\n",
       "count                                              128535   \n",
       "unique                                                  2   \n",
       "top     Airborne rhinovirus detection and effect of ul...   \n",
       "freq                                                82764   \n",
       "\n",
       "                                                 abstract publish_time  \\\n",
       "count                                              128535       128535   \n",
       "unique                                                  2            2   \n",
       "top     BACKGROUND: Rhinovirus, the most common cause ...   2003-01-13   \n",
       "freq                                                82764        82764   \n",
       "\n",
       "                                                  authors  \n",
       "count                                              128535  \n",
       "unique                                                  2  \n",
       "top     Myatt, Theodore A; Johnston, Sebastian L; Rudn...  \n",
       "freq                                                82764  "
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the CSV file with the combined data\n",
    "all_data = pd.read_csv(\"output_from_using_REBEL/REBEL_all_triples_with_metadata_combined.csv\")\n",
    "all_data.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(128535, 9)"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_data.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(627, 3)\n"
     ]
    }
   ],
   "source": [
    "print(all_data[['subject', 'relationship', 'object']].drop_duplicates().shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_data_non_duplicates = all_data[['subject', 'relationship', 'object']].drop_duplicates()\n",
    "\n",
    "# Print the output to the console\n",
    "all_data_non_duplicates.to_csv('output_from_using_REBEL/REBEL_all_triples_No_duplicate.csv', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Entity Type Adding Process"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sentences saved to 'output_from_using_REBEL/rebel_triples_sentences.csv'\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the CSV file with the triples\n",
    "triples_df = pd.read_csv('output_from_using_REBEL/rebel_triples.csv')\n",
    "\n",
    "# Function to convert a triple into a sentence\n",
    "def triple_to_sentence(triple):\n",
    "    return f\"{triple['subject']} {triple['relationship']} {triple['object']}.\"\n",
    "\n",
    "# Apply the function to each row in the DataFrame\n",
    "triples_df['sentence'] = triples_df.apply(triple_to_sentence, axis=1)\n",
    "\n",
    "# Display the resulting DataFrame with sentences\n",
    "triples_df.head()\n",
    "\n",
    "# Save the sentences to a new CSV file\n",
    "triples_df.to_csv('output_from_using_REBEL/rebel_triples_sentences.csv', index=False)\n",
    "print(\"Sentences saved to 'output_from_using_REBEL/rebel_triples_sentences.csv'\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "triples_sen = pd.read_csv(\"output_from_using_REBEL/rebel_triples_sentences.csv\")\n",
    "triples_sen"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Fetching 10 files: 100%|██████████| 10/10 [00:00<00:00, 248183.67it/s]\n",
      "Asking to truncate to max_length but no maximum length is provided and the model has no predefined maximum length. Default to no truncation.\n"
     ]
    }
   ],
   "source": [
    "from gliner import GLiNER\n",
    "\n",
    "model = GLiNER.from_pretrained(\"gliner-community/gliner_large-v2.5\")\n",
    "\n",
    "triples_sen = pd.read_csv(\"output_from_using_REBEL/rebel_triples_sentences.csv\")\n",
    "\n",
    "text = triples_sen[\"sentence\"].tolist()\n",
    "\n",
    "labels = [\n",
    "    \"virus\", \"disease\", \"symptom\", \"drug\", \"vaccine\", \n",
    "    \"gene\", \"diagnostic\", \"treatment\", \"transmission\",\n",
    "    \"epidemiology\", \"public health\", \"host\", \"immune response\",\n",
    "    \"organization\", \"measurement\", \"publication\", \"environment\", \"substance\", \n",
    "    \"molecule\", \"chemical\", \"bacteria\", \"country\", \"region\", \"city\", \"province\", \n",
    "    \"state\", \"location\", \"event\", \"method\", \"person\", \"people\"\n",
    "]\n",
    "\n",
    "entity_label_dict = {}\n",
    "\n",
    "# Iterate through the sentences and predict entities\n",
    "for sentence in text:\n",
    "    entities = model.predict_entities(sentence, labels, threshold=0.7)\n",
    "    for entity in entities:\n",
    "        entity_text = entity[\"text\"]\n",
    "        entity_label = entity[\"label\"]\n",
    "        # Store the entity and label in the dictionary\n",
    "        entity_label_dict[entity_text] = entity_label\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print(len((entity_label_dict)))\n",
    "entity_label_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity label dictionary saved to 'entity_label_dict.csv'\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Convert the dictionary to a DataFrame\n",
    "entity_label_df = pd.DataFrame(list(entity_label_dict.items()), columns=['Entity Text', 'Entity Label'])\n",
    "\n",
    "# Save the DataFrame to a CSV file\n",
    "entity_label_df.to_csv('output_from_using_REBEL/entity_label_dict.csv', index=False)\n",
    "\n",
    "print(\"Entity label dictionary saved to 'entity_label_dict.csv'\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 15866 entries, 0 to 15865\n",
      "Data columns (total 5 columns):\n",
      " #   Column        Non-Null Count  Dtype \n",
      "---  ------        --------------  ----- \n",
      " 0   subject       15866 non-null  object\n",
      " 1   relationship  15866 non-null  object\n",
      " 2   object        15864 non-null  object\n",
      " 3   chunk         15866 non-null  object\n",
      " 4   sentence      15866 non-null  object\n",
      "dtypes: object(5)\n",
      "memory usage: 619.9+ KB\n"
     ]
    }
   ],
   "source": [
    "triples_df = pd.read_csv('output_from_using_REBEL/rebel_triples_sentences.csv')\n",
    "triples_df.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Updated triples with labels saved to 'output_from_using_REBEL/rebel_triples_labels.csv'\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the triples and entity label data\n",
    "triples_df = pd.read_csv('output_from_using_REBEL/rebel_triples_sentences.csv')\n",
    "entity_label_df = pd.read_csv('output_from_using_REBEL/entity_label_dict.csv')\n",
    "\n",
    "# Create a dictionary from the entity label DataFrame\n",
    "entity_label_dict = {row[\"Entity Text\"].strip().lower(): row[\"Entity Label\"].strip() for _, row in entity_label_df.iterrows()}\n",
    "\n",
    "# Function to get the entity label\n",
    "def get_entity_label(entity):\n",
    "    return entity_label_dict.get(entity.strip().lower(), 'unknown')\n",
    "\n",
    "# Fill NaN values with an empty string\n",
    "triples_df['subject'] = triples_df['subject'].fillna('')\n",
    "triples_df['object'] = triples_df['object'].fillna('')\n",
    "\n",
    "# Add the labels to the triples DataFrame\n",
    "triples_df['subject_label'] = triples_df['subject'].apply(get_entity_label)\n",
    "triples_df['object_label'] = triples_df['object'].apply(get_entity_label)\n",
    "\n",
    "# Save the updated DataFrame to a new CSV file\n",
    "triples_df.to_csv('output_from_using_REBEL/rebel_triples_labels.csv', index=False)\n",
    "\n",
    "print(\"Updated triples with labels saved to 'output_from_using_REBEL/rebel_triples_labels.csv'\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>subject</th>\n",
       "      <th>relationship</th>\n",
       "      <th>object</th>\n",
       "      <th>chunk</th>\n",
       "      <th>sentence</th>\n",
       "      <th>subject_label</th>\n",
       "      <th>object_label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Rhinovirus</td>\n",
       "      <td>has effect</td>\n",
       "      <td>upper respiratory tract infections</td>\n",
       "      <td>['BACKGROUND: Rhinovirus, the most common caus...</td>\n",
       "      <td>Rhinovirus has effect upper respiratory tract ...</td>\n",
       "      <td>virus</td>\n",
       "      <td>disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>aerosol transmission</td>\n",
       "      <td>facet of</td>\n",
       "      <td>rhinovirus</td>\n",
       "      <td>['BACKGROUND: Rhinovirus, the most common caus...</td>\n",
       "      <td>aerosol transmission facet of rhinovirus.</td>\n",
       "      <td>transmission</td>\n",
       "      <td>virus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>detecting infectious agents</td>\n",
       "      <td>studies</td>\n",
       "      <td>detecting aerosolized rhinovirus</td>\n",
       "      <td>['As a first step in studies of possible airbo...</td>\n",
       "      <td>detecting infectious agents studies detecting ...</td>\n",
       "      <td>unknown</td>\n",
       "      <td>unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>aerosol chamber</td>\n",
       "      <td>use</td>\n",
       "      <td>aerosol</td>\n",
       "      <td>['As a first step in studies of possible airbo...</td>\n",
       "      <td>aerosol chamber use aerosol.</td>\n",
       "      <td>unknown</td>\n",
       "      <td>environment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rhinoviral</td>\n",
       "      <td>subclass of</td>\n",
       "      <td>aerosols</td>\n",
       "      <td>['To determine the effect of UV irradiation on...</td>\n",
       "      <td>rhinoviral subclass of aerosols.</td>\n",
       "      <td>unknown</td>\n",
       "      <td>substance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15861</th>\n",
       "      <td>GeoDRM</td>\n",
       "      <td>facet of</td>\n",
       "      <td>Web-based geospatial technologies</td>\n",
       "      <td>['Such developments are of interest to the geo...</td>\n",
       "      <td>GeoDRM facet of Web-based geospatial technolog...</td>\n",
       "      <td>method</td>\n",
       "      <td>method</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15862</th>\n",
       "      <td>OGC GeoDRM Working Group</td>\n",
       "      <td>field of work</td>\n",
       "      <td>digital rights management</td>\n",
       "      <td>['The mission of OGC GeoDRM Working Group is t...</td>\n",
       "      <td>OGC GeoDRM Working Group field of work digital...</td>\n",
       "      <td>organization</td>\n",
       "      <td>method</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15863</th>\n",
       "      <td>Google</td>\n",
       "      <td>product or material produced</td>\n",
       "      <td>maps</td>\n",
       "      <td>['The mission of OGC GeoDRM Working Group is t...</td>\n",
       "      <td>Google product or material produced maps.</td>\n",
       "      <td>organization</td>\n",
       "      <td>location</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15864</th>\n",
       "      <td>map</td>\n",
       "      <td>part of</td>\n",
       "      <td>visual language of geography</td>\n",
       "      <td>['Building on the powerful and universal visua...</td>\n",
       "      <td>map part of visual language of geography.</td>\n",
       "      <td>environment</td>\n",
       "      <td>unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15865</th>\n",
       "      <td>Google</td>\n",
       "      <td>subsidiary</td>\n",
       "      <td>MSN</td>\n",
       "      <td>['Building on the powerful and universal visua...</td>\n",
       "      <td>Google subsidiary MSN.</td>\n",
       "      <td>organization</td>\n",
       "      <td>organization</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>15866 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           subject                  relationship  \\\n",
       "0                       Rhinovirus                    has effect   \n",
       "1             aerosol transmission                      facet of   \n",
       "2      detecting infectious agents                       studies   \n",
       "3                  aerosol chamber                           use   \n",
       "4                       rhinoviral                   subclass of   \n",
       "...                            ...                           ...   \n",
       "15861                       GeoDRM                      facet of   \n",
       "15862     OGC GeoDRM Working Group                 field of work   \n",
       "15863                       Google  product or material produced   \n",
       "15864                          map                       part of   \n",
       "15865                       Google                    subsidiary   \n",
       "\n",
       "                                   object  \\\n",
       "0      upper respiratory tract infections   \n",
       "1                              rhinovirus   \n",
       "2        detecting aerosolized rhinovirus   \n",
       "3                                 aerosol   \n",
       "4                                aerosols   \n",
       "...                                   ...   \n",
       "15861   Web-based geospatial technologies   \n",
       "15862           digital rights management   \n",
       "15863                                maps   \n",
       "15864        visual language of geography   \n",
       "15865                                 MSN   \n",
       "\n",
       "                                                   chunk  \\\n",
       "0      ['BACKGROUND: Rhinovirus, the most common caus...   \n",
       "1      ['BACKGROUND: Rhinovirus, the most common caus...   \n",
       "2      ['As a first step in studies of possible airbo...   \n",
       "3      ['As a first step in studies of possible airbo...   \n",
       "4      ['To determine the effect of UV irradiation on...   \n",
       "...                                                  ...   \n",
       "15861  ['Such developments are of interest to the geo...   \n",
       "15862  ['The mission of OGC GeoDRM Working Group is t...   \n",
       "15863  ['The mission of OGC GeoDRM Working Group is t...   \n",
       "15864  ['Building on the powerful and universal visua...   \n",
       "15865  ['Building on the powerful and universal visua...   \n",
       "\n",
       "                                                sentence subject_label  \\\n",
       "0      Rhinovirus has effect upper respiratory tract ...         virus   \n",
       "1              aerosol transmission facet of rhinovirus.  transmission   \n",
       "2      detecting infectious agents studies detecting ...       unknown   \n",
       "3                           aerosol chamber use aerosol.       unknown   \n",
       "4                       rhinoviral subclass of aerosols.       unknown   \n",
       "...                                                  ...           ...   \n",
       "15861  GeoDRM facet of Web-based geospatial technolog...        method   \n",
       "15862  OGC GeoDRM Working Group field of work digital...  organization   \n",
       "15863          Google product or material produced maps.  organization   \n",
       "15864          map part of visual language of geography.   environment   \n",
       "15865                             Google subsidiary MSN.  organization   \n",
       "\n",
       "       object_label  \n",
       "0           disease  \n",
       "1             virus  \n",
       "2           unknown  \n",
       "3       environment  \n",
       "4         substance  \n",
       "...             ...  \n",
       "15861        method  \n",
       "15862        method  \n",
       "15863      location  \n",
       "15864       unknown  \n",
       "15865  organization  \n",
       "\n",
       "[15866 rows x 7 columns]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "triples_labels = pd.read_csv(\"output_from_using_REBEL/rebel_triples_labels.csv\")\n",
    "triples_labels"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>subject</th>\n",
       "      <th>relationship</th>\n",
       "      <th>object</th>\n",
       "      <th>chunk</th>\n",
       "      <th>sentence</th>\n",
       "      <th>subject_label</th>\n",
       "      <th>object_label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>15866</td>\n",
       "      <td>15866</td>\n",
       "      <td>15864</td>\n",
       "      <td>15866</td>\n",
       "      <td>15866</td>\n",
       "      <td>15866</td>\n",
       "      <td>15866</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unique</th>\n",
       "      <td>8384</td>\n",
       "      <td>107</td>\n",
       "      <td>6630</td>\n",
       "      <td>7325</td>\n",
       "      <td>13686</td>\n",
       "      <td>32</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>top</th>\n",
       "      <td>genome</td>\n",
       "      <td>subclass of</td>\n",
       "      <td>protein</td>\n",
       "      <td>['To identify RBD amino acid residues that mig...</td>\n",
       "      <td>John F. Kennedy School of Government part of H...</td>\n",
       "      <td>unknown</td>\n",
       "      <td>unknown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freq</th>\n",
       "      <td>236</td>\n",
       "      <td>4554</td>\n",
       "      <td>281</td>\n",
       "      <td>12</td>\n",
       "      <td>39</td>\n",
       "      <td>4934</td>\n",
       "      <td>3379</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       subject relationship   object  \\\n",
       "count    15866        15866    15864   \n",
       "unique    8384          107     6630   \n",
       "top     genome  subclass of  protein   \n",
       "freq       236         4554      281   \n",
       "\n",
       "                                                    chunk  \\\n",
       "count                                               15866   \n",
       "unique                                               7325   \n",
       "top     ['To identify RBD amino acid residues that mig...   \n",
       "freq                                                   12   \n",
       "\n",
       "                                                 sentence subject_label  \\\n",
       "count                                               15866         15866   \n",
       "unique                                              13686            32   \n",
       "top     John F. Kennedy School of Government part of H...       unknown   \n",
       "freq                                                   39          4934   \n",
       "\n",
       "       object_label  \n",
       "count         15866  \n",
       "unique           32  \n",
       "top         unknown  \n",
       "freq           3379  "
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "triples_labels.describe()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## LLM refinement"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Predicting labels for unknown subjects:   0%|          | 0/4934 [00:00<?, ?it/s]/var/folders/dr/fn2_r29930s6nq_wqwxq_ckm0000gn/T/ipykernel_7900/664957814.py:39: LangChainDeprecationWarning: The method `BaseChatModel.__call__` was deprecated in langchain-core 0.1.7 and will be removed in 1.0. Use :meth:`~invoke` instead.\n",
      "  response = llm(\n",
      "Predicting labels for unknown subjects: 100%|██████████| 4934/4934 [1:56:36<00:00,  1.42s/it]  \n",
      "Predicting labels for unknown objects: 100%|██████████| 3379/3379 [1:18:40<00:00,  1.40s/it]\n"
     ]
    }
   ],
   "source": [
    "from langchain_ollama.chat_models import ChatOllama\n",
    "from langchain.prompts import SystemMessagePromptTemplate, HumanMessagePromptTemplate\n",
    "from langchain.schema import AIMessage, HumanMessage, SystemMessage\n",
    "from tqdm import tqdm\n",
    "\n",
    "# Initialize the Ollama model\n",
    "llm = ChatOllama(model=\"qwen2.5\", temperature=0, format=\"json\")\n",
    "\n",
    "\n",
    "# Function to predict the label using LLM\n",
    "def predict_label(text, chunk):\n",
    "    system_message_template = \"\"\"\n",
    "        Predict the label for the following entity based on the given context. \n",
    "        The label must be chosen strictly from the predefined list below. Return a JSON object, \n",
    "        strictly formatted as valid JSON. Do not add any text, explanations, or comments outside the JSON.\n",
    "\n",
    "        Labels: [\n",
    "            \"virus\", \"disease\", \"symptom\", \"drug\", \"vaccine\", \n",
    "            \"gene\", \"protein\", \"diagnostic\", \"treatment\", \"transmission\",\n",
    "            \"epidemiology\", \"public health\", \"host\", \"pathway\", \"immune response\",\n",
    "            \"clinical trial\", \"organization\", \"measurement\", \"publication\", \"environment\", \n",
    "            \"country\", \"region\", \"city\", \"person\", \"location\", \"event\", \"method\"\n",
    "            ]\n",
    "\n",
    "        For example: \n",
    "        [\n",
    "            {{\"entity\": \"Eiffel Tower\", \"label\": \"location\"}},\n",
    "            {{\"entity\": \"Albert Einstein\", \"label\": \"person\"}},\n",
    "            {{\"entity\": \"COVID-19\", \"label\": \"virus\"}}\n",
    "        ]\n",
    "        Output must be valid JSON with no additional text.\n",
    "\n",
    "        Context: \\n\\n{chunk}\n",
    "        Entity: \\n\\n{entity} \n",
    "    \"\"\"\n",
    "    system_message = system_message_template.format(entity=text, chunk=chunk)\n",
    "    user_message = f\"Entity: {text}\\n\\nLabel:\"\n",
    "\n",
    "    response = llm(\n",
    "        [SystemMessage(content=system_message), HumanMessage(content=user_message)]\n",
    "    )\n",
    "    return response.content.strip()\n",
    "\n",
    "\n",
    "# Filter rows with unknown labels\n",
    "unknown_subjects = triples_labels[triples_labels[\"subject_label\"] == \"unknown\"]\n",
    "unknown_objects = triples_labels[triples_labels[\"object_label\"] == \"unknown\"]\n",
    "\n",
    "subject_labels = []\n",
    "object_labels = []\n",
    "\n",
    "# Predict labels for unknown subjects\n",
    "for index, row in tqdm(\n",
    "    unknown_subjects.iterrows(),\n",
    "    total=unknown_subjects.shape[0],\n",
    "    desc=\"Predicting labels for unknown subjects\",\n",
    "):\n",
    "    predicted_label = predict_label(row[\"subject\"], row[\"chunk\"])\n",
    "    subject_labels.append(predicted_label)\n",
    "\n",
    "# Predict labels for unknown objects\n",
    "for index, row in tqdm(\n",
    "    unknown_objects.iterrows(),\n",
    "    total=unknown_objects.shape[0],\n",
    "    desc=\"Predicting labels for unknown objects\",\n",
    "):\n",
    "    predicted_label = predict_label(row[\"object\"], row[\"chunk\"])\n",
    "    object_labels.append(predicted_label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Subject Labels Dicts: [{'entity': 'detecting infectious agents', 'label': 'method'}, {'entity': 'aerosol chamber', 'label': 'method'}, {'entity': 'rhinoviral', 'label': 'virus'}, {'entity': 'TCID(50)', 'label': 'measurement'}, {'entity': 'germicidal', 'label': 'virus'}, {'entity': 'infectious viral particles', 'label': 'virus'}, {'entity': 'autumn outbreak period', 'label': 'event'}, {'entity': 'one experimental study', 'label': 'publication'}, {'entity': 'ultraviolet germicidal irradiation', 'label': 'method'}, {'entity': 'bacilli Calmette-Guérin', 'label': 'vaccine'}, {'entity': 'drying jar', 'label': 'device'}, {'entity': 'Collison', 'label': 'device'}, {'entity': 'UV exposure chamber', 'label': 'equipment'}, {'entity': 'filters', 'label': 'method'}, {'entity': 'Teflon filter', 'label': 'diagnostic'}, {'entity': 'bypass filter', 'label': 'device'}, {'entity': 'semi-nested', 'label': 'method'}, {'entity': 'spiked', 'label': 'vaccine'}, {'entity': 'single pass air', 'label': 'environment'}, {'entity': 'Qiagen QIAamp Viral RNA Mini Kit', 'label': 'vaccine'}, {'entity': 'microfuge', 'label': 'device'}, {'entity': 'Invitrogen SuperScript', 'label': 'method'}, {'entity': 'uranine analysis', 'label': 'method'}, {'entity': '100 mm test tube', 'label': 'measurement'}, {'entity': 'Mannedorf', 'label': 'location'}, {'entity': 'dilution gradient filter', 'label': 'method'}, {'entity': 'TCID 50', 'label': 'measurement'}, {'entity': 'extraction and detection assay', 'label': 'diagnostic'}, {'entity': 'airborne infectious droplet nuclei', 'label': 'transmission'}, {'entity': 'air sampling', 'label': 'method'}, {'entity': 'enteroviral aerosols', 'label': 'virus'}, {'entity': 'Collison', 'label': 'device'}, {'entity': 'nebulization solution', 'label': 'method'}, {'entity': 'spiking experiments', 'label': 'clinical trial'}, {'entity': 'drying jar', 'label': 'location'}, {'entity': '1966', 'label': 'publication'}, {'entity': 'National Institute of Allergy and Infectious Disease', 'label': 'organization'}, {'entity': 'TCID 50', 'label': 'measurement'}, {'entity': 'aerosol data from 1966', 'label': 'diagnostic'}, {'entity': 'spiking experiment', 'label': 'method'}, {'entity': 'filter collection', 'label': 'method'}, {'entity': 'UV irradiated aerosols', 'label': 'detection'}, {'entity': 'nonexposed aerosols', 'label': 'disease'}, {'entity': 'hypothesized pattern', 'label': 'diagnosis'}, {'entity': 'evolutionarily significant', 'label': 'vocabulary'}, {'entity': 'Claw', 'label': 'structure'}, {'entity': 'unique', 'label': 'virus'}, {'entity': 'vaccine research', 'label': 'clinical trial'}, {'entity': 'If H.', 'label': 'virus'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'different from each other', 'label': 'clinical trial'}, {'entity': 'clusters', 'label': 'method'}, {'entity': 'swept back and forth', 'label': 'event'}, {'entity': 'colonial conquest', 'label': 'event'}, {'entity': 'The relationships between human populations', 'label': 'host'}, {'entity': 'tree', 'label': 'method'}, {'entity': 'clusters', 'label': 'pathway'}, {'entity': 'network of relationships', 'label': 'pathway'}, {'entity': 'expansion', 'label': 'event'}, {'entity': 'intraspecific', 'label': 'gene'}, {'entity': 'multifocal leukoencephalopathy', 'label': 'disease'}, {'entity': 'subtypes', 'label': 'virus'}, {'entity': 'evolutionary history', 'label': 'epidemiology'}, {'entity': 'human evolution and history', 'label': 'event'}, {'entity': 'JCV subtypes', 'label': 'virus'}, {'entity': '[7]', 'label': 'publication'}, {'entity': 'Taiwan epidemic', 'label': 'event'}, {'entity': 'Taiwan indigenous peoples', 'label': 'host'}, {'entity': 'Taiwanese general population', 'label': 'population'}, {'entity': 'Taiwan was stroked', 'label': 'region'}, {'entity': 'Taipei Municipal Hoping hospital', 'label': 'organization'}, {'entity': 'health care workers', 'label': 'person'}, {'entity': 'health care workers', 'label': 'person'}, {'entity': 'disease protection phenotype', 'label': 'trait'}, {'entity': 'probable SARS', 'label': 'disease'}, {'entity': 'excluded fever patients', 'label': 'disease'}, {'entity': 'SARS-CoV RNA test', 'label': 'diagnostic'}, {'entity': 'Binax', 'label': 'organization'}, {'entity': 'suspected probable case', 'label': 'disease'}, {'entity': 'rapid severe', 'label': 'symptom'}, {'entity': 'probable SARS', 'label': 'disease'}, {'entity': 'Excluded fever patients', 'label': 'disease'}, {'entity': 'vigorous measures against infection', 'label': 'treatment'}, {'entity': 'non-infected', 'label': 'disease'}, {'entity': 'allele typing', 'label': 'method'}, {'entity': 'Wirral', 'label': 'location'}, {'entity': 'class I', 'label': 'gene'}, {'entity': \"Woolf's method\", 'label': 'method'}, {'entity': \"Fisher's exact test\", 'label': 'method'}, {'entity': 'type I error', 'label': 'measurement'}, {'entity': 'corrected P-value', 'label': 'measurement'}, {'entity': 'table of correspondence', 'label': 'method'}, {'entity': 'SARS coronavirus infection', 'label': 'disease'}, {'entity': 'B* 3901/B39', 'label': 'gene'}, {'entity': 'central mountain area', 'label': 'location'}, {'entity': 'coronavirus infection', 'label': 'disease'}, {'entity': 'Hong River', 'label': 'location'}, {'entity': 'Taiwan indigenous', 'label': 'person'}, {'entity': 'Taiwan indigenous peoples', 'label': 'person'}, {'entity': 'HLA-B* 4601/B46', 'label': 'gene'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'urbanized', 'label': 'location'}, {'entity': 'tropism', 'label': 'virus'}, {'entity': 'first milder outbreak', 'label': 'event'}, {'entity': 'first epidemic', 'label': 'event'}, {'entity': 'Kow-loon', 'label': 'location'}, {'entity': 'SAR Hong Kong', 'label': 'location'}, {'entity': 'estate', 'label': 'location'}, {'entity': 'pattern', 'label': 'epidemiology'}, {'entity': 'hypothesis of the double epidemic model for SARS', 'label': 'clinical trial'}, {'entity': 'Another epidemic', 'label': 'epidemiology'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'SARS in China', 'label': 'disease'}, {'entity': 'outbreak of SARS', 'label': 'event'}, {'entity': 'isolated', 'label': 'virus'}, {'entity': 'class', 'label': 'vocabulary'}, {'entity': 'admitted', 'label': 'virus'}, {'entity': 'nonlinear', 'label': 'method'}, {'entity': 'control parameters', 'label': 'parameter'}, {'entity': 'initial value', 'label': 'measurement'}, {'entity': 'final value', 'label': 'measurement'}, {'entity': 'exponentially slow', 'label': 'measurement'}, {'entity': 'critical point', 'label': 'treatment'}, {'entity': 'SARS in Hong Kong', 'label': 'disease'}, {'entity': 'general shape of a graph', 'label': 'diagram'}, {'entity': 'Typical dynamics for the SIR model', 'label': 'diagram'}, {'entity': 'curves', 'label': 'diagram'}, {'entity': 'parameters estimations', 'label': 'measurement'}, {'entity': 'case', 'label': 'disease'}, {'entity': 'invalidates', 'label': 'clinical trial'}, {'entity': 'effective population', 'label': 'measurement'}, {'entity': 'different, but similar, disease that spread before', 'label': 'disease'}, {'entity': 'SEIRP model', 'label': 'model'}, {'entity': 'outbreak of infections', 'label': 'epidemiology'}, {'entity': 'double epidemic interaction', 'label': 'epidemiology'}, {'entity': 'non-linear', 'label': 'method'}, {'entity': 'directly transmitted diseases', 'label': 'disease'}, {'entity': 'classes', 'label': 'clinical trial'}, {'entity': 'Exposed class E', 'label': 'disease'}, {'entity': 'transmit it', 'label': 'transmission'}, {'entity': 'Removed class R', 'label': 'clinical trial'}, {'entity': 'class I P', 'label': 'clinical trial'}, {'entity': 'class R P', 'label': 'clinical trial'}, {'entity': 'disease B', 'label': 'disease'}, {'entity': 'disease A', 'label': 'disease'}, {'entity': 'class I', 'label': 'clinical trial'}, {'entity': 'class E', 'label': 'clinical trial'}, {'entity': 'I(t)', 'label': 'variable'}, {'entity': 'nonlinear', 'label': 'method'}, {'entity': 'transmit infection', 'label': 'transmission'}, {'entity': 'non-negative', 'label': 'measurement'}, {'entity': 'indirect evidence', 'label': 'diagnostic'}, {'entity': 'epidemic A', 'label': 'disease'}, {'entity': 'protective epidemic', 'label': 'disease'}, {'entity': 'Susceptibles', 'label': 'virus'}, {'entity': 'Dynamic', 'label': 'method'}, {'entity': 'case a', 'label': 'event'}, {'entity': 'outbreak of SARS', 'label': 'event'}, {'entity': 'outbreak of SARS in Hong Kong', 'label': 'event'}, {'entity': 'disease B', 'label': 'disease'}, {'entity': 'population of Hong Kong', 'label': 'measurement'}, {'entity': '2003', 'label': 'year'}, {'entity': '1011', 'label': 'measurement'}, {'entity': '11.83', 'label': 'measurement'}, {'entity': 'S(0)', 'label': 'measurement'}, {'entity': 'confirmed cases of SARS', 'label': 'disease'}, {'entity': '1033', 'label': 'measurement'}, {'entity': 'outbreak of SARS', 'label': 'event'}, {'entity': 'SARS cases in Beijing', 'label': 'disease'}, {'entity': 'E(0)', 'label': 'measurement'}, {'entity': 'outbreak of SARS in Inner Mongolia', 'label': 'event'}, {'entity': 'E(0)', 'label': 'measurement'}, {'entity': 'I P', 'label': 'parameter'}, {'entity': 'class E', 'label': 'diagnosis'}, {'entity': 'class I', 'label': 'disease'}, {'entity': 'b parameter', 'label': 'measurement'}, {'entity': 'incubation period', 'label': 'measurement'}, {'entity': 'different scenarios', 'label': 'event'}, {'entity': 'protective epidemic', 'label': 'disease'}, {'entity': 'severe draught', 'label': 'environment'}, {'entity': 'Severe Acute Respiratory Syndrome outbreak', 'label': 'event'}, {'entity': 'next autumn', 'label': 'event'}, {'entity': 'next year', 'label': 'time'}, {'entity': 'descriptive', 'label': 'method'}, {'entity': 'hypotheses', 'label': 'method'}, {'entity': 'The Model', 'label': 'method'}, {'entity': 'Susceptible-Infected-Removed (SIR) epidemic model', 'label': 'epidemiology'}, {'entity': 'trailing', 'label': 'event'}, {'entity': 'unscientific', 'label': 'word'}, {'entity': 'causes', 'label': 'epidemiology'}, {'entity': 'Ptolemean system', 'label': 'method'}, {'entity': 'continuous model', 'label': 'method'}, {'entity': 'stochastic parameters', 'label': 'method'}, {'entity': 'outbreak of SARS in Hong Kong', 'label': 'event'}, {'entity': 'provisional hypothesis', 'label': 'clinical trial'}, {'entity': 'Observed cummulative cases per day Predicted cummulative cases data', 'label': 'measurement'}, {'entity': 'under control', 'label': 'treatment'}, {'entity': 'Exposed(E)', 'label': 'epidemiology'}, {'entity': 'tail', 'label': 'symptom'}, {'entity': 'identifying patients', 'label': 'method'}, {'entity': 'mild infection', 'label': 'symptom'}, {'entity': 'vaccination against coronavirus', 'label': 'vaccine'}, {'entity': 'variety of virulence', 'label': 'disease'}, {'entity': 'Our model', 'label': 'clinical trial'}, {'entity': 'Amoy Gardens Estate outbreak', 'label': 'event'}, {'entity': 'simple model', 'label': 'model'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': 'core structure', 'label': 'protein'}, {'entity': 'Figure 1 Wheel projection of the N-HR', 'label': 'diagram'}, {'entity': 'amino acid sequence', 'label': 'protein'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': 'loop structure', 'label': 'protein'}, {'entity': 'C-terminus', 'label': 'protein'}, {'entity': 'structural motifs', 'label': 'protein'}, {'entity': 'Spartan scytale', 'label': 'device'}, {'entity': 'letter', 'label': 'method'}, {'entity': 'C34', 'label': 'drug'}, {'entity': 'C34', 'label': 'drug'}, {'entity': 'SARS-CoV E2 precursor protein accession number', 'label': 'protein'}, {'entity': 'drafted', 'label': 'method'}, {'entity': 'neighbor-joining algorithm', 'label': 'method'}, {'entity': 'embedded tetra-nucleotide usage', 'label': 'method'}, {'entity': 'virus classification', 'label': 'disease'}, {'entity': 'Transmissible Gastroenteritis Virus', 'label': 'virus'}, {'entity': 'Porcine reproductive and Respiratory syndrome Virus', 'label': 'virus'}, {'entity': 'di-nucleotide bias in six RNA viruses', 'label': 'measurement'}, {'entity': 'N(w) and E(w)', 'label': 'measurement'}, {'entity': 'Tetra-nucleotide bias', 'label': 'measurement'}, {'entity': 'tetra-nucleotide bias', 'label': 'measurement'}, {'entity': 'N(w) and E(w)', 'label': 'measurement'}, {'entity': 'distance tree', 'label': 'method'}, {'entity': 'tetra-nucleotide usage pattern', 'label': 'measurement'}, {'entity': 'negative strand', 'label': 'virus'}, {'entity': '3 rd', 'label': 'measurement'}, {'entity': 'tetra-nucleotide usage pattern', 'label': 'measurement'}, {'entity': 'word usage', 'label': 'virus'}, {'entity': 'Transmissible Gastroenteritis Virus', 'label': 'virus'}, {'entity': 'usage pattern', 'label': 'virus'}, {'entity': 'order', 'label': 'method'}, {'entity': 'different species', 'label': 'virus'}, {'entity': 'background properties', 'label': 'measurement'}, {'entity': 'middle figure', 'label': 'diagram'}, {'entity': 'top figure', 'label': 'figure'}, {'entity': 'Sun Fire', 'label': 'organization'}, {'entity': 'Matlab', 'label': 'tool'}, {'entity': 'UAU-GAGAGAUCCGAGA', 'label': 'gene'}, {'entity': 'word', 'label': 'virus'}, {'entity': 'V', 'label': 'virus'}, {'entity': 'pros', 'label': 'method'}, {'entity': 'word set', 'label': 'measurement'}, {'entity': 'sub-word', 'label': 'term'}, {'entity': 'bias of word with sizes less than n', 'label': 'measurement'}, {'entity': 'word size m -1', 'label': 'measurement'}, {'entity': 'Markov model', 'label': 'method'}, {'entity': 'quantifying', 'label': 'method'}, {'entity': 'tetra-nucleotide bias', 'label': 'measurement'}, {'entity': 'bias', 'label': 'measurement'}, {'entity': 'Flowchart for studying the tetra-nucleotide usage pattern', 'label': 'diagram'}, {'entity': 'neighbor joining', 'label': 'method'}, {'entity': 'linear relationship', 'label': 'measurement'}, {'entity': 'perfect linearity', 'label': 'measurement'}, {'entity': 'positive linearity', 'label': 'measurement'}, {'entity': 'Positive magnitude of r', 'label': 'measurement'}, {'entity': 'negative magnitude of r', 'label': 'measurement'}, {'entity': 'triangular matrix', 'label': 'measurement'}, {'entity': 'factors', 'label': 'method'}, {'entity': 'scatter-plot', 'label': 'method'}, {'entity': 'first eigen-vector', 'label': 'measurement'}, {'entity': 'subsequent eigen-vectors', 'label': 'method'}, {'entity': 'Matlab', 'label': 'tool'}, {'entity': 'pathogen discovery', 'label': 'treatment'}, {'entity': 'unsequenced', 'label': 'virus'}, {'entity': 'known', 'label': 'virus'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'transmissible gastroenteritis virus', 'label': 'virus'}, {'entity': 'hybridized', 'label': 'virus'}, {'entity': 'viral nucleic acid', 'label': 'virus'}, {'entity': 'hybridized', 'label': 'diagnostic'}, {'entity': 'NucliSens', 'label': 'method'}, {'entity': 'extractor workstation', 'label': 'equipment'}, {'entity': 'eluate', 'label': 'measurement'}, {'entity': 'hybridized', 'label': 'method'}, {'entity': '70mer', 'label': 'gene'}, {'entity': 'Cy5', 'label': 'measurement'}, {'entity': '1500 intensity units', 'label': 'measurement'}, {'entity': 'array element', 'label': 'method'}, {'entity': 'sequencing cocktail', 'label': 'method'}, {'entity': 'classical late-infantile neuronal ceroid lipofuscinosis', 'label': 'disease'}, {'entity': 'three-dimensional', 'label': 'measurement'}, {'entity': 'life cycle', 'label': 'treatment'}, {'entity': 'binding pocket', 'label': 'protein'}, {'entity': 'classical late-infantile neuronal ceroid lipofuscinosis', 'label': 'disease'}, {'entity': 'physiological role', 'label': 'measurement'}, {'entity': 'crystallization', 'label': 'method'}, {'entity': 'molecular modeling', 'label': 'method'}, {'entity': 'molecular homology modeling', 'label': 'method'}, {'entity': 'nucleotide pair', 'label': 'gene'}, {'entity': 'fungal', 'label': 'virus'}, {'entity': 'prosegment', 'label': 'protein'}, {'entity': 'fish', 'label': 'organism'}, {'entity': 'protein modeling', 'label': 'method'}, {'entity': 'r.m.s.', 'label': 'measurement'}, {'entity': 'Corrected gene sequence of the zebrafish CLN2', 'label': 'gene'}, {'entity': 'steric barrier', 'label': 'protein'}, {'entity': 'P1 residue', 'label': 'protein'}, {'entity': 'Sequences of the catalytic domains of CLN2', 'label': 'gene'}, {'entity': 'green', 'label': 'color'}, {'entity': 'S1 pocket', 'label': 'protein'}, {'entity': 'S2 pocket', 'label': 'protein'}, {'entity': 'Figure 4 Sequence alignment of bacterial and mammalian enzymes', 'label': 'publication'}, {'entity': 'A homology-derived model of human CLN2 Figure 5', 'label': 'diagram'}, {'entity': 'Cα trace', 'label': 'protein'}, {'entity': 'mutated', 'label': 'gene'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': 'lipofusci', 'label': 'disease'}, {'entity': 'truncated enzyme', 'label': 'protein'}, {'entity': 'catalytic', 'label': 'protein'}, {'entity': 'truncations', 'label': 'gene'}, {'entity': 'P1', 'label': 'protein'}, {'entity': 'DINO', 'label': 'method'}, {'entity': 'Arg', 'label': 'protein'}, {'entity': 'glucagon', 'label': 'protein'}, {'entity': 'cholecystokinin', 'label': 'protein'}, {'entity': 'type I collagen', 'label': 'protein'}, {'entity': 'catalytic', 'label': 'protein'}, {'entity': 'free N termini', 'label': 'protein'}, {'entity': 'crystallize', 'label': 'method'}, {'entity': 'subfamilies', 'label': 'protein'}, {'entity': 'DeepView', 'label': 'software'}, {'entity': 'covalent geometry', 'label': 'measurement'}, {'entity': 'PROCHECK', 'label': 'software'}, {'entity': 'SARS case fatality', 'label': 'virus'}, {'entity': 'ground-level ozone', 'label': 'measurement'}, {'entity': 'long-term', 'label': 'treatment'}, {'entity': 'low APIs', 'label': 'measurement'}, {'entity': 'high APIs', 'label': 'measurement'}, {'entity': 'air pollution indices', 'label': 'measurement'}, {'entity': 'uncontrolled confounding effect', 'label': 'measurement'}, {'entity': 'ground-level ozone', 'label': 'measurement'}, {'entity': '200', 'label': 'measurement'}, {'entity': 'April', 'label': 'month'}, {'entity': 'SARS case fatality', 'label': 'disease'}, {'entity': 'fitted', 'label': 'method'}, {'entity': 'short-term exposure', 'label': 'event'}, {'entity': 'high API', 'label': 'measurement'}, {'entity': 'low API', 'label': 'measurement'}, {'entity': 'recovered', 'label': 'diagnostic'}, {'entity': '2003', 'label': 'year'}, {'entity': 'Shanxi', 'label': 'region'}, {'entity': 'SARS fatality', 'label': 'disease'}, {'entity': 'moderate API', 'label': 'measurement'}, {'entity': 'University of California, Berkeley', 'label': 'organization'}, {'entity': 'different in medical facilities', 'label': 'measurement'}, {'entity': 'Exposure misclassification', 'label': 'measurement'}, {'entity': 'air pollution indices', 'label': 'measurement'}, {'entity': '2003 SARS pandemic', 'label': 'event'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'medical leave', 'label': 'vaccine'}, {'entity': 'bus', 'label': 'transmission'}, {'entity': 'cafes', 'label': 'location'}, {'entity': 'surgical mask', 'label': 'vaccine'}, {'entity': 'mid-semester', 'label': 'event'}, {'entity': 'Singapore government', 'label': 'organization'}, {'entity': 'electronic tag', 'label': 'device'}, {'entity': '2003', 'label': 'year'}, {'entity': '1950s', 'label': 'year'}, {'entity': 'stress', 'label': 'symptom'}, {'entity': 'groping', 'label': 'symptom'}, {'entity': 'Michael Smith Genome Sciences Centre', 'label': 'organization'}, {'entity': 'SARS-CoV diagnostic kit', 'label': 'diagnostic'}, {'entity': 'SARS pandemic', 'label': 'virus'}, {'entity': 'ubiquitin associated', 'label': 'protein'}, {'entity': 'Ub-Ub conjugation', 'label': 'protein'}, {'entity': 'proteasome', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'pathway'}, {'entity': 'endocytic pathway', 'label': 'pathway'}, {'entity': 'sequestosome', 'label': 'protein'}, {'entity': 'polyubiquitinated', 'label': 'gene'}, {'entity': 'Gold TNT SP6 Express 96 plate', 'label': 'method'}, {'entity': 'En 3 HANCE', 'label': 'method'}, {'entity': 'LB media', 'label': 'measurement'}, {'entity': 'TNT Quick', 'label': 'method'}, {'entity': 'secondary antibody', 'label': 'protein'}, {'entity': 'ProteoLink IVEC system', 'label': 'method'}, {'entity': 'mixed pool', 'label': 'protein'}, {'entity': '3B', 'label': 'figure'}, {'entity': 'p62 UBA', 'label': 'protein'}, {'entity': '3C', 'label': 'event'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'LB media', 'label': 'measurement'}, {'entity': 'TNT Quick', 'label': 'method'}, {'entity': 'mixed TNT reaction', 'label': 'method'}, {'entity': 'individual TNT reaction', 'label': 'method'}, {'entity': 'β-amyloid', 'label': 'protein'}, {'entity': \"Alzheimer's type brain\", 'label': 'disease'}, {'entity': 'hindbrain', 'label': 'protein'}, {'entity': '6', 'label': 'number'}, {'entity': 'non-ubiquitinated', 'label': 'protein'}, {'entity': 'binding partner', 'label': 'protein'}, {'entity': 'lysine linkage', 'label': 'protein'}, {'entity': 'ubiquitin chain synthesis', 'label': 'protein'}, {'entity': '8A', 'label': 'figure'}, {'entity': 'cultural phenomena', 'label': 'environment'}, {'entity': 'Interaction', 'label': 'event'}, {'entity': 'evidence of harm', 'label': 'disease'}, {'entity': 'workflow', 'label': 'method'}, {'entity': 'content quality', 'label': 'measurement'}, {'entity': 'multiple use of data', 'label': 'method'}, {'entity': 'hardware', 'label': 'organization'}, {'entity': 'retrospective data', 'label': 'measurement'}, {'entity': 'complementary', 'label': 'term'}, {'entity': 'hardware', 'label': 'method'}, {'entity': 'information technology', 'label': 'method'}, {'entity': 'digital cartography', 'label': 'method'}, {'entity': 'real-time GIS', 'label': 'method'}, {'entity': 'UK NHS', 'label': 'organization'}, {'entity': 'data/analysis errors', 'label': 'measurement'}, {'entity': 'University of California, Berkeley', 'label': 'organization'}, {'entity': '2001', 'label': 'year'}, {'entity': 'buffer generation', 'label': 'method'}, {'entity': 'robust conclusions', 'label': 'clinical trial'}, {'entity': 'electronic map', 'label': 'method'}, {'entity': 'multivariate', 'label': 'method'}, {'entity': 'surface statistical', 'label': 'measurement'}, {'entity': 'probability estimation', 'label': 'method'}, {'entity': 'spatial relationship', 'label': 'method'}, {'entity': 'health outcomes', 'label': 'disease'}, {'entity': 'resource allocation', 'label': 'method'}, {'entity': 'points of strength and weakness', 'label': 'measurement'}, {'entity': 'spatial entities', 'label': 'location'}, {'entity': 'Choropleth map', 'label': 'method'}, {'entity': 'shaded value', 'label': 'measurement'}, {'entity': 'polygon', 'label': 'region'}, {'entity': 'contiguous area', 'label': 'region'}, {'entity': 'larger areas', 'label': 'region'}, {'entity': 'spatial filter', 'label': 'method'}, {'entity': 'privacy', 'label': 'measurement'}, {'entity': 'smoothed map', 'label': 'method'}, {'entity': 'ratio smoother', 'label': 'method'}, {'entity': 'census area', 'label': 'location'}, {'entity': 'grid point', 'label': 'location'}, {'entity': 'overlap', 'label': 'method'}, {'entity': 'grid point', 'label': 'location'}, {'entity': 'smoothed map', 'label': 'diagnostic'}, {'entity': 'Talbot et al', 'label': 'organization'}, {'entity': 'disease epidemiology', 'label': 'epidemiology'}, {'entity': 'Abbey', 'label': 'location'}, {'entity': 'Twerton', 'label': 'location'}, {'entity': 'ArcView GIS', 'label': 'software'}, {'entity': 'animation', 'label': 'method'}, {'entity': 'raster', 'label': 'measurement'}, {'entity': 'associative', 'label': 'method'}, {'entity': 'county-level', 'label': 'measurement'}, {'entity': 'Trend surface analysis', 'label': 'method'}, {'entity': 'disease epidemics', 'label': 'epidemiology'}, {'entity': 'Drug Incidence and Prevalence Estimation Program', 'label': 'clinical trial'}, {'entity': 'trend surface map', 'label': 'method'}, {'entity': 'ESRI ArcGIS Tracking Analyst', 'label': 'software'}, {'entity': 'ArcGIS 3D Analyst', 'label': 'organization'}, {'entity': 'population growth', 'label': 'measurement'}, {'entity': 'spatial disease clustering', 'label': 'disease'}, {'entity': 'nonparametric', 'label': 'method'}, {'entity': \"Rogerson's spatial pattern surveillance technique\", 'label': 'method'}, {'entity': 'clusters', 'label': 'event'}, {'entity': 'cumulative sum statistic', 'label': 'method'}, {'entity': 'Sentient', 'label': 'organization'}, {'entity': 'problem solving', 'label': 'method'}, {'entity': 'data entry', 'label': 'method'}, {'entity': 'wireless', 'label': 'technology'}, {'entity': 'spatially referenced', 'label': 'method'}, {'entity': 'isochrones', 'label': 'method'}, {'entity': 'outpatient medical records', 'label': 'clinical trial'}, {'entity': \"L'Orfeo\", 'label': 'organization'}, {'entity': 'Richards et al', 'label': 'organization'}, {'entity': 'industrial hygienist', 'label': 'person'}, {'entity': 'public decision-making', 'label': 'public health'}, {'entity': 'famine early warning system', 'label': 'diagnostic'}, {'entity': 'HealthMapper', 'label': 'tool'}, {'entity': 'epidemics management', 'label': 'epidemiology'}, {'entity': 'health geography', 'label': 'epidemiology'}, {'entity': 'healthcare commissioning', 'label': 'clinical trial'}, {'entity': 'population screening', 'label': 'method'}, {'entity': 'Field et al', 'label': 'organization'}, {'entity': 'retrospective', 'label': 'method'}, {'entity': 'forecasting', 'label': 'method'}, {'entity': 'tourism', 'label': 'event'}, {'entity': 'Regional Office for Europe', 'label': 'organization'}, {'entity': 'time chart', 'label': 'method'}, {'entity': 'Atlas of Health in Europe', 'label': 'publication'}, {'entity': 'NIPH', 'label': 'organization'}, {'entity': 'determinant', 'label': 'measurement'}, {'entity': '2004', 'label': 'year'}, {'entity': 'healthcare delivery', 'label': 'clinical trial'}, {'entity': 'spatial interaction model', 'label': 'method'}, {'entity': 'networking capabilities', 'label': 'method'}, {'entity': 'accessibility patterns of service utilisation', 'label': 'measurement'}, {'entity': 'access block', 'label': 'disease'}, {'entity': 'Access block', 'label': 'measurement'}, {'entity': 'transferred', 'label': 'measurement'}, {'entity': 'supply/demand mismatch', 'label': 'symptom'}, {'entity': 'casuality queues', 'label': 'symptom'}, {'entity': 'patient activity system', 'label': 'clinical trial'}, {'entity': 'in-hospital', 'label': 'location'}, {'entity': 'health-based GIS', 'label': 'method'}, {'entity': 'Information Policy Unit', 'label': 'organization'}, {'entity': 'matching of demand', 'label': 'method'}, {'entity': 'NHS Online Health Informatics Community Portal', 'label': 'organization'}, {'entity': 'aggregated patient records', 'label': 'measurement'}, {'entity': 'data-handling', 'label': 'method'}, {'entity': '1990s', 'label': 'region'}, {'entity': 'reforms of the early 1990s', 'label': 'event'}, {'entity': 'managers', 'label': 'person'}, {'entity': 'Staff turnover', 'label': 'event'}, {'entity': 'regional health GIS centre', 'label': 'organization'}, {'entity': 'spatial data collection', 'label': 'method'}, {'entity': 'NHS region', 'label': 'region'}, {'entity': '2003', 'label': 'publication'}, {'entity': 'statistical data', 'label': 'measurement'}, {'entity': 'rate per 100,000 population', 'label': 'measurement'}, {'entity': 'Cancer Intelligence Unit', 'label': 'organization'}, {'entity': 'data confidentiality issues', 'label': 'measurement'}, {'entity': 'data held by other organisations', 'label': 'measurement'}, {'entity': 'Higgs et al', 'label': 'organization'}, {'entity': 'cultural', 'label': 'environment'}, {'entity': 'culture of data sharing', 'label': 'concept'}, {'entity': 'Education, training, and capacity building', 'label': 'clinical trial'}, {'entity': 'UNIGIS International', 'label': 'organization'}, {'entity': 'Unambiguous', 'label': 'vocabulary'}, {'entity': 'information and services', 'label': 'service'}, {'entity': 'stakeholders', 'label': 'organization'}, {'entity': 'business model', 'label': 'method'}, {'entity': 'adapt to changing technologies', 'label': 'method'}, {'entity': 'institutional framework', 'label': 'organization'}, {'entity': 'lead role', 'label': 'role'}, {'entity': 'semantics', 'label': 'method'}, {'entity': 'proof of concept', 'label': 'clinical trial'}, {'entity': 'market-linked application', 'label': 'method'}, {'entity': 'EIS-AFRICA', 'label': 'organization'}, {'entity': 'building of roads', 'label': 'infrastructure'}, {'entity': 'basic infrastructures', 'label': 'organization'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'documentation', 'label': 'method'}, {'entity': 'relevance', 'label': 'measurement'}, {'entity': 'standardised', 'label': 'measurement'}, {'entity': 'copyright', 'label': 'publication'}, {'entity': 'data access', 'label': 'measurement'}, {'entity': 'storage', 'label': 'method'}, {'entity': 'Raising awareness', 'label': 'event'}, {'entity': 'policy dialogue', 'label': 'event'}, {'entity': 'Regional office for Africa', 'label': 'organization'}, {'entity': 'health hazard mapping', 'label': 'method'}, {'entity': 'An incremental approach', 'label': 'method'}, {'entity': 'Public Health Reports', 'label': 'publication'}, {'entity': 'CD-ROM', 'label': 'measurement'}, {'entity': 'maintenance', 'label': 'method'}, {'entity': 'data confidentiality issues', 'label': 'event'}, {'entity': 'An incremental approach', 'label': 'method'}, {'entity': 'sharing agreement', 'label': 'agreement'}, {'entity': 'backup', 'label': 'method'}, {'entity': 'over-the-counter', 'label': 'drug'}, {'entity': 'industry expert', 'label': 'person'}, {'entity': 'public health official', 'label': 'public health'}, {'entity': 'Most recent published survey of levels of GIS use in the NHS', 'label': 'publication'}, {'entity': 'business case', 'label': 'clinical trial'}, {'entity': 'funding', 'label': 'organization'}, {'entity': 'Geospatial readiness', 'label': 'concept'}, {'entity': 'Community/university research collaboration', 'label': 'organization'}, {'entity': 'national spatial health information infrastructure', 'label': 'organization'}, {'entity': 'determinants of health', 'label': 'concept'}, {'entity': 'data model', 'label': 'method'}, {'entity': 'cartographic', 'label': 'measurement'}, {'entity': 'cartographic file', 'label': 'measurement'}, {'entity': 'Hospital Episode Statistics in England', 'label': 'measurement'}, {'entity': 'ad hoc metadata', 'label': 'measurement'}, {'entity': 'directories', 'label': 'organization'}, {'entity': 'acquisition', 'label': 'process'}, {'entity': 'privacy concerns', 'label': 'disease'}, {'entity': 'privacy', 'label': 'vaccine'}, {'entity': 'map type', 'label': 'method'}, {'entity': 'analysis of data', 'label': 'method'}, {'entity': 'iterative', 'label': 'method'}, {'entity': 'design model', 'label': 'method'}, {'entity': 'user level', 'label': 'method'}, {'entity': 'decision support systems', 'label': 'clinical trial'}, {'entity': 'institutional culture', 'label': 'organization'}, {'entity': 'research rigor', 'label': 'treatment'}, {'entity': 'uncertainty', 'label': 'treatment'}, {'entity': 'collaborative process', 'label': 'event'}, {'entity': 'language', 'label': 'vocabulary'}, {'entity': 'academic disciplines', 'label': 'organization'}, {'entity': 'Trust', 'label': 'vaccine'}, {'entity': 'resources', 'label': 'vaccine'}, {'entity': 'trust', 'label': 'immune response'}, {'entity': 'uncertainty', 'label': 'symptom'}, {'entity': 'uncertainty', 'label': 'vocabulary'}, {'entity': 'ambiguity', 'label': 'vocabulary'}, {'entity': 'collaborative research', 'label': 'method'}, {'entity': 'funding agencies', 'label': 'organization'}, {'entity': 'community/university research partnership', 'label': 'organization'}, {'entity': 'San Diego Association of Governments', 'label': 'organization'}, {'entity': 'Window to My Environment', 'label': 'organization'}, {'entity': 'federal', 'label': 'organization'}, {'entity': 'timeliness', 'label': 'measurement'}, {'entity': 'Public health database', 'label': 'public health'}, {'entity': 'ontologies', 'label': 'method'}, {'entity': 'integration of data', 'label': 'method'}, {'entity': 'health hazard mapping', 'label': 'method'}, {'entity': 'one-size-fits-all', 'label': 'measurement'}, {'entity': 'hazard of interest', 'label': 'disease'}, {'entity': 'concept', 'label': 'vocabulary'}, {'entity': 'head of population', 'label': 'measurement'}, {'entity': 'aggregation level', 'label': 'measurement'}, {'entity': 'hazard', 'label': 'disease'}, {'entity': 'unnecessary disease', 'label': 'disease'}, {'entity': 'public health governing bodies', 'label': 'organization'}, {'entity': 'research organisations', 'label': 'organization'}, {'entity': 'facility emergency response', 'label': 'emergency response'}, {'entity': 'man-made disasters', 'label': 'event'}, {'entity': 'proprietary', 'label': 'organization'}, {'entity': 'interoperability', 'label': 'method'}, {'entity': 'Web GIS', 'label': 'organization'}, {'entity': 'Geography Markup Language', 'label': 'organization'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'cities', 'label': 'city'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'Existing SDIs and SDI initiatives worldwide', 'label': 'organization'}, {'entity': 'Web Feature Service', 'label': 'organization'}, {'entity': 'proprietary software', 'label': 'organization'}, {'entity': 'MasterMap of Great Britain', 'label': 'publication'}, {'entity': 'MasterMap', 'label': 'organization'}, {'entity': 'OS MasterMap', 'label': 'organization'}, {'entity': 'presentation', 'label': 'method'}, {'entity': 'features', 'label': 'vocabulary'}, {'entity': 'stylesheets', 'label': 'method'}, {'entity': 'WFS query', 'label': 'method'}, {'entity': 'schema', 'label': 'publication'}, {'entity': 'features', 'label': 'measurement'}, {'entity': 'stylesheets', 'label': 'method'}, {'entity': 'Web Feature Service', 'label': 'organization'}, {'entity': 'gridded data', 'label': 'measurement'}, {'entity': 'schema components', 'label': 'publication'}, {'entity': 'International Classification of Diseases', 'label': 'publication'}, {'entity': 'spatial database', 'label': 'measurement'}, {'entity': 'feature-streaming', 'label': 'method'}, {'entity': 'query', 'label': 'method'}, {'entity': 'database optimiser', 'label': 'method'}, {'entity': 'optimiser', 'label': 'method'}, {'entity': 'joins', 'label': 'method'}, {'entity': 'federated database', 'label': 'technology'}, {'entity': 'IBM DB2', 'label': 'organization'}, {'entity': 'optimiser', 'label': 'method'}, {'entity': 'postcodes', 'label': 'location'}, {'entity': 'ProADDRESS', 'label': 'software'}, {'entity': 'bioterrorist', 'label': 'event'}, {'entity': 'automated conflation', 'label': 'method'}, {'entity': '2001 World Trade Centre attack', 'label': 'event'}, {'entity': 'city planning', 'label': 'organization'}, {'entity': '2001 terrorist attack', 'label': 'event'}, {'entity': 'CD-ROM', 'label': 'measurement'}, {'entity': 'community health planning', 'label': 'public health'}, {'entity': 'data silos', 'label': 'organization'}, {'entity': 'map boundary', 'label': 'location'}, {'entity': 'fault-tolerant', 'label': 'method'}, {'entity': 'decision-making process', 'label': 'public health'}, {'entity': 'learning', 'label': 'method'}, {'entity': 'tools', 'label': 'vaccine'}, {'entity': 'graphs', 'label': 'diagnostic'}, {'entity': 'computing infrastructure', 'label': 'organization'}, {'entity': 'blind', 'label': 'person'}, {'entity': 'his article', 'label': 'publication'}, {'entity': 'data confidentiality', 'label': 'measurement'}, {'entity': \"Megan's Law\", 'label': 'organization'}, {'entity': 'Spatial data confidentiality', 'label': 'public health'}, {'entity': 'linked', 'label': 'method'}, {'entity': 'disaggregate data', 'label': 'measurement'}, {'entity': 'micro-data', 'label': 'measurement'}, {'entity': 'public health concerns', 'label': 'public health'}, {'entity': 'micro-data', 'label': 'measurement'}, {'entity': 'healthcare utilisation', 'label': 'measurement'}, {'entity': 'aggregated data', 'label': 'measurement'}, {'entity': 'fine-level', 'label': 'measurement'}, {'entity': 'solution group', 'label': 'clinical trial'}, {'entity': 'health record', 'label': 'disease'}, {'entity': 'affine point transformation', 'label': 'method'}, {'entity': 'random perturbation', 'label': 'method'}, {'entity': 'geographic mask', 'label': 'method'}, {'entity': 'surrogate location', 'label': 'location'}, {'entity': 'geographic identifier', 'label': 'location'}, {'entity': 'disclosure risk', 'label': 'measurement'}, {'entity': 'aggregation mask', 'label': 'method'}, {'entity': 'Real aggregation', 'label': 'method'}, {'entity': 'South Carolina Department of Health and Environmental Control', 'label': 'organization'}, {'entity': 'health problems', 'label': 'disease'}, {'entity': 'census tract', 'label': 'location'}, {'entity': 'census tract', 'label': 'location'}, {'entity': 'census tracts', 'label': 'location'}, {'entity': 'Census tracts', 'label': 'location'}, {'entity': 'Vital Health and Census Data Integration System', 'label': 'system'}, {'entity': 'postcodes', 'label': 'location'}, {'entity': 'Software agent', 'label': 'method'}, {'entity': 'confidential health records', 'label': 'measurement'}, {'entity': 'intelligent analysis agent', 'label': 'method'}, {'entity': 'Health System Resident Component', 'label': 'organization'}, {'entity': 'protected data', 'label': 'measurement'}, {'entity': 'XACML', 'label': 'organization'}, {'entity': 'Digital Rights Manangement', 'label': 'organization'}, {'entity': 'Windows Server 2003', 'label': 'organization'}, {'entity': 'Private Secured Geography Networks', 'label': 'organization'}, {'entity': 'anonymised', 'label': 'organization'}, {'entity': 'LIBCSP', 'label': 'organization'}, {'entity': 'Washington State Health Department', 'label': 'organization'}, {'entity': 'privacy', 'label': 'measurement'}, {'entity': 'laws regarding privacy', 'label': 'legal'}, {'entity': 'patient consent', 'label': 'vaccine'}, {'entity': 'Health and Social Care Act 2001', 'label': 'organization'}, {'entity': 'TC 251', 'label': 'organization'}, {'entity': 'Health Insurance Portability and Accountability Act', 'label': 'organization'}, {'entity': 'September 2001 events', 'label': 'event'}, {'entity': 'confidentiality', 'label': 'measurement'}, {'entity': 'Tounderstand', 'label': 'measurement'}, {'entity': 'confounding factors', 'label': 'measurement'}, {'entity': 'Monmonier', 'label': 'person'}, {'entity': \"cartographer's paradox\", 'label': 'concept'}, {'entity': 'small numbers problem', 'label': 'epidemiology'}, {'entity': 'statistically unstable', 'label': 'measurement'}, {'entity': 'true pattern', 'label': 'diagnostic'}, {'entity': 'quantifying', 'label': 'method'}, {'entity': 'continuous monitoring of health data', 'label': 'clinical trial'}, {'entity': 'ClusterSeer', 'label': 'method'}, {'entity': 'Clus-terSeer', 'label': 'software'}, {'entity': 'GeoVISTA Studio', 'label': 'organization'}, {'entity': 'GeoVISTA Studio', 'label': 'organization'}, {'entity': 'geographic visualisation', 'label': 'method'}, {'entity': 'informaticians', 'label': 'organization'}, {'entity': 'A balanced approach', 'label': 'method'}, {'entity': 'aetiological', 'label': 'epidemiology'}, {'entity': 'longitudinal database', 'label': 'diagnostic'}, {'entity': '1992', 'label': 'year'}, {'entity': 'interpolation', 'label': 'method'}, {'entity': 'misleading', 'label': 'measurement'}, {'entity': 'optimum scale', 'label': 'measurement'}, {'entity': 'US population', 'label': 'location'}, {'entity': \"TerraSeer's Cancer Atlas Viewer\", 'label': 'tool'}, {'entity': 'scatterplots', 'label': 'method'}, {'entity': 'cancer research', 'label': 'clinical trial'}, {'entity': 'health conditions', 'label': 'disease'}, {'entity': 'uncertainty', 'label': 'measurement'}, {'entity': 'MARA', 'label': 'organization'}, {'entity': 'maintainer', 'label': 'organization'}, {'entity': 'maintain', 'label': 'method'}, {'entity': 'spatial information infrastructure', 'label': 'concept'}, {'entity': 'distributed geolibrary', 'label': 'organization'}, {'entity': 'Distributed geolibraries', 'label': 'organization'}, {'entity': 'location map', 'label': 'measurement'}, {'entity': 'multidisciplinary research', 'label': 'method'}, {'entity': 'wireless communications', 'label': 'method'}, {'entity': 'geolibrary', 'label': 'organization'}, {'entity': 'intellectual property', 'label': 'organization'}, {'entity': 'US NSDI', 'label': 'organization'}, {'entity': 'US government', 'label': 'organization'}, {'entity': 'federal government', 'label': 'organization'}, {'entity': 'Census 2001', 'label': 'publication'}, {'entity': 'MetaGenie', 'label': 'tool'}, {'entity': 'UK Association for Geographic Information', 'label': 'organization'}, {'entity': 'infrastructure for spatial information', 'label': 'organization'}, {'entity': 'GIgateway Data Locator', 'label': 'organization'}, {'entity': 'interactive map', 'label': 'method'}, {'entity': 'Australian Spatial Data Infrastructure', 'label': 'organization'}, {'entity': 'Australian Spatial Data Directory', 'label': 'organization'}, {'entity': 'comparison', 'label': 'method'}, {'entity': 'Framework', 'label': 'organization'}, {'entity': 'base layer', 'label': 'location'}, {'entity': 'attributes', 'label': 'measurement'}, {'entity': 'distributed search', 'label': 'method'}, {'entity': 'technology', 'label': 'method'}, {'entity': 'geographic information provider', 'label': 'person'}, {'entity': 'specifications', 'label': 'measurement'}, {'entity': 'occupational safety and health', 'label': 'public health'}, {'entity': 'disease surveillance', 'label': 'public health'}, {'entity': 'signs of disease', 'label': 'symptom'}, {'entity': 'health monitoring system', 'label': 'diagnostic'}, {'entity': 'covert', 'label': 'event'}, {'entity': 'informaticians', 'label': 'organization'}, {'entity': 'outbreak detection', 'label': 'method'}, {'entity': 'unique', 'label': 'measurement'}, {'entity': 'very rare outbreaks', 'label': 'event'}, {'entity': 'spatial health information infrastructure', 'label': 'measurement'}, {'entity': 'man-made disasters', 'label': 'event'}, {'entity': 'September 2001 attacks', 'label': 'event'}, {'entity': 'US Federal Emergency Management Agency', 'label': 'organization'}, {'entity': 'animal health professionals', 'label': 'person'}, {'entity': 'earthquake model', 'label': 'diagnostic'}, {'entity': 'damage assessment', 'label': 'measurement'}, {'entity': 'WHO Regional Offices', 'label': 'organization'}, {'entity': 'outbreak alert and response', 'label': 'clinical trial'}, {'entity': 'climatic', 'label': 'environment'}, {'entity': 'Global Atlas of Infectious Diseases', 'label': 'organization'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'graduated circles', 'label': 'method'}, {'entity': 'street', 'label': 'location'}, {'entity': 'disease outbreak', 'label': 'event'}, {'entity': 'disease surveillance system', 'label': 'epidemiology'}, {'entity': 'acute care hospital', 'label': 'location'}, {'entity': 'laboratory work', 'label': 'diagnostic'}, {'entity': 'health levels', 'label': 'disease'}, {'entity': 'constitutional', 'label': 'symptom'}, {'entity': 'multivariate', 'label': 'method'}, {'entity': 'over-the-counter', 'label': 'drug'}, {'entity': 'scanners', 'label': 'tool'}, {'entity': 'cohort exposure', 'label': 'transmission'}, {'entity': 'OTC', 'label': 'drug'}, {'entity': 'standard coding system', 'label': 'method'}, {'entity': 'false alarms', 'label': 'detection'}, {'entity': 'epidemic plot', 'label': 'diagram'}, {'entity': 'NRDM map', 'label': 'diagnostic'}, {'entity': 'green', 'label': 'color'}, {'entity': 'yellow', 'label': 'color'}, {'entity': 'orange', 'label': 'color'}, {'entity': 'anomaly', 'label': 'measurement'}, {'entity': 'green', 'label': 'color'}, {'entity': 'water contamination', 'label': 'disease'}, {'entity': 'differ-ent', 'label': 'method'}, {'entity': 'spatial scan', 'label': 'method'}, {'entity': 'ready-to-use', 'label': 'measurement'}, {'entity': 'network engineer', 'label': 'person'}, {'entity': 'application service provider', 'label': 'organization'}, {'entity': 'coalitions', 'label': 'organization'}, {'entity': 'well-being', 'label': 'clinical trial'}, {'entity': 'Environmental Health Surveillance System for Scotland', 'label': 'organization'}, {'entity': 'death record field', 'label': 'disease'}, {'entity': 'London Air Quality Network', 'label': 'organization'}, {'entity': '1999', 'label': 'year'}, {'entity': 'South East England', 'label': 'region'}, {'entity': 'fitness', 'label': 'measurement'}, {'entity': 'spatial statistical', 'label': 'method'}, {'entity': 'digital cartography', 'label': 'method'}, {'entity': 'mitigable', 'label': 'treatment'}, {'entity': 'business plan', 'label': 'method'}, {'entity': 'healthcare system', 'label': 'organization'}, {'entity': 'health geography', 'label': 'epidemiology'}, {'entity': 'healthcare commissioning', 'label': 'clinical trial'}, {'entity': 'expertise', 'label': 'method'}, {'entity': 'UK initiative', 'label': 'organization'}, {'entity': 'GIS business plan', 'label': 'method'}, {'entity': 'health condition', 'label': 'disease'}, {'entity': \"Tomlison's methodology\", 'label': 'method'}, {'entity': 'retrospective data', 'label': 'measurement'}, {'entity': 'foundation data layer', 'label': 'measurement'}, {'entity': 'foundation spatial data', 'label': 'data'}, {'entity': 'courses', 'label': 'clinical trial'}, {'entity': 'Boscoe and Pickle', 'label': 'organization'}, {'entity': 'networking', 'label': 'method'}, {'entity': 'confidentiality', 'label': 'measurement'}, {'entity': 'fine-level', 'label': 'measurement'}, {'entity': 'ontologies', 'label': 'vocabulary'}, {'entity': 'Semantic Web', 'label': 'method'}, {'entity': 'small numbers problem', 'label': 'measurement'}, {'entity': 'uncertainty', 'label': 'measurement'}, {'entity': 'public health workflows', 'label': 'workflow'}, {'entity': 'bottom-line conclusions', 'label': 'diagnostic'}, {'entity': 'fault-tolerant', 'label': 'measurement'}, {'entity': 'nested', 'label': 'method'}, {'entity': 'autonomous', 'label': 'method'}, {'entity': 'pattern spotters', 'label': 'method'}, {'entity': 'systematic', 'label': 'method'}, {'entity': 'proof of concept', 'label': 'clinical trial'}, {'entity': '2003 SARS outbreak', 'label': 'event'}, {'entity': 'interrelated', 'label': 'relation'}, {'entity': 'spatial health information infrastructure', 'label': 'organization'}, {'entity': 'graduated circles', 'label': 'method'}, {'entity': 'international air routes', 'label': 'transmission'}, {'entity': 'global alert', 'label': 'event'}, {'entity': 'travel advisories', 'label': 'public health'}, {'entity': 'outbreak alert and response', 'label': 'epidemiology'}, {'entity': 'climatic', 'label': 'environment'}, {'entity': 'map layer', 'label': 'method'}, {'entity': '2003 outbreak', 'label': 'event'}, {'entity': 'SARS map for 29 April 2003', 'label': 'diagram'}, {'entity': 'world map of SARS', 'label': 'diagram'}, {'entity': 'epidemic curves of SARS', 'label': 'diagram'}, {'entity': '2003 SARS outbreak', 'label': 'event'}, {'entity': 'choropleth rendition', 'label': 'diagram'}, {'entity': '2003 SARS outbreak', 'label': 'event'}, {'entity': 'street', 'label': 'location'}, {'entity': 'SARS distribution', 'label': 'disease'}, {'entity': 'SARS control measures', 'label': 'treatment'}, {'entity': 'provincial SARS distribution map of China', 'label': 'publication'}, {'entity': '2003 outbreak', 'label': 'event'}, {'entity': 'ESRI China', 'label': 'organization'}, {'entity': 'SARS GIS', 'label': 'organization'}, {'entity': 'Hong Kong Department of Health', 'label': 'organization'}, {'entity': 'Hong Kong maps', 'label': 'publication'}, {'entity': 'ESRI ArcIMS', 'label': 'organization'}, {'entity': \"SARS Figure 1 Corda's world map of SARS\", 'label': 'publication'}, {'entity': 'SARS deaths by country', 'label': 'measurement'}, {'entity': 'district hospital', 'label': 'location'}, {'entity': 'surveillance statistics', 'label': 'measurement'}, {'entity': 'Macromedia Flash', 'label': 'organization'}, {'entity': 'classification', 'label': 'method'}, {'entity': 'Sunday Communications', 'label': 'organization'}, {'entity': 'confirmed', 'label': 'disease'}, {'entity': 'HKU map', 'label': 'diagram'}, {'entity': 'SARS GIS', 'label': 'diagnostic'}, {'entity': 'ArcView GIS', 'label': 'software'}, {'entity': 'web map', 'label': 'method'}, {'entity': '21st century', 'label': 'time period'}, {'entity': 'green', 'label': 'measurement'}, {'entity': '2003 outbreak', 'label': 'event'}, {'entity': 'street', 'label': 'location'}, {'entity': 'Monmonier', 'label': 'person'}, {'entity': \"cartographer's paradox\", 'label': 'concept'}, {'entity': 'suppression', 'label': 'method'}, {'entity': 'SARS mapping in Hong Kong', 'label': 'diagnostic'}, {'entity': 'SARS GIS', 'label': 'organization'}, {'entity': 'interactive mapping', 'label': 'method'}, {'entity': 'information sharing', 'label': 'method'}, {'entity': 'research data', 'label': 'publication'}, {'entity': 'anecdotes', 'label': 'clinical trial'}, {'entity': 'potential downsides', 'label': 'symptom'}, {'entity': 'respiratory syncytial virus pneumonitis', 'label': 'disease'}, {'entity': 'Systemic steroid', 'label': 'drug'}, {'entity': 'use of steroids in the treatment of SARS', 'label': 'treatment'}, {'entity': 'significant adverse effects', 'label': 'treatment'}, {'entity': 'case series', 'label': 'clinical trial'}, {'entity': 'nonblinded', 'label': 'method'}, {'entity': 'high doses', 'label': 'treatment'}, {'entity': 'inflammatory damage', 'label': 'host'}, {'entity': 'effect of steroids', 'label': 'clinical trial'}, {'entity': 'steroid-related complications', 'label': 'disease'}, {'entity': 'retrospective', 'label': 'clinical trial'}, {'entity': 'anecdotes', 'label': 'clinical trial'}, {'entity': 'contaminated nasal secretions', 'label': 'transmission'}, {'entity': 'routes of HCV transmission', 'label': 'transmission'}, {'entity': 'injection risk', 'label': 'virus'}, {'entity': 'bleeding of nasal membranes', 'label': 'symptom'}, {'entity': 'Retrovirus Epidemiology Donor Study', 'label': 'organization'}, {'entity': 'list of criteria used to screen potential blood donors', 'label': 'method'}, {'entity': 'HCV antibodies', 'label': 'virus'}, {'entity': 'Superscript™ First Strand cDNA Synthesis kit', 'label': 'method'}, {'entity': 'HCV RNA detection', 'label': 'diagnostic'}, {'entity': 'serology', 'label': 'measurement'}, {'entity': 'virological', 'label': 'virus'}, {'entity': 'dissimilarity', 'label': 'measurement'}, {'entity': 'multiple SNPs', 'label': 'gene'}, {'entity': 'previous results', 'label': 'clinical trial'}, {'entity': 'Amino acid changes', 'label': 'protein'}, {'entity': 'subgroup', 'label': 'virus'}, {'entity': 'strain differences', 'label': 'treatment'}, {'entity': 'origin', 'label': 'virus'}, {'entity': \"5' end\", 'label': 'gene'}, {'entity': 'gbk', 'label': 'format'}, {'entity': 'subgroup', 'label': 'virus'}, {'entity': '29573', 'label': 'measurement'}, {'entity': 'insertions and deletions', 'label': 'sequence polymorphism'}, {'entity': 'primary structure', 'label': 'measurement'}, {'entity': 'TWH position', 'label': 'location'}, {'entity': 'poly-a', 'label': 'virus'}, {'entity': 'AS', 'label': 'gene'}, {'entity': \"5' end 69\", 'label': 'measurement'}, {'entity': 'TWJ', 'label': 'virus'}, {'entity': 'ZMY 1', 'label': 'virus'}, {'entity': 'TWJ', 'label': 'virus'}, {'entity': 'SZ3', 'label': 'virus'}, {'entity': 'III', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'B group', 'label': 'protein'}, {'entity': 'structural tree', 'label': 'diagram'}, {'entity': 'regularities', 'label': 'measurement'}, {'entity': \"3' end\", 'label': 'protein'}, {'entity': 'T-T-T-T / C-G-C-C', 'label': 'sequence'}, {'entity': '3', 'label': 'measurement'}, {'entity': '1ab', 'label': 'protein'}, {'entity': 'Phylogenetic tree of 36 SARS-CoV complete genome isolates', 'label': 'diagram'}, {'entity': 'Phylogenetic tree of 36 SARS-CoV complete genome isolates', 'label': 'diagram'}, {'entity': 'ZMY 1', 'label': 'virus'}, {'entity': 'B group', 'label': 'virus'}, {'entity': 'structural tree', 'label': 'method'}, {'entity': 'pairwise', 'label': 'method'}, {'entity': \"5'\", 'label': 'gene'}, {'entity': 'long insertions', 'label': 'virus'}, {'entity': '2003', 'label': 'year'}, {'entity': 'sites', 'label': 'virus'}, {'entity': 'deleted', 'label': 'method'}, {'entity': \"poly-'a\", 'label': 'gene'}, {'entity': 'parts', 'label': 'measurement'}, {'entity': 'Mismx', 'label': 'file'}, {'entity': 'first outbreak', 'label': 'event'}, {'entity': 'viral proteinase', 'label': 'protein'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'cystic fibrosis transmembrane conductance regulator', 'label': 'protein'}, {'entity': 'coronavirus pathology', 'label': 'disease'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': '2A', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'logo', 'label': 'diagram'}, {'entity': 'coronavirus 3C-like proteinase', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'LQ', 'label': 'pattern'}, {'entity': 'trained', 'label': 'method'}, {'entity': 'positive answer', 'label': 'clinical trial'}, {'entity': 'possibly cleaved', 'label': 'treatment'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'virus proteinase', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'IIIC', 'label': 'gene'}, {'entity': 'zipper motif', 'label': 'protein'}, {'entity': 'picornavirus 3C', 'label': 'protein'}, {'entity': 'RP/EB', 'label': 'protein'}, {'entity': 'early symptoms of SARS', 'label': 'symptom'}, {'entity': 'cystic fibrosis transmembrane conductance regulator', 'label': 'gene'}, {'entity': 'accessible region', 'label': 'protein'}, {'entity': 'C-terminus', 'label': 'protein'}, {'entity': 'coronavirus proteinase 3CL pro', 'label': 'protein'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'Fraction', 'label': 'measurement'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'KSSVQSVAG', 'label': 'protein'}, {'entity': 'drug discovery', 'label': 'method'}, {'entity': 'sequence logo', 'label': 'method'}, {'entity': 'Shannon information content', 'label': 'measurement'}, {'entity': 'p k', 'label': 'measurement'}, {'entity': 'feed-forward', 'label': 'method'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'data set', 'label': 'measurement'}, {'entity': 'histogram', 'label': 'measurement'}, {'entity': 'P(X l)', 'label': 'measurement'}, {'entity': 'non-cleavage', 'label': 'measurement'}, {'entity': 'prediction web server', 'label': 'organization'}, {'entity': 'Swiss-Prot', 'label': 'organization'}, {'entity': 'methicillin-resistant Staphylococcus aureus', 'label': 'virus'}, {'entity': 'MRSA colonization', 'label': 'disease'}, {'entity': 'Hill', 'label': 'person'}, {'entity': 'reversibility', 'label': 'measurement'}, {'entity': 'detection and control', 'label': 'treatment'}, {'entity': 'surveillance cultures/contact precautions', 'label': 'treatment'}, {'entity': 'cost', 'label': 'measurement'}, {'entity': 'unrandomized', 'label': 'clinical trial'}, {'entity': 'falls', 'label': 'symptom'}, {'entity': 'inadequate isolation', 'label': 'disease'}, {'entity': 'mec IV', 'label': 'virus'}, {'entity': 'cohort nursing', 'label': 'method'}, {'entity': 'MRSA reservoir', 'label': 'disease'}, {'entity': 'prospective', 'label': 'clinical trial'}, {'entity': 'evidence that it may be detrimental', 'label': 'clinical trial'}, {'entity': 'nonisolated', 'label': 'disease'}, {'entity': 'cohort nursing', 'label': 'method'}, {'entity': '48-hour', 'label': 'measurement'}, {'entity': 'nosocomial MSSA infection', 'label': 'disease'}, {'entity': 'cohort', 'label': 'virus'}, {'entity': 'Java Web Start', 'label': 'method'}, {'entity': 'gene families', 'label': 'gene'}, {'entity': 'nucleotide composition', 'label': 'measurement'}, {'entity': 'sequence editor', 'label': 'method'}, {'entity': 'LINUX', 'label': 'organization'}, {'entity': 'LINUX', 'label': 'organization'}, {'entity': 'Web Start', 'label': 'method'}, {'entity': 'VGO', 'label': 'vaccine'}, {'entity': 'Bioinformatics Sequence Markup Language', 'label': 'method'}, {'entity': 'BSML', 'label': 'file format'}, {'entity': 'BBB window', 'label': 'method'}, {'entity': 'gaps', 'label': 'measurement'}, {'entity': 'second', 'label': 'measurement'}, {'entity': 'third', 'label': 'sequence'}, {'entity': 'conserved/variable regions', 'label': 'vaccine'}, {'entity': 'scrolling', 'label': 'method'}, {'entity': 'Java', 'label': 'method'}, {'entity': 'unmasked', 'label': 'measurement'}, {'entity': '3frame translation', 'label': 'method'}, {'entity': '.bbb', 'label': 'publication'}, {'entity': 'Comment', 'label': 'vocabulary'}, {'entity': 'main window', 'label': 'clinical trial'}, {'entity': 'column header', 'label': 'method'}, {'entity': 'alignment window', 'label': 'method'}, {'entity': 'consensus refresh', 'label': 'method'}, {'entity': 'regions of mostly substitutions', 'label': 'disease'}, {'entity': 'PNG', 'label': 'file format'}, {'entity': 'user-specified width', 'label': 'measurement'}, {'entity': 'Edit menu', 'label': 'method'}, {'entity': 'Alignment Info', 'label': 'method'}, {'entity': 'spreadsheet', 'label': 'tool'}, {'entity': 'alignment set', 'label': 'measurement'}, {'entity': 'fuzzy motif search', 'label': 'method'}, {'entity': 'table format', 'label': 'measurement'}, {'entity': 'long sequences', 'label': 'measurement'}, {'entity': 'icon', 'label': 'method'}, {'entity': 'ORO', 'label': 'organization'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'founder alignment', 'label': 'virus'}, {'entity': 'ORF map', 'label': 'protein'}, {'entity': 'gene feature information', 'label': 'gene'}, {'entity': 'directory', 'label': 'location'}, {'entity': 'intergenic', 'label': 'gene'}, {'entity': 'hypothesis', 'label': 'clinical trial'}, {'entity': 'GNU General Public License', 'label': 'publication'}, {'entity': 'last common ancestor', 'label': 'gene'}, {'entity': 'comparative', 'label': 'method'}, {'entity': 'root', 'label': 'method'}, {'entity': 'last common ancestor', 'label': 'treatment'}, {'entity': 'a global network of international collaborating laboratories', 'label': 'organization'}, {'entity': 'British Columbia Centre for Disease Control', 'label': 'organization'}, {'entity': 'viral strains', 'label': 'virus'}, {'entity': 'root', 'label': 'gene'}, {'entity': 'base', 'label': 'measurement'}, {'entity': 'root', 'label': 'gene'}, {'entity': 'routes', 'label': 'method'}, {'entity': 'root', 'label': 'term'}, {'entity': \"E(D(S,S'))\", 'label': 'measurement'}, {'entity': 'partition', 'label': 'method'}, {'entity': 'S 0', 'label': 'gene'}, {'entity': 'intermediate sequences', 'label': 'sequence'}, {'entity': 'time factor', 'label': 'measurement'}, {'entity': 'Figure 1 Phylogenetic Tree', 'label': 'diagram'}, {'entity': 'last common ancestor', 'label': 'term'}, {'entity': 'iterations', 'label': 'measurement'}, {'entity': 'wide time distribution', 'label': 'measurement'}, {'entity': 'date of host death', 'label': 'measurement'}, {'entity': 'situation', 'label': 'environment'}, {'entity': 'fledgling epidemic', 'label': 'epidemiology'}, {'entity': 'error of time', 'label': 'measurement'}, {'entity': 'gene entrapment', 'label': 'method'}, {'entity': 'disassembly', 'label': 'virus'}, {'entity': 'IGF-II signalling pathway', 'label': 'pathway'}, {'entity': 'virus resistant cells', 'label': 'host'}, {'entity': 'virus-resistant cells', 'label': 'cell'}, {'entity': 'resistant clones', 'label': 'virus'}, {'entity': 'heterozygosity', 'label': 'gene'}, {'entity': 'imputed', 'label': 'gene'}, {'entity': 'anchorage independent growth', 'label': 'treatment'}, {'entity': 'virus resistant clones', 'label': 'host'}, {'entity': 'intrinsically resistant', 'label': 'virus'}, {'entity': 'In vitro infection', 'label': 'treatment'}, {'entity': 'non-redundant sequences', 'label': 'database'}, {'entity': 'known or presumptive genes', 'label': 'gene'}, {'entity': 'imputed', 'label': 'gene'}, {'entity': 'Figure 1 Persistently infected RIE-1 cells fail to survive in serum-free media', 'label': 'diagnostic'}, {'entity': 'complete medium', 'label': 'measurement'}, {'entity': 'wells', 'label': 'symptom'}, {'entity': 'post-infection', 'label': 'disease'}, {'entity': '89-residue carboxy-terminal E peptide', 'label': 'protein'}, {'entity': 'IGF-II 62T', 'label': 'protein'}, {'entity': 'serotypes', 'label': 'virus'}, {'entity': 'σ3', 'label': 'protein'}, {'entity': 'anti-sense Igf2', 'label': 'gene'}, {'entity': 'IGF-II 62T', 'label': 'protein'}, {'entity': 'soft agar', 'label': 'measurement'}, {'entity': 'phenotypes', 'label': 'disease'}, {'entity': 'reovirus resistance', 'label': 'treatment'}, {'entity': 'Figure 9 Anchorage-independent growth phenotypes of RIE-1 cell clones', 'label': 'diagram'}, {'entity': 'IGF2 SV splice-variant', 'label': 'gene'}, {'entity': 'repacking', 'label': 'virus'}, {'entity': 'zoster virus infection', 'label': 'disease'}, {'entity': 'anchorageindependence', 'label': 'treatment'}, {'entity': 'Virus resistance', 'label': 'virus'}, {'entity': 'disassembly', 'label': 'disease'}, {'entity': 'cell clones', 'label': 'treatment'}, {'entity': 'IGF-II signaling pathway', 'label': 'pathway'}, {'entity': 'disassembly', 'label': 'disease'}, {'entity': 'Palo Alto, CA', 'label': 'location'}, {'entity': 'libraries', 'label': 'organization'}, {'entity': 'plaque forming unit', 'label': 'measurement'}, {'entity': 'Logan, UT', 'label': 'location'}, {'entity': 'colonies', 'label': 'treatment'}, {'entity': 'Lang', 'label': 'virus'}, {'entity': 'Dearing', 'label': 'virus'}, {'entity': 'type 1', 'label': 'virus'}, {'entity': 'type 3', 'label': 'virus'}, {'entity': 'gentian violet', 'label': 'diagnostic'}, {'entity': 'Titres', 'label': 'measurement'}, {'entity': 'Carl Zeiss, Inc', 'label': 'organization'}, {'entity': 'Thornwood, NY', 'label': 'city'}, {'entity': 'Tucson, AZ', 'label': 'location'}, {'entity': 'shuttle-vector', 'label': 'gene'}, {'entity': 'Foster City, CA', 'label': 'city'}, {'entity': 'by chance', 'label': 'measurement'}, {'entity': 'non-repetitive', 'label': 'gene'}, {'entity': 'murine', 'label': 'gene'}, {'entity': 'Gaithersburg, MD', 'label': 'location'}, {'entity': 'Cedar Creek, TX', 'label': 'location'}, {'entity': 'West Grove, PA', 'label': 'location'}, {'entity': 'Valencia, CA', 'label': 'location'}, {'entity': 'agarose solution', 'label': 'measurement'}, {'entity': 'plates', 'label': 'vessel'}, {'entity': 'Hercules, CA', 'label': 'location'}, {'entity': 'West Grove, PA', 'label': 'location'}, {'entity': 'CellTiter 96 Aqueous Non-Radioactiver Cell Proliferation Assay', 'label': 'method'}, {'entity': '2 hours', 'label': 'measurement'}, {'entity': 'constant mutation rate model', 'label': 'method'}, {'entity': 'Vero cell passage', 'label': 'method'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': \"3' – most portion\", 'label': 'protein'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': \"3'\", 'label': 'measurement'}, {'entity': 'monitoring and therapy', 'label': 'treatment'}, {'entity': 'homogenizing', 'label': 'method'}, {'entity': 'viral cell culture', 'label': 'method'}, {'entity': 'sequencing errors', 'label': 'measurement'}, {'entity': 'PAUP*', 'label': 'software'}, {'entity': 'noisy variations', 'label': 'virus'}, {'entity': 'constant mutation rate', 'label': 'measurement'}, {'entity': 'least square fitting', 'label': 'method'}, {'entity': 'Rsquare statistics', 'label': 'measurement'}, {'entity': 'SARS-CoV evolution', 'label': 'disease'}, {'entity': 'constant mutation rate model', 'label': 'method'}, {'entity': 'date of origin', 'label': 'date'}, {'entity': 'mass spectroscopic', 'label': 'method'}, {'entity': '18372', 'label': 'measurement'}, {'entity': 'unusual incident', 'label': 'event'}, {'entity': 'sequencing errors', 'label': 'virus'}, {'entity': 'SARS sequence databases', 'label': 'publication'}, {'entity': 'sequencing error', 'label': 'virus'}, {'entity': 'filtering scheme', 'label': 'method'}, {'entity': 'Common substitution sites', 'label': 'measurement'}, {'entity': 'probability of finding a false mutation', 'label': 'measurement'}, {'entity': 'error estimates', 'label': 'measurement'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'molecular sub-branch', 'label': 'pathway'}, {'entity': 'Hotel M', 'label': 'location'}, {'entity': 'synonymous', 'label': 'gene'}, {'entity': 'insertion', 'label': 'mutation'}, {'entity': 'overlapping nature', 'label': 'clinical trial'}, {'entity': 'rate of mutation in culture', 'label': 'measurement'}, {'entity': 'North China', 'label': 'region'}, {'entity': 'molecular sequence association', 'label': 'diagnostic'}, {'entity': 'SARS emergence', 'label': 'event'}, {'entity': 'PAUP*', 'label': 'software'}, {'entity': 'tree', 'label': 'clinical trial'}, {'entity': 'Singapore cases', 'label': 'location'}, {'entity': 'kinetics of viral replication', 'label': 'measurement'}, {'entity': 'SARS specific antibodies', 'label': 'vaccine'}, {'entity': \"5'-end\", 'label': 'protein'}, {'entity': 'feline enteric coronavirus', 'label': 'virus'}, {'entity': 'world-wide problem', 'label': 'epidemiology'}, {'entity': 'largest number of infected patients', 'label': 'measurement'}, {'entity': 'most appropriate treatment regime', 'label': 'treatment'}, {'entity': 'cascade of responses', 'label': 'immune response'}, {'entity': 'median logarithmic ratio', 'label': 'measurement'}, {'entity': 'missing spots', 'label': 'measurement'}, {'entity': 'reciprocal labeling', 'label': 'method'}, {'entity': 'gene features', 'label': 'gene'}, {'entity': 'expression range', 'label': 'measurement'}, {'entity': 'Friendly Neighbor', 'label': 'method'}, {'entity': 'inoculum', 'label': 'virus'}, {'entity': '1B', 'label': 'measurement'}, {'entity': 'lag phase', 'label': 'symptom'}, {'entity': 'day 4', 'label': 'measurement'}, {'entity': 'donor e', 'label': 'person'}, {'entity': 'SARS-CoV viral titer', 'label': 'measurement'}, {'entity': 'Reciprocal dye swap replicate hybridization', 'label': 'method'}, {'entity': 'specific trafficking', 'label': 'pathway'}, {'entity': 'innate host inflammatory response', 'label': 'immune response'}, {'entity': 'in vitro model', 'label': 'method'}, {'entity': 'post-exposure vaccination', 'label': 'treatment'}, {'entity': 'preparations to improve our ability to detect and respond to an outbreak of smallpox', 'label': 'treatment'}, {'entity': 'network simulation', 'label': 'method'}, {'entity': 'Mathematical model', 'label': 'method'}, {'entity': 'computer model', 'label': 'method'}, {'entity': 'stochastic', 'label': 'method'}, {'entity': 'individual-based model', 'label': 'clinical trial'}, {'entity': 'event-driven', 'label': 'method'}, {'entity': 'Mild', 'label': 'symptom'}, {'entity': 'Natural history of smallpox', 'label': 'epidemiology'}, {'entity': 'different clinical classifications', 'label': 'clinical trial'}, {'entity': 'mild', 'label': 'symptom'}, {'entity': 'successfully vaccinated', 'label': 'vaccine'}, {'entity': 'severe smallpox', 'label': 'disease'}, {'entity': 'reduce the severity', 'label': 'treatment'}, {'entity': 'Vaccine protection', 'label': 'vaccine'}, {'entity': 'close contacts', 'label': 'disease'}, {'entity': 'severe', 'label': 'disease'}, {'entity': 'mild', 'label': 'symptom'}, {'entity': 'pre-eruptive period', 'label': 'disease'}, {'entity': 'Stage 4', 'label': 'epidemiology'}, {'entity': 'uniform distribution', 'label': 'method'}, {'entity': 'epidemic potential', 'label': 'epidemiology'}, {'entity': 'alternative (ad hoc) index of epidemic potential', 'label': 'index'}, {'entity': 'Smallpox stages used in the simulation model', 'label': 'diagnostic'}, {'entity': 'horizontal line shading', 'label': 'diagnostic'}, {'entity': 'epidemic extinction', 'label': 'disease'}, {'entity': 'traceable', 'label': 'method'}, {'entity': 'tracing only of direct contacts', 'label': 'method'}, {'entity': 'case', 'label': 'disease'}, {'entity': 'Hypercube Sample', 'label': 'method'}, {'entity': 'random', 'label': 'transmission'}, {'entity': 'historic smallpox', 'label': 'disease'}, {'entity': 'case finding time', 'label': 'measurement'}, {'entity': 'Hypercube', 'label': 'method'}, {'entity': 'index case', 'label': 'disease'}, {'entity': 'pre-eruptive infectious period', 'label': 'disease'}, {'entity': 'household tracing delay', 'label': 'treatment'}, {'entity': 'mild cases', 'label': 'disease'}, {'entity': 'the far larger number of individuals at risk', 'label': 'vaccine'}, {'entity': 'φ', 'label': 'symptom'}, {'entity': 'routine vaccination', 'label': 'vaccine'}, {'entity': 'casual', 'label': 'transmission'}, {'entity': 'first week of the rash', 'label': 'symptom'}, {'entity': 'Diagnosis time', 'label': 'diagnostic'}, {'entity': 'index case', 'label': 'epidemiology'}, {'entity': 'direct contacts', 'label': 'contact'}, {'entity': 'Stochastic', 'label': 'method'}, {'entity': 'relative hazard for infection due to workplace/social contacts', 'label': 'measurement'}, {'entity': '10000', 'label': 'measurement'}, {'entity': 'case', 'label': 'disease'}, {'entity': 'hazard for close contact transmission', 'label': 'transmission'}, {'entity': 'hazard for random transmission', 'label': 'transmission'}, {'entity': 'control conditions', 'label': 'method'}, {'entity': 'contact finding', 'label': 'measurement'}, {'entity': 'widespread community awareness', 'label': 'event'}, {'entity': 'realizations', 'label': 'clinical trial'}, {'entity': '1000 containment fractions', 'label': 'measurement'}, {'entity': 'household contact', 'label': 'virus'}, {'entity': 'contact finding', 'label': 'method'}, {'entity': 'contact finding probability', 'label': 'measurement'}, {'entity': '1000 \"calibrated\" parameter sets', 'label': 'measurement'}, {'entity': 'effective one day delay', 'label': 'method'}, {'entity': 'calibrated uncertainty analysis', 'label': 'method'}, {'entity': 'fast scenario', 'label': 'event'}, {'entity': 'Faster contact tracing', 'label': 'method'}, {'entity': 'contacts of contacts not traced', 'label': 'vaccination'}, {'entity': 'fast scenario', 'label': 'scenario'}, {'entity': 'slow scenario', 'label': 'event'}, {'entity': 'index case', 'label': 'event'}, {'entity': 'highly contagious', 'label': 'transmission'}, {'entity': 'contact finding probability', 'label': 'measurement'}, {'entity': 'initial cases', 'label': 'clinical trial'}, {'entity': 'More rapid diagnosis', 'label': 'diagnostic'}, {'entity': 'high contact finding probability', 'label': 'transmission'}, {'entity': 'prodromal', 'label': 'symptom'}, {'entity': 'Scenario a', 'label': 'scenario'}, {'entity': 'case a', 'label': 'clinical trial'}, {'entity': 'secondary cases', 'label': 'disease'}, {'entity': 'case b', 'label': 'disease'}, {'entity': 'discontinuation of routine vaccination', 'label': 'event'}, {'entity': 'long diagnosis', 'label': 'diagnostic'}, {'entity': 'prodromal', 'label': 'symptom'}, {'entity': 'scenario a', 'label': 'scenario'}, {'entity': 'scenario b', 'label': 'event'}, {'entity': 'con-tact', 'label': 'contact'}, {'entity': 'mild smallpox', 'label': 'disease'}, {'entity': 'discontinuation of routine vaccination', 'label': 'event'}, {'entity': 'loss of containment', 'label': 'event'}, {'entity': 'vaccine protection', 'label': 'vaccine'}, {'entity': '1000 scenarios', 'label': 'measurement'}, {'entity': 'small pie chart', 'label': 'diagram'}, {'entity': 'calibrated scenarios', 'label': 'clinical trial'}, {'entity': 'planned for', 'label': 'clinical trial'}, {'entity': 'contact finding probability', 'label': 'measurement'}, {'entity': 'moderately severe', 'label': 'disease'}, {'entity': 'mortuary worker', 'label': 'person'}, {'entity': 'first responders', 'label': 'person'}, {'entity': 'adverse events of the vaccine', 'label': 'vaccine'}, {'entity': 'transmission potential', 'label': 'transmission'}, {'entity': 'high coverage', 'label': 'treatment'}, {'entity': 'transient', 'label': 'disease'}, {'entity': 'tracing of contacts of contacts', 'label': 'contact tracing'}, {'entity': 'isolation of contacts', 'label': 'treatment'}, {'entity': 'misinformation', 'label': 'disease'}, {'entity': 'All the Virology', 'label': 'organization'}, {'entity': 'freely and universally accessible', 'label': 'measurement'}, {'entity': 'cited', 'label': 'method'}, {'entity': 'online repository', 'label': 'organization'}, {'entity': 'US National Library of Medicine', 'label': 'organization'}, {'entity': 'No barriers to access', 'label': 'measurement'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'literature searching', 'label': 'method'}, {'entity': 'subscription', 'label': 'organization'}, {'entity': 'UK government', 'label': 'organization'}, {'entity': 'economy', 'label': 'country'}, {'entity': 'editor', 'label': 'person'}, {'entity': 'monitoring editor', 'label': 'person'}, {'entity': 'ATV', 'label': 'organization'}, {'entity': 'competing interests', 'label': 'clinical trial'}, {'entity': 'commercial development', 'label': 'treatment'}, {'entity': 'biodefense study section', 'label': 'organization'}, {'entity': 'per diem', 'label': 'measurement'}, {'entity': 'virus culture', 'label': 'diagnostic'}, {'entity': 'males', 'label': 'person'}, {'entity': 'sputa', 'label': 'diagnostic'}, {'entity': 'strip and touchdown protocol', 'label': 'method'}, {'entity': 'virus culture', 'label': 'diagnostic'}, {'entity': 'virus culture', 'label': 'diagnostic'}, {'entity': 'nucleic acid amplification', 'label': 'diagnostic'}, {'entity': 'contamination issues', 'label': 'disease'}, {'entity': 'primer set', 'label': 'virus'}, {'entity': 'clinically validated', 'label': 'clinical trial'}, {'entity': 'touchdown amplification', 'label': 'method'}, {'entity': 'specific base-pair priming', 'label': 'protein'}, {'entity': 'optimum annealing temperature', 'label': 'measurement'}, {'entity': 'gold standard', 'label': 'vaccine'}, {'entity': 'ADV', 'label': 'virus'}, {'entity': 'IF sera', 'label': 'vaccine'}, {'entity': 'present study', 'label': 'clinical trial'}, {'entity': 'clinical reporting algorithm', 'label': 'method'}, {'entity': 'acute virus respiratory infections', 'label': 'disease'}, {'entity': 'implications', 'label': 'clinical trial'}, {'entity': 'throat', 'label': 'location'}, {'entity': 'throat', 'label': 'location'}, {'entity': 'male', 'label': 'person'}, {'entity': 'parainfluenza type 3', 'label': 'virus'}, {'entity': \"5' non-coding region\", 'label': 'gene'}, {'entity': \"5'\", 'label': 'measurement'}, {'entity': 'multiplexing', 'label': 'method'}, {'entity': 'head-to-head comparison', 'label': 'clinical trial'}, {'entity': 'under reporting', 'label': 'disease'}, {'entity': 'molecular primer', 'label': 'method'}, {'entity': 'generic touchdown protocol', 'label': 'method'}, {'entity': 'cycling parameters', 'label': 'method'}, {'entity': 'critical evaluation', 'label': 'method'}, {'entity': 'H1', 'label': 'virus'}, {'entity': 'H3', 'label': 'virus'}, {'entity': 'specimen extract', 'label': 'diagnostic'}, {'entity': 'mastermix', 'label': 'vaccine'}, {'entity': 'ice', 'label': 'measurement'}, {'entity': 'freeze-thaw cycle', 'label': 'measurement'}, {'entity': 'virus culture', 'label': 'diagnostic'}, {'entity': 'single round system', 'label': 'method'}, {'entity': 'trained', 'label': 'person'}, {'entity': 'Quality Control for Molecular Diagnostics', 'label': 'organization'}, {'entity': 'lower copy number', 'label': 'measurement'}, {'entity': 'cost', 'label': 'measurement'}, {'entity': 'utilities', 'label': 'environment'}, {'entity': 'swab', 'label': 'diagnostic'}, {'entity': 'touchdown protocol', 'label': 'method'}, {'entity': 'Positive Control detection threshold', 'label': 'diagnostic'}, {'entity': 'viral respiratory pathogenesis', 'label': 'pathway'}, {'entity': 'co-pathogen', 'label': 'disease'}, {'entity': 'virus culture', 'label': 'diagnostic'}, {'entity': 'strip and touchdown protocol', 'label': 'method'}, {'entity': 'VTM', 'label': 'virus transport medium'}, {'entity': 'multi-well slide', 'label': 'measurement'}, {'entity': 'Ely', 'label': 'location'}, {'entity': '8 well', 'label': 'measurement'}, {'entity': 'influenza A (H3)', 'label': 'virus'}, {'entity': 'influenza A (H1)', 'label': 'virus'}, {'entity': 'respiratory syncytial virus type A', 'label': 'virus'}, {'entity': 'respiratory syncytial virus type B', 'label': 'virus'}, {'entity': 'parainfluenza virus type 1', 'label': 'virus'}, {'entity': 'multiplex', 'label': 'method'}, {'entity': 'influenza A (H1)', 'label': 'virus'}, {'entity': 'master-mix', 'label': 'vaccine'}, {'entity': 'parainfluenza virus type 1', 'label': 'virus'}, {'entity': 'Southampton', 'label': 'location'}, {'entity': 'control band', 'label': 'diagnostic'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'long-term care', 'label': 'location'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'policy development', 'label': 'treatment'}, {'entity': 'interpersonal skills', 'label': 'method'}, {'entity': 'compe-tencies', 'label': 'clinical trial'}, {'entity': 'core competencies', 'label': 'clinical trial'}, {'entity': 'skills', 'label': 'method'}, {'entity': 'ethical problem', 'label': 'clinical trial'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'affiliated hospitals', 'label': 'organization'}, {'entity': 'eth-ics', 'label': 'clinical trial'}, {'entity': \"JCB's\", 'label': 'organization'}, {'entity': 'academic', 'label': 'organization'}, {'entity': 'general', 'label': 'clinical trial'}, {'entity': 'bioethics training', 'label': 'clinical trial'}, {'entity': 'general', 'label': 'clinical trial'}, {'entity': 'affiliation', 'label': 'organization'}, {'entity': 'stipend', 'label': 'measurement'}, {'entity': 'stakeholders', 'label': 'person'}, {'entity': 'end-of-life care', 'label': 'clinical trial'}, {'entity': 'case conferences', 'label': 'event'}, {'entity': 'ethical questions', 'label': 'event'}, {'entity': 'confidentiality', 'label': 'ethical issue'}, {'entity': 'hospital ethicist', 'label': 'person'}, {'entity': 'quality assurance', 'label': 'treatment'}, {'entity': 'clinical ethics fellows', 'label': 'person'}, {'entity': 'health record', 'label': 'diagnostic'}, {'entity': 'chairing', 'label': 'method'}, {'entity': 'students', 'label': 'person'}, {'entity': 'skill', 'label': 'vocabulary'}, {'entity': 'blood product', 'label': 'vaccine'}, {'entity': 'research ethics board', 'label': 'organization'}, {'entity': 'sustainable', 'label': 'treatment'}, {'entity': 'skill', 'label': 'vocabulary'}, {'entity': 'health ethics', 'label': 'clinical trial'}, {'entity': 'research ethics', 'label': 'organization'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'retrospective case', 'label': 'clinical trial'}, {'entity': 'real', 'label': 'clinical trial'}, {'entity': 'families', 'label': 'organization'}, {'entity': 'pager', 'label': 'object'}, {'entity': 'public relations', 'label': 'method'}, {'entity': 'mediation', 'label': 'method'}, {'entity': 'wisdom', 'label': 'immune response'}, {'entity': 'senior hospital staff member', 'label': 'person'}, {'entity': 'interpersonal skills', 'label': 'clinical trial'}, {'entity': 'conflict negotiation', 'label': 'method'}, {'entity': 'organizational ethics', 'label': 'clinical trial'}, {'entity': 'ethical sensitivity', 'label': 'immune response'}, {'entity': 'committee work', 'label': 'organization'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'clinical ethics practice', 'label': 'method'}, {'entity': 'legal expert', 'label': 'role'}, {'entity': 'humility', 'label': 'character trait'}, {'entity': 'desirable', 'label': 'vocabulary'}, {'entity': 'deliberation', 'label': 'clinical trial'}, {'entity': 'self-awareness', 'label': 'person'}, {'entity': 'perspective', 'label': 'immune response'}, {'entity': 'ethics consultation', 'label': 'clinical trial'}, {'entity': 'selfawareness', 'label': 'trait'}, {'entity': 'ethical dilemmas', 'label': 'immune response'}, {'entity': 'decision-making', 'label': 'clinical trial'}, {'entity': 'ethical issues', 'label': 'event'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'outsider', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'SARS precautions', 'label': 'treatment'}, {'entity': 'legal', 'label': 'clinical trial'}, {'entity': 'mediation', 'label': 'method'}, {'entity': 'Joint Centre for Bioethics Clinical Ethics Fellowship', 'label': 'organization'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'interpretation', 'label': 'method'}, {'entity': 'interpretation', 'label': 'method'}, {'entity': 'antimicrobial rinse-free hand sanitizers', 'label': 'treatment'}, {'entity': 'rinse-free', 'label': 'method'}, {'entity': 'Absenteeism due to communicable illness', 'label': 'symptom'}, {'entity': 'significant difference', 'label': 'clinical trial'}, {'entity': 'rinse-free', 'label': 'method'}, {'entity': 'benefit', 'label': 'treatment'}, {'entity': 'student absenteeism', 'label': 'symptom'}, {'entity': '2 percent', 'label': 'measurement'}, {'entity': 'school environment', 'label': 'environment'}, {'entity': 'antimicrobial rinse-free', 'label': 'treatment'}, {'entity': 'antimicrobial rinse-free', 'label': 'drug'}, {'entity': 'complete protocol', 'label': 'publication'}, {'entity': 'reviews', 'label': 'publication'}, {'entity': 'bibliographic database', 'label': 'publication'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'elementary school', 'label': 'location'}, {'entity': 'senior kindergarten', 'label': 'location'}, {'entity': 'rinse-free', 'label': 'treatment'}, {'entity': 'cluster non-randomized controlled trial', 'label': 'clinical trial'}, {'entity': 'blinding', 'label': 'method'}, {'entity': 'abstracts', 'label': 'publication'}, {'entity': '2005', 'label': 'year'}, {'entity': 'NLS', 'label': 'person'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'English', 'label': 'language'}, {'entity': 'LOE', 'label': 'measurement'}, {'entity': 'NLS', 'label': 'person'}, {'entity': 'rinse-free', 'label': 'method'}, {'entity': 'crossover', 'label': 'clinical trial'}, {'entity': 'double blinding', 'label': 'method'}, {'entity': \"Cochrane Reviewers' Handbook\", 'label': 'publication'}, {'entity': '95 percent confidence interval', 'label': 'measurement'}, {'entity': '95 percent confidence interval', 'label': 'measurement'}, {'entity': 'measures of association', 'label': 'measurement'}, {'entity': 'potentially relevant trials', 'label': 'clinical trial'}, {'entity': 'abstracts', 'label': 'publication'}, {'entity': 'Inappropriate intervention', 'label': 'clinical trial'}, {'entity': 'inclusion criteria', 'label': 'clinical trial'}, {'entity': \"McNemars's test\", 'label': 'method'}, {'entity': 'Thompson 2004', 'label': 'publication'}, {'entity': '2000', 'label': 'year'}, {'entity': '2004', 'label': 'year'}, {'entity': 'elementary school', 'label': 'organization'}, {'entity': 'RCT', 'label': 'clinical trial'}, {'entity': '15', 'label': 'clinical trial'}, {'entity': 'Quality of reporting', 'label': 'measurement'}, {'entity': 'double-blinded', 'label': 'clinical trial'}, {'entity': 'dropouts', 'label': 'clinical trial'}, {'entity': 'dropouts', 'label': 'virus'}, {'entity': 'overall agreement', 'label': 'measurement'}, {'entity': 'percent relative effect', 'label': 'measurement'}, {'entity': 'one such studied completed', 'label': 'event'}, {'entity': '1999', 'label': 'year'}, {'entity': 'outbreaks do occur', 'label': 'event'}, {'entity': 'elementary school', 'label': 'location'}, {'entity': 'antimicrobial rinse-free', 'label': 'drug'}, {'entity': 'cluster controlled', 'label': 'clinical trial'}, {'entity': 'non-randomized', 'label': 'method'}, {'entity': 'double-blinded', 'label': 'clinical trial'}, {'entity': 'exaggerated treatment effect', 'label': 'treatment'}, {'entity': 'rinsefree', 'label': 'drug'}, {'entity': 'unpublished', 'label': 'publication'}, {'entity': 'subgroup', 'label': 'clinical trial'}, {'entity': '1999', 'label': 'year'}, {'entity': 'inexpensive', 'label': 'measurement'}, {'entity': 'blinding', 'label': 'method'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'structural motifs', 'label': 'protein'}, {'entity': 'α-and β-penetrenes', 'label': 'protein'}, {'entity': 'structures', 'label': 'protein'}, {'entity': 'β sheets', 'label': 'protein'}, {'entity': 'β sheet domain', 'label': 'protein'}, {'entity': 'endocytic', 'label': 'virus'}, {'entity': 'insect vector', 'label': 'virus'}, {'entity': 'rodent vector', 'label': 'virus'}, {'entity': 'β-penetrenes', 'label': 'protein'}, {'entity': 'surface glycoprotein', 'label': 'protein'}, {'entity': 'blosum50', 'label': 'measurement'}, {'entity': 'HDPsec', 'label': 'method'}, {'entity': 'TMpred', 'label': 'tool'}, {'entity': 'TMbase', 'label': 'database'}, {'entity': 'Membrane Protein eXplorer', 'label': 'tool'}, {'entity': 'Ebola virus', 'label': 'virus'}, {'entity': 'domain Ia', 'label': 'protein'}, {'entity': 'SIN E1 fusion peptide', 'label': 'protein'}, {'entity': 'TMpred', 'label': 'method'}, {'entity': 'domain IIb', 'label': 'protein'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'Wimley-White interfacial hydrophobicity scale', 'label': 'method'}, {'entity': 'WWIHS score', 'label': 'measurement'}, {'entity': 'pre-anchor', 'label': 'protein'}, {'entity': 'E of DEN', 'label': 'protein'}, {'entity': 'PHDsec', 'label': 'algorithm'}, {'entity': 'scaffold', 'label': 'protein'}, {'entity': 'β sheet', 'label': 'protein'}, {'entity': 'Bunyavirus Gc', 'label': 'protein'}, {'entity': 'phlebovirus Gc', 'label': 'virus'}, {'entity': 'MacMolly alignment', 'label': 'method'}, {'entity': 'carboxyl terminal', 'label': 'protein'}, {'entity': 'domain III', 'label': 'protein'}, {'entity': 'olive', 'label': 'organism'}, {'entity': 'sequence similarity', 'label': 'measurement'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'WWIHS scores', 'label': 'measurement'}, {'entity': 'structural', 'label': 'protein'}, {'entity': 'phlebovirus Gc', 'label': 'virus'}, {'entity': 'Stone', 'label': 'object'}, {'entity': 'Common order of proteins in Bunyavirus M segment polyproteins', 'label': 'protein'}, {'entity': 'TMpred', 'label': 'method'}, {'entity': 'divergent evolution', 'label': 'process'}, {'entity': 'internal fusion peptides', 'label': 'protein'}, {'entity': 'endocytic', 'label': 'virus'}, {'entity': 'trimers', 'label': 'protein'}, {'entity': 'segments', 'label': 'protein'}, {'entity': 'nipple', 'label': 'symptom'}, {'entity': 'domains I-III', 'label': 'protein'}, {'entity': '6E', 'label': 'measurement'}, {'entity': '6G', 'label': 'vaccine'}, {'entity': 'Gallaher HIV-1 TM fusion protein', 'label': 'protein'}, {'entity': \"5' end\", 'label': 'measurement'}, {'entity': 'mutational', 'label': 'virus'}, {'entity': 'strand synthesis', 'label': 'process'}, {'entity': \"1960's\", 'label': 'year'}, {'entity': 'Coronavirus replication', 'label': 'process'}, {'entity': \"5' end\", 'label': 'protein'}, {'entity': 'body TRS', 'label': 'gene'}, {'entity': 'sg mRNA', 'label': 'protein'}, {'entity': 'sg', 'label': 'gene'}, {'entity': \"Nucleotide content of individual HCoV-NL63 genes and the 5'/3' untranslated regions (UTR) Figure 2\", 'label': 'measurement'}, {'entity': 'gene order', 'label': 'gene'}, {'entity': 'strand RNA', 'label': 'virus'}, {'entity': \"5'\", 'label': 'gene'}, {'entity': \"3'\", 'label': 'gene'}, {'entity': 'G anti-peaks', 'label': 'protein'}, {'entity': 'Cumulative GC skew graph', 'label': 'diagram'}, {'entity': 'discontinuous transcription', 'label': 'method'}, {'entity': 'alternative mechanistic explanation', 'label': 'method'}, {'entity': 'Ucount', 'label': 'measurement'}, {'entity': \"3'\", 'label': 'gene'}, {'entity': 'strand synthesis', 'label': 'method'}, {'entity': 'leader', 'label': 'gene'}, {'entity': 'sg', 'label': 'protein'}, {'entity': \"3' terminus\", 'label': 'gene'}, {'entity': 'pathogenicity islands', 'label': 'gene'}, {'entity': \"3'\", 'label': 'gene'}, {'entity': \"3' one-third\", 'label': 'gene'}, {'entity': 'alternative mechanism', 'label': 'method'}, {'entity': 'sub-genomic', 'label': 'gene'}, {'entity': '1b', 'label': 'gene'}, {'entity': 'synonymous codon', 'label': 'gene'}, {'entity': 'genome drift', 'label': 'virus'}, {'entity': \"3'\", 'label': 'gene'}, {'entity': 'E sg', 'label': 'gene'}, {'entity': 'single stranded loop', 'label': 'protein'}, {'entity': 'motifs', 'label': 'protein'}, {'entity': 'sg', 'label': 'gene'}, {'entity': 'leader TRS', 'label': 'gene'}, {'entity': 'Expression levels of the HCoV-NL63 genomic and sg mRNAs Figure 8 Expression levels of the HCoV-NL63 genomic and sg mRNAs', 'label': 'measurement'}, {'entity': \"5'\", 'label': 'measurement'}, {'entity': 'Excel', 'label': 'software'}, {'entity': 'SPSS', 'label': 'software'}, {'entity': 'Cumulative GC-skew graph', 'label': 'diagnostic'}, {'entity': 'virus passage 7', 'label': 'virus'}, {'entity': 'MHV strain A59', 'label': 'virus'}, {'entity': 'DNA primers', 'label': 'gene'}, {'entity': \"5' primer\", 'label': 'method'}, {'entity': 'high stringency buffer', 'label': 'measurement'}, {'entity': 'sg mRNA', 'label': 'measurement'}, {'entity': 'homology', 'label': 'protein'}, {'entity': 'complementary', 'label': 'virus'}, {'entity': 'poly-A tail', 'label': 'measurement'}, {'entity': '908 bend', 'label': 'protein'}, {'entity': 'RNA 530 loop', 'label': 'protein'}, {'entity': 'SARS variants', 'label': 'virus'}, {'entity': 'helical base-pairing', 'label': 'protein'}, {'entity': 'color-coded brackets', 'label': 'method'}, {'entity': 'secondary structure', 'label': 'structure'}, {'entity': 's2m domain', 'label': 'protein'}, {'entity': 'canonical A-form', 'label': 'protein'}, {'entity': 'tertiary', 'label': 'protein'}, {'entity': 'Fold of the s2m RNA', 'label': 'protein'}, {'entity': 'GNRA tetraloop', 'label': 'protein'}, {'entity': 'U(25)', 'label': 'protein'}, {'entity': '908 kink', 'label': 'protein'}, {'entity': 'P6522', 'label': 'spacegroup'}, {'entity': 'GNRA tetraloop', 'label': 'protein'}, {'entity': 'Hoogsteen faces', 'label': 'protein'}, {'entity': 'A(33)', 'label': 'protein'}, {'entity': 'three-purine asymmetric bulge', 'label': 'protein'}, {'entity': 'seven-nucleotide asymmetric bubble', 'label': 'protein'}, {'entity': 'A(38)', 'label': 'gene'}, {'entity': 'Watson-Crick-like', 'label': 'protein'}, {'entity': 'adenosine platform motif', 'label': 'gene'}, {'entity': 'domain IV', 'label': 'pathway'}, {'entity': 'G(11), A(12), and A(33', 'label': 'gene'}, {'entity': 'tunnel', 'label': 'pathway'}, {'entity': 'tertiary', 'label': 'protein'}, {'entity': 'A(23)', 'label': 'protein'}, {'entity': 'C(27)', 'label': 'protein'}, {'entity': 'Figure 4B', 'label': 'figure'}, {'entity': 'exocyclic', 'label': 'protein'}, {'entity': '1-cyclohexyl-3-(2-morpholinoethyl)carbodiimide metho-p-toluene sulfonate', 'label': 'drug'}, {'entity': '39 UTR', 'label': 'gene'}, {'entity': 'anti-sense RNA', 'label': 'gene'}, {'entity': 'sequence complement', 'label': 'gene'}, {'entity': 'Watson-Crick', 'label': 'gene'}, {'entity': 'inter-molecular contacts', 'label': 'measurement'}, {'entity': 'nonessential', 'label': 'gene'}, {'entity': 'U(25)', 'label': 'protein'}, {'entity': 'pbio', 'label': 'publication'}, {'entity': 'helical interaction', 'label': 'pathway'}, {'entity': 'backbone', 'label': 'protein'}, {'entity': 'SARS s2m RNA', 'label': 'gene'}, {'entity': 'small', 'label': 'protein'}, {'entity': 'large', 'label': 'protein'}, {'entity': '530 stem-loop', 'label': 'protein'}, {'entity': '530 stem-loop', 'label': 'structure'}, {'entity': '530 loop', 'label': 'protein'}, {'entity': 'macromolecular mimicry', 'label': 'pathway'}, {'entity': 'testable', 'label': 'clinical trial'}, {'entity': 'OB fold', 'label': 'protein'}, {'entity': 'OB fold', 'label': 'protein'}, {'entity': 'single- strand', 'label': 'protein'}, {'entity': 'structural genomics', 'label': 'method'}, {'entity': 'OBlike', 'label': 'protein'}, {'entity': 's2m RNA Tunnel Is an Attractive Target for the Design of Anti-SARS Drugs', 'label': 'protein'}, {'entity': 'anti-SARS therapeutic', 'label': 'drug'}, {'entity': 'RNA transcript', 'label': 'virus'}, {'entity': 'Harker section', 'label': 'method'}, {'entity': 'phase extension', 'label': 'method'}, {'entity': 'poly-C', 'label': 'gene'}, {'entity': 'residue 48', 'label': 'protein'}, {'entity': 'MacPymol', 'label': 'software'}, {'entity': 'phasing', 'label': 'method'}, {'entity': 'Chemical probing', 'label': 'method'}, {'entity': '39', 'label': 'measurement'}, {'entity': 'accountability for reasonableness', 'label': 'method'}, {'entity': 'axial coding', 'label': 'method'}, {'entity': 'reasoning used', 'label': 'method'}, {'entity': 'hospital system', 'label': 'organization'}, {'entity': 'SARS in Toronto in 2003', 'label': 'disease'}, {'entity': 'legitimacy', 'label': 'concept'}, {'entity': 'Accountability for reasonableness', 'label': 'method'}, {'entity': 'relevance', 'label': 'clinical trial'}, {'entity': 'outbreak of SARS', 'label': 'event'}, {'entity': 'decision making', 'label': 'method'}, {'entity': 'accountability for reasonableness', 'label': 'method'}, {'entity': 'context-dependent', 'label': 'method'}, {'entity': \"four conditions of 'accountability for reasonableness'\", 'label': 'clinical trial'}, {'entity': 'key', 'label': 'person'}, {'entity': 'audiotaped', 'label': 'method'}, {'entity': 'pilot interview', 'label': 'method'}, {'entity': 'axial coding', 'label': 'method'}, {'entity': 'rationales', 'label': 'clinical trial'}, {'entity': 'idea', 'label': 'concept'}, {'entity': 'reasonableness', 'label': 'clinical trial'}, {'entity': 'good practice', 'label': 'clinical trial'}, {'entity': 'bias', 'label': 'method'}, {'entity': 'member check', 'label': 'method'}, {'entity': 'research team', 'label': 'organization'}, {'entity': 'supportive reasoning', 'label': 'method'}, {'entity': 'four conditions', 'label': 'clinical trial'}, {'entity': 'types of priority setting decisions', 'label': 'clinical trial'}, {'entity': 'contain the spread of the virus', 'label': 'treatment'}, {'entity': 'Ontario Ministry of Health and Long-Term Care', 'label': 'organization'}, {'entity': 'Second', 'label': 'event'}, {'entity': 'General medicine', 'label': 'disease'}, {'entity': 'intensivist', 'label': 'person'}, {'entity': 'operating room', 'label': 'location'}, {'entity': 'command center', 'label': 'organization'}, {'entity': 'nurse manager', 'label': 'person'}, {'entity': 'Priority setting reasoning', 'label': 'clinical trial'}, {'entity': 'Need', 'label': 'symptom'}, {'entity': 'emergent', 'label': 'symptom'}, {'entity': 'reasons used', 'label': 'clinical trial'}, {'entity': 'division', 'label': 'organization'}, {'entity': 'debate', 'label': 'event'}, {'entity': 'Sickly squeaky wheel method of appeal', 'label': 'method'}, {'entity': 'high level administration', 'label': 'organization'}, {'entity': 'families', 'label': 'person'}, {'entity': 'ethically', 'label': 'method'}, {'entity': 'trust', 'label': 'measurement'}, {'entity': 'crisis management', 'label': 'method'}, {'entity': 'four conditions', 'label': 'clinical trial'}, {'entity': 'accountability for reasonableness', 'label': 'concept'}, {'entity': 'life', 'label': 'immune response'}, {'entity': 'decision making culture', 'label': 'organization'}, {'entity': 'proliferative', 'label': 'signal'}, {'entity': 'organogenesis', 'label': 'pathway'}, {'entity': 'abnormalities', 'label': 'symptom'}, {'entity': 'ubiquitously expressed', 'label': 'measurement'}, {'entity': 'Ras-Raf-MEK-ERK pathway', 'label': 'pathway'}, {'entity': 'inositol ring', 'label': 'protein'}, {'entity': 'Ser', 'label': 'protein'}, {'entity': 'Forkhead', 'label': 'protein'}, {'entity': 'MAPK ERK1/2', 'label': 'protein'}, {'entity': 'DNA fragmentation analysis', 'label': 'measurement'}, {'entity': 'Ras-Raf-MEK-ERK pathway', 'label': 'pathway'}, {'entity': 'anti-tubulin', 'label': 'protein'}, {'entity': 'monolayer cell', 'label': 'cell'}, {'entity': 'RV-induced apoptotic signaling', 'label': 'pathway'}, {'entity': 'floaters', 'label': 'symptom'}, {'entity': 'RV-induced CPE', 'label': 'disease'}, {'entity': '3A', 'label': 'measurement'}, {'entity': 'D', 'label': 'virus'}, {'entity': 'capsid gene', 'label': 'gene'}, {'entity': '24 hours', 'label': 'measurement'}, {'entity': 'pUSEamp(+)', 'label': 'virus'}, {'entity': 'cellular phosphatase', 'label': 'protein'}, {'entity': 'apoptotic laddering', 'label': 'disease'}, {'entity': 'monolayer death', 'label': 'symptom'}, {'entity': 'survival signal', 'label': 'pathway'}, {'entity': 'RV replication', 'label': 'treatment'}, {'entity': 'DNA replication', 'label': 'virus'}, {'entity': 'MEK1/2', 'label': 'protein'}, {'entity': 'malignant cells', 'label': 'host'}, {'entity': '2000', 'label': 'publication'}, {'entity': 'modulate cellular responses', 'label': 'immune response'}, {'entity': 'CRS', 'label': 'disease'}, {'entity': 'Paisley', 'label': 'location'}, {'entity': 'Dako', 'label': 'organization'}, {'entity': 'culture plates', 'label': 'location'}, {'entity': 'test wavelength', 'label': 'measurement'}, {'entity': 'Nikon Eclipse TS100', 'label': 'method'}, {'entity': 'COOLPIX 4500', 'label': 'method'}, {'entity': \"5'-CCTGTACGTGGGGCCTTTAA-3'\", 'label': 'protein'}, {'entity': 'cycling conditions', 'label': 'method'}, {'entity': 'viral domain', 'label': 'protein'}, {'entity': 'functional aspects', 'label': 'treatment'}, {'entity': 'positive-strand', 'label': 'virus'}, {'entity': 'methylating', 'label': 'environment'}, {'entity': '2-oxoglutarate (2OG)-and Fe(II)dependent oxygenase superfamily', 'label': 'protein'}, {'entity': 'jelly roll', 'label': 'protein'}, {'entity': 'AlkB homologues', 'label': 'gene'}, {'entity': 'functional mapping', 'label': 'method'}, {'entity': 'H', 'label': 'protein'}, {'entity': 'checkpointing', 'label': 'immune response'}, {'entity': 'RNA repair', 'label': 'disease'}, {'entity': 'NCBI nr', 'label': 'database'}, {'entity': '11', 'label': 'measurement'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'HEL', 'label': 'protein'}, {'entity': 'aligned', 'label': 'method'}, {'entity': 'MT', 'label': 'protein'}, {'entity': 'MT', 'label': 'measurement'}, {'entity': 'AlkB tree', 'label': 'treatment'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'HEL', 'label': 'gene'}, {'entity': 'scatter plot', 'label': 'diagram'}, {'entity': 'nr database', 'label': 'publication'}, {'entity': 'grapevine', 'label': 'host'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'Multiple alignment of sequence regions', 'label': 'method'}, {'entity': 'evolutionary distance', 'label': 'measurement'}, {'entity': 'Blosum50 score matrix', 'label': 'method'}, {'entity': 'gi3702789', 'label': 'gene'}, {'entity': 'gi9630789', 'label': 'gene'}, {'entity': 'Nostoc sp', 'label': 'organism'}, {'entity': 'conserved', 'label': 'protein'}, {'entity': 'Potato virus M', 'label': 'virus'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'Potato virus M', 'label': 'virus'}, {'entity': 'Non-essential', 'label': 'gene'}, {'entity': 'Potato virus M', 'label': 'virus'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'truncated end', 'label': 'protein'}, {'entity': 'subfamilies', 'label': 'classification'}, {'entity': 'OTU', 'label': 'protein'}, {'entity': 'peptidase domain', 'label': 'protein'}, {'entity': 'Potato virus M', 'label': 'virus'}, {'entity': 'MT', 'label': 'protein'}, {'entity': 'OTU', 'label': 'protein'}, {'entity': 'HEL', 'label': 'protein'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'HEL', 'label': 'gene'}, {'entity': 'recombined', 'label': 'method'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'integration', 'label': 'event'}, {'entity': 'functional', 'label': 'method'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'HEL', 'label': 'protein'}, {'entity': 'OTU domain', 'label': 'gene'}, {'entity': 'peptidase domain', 'label': 'protein'}, {'entity': 'alternative explanation', 'label': 'method'}, {'entity': 'integrated domain', 'label': 'protein'}, {'entity': 'simulations', 'label': 'method'}, {'entity': 'repression of transcription', 'label': 'gene'}, {'entity': 'RNA interference', 'label': 'protein'}, {'entity': '2b', 'label': 'protein'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'RNA methylation', 'label': 'protein'}, {'entity': 'alternative hypothesis', 'label': 'concept'}, {'entity': 'source', 'label': 'virus'}, {'entity': 'grapevine', 'label': 'host'}, {'entity': 'viral replication complex', 'label': 'virus'}, {'entity': 'viral domain', 'label': 'protein'}, {'entity': 'effective mutation rate', 'label': 'measurement'}, {'entity': 'MT', 'label': 'gene'}, {'entity': 'HEL', 'label': 'protein'}, {'entity': 'divergence of most sequence', 'label': 'event'}, {'entity': 'noncytopatogenic', 'label': 'virus'}, {'entity': 'minus-strand synthesis', 'label': 'process'}, {'entity': 'PTGS system', 'label': 'disease'}, {'entity': 'groff', 'label': 'software'}, {'entity': 'reference sequence database', 'label': 'publication'}, {'entity': 'PLA General Hospital', 'label': 'organization'}, {'entity': 'transfer coefficient for carbon monoxide', 'label': 'measurement'}, {'entity': 'femoral head', 'label': 'location'}, {'entity': 'bones', 'label': 'immune response'}, {'entity': 'joints', 'label': 'symptom'}, {'entity': 'Clinical Diagnosis Standard for SARS Patients', 'label': 'publication'}, {'entity': 'high resolution computerized tomography', 'label': 'diagnostic'}, {'entity': 'femoral head', 'label': 'location'}, {'entity': 'femoral head', 'label': 'location'}, {'entity': 'cluster of cases of SARS', 'label': 'epidemiology'}, {'entity': 'secondary infected SARS cases', 'label': 'disease'}, {'entity': 'imaging profile', 'label': 'diagnostic'}, {'entity': '1+2+4', 'label': 'clinical trial'}, {'entity': 'diluent buffer', 'label': 'measurement'}, {'entity': 'washing buffer', 'label': 'measurement'}, {'entity': 'calibration curve', 'label': 'measurement'}, {'entity': 'Yorba Linda', 'label': 'location'}, {'entity': 'diffusion deficit', 'label': 'diagnostic'}, {'entity': 'Frontal chest X-ray', 'label': 'diagnostic'}, {'entity': 'GE Light Speed', 'label': 'organization'}, {'entity': 'inspiration', 'label': 'clinical trial'}, {'entity': 'septal', 'label': 'disease'}, {'entity': 'calcification', 'label': 'disease'}, {'entity': 'pleural', 'label': 'disease'}, {'entity': 'Milwaukee, WI', 'label': 'location'}, {'entity': 'bone', 'label': 'symptom'}, {'entity': 'coronal', 'label': 'measurement'}, {'entity': 'first follow-up visit', 'label': 'event'}, {'entity': 'femoral heads', 'label': 'location'}, {'entity': 'secondary', 'label': 'disease'}, {'entity': 'CDC SARS case definition', 'label': 'clinical trial'}, {'entity': 'SARS-CoV test', 'label': 'diagnostic'}, {'entity': 'SARS vaccine', 'label': 'vaccine'}, {'entity': 'ground-glass-like', 'label': 'symptom'}, {'entity': 'femoral head', 'label': 'location'}, {'entity': 'femoral head', 'label': 'location'}, {'entity': 'local kerotherapy', 'label': 'treatment'}, {'entity': 'emerging sequelae', 'label': 'disease'}, {'entity': 'Gram positive', 'label': 'measurement'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'secreted', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'signal peptide', 'label': 'protein'}, {'entity': 'Gram positive', 'label': 'gene'}, {'entity': 'digestive tract', 'label': 'location'}, {'entity': 'digestive tract', 'label': 'location'}, {'entity': 'signal peptide', 'label': 'protein'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'secreted', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'housekeeping protease', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'Clp complex', 'label': 'protein'}, {'entity': 'secreted', 'label': 'protein'}, {'entity': 'abortus ribosomal protein', 'label': 'protein'}, {'entity': 'Gram negative', 'label': 'virus'}, {'entity': 'L12', 'label': 'protein'}, {'entity': 'secreted form', 'label': 'protein'}, {'entity': 'cytoplasmic form', 'label': 'protein'}, {'entity': 'late-exponential phase', 'label': 'phase'}, {'entity': 'ovine IFN-ω', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'SP', 'label': 'protein'}, {'entity': 'homologous', 'label': 'vocabulary'}, {'entity': 'homologous', 'label': 'vocabulary'}, {'entity': 'Slayer', 'label': 'protein'}, {'entity': 'FedF', 'label': 'gene'}, {'entity': 'SP', 'label': 'protein'}, {'entity': 'L7', 'label': 'protein'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': 'SP Nuc', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'secreted', 'label': 'protein'}, {'entity': 'chaperone', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'method'}, {'entity': 'allergenic', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'GroEL', 'label': 'protein'}, {'entity': 'housekeeping protease', 'label': 'protein'}, {'entity': 'synthesis', 'label': 'protein'}, {'entity': 'SE', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'fusion', 'label': 'method'}, {'entity': 'PrtB carrier', 'label': 'protein'}, {'entity': 'stabilization effect', 'label': 'measurement'}, {'entity': 'heterodimeric', 'label': 'protein'}, {'entity': 'exponential', 'label': 'measurement'}, {'entity': 'house keeping protease', 'label': 'protein'}, {'entity': 'unfolded state', 'label': 'protein'}, {'entity': 'host factor', 'label': 'host'}, {'entity': 'housekeeping protease', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'protein'}, {'entity': 'Heterologous', 'label': 'protein'}, {'entity': 'exponential', 'label': 'growth phase'}, {'entity': 'stationary phase', 'label': 'growth phase'}, {'entity': 'exponential', 'label': 'measurement'}, {'entity': 'stationary', 'label': 'treatment'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'lactococcal Sec machinery', 'label': 'protein'}, {'entity': 'cell wall', 'label': 'structure'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'protein'}, {'entity': 'yields', 'label': 'measurement'}, {'entity': 'secretion process', 'label': 'process'}, {'entity': 'food-grade', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'health care worker', 'label': 'person'}, {'entity': 'hospital admission', 'label': 'treatment'}, {'entity': 'newborn', 'label': 'host'}, {'entity': 'English', 'label': 'language'}, {'entity': 'Braun Thermoscan', 'label': 'device'}, {'entity': 'clinically significant difference', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'human error', 'label': 'error'}, {'entity': 'Sixty', 'label': 'measurement'}, {'entity': 'males', 'label': 'person'}, {'entity': '5 minutes', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': '0.80', 'label': 'measurement'}, {'entity': 'home tympanic thermometer', 'label': 'measurement'}, {'entity': 'hospital tympanic thermometer', 'label': 'measurement'}, {'entity': \"St John's Hospital\", 'label': 'organization'}, {'entity': 'cooling', 'label': 'symptom'}, {'entity': '2011', 'label': 'publication'}, {'entity': 'mercury thermometer', 'label': 'measurement'}, {'entity': 'temporal artery thermometer', 'label': 'diagnostic'}, {'entity': 'limits of agreement', 'label': 'measurement'}, {'entity': 'Scatter plot', 'label': 'diagnostic'}, {'entity': 'trade liberalization', 'label': 'event'}, {'entity': 'empire', 'label': 'country'}, {'entity': 'international trade', 'label': 'organization'}, {'entity': 'trade liberalisation', 'label': 'event'}, {'entity': 'Darmstadt', 'label': 'location'}, {'entity': 'infected food products', 'label': 'disease'}, {'entity': 'sidebar', 'label': 'event'}, {'entity': 'ban', 'label': 'event'}, {'entity': 'dispute settlement', 'label': 'organization'}, {'entity': 'diffi cult', 'label': 'vocabulary'}, {'entity': 'trade lobby', 'label': 'organization'}, {'entity': 'insuffi cient', 'label': 'vocabulary'}, {'entity': 'Consumer groups', 'label': 'organization'}, {'entity': 'systemic impacts', 'label': 'pathway'}, {'entity': 'irreversible consequences', 'label': 'effect'}, {'entity': 'classification', 'label': 'term'}, {'entity': 'tobacco product', 'label': 'product'}, {'entity': 'Public health advocates', 'label': 'organization'}, {'entity': 'regional trade negotiations', 'label': 'event'}, {'entity': 'trade in services', 'label': 'organization'}, {'entity': 'trademark', 'label': 'vocabulary'}, {'entity': 'pharmaceutical industry', 'label': 'organization'}, {'entity': 'Doha Declaration on the TRIPS Agreement and Public Health', 'label': 'publication'}, {'entity': 'compulsory licensing', 'label': 'treatment'}, {'entity': 'continued disagreement', 'label': 'event'}, {'entity': 'breakdown in multilateral negotiations', 'label': 'event'}, {'entity': 'competitive', 'label': 'environment'}, {'entity': 'research and development', 'label': 'clinical trial'}, {'entity': 'marketing monopoly', 'label': 'vaccine'}, {'entity': 'e-commerce', 'label': 'vaccine'}, {'entity': 'managed competition', 'label': 'method'}, {'entity': 'ambulance', 'label': 'service'}, {'entity': 'nonprofi t', 'label': 'organization'}, {'entity': 'health-related service', 'label': 'service'}, {'entity': 'social security fund', 'label': 'organization'}, {'entity': 'minimum trade restrictiveness', 'label': 'measurement'}, {'entity': 'non-profit basis', 'label': 'organization'}, {'entity': 'restricting market forces', 'label': 'event'}, {'entity': 'international trade', 'label': 'organization'}, {'entity': 'multilateral agreements', 'label': 'organization'}, {'entity': 'value-based assumption', 'label': 'concept'}, {'entity': 'trade policy', 'label': 'method'}, {'entity': 'insuffi cient', 'label': 'measurement'}, {'entity': 'Watson-Crick base-pairing', 'label': 'method'}, {'entity': 'nucleotide analogues', 'label': 'drug'}, {'entity': 'non-base-paired', 'label': 'gene'}, {'entity': 'duration of action', 'label': 'measurement'}, {'entity': '5 0 end', 'label': 'gene'}, {'entity': 'commercial re-use', 'label': 'method'}, {'entity': \"3'endo\", 'label': 'protein'}, {'entity': 'A-form', 'label': 'structure'}, {'entity': 'Hørsholm', 'label': 'location'}, {'entity': 'SARS 1-3', 'label': 'virus'}, {'entity': 'non-overlapping system', 'label': 'method'}, {'entity': 'Madison WI', 'label': 'location'}, {'entity': 'PTP-6 Peltier System', 'label': 'equipment'}, {'entity': 'denatured', 'label': 'method'}, {'entity': 'slit', 'label': 'measurement'}, {'entity': 'PC12', 'label': 'cell line'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'transfection mix', 'label': 'method'}, {'entity': 'unrelated siRNA', 'label': 'vaccine'}, {'entity': 'SARS 3', 'label': 'gene'}, {'entity': 'Offenburg', 'label': 'city'}, {'entity': 'ABI Prism SDS Software', 'label': 'method'}, {'entity': 'TBE', 'label': 'measurement'}, {'entity': '3 0 end protected', 'label': 'measurement'}, {'entity': '3 0', 'label': 'measurement'}, {'entity': '3 0', 'label': 'measurement'}, {'entity': 'EC 50', 'label': 'measurement'}, {'entity': \"5' antisense\", 'label': 'gene'}, {'entity': 'Figure 2c', 'label': 'publication'}, {'entity': '5 0', 'label': 'measurement'}, {'entity': \"3' sense end\", 'label': 'gene'}, {'entity': '3 0', 'label': 'gene'}, {'entity': \"5' antisense end\", 'label': 'gene'}, {'entity': 'catalytic site', 'label': 'protein'}, {'entity': 'A-form', 'label': 'protein'}, {'entity': 'effective', 'label': 'measurement'}, {'entity': 'triplicate', 'label': 'measurement'}, {'entity': 'complementary', 'label': 'gene'}, {'entity': '5 0 sense end', 'label': 'gene'}, {'entity': 'sense', 'label': 'gene'}, {'entity': 'SARS siRNA', 'label': 'virus'}, {'entity': 'strand activity', 'label': 'measurement'}, {'entity': '5(23)', 'label': 'publication'}, {'entity': '5 0 sense end', 'label': 'gene'}, {'entity': 'sense strands', 'label': 'gene'}, {'entity': 'strand-bias', 'label': 'measurement'}, {'entity': '5 0 sense position', 'label': 'measurement'}, {'entity': 'indirect', 'label': 'method'}, {'entity': \"5' sense end\", 'label': 'gene'}, {'entity': 'permissive', 'label': 'measurement'}, {'entity': 'SARS 1-3', 'label': 'gene'}, {'entity': 'triplicate', 'label': 'measurement'}, {'entity': 'new ones', 'label': 'clinical trial'}, {'entity': 'Last Universal Common Ancestor', 'label': 'gene'}, {'entity': 'prokaryote', 'label': 'organism'}, {'entity': 'dramatic homologies', 'label': 'measurement'}, {'entity': '20', 'label': 'publication'}, {'entity': '21', 'label': 'publication'}, {'entity': '16', 'label': 'measurement'}, {'entity': 'species of origin', 'label': 'gene'}, {'entity': 'singularities', 'label': 'measurement'}, {'entity': 'speciesspecific', 'label': 'gene'}, {'entity': 'regions of interest', 'label': 'gene'}, {'entity': 'short oligonucleotide', 'label': 'gene'}, {'entity': 'Chaos Game Representation', 'label': 'method'}, {'entity': 'word thereafter', 'label': 'measurement'}, {'entity': '12', 'label': 'measurement'}, {'entity': 'archaea', 'label': 'virus'}, {'entity': 'eukarya', 'label': 'organism'}, {'entity': 'local signature', 'label': 'measurement'}, {'entity': 'speciesspecific', 'label': 'measurement'}, {'entity': 'small window', 'label': 'measurement'}, {'entity': 'local signature', 'label': 'method'}, {'entity': 'lines', 'label': 'method'}, {'entity': 'columns', 'label': 'method'}, {'entity': 'recipient species signature', 'label': 'gene'}, {'entity': 'genome signature', 'label': 'measurement'}, {'entity': 'clusters of local signatures', 'label': 'treatment'}, {'entity': 'genome signature', 'label': 'gene'}, {'entity': 'high dimensional', 'label': 'measurement'}, {'entity': 'square root', 'label': 'measurement'}, {'entity': 'host signature', 'label': 'host'}, {'entity': 'distribution of distances', 'label': 'measurement'}, {'entity': 'isolated signature', 'label': 'virus'}, {'entity': 'Genome Information Broker', 'label': 'organization'}, {'entity': 'BlastN search', 'label': 'method'}, {'entity': 'atypical region', 'label': 'region'}, {'entity': 'donor-compatible', 'label': 'measurement'}, {'entity': 'fluctuations', 'label': 'measurement'}, {'entity': 'host signature', 'label': 'signature'}, {'entity': 'mean distance to the host', 'label': 'measurement'}, {'entity': 'most populated class', 'label': 'clustering'}, {'entity': 'Figure 2b', 'label': 'figure'}, {'entity': 'host signature', 'label': 'signature'}, {'entity': 'B.subtilis typical', 'label': 'gene'}, {'entity': 'clusters', 'label': 'pathway'}, {'entity': 'intra-genomic range', 'label': 'measurement'}, {'entity': 'small values', 'label': 'measurement'}, {'entity': 'contrasted results', 'label': 'diagnostic'}, {'entity': 'homologous', 'label': 'gene'}, {'entity': 'bacteriophage gene', 'label': 'gene'}, {'entity': 'species signature', 'label': 'host'}, {'entity': 'H.influenzae signature', 'label': 'measurement'}, {'entity': 'Mean distance to host and cut-off distance', 'label': 'measurement'}, {'entity': '5 kb', 'label': 'measurement'}, {'entity': 'typical', 'label': 'clinical trial'}, {'entity': 'species signature', 'label': 'host'}, {'entity': 'comparison with literature', 'label': 'method'}, {'entity': 'caveats', 'label': 'measurement'}, {'entity': 'Lawrence and Ochman', 'label': 'person'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'Kappa of one', 'label': 'measurement'}, {'entity': 'Kappa of zero', 'label': 'measurement'}, {'entity': 'consensus set', 'label': 'gene'}, {'entity': 'Kappas', 'label': 'measurement'}, {'entity': 'Nakamura approach', 'label': 'method'}, {'entity': 'every other approach', 'label': 'method'}, {'entity': 'long', 'label': 'measurement'}, {'entity': 'method-specific', 'label': 'measurement'}, {'entity': 'single vote', 'label': 'gene'}, {'entity': \"'single vote' genes\", 'label': 'gene'}, {'entity': 'single vote gene', 'label': 'gene'}, {'entity': 'single vote', 'label': 'gene'}, {'entity': 'atypicality', 'label': 'gene'}, {'entity': 'amelioration', 'label': 'gene'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'K12 strains', 'label': 'strain'}, {'entity': 'clustered', 'label': 'method'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'strains', 'label': 'gene'}, {'entity': 'strainspecific', 'label': 'gene'}, {'entity': 'homologous', 'label': 'gene'}, {'entity': '12 000 signature database', 'label': 'organization'}, {'entity': 'Ter', 'label': 'gene'}, {'entity': 'caveats', 'label': 'clinical trial'}, {'entity': 'ameliorated', 'label': 'clinical trial'}, {'entity': 'screening window', 'label': 'measurement'}, {'entity': '1 kb', 'label': 'measurement'}, {'entity': 'atypical', 'label': 'measurement'}, {'entity': 'G+C% atypicality', 'label': 'measurement'}, {'entity': 'Nicolas et al.', 'label': 'organization'}, {'entity': 'outlier', 'label': 'measurement'}, {'entity': 'atypical signature', 'label': 'measurement'}, {'entity': 'HMM', 'label': 'method'}, {'entity': 'atypical', 'label': 'disease'}, {'entity': 'learning process', 'label': 'method'}, {'entity': 'vicinity', 'label': 'location'}, {'entity': 'signature database', 'label': 'organization'}, {'entity': 'transfer event', 'label': 'event'}, {'entity': 'outliers', 'label': 'virus'}, {'entity': 'coding regions', 'label': 'gene'}, {'entity': 'strains', 'label': 'host'}, {'entity': 'species of origin', 'label': 'virus'}, {'entity': 'mechanisms of evolution', 'label': 'concept'}, {'entity': 'species specificity', 'label': 'gene'}, {'entity': 'environmental considerations', 'label': 'environment'}, {'entity': 'signature method', 'label': 'method'}, {'entity': '49', 'label': 'publication'}, {'entity': 'bias', 'label': 'measurement'}, {'entity': 'species specificity', 'label': 'virus'}, {'entity': 'filtering criteria', 'label': 'method'}, {'entity': 'oligo nucleotide microarrays', 'label': 'method'}, {'entity': 'Pearson Correlation factor', 'label': 'measurement'}, {'entity': \"Student's unpaired T-test\", 'label': 'method'}, {'entity': 'genetic profile', 'label': 'gene'}, {'entity': 'Bactericidal Permeability Increasing Protein', 'label': 'protein'}, {'entity': 'Gram negative', 'label': 'disease'}, {'entity': 'Immune-response related genes', 'label': 'gene'}, {'entity': 'DNA synthesis', 'label': 'gene'}, {'entity': 'Lactoferrin Lipocalin', 'label': 'gene'}, {'entity': 'genetic profile', 'label': 'gene'}, {'entity': 'fragmentation buffer', 'label': 'method'}, {'entity': 'HG-Focus Array', 'label': 'diagnostic'}, {'entity': 'differentially expressed', 'label': 'measurement'}, {'entity': 'differentially expressed', 'label': 'gene'}, {'entity': 'pathogenesis of SARS', 'label': 'pathway'}, {'entity': '2002', 'label': 'year'}, {'entity': 'development', 'label': 'treatment'}, {'entity': 'potential treatment', 'label': 'treatment'}, {'entity': 'molecular mechanisms', 'label': 'treatment'}, {'entity': 'gastrointestinal', 'label': 'disease'}, {'entity': \"3' co-terminal\", 'label': 'gene'}, {'entity': 'southern', 'label': 'region'}, {'entity': 'jump', 'label': 'transmission'}, {'entity': 'human coronavirus OC43', 'label': 'virus'}, {'entity': 'human coronavirus 229E', 'label': 'virus'}, {'entity': 'chicken infectious bronchitis virus', 'label': 'virus'}, {'entity': 'N-terminal domain', 'label': 'protein'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'coding sequences', 'label': 'gene'}, {'entity': 'initial responses', 'label': 'protein'}, {'entity': 'gene arrangement', 'label': 'gene'}, {'entity': 'virus-encoded proteinase', 'label': 'protein'}, {'entity': 'research efforts', 'label': 'clinical trial'}, {'entity': 'unconventionally translated', 'label': 'method'}, {'entity': 'SARS-CoV virulence', 'label': 'virus'}, {'entity': 'genome expression', 'label': 'pathway'}, {'entity': 'coronavirus fusion', 'label': 'protein'}, {'entity': 'envelope and membrane proteins', 'label': 'protein'}, {'entity': '2003 epidemy', 'label': 'event'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'chronic carrier', 'label': 'disease'}, {'entity': 'persistent infection', 'label': 'disease'}, {'entity': 'small intestine', 'label': 'location'}, {'entity': 'bi-or triphasic', 'label': 'clinical trial'}, {'entity': 'second phase', 'label': 'clinical trial'}, {'entity': 'third phase', 'label': 'event'}, {'entity': 'critical phase', 'label': 'disease'}, {'entity': 'children younger than 12 years', 'label': 'host'}, {'entity': 'developmental stage', 'label': 'immune response'}, {'entity': 'first-generation RT-PCR assay', 'label': 'diagnostic'}, {'entity': 'best treatment strategy for SARS among children', 'label': 'treatment'}, {'entity': 'short-to medium-term', 'label': 'treatment'}, {'entity': 'long-term complications', 'label': 'symptom'}, {'entity': 'cytocidal', 'label': 'disease'}, {'entity': 'viral replication complex', 'label': 'protein'}, {'entity': 'adaptive', 'label': 'immune response'}, {'entity': 'spike(S)', 'label': 'protein'}, {'entity': 'respiratory epithelial cells', 'label': 'cell'}, {'entity': 'respiratory tract', 'label': 'disease'}, {'entity': 'S1 domain', 'label': 'protein'}, {'entity': 'S1 domain', 'label': 'protein'}, {'entity': 'SARS-CoV fusion', 'label': 'protein'}, {'entity': 'SARS-CoV RdRp', 'label': 'protein'}, {'entity': 'endo sugar puckering', 'label': 'protein'}, {'entity': 'human immunodeficiency virus type 1 reverse transcriptase', 'label': 'virus'}, {'entity': 'RNA replication', 'label': 'virus'}, {'entity': 'viral polyproteins', 'label': 'protein'}, {'entity': 'SARS-unique domain', 'label': 'protein'}, {'entity': 'α-helical', 'label': 'protein'}, {'entity': 'catalytically incompetent conformation', 'label': 'protein'}, {'entity': 'catalytically incompetent', 'label': 'protein'}, {'entity': 'M pro', 'label': 'protein'}, {'entity': 'active monomer', 'label': 'protein'}, {'entity': 'transmissible gastroenteritis virus', 'label': 'virus'}, {'entity': 'L-700,417', 'label': 'drug'}, {'entity': 'experimentally', 'label': 'method'}, {'entity': 'intermediate compartment', 'label': 'pathway'}, {'entity': 'secretory pathways', 'label': 'pathway'}, {'entity': 'endogenous', 'label': 'drug'}, {'entity': '21-mer', 'label': 'measurement'}, {'entity': 'Live-attenuated coronavirus vaccine', 'label': 'vaccine'}, {'entity': 'attenuate', 'label': 'virus'}, {'entity': 'coronavirus vaccine', 'label': 'vaccine'}, {'entity': 'S1 domain', 'label': 'protein'}, {'entity': 'cellmediated', 'label': 'immune response'}, {'entity': 'coronavirus vaccination', 'label': 'vaccine'}, {'entity': 'potential variations', 'label': 'virus'}, {'entity': 'coronavirus vaccine', 'label': 'vaccine'}, {'entity': 'bridged conjugation', 'label': 'method'}, {'entity': 'Viro-chip', 'label': 'diagnostic'}, {'entity': 'solid-phase array', 'label': 'method'}, {'entity': 'multiplexed', 'label': 'method'}, {'entity': 'secondary reagent', 'label': 'method'}, {'entity': 'beacon-conjugated', 'label': 'protein'}, {'entity': 'diagnostic capabilities', 'label': 'diagnostic'}, {'entity': 'complementary', 'label': 'gene'}, {'entity': 'loop sequence', 'label': 'gene'}, {'entity': 'beacon sequence', 'label': 'measurement'}, {'entity': 'Biolegio', 'label': 'organization'}, {'entity': 'complementary', 'label': 'gene'}, {'entity': 'complementary sequence', 'label': 'virus'}, {'entity': 'Target DNA', 'label': 'measurement'}, {'entity': 'beacon', 'label': 'measurement'}, {'entity': 'biotinylated', 'label': 'measurement'}, {'entity': 'doublet', 'label': 'virus'}, {'entity': 'HIV control', 'label': 'disease'}, {'entity': 'limit of detection', 'label': 'measurement'}, {'entity': 'HCV sequence', 'label': 'gene'}, {'entity': '80 C', 'label': 'measurement'}, {'entity': 'complementary positive control', 'label': 'diagnostic'}, {'entity': 'complete mismatch oligo', 'label': 'measurement'}, {'entity': '-N', 'label': 'virus'}, {'entity': 'beacon sequence', 'label': 'method'}, {'entity': 'representative dot plot', 'label': 'diagram'}, {'entity': 'multiplex system', 'label': 'diagnostic'}, {'entity': 'multiplex', 'label': 'method'}, {'entity': 'complementary', 'label': 'gene'}, {'entity': 'solid state nucleic acid detection', 'label': 'method'}, {'entity': 'window of discrimination', 'label': 'measurement'}, {'entity': 'homologous sequence', 'label': 'gene'}, {'entity': \"Gibb's free energy assignments\", 'label': 'measurement'}, {'entity': 'Unique sequence', 'label': 'measurement'}, {'entity': 'complex mixture', 'label': 'measurement'}, {'entity': 'control gene', 'label': 'gene'}, {'entity': 'GAP', 'label': 'gene'}, {'entity': 'Act', 'label': 'gene'}, {'entity': 'Act', 'label': 'gene'}, {'entity': 'PPI', 'label': 'gene'}, {'entity': 'PPI', 'label': 'gene'}, {'entity': 'GAP', 'label': 'gene'}, {'entity': 'Act', 'label': 'gene'}, {'entity': 'PPI', 'label': 'gene'}, {'entity': 'GAP', 'label': 'gene'}, {'entity': 'Act', 'label': 'gene'}, {'entity': 'PPI', 'label': 'gene'}, {'entity': 'Capa-ble', 'label': 'measurement'}, {'entity': 'PPI', 'label': 'gene'}, {'entity': 'GAP', 'label': 'gene'}, {'entity': 'PPI', 'label': 'protein'}, {'entity': 'gene expression studies', 'label': 'method'}, {'entity': 'Huntingdon', 'label': 'location'}, {'entity': 'Perkin Elmer 7700 Sequence Detection System', 'label': 'method'}, {'entity': 'SARS associated coronavirus', 'label': 'virus'}, {'entity': 'enteric', 'label': 'disease'}, {'entity': 'human coronavirus 229E', 'label': 'virus'}, {'entity': 'winter', 'label': 'season'}, {'entity': 'genome organisation', 'label': 'gene'}, {'entity': 'cocirculating', 'label': 'virus'}, {'entity': '1 month', 'label': 'measurement'}, {'entity': 'RSV epidemic period', 'label': 'epidemiology'}, {'entity': 'Westburg', 'label': 'organization'}, {'entity': 'Foster City, CA', 'label': 'location'}, {'entity': 'one-step RT-PCR', 'label': 'method'}, {'entity': '50°C', 'label': 'measurement'}, {'entity': 'RT-nested', 'label': 'method'}, {'entity': 'outer PCR product', 'label': 'measurement'}, {'entity': 'HCoV-NL63 positive control', 'label': 'virus'}, {'entity': 'NCBI BLAST', 'label': 'method'}, {'entity': 'GeneDoc Alignment', 'label': 'software'}, {'entity': 'MEGA', 'label': 'software'}, {'entity': 'epidemic period', 'label': 'epidemiology'}, {'entity': 'type B', 'label': 'virus'}, {'entity': '1 month', 'label': 'person'}, {'entity': 'underlying disease', 'label': 'disease'}, {'entity': 'neighbor-joining', 'label': 'method'}, {'entity': 'RT-nested', 'label': 'method'}, {'entity': 'neighbor-joining', 'label': 'method'}, {'entity': 'bootstrap', 'label': 'method'}, {'entity': 'severe respiratory illness', 'label': 'disease'}, {'entity': '1 month', 'label': 'measurement'}, {'entity': 'seasonal outbreak', 'label': 'epidemiology'}, {'entity': 'February 2003', 'label': 'measurement'}, {'entity': 'winter', 'label': 'season'}, {'entity': 'summer', 'label': 'season'}, {'entity': '2001', 'label': 'year'}, {'entity': 'GenBank Figure 5', 'label': 'publication'}, {'entity': 'cocirculating', 'label': 'virus'}, {'entity': 'spike insert region', 'label': 'gene'}, {'entity': '1988', 'label': 'year'}, {'entity': 'winter', 'label': 'epidemiology'}, {'entity': 'coronavirus group 1', 'label': 'virus'}, {'entity': 'severe respiratory infections', 'label': 'disease'}, {'entity': '1988', 'label': 'year'}, {'entity': 'cocirculating', 'label': 'virus'}, {'entity': 'two base deletions', 'label': 'measurement'}, {'entity': 'megabases', 'label': 'measurement'}, {'entity': 'heterozygous', 'label': 'gene'}, {'entity': 'sequence change', 'label': 'gene'}, {'entity': 'journals oupjournals', 'label': 'organization'}, {'entity': 'journals oupjournals', 'label': 'organization'}, {'entity': 'non-enzymatic', 'label': 'method'}, {'entity': 'ATM', 'label': 'gene'}, {'entity': '3161 C/G', 'label': ' SNP'}, {'entity': 'bases', 'label': 'measurement'}, {'entity': 'position', 'label': 'gene'}, {'entity': 'geometric mean', 'label': 'measurement'}, {'entity': 'antilog', 'label': 'measurement'}, {'entity': 'ATM base', 'label': 'measurement'}, {'entity': 'base', 'label': 'measurement'}, {'entity': 'moving average', 'label': 'measurement'}, {'entity': 'insertions', 'label': 'gene'}, {'entity': 'ATM', 'label': 'gene'}, {'entity': 'PM', 'label': 'measurement'}, {'entity': 'hybridization specificity ratio', 'label': 'measurement'}, {'entity': 'single base substitution', 'label': 'measurement'}, {'entity': 'specificity bin', 'label': 'measurement'}, {'entity': 'ATM', 'label': 'gene'}, {'entity': 'confounding factors', 'label': 'measurement'}, {'entity': 'sense', 'label': 'measurement'}, {'entity': 'base deletion', 'label': 'measurement'}, {'entity': 'hybridization specificity ratio', 'label': 'measurement'}, {'entity': 'homopolymer', 'label': 'gene'}, {'entity': 'homopolymeric', 'label': 'gene'}, {'entity': 'wild-type target', 'label': 'disease'}, {'entity': 'triple helices', 'label': 'pathway'}, {'entity': 'homopyrimidine tract', 'label': 'gene'}, {'entity': 'potential', 'label': 'measurement'}, {'entity': 'mismatch probe hybridization', 'label': 'measurement'}, {'entity': 'single base substitution', 'label': 'measurement'}, {'entity': 'one', 'label': 'measurement'}, {'entity': 'sense', 'label': 'gene'}, {'entity': 'base deletion', 'label': 'measurement'}, {'entity': 'insertion and deletion probes', 'label': 'gene'}, {'entity': 'bulges', 'label': 'measurement'}, {'entity': 'nucleotide mismatches', 'label': 'virus'}, {'entity': 'mismatch', 'label': 'measurement'}, {'entity': '40', 'label': 'measurement'}, {'entity': 'DG', 'label': 'measurement'}, {'entity': 'weak', 'label': 'measurement'}, {'entity': '(T)5', 'label': 'sequence'}, {'entity': 'arrayed probe', 'label': 'measurement'}, {'entity': 'insertion', 'label': 'method'}, {'entity': 'dye incorporation', 'label': 'measurement'}, {'entity': 'further investigations', 'label': 'clinical trial'}, {'entity': 'Base', 'label': 'gene'}, {'entity': 'sequence change', 'label': 'measurement'}, {'entity': 'cytoplasmic tail', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': 'accessory', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'endocytotic', 'label': 'protein'}, {'entity': 'SARS-CoV S', 'label': 'protein'}, {'entity': 'ExD', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'SARS-CoV 3a', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'protein domain', 'label': 'protein'}, {'entity': 'endocytotic', 'label': 'protein'}, {'entity': 'syncytium', 'label': 'disease'}, {'entity': 'life cycle', 'label': 'virus'}, {'entity': 'HEV', 'label': 'protein'}, {'entity': 'H', 'label': 'protein'}, {'entity': 'trafficking', 'label': 'pathway'}, {'entity': 'declare', 'label': 'method'}, {'entity': 'unmet', 'label': 'disease'}, {'entity': 'technology', 'label': 'method'}, {'entity': '1990', 'label': 'year'}, {'entity': 'research spending', 'label': 'measurement'}, {'entity': 'generic competition', 'label': 'treatment'}, {'entity': 'access to lifesaving medicines', 'label': 'vaccine'}, {'entity': 'treaty', 'label': 'organization'}, {'entity': 'imitation', 'label': 'drug'}, {'entity': 'poorest', 'label': 'person'}, {'entity': '2002', 'label': 'year'}, {'entity': 'welfare losses', 'label': 'measurement'}, {'entity': 'marketing', 'label': 'drug'}, {'entity': 'blockbuster business model', 'label': 'method'}, {'entity': 'tax breaks', 'label': 'measurement'}, {'entity': 'Doha Declaration on TRIPS and Public Health', 'label': 'organization'}, {'entity': 'drug costs high', 'label': 'measurement'}, {'entity': 'crisis in pharmaceutical innovation', 'label': 'event'}, {'entity': 'international solution', 'label': 'method'}, {'entity': 'defense', 'label': 'organization'}, {'entity': 'mobilize', 'label': 'method'}, {'entity': 'infectious disease monitoring', 'label': 'disease'}, {'entity': 'international treaty', 'label': 'organization'}, {'entity': 'innovation', 'label': 'method'}, {'entity': 'drug prices', 'label': 'vaccine'}, {'entity': 'Profi t Motive', 'label': 'concept'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'drug development', 'label': 'drug'}, {'entity': 'Afi can sleeping sickness', 'label': 'disease'}, {'entity': 'knowledge-sharing', 'label': 'method'}, {'entity': 'developing new drugs', 'label': 'treatment'}, {'entity': 'Cambia', 'label': 'organization'}, {'entity': 'drug development', 'label': 'drug'}, {'entity': 'in-kind contributions', 'label': 'treatment'}, {'entity': 'patent abuse on drugs', 'label': 'vaccine'}, {'entity': 'diffi culties', 'label': 'event'}, {'entity': 'anthrax scare in the US', 'label': 'event'}, {'entity': 'treaty on plant genetic resources', 'label': 'organization'}, {'entity': 'development strategy', 'label': 'clinical trial'}, {'entity': 'research community', 'label': 'organization'}, {'entity': 'diffi cult', 'label': 'measurement'}, {'entity': 'science', 'label': 'clinical trial'}, {'entity': 'treaty', 'label': 'organization'}, {'entity': 'Singapore outbreak', 'label': 'event'}, {'entity': 'acquisition of SARS', 'label': 'disease'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'Singapore outbreak', 'label': 'event'}, {'entity': 'acquisition of SARS', 'label': 'event'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'infectious disease outbreak', 'label': 'event'}, {'entity': 'worldwide effort', 'label': 'event'}, {'entity': 'sensitivity study', 'label': 'clinical trial'}, {'entity': 'Singapore SARS outbreak', 'label': 'event'}, {'entity': 'probable SARS', 'label': 'disease'}, {'entity': 'endotracheal tube', 'label': 'vessel'}, {'entity': 'isolated', 'label': 'virus'}, {'entity': 'Valencia, California', 'label': 'location'}, {'entity': 'Carlsbad, California', 'label': 'city'}, {'entity': 'ethanol precipitation', 'label': 'method'}, {'entity': 'Lightcycler Sars-CoV quantification kit', 'label': 'vaccine'}, {'entity': 'San Diego, California', 'label': 'location'}, {'entity': 'human blood samples', 'label': 'sample'}, {'entity': '95% probability of detection', 'label': 'measurement'}, {'entity': 'T:C:T', 'label': 'sequence variant'}, {'entity': 'C:C:C', 'label': 'sequence'}, {'entity': 'fifth-and six-generation isolate', 'label': 'virus'}, {'entity': 'T T', 'label': 'measurement'}, {'entity': 'primary tissue', 'label': 'sample'}, {'entity': 'T/C:T/C:T/C variant', 'label': 'gene'}, {'entity': 'T/C:T/C:T/C', 'label': 'variant'}, {'entity': 'T:T:C', 'label': 'variant'}, {'entity': 'novel sequence', 'label': 'virus'}, {'entity': 'genotyped', 'label': 'method'}, {'entity': 'SARS outbreak', 'label': 'epidemiology'}, {'entity': 'Singapore outbreak', 'label': 'event'}, {'entity': 'Singapore SARS outbreak', 'label': 'event'}, {'entity': 'index case', 'label': 'virus'}, {'entity': 'reverse mutation', 'label': 'virus'}, {'entity': 'Hotel M', 'label': 'location'}, {'entity': 'novel mutation', 'label': 'virus'}, {'entity': 'Sin2679 contracted SARS', 'label': 'virus'}, {'entity': 'early-generation ones', 'label': 'clinical trial'}, {'entity': 'Hong Kong outbreak', 'label': 'event'}, {'entity': 'molecular fingerprint', 'label': 'measurement'}, {'entity': 'MSbased genetic analysis', 'label': 'method'}, {'entity': 'multiplexing', 'label': 'method'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'SARS-CoV transmission route in Singapore', 'label': 'transmission'}, {'entity': '6', 'label': 'measurement'}, {'entity': 'spermatogenic', 'label': 'gene'}, {'entity': 'determinant', 'label': 'protein'}, {'entity': 'cellular differentiation', 'label': 'treatment'}, {'entity': '1N DNA content cell', 'label': 'cell'}, {'entity': 'germ cell differentiation', 'label': 'pathway'}, {'entity': 'interstitial cells', 'label': 'cell'}, {'entity': 'differential display', 'label': 'method'}, {'entity': 'excised', 'label': 'method'}, {'entity': 'Human Testis Large-Insert cDNA Library', 'label': 'organization'}, {'entity': 'Cyclone', 'label': 'organization'}, {'entity': 'separation', 'label': 'method'}, {'entity': 'hybridization buffer', 'label': 'measurement'}, {'entity': 'BxPC-3 cell', 'label': 'cell'}, {'entity': 'fluorescence microscope', 'label': 'method'}, {'entity': 'sialic acid 9-O-acetylesterase', 'label': 'protein'}, {'entity': 'pattern', 'label': 'diagnostic'}, {'entity': 'colon', 'label': 'location'}, {'entity': 'small intestine', 'label': 'location'}, {'entity': 'germ cell', 'label': 'cell'}, {'entity': 'mitotic', 'label': 'phase'}, {'entity': 'chromosome pairing', 'label': 'gene'}, {'entity': 'cross-talk', 'label': 'method'}, {'entity': 'cellular communications', 'label': 'pathway'}, {'entity': 'spermatogenic', 'label': 'clinical trial'}, {'entity': '2C cells', 'label': 'cell'}, {'entity': 'primary spermatocyte', 'label': 'cell'}, {'entity': 'germ cell', 'label': 'cell'}, {'entity': 'G N F T Y M S A V C W L Y D T L Y D T L Q Y I E V W S R R T L K A C G V P N T 143 m-Lse 181', 'label': 'protein'}, {'entity': 'embryo development', 'label': 'pathway'}, {'entity': 'retina', 'label': 'organ'}, {'entity': 'B lymphocyte', 'label': 'cell'}, {'entity': 'differentiation', 'label': 'clinical trial'}, {'entity': 'differentiation', 'label': 'pathway'}, {'entity': 'germ cell', 'label': 'cell'}, {'entity': 'germ cell differentiation', 'label': 'pathway'}, {'entity': 'viral RNA polymerase', 'label': 'protein'}, {'entity': 'crossover', 'label': 'gene'}, {'entity': 'copychoice model', 'label': 'method'}, {'entity': 'fitness', 'label': 'measurement'}, {'entity': 'lines', 'label': 'method'}, {'entity': 'samplinglike', 'label': 'method'}, {'entity': 'passaged', 'label': 'virus'}, {'entity': 'hot-spot', 'label': 'virus'}, {'entity': 'non-canonical', 'label': 'protein'}, {'entity': 'RNA folding', 'label': 'protein'}, {'entity': \"2003 of significant developments in Australia's official international health relations\", 'label': 'event'}, {'entity': 'Australian foreign policy', 'label': 'organization'}, {'entity': '2003', 'label': 'year'}, {'entity': 'treaties', 'label': 'agreement'}, {'entity': 'international treaties', 'label': 'organization'}, {'entity': 'World Health Assembly of WHO', 'label': 'organization'}, {'entity': 'tobacco epidemic', 'label': 'disease'}, {'entity': 'levels of government', 'label': 'organization'}, {'entity': 'health care agreement', 'label': 'treatment'}, {'entity': 'cost neutral', 'label': 'measurement'}, {'entity': 'insurance premium', 'label': 'measurement'}, {'entity': 'treaty', 'label': 'organization'}, {'entity': 'complementary medicines', 'label': 'drug'}, {'entity': 'New Zealand counterpart', 'label': 'organization'}, {'entity': 'Food Standards Australia New Zealand', 'label': 'organization'}, {'entity': '2003', 'label': 'year'}, {'entity': 'Sixth Australia Indonesia Ministerial Forum', 'label': 'event'}, {'entity': 'Australia Japan Conference', 'label': 'event'}, {'entity': 'preferential trade agreements', 'label': 'organization'}, {'entity': 'intersectoral', 'label': 'organization'}, {'entity': 'tourism', 'label': 'industry'}, {'entity': 'global alert', 'label': 'diagnostic'}, {'entity': 'task force', 'label': 'organization'}, {'entity': 'Support for essential services', 'label': 'organization'}, {'entity': 'health advisors', 'label': 'organization'}, {'entity': 'liberalization of immigration arrangements', 'label': 'policy'}, {'entity': 'brain drain', 'label': 'event'}, {'entity': 'developing nations', 'label': 'country'}, {'entity': 'endorsed the code', 'label': 'clinical trial'}, {'entity': \"Australia's international health relations in 2003\", 'label': 'event'}, {'entity': 'immediate early', 'label': 'gene'}, {'entity': 'lytic replication cycle', 'label': 'pathway'}, {'entity': 'IU', 'label': 'measurement'}, {'entity': 'IC 95 isoboles', 'label': 'measurement'}, {'entity': 'vehicle', 'label': 'drug'}, {'entity': 'HCMV IE', 'label': 'gene'}, {'entity': 'synergy', 'label': 'measurement'}, {'entity': 'antagonism', 'label': 'clinical trial'}, {'entity': 'IE72/86', 'label': 'protein'}, {'entity': 'IE72/86', 'label': 'protein'}, {'entity': 'antiviral state', 'label': 'immune response'}, {'entity': 'synergistic', 'label': 'immune response'}, {'entity': 'viral attachment', 'label': 'pathway'}, {'entity': 'Yamamoto, et al.', 'label': 'organization'}, {'entity': 'IE', 'label': 'protein'}, {'entity': 'intranuclear', 'label': 'protein'}, {'entity': 'described', 'label': 'vaccine'}, {'entity': 'synergism', 'label': 'immune response'}, {'entity': 'antagonism', 'label': 'immune response'}, {'entity': 'IC 95', 'label': 'measurement'}, {'entity': 'Concave isobole', 'label': 'method'}, {'entity': 'convex isobole', 'label': 'diagnostic'}, {'entity': 'HCMV IE', 'label': 'gene'}, {'entity': 'noisy', 'label': 'measurement'}, {'entity': 'initial orientation', 'label': 'treatment'}, {'entity': 'weighted projection matching', 'label': 'method'}, {'entity': 'hierarchical', 'label': 'method'}, {'entity': \"L'Orfeo\", 'label': 'event'}, {'entity': 'microscope', 'label': 'tool'}, {'entity': 'sindbis', 'label': 'virus'}, {'entity': 'particle orientation', 'label': 'method'}, {'entity': 'Fourier Common Line', 'label': 'method'}, {'entity': '3D model', 'label': 'method'}, {'entity': 'contrast', 'label': 'measurement'}, {'entity': 'Different defocus images with their spatial frequency resolution', 'label': 'method'}, {'entity': 'LaB6 gun', 'label': 'equipment'}, {'entity': 'signal above noise', 'label': 'measurement'}, {'entity': 'contrast', 'label': 'measurement'}, {'entity': 'high-resolution reconstruction', 'label': 'measurement'}, {'entity': 'Å resolution', 'label': 'measurement'}, {'entity': 'archiving', 'label': 'method'}, {'entity': 'low contrast', 'label': 'measurement'}, {'entity': 'hierarchical weighted projection matching', 'label': 'method'}, {'entity': 'wavelett', 'label': 'method'}, {'entity': 'details', 'label': 'method'}, {'entity': 'farfrom-focus', 'label': 'measurement'}, {'entity': 'Hierarchical Wavelet Projection Matching', 'label': 'method'}, {'entity': 'close to focus', 'label': 'measurement'}, {'entity': 'asymmetric triangle', 'label': 'structure'}, {'entity': '2006', 'label': 'year'}, {'entity': 'classes', 'label': 'measurement'}, {'entity': 'wccf', 'label': 'method'}, {'entity': 'hierarchical', 'label': 'method'}, {'entity': '3D', 'label': 'measurement'}, {'entity': 'Local-refinement', 'label': 'method'}, {'entity': 'global refinement', 'label': 'method'}, {'entity': 'focal pair', 'label': 'method'}, {'entity': 'low resolution model', 'label': 'measurement'}, {'entity': '[4]', 'label': 'publication'}, {'entity': 'sampling step size', 'label': 'measurement'}, {'entity': 'lines angles', 'label': 'measurement'}, {'entity': '5 pixels', 'label': 'measurement'}, {'entity': 'far to focus', 'label': 'method'}, {'entity': 'close to focus', 'label': 'measurement'}, {'entity': 'iterative', 'label': 'method'}, {'entity': '300 HSV-1', 'label': 'virus'}, {'entity': 'wavelet-projection matching', 'label': 'method'}, {'entity': '1 standard deviation above the mean', 'label': 'measurement'}, {'entity': 'phase difference', 'label': 'measurement'}, {'entity': 'Fig.(?<=[?!])\\\\s+6', 'label': 'figure'}, {'entity': 'image', 'label': 'measurement'}, {'entity': 'asymmetric triangle', 'label': 'method'}, {'entity': 'icosahedrally symmetric model', 'label': 'model'}, {'entity': 'asymmetric triangle', 'label': 'method'}, {'entity': 'wccf', 'label': 'method'}, {'entity': 'orientation', 'label': 'method'}, {'entity': 'wccf', 'label': 'method'}, {'entity': 'Improved Common Line', 'label': 'method'}, {'entity': 'focal pair', 'label': 'method'}, {'entity': 'phase residual criterion', 'label': 'method'}, {'entity': 'polar transformation', 'label': 'method'}, {'entity': 'wavelet denoising', 'label': 'method'}, {'entity': 'intermediate resolution reconstruction', 'label': 'method'}, {'entity': 'noisy', 'label': 'measurement'}, {'entity': 'HSV data', 'label': 'measurement'}, {'entity': 'closer defocus', 'label': 'measurement'}, {'entity': 'focal pair', 'label': 'method'}, {'entity': 'structure resolution', 'label': 'measurement'}, {'entity': 'farfocus', 'label': 'measurement'}, {'entity': 'Figure 7 Surface visualization of P22 empty shell capsid', 'label': 'diagnostic'}, {'entity': 'two examples of real reconstructions', 'label': 'example'}, {'entity': 'contrast', 'label': 'measurement'}, {'entity': 'wavelet projection matching', 'label': 'method'}, {'entity': 'Gaussian filtered projection matching', 'label': 'method'}, {'entity': 'PH sindbis', 'label': 'virus'}, {'entity': 'sindbis', 'label': 'virus'}, {'entity': 'single particle reconstruction', 'label': 'method'}, {'entity': 'hierarchical', 'label': 'method'}, {'entity': 'microscope', 'label': 'tool'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'approximation', 'label': 'method'}, {'entity': 'centers of classes', 'label': 'method'}, {'entity': 'factor', 'label': 'measurement'}, {'entity': 'A 1', 'label': 'measurement'}, {'entity': 'noise effect', 'label': 'measurement'}, {'entity': 'approximation', 'label': 'method'}, {'entity': 'modelbased projection matching', 'label': 'method'}, {'entity': 'level two', 'label': 'measurement'}, {'entity': 'contrast enhancement', 'label': 'method'}, {'entity': 'level two', 'label': 'method'}, {'entity': 'angular grid', 'label': 'measurement'}, {'entity': 'parallel computer', 'label': 'organization'}, {'entity': 'classes', 'label': 'method'}, {'entity': 'asymmetric triangle', 'label': 'diagram'}, {'entity': 'classification', 'label': 'method'}, {'entity': 'wccf', 'label': 'method'}, {'entity': 'orientation', 'label': 'method'}, {'entity': 'replication cycle', 'label': 'virus'}, {'entity': 'virus adaptation', 'label': 'pathway'}, {'entity': 'population recombination rate', 'label': 'measurement'}, {'entity': 'selective neutrality', 'label': 'measurement'}, {'entity': 'evolution of viruses', 'label': 'epidemiology'}, {'entity': 'Brassica rapa', 'label': 'host'}, {'entity': 'complementation techniques', 'label': 'method'}, {'entity': 'recombinant genome', 'label': 'gene'}, {'entity': 'viral reverse transcriptase', 'label': 'protein'}, {'entity': 'aþcd/þbþþ', 'label': 'genotype'}, {'entity': 'abþd/þþcþ', 'label': 'genotype'}, {'entity': 'abcþ/þþþd', 'label': 'genotype'}, {'entity': 'þþcd/abþþ', 'label': 'genotype'}, {'entity': 'replication cycle', 'label': 'process'}, {'entity': 'replications', 'label': 'measurement'}, {'entity': 'halving', 'label': 'measurement'}, {'entity': 'phylogenetically', 'label': 'gene'}, {'entity': 'temporal', 'label': 'method'}, {'entity': 'inclusion bodies', 'label': 'virus'}, {'entity': 'interferences', 'label': 'measurement'}, {'entity': 'virus genotype', 'label': 'virus'}, {'entity': 'models', 'label': 'method'}, {'entity': 'clonally', 'label': 'method'}, {'entity': 'panmictically', 'label': 'term'}, {'entity': 'template-switching', 'label': 'method'}, {'entity': '19S', 'label': 'gene'}, {'entity': 'second-strand DNA', 'label': 'gene'}, {'entity': 'maximum likelihood', 'label': 'method'}, {'entity': 'D1', 'label': 'gene'}, {'entity': 'D2', 'label': 'gene'}, {'entity': 'D3', 'label': 'gene'}, {'entity': 'coinfected', 'label': 'clinical trial'}, {'entity': 'physiology', 'label': 'measurement'}, {'entity': 'neutral marker', 'label': 'virus'}, {'entity': '16/8', 'label': 'measurement'}, {'entity': 'harvested', 'label': 'method'}, {'entity': 'Ronkonkoma, New York', 'label': 'location'}, {'entity': 'Fitchburg, Wisconsin', 'label': 'location'}, {'entity': 'Image program', 'label': 'method'}, {'entity': 'assumption', 'label': 'method'}, {'entity': 'Wilcoxon signed-rank', 'label': 'method'}, {'entity': 'frequency estimates', 'label': 'measurement'}, {'entity': 'neutral', 'label': 'treatment'}, {'entity': 'selective effects', 'label': 'virus'}, {'entity': 'genotyped', 'label': 'method'}, {'entity': 'viral replication cycle', 'label': 'virus'}, {'entity': 'replication cycle', 'label': 'measurement'}, {'entity': 'circularization', 'label': 'process'}, {'entity': 'Mathematica', 'label': 'software'}, {'entity': 'i 12', 'label': 'parameter'}, {'entity': 'fourmarker genotypes', 'label': 'measurement'}, {'entity': 'second decimal', 'label': 'measurement'}, {'entity': 'eventual containment and treatment', 'label': 'treatment'}, {'entity': 'fears', 'label': 'symptom'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'visitation restriction', 'label': 'public health'}, {'entity': 'rights', 'label': 'person'}, {'entity': 'collateral damage', 'label': 'symptom'}, {'entity': 'loss of civil liberties', 'label': 'event'}, {'entity': 'scientific perspective', 'label': 'epidemiology'}, {'entity': 'transparency', 'label': 'clinical trial'}, {'entity': 'exceptional latitude', 'label': 'clinical trial'}, {'entity': 'exceptionality', 'label': 'clinical trial'}, {'entity': 'transparency', 'label': 'measurement'}, {'entity': 'ethical considerations', 'label': 'clinical trial'}, {'entity': 'professional code of conduct', 'label': 'vaccine'}, {'entity': 'reciprocity', 'label': 'organization'}, {'entity': 'exceptionality', 'label': 'term'}, {'entity': 'privacy', 'label': 'public health'}, {'entity': 'visitation restrictions', 'label': 'policy'}, {'entity': 'privacy strategies', 'label': 'method'}, {'entity': 'protection of communities from undue stigmatization', 'label': 'concept'}, {'entity': 'universal management strategies', 'label': 'treatment'}, {'entity': 'discrimination', 'label': 'event'}, {'entity': 'due diligence', 'label': 'method'}, {'entity': 'individual rights', 'label': 'person'}, {'entity': 'exceptional latitude', 'label': 'concept'}, {'entity': 'adjudicate', 'label': 'method'}, {'entity': 'academic intensive care unit', 'label': 'organization'}, {'entity': 'computer knowledge access system', 'label': 'clinical trial'}, {'entity': 'blinded', 'label': 'method'}, {'entity': 'medical software', 'label': 'tool'}, {'entity': 'handheld computer', 'label': 'device'}, {'entity': 'handheld computer', 'label': 'organization'}, {'entity': 'point-of-care', 'label': 'clinical trial'}, {'entity': 'University of California, Berkeley', 'label': 'organization'}, {'entity': 'medical knowledge', 'label': 'disease'}, {'entity': 'point-of-care', 'label': 'clinical trial'}, {'entity': 'handheld devices', 'label': 'device'}, {'entity': 'Palm M505', 'label': 'organization'}, {'entity': 'Skokie, IL', 'label': 'location'}, {'entity': \"What's New\", 'label': 'publication'}, {'entity': 'article abstract', 'label': 'publication'}, {'entity': 'Redwood City, CA', 'label': 'location'}, {'entity': 'clinical practice', 'label': 'treatment'}, {'entity': 'telephone', 'label': 'method'}, {'entity': 'e-mail', 'label': 'method'}, {'entity': 'Feasibility Feasibility', 'label': 'clinical trial'}, {'entity': 'handheld computer', 'label': 'organization'}, {'entity': 'handheld reference system', 'label': 'method'}, {'entity': 'physician group', 'label': 'organization'}, {'entity': 'Wappingers Falls, NY', 'label': 'location'}, {'entity': 'baseline scenario', 'label': 'event'}, {'entity': 'management decisions', 'label': 'clinical trial'}, {'entity': 'The internet-based data transfer system', 'label': 'method'}, {'entity': 'study server', 'label': 'organization'}, {'entity': 'The study time course', 'label': 'clinical trial'}, {'entity': 'screen capture', 'label': 'method'}, {'entity': 'handheld', 'label': 'tool'}, {'entity': 'scoring system', 'label': 'method'}, {'entity': 'critical care physician', 'label': 'person'}, {'entity': 'Negative points', 'label': 'diagnostic'}, {'entity': 'admission order', 'label': 'clinical trial'}, {'entity': 'α', 'label': 'measurement'}, {'entity': 'SAS System for Window', 'label': 'organization'}, {'entity': 'Themes', 'label': 'clinical trial'}, {'entity': 'medical software', 'label': 'publication'}, {'entity': 'Palm', 'label': 'organization'}, {'entity': 'Windows CE', 'label': 'tool'}, {'entity': 'handhelds', 'label': 'device'}, {'entity': 'clinical practice', 'label': 'treatment'}, {'entity': \"What's New\", 'label': 'clinical trial'}, {'entity': 'desktop computer', 'label': 'device'}, {'entity': 'final scenarios', 'label': 'clinical trial'}, {'entity': 'electronic textbooks', 'label': 'publication'}, {'entity': 'handheld device', 'label': 'device'}, {'entity': 'Google Chrome', 'label': 'organization'}, {'entity': 'scenario assignment', 'label': 'clinical trial'}, {'entity': 'electronic knowledge translation', 'label': 'method'}, {'entity': 'handheld devices', 'label': 'device'}, {'entity': 'point-ofcare', 'label': 'clinical trial'}, {'entity': 'handheld devices', 'label': 'device'}, {'entity': 'allocation of scenarios', 'label': 'method'}, {'entity': 'difficulty', 'label': 'measurement'}, {'entity': 'outbreak of SARS', 'label': 'event'}, {'entity': 'inadequate training', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'year'}, {'entity': 'wireless', 'label': 'technology'}, {'entity': 'all-in-one', 'label': 'method'}, {'entity': 'information system', 'label': 'clinical trial'}, {'entity': 'demand', 'label': 'measurement'}, {'entity': 'standardize', 'label': 'method'}, {'entity': 'electronic knowledge translation system', 'label': 'method'}, {'entity': 'tablet computer', 'label': 'organization'}, {'entity': 'point-of-care', 'label': 'clinical trial'}, {'entity': 'clinical decision-making', 'label': 'treatment'}, {'entity': 'NPfIT', 'label': 'organization'}, {'entity': 'horizon scanning', 'label': 'method'}, {'entity': 'biochemical markers', 'label': 'diagnostic'}, {'entity': 'San Diego, CA', 'label': 'location'}, {'entity': 'intensivist', 'label': 'person'}, {'entity': 'Normoglycaemia in Intensive CarE', 'label': 'clinical trial'}, {'entity': 'continuous in vivo glucose monitoring', 'label': 'diagnostic'}, {'entity': 'Northridge, CA', 'label': 'location'}, {'entity': 'Glucoday', 'label': 'organization'}, {'entity': 'automated', 'label': 'method'}, {'entity': 'Milwaukee, WI', 'label': 'location'}, {'entity': 'point-of-care', 'label': 'diagnostic'}, {'entity': 'drug resistance', 'label': 'treatment'}, {'entity': 'Acquired bloodstream infection', 'label': 'disease'}, {'entity': 'Boca Raton, FL', 'label': 'location'}, {'entity': 'Health technology assessment', 'label': 'clinical trial'}, {'entity': 'stopcock', 'label': 'device'}, {'entity': 'Fresenius', 'label': 'organization'}, {'entity': 'ISABEL', 'label': 'diagnostic'}, {'entity': 'UK Department of Health', 'label': 'organization'}, {'entity': 'inference engine', 'label': 'method'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'ISABEL', 'label': 'organization'}, {'entity': 'National Project for IT', 'label': 'organization'}, {'entity': 'electronic patient record', 'label': 'treatment'}, {'entity': 'massively interconnected', 'label': 'environment'}, {'entity': 'collapse', 'label': 'symptom'}, {'entity': 'social service', 'label': 'organization'}, {'entity': 'UK Government', 'label': 'organization'}, {'entity': 'NPfIT', 'label': 'organization'}, {'entity': 'independent research', 'label': 'clinical trial'}, {'entity': 'electronic patient record', 'label': 'treatment'}, {'entity': 'NPfIT', 'label': 'organization'}, {'entity': 'emerging technologies', 'label': 'technology'}, {'entity': 'Edr stimulatory RNA', 'label': 'protein'}, {'entity': '3 0', 'label': 'measurement'}, {'entity': 'H-(hairpin)-type pseudoknot', 'label': 'protein'}, {'entity': '2005', 'label': 'publication'}, {'entity': 'embryonal carcinoma differentiation regulated', 'label': 'gene'}, {'entity': 'musculoskeletal system', 'label': 'pathway'}, {'entity': 'development', 'label': 'pathway'}, {'entity': 'domain', 'label': 'gene'}, {'entity': 'retroviral gag', 'label': 'protein'}, {'entity': 'Edr stimulatory RNA', 'label': 'protein'}, {'entity': 'Kunkel', 'label': 'person'}, {'entity': 'pSP64T/Edr(20)', 'label': 'organization'}, {'entity': 'Sephadex G-50 chromatography', 'label': 'method'}, {'entity': 'ribosomal frameshift', 'label': 'method'}, {'entity': 'SDS-15%(w/v)', 'label': 'measurement'}, {'entity': 'differential', 'label': 'measurement'}, {'entity': 'end-labelled', 'label': 'method'}, {'entity': 'Enzymatic', 'label': 'method'}, {'entity': 'Chemical probing', 'label': 'method'}, {'entity': 'carrier rRNA', 'label': 'protein'}, {'entity': 'stop', 'label': 'gene'}, {'entity': 'wild-type efficiency', 'label': 'measurement'}, {'entity': 'pKT0/Edr/D24nt', 'label': 'virus'}, {'entity': 'uncertain', 'label': 'virus'}, {'entity': 'frameshift site', 'label': 'gene'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'non-frameshifted', 'label': 'protein'}, {'entity': 'A-site', 'label': 'protein'}, {'entity': 'GGGAAAC stretch', 'label': 'gene'}, {'entity': 'pseudowild-type', 'label': 'virus'}, {'entity': '1b', 'label': 'stem'}, {'entity': 'pseudowild', 'label': 'gene'}, {'entity': 'frameshift stimulatory RNA', 'label': 'gene'}, {'entity': 'RRL(38)', 'label': 'measurement'}, {'entity': '-ACA-bulge', 'label': 'protein'}, {'entity': 'loop 1', 'label': 'structure'}, {'entity': 'single-strand-specific enzymes', 'label': 'protein'}, {'entity': 'cleavage pattern', 'label': 'measurement'}, {'entity': 'denaturing', 'label': 'method'}, {'entity': 'A-form', 'label': 'protein'}, {'entity': 'equilibrium', 'label': 'concept'}, {'entity': 'bulge triplet', 'label': 'protein'}, {'entity': 'S1 arm', 'label': 'protein'}, {'entity': 'CL3', 'label': 'protein'}, {'entity': 'flips', 'label': 'protein'}, {'entity': 'RSV pseudoknot', 'label': 'structure'}, {'entity': 'secondary structure', 'label': 'structure'}, {'entity': 'simian T-cell lymphotrophic virus type 1', 'label': 'virus'}, {'entity': 'CL3', 'label': 'protein'}, {'entity': 'RSV pseudoknot', 'label': 'structure'}, {'entity': 'pro/pol', 'label': 'gene'}, {'entity': 'gag/pro region', 'label': 'gene'}, {'entity': 'mismatches', 'label': 'measurement'}, {'entity': 'non-functional', 'label': 'disease'}, {'entity': 'de novo synthesis', 'label': 'method'}, {'entity': 'plaque formation', 'label': 'diagnostic'}, {'entity': 'mismatch', 'label': 'method'}, {'entity': 'mismatches', 'label': 'detection'}, {'entity': 'affinity capture', 'label': 'method'}, {'entity': 'MutS error filtering', 'label': 'method'}, {'entity': 'Slide-A-Lyzer', 'label': 'diagnostic'}, {'entity': 'sonicated', 'label': 'method'}, {'entity': 'Amicon Ultra', 'label': 'method'}, {'entity': 'extinction coefficient', 'label': 'measurement'}, {'entity': 'cycles', 'label': 'method'}, {'entity': 'outside primer', 'label': 'method'}, {'entity': 'cycles', 'label': 'measurement'}, {'entity': 'Qiagen QIAquick PCR purification kit', 'label': 'method'}, {'entity': 'previous report', 'label': 'publication'}, {'entity': 'homoduplex', 'label': 'gene'}, {'entity': 'transition', 'label': 'event'}, {'entity': 'DNase I digestion', 'label': 'method'}, {'entity': 'error filtering', 'label': 'method'}, {'entity': 'error filtered', 'label': 'disease'}, {'entity': 'P(F)', 'label': 'measurement'}, {'entity': 'MutS selectivity factor', 'label': 'measurement'}, {'entity': 'assembly product', 'label': 'treatment'}, {'entity': 'MutS selectivity factor', 'label': 'gene'}, {'entity': 'fragment size', 'label': 'measurement'}, {'entity': 'multiple iterations', 'label': 'method'}, {'entity': 'mismatch specificity', 'label': 'gene'}, {'entity': 'non-ribosomal hexanucleotide', 'label': 'gene'}, {'entity': 'class(5)', 'label': 'method'}, {'entity': 'KS', 'label': 'disease'}, {'entity': 'GREG table', 'label': 'organization'}, {'entity': 'GREG table', 'label': 'table'}, {'entity': 'EMBL database', 'label': 'organization'}, {'entity': 'taxonomic table', 'label': 'publication'}, {'entity': 'taxonomic table', 'label': 'publication'}, {'entity': 'longest sequence', 'label': 'measurement'}, {'entity': 'Genomic sequence of viral species', 'label': 'virus'}, {'entity': 'confluence', 'label': 'measurement'}, {'entity': 'FBS', 'label': 'measurement'}, {'entity': 'notified', 'label': 'virus'}, {'entity': \"Eagle's minimum essential medium\", 'label': 'measurement'}, {'entity': 'notified', 'label': 'organization'}, {'entity': 'Gene Elute Purification Kit', 'label': 'organization'}, {'entity': 'linkers', 'label': 'method'}, {'entity': 'hybridized', 'label': 'method'}, {'entity': 'denaturation step', 'label': 'method'}, {'entity': 'BcaBest Labeling kit', 'label': 'organization'}, {'entity': 'satellite sequence', 'label': 'gene'}, {'entity': '28S ribosomal', 'label': 'gene'}, {'entity': \"Poisson's distribution\", 'label': 'method'}, {'entity': 'plasmid RNA', 'label': 'gene'}, {'entity': 'reverse transcribed', 'label': 'method'}, {'entity': 'mixed RNA', 'label': 'diagnosis'}, {'entity': 'ribosomal cDNA', 'label': 'gene'}, {'entity': 'random primer', 'label': 'method'}, {'entity': 'pCIneo', 'label': 'gene'}, {'entity': 'hybridized', 'label': 'diagnostic'}, {'entity': 'random primer', 'label': 'method'}, {'entity': 'SARS-CoV-specific primers', 'label': 'method'}, {'entity': 'messenger', 'label': 'gene'}, {'entity': 'competitive PCR', 'label': 'method'}, {'entity': 'mixed primer', 'label': 'gene'}, {'entity': 'transcription control', 'label': 'immune response'}, {'entity': 'driver amplifier', 'label': 'vaccine'}, {'entity': 'nonribosomal', 'label': 'gene'}, {'entity': 'total RNA', 'label': 'measurement'}, {'entity': 'double-membraned', 'label': 'protein'}, {'entity': 'RNA interference', 'label': 'method'}, {'entity': 'RNA replication', 'label': 'protein'}, {'entity': 'normal development', 'label': 'process'}, {'entity': 'cellular compartments', 'label': 'protein'}, {'entity': 'electrondense', 'label': 'measurement'}, {'entity': 'double membrane', 'label': 'structure'}, {'entity': '3A', 'label': 'protein'}, {'entity': '3A', 'label': 'protein'}, {'entity': '2C', 'label': 'protein'}, {'entity': 'autophagosomal', 'label': 'pathway'}, {'entity': 'bacterial clearance', 'label': 'disease'}, {'entity': 'membrane', 'label': 'protein'}, {'entity': 'luminal', 'label': 'protein'}, {'entity': 'RNA replication protein 3A', 'label': 'protein'}, {'entity': 'poliovirus 3A', 'label': 'virus'}, {'entity': 'serotypes', 'label': 'virus'}, {'entity': '3A', 'label': 'protein'}, {'entity': 'poliovirus 3A', 'label': 'protein'}, {'entity': 'autophagosomal', 'label': 'disease'}, {'entity': 'singlemembrane', 'label': 'vocabulary'}, {'entity': 'replication complex', 'label': 'protein'}, {'entity': 'multilamellar', 'label': 'virus'}, {'entity': 'poliovirus infection', 'label': 'disease'}, {'entity': 'poliovirus infection', 'label': 'disease'}, {'entity': 'host defenses', 'label': 'immune response'}, {'entity': 'lipase', 'label': 'protein'}, {'entity': 'luminal', 'label': 'protein'}, {'entity': 'Dot', 'label': 'gene'}, {'entity': 'degradative', 'label': 'vocabulary'}, {'entity': 'mitochondrial membrane', 'label': 'protein'}, {'entity': 'lytic', 'label': 'virus'}, {'entity': 'polarized', 'label': 'treatment'}, {'entity': 'cytosolic lumen', 'label': 'pathway'}, {'entity': 'double-membrane structure', 'label': 'virus'}, {'entity': 'secondary antibody', 'label': 'protein'}, {'entity': 'cytosolic', 'label': 'protein'}, {'entity': 'Palo Alto, California', 'label': 'location'}, {'entity': 'HeLa H1', 'label': 'cell'}, {'entity': 'Manassas, Virginia', 'label': 'location'}, {'entity': 'Carlsbad, California', 'label': 'city'}, {'entity': '3A', 'label': 'protein'}, {'entity': 'Vectashield', 'label': 'vaccine'}, {'entity': 'Burlingame, California', 'label': 'location'}, {'entity': 'Eugene, Oregon', 'label': 'location'}, {'entity': 'SoftWorx', 'label': 'software'}, {'entity': '528nm emission', 'label': 'measurement'}, {'entity': 'Adobe Photoshop', 'label': 'software'}, {'entity': 'Lafayette, Colorado', 'label': 'location'}, {'entity': 'Carlsbad, California', 'label': 'location'}, {'entity': 'San Jose, California', 'label': 'location'}, {'entity': 'Zymed', 'label': 'organization'}, {'entity': 'South San Francisco, California', 'label': 'location'}, {'entity': 'Research Diagnostics', 'label': 'organization'}, {'entity': 'Flanders, New Jersey', 'label': 'location'}, {'entity': 'High-pressure freezing', 'label': 'method'}, {'entity': 'Balzers HPM 10', 'label': 'organization'}, {'entity': 'CM10 EM', 'label': 'equipment'}, {'entity': 'SARS epidemic in Taiwan', 'label': 'event'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'regional hospital', 'label': 'organization'}, {'entity': 'peak SARS period', 'label': 'event'}, {'entity': 'regional hospital', 'label': 'location'}, {'entity': 'first probable case of severe acute respiratory syndrome (SARS) in Taiwan', 'label': 'disease'}, {'entity': 'April', 'label': 'month'}, {'entity': 'regional hospitals', 'label': 'organization'}, {'entity': 'Delivery of a child', 'label': 'event'}, {'entity': 'mode of child delivery', 'label': 'treatment'}, {'entity': 'neo-natal', 'label': 'disease'}, {'entity': 'Health consequences', 'label': 'disease'}, {'entity': 'public health professionals', 'label': 'organization'}, {'entity': 'neo-natal', 'label': 'clinical trial'}, {'entity': 'cesarean', 'label': 'procedure'}, {'entity': 'DRG (diagnosis-related groups) payment system', 'label': 'method'}, {'entity': 'itemized', 'label': 'measurement'}, {'entity': 'quality of care', 'label': 'treatment'}, {'entity': 'regional hospital', 'label': 'organization'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'in-patient', 'label': 'clinical trial'}, {'entity': 'provider identification number', 'label': 'measurement'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'significance', 'label': 'measurement'}, {'entity': 'Accreditation level of provider', 'label': 'measurement'}, {'entity': 'regional hospital', 'label': 'organization'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'regional hospitals', 'label': 'organization'}, {'entity': 'pre-SARS period', 'label': 'period'}, {'entity': 'peak SARS period', 'label': 'event'}, {'entity': 'regional hospital', 'label': 'organization'}, {'entity': 'pre', 'label': 'period'}, {'entity': 'peak SARS period', 'label': 'event'}, {'entity': 'lower level hospitals/clinics', 'label': 'location'}, {'entity': 'post-SARS period', 'label': 'period'}, {'entity': 'pre-SARS period', 'label': 'period'}, {'entity': 'regional hospital', 'label': 'location'}, {'entity': 'LOS', 'label': 'measurement'}, {'entity': 'in-patient', 'label': 'clinical trial'}, {'entity': 'continuity of that care', 'label': 'clinical trial'}, {'entity': 'limitations', 'label': 'clinical trial'}, {'entity': 'perinatal', 'label': 'disease'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'childbirth', 'label': 'event'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'regional hospitals', 'label': 'organization'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'direct causalities', 'label': 'symptom'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'search for preventive strategies', 'label': 'treatment'}, {'entity': 'last case', 'label': 'event'}, {'entity': 'outbreak of SARS', 'label': 'disease'}, {'entity': 'small', 'label': 'pathway'}, {'entity': 'fecaloral', 'label': 'transmission'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'SARS epidemiology', 'label': 'epidemiology'}, {'entity': 'etiology of the virus', 'label': 'disease'}, {'entity': 'a cure', 'label': 'treatment'}, {'entity': 'human coronavirus 229E', 'label': 'virus'}, {'entity': 'Bovine, equine, and rat coronavirus', 'label': 'virus'}, {'entity': 'pheasant coronavirus', 'label': 'virus'}, {'entity': 'SARS-CoV polymerase', 'label': 'gene'}, {'entity': 'tree analysis', 'label': 'method'}, {'entity': 'SARS virus pathogenesis in vivo', 'label': 'pathway'}, {'entity': 'viral replicative strategy', 'label': 'virus'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'SARS-CoV epidemic', 'label': 'disease'}, {'entity': 'NTPase/helicase', 'label': 'protein'}, {'entity': \"3' untranslated region\", 'label': 'region'}, {'entity': '530 loop', 'label': 'protein'}, {'entity': 'negative charge', 'label': 'protein'}, {'entity': 'The balance of cell survival and cell death in response to SARS-CoV infection', 'label': 'clinical trial'}, {'entity': 'SARS CoV vaccine', 'label': 'vaccine'}, {'entity': 'boosting', 'label': 'vaccine'}, {'entity': 'human vaccine', 'label': 'vaccine'}, {'entity': 'SARS vaccine', 'label': 'vaccine'}, {'entity': 'attenuated', 'label': 'virus'}, {'entity': 'SARS vaccine', 'label': 'vaccine'}, {'entity': 'virus receptor', 'label': 'protein'}, {'entity': 'cell biology', 'label': 'method'}, {'entity': 'basic research', 'label': 'clinical trial'}, {'entity': 'human outbreaks in early 2003', 'label': 'event'}, {'entity': 'dichotomous key', 'label': 'method'}, {'entity': 'character', 'label': 'method'}, {'entity': 'binary', 'label': 'method'}, {'entity': 'moth', 'label': 'organism'}, {'entity': 'sub-sequence', 'label': 'gene'}, {'entity': 'taxonomy', 'label': 'method'}, {'entity': 'questions', 'label': 'method'}, {'entity': 'groups', 'label': 'method'}, {'entity': 'binary', 'label': 'method'}, {'entity': 'binary', 'label': 'method'}, {'entity': 'ideal characters', 'label': 'method'}, {'entity': 'target-specific characters', 'label': 'method'}, {'entity': 'target-specific', 'label': 'method'}, {'entity': 'least efficient strategy', 'label': 'method'}, {'entity': 'shared', 'label': 'method'}, {'entity': 'binary', 'label': 'method'}, {'entity': 'subsequences', 'label': 'gene'}, {'entity': 'complementary sequences', 'label': 'method'}, {'entity': 'Sub-sequence', 'label': 'virus'}, {'entity': 'diversity', 'label': 'measurement'}, {'entity': 'gaps', 'label': 'measurement'}, {'entity': 'sub-sequence', 'label': 'gene'}, {'entity': 'questions', 'label': 'clinical trial'}, {'entity': 'singletons', 'label': 'vocabulary'}, {'entity': 'sub-sequence', 'label': 'gene'}, {'entity': 'Distinguishing sub-sequences', 'label': 'virus'}, {'entity': 'singletons', 'label': 'measurement'}, {'entity': 'sub-sequence', 'label': 'measurement'}, {'entity': '2A', 'label': 'measurement'}, {'entity': '2B', 'label': 'measurement'}, {'entity': 'taxonomic key', 'label': 'method'}, {'entity': 'random data', 'label': 'measurement'}, {'entity': 'real sequences', 'label': 'disease'}, {'entity': '4A', 'label': 'figure'}, {'entity': 'groups', 'label': 'method'}, {'entity': 'variant sequence', 'label': 'gene'}, {'entity': 'AGAAGTATTTAAATTACG', 'label': 'sequence'}, {'entity': 'sub-sequence', 'label': 'gene'}, {'entity': 'gene sub-sequences', 'label': 'gene'}, {'entity': 'region-specific primer', 'label': 'method'}, {'entity': 'subtype', 'label': 'virus'}, {'entity': 'subtype', 'label': 'region'}, {'entity': 'redundant', 'label': 'measurement'}, {'entity': 'redundant', 'label': 'measurement'}, {'entity': 'groups', 'label': 'vocabulary'}, {'entity': 'variant sequences', 'label': 'measurement'}, {'entity': 'biochemically', 'label': 'method'}, {'entity': 'variant sequence', 'label': 'gene'}, {'entity': 'multiplexing', 'label': 'method'}, {'entity': 'Lepidoptera', 'label': 'organism'}, {'entity': 'Fortran 95', 'label': 'method'}, {'entity': 'degapped', 'label': 'method'}, {'entity': 'replicates', 'label': 'virus'}, {'entity': 'array', 'label': 'method'}, {'entity': 'distinguishing array', 'label': 'method'}, {'entity': 'DSS selection', 'label': 'method'}, {'entity': 'complete set', 'label': 'definition'}, {'entity': 'genetic predisposition', 'label': 'gene'}, {'entity': 'ACE insertion/deletion', 'label': 'gene'}, {'entity': 'multivariate', 'label': 'method'}, {'entity': 'deaths worldwide', 'label': 'measurement'}, {'entity': 'survivors', 'label': 'person'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'reverse', 'label': 'gene'}, {'entity': 'AmpliTaq Gold DNA polymerase', 'label': 'protein'}, {'entity': 'chisquare test', 'label': 'method'}, {'entity': 'multivariate', 'label': 'method'}, {'entity': 'control subjects', 'label': 'disease'}, {'entity': 'smoking habits', 'label': 'disease'}, {'entity': 'severe', 'label': 'disease'}, {'entity': 'table 3a', 'label': 'publication'}, {'entity': 'table 3b', 'label': 'publication'}, {'entity': 'significant factor', 'label': 'measurement'}, {'entity': 'male', 'label': 'host'}, {'entity': 'association studies', 'label': 'clinical trial'}, {'entity': 'ACE polymorphism', 'label': 'gene'}, {'entity': 'DD', 'label': 'gene'}, {'entity': 'index patient', 'label': 'person'}, {'entity': 'Endpoint', 'label': 'clinical trial'}, {'entity': 'DD', 'label': 'genotype'}, {'entity': 'SARS-coronavirus infection in Chinese', 'label': 'disease'}, {'entity': 'A-form', 'label': 'structure'}, {'entity': 'stems 2', 'label': 'pathway'}, {'entity': 'rapid response', 'label': 'event'}, {'entity': 'Pknots', 'label': 'software'}, {'entity': 'Pknot', 'label': 'method'}, {'entity': 'stems 1', 'label': 'pathway'}, {'entity': 'kissing loop', 'label': 'protein'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'avian infectious bronchitis virus', 'label': 'virus'}, {'entity': 'general classes', 'label': 'concept'}, {'entity': 'bases', 'label': 'measurement'}, {'entity': 'loop 2 region', 'label': 'structure'}, {'entity': 'A13446', 'label': 'gene'}, {'entity': '59', 'label': 'measurement'}, {'entity': 'two-dimensional', 'label': 'measurement'}, {'entity': 'SARS pseudoknot', 'label': 'protein'}, {'entity': 'imino nitrogen', 'label': 'protein'}, {'entity': 'mRNA pseudoknot', 'label': 'structure'}, {'entity': 'large errors', 'label': 'measurement'}, {'entity': 'yeast system', 'label': 'organism'}, {'entity': 'basepairing', 'label': 'method'}, {'entity': 'oligonucleotide site-directed mutagenesis', 'label': 'method'}, {'entity': 'À1 PRF', 'label': 'protein'}, {'entity': 'bulged adenosine', 'label': 'protein'}, {'entity': 'basic stem 2 structure', 'label': 'protein'}, {'entity': 'U13424', 'label': 'gene'}, {'entity': 'À1 PRF', 'label': 'gene'}, {'entity': 'À1 PRF', 'label': 'protein'}, {'entity': 'fold wild-type', 'label': 'measurement'}, {'entity': 'bulged residue', 'label': 'protein'}, {'entity': 'Frameshifting with S3A', 'label': 'protein'}, {'entity': 'S2 series', 'label': 'measurement'}, {'entity': 'potential significance', 'label': 'clinical trial'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'group 1 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'À1 PRF', 'label': 'protein'}, {'entity': 'À1 PRF', 'label': 'protein'}, {'entity': 'translation competent', 'label': 'disease'}, {'entity': 'replication competent', 'label': 'virus'}, {'entity': 'kissing-loop', 'label': 'protein'}, {'entity': 'MFE', 'label': 'measurement'}, {'entity': 'Pknots', 'label': 'method'}, {'entity': 'À1 PRF', 'label': 'gene'}, {'entity': 'HÀ', 'label': 'publication'}, {'entity': 'Walkersville, Maine', 'label': 'location'}, {'entity': 'Fitchburg, Wisconsin', 'label': 'location'}, {'entity': 'Austin, Texas', 'label': 'location'}, {'entity': 'normally distributed', 'label': 'measurement'}, {'entity': 'Coralville, Iowa', 'label': 'location'}, {'entity': 'La Jolla, California', 'label': 'location'}, {'entity': 'complementary mutation', 'label': 'mutation'}, {'entity': 'CAGGGC', 'label': 'gene'}, {'entity': 'end labeled', 'label': 'method'}, {'entity': 'Sunnyvale, California', 'label': 'location'}, {'entity': 'Manassas, Virginia', 'label': 'city'}, {'entity': 'Piscataway, New Jersey', 'label': 'location'}, {'entity': 'Billerica, Massachusetts', 'label': 'location'}, {'entity': 'Hercules, California', 'label': 'location'}, {'entity': 'Bruker Avance', 'label': 'organization'}, {'entity': 'One-dimensional', 'label': 'method'}, {'entity': '1 H,15 N-heteronuclear multiple quantum correlation', 'label': 'method'}, {'entity': 'XLS', 'label': 'publication'}, {'entity': 'Microbial Rosetta Stone', 'label': 'organization'}, {'entity': 'classifications', 'label': 'disease'}, {'entity': 'emerging infectious organisms', 'label': 'disease'}, {'entity': 'Microbial Rosetta Stone', 'label': 'publication'}, {'entity': 'taxonomic', 'label': 'measurement'}, {'entity': 'nomenclature', 'label': 'vocabulary'}, {'entity': 'common name', 'label': 'term'}, {'entity': 'alternative name', 'label': 'vocabulary'}, {'entity': 'emerging infectious organisms', 'label': 'disease'}, {'entity': 'United States Department of Agriculture', 'label': 'organization'}, {'entity': 'nomenclature', 'label': 'vocabulary'}, {'entity': 'synonyms', 'label': 'virus'}, {'entity': 'agent', 'label': 'disease causing agents'}, {'entity': 'nomenclature', 'label': 'vocabulary'}, {'entity': 'Microbial Rosetta Stone', 'label': 'organization'}, {'entity': 'phylogenetic hierarchy', 'label': 'method'}, {'entity': 'taxonomic', 'label': 'gene'}, {'entity': 'Microbial Rosetta Stone', 'label': 'organization'}, {'entity': 'query', 'label': 'method'}, {'entity': 'bioterrorist', 'label': 'virus'}, {'entity': 'developing', 'label': 'country'}, {'entity': 'Mycoplasma pneumonia', 'label': 'disease'}, {'entity': 'taxonomic', 'label': 'vocabulary'}, {'entity': 'developing', 'label': 'country'}, {'entity': 'notifiable disease list', 'label': 'diagnostic'}, {'entity': 'etiological agent', 'label': 'disease'}, {'entity': 'Figure 1 Symbol key', 'label': 'diagnostic'}, {'entity': 'Hugenholtz', 'label': 'person'}, {'entity': 'branches', 'label': 'method'}, {'entity': 'taxonomy', 'label': 'measurement'}, {'entity': 'niches', 'label': 'environment'}, {'entity': 'cultivatable organisms', 'label': 'organism'}, {'entity': 'chronic sequelae', 'label': 'disease'}, {'entity': 'food-borne illness in the United States', 'label': 'disease'}, {'entity': 'Mishandling of food', 'label': 'disease'}, {'entity': 'sub-therapeutic', 'label': 'measurement'}, {'entity': 'commensal', 'label': 'disease'}, {'entity': 'intermediate vector', 'label': 'transmission'}, {'entity': 'outbreak of monkeypox', 'label': 'event'}, {'entity': 'zoonotic', 'label': 'disease'}, {'entity': 'AIDS epidemic', 'label': 'disease'}, {'entity': 'cereal', 'label': 'virus'}, {'entity': 'infant', 'label': 'person'}, {'entity': 'older children', 'label': 'disease'}, {'entity': 'ideological', 'label': 'virus'}, {'entity': 'Additional File 5', 'label': 'publication'}, {'entity': 'biological source', 'label': 'virus'}, {'entity': 'source organism', 'label': 'organism'}, {'entity': 'food contamination', 'label': 'disease'}, {'entity': 'Ricin', 'label': 'drug'}, {'entity': 'genetically altered microbes', 'label': 'virus'}, {'entity': 'bioterrorist engineering', 'label': 'method'}, {'entity': 'vaccine research', 'label': 'vaccine'}, {'entity': '2002', 'label': 'year'}, {'entity': 'reverse genetics', 'label': 'method'}, {'entity': 'reverse genetics', 'label': 'method'}, {'entity': 'virulence factor', 'label': 'gene'}, {'entity': 'antibiotic-resistant', 'label': 'virus'}, {'entity': 'Microbial Rosetta Stone Database', 'label': 'organization'}, {'entity': 'species name', 'label': 'virus'}, {'entity': 'species synonyms', 'label': 'vocabulary'}, {'entity': 'Shigella dysenteriae', 'label': 'disease'}, {'entity': 'disease name', 'label': 'disease'}, {'entity': 'Hugenholtz et al.', 'label': 'organization'}, {'entity': 'gene sequence', 'label': 'gene'}, {'entity': 'eukaryotic tree', 'label': 'diagram'}, {'entity': 'RNA virus phylogenetic trees', 'label': 'diagnostic'}, {'entity': 'polyphyly', 'label': 'virus'}, {'entity': 'developing nations', 'label': 'region'}, {'entity': 'leading causes of death worldwide', 'label': 'disease'}, {'entity': 'CDC-notifiable-disease list', 'label': 'public health'}, {'entity': 'notifiable disease list', 'label': 'list'}, {'entity': 'etiological agent', 'label': 'disease'}, {'entity': 'unspecific', 'label': 'disease'}, {'entity': 'Additional File 4', 'label': 'publication'}, {'entity': 'emerging', 'label': 'virus'}, {'entity': 'Emerging disease table', 'label': 'diagnostic'}, {'entity': 'Potential bioweapons', 'label': 'virus'}, {'entity': 'modified', 'label': 'treatment'}, {'entity': 'Bioengineering for peaceful purposes', 'label': 'method'}, {'entity': 'critical human pathogens', 'label': 'virus'}, {'entity': 'writing', 'label': 'method'}, {'entity': 'TAH', 'label': 'person'}, {'entity': 'taxonomy', 'label': 'virus'}, {'entity': '3CL(Pro)', 'label': 'protein'}, {'entity': 'bio-safety level 2', 'label': 'epidemiology'}, {'entity': 'PCR cycle', 'label': 'method'}, {'entity': '15mer', 'label': 'protein'}, {'entity': 'thermal flask', 'label': 'object'}, {'entity': 'evaporated', 'label': 'method'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'Agilent Zorbax Extend', 'label': 'method'}, {'entity': 'SARS 3CL Pro', 'label': 'protein'}, {'entity': 'reverse phase', 'label': 'method'}, {'entity': 'EC', 'label': 'protein'}, {'entity': 'ECg', 'label': 'protein'}, {'entity': 'TF2', 'label': 'protein'}, {'entity': 'fluorescence plate reader', 'label': 'method'}, {'entity': 'Human coronavirus', 'label': 'virus'}, {'entity': 'plasminogen activator', 'label': 'protein'}, {'entity': 'enteric', 'label': 'disease'}, {'entity': 'zinc fi ngers', 'label': 'protein'}, {'entity': 'H type', 'label': 'protein'}, {'entity': 'loop', 'label': 'protein'}, {'entity': 'loops', 'label': 'gene'}, {'entity': 'diverse roles', 'label': 'role'}, {'entity': 'double-pseudoknot conformation', 'label': 'protein'}, {'entity': 'HDV ribozyme', 'label': 'protein'}, {'entity': 'fi ve', 'label': 'measurement'}, {'entity': 'nested double-pseudoknot conformation', 'label': 'protein'}, {'entity': 'self-cleaving', 'label': 'protein'}, {'entity': 'pseudoknot belt', 'label': 'protein'}, {'entity': 'hydrolysis', 'label': 'enzyme reaction'}, {'entity': 'pseudoknot architecture', 'label': 'pathway'}, {'entity': '5′ end', 'label': 'gene'}, {'entity': 'pseudoknot fold', 'label': 'protein'}, {'entity': 'fl anks', 'label': 'protein'}, {'entity': '-1 frameshifting', 'label': 'process'}, {'entity': 'frameshift-inducing pseudoknot', 'label': 'protein'}, {'entity': 'P1', 'label': 'gene'}, {'entity': 'P2', 'label': 'gene'}, {'entity': 'SARS-CoV pseudoknot', 'label': 'protein'}, {'entity': 'RNA architecture', 'label': 'structure'}, {'entity': 'molecular functions', 'label': 'gene'}, {'entity': 'dynamic', 'label': 'clinical trial'}, {'entity': 'dynamic', 'label': 'pathway'}, {'entity': 'dynamic', 'label': 'treatment'}, {'entity': 'attributes', 'label': 'method'}, {'entity': 'formal', 'label': 'vocabulary'}, {'entity': 'Ontologies', 'label': 'vocabulary'}, {'entity': 'annotation', 'label': 'method'}, {'entity': 'software model', 'label': 'method'}, {'entity': 'ontologies', 'label': 'vocabulary'}, {'entity': 'directed acyclic graph', 'label': 'term'}, {'entity': 'GO definition', 'label': 'term'}, {'entity': 'Gene Ontology Annotation', 'label': 'organization'}, {'entity': 'mine', 'label': 'method'}, {'entity': 'GO domain', 'label': 'gene'}, {'entity': 'inheritance', 'label': 'method'}, {'entity': 'high-level concepts', 'label': 'concept'}, {'entity': 'dynamics', 'label': 'measurement'}, {'entity': 'dynamic', 'label': 'method'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'biological entities', 'label': 'gene'}, {'entity': 'molecular function domain', 'label': 'term'}, {'entity': 'unified modeling language', 'label': 'method'}, {'entity': 'unified modeling language', 'label': 'method'}, {'entity': 'unified modeling language', 'label': 'method'}, {'entity': 'tumor development', 'label': 'disease'}, {'entity': 'Smad Anchor for Receptor Activation', 'label': 'protein'}, {'entity': 'Smad interaction', 'label': 'protein'}, {'entity': 'dynamic', 'label': 'method'}, {'entity': 'ontological terms', 'label': 'term'}, {'entity': 'class diagram', 'label': 'method'}, {'entity': 'more specific', 'label': 'term'}, {'entity': 'less specific', 'label': 'term'}, {'entity': 'A specific example', 'label': 'example'}, {'entity': 'class', 'label': 'method'}, {'entity': 'Inheritance Relationships', 'label': 'method'}, {'entity': 'inheritance', 'label': 'method'}, {'entity': 'inheritance', 'label': 'method'}, {'entity': 'object functions', 'label': 'method'}, {'entity': 'GO molecular function', 'label': 'term'}, {'entity': 'molecular function domain', 'label': 'term'}, {'entity': 'molecular function domain', 'label': 'ontology'}, {'entity': 'GO molecular function', 'label': 'ontology'}, {'entity': 'reverse engineering', 'label': 'method'}, {'entity': 'Figure 1 The GO terms associated with the process, TGF-beta receptor complex assembly', 'label': 'diagram'}, {'entity': 'Figure 1A', 'label': 'figure'}, {'entity': 'GO notation', 'label': 'method'}, {'entity': 'complex assembly', 'label': 'pathway'}, {'entity': 'sequence diagram', 'label': 'method'}, {'entity': 'molecular function domain', 'label': 'ontology'}, {'entity': 'interact', 'label': 'method'}, {'entity': 'function', 'label': 'method'}, {'entity': 'autoactivation', 'label': 'method'}, {'entity': 'TGF-beta RII complex', 'label': 'protein'}, {'entity': 'term', 'label': 'method'}, {'entity': 'self-call', 'label': 'method'}, {'entity': 'protein homodimerization', 'label': 'protein'}, {'entity': 'translate()', 'label': 'method'}, {'entity': 'Biological process', 'label': 'pathway'}, {'entity': 'events', 'label': 'event'}, {'entity': 'alternative scenarios', 'label': 'event'}, {'entity': 'lipid raft-dependent', 'label': 'pathway'}, {'entity': 'signal promotion', 'label': 'pathway'}, {'entity': 'final state', 'label': 'state'}, {'entity': 'complex assembly', 'label': 'pathway'}, {'entity': 'functions', 'label': 'gene'}, {'entity': 'functions', 'label': 'protein'}, {'entity': 'attributes', 'label': 'measurement'}, {'entity': 'inheritance', 'label': 'method'}, {'entity': 'interacting', 'label': 'method'}, {'entity': 'product description', 'label': 'gene'}, {'entity': 'GO vocabulary', 'label': 'vocabulary'}, {'entity': 'objectoriented', 'label': 'method'}, {'entity': 'process', 'label': 'pathway'}, {'entity': 'central dogma', 'label': 'concept'}, {'entity': 'inheritance', 'label': 'concept'}, {'entity': 'inheritance', 'label': 'method'}, {'entity': 'layered architecture', 'label': 'method'}, {'entity': 'modular structure', 'label': 'concept'}, {'entity': 'functions', 'label': 'gene'}, {'entity': 'outcome', 'label': 'clinical trial'}, {'entity': 'complex assembly', 'label': 'pathway'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'function term', 'label': 'term'}, {'entity': 'polymorphic behavior', 'label': 'protein'}, {'entity': 'heterodimerize', 'label': 'protein'}, {'entity': 'dimerization function', 'label': 'protein'}, {'entity': 'protein AB', 'label': 'protein'}, {'entity': 'dynamics', 'label': 'method'}, {'entity': 'biological process', 'label': 'process'}, {'entity': 'biological process', 'label': 'process'}, {'entity': 'dynamics of a process', 'label': 'pathway'}, {'entity': 'disconnected domains', 'label': 'concept'}, {'entity': 'systems', 'label': 'pathway'}, {'entity': 'system integration', 'label': 'clinical trial'}, {'entity': 'capturing biological systems', 'label': 'method'}, {'entity': 'gene ontologies', 'label': 'gene'}, {'entity': 'Java', 'label': 'method'}, {'entity': 'automation', 'label': 'method'}, {'entity': 'biological system integration', 'label': 'process'}, {'entity': 'modules', 'label': 'organization'}, {'entity': 'Visio Pro', 'label': 'software'}, {'entity': 'class-responsibility collaboration', 'label': 'method'}, {'entity': 'complex assembly', 'label': 'process'}, {'entity': 'boundary', 'label': 'measurement'}, {'entity': 'complex assembly', 'label': 'pathway'}, {'entity': 'components', 'label': 'organization'}, {'entity': 'state diagram', 'label': 'method'}, {'entity': 'concurrent state', 'label': 'state'}, {'entity': 'essential attributes', 'label': 'measurement'}, {'entity': 'SPR Opt', 'label': 'method'}, {'entity': 'co-segregation', 'label': 'method'}, {'entity': 'unsequenced', 'label': 'virus'}, {'entity': 'free', 'label': 'vocabulary_word'}, {'entity': '2001 anthrax attacks', 'label': 'event'}, {'entity': 'HIV fragments', 'label': 'virus'}, {'entity': 'genome sequencing', 'label': 'method'}, {'entity': 'hotspots of variation', 'label': 'virus'}, {'entity': 'Full sequence information', 'label': 'measurement'}, {'entity': 'different', 'label': 'treatment'}, {'entity': 'knowledge bases', 'label': 'diagnostic'}, {'entity': 'bench work', 'label': 'method'}, {'entity': 'human crime forensics', 'label': 'forensic'}, {'entity': 'SPR Opt', 'label': 'software'}, {'entity': 'SPR Opt', 'label': 'software'}, {'entity': 'length polymorphism', 'label': 'measurement'}, {'entity': 'non-tandem repeat', 'label': 'gene'}, {'entity': 'selection', 'label': 'method'}, {'entity': 'forensic markers', 'label': 'method'}, {'entity': 'unsequenced', 'label': 'virus'}, {'entity': 'source code', 'label': 'method'}, {'entity': 'output files', 'label': 'publication'}, {'entity': 'variable sites', 'label': 'measurement'}, {'entity': 'restriction digest', 'label': 'method'}, {'entity': 'Amplicon X Unique Fragment Length Distributions', 'label': 'measurement'}, {'entity': 'clusters', 'label': 'gene'}, {'entity': 'SNP forensics', 'label': 'method'}, {'entity': 'SPR Opt', 'label': 'method'}, {'entity': 'unresolved cluster', 'label': 'virus'}, {'entity': 'heterogeneous viruses', 'label': 'virus'}, {'entity': 'inter-specific conservation', 'label': 'gene'}, {'entity': 'SPR Opt', 'label': 'software'}, {'entity': 'Figure 2 Phylogram of SARS', 'label': 'diagram'}, {'entity': 'Microsoft Windows', 'label': 'organization'}, {'entity': 'dots', 'label': 'measurement'}, {'entity': 'Base', 'label': 'gene'}, {'entity': 'conservation gestalt', 'label': 'method'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'downstream', 'label': 'gene'}, {'entity': 'conserved sequence', 'label': 'sequence'}, {'entity': 'input sequence', 'label': 'measurement'}, {'entity': 'insertion/deletion', 'label': 'gene'}, {'entity': 'length requirement', 'label': 'measurement'}, {'entity': 'conservation gestalt', 'label': 'term'}, {'entity': 'jump base', 'label': 'measurement'}, {'entity': 'sites', 'label': 'measurement'}, {'entity': 'primer3', 'label': 'software'}, {'entity': 'source code', 'label': 'publication'}, {'entity': 'regular expression pattern matching', 'label': 'method'}, {'entity': 'restriction digest', 'label': 'method'}, {'entity': 'alternative enzymes', 'label': 'method'}, {'entity': 'buffers', 'label': 'measurement'}, {'entity': 'non-palindromic', 'label': 'measurement'}, {'entity': 'variable', 'label': 'measurement'}, {'entity': 'character haplotypes', 'label': 'pathway'}, {'entity': 'character', 'label': 'gene'}, {'entity': 'character', 'label': 'gene'}, {'entity': 'locus', 'label': 'gene'}, {'entity': 'sequence cluster', 'label': 'diagnostic'}, {'entity': 'alternative solution set', 'label': 'method'}, {'entity': 'genome group', 'label': 'organization'}, {'entity': 'combinatoric', 'label': 'method'}, {'entity': 'demands', 'label': 'measurement'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'bit vector', 'label': 'method'}, {'entity': 'haplotypess', 'label': 'gene'}, {'entity': 'output', 'label': 'method'}, {'entity': 'solution set i', 'label': 'variable'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'index i', 'label': 'variable'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'list others(A)', 'label': 'method'}, {'entity': 'cannot be uniquely discriminated', 'label': 'clinical trial'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'others(i,A)', 'label': 'gene'}, {'entity': 'others(j,A)', 'label': 'gene'}, {'entity': 'genome groups', 'label': 'measurement'}, {'entity': 'all_discriminating_sets', 'label': 'publication'}, {'entity': 'prepared group', 'label': 'virus'}, {'entity': 'genome groups with many genomes', 'label': 'virus'}, {'entity': 'genome groups with the fewest genomes', 'label': 'gene'}, {'entity': 'pare_groups', 'label': 'variable'}, {'entity': 'genome group', 'label': 'vocabulary'}, {'entity': 'unresolved cluster', 'label': 'cluster'}, {'entity': '3', 'label': 'measurement'}, {'entity': '4', 'label': 'measurement'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'character', 'label': 'method'}, {'entity': 'global optimum', 'label': 'measurement'}, {'entity': 'bases', 'label': 'measurement'}, {'entity': 'SPR Opt', 'label': 'method'}, {'entity': 'PHYLIP', 'label': 'software'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'minimum', 'label': 'measurement'}, {'entity': 'PHYLIP', 'label': 'software'}, {'entity': 'PHYLIP', 'label': 'organization'}, {'entity': 'groups', 'label': 'measurement'}, {'entity': 'FLPs_all', 'label': 'publication'}, {'entity': 'anti-PMN', 'label': 'drug'}, {'entity': 'mucus secretion', 'label': 'symptom'}, {'entity': 'host immune response', 'label': 'immune response'}, {'entity': 'IL-8/CXCL8', 'label': 'protein'}, {'entity': '24', 'label': 'publication'}, {'entity': 'butterfly needle', 'label': 'method'}, {'entity': 'lavaged', 'label': 'method'}, {'entity': 'Santa Cruz Biotechnology Inc', 'label': 'organization'}, {'entity': 'secondary biotinylated', 'label': 'method'}, {'entity': 'murine', 'label': 'host'}, {'entity': 'threshold cycle number', 'label': 'measurement'}, {'entity': 'Evans blue dye extravasation technique', 'label': 'method'}, {'entity': 'bronchi', 'label': 'anatomy'}, {'entity': 'dual-wavelength', 'label': 'method'}, {'entity': 'corrected absorbance at 620 nm', 'label': 'measurement'}, {'entity': 'hilum', 'label': 'location'}, {'entity': 'vacuum oven', 'label': 'equipment'}, {'entity': 'detection antibody', 'label': 'protein'}, {'entity': 'Whole lung single cell suspension', 'label': 'method'}, {'entity': 'Becton Dickinson Immunocytometry Systems', 'label': 'organization'}, {'entity': 'surface airway epithelium', 'label': 'treatment'}, {'entity': 'intraparenchymal', 'label': 'treatment'}, {'entity': 'infiltrates', 'label': 'symptom'}, {'entity': 'lung vascular permeability', 'label': 'measurement'}, {'entity': 'wet', 'label': 'measurement'}, {'entity': '3B', 'label': 'measurement'}, {'entity': '4A', 'label': 'figure'}, {'entity': 'KC/CXCL1', 'label': 'protein'}, {'entity': 'MIP-2/CXCL2/3', 'label': 'protein'}, {'entity': 'KC/CXCL1', 'label': 'protein'}, {'entity': '5A', 'label': 'figure'}, {'entity': 'D', 'label': 'protein'}, {'entity': 'anti-murine', 'label': 'protein'}, {'entity': '7A', 'label': 'figure'}, {'entity': '7B', 'label': 'measurement'}, {'entity': '7C', 'label': 'measurement'}, {'entity': 'innate component', 'label': 'immune response'}, {'entity': 'adaptive component', 'label': 'immune response'}, {'entity': 'CC', 'label': 'protein'}, {'entity': '10 ug/g', 'label': 'measurement'}, {'entity': '72 h timepoint', 'label': 'measurement'}, {'entity': 'bronchial epithelial cells', 'label': 'protein'}, {'entity': 'acute phase', 'label': 'symptom'}, {'entity': 'Persistent high-resolution computed tomography', 'label': 'diagnostic'}, {'entity': 'repeat bronchoscopy', 'label': 'procedure'}, {'entity': 'alveolar cell', 'label': 'disease'}, {'entity': 'multinucleated giant cell', 'label': 'cell'}, {'entity': 'groundglass', 'label': 'diagnostic'}, {'entity': 'inflammatory abnormalities', 'label': 'disease'}, {'entity': 'acute phase', 'label': 'disease'}, {'entity': 'Chang Gung Memorial Hospital', 'label': 'organization'}, {'entity': 'modified Centers for Disease Control and Prevention (CDC) case definition of SARS', 'label': 'definition'}, {'entity': 'bronchoscopic', 'label': 'method'}, {'entity': 'supine', 'label': 'clinical trial'}, {'entity': 'inferior pulmonary vein', 'label': 'location'}, {'entity': 'CT score', 'label': 'measurement'}, {'entity': 'subsegmental bronchus', 'label': 'location'}, {'entity': 'cell pellet', 'label': 'measurement'}, {'entity': 'anti-IgG1', 'label': 'protein'}, {'entity': 'Becton Dickinson BD LYSYS II', 'label': 'method'}, {'entity': 'virus detection', 'label': 'diagnostic'}, {'entity': 'Hitachi, Tokyo', 'label': 'location'}, {'entity': 'factors', 'label': 'method'}, {'entity': 'stepwise', 'label': 'method'}, {'entity': 'ventilatory support', 'label': 'treatment'}, {'entity': 'consolidation', 'label': 'disease'}, {'entity': 'Honeycombing', 'label': 'symptom'}, {'entity': 'factors', 'label': 'clinical trial'}, {'entity': 'influenza viruses A', 'label': 'virus'}, {'entity': 'last epidemic of SARS in Taiwan', 'label': 'event'}, {'entity': '2004', 'label': 'year'}, {'entity': 'Residual abnormality on HRCT', 'label': 'symptom'}, {'entity': 'coronavirus infection', 'label': 'disease'}, {'entity': '90th day of illness', 'label': 'event'}, {'entity': 'acute phase', 'label': 'clinical trial'}, {'entity': 'outcome', 'label': 'clinical trial'}, {'entity': 'non-randomized', 'label': 'clinical trial'}, {'entity': 'outcome', 'label': 'disease'}, {'entity': 'CD8+ T cell', 'label': 'cell'}, {'entity': 'class I', 'label': 'gene'}, {'entity': 'PRED(BALB/c)', 'label': 'clinical trial'}, {'entity': 'H2(d)', 'label': 'gene'}, {'entity': 'Self protein', 'label': 'protein'}, {'entity': 'class I', 'label': 'gene'}, {'entity': 'H2-K k', 'label': 'gene'}, {'entity': 'anchor positions', 'label': 'protein'}, {'entity': 'validated', 'label': 'method'}, {'entity': 'JenPep', 'label': 'organization'}, {'entity': '22', 'label': 'measurement'}, {'entity': '23', 'label': 'measurement'}, {'entity': 'training set', 'label': 'measurement'}, {'entity': 'I-E d', 'label': 'protein'}, {'entity': 'I-A d', 'label': 'protein'}, {'entity': 'nucleocapsid DNA vaccine', 'label': 'vaccine'}, {'entity': 'PRED BALB/c', 'label': 'software'}, {'entity': 'SunOS', 'label': 'environment'}, {'entity': 'PRED BALB/c', 'label': 'method'}, {'entity': 'amino acid codes', 'label': 'vocabulary'}, {'entity': 'contiguous', 'label': 'protein'}, {'entity': 'binding scores', 'label': 'measurement'}, {'entity': 'SP', 'label': 'measurement'}, {'entity': 'carriage returns', 'label': 'method'}, {'entity': 'line breaks', 'label': 'method'}, {'entity': '9mer peptide', 'label': 'protein'}, {'entity': 'YPILPEYLQCVK', 'label': 'protein'}, {'entity': 'molecular biology', 'label': 'biology'}, {'entity': 'STYLE', 'label': 'method'}, {'entity': 'DNA style', 'label': 'gene'}, {'entity': 'chaos game representation', 'label': 'method'}, {'entity': '4', 'label': 'measurement'}, {'entity': '8', 'label': 'measurement'}, {'entity': '16', 'label': 'measurement'}, {'entity': '17', 'label': 'measurement'}, {'entity': 'segment', 'label': 'gene'}, {'entity': 'taxonomic', 'label': 'gene'}, {'entity': 'DNA style', 'label': 'method'}, {'entity': '31', 'label': 'publication'}, {'entity': 'homologous sequences', 'label': 'disease'}, {'entity': 'derivative work', 'label': 'publication'}, {'entity': 'collecting', 'label': 'method'}, {'entity': 'slow growth', 'label': 'disease'}, {'entity': 'Distance between signatures', 'label': 'measurement'}, {'entity': \"User's sequence\", 'label': 'virus'}, {'entity': 'charts', 'label': 'method'}, {'entity': 'subsets', 'label': 'clinical trial'}, {'entity': 'Figure 1A', 'label': 'figure'}, {'entity': 'nt long', 'label': 'measurement'}, {'entity': 'counterparts', 'label': 'gene'}, {'entity': 'SARS Extranet', 'label': 'organization'}, {'entity': 'think-aloud', 'label': 'method'}, {'entity': 'communications strategies used during the outbreak', 'label': 'method'}, {'entity': 'web browser', 'label': 'tool'}, {'entity': 'managed health care organization', 'label': 'organization'}, {'entity': 'web-based communication systems', 'label': 'technology'}, {'entity': 'medical records', 'label': 'diagnostic'}, {'entity': 'bioterrorist', 'label': 'disease'}, {'entity': 'provincial Severe Acute Respiratory Syndrome (SARS) emergency', 'label': 'event'}, {'entity': 'Health Canada', 'label': 'organization'}, {'entity': 'version control', 'label': 'method'}, {'entity': 'centralized access point', 'label': 'organization'}, {'entity': 'Virtual Private Network', 'label': 'method'}, {'entity': 'private network', 'label': 'organization'}, {'entity': 'version control', 'label': 'method'}, {'entity': 'web master', 'label': 'organization'}, {'entity': 'information dissemination', 'label': 'clinical trial'}, {'entity': 'ethics review board', 'label': 'organization'}, {'entity': 'paper-based', 'label': 'method'}, {'entity': 'SARS Extranet', 'label': 'organization'}, {'entity': 'think-aloud', 'label': 'method'}, {'entity': 'ask questions', 'label': 'method'}, {'entity': 'listening', 'label': 'method'}, {'entity': 'branching', 'label': 'method'}, {'entity': 'web version', 'label': 'publication'}, {'entity': '2003', 'label': 'year'}, {'entity': 'SPSS', 'label': 'software'}, {'entity': 'SARS Steering Committee', 'label': 'organization'}, {'entity': 'communication strategies used during the outbreak', 'label': 'method'}, {'entity': 'Ontario Ministry of Health and Long-Term Care', 'label': 'organization'}, {'entity': 'Hamilton SARS Extranet', 'label': 'organization'}, {'entity': 'why they visited', 'label': 'event'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'username', 'label': 'measurement'}, {'entity': 'Open-ended questions', 'label': 'method'}, {'entity': 'access/password', 'label': 'method'}, {'entity': 'public health emergency', 'label': 'public health'}, {'entity': 'public health emergency', 'label': 'public health'}, {'entity': 'interactive', 'label': 'method'}, {'entity': 'bulletin board', 'label': 'treatment'}, {'entity': 'Twoway', 'label': 'method'}, {'entity': 'interactive technologies', 'label': 'measurement'}, {'entity': 'text chat', 'label': 'method'}, {'entity': 'Internet site', 'label': 'environment'}, {'entity': 'Hamilton directive', 'label': 'publication'}, {'entity': 'provincial SARS site', 'label': 'organization'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'web communication', 'label': 'method'}, {'entity': 'sharing passwords', 'label': 'method'}, {'entity': 'password-protected', 'label': 'method'}, {'entity': 'security policies', 'label': 'method'}, {'entity': 'Communication on communicable disease', 'label': 'diagnostic'}, {'entity': 'SARS emergency', 'label': 'event'}, {'entity': 'initial SARS outbreak', 'label': 'event'}, {'entity': 'sites', 'label': 'location'}, {'entity': 'interactivity', 'label': 'measurement'}, {'entity': 'pre-hospital', 'label': 'treatment'}, {'entity': 'disaster medicine', 'label': 'clinical trial'}, {'entity': 'ambulances to nowhere', 'label': 'event'}, {'entity': 'Madrid bombing explosions', 'label': 'event'}, {'entity': 'critical care infrastructure', 'label': 'organization'}, {'entity': 'Tokyo Sarin gas subway attack', 'label': 'event'}, {'entity': 'overcrowding', 'label': 'symptom'}, {'entity': 'critical care nurses', 'label': 'person'}, {'entity': 'infectious disease epidemic', 'label': 'disease'}, {'entity': '2003', 'label': 'year'}, {'entity': 'critical care personnel', 'label': 'person'}, {'entity': 'imperatives', 'label': 'clinical trial'}, {'entity': 'severe acute respiratory syndrome outbreak of 2003', 'label': 'event'}, {'entity': 'disaster medical education', 'label': 'organization'}, {'entity': 'dialog', 'label': 'clinical trial'}, {'entity': 'healthcare professionals', 'label': 'person'}, {'entity': 'self-preservation training', 'label': 'method'}, {'entity': 'healthcare professionals', 'label': 'person'}, {'entity': 'The content of this program', 'label': 'clinical trial'}, {'entity': 'responders', 'label': 'person'}, {'entity': 'effective', 'label': 'clinical trial'}, {'entity': 'resource management', 'label': 'method'}, {'entity': 'just-in-time training', 'label': 'method'}, {'entity': 'queried', 'label': 'method'}, {'entity': 'labor intensive', 'label': 'measurement'}, {'entity': 'just-in-time training', 'label': 'method'}, {'entity': 'professional', 'label': 'person'}, {'entity': 'plan', 'label': 'treatment'}, {'entity': 'efficient', 'label': 'measurement'}, {'entity': 'pre-planning', 'label': 'method'}, {'entity': 'disaster medical education', 'label': 'clinical trial'}, {'entity': 'cost', 'label': 'measurement'}, {'entity': 'dual-use modalities', 'label': 'method'}, {'entity': 'critical care unit', 'label': 'disease'}, {'entity': 'tiered', 'label': 'method'}, {'entity': 'mass casualty', 'label': 'event'}, {'entity': 'disaster medical response', 'label': 'event'}, {'entity': 'prehospital', 'label': 'clinical trial'}, {'entity': 'present article', 'label': 'publication'}, {'entity': 'cell-based systems', 'label': 'method'}, {'entity': 'characterization', 'label': 'method'}, {'entity': 'robustness of measurements', 'label': 'measurement'}, {'entity': 'identification methods', 'label': 'diagnostic'}, {'entity': 'revolution in new high-throughput approaches', 'label': 'method'}, {'entity': 'limitations', 'label': 'treatment'}, {'entity': 'cell-based systems', 'label': 'method'}, {'entity': 'simultaneously', 'label': 'method'}, {'entity': 'viral chips', 'label': 'diagnostic'}, {'entity': '1999', 'label': 'year'}, {'entity': 'early', 'label': 'treatment'}, {'entity': 'staging', 'label': 'diagnostic'}, {'entity': '2002', 'label': 'year'}, {'entity': 'detection and identification', 'label': 'diagnostic'}, {'entity': 'viral barcodes', 'label': 'diagnostic'}, {'entity': 'serotype', 'label': 'virus'}, {'entity': 'subtype', 'label': 'virus'}, {'entity': 'detection and distinction', 'label': 'diagnostic'}, {'entity': 'microarray analysis of viral recombination', 'label': 'method'}, {'entity': 'reverse-transcribed', 'label': 'method'}, {'entity': 'host chip', 'label': 'organization'}, {'entity': 'viral barcode', 'label': 'diagnostic'}, {'entity': 'serum antibodies', 'label': 'protein'}, {'entity': 'protein analyte', 'label': 'protein'}, {'entity': 'identifier', 'label': 'virus'}, {'entity': 'polymers', 'label': 'vaccine'}, {'entity': 'complementary technologies', 'label': 'technology'}, {'entity': 'hybridization signature', 'label': 'diagnostic'}, {'entity': 'engineered cells', 'label': 'gene'}, {'entity': 'technology', 'label': 'method'}, {'entity': 'Array', 'label': 'method'}, {'entity': 'detection system', 'label': 'method'}, {'entity': 'bench', 'label': 'method'}, {'entity': 'barcodes', 'label': 'viral barcodes'}, {'entity': 'Infection Profile Database', 'label': 'organization'}, {'entity': 'synthetic sensors', 'label': 'method'}, {'entity': 'unique signature', 'label': 'virus'}, {'entity': 'various responses', 'label': 'immune response'}, {'entity': 'realistic', 'label': 'clinical trial'}, {'entity': 'quality assurance', 'label': 'measurement'}, {'entity': 'Wimley-White interfacial hydrophobicity scale', 'label': 'method'}, {'entity': '50% inhibitory concentration', 'label': 'measurement'}, {'entity': 'stem domain', 'label': 'protein'}, {'entity': 'WN83', 'label': 'protein'}, {'entity': 'metastable', 'label': 'protein'}, {'entity': 'lipid bilayer', 'label': 'structure'}, {'entity': 'viral envelope protein', 'label': 'protein'}, {'entity': 'insect vector', 'label': 'virus'}, {'entity': 'DF', 'label': 'disease'}, {'entity': 'Cross-reactive', 'label': 'vaccine'}, {'entity': 'four serotypes', 'label': 'virus'}, {'entity': 'WNV E', 'label': 'protein'}, {'entity': 'pre-anchor domain', 'label': 'protein'}, {'entity': 'DENV E', 'label': 'virus'}, {'entity': 'domain III', 'label': 'protein'}, {'entity': 'DENV E', 'label': 'protein'}, {'entity': 'WNV E', 'label': 'virus'}, {'entity': 'virus adsorption', 'label': 'process'}, {'entity': '50% inhibitory concentration', 'label': 'measurement'}, {'entity': 'WN83', 'label': 'protein'}, {'entity': 'pre-anchor domain', 'label': 'protein'}, {'entity': 'WN83', 'label': 'drug'}, {'entity': 'WN83', 'label': 'virus'}, {'entity': 'cell metabolism', 'label': 'measurement'}, {'entity': 'E domain', 'label': 'protein'}, {'entity': '10 µM', 'label': 'measurement'}, {'entity': 'primary sequence', 'label': 'protein'}, {'entity': \"Dunnett's posthoc test\", 'label': 'method'}, {'entity': 'Effect of scrambled peptide order on inhibitory function', 'label': 'clinical trial'}, {'entity': 'WN83', 'label': 'protein'}, {'entity': 'trimer', 'label': 'protein'}, {'entity': 'stem domain', 'label': 'protein'}, {'entity': 'I/II junction', 'label': 'protein'}, {'entity': 'DENV E', 'label': 'protein'}, {'entity': 'WN83', 'label': 'virus'}, {'entity': 'flavivirus neutralizing antibodies', 'label': 'protein'}, {'entity': 'ini-tial', 'label': 'clinical trial'}, {'entity': 'resistant', 'label': 'treatment'}, {'entity': 'Viral plaque reduction assay', 'label': 'diagnostic'}, {'entity': 'fluorescing', 'label': 'virus'}, {'entity': 'polyclonal', 'label': 'vaccine'}, {'entity': 'TACS™', 'label': 'method'}, {'entity': 'control measures', 'label': 'treatment'}, {'entity': 'biases', 'label': 'method'}, {'entity': 'influenza outbreak', 'label': 'epidemiology'}, {'entity': 'a priori', 'label': 'method'}, {'entity': 'model structure', 'label': 'method'}, {'entity': 'classical frameworks', 'label': 'epidemiology'}, {'entity': 'control measures', 'label': 'treatment'}, {'entity': 'biases', 'label': 'virus'}, {'entity': 'disease outbreak', 'label': 'event'}, {'entity': '2001 outbreak of foot-and-mouth disease in the United Kingdom', 'label': 'event'}, {'entity': 'systematic bias', 'label': 'measurement'}, {'entity': 'susceptible-infectious-recovered', 'label': 'epidemiology'}, {'entity': 'infectious class', 'label': 'disease'}, {'entity': 'latent class', 'label': 'disease'}, {'entity': 'gamma', 'label': 'measurement'}, {'entity': 'subclasses', 'label': 'method'}, {'entity': 'asymptotic', 'label': 'measurement'}, {'entity': 'schools', 'label': 'location'}, {'entity': 'equation 4', 'label': 'method'}, {'entity': 'SARS outbreak of 2003', 'label': 'event'}, {'entity': 'transmission potential', 'label': 'transmission'}, {'entity': 'model parameters', 'label': 'clinical trial'}, {'entity': 'exponential', 'label': 'method'}, {'entity': 'influenza outbreak', 'label': 'disease'}, {'entity': 'dominant eigenvalue', 'label': 'measurement'}, {'entity': 'epidemic growth rate', 'label': 'measurement'}, {'entity': 'exposed class', 'label': 'model'}, {'entity': 'shape', 'label': 'measurement'}, {'entity': 'classical exponentially distributed model', 'label': 'model'}, {'entity': 'periods of fixed lengths', 'label': 'measurement'}, {'entity': 'Estimating R 0 from Trajectory Matching', 'label': 'method'}, {'entity': 'influenza outbreak', 'label': 'event'}, {'entity': 'least squares error', 'label': 'measurement'}, {'entity': 'best-fit parameters', 'label': 'measurement'}, {'entity': 'best fit', 'label': 'clinical trial'}, {'entity': 'SIR best-fit', 'label': 'model'}, {'entity': 'SEIR best-fit', 'label': 'method'}, {'entity': 'gamma-distributed SEIR model', 'label': 'model'}, {'entity': 'epidemic growth rate', 'label': 'measurement'}, {'entity': 'individual-level control', 'label': 'treatment'}, {'entity': 'asymptomatic phase', 'label': 'symptom'}, {'entity': 'Figure 1A', 'label': 'publication'}, {'entity': 'infectious class', 'label': 'treatment'}, {'entity': 'gamma-distributed infectious period', 'label': 'measurement'}, {'entity': 'quarantine period', 'label': 'treatment'}, {'entity': 'models in epidemiology', 'label': 'epidemiology'}, {'entity': 'classical frameworks', 'label': 'epidemiology'}, {'entity': 'cis', 'label': 'gene'}, {'entity': '-1 frameshifting', 'label': 'protein'}, {'entity': 'bronchitis virus', 'label': 'virus'}, {'entity': 'linker', 'label': 'gene'}, {'entity': 'atypical pseudoknot', 'label': 'protein'}, {'entity': 'attenuation signal', 'label': 'virus'}, {'entity': 'cis RNA attenuator', 'label': 'virus'}, {'entity': 'interference of viral propagation', 'label': 'treatment'}, {'entity': 'stimulator', 'label': 'gene'}, {'entity': 'helical stems', 'label': 'protein'}, {'entity': 'non-pseudoknot', 'label': 'virus'}, {'entity': 'E-site', 'label': 'protein'}, {'entity': 'efficiency modulation', 'label': 'host'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': 'secondary structure', 'label': 'structure'}, {'entity': 'DNA primers', 'label': 'method'}, {'entity': 'Frameshifting stimulator', 'label': 'gene'}, {'entity': 'reverse', 'label': 'method'}, {'entity': 'quick-change mutagenesis kit', 'label': 'method'}, {'entity': 'sense', 'label': 'gene'}, {'entity': 'reverse', 'label': 'method'}, {'entity': 'SARS 13 377-13 475', 'label': 'gene'}, {'entity': '5 0 end', 'label': 'measurement'}, {'entity': '50 ml', 'label': 'measurement'}, {'entity': '5 0 end labeled', 'label': 'measurement'}, {'entity': 'denatured', 'label': 'method'}, {'entity': 'unfolded RNA', 'label': 'protein'}, {'entity': '20 C', 'label': 'measurement'}, {'entity': 'phosphorimagery', 'label': 'method'}, {'entity': 'T7-coupled transcription/translation', 'label': 'method'}, {'entity': 'S-labeled', 'label': 'measurement'}, {'entity': '100-1000 ng', 'label': 'measurement'}, {'entity': 'ABI TR 717', 'label': 'measurement'}, {'entity': 'firefly ORF', 'label': 'protein'}, {'entity': 'SARS 13 369-13 520 RNA', 'label': 'virus'}, {'entity': 'SARS 13 377-13 475 RNA', 'label': 'gene'}, {'entity': 'gel analysis', 'label': 'method'}, {'entity': 'duplex conformations', 'label': 'protein'}, {'entity': 'base', 'label': 'protein'}, {'entity': 'SARS 13 369-13 520 RNA', 'label': 'virus'}, {'entity': 'lane 3', 'label': 'measurement'}, {'entity': 'SARS 13 222-13 368', 'label': 'virus'}, {'entity': 'reporter', 'label': 'gene'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'A residue', 'label': 'protein'}, {'entity': 'U residue', 'label': 'protein'}, {'entity': 'SARS 13 222-13 368 RNA', 'label': 'gene'}, {'entity': 'slippery site', 'label': 'gene'}, {'entity': 'extension', 'label': 'method'}, {'entity': 'shifted', 'label': 'measurement'}, {'entity': 'cis', 'label': 'gene'}, {'entity': 'slippery site', 'label': 'protein'}, {'entity': 'Figure 6A', 'label': 'figure'}, {'entity': 'IBV pseudoknot', 'label': 'gene'}, {'entity': 'downstream', 'label': 'gene'}, {'entity': '4A', 'label': 'publication'}, {'entity': 'reporter system', 'label': 'method'}, {'entity': 'pseudoknot of long S1', 'label': 'protein'}, {'entity': 'SARS CoV pseudoknot', 'label': 'protein'}, {'entity': 'SARS CoV pseudoknot', 'label': 'protein'}, {'entity': 'transmissible gastroenteritis virus', 'label': 'virus'}, {'entity': 'SARS CoV pseudoknot', 'label': 'protein'}, {'entity': 'BWYV pseudoknot', 'label': 'protein'}, {'entity': 'upstream attenuation', 'label': 'gene'}, {'entity': 'mutational', 'label': 'gene'}, {'entity': 'intervenient spacer', 'label': 'gene'}, {'entity': 'À1 frameshifting', 'label': 'protein'}, {'entity': 'BWYV pseudoknot', 'label': 'gene'}, {'entity': 'potential hydrogen numbers', 'label': 'measurement'}, {'entity': 'source information', 'label': 'measurement'}, {'entity': 'destination information', 'label': 'measurement'}, {'entity': 'computer program', 'label': 'method'}, {'entity': 'PUNS', 'label': 'software'}, {'entity': 'PRIMEX', 'label': 'method'}, {'entity': 'base digitalization', 'label': 'method'}, {'entity': \"Shannon's\", 'label': 'method'}, {'entity': \"Shannon's information theory\", 'label': 'method'}, {'entity': 'conserved regions', 'label': 'measurement'}, {'entity': 'Candidates of Annealing Sites', 'label': 'protein'}, {'entity': 'TU', 'label': 'variable'}, {'entity': 'perfect match', 'label': 'measurement'}, {'entity': 'set I up', 'label': 'measurement'}, {'entity': 'arithmetic average', 'label': 'method'}, {'entity': 'geometric average', 'label': 'method'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'downstream', 'label': 'gene'}, {'entity': 'Win32', 'label': 'environment'}, {'entity': 'text file', 'label': 'publication'}, {'entity': 'result file', 'label': 'publication'}, {'entity': 'upstream', 'label': 'measurement'}, {'entity': 'predicted products', 'label': 'measurement'}, {'entity': 'Virtual PCR', 'label': 'clinical trial'}, {'entity': 'gene families', 'label': 'gene'}, {'entity': 'threshold parameters', 'label': 'measurement'}, {'entity': 'binding domain', 'label': 'protein'}, {'entity': 'IN-2(+)/IN-4(-)', 'label': 'primer'}, {'entity': 'taxonomy', 'label': 'epidemiology'}, {'entity': 'PCR primers', 'label': 'method'}, {'entity': 'I up', 'label': 'parameter'}, {'entity': 'change of primer pairs', 'label': 'treatment'}, {'entity': 'primer length', 'label': 'measurement'}, {'entity': 'product prediction', 'label': 'method'}, {'entity': 'fidelity', 'label': 'measurement'}, {'entity': 'degenerate primers', 'label': 'gene'}, {'entity': 'complex templates', 'label': 'measurement'}, {'entity': 'base stacking', 'label': 'method'}, {'entity': 'SPCR program', 'label': 'organization'}, {'entity': 'Win32', 'label': 'environment'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': 'replicase 1a/1b', 'label': 'gene'}, {'entity': '8a', 'label': 'gene'}, {'entity': '8b', 'label': 'gene'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': \"6T's\", 'label': 'measurement'}, {'entity': '3a', 'label': 'gene'}, {'entity': \"6T's\", 'label': 'mutation'}, {'entity': \"9T's\", 'label': 'mutation'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'in-frame', 'label': 'measurement'}, {'entity': '3a', 'label': 'gene'}, {'entity': '3a', 'label': 'gene'}, {'entity': 'N-terminus', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': '3b', 'label': 'gene'}, {'entity': 'foot-and-mouth disease virus', 'label': 'virus'}, {'entity': '3a', 'label': 'gene'}, {'entity': 'endocytotic', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'monofunctional PSG', 'label': 'gene'}, {'entity': 'homogenisation', 'label': 'gene'}, {'entity': 'homogenisation', 'label': 'term'}, {'entity': 'unequal crossover', 'label': 'method'}, {'entity': 'families', 'label': 'gene'}, {'entity': 'programmed mutational mechanisms', 'label': 'gene'}, {'entity': 'immunoglobulin heavy-chain variableregion', 'label': 'protein'}, {'entity': 'divergence', 'label': 'gene'}, {'entity': 'PSG family', 'label': 'protein'}, {'entity': 'tails', 'label': 'protein'}, {'entity': 'A domain', 'label': 'protein'}, {'entity': 'parent-offspring', 'label': 'gene'}, {'entity': 'offspring', 'label': 'host'}, {'entity': 'extracellular matrix', 'label': 'protein'}, {'entity': 'trophoblast', 'label': 'protein'}, {'entity': 'PSG sequence divergence', 'label': 'gene'}, {'entity': 'CEA family', 'label': 'gene'}, {'entity': 'domain', 'label': 'pathway'}, {'entity': 'conservation', 'label': 'measurement'}, {'entity': 'human B2', 'label': 'gene'}, {'entity': 'alternative leader and N1 domain exons', 'label': 'gene'}, {'entity': 'predicted sequences', 'label': 'measurement'}, {'entity': 'McLenachan et al.', 'label': 'organization'}, {'entity': 'branch points', 'label': 'method'}, {'entity': 'evolutionary tree', 'label': 'method'}, {'entity': 'domain', 'label': 'gene'}, {'entity': 'N1 domain exon', 'label': 'gene'}, {'entity': '2011', 'label': 'year'}, {'entity': 'Box and whisker plots for Dayhoff PAM 250 scores', 'label': 'method'}, {'entity': 'human PSG', 'label': 'gene'}, {'entity': 'box and whisker plot', 'label': 'diagram'}, {'entity': 'N/N1', 'label': 'protein'}, {'entity': 'Psg30', 'label': 'protein'}, {'entity': '3A', 'label': 'figure'}, {'entity': 'SplitsTree graph', 'label': 'method'}, {'entity': 'orthologous', 'label': 'gene'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'PSG36', 'label': 'gene'}, {'entity': 'Dayhoff chart', 'label': 'publication'}, {'entity': '10', 'label': 'figure'}, {'entity': 'protein folding', 'label': 'protein'}, {'entity': 'box', 'label': 'diagnostic'}, {'entity': '11', 'label': 'measurement'}, {'entity': '45° line of equivalence', 'label': 'measurement'}, {'entity': 'CFG face', 'label': 'diagnostic'}, {'entity': 'non-synonymous', 'label': 'gene'}, {'entity': 'PSG N1 domain', 'label': 'protein'}, {'entity': 'N1', 'label': 'region'}, {'entity': 'CGA GGA GAT', 'label': 'gene'}, {'entity': 'motifs', 'label': 'gene'}, {'entity': 'transition', 'label': 'method'}, {'entity': 'RGDlike', 'label': 'protein'}, {'entity': 'primate', 'label': 'gene'}, {'entity': 'PSG30', 'label': 'gene'}, {'entity': 'PAE motif', 'label': 'protein'}, {'entity': 'HAE', 'label': 'protein'}, {'entity': 'snake venom disintegrins', 'label': 'protein'}, {'entity': 'coevolution', 'label': 'process'}, {'entity': 'Windows 2000', 'label': 'environment'}, {'entity': 'PROSITE', 'label': 'organization'}, {'entity': 'human coronavirus NL63', 'label': 'virus'}, {'entity': 'Paediatric Respiratory Infection in Germany', 'label': 'event'}, {'entity': 'PRI.DE study', 'label': 'clinical trial'}, {'entity': 'snap freeze', 'label': 'method'}, {'entity': 'generating random numbers', 'label': 'method'}, {'entity': 'proper representation', 'label': 'measurement'}, {'entity': 'winter', 'label': 'season'}, {'entity': 'Mortlake', 'label': 'location'}, {'entity': 'Carlsbad, California', 'label': 'location'}, {'entity': 'Madison, Wisconsin', 'label': 'location'}, {'entity': 'detection limit', 'label': 'measurement'}, {'entity': 'dilution', 'label': 'method'}, {'entity': 'crossing point', 'label': 'measurement'}, {'entity': \"Wilcoxon's two-sample test\", 'label': 'method'}, {'entity': 'dependent variable', 'label': 'variable'}, {'entity': 'date of sampling', 'label': 'measurement'}, {'entity': 'outpatient', 'label': 'clinical trial'}, {'entity': 'coinfected', 'label': 'symptom'}, {'entity': 'previous survey', 'label': 'clinical trial'}, {'entity': '2000-2001', 'label': 'year'}, {'entity': 'fall', 'label': 'event'}, {'entity': 'winter', 'label': 'transmission'}, {'entity': \"Fisher's exact test\", 'label': 'method'}, {'entity': 'viral clearance', 'label': 'virus'}, {'entity': 'defrosted', 'label': 'method'}, {'entity': 'Severe Acute Respiratory Syndrome coronavirus', 'label': 'virus'}, {'entity': 'C- terminus', 'label': 'protein'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'PSORT II', 'label': 'method'}, {'entity': 'forward scatter', 'label': 'measurement'}, {'entity': 'intra-culture', 'label': 'treatment'}, {'entity': 'positive cells', 'label': 'protein'}, {'entity': 'G0/G1 phase', 'label': 'protein'}, {'entity': '2005', 'label': 'year'}, {'entity': 'sub-G1', 'label': 'measurement'}, {'entity': 'G0/G1 phase', 'label': 'pathway'}, {'entity': '293', 'label': 'cell'}, {'entity': 'G0/', 'label': 'cell cycle stage'}, {'entity': 'G0/G1 phase arrest', 'label': 'treatment'}, {'entity': '3b', 'label': 'protein'}, {'entity': '3b', 'label': 'gene'}, {'entity': 'multiple-membrane-spanning', 'label': 'protein'}, {'entity': 'glycosylated', 'label': 'protein'}, {'entity': 'glycosyla-tion', 'label': 'protein'}, {'entity': 'virus fusion', 'label': 'pathway'}, {'entity': 'receptor glycosylation', 'label': 'protein'}, {'entity': 'Glycosylation-specific enhancement', 'label': 'method'}, {'entity': 'non-specific enhancement', 'label': 'clinical trial'}, {'entity': 'unglycosylated', 'label': 'protein'}, {'entity': '35 µM PS', 'label': 'measurement'}, {'entity': '293T', 'label': 'cell'}, {'entity': 'virus fusion', 'label': 'virus'}, {'entity': 'trypsinized', 'label': 'method'}, {'entity': 'repeated', 'label': 'method'}, {'entity': 'non-specific enhancement', 'label': 'treatment'}, {'entity': 'outer leaflet', 'label': 'location'}, {'entity': 'saturation', 'label': 'measurement'}, {'entity': 'University of California, Berkeley', 'label': 'organization'}, {'entity': 'virus fusion', 'label': 'pathway'}, {'entity': 'cognate receptor', 'label': 'protein'}, {'entity': 'fusogenic', 'label': 'protein'}, {'entity': 'NIH 3T3/ASCT2', 'label': 'organization'}, {'entity': 'lipid bilayer', 'label': 'protein'}, {'entity': 'Eagle medium', 'label': 'measurement'}, {'entity': 'Minimal Essential Medium Alpha', 'label': 'measurement'}, {'entity': 'Annexin-V labeling', 'label': 'method'}, {'entity': 'W-385', 'label': 'equipment'}, {'entity': '2′-O-methyl nucleotide', 'label': 'measurement'}, {'entity': 'nearest-neighbor free energy increments', 'label': 'measurement'}, {'entity': 'modified', 'label': 'method'}, {'entity': 'nearestneighbor model', 'label': 'method'}, {'entity': 'binding affinity', 'label': 'measurement'}, {'entity': 'nearest-neighbor model', 'label': 'method'}, {'entity': 'Beckman DU 640', 'label': 'equipment'}, {'entity': 'MeltWin', 'label': 'software'}, {'entity': 'Microsoft Excel', 'label': 'organization'}, {'entity': 'T m À1 versus ln', 'label': 'measurement'}, {'entity': '5 0', 'label': 'measurement'}, {'entity': '5 0', 'label': 'measurement'}, {'entity': 'GC pair', 'label': 'gene'}, {'entity': '37 C', 'label': 'measurement'}, {'entity': 'neighbor', 'label': 'gene'}, {'entity': '5 0', 'label': 'measurement'}, {'entity': 'nearest-neighbor combination', 'label': 'method'}, {'entity': 'CAAGCC', 'label': 'sequence'}, {'entity': 'nearest neighbor', 'label': 'method'}, {'entity': '2 0 -O-methyl RNA', 'label': 'measurement'}, {'entity': 'RNA/RNA duplex', 'label': 'measurement'}, {'entity': 'A-form', 'label': 'structure'}, {'entity': '2 0 -O-methyl RNA', 'label': 'molecule'}, {'entity': '2 0 -O-methyl RNA/RNA duplex', 'label': 'protein'}, {'entity': 'CA nearest neighbor', 'label': 'measurement'}, {'entity': '2 0 -O-methyl RNA/RNA duplexes', 'label': 'measurement'}, {'entity': 'RNA/RNA duplex', 'label': 'protein'}, {'entity': 'sequence design', 'label': 'method'}, {'entity': 'experimental error', 'label': 'measurement'}, {'entity': 'Watson-Crick complementary duplex', 'label': 'measurement'}, {'entity': 'GU pair', 'label': 'protein'}, {'entity': '2 0 -O-methyl', 'label': 'measurement'}, {'entity': 'mismatch penalty', 'label': 'measurement'}, {'entity': '2 0 -O-methyl RNA', 'label': 'protein'}, {'entity': 'internal LNAs', 'label': 'protein'}, {'entity': '5 0', 'label': 'measurement'}, {'entity': 'non-terminal', 'label': 'measurement'}, {'entity': 'sequence independent', 'label': 'measurement'}, {'entity': 'internal', 'label': 'measurement'}, {'entity': 'mismatch penalty', 'label': 'measurement'}, {'entity': 'mismatch', 'label': 'virus'}, {'entity': 'simulated cough machine', 'label': 'method'}, {'entity': 'crosslinking', 'label': 'measurement'}, {'entity': 'providing care', 'label': 'treatment'}, {'entity': 'SARS outbreak', 'label': 'disease'}, {'entity': 'face shield', 'label': 'vaccine'}, {'entity': 'seal', 'label': 'vaccine'}, {'entity': 'cost-effective', 'label': 'measurement'}, {'entity': 'new active cases of TB', 'label': 'disease'}, {'entity': '2002', 'label': 'year'}, {'entity': 'multiple, first-line anti-TB drugs', 'label': 'drug'}, {'entity': 'transmissor', 'label': 'transmission'}, {'entity': 'airway surface fluid', 'label': 'diagnostic'}, {'entity': 'mucociliary', 'label': 'protein'}, {'entity': 'Schematic diagram of the macromolecules of mucus with the different types of crosslinks', 'label': 'diagram'}, {'entity': 'divalent', 'label': 'protein'}, {'entity': 'mucosa', 'label': 'symptom'}, {'entity': 'direct', 'label': 'transmission'}, {'entity': 'mucus retention', 'label': 'disease'}, {'entity': 'co-adjuvant', 'label': 'drug'}, {'entity': 'mucus simulant', 'label': 'diagnostic'}, {'entity': 'mucus simulant', 'label': 'diagnostic'}, {'entity': 'simulated cough machine', 'label': 'device'}, {'entity': 'pharynx', 'label': 'anatomy'}, {'entity': 'frog Ringer solution', 'label': 'measurement'}, {'entity': 'frog chamber', 'label': 'location'}, {'entity': 'Pari jet', 'label': 'device'}, {'entity': 'frog Ringer', 'label': 'measurement'}, {'entity': 'stereomicroscope', 'label': 'method'}, {'entity': 'MTV', 'label': 'measurement'}, {'entity': 'graduated', 'label': 'measurement'}, {'entity': 'palate', 'label': 'location'}, {'entity': 'sucked in', 'label': 'method'}, {'entity': 'voltage supply', 'label': 'measurement'}, {'entity': 'mouth', 'label': 'location'}, {'entity': 'airflow pattern', 'label': 'measurement'}, {'entity': 'end-piece', 'label': 'vaccine'}, {'entity': 'solenoid', 'label': 'device'}, {'entity': 'cough clearability', 'label': 'measurement'}, {'entity': \"Millonig's buffer\", 'label': 'measurement'}, {'entity': 'absolute ethanol', 'label': 'measurement'}, {'entity': 'vacuum desiccator', 'label': 'equipment'}, {'entity': 'image file', 'label': 'measurement'}, {'entity': 'cough maneuver', 'label': 'method'}, {'entity': 'mucin exocytosis', 'label': 'process'}, {'entity': 'watery', 'label': 'measurement'}, {'entity': 'frog Ringer solution', 'label': 'measurement'}, {'entity': 'significant differences', 'label': 'measurement'}, {'entity': 'mucus simulant', 'label': 'measurement'}, {'entity': 'cough maneuver', 'label': 'method'}, {'entity': 'significant difference', 'label': 'measurement'}, {'entity': 'mouth', 'label': 'location'}, {'entity': 'fine aerosol', 'label': 'measurement'}, {'entity': 'cough maneuver', 'label': 'method'}, {'entity': 'p < 0.05', 'label': 'measurement'}, {'entity': 'frog Ringer solution', 'label': 'diagnostic'}, {'entity': 'ex vivo', 'label': 'method'}, {'entity': 'crosslinking', 'label': 'method'}, {'entity': 'airway mucus simulant', 'label': 'drug'}, {'entity': 'cough maneuver', 'label': 'method'}, {'entity': 'modulation', 'label': 'treatment'}, {'entity': 'SE micrograph', 'label': 'diagnostic'}, {'entity': 'ciliated surface', 'label': 'structure'}, {'entity': 'pre-clinical studies', 'label': 'clinical trial'}, {'entity': 'no competing interests', 'label': 'organization'}, {'entity': 'health implications', 'label': 'epidemiology'}, {'entity': 'proximal determinants', 'label': 'conceptual model'}, {'entity': 'think-model', 'label': 'concept'}, {'entity': 'rich', 'label': 'person'}, {'entity': 'poor', 'label': 'host'}, {'entity': 'economic process', 'label': 'method'}, {'entity': 'events', 'label': 'event'}, {'entity': 'effects of globalisation', 'label': 'event'}, {'entity': 'think-model', 'label': 'method'}, {'entity': 'economic factors', 'label': 'measurement'}, {'entity': 'health effects of the globalisation process', 'label': 'clinical trial'}, {'entity': 'interconnected', 'label': 'concept'}, {'entity': 'physical morbidity', 'label': 'disease'}, {'entity': 'Huynen et al', 'label': 'organization'}, {'entity': 'multi-causality', 'label': 'concept'}, {'entity': 'hierarchical levels of causality', 'label': 'concept'}, {'entity': 'proximal', 'label': 'clinical trial'}, {'entity': 'contextual determinant', 'label': 'epidemiology'}, {'entity': 'macro-level', 'label': 'epidemiology'}, {'entity': 'ill health', 'label': 'disease'}, {'entity': 'cultural', 'label': 'environment'}, {'entity': 'supranational', 'label': 'organization'}, {'entity': 'interdeterminacy', 'label': 'concept'}, {'entity': 'diffusion of information', 'label': 'diagnostic'}, {'entity': 'cross-cultural interaction', 'label': 'event'}, {'entity': 'health determinants', 'label': 'disease'}, {'entity': 'other levels', 'label': 'concept'}, {'entity': 'institutional response', 'label': 'organization'}, {'entity': 'WTO agreements', 'label': 'organization'}, {'entity': 'New global governance structure', 'label': 'organization'}, {'entity': 'Global mobility', 'label': 'event'}, {'entity': 'Globalising cultural flows', 'label': 'event'}, {'entity': 'loss of biodiversity', 'label': 'environment'}, {'entity': 'economic globalisation', 'label': 'concept'}, {'entity': 'Dollar', 'label': 'organization'}, {'entity': 'Kraay', 'label': 'person'}, {'entity': 'health of the poor', 'label': 'disease'}, {'entity': \"1980's\", 'label': 'period'}, {'entity': '1950', 'label': 'year'}, {'entity': '1997', 'label': 'year'}, {'entity': 'trade internationally', 'label': 'event'}, {'entity': 'primary commodities', 'label': 'measurement'}, {'entity': 'illegal drug trade', 'label': 'drug'}, {'entity': 'transportation', 'label': 'transmission'}, {'entity': 'tourism', 'label': 'event'}, {'entity': 'pleasure', 'label': 'location'}, {'entity': 'resettlement by refugees', 'label': 'event'}, {'entity': '1970s', 'label': 'year'}, {'entity': 'cross-cultural', 'label': 'event'}, {'entity': 'religious fundamentalism', 'label': 'event'}, {'entity': 'intralevel', 'label': 'measurement'}, {'entity': 'cross-cultural interaction', 'label': 'event'}, {'entity': 'social exclusion', 'label': 'event'}, {'entity': 'intimate social contacts', 'label': 'host'}, {'entity': 'peace movement', 'label': 'organization'}, {'entity': 'informal social network', 'label': 'organization'}, {'entity': 'radio', 'label': 'method'}, {'entity': 'social exclusion', 'label': 'event'}, {'entity': 'global mobility', 'label': 'environment'}, {'entity': 'globalisation of education', 'label': 'event'}, {'entity': 'transworld textbooks', 'label': 'publication'}, {'entity': 'empirical data', 'label': 'measurement'}, {'entity': 'literacy', 'label': 'method'}, {'entity': 'pen and paper', 'label': 'tool'}, {'entity': 'film', 'label': 'media'}, {'entity': 'Intergovernmental Panel on Climate Change', 'label': 'organization'}, {'entity': 'ecosystem functioning', 'label': 'environment'}, {'entity': 'ecosystem function', 'label': 'function'}, {'entity': 'The most important relationships', 'label': 'event'}, {'entity': 'health care policy', 'label': 'treatment'}, {'entity': 'neoliberal', 'label': 'concept'}, {'entity': 'privatisation of health services', 'label': 'event'}, {'entity': 'illegal trading of drugs', 'label': 'drug'}, {'entity': 'developing', 'label': 'country'}, {'entity': 'emotional', 'label': 'immune response'}, {'entity': 'social exclusion', 'label': 'event'}, {'entity': 'violence', 'label': 'event'}, {'entity': 'treaties', 'label': 'treatment'}, {'entity': 'social exclusion', 'label': 'vaccine'}, {'entity': 'lifestyle', 'label': 'disease'}, {'entity': 'product marketing', 'label': 'method'}, {'entity': 'marketing', 'label': 'organization'}, {'entity': 'cigarette smuggling', 'label': 'event'}, {'entity': 'Illegal trade', 'label': 'disease'}, {'entity': 'excise', 'label': 'vaccine'}, {'entity': 'spread of infectious diseases', 'label': 'disease'}, {'entity': 'goods', 'label': 'transmission'}, {'entity': 'global alerts', 'label': 'diagnostic'}, {'entity': 'subsidise', 'label': 'method'}, {'entity': 'World Health Report 2001', 'label': 'publication'}, {'entity': 'production', 'label': 'method'}, {'entity': 'food consumption', 'label': 'measurement'}, {'entity': 'Accelerated economic growth', 'label': 'event'}, {'entity': 'Lang', 'label': 'publication'}, {'entity': 'shocks', 'label': 'event'}, {'entity': 'food insecure', 'label': 'condition'}, {'entity': 'global level', 'label': 'region'}, {'entity': 'demand', 'label': 'measurement'}, {'entity': 'technologies', 'label': 'method'}, {'entity': 'threatened', 'label': 'environment'}, {'entity': 'sub-Saharan', 'label': 'region'}, {'entity': 'effects of globalisation', 'label': 'event'}, {'entity': 'privatisation', 'label': 'organization'}, {'entity': 'privatisation', 'label': 'organization'}, {'entity': 'inequalities in access', 'label': 'disease'}, {'entity': 'virtual water', 'label': 'measurement'}, {'entity': 'global trade', 'label': 'event'}, {'entity': 'technologies', 'label': 'method'}, {'entity': 'intergenerational equity', 'label': 'concept'}, {'entity': 'health impacts', 'label': 'disease'}, {'entity': 'think-model', 'label': 'method'}, {'entity': 'health impacts of different globalisation pathways', 'label': 'event'}, {'entity': 'We claim', 'label': 'organization'}, {'entity': 'holistic approach towards globalisation', 'label': 'conceptual framework'}, {'entity': 'holistic approach', 'label': 'method'}, {'entity': 'institutional', 'label': 'organization'}, {'entity': 'Another influenza pandemic', 'label': 'event'}, {'entity': 'all but inevitable', 'label': 'event'}, {'entity': 'first pandemic wave', 'label': 'event'}, {'entity': 'next influenza pandemic', 'label': 'event'}, {'entity': 'effective planning', 'label': 'treatment'}, {'entity': 'population to general practitioner', 'label': 'measurement'}, {'entity': 'population health outcomes', 'label': 'epidemiology'}, {'entity': 'workday', 'label': 'measurement'}, {'entity': 'high-risk', 'label': 'virus'}, {'entity': 'work', 'label': 'measurement'}, {'entity': 'general practice', 'label': 'organization'}, {'entity': 'stochastically simulated', 'label': 'method'}, {'entity': 'lost workdays', 'label': 'measurement'}, {'entity': 'peak pandemic', 'label': 'event'}, {'entity': 'peak week', 'label': 'event'}, {'entity': '1918 pandemic in New Zealand', 'label': 'event'}, {'entity': 'uncomplicated', 'label': 'disease'}, {'entity': 'more severe', 'label': 'event'}, {'entity': 'mask', 'label': 'vaccine'}, {'entity': 'national level', 'label': 'location'}, {'entity': 'SARS preparedness and response plan', 'label': 'clinical trial'}, {'entity': 'CDC website', 'label': 'organization'}, {'entity': 'community-governed', 'label': 'organization'}, {'entity': 'medical workforce in New Zealand', 'label': 'organization'}, {'entity': 'uncertainties associated with pandemic influenza', 'label': 'epidemiology'}, {'entity': 'health care worker', 'label': 'person'}, {'entity': 'professional obligations', 'label': 'concept'}, {'entity': 'care for sick relatives', 'label': 'activity'}, {'entity': 'particularly virulent strains', 'label': 'virus'}, {'entity': 'fear of infection', 'label': 'symptom'}, {'entity': 'beginning battle', 'label': 'event'}, {'entity': 'alliance', 'label': 'host'}, {'entity': 'redundancy', 'label': 'virus'}, {'entity': 'invading virus', 'label': 'virus'}, {'entity': 'weapons', 'label': 'method'}, {'entity': 'conflict', 'label': 'virus'}, {'entity': 'opening battle', 'label': 'event'}, {'entity': 'latent state', 'label': 'virus'}, {'entity': 'adapted to the human host and environment', 'label': 'host'}, {'entity': 'mucosal', 'label': 'symptom'}, {'entity': 'colinear', 'label': 'gene'}, {'entity': 'unglycosylated', 'label': 'protein'}, {'entity': 'viral DNA', 'label': 'gene'}, {'entity': 'laminas', 'label': 'location'}, {'entity': '41', 'label': 'publication'}, {'entity': 'HSV composition and entry', 'label': 'publication'}, {'entity': 'fusion of membranes', 'label': 'process'}, {'entity': 'proteasome', 'label': 'protein'}, {'entity': 'nuclear membrane', 'label': 'protein'}, {'entity': 'posttranscriptional', 'label': 'protein'}, {'entity': 'inner nuclear membrane', 'label': 'structure'}, {'entity': 'viral egress', 'label': 'treatment'}, {'entity': 'de-envelopment', 'label': 'virus'}, {'entity': '1967', 'label': 'year'}, {'entity': 'ethnic', 'label': 'country'}, {'entity': 'improved socioeconomic conditions', 'label': 'environment'}, {'entity': 'mental', 'label': 'symptom'}, {'entity': 'high-dose acyclovir', 'label': 'treatment'}, {'entity': 'high-dose acyclovir', 'label': 'drug'}, {'entity': 'HSV stromal keratitis', 'label': 'disease'}, {'entity': 'molecular mimicry', 'label': 'immune response'}, {'entity': 'dissemination of the infection', 'label': 'disease'}, {'entity': 'proliferation', 'label': 'process'}, {'entity': 'monocytes', 'label': 'protein'}, {'entity': 'leukocytes', 'label': 'cell'}, {'entity': 'molecular level', 'label': 'measurement'}, {'entity': 'numerical nomenclature of interleukin', 'label': 'method'}, {'entity': 'functional anchorage', 'label': 'measurement'}, {'entity': 'prototypes', 'label': 'protein'}, {'entity': 'kinases', 'label': 'protein'}, {'entity': 'IFN-ξ/limitin', 'label': 'protein'}, {'entity': 'allelic', 'label': 'gene'}, {'entity': 'functional', 'label': 'measurement'}, {'entity': 'IFN-ξ/limitin', 'label': 'protein'}, {'entity': 'macrophage activating factor', 'label': 'protein'}, {'entity': 'macro-phage', 'label': 'protein'}, {'entity': 'knock out', 'label': 'method'}, {'entity': 'LIGHT', 'label': 'protein'}, {'entity': 'post translational modification', 'label': 'protein'}, {'entity': 'phagocyte', 'label': 'cell type'}, {'entity': 'homotrimers', 'label': 'protein'}, {'entity': 'life', 'label': 'immune response'}, {'entity': 'extracellular domain', 'label': 'protein'}, {'entity': 'chromosomes 3', 'label': 'gene'}, {'entity': 'dif-ferent kinetics', 'label': 'measurement'}, {'entity': 'IL-12 family', 'label': 'protein'}, {'entity': 'B cell growth factor', 'label': 'protein'}, {'entity': 'course of infection', 'label': 'disease'}, {'entity': 'in vivo infection', 'label': 'clinical trial'}, {'entity': \"2'-5'-oligoadenylate synthetase\", 'label': 'protein'}, {'entity': 'regulatory signals', 'label': 'immune response'}, {'entity': 'IFN-α/β system', 'label': 'protein'}, {'entity': 'co-dominant', 'label': 'gene'}, {'entity': 'IFN-activated OAS', 'label': 'protein'}, {'entity': 'inbred', 'label': 'organization'}, {'entity': 'HSV vhs-protein', 'label': 'protein'}, {'entity': 'IFN-α/β system', 'label': 'protein'}, {'entity': 'intrinsic', 'label': 'virus'}, {'entity': 'phagolysosome', 'label': 'pathway'}, {'entity': 'local infection', 'label': 'disease'}, {'entity': 'first replication cycle', 'label': 'virus'}, {'entity': 'effects', 'label': 'symptom'}, {'entity': 'positive feed-back mechanisms', 'label': 'pathway'}, {'entity': '3', 'label': 'figure'}, {'entity': 'KO', 'label': 'gene'}, {'entity': 'local titres', 'label': 'measurement'}, {'entity': 'guanidino nitrogen', 'label': 'protein'}, {'entity': 'second wave', 'label': 'treatment'}, {'entity': 'antiviral system', 'label': 'virus'}, {'entity': 'influenza virus infections', 'label': 'disease'}, {'entity': 'closed down', 'label': 'treatment'}, {'entity': 'peritoneal', 'label': 'location'}, {'entity': 'DNAbinding', 'label': 'gene'}, {'entity': 'stimulators', 'label': 'virus'}, {'entity': 'generalized', 'label': 'disease'}, {'entity': 'non-specific antiviral defence', 'label': 'immune response'}, {'entity': 'molecular signalling', 'label': 'method'}, {'entity': 'inhibitors of TNF', 'label': 'drug'}, {'entity': 'no competing interests', 'label': 'organization'}, {'entity': 'snowball', 'label': 'method'}, {'entity': 'international spread', 'label': 'transmission'}, {'entity': 'cross border', 'label': 'region'}, {'entity': 'HIV among injecting drug users', 'label': 'disease'}, {'entity': 'IDU', 'label': 'person'}, {'entity': 'subtypes', 'label': 'virus'}, {'entity': 'HIV prevention efforts', 'label': 'treatment'}, {'entity': 'HIV transmission', 'label': 'transmission'}, {'entity': '1996', 'label': 'year'}, {'entity': 'commercial goods', 'label': 'measurement'}, {'entity': 'informal crossing', 'label': 'pathway'}, {'entity': 'village', 'label': 'location'}, {'entity': 'town', 'label': 'location'}, {'entity': 'Drug dealer', 'label': 'person'}, {'entity': 'sell drugs', 'label': 'drug'}, {'entity': 'outbreak of severe acute respiratory syndrome', 'label': 'disease'}, {'entity': 'porters', 'label': 'occupation'}, {'entity': 'intermarriage', 'label': 'event'}, {'entity': 'snowball', 'label': 'method'}, {'entity': 'project center', 'label': 'location'}, {'entity': 'yuan', 'label': 'measurement'}, {'entity': 'project sites', 'label': 'location'}, {'entity': 'shooting places', 'label': 'location'}, {'entity': 'clusters', 'label': 'organization'}, {'entity': 'shooting places', 'label': 'location'}, {'entity': 'Vietnamese dong', 'label': 'measurement'}, {'entity': 'National AIDS Standing Bureau', 'label': 'organization'}, {'entity': 'signed informed consent', 'label': 'clinical trial'}, {'entity': 'project participation card', 'label': 'object'}, {'entity': 'Drug Injection Study, Phase II', 'label': 'clinical trial'}, {'entity': 'trained interviewers', 'label': 'organization'}, {'entity': 'distance to the border', 'label': 'measurement'}, {'entity': 'phlebotomist', 'label': 'person'}, {'entity': 'identification number', 'label': 'method'}, {'entity': 'labeled', 'label': 'vocabulary'}, {'entity': 'Guangxi Center for HIV/AIDS Prevention and Control', 'label': 'organization'}, {'entity': 'Guangxi Center for HIV/AIDS Prevention and Control', 'label': 'organization'}, {'entity': '1822 election', 'label': 'event'}, {'entity': 'pattern', 'label': 'clinical trial'}, {'entity': 'crackdowns', 'label': 'event'}, {'entity': 'family ties', 'label': 'host'}, {'entity': 'syringe', 'label': 'vaccine'}, {'entity': 'border crossing', 'label': 'location'}, {'entity': 'ethnic minority status', 'label': 'person'}, {'entity': 'drug distribution', 'label': 'treatment'}, {'entity': 'syringe', 'label': 'vaccine'}, {'entity': 'syringes', 'label': 'vaccine'}, {'entity': 'syringe', 'label': 'vaccine'}, {'entity': 'international transmission', 'label': 'transmission'}, {'entity': 'admissible', 'label': 'clinical trial'}, {'entity': 'SARS CoV infection', 'label': 'disease'}, {'entity': 'envelope spike', 'label': 'protein'}, {'entity': 'in vivo inhibitors of the virus', 'label': 'drug'}, {'entity': 'Golgi vesicles', 'label': 'protein'}, {'entity': 'independent', 'label': 'clinical trial'}, {'entity': 'endosomemediated virus', 'label': 'virus'}, {'entity': '5 mM', 'label': 'measurement'}, {'entity': 'SARS CoV infection', 'label': 'disease'}, {'entity': '5A', 'label': 'figure'}, {'entity': 'Golgimodified', 'label': 'protein'}, {'entity': 'SARS-specific polyclonal antibody', 'label': 'protein'}, {'entity': 'previous analysis', 'label': 'publication'}, {'entity': 'SARS-CoV spike protein', 'label': 'protein'}, {'entity': 'chased', 'label': 'method'}, {'entity': '3', 'label': 'measurement'}, {'entity': 'protein precursor', 'label': 'protein'}, {'entity': 'Golgi form', 'label': 'protein'}, {'entity': 'lanes 3', 'label': 'measurement'}, {'entity': 'terminal', 'label': 'protein'}, {'entity': 'mechanism(s) of action', 'label': 'method'}, {'entity': 'SARS-CoV replication', 'label': 'virus'}, {'entity': 'SARS-CoV spike protein', 'label': 'protein'}, {'entity': 'anti-mouse fluorescein-coupled antibody', 'label': 'protein'}, {'entity': 'RIPA', 'label': 'method'}, {'entity': 'Eagle medium', 'label': 'measurement'}, {'entity': 'swine anti-goat IgG antibody', 'label': 'protein'}, {'entity': 'FAC-SCalibur', 'label': 'equipment'}, {'entity': 'RIPA', 'label': 'method'}, {'entity': 'no competing interests', 'label': 'publication'}, {'entity': 'spike envelope', 'label': 'protein'}, {'entity': 'periphery', 'label': 'protein'}, {'entity': 'structural similarity', 'label': 'treatment'}, {'entity': 'SCV S', 'label': 'protein'}, {'entity': 'sites', 'label': 'measurement'}, {'entity': 'Figure 1 Expression and binding of soluble S fragments containing the RBD', 'label': 'publication'}, {'entity': 'anti-c-myc epitope', 'label': 'protein'}, {'entity': 'S272-537', 'label': 'protein'}, {'entity': 'residues 327 and 490', 'label': 'protein'}, {'entity': \"Lee and Richards' algorithm\", 'label': 'method'}, {'entity': 'Glycosylation of S fragment containing the RBD Figure 2 Glycosylation of S fragment containing the RBD', 'label': 'protein'}, {'entity': 'Effects of glycosylation on expression and binding of RBD-containing fragments Figure 3', 'label': 'measurement'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'non-redundant', 'label': 'vocabulary'}, {'entity': 'SCV', 'label': 'virus'}, {'entity': 'InsightII', 'label': 'software'}, {'entity': 'Connolly molecular surface', 'label': 'protein'}, {'entity': 'sites', 'label': 'protein'}, {'entity': 'yellow color', 'label': 'color'}, {'entity': '330', 'label': 'measurement'}, {'entity': '357', 'label': 'gene'}, {'entity': 'sites', 'label': 'protein'}, {'entity': 'Figure 6 Mapping of the S RBD mutants on the structure', 'label': 'method'}, {'entity': 'solid', 'label': 'measurement'}, {'entity': 'transparency', 'label': 'measurement'}, {'entity': 'homology model', 'label': 'method'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'three-dimensional', 'label': 'measurement'}, {'entity': '3D model', 'label': 'method'}, {'entity': 'inter-connecting loop', 'label': 'protein'}, {'entity': 'polar', 'label': 'protein'}, {'entity': 'dissociation rate constant', 'label': 'measurement'}, {'entity': '330', 'label': 'protein'}, {'entity': '6', 'label': 'number'}, {'entity': 'sites', 'label': 'protein'}, {'entity': 'oligomeric', 'label': 'protein'}, {'entity': 'Polyfect transfection kit', 'label': 'organization'}, {'entity': 'anti-c-Myc epitope', 'label': 'protein'}, {'entity': 'anti-His tag', 'label': 'protein'}, {'entity': 'anti-c-Myc epitope', 'label': 'protein'}, {'entity': 'anti-c-myc epitope', 'label': 'protein'}, {'entity': 'RT', 'label': 'measurement'}, {'entity': 'no competing interests', 'label': 'organization'}, {'entity': 'microbial communities', 'label': 'organism'}, {'entity': 'shotgun expression libraries', 'label': 'method'}, {'entity': 'viral pathogen', 'label': 'virus'}, {'entity': 'closely related', 'label': 'vocabulary'}, {'entity': 'New analytical tools', 'label': 'method'}, {'entity': 'hybridized', 'label': 'diagnostic'}, {'entity': 'nearest neighbor method', 'label': 'method'}, {'entity': 'BLAST alignment', 'label': 'method'}, {'entity': 'raw similarity scores', 'label': 'measurement'}, {'entity': 'null probability distribution', 'label': 'measurement'}, {'entity': 'theoretical hybridization energy profiles', 'label': 'measurement'}, {'entity': 'Theoretical hybridization energy profiles', 'label': 'method'}, {'entity': 'row', 'label': 'method'}, {'entity': 'nor-malization', 'label': 'method'}, {'entity': 'direct fluorescent antibody', 'label': 'method'}, {'entity': 'unit-vector', 'label': 'method'}, {'entity': 'uncentered Pearson correlation', 'label': 'method'}, {'entity': 'energy vector normalization', 'label': 'method'}, {'entity': 'intrafamily separation', 'label': 'measurement'}, {'entity': 'best scoring profiles', 'label': 'virus'}, {'entity': 'virus profile', 'label': 'vaccine'}, {'entity': 'sample diversity', 'label': 'measurement'}, {'entity': 'normally distributed', 'label': 'measurement'}, {'entity': 'tails', 'label': 'measurement'}, {'entity': 'null hypothesis', 'label': 'diagnostic'}, {'entity': 'RSV-positive samples', 'label': 'disease'}, {'entity': 'red circles', 'label': 'diagnostic'}, {'entity': 'Example 3', 'label': 'event'}, {'entity': 'viral etiologies', 'label': 'disease'}, {'entity': 'GEO database', 'label': 'organization'}, {'entity': 'GEO database', 'label': 'organization'}, {'entity': 'P < 0.01', 'label': 'measurement'}, {'entity': '2003 outbreak of SARS', 'label': 'event'}, {'entity': 'alternative oligonucleotide', 'label': 'virus'}, {'entity': 'Additional data file', 'label': 'publication'}, {'entity': 'GEO database', 'label': 'organization'}, {'entity': 'clinical sample', 'label': 'sample'}, {'entity': 'E-Predict similarity scores', 'label': 'method'}, {'entity': 'user bias', 'label': 'immune response'}, {'entity': 'negatives', 'label': 'measurement'}, {'entity': 'positives', 'label': 'measurement'}, {'entity': 'experimental', 'label': 'method'}, {'entity': 'score distribution', 'label': 'measurement'}, {'entity': 'significance', 'label': 'measurement'}, {'entity': 'P value estimation', 'label': 'method'}, {'entity': 'node', 'label': 'organization'}, {'entity': 'misleading predictions', 'label': 'detection'}, {'entity': 'first iteration', 'label': 'clinical trial'}, {'entity': 'null probability distribution', 'label': 'method'}, {'entity': 'two iterations', 'label': 'measurement'}, {'entity': 'noniterative', 'label': 'method'}, {'entity': 'detection of genes', 'label': 'gene'}, {'entity': 'Eagle medium', 'label': 'measurement'}, {'entity': 'Carlsbad, CA', 'label': 'location'}, {'entity': 'Qiagen USA', 'label': 'organization'}, {'entity': 'viral culture', 'label': 'disease'}, {'entity': 'Fermentas USA', 'label': 'organization'}, {'entity': 'Austin, TX', 'label': 'location'}, {'entity': 'previously described', 'label': 'method'}, {'entity': 'GEO database', 'label': 'organization'}, {'entity': 'poly-lysine', 'label': 'method'}, {'entity': 'gridding', 'label': 'method'}, {'entity': 'Round A-B random PCR', 'label': 'method'}, {'entity': 'second strand synthesis', 'label': 'method'}, {'entity': 'primer B', 'label': 'method'}, {'entity': 'gridding', 'label': 'method'}, {'entity': 'Union City, CA', 'label': 'location'}, {'entity': 'GEO database', 'label': 'organization'}, {'entity': 'intensi-ties', 'label': 'measurement'}, {'entity': 'Additional data file', 'label': 'publication'}, {'entity': 'Control oligonucleotide', 'label': 'virus'}, {'entity': 'sum-normalized intensity', 'label': 'measurement'}, {'entity': 'array intensities', 'label': 'measurement'}, {'entity': 'unit-vector', 'label': 'method'}, {'entity': 'uncentered Pearson correlation', 'label': 'method'}, {'entity': 'uncentered Pearson correlation', 'label': 'method'}, {'entity': 'log-transformed', 'label': 'method'}, {'entity': 'Shapiro-Wilk normality test', 'label': 'method'}, {'entity': 'virus profiles', 'label': 'virus'}, {'entity': 'significance', 'label': 'measurement'}, {'entity': 'first iteration', 'label': 'method'}, {'entity': 'second iteration', 'label': 'method'}, {'entity': 'uncentered Pearson correlation', 'label': 'method'}, {'entity': 'TreeView', 'label': 'method'}, {'entity': 'Microsoft', 'label': 'organization'}, {'entity': 'Microsoft', 'label': 'organization'}, {'entity': 'Google Maps API', 'label': 'organization'}, {'entity': 'copyright', 'label': 'vaccine'}, {'entity': 'London July 2005 Terrorist Attacks', 'label': 'event'}, {'entity': 'altitude', 'label': 'measurement'}, {'entity': 'MultiMap', 'label': 'organization'}, {'entity': 'A9 Block View', 'label': 'tool'}, {'entity': 'aerial photos', 'label': 'measurement'}, {'entity': 'Microsoft Virtual Earth', 'label': 'organization'}, {'entity': 'jibble', 'label': 'organization'}, {'entity': 'worldKit', 'label': 'organization'}, {'entity': 'Google Maps API', 'label': 'organization'}, {'entity': 'Google Maps API', 'label': 'organization'}, {'entity': 'API key', 'label': 'organization'}, {'entity': 'SHA Locator', 'label': 'organization'}, {'entity': 'SHA Locator', 'label': 'organization'}, {'entity': 'tooltips', 'label': 'method'}, {'entity': 'Google Earth KML', 'label': 'organization'}, {'entity': 'Internet Explorer', 'label': 'organization'}, {'entity': 'Virtual Earth Map Control', 'label': 'tool'}, {'entity': 'Virtual Earth Map Control', 'label': 'organization'}, {'entity': 'Web portal', 'label': 'organization'}, {'entity': 'Google Maps API', 'label': 'organization'}, {'entity': 'Web application', 'label': 'clinical trial'}, {'entity': 'SHA Locator', 'label': 'organization'}, {'entity': 'lower pane', 'label': 'location'}, {'entity': 'Wikipedia -The Free Encyclopedia', 'label': 'organization'}, {'entity': 'mapwikis', 'label': 'publication'}, {'entity': 'worldKit GeoWiki', 'label': 'organization'}, {'entity': 'Tate Museum', 'label': 'organization'}, {'entity': 'Internet GIS', 'label': 'technology'}, {'entity': \"Megan's Law\", 'label': 'organization'}, {'entity': 'SARS (Severe Acute Respiratory Syndrome) mapping in Hong Kong in 2003', 'label': 'disease'}, {'entity': 'September 2001 terrorist events', 'label': 'event'}, {'entity': 'protection of information', 'label': 'concept'}, {'entity': 'map data privacy and confidentiality', 'label': 'measurement'}, {'entity': 'copyright minefield', 'label': 'vocabulary'}, {'entity': 'mobile phones', 'label': 'device'}, {'entity': 'Web map', 'label': 'organization'}, {'entity': 'MSN Virtual Earth Map Control', 'label': 'organization'}, {'entity': 'Geospatial Digital Rights Management', 'label': 'organization'}, {'entity': 'aracti', 'label': 'organization'}, {'entity': 'geospatial data providers', 'label': 'organization'}]\n",
      "Object Labels Dicts: [{'entity': 'detecting aerosolized rhinovirus', 'label': 'method'}, {'entity': 'Valencia, California', 'label': 'location'}, {'entity': 'aerosolized rhinovirus', 'label': 'virus'}, {'entity': 'airborne viruses', 'label': 'virus'}, {'entity': 'Airborne transmission of infection', 'label': 'transmission'}, {'entity': 'laboratory studies', 'label': 'clinical trial'}, {'entity': 'highly contagious', 'label': 'virus'}, {'entity': 'Waltham, MA', 'label': 'location'}, {'entity': 'tracer', 'label': 'measurement'}, {'entity': 'Memphis, Tennessee', 'label': 'location'}, {'entity': 'Ann Arbor, Michigan', 'label': 'location'}, {'entity': 'Makeup air', 'label': 'environment'}, {'entity': 'Filtered air', 'label': 'measurement'}, {'entity': 'Bourne, Massachusetts', 'label': 'location'}, {'entity': 'air sample', 'label': 'sample'}, {'entity': '60', 'label': 'measurement'}, {'entity': 'Thorofare, New Jersey', 'label': 'location'}, {'entity': 'pipeted', 'label': 'method'}, {'entity': 'uranine analysis', 'label': 'method'}, {'entity': 'nebulizing fluid', 'label': 'sample'}, {'entity': 'spiking', 'label': 'method'}, {'entity': '50', 'label': 'measurement'}, {'entity': '%RH', 'label': 'measurement'}, {'entity': 'spiking', 'label': 'method'}, {'entity': 'expelled', 'label': 'virus'}, {'entity': '1966', 'label': 'publication'}, {'entity': 'ID 50', 'label': 'measurement'}, {'entity': '1966', 'label': 'year'}, {'entity': 'impingers', 'label': 'diagnostic'}, {'entity': 'nonexposed aerosols', 'label': 'disease'}, {'entity': 'UV irradiated aerosols', 'label': 'detection'}, {'entity': 'human population migrations', 'label': 'event'}, {'entity': 'evolutionarily significant', 'label': 'vocabulary'}, {'entity': 'limb', 'label': 'anatomy'}, {'entity': 'vaccine research', 'label': 'clinical trial'}, {'entity': 'evolutionary history', 'label': 'epidemiology'}, {'entity': 'If', 'label': 'virus'}, {'entity': 'subpopulation', 'label': 'virus'}, {'entity': 'human populations', 'label': 'host'}, {'entity': 'evolutionary relationships', 'label': 'pathway'}, {'entity': 'lineage', 'label': 'gene'}, {'entity': 'migration', 'label': 'event'}, {'entity': 'divergence', 'label': 'measurement'}, {'entity': 'distributions', 'label': 'measurement'}, {'entity': 'evolutionary history', 'label': 'pathway'}, {'entity': 'human pathogens', 'label': 'virus'}, {'entity': 'mitochondrial DNA evolution', 'label': 'transmission'}, {'entity': 'adapted from', 'label': 'method'}, {'entity': 'Taiwan epidemic', 'label': 'event'}, {'entity': 'control set', 'label': 'method'}, {'entity': 'SARS coronavirus infection', 'label': 'disease'}, {'entity': 'Taiwanese general population', 'label': 'population'}, {'entity': 'Taiwan indigenous peoples', 'label': 'host'}, {'entity': 'patient group', 'label': 'clinical trial'}, {'entity': 'Taiwan was stroked', 'label': 'region'}, {'entity': 'SARS panic', 'label': 'event'}, {'entity': '24 hours later', 'label': 'measurement'}, {'entity': 'HLA-class II system', 'label': 'gene'}, {'entity': 'disease susceptibility', 'label': 'virus'}, {'entity': 'fatal cases', 'label': 'disease'}, {'entity': 'Control A Blood', 'label': 'sample'}, {'entity': 'health care workers', 'label': 'person'}, {'entity': 'April 2003', 'label': 'event'}, {'entity': 'health care workers', 'label': 'person'}, {'entity': 'serological methods', 'label': 'method'}, {'entity': 'allele frequencies', 'label': 'measurement'}, {'entity': 'allele frequencies', 'label': 'measurement'}, {'entity': 'bias', 'label': 'measurement'}, {'entity': 'Singapore Chinese', 'label': 'person'}, {'entity': 'Thai Chinese', 'label': 'person'}, {'entity': 'Taiwan indigenous peoples', 'label': 'person'}, {'entity': 'SAR Hong Kong', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'Mainland China', 'label': 'location'}, {'entity': 'entry', 'label': 'event'}, {'entity': 'class', 'label': 'definition'}, {'entity': 'ordinary differential equations', 'label': 'method'}, {'entity': 'propagation of the disease', 'label': 'disease'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'final value', 'label': 'measurement'}, {'entity': 'initial value', 'label': 'measurement'}, {'entity': 'exponentially slow', 'label': 'measurement'}, {'entity': 'function', 'label': 'method'}, {'entity': 'graph of the function I', 'label': 'diagram'}, {'entity': 'Typical dynamics for the SIR model', 'label': 'diagram'}, {'entity': 'class', 'label': 'treatment'}, {'entity': 'R ∞', 'label': 'measurement'}, {'entity': 'classes', 'label': 'pathway'}, {'entity': 'classes', 'label': 'pathway'}, {'entity': 'classes', 'label': 'pathway'}, {'entity': 'classes', 'label': 'pathway'}, {'entity': 'S(0)', 'label': 'measurement'}, {'entity': 'errors', 'label': 'measurement'}, {'entity': '17', 'label': 'publication'}, {'entity': 'differential equation', 'label': 'method'}, {'entity': 'Comparison of the SEIR (red) and SIR (green) models', 'label': 'diagram'}, {'entity': 'disease A', 'label': 'disease'}, {'entity': 'class R P', 'label': 'clinical trial'}, {'entity': 'class I P', 'label': 'clinical trial'}, {'entity': 'disease A', 'label': 'disease'}, {'entity': 'disease B', 'label': 'disease'}, {'entity': 'class I', 'label': 'clinical trial'}, {'entity': 'time unit', 'label': 'measurement'}, {'entity': 'positive number', 'label': 'measurement'}, {'entity': 'time unit', 'label': 'measurement'}, {'entity': 'positive number', 'label': 'measurement'}, {'entity': 'ordinary differential equations', 'label': 'method'}, {'entity': 'constant', 'label': 'measurement'}, {'entity': 'transmit infection', 'label': 'transmission'}, {'entity': 'population size', 'label': 'measurement'}, {'entity': 'epidemic A', 'label': 'disease'}, {'entity': 'class', 'label': 'vocabulary'}, {'entity': 'protection assumption', 'label': 'method'}, {'entity': 'Hong-Kong simulations', 'label': 'clinical trial'}, {'entity': 'outbreak of SARS in Hong Kong', 'label': 'event'}, {'entity': 'disease B', 'label': 'disease'}, {'entity': '2003', 'label': 'year'}, {'entity': 'limit', 'label': 'measurement'}, {'entity': 'sum', 'label': 'measurement'}, {'entity': '10', 'label': 'measurement'}, {'entity': 'E(0)', 'label': 'measurement'}, {'entity': '17 March, 2003', 'label': 'date'}, {'entity': '10', 'label': 'measurement'}, {'entity': 'initial population', 'label': 'measurement'}, {'entity': 'daily new case', 'label': 'measurement'}, {'entity': 'I P', 'label': 'parameter'}, {'entity': 'class I', 'label': 'disease'}, {'entity': 'class E', 'label': 'diagnosis'}, {'entity': 'hypothesis', 'label': 'clinical trial'}, {'entity': 'onset to admission interval', 'label': 'measurement'}, {'entity': 'scenarios', 'label': 'event'}, {'entity': 'recently more or less submitted to a severe draught', 'label': 'environment'}, {'entity': 'next year', 'label': 'time'}, {'entity': 'next autumn', 'label': 'event'}, {'entity': 'phenomenological', 'label': 'method'}, {'entity': 'consequences', 'label': 'clinical trial'}, {'entity': 'planets', 'label': 'environment'}, {'entity': 'differential equation', 'label': 'method'}, {'entity': 'discrete model', 'label': 'method'}, {'entity': 'data set', 'label': 'measurement'}, {'entity': 'phenomenological', 'label': 'method'}, {'entity': '10', 'label': 'measurement'}, {'entity': 'validate', 'label': 'method'}, {'entity': 'time series', 'label': 'measurement'}, {'entity': 'dynamics', 'label': 'method'}, {'entity': 'peak', 'label': 'symptom'}, {'entity': 'Predicted cases', 'label': 'measurement'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'reality', 'label': 'epidemiology'}, {'entity': 'conformational change', 'label': 'protein'}, {'entity': 'Wheel projection of the N-HR', 'label': 'protein'}, {'entity': 'inhibitors of HIV-1 entry into cells', 'label': 'drug'}, {'entity': 'cryptographic device', 'label': 'device'}, {'entity': 'SARS-CoV sequence', 'label': 'virus'}, {'entity': 'SARS-CoV S2 protein', 'label': 'protein'}, {'entity': 'gi|30023954', 'label': 'protein'}, {'entity': 'Y.Y.', 'label': 'person'}, {'entity': 'relationship tree', 'label': 'treatment'}, {'entity': 'embedded tetra-nucleotide usage', 'label': 'method'}, {'entity': 'newly identified infectious disease', 'label': 'disease'}, {'entity': 'mutate', 'label': 'virus'}, {'entity': 'origin', 'label': 'virus'}, {'entity': 'dinucleotide UpA', 'label': 'gene'}, {'entity': 'evolutionary paths', 'label': 'pathway'}, {'entity': 'Relationship tree', 'label': 'method'}, {'entity': 'usage pattern', 'label': 'measurement'}, {'entity': 'negative strand', 'label': 'virus'}, {'entity': 'relationship tree', 'label': 'method'}, {'entity': 'word usage', 'label': 'method'}, {'entity': 'word usage', 'label': 'method'}, {'entity': '3-D plot', 'label': 'diagnostic'}, {'entity': 'order', 'label': 'method'}, {'entity': 'bottom figure', 'label': 'figure'}, {'entity': 'Technical Programming language', 'label': 'method'}, {'entity': '6', 'label': 'position'}, {'entity': 'cons', 'label': 'virus'}, {'entity': 'tetra-nucleotide usage patterns', 'label': 'measurement'}, {'entity': 'computer tractability', 'label': 'measurement'}, {'entity': 'word', 'label': 'method'}, {'entity': 'occurrences', 'label': 'measurement'}, {'entity': 'vector components', 'label': 'measurement'}, {'entity': 'bias', 'label': 'measurement'}, {'entity': 'model chromosome', 'label': 'vocabulary'}, {'entity': 'identity', 'label': 'measurement'}, {'entity': 'negative magnitude of r', 'label': 'measurement'}, {'entity': 'Positive magnitude of r', 'label': 'measurement'}, {'entity': 'virus Y', 'label': 'virus'}, {'entity': 'scatter-plot', 'label': 'method'}, {'entity': 'variables', 'label': 'measurement'}, {'entity': 'subsequent eigen-vectors', 'label': 'method'}, {'entity': 'first eigen-vector', 'label': 'measurement'}, {'entity': 'extracted', 'label': 'method'}, {'entity': 'Technical Programming Language', 'label': 'method'}, {'entity': 'molecular methods', 'label': 'method'}, {'entity': '1998', 'label': 'publication'}, {'entity': 'unknown', 'label': 'virus'}, {'entity': 'families', 'label': 'virus'}, {'entity': 'Durham, North Carolina', 'label': 'location'}, {'entity': 'Microarray hybridization and analysis', 'label': 'method'}, {'entity': '1500', 'label': 'measurement'}, {'entity': 'pocket', 'label': 'protein'}, {'entity': 'enzymes', 'label': 'protein'}, {'entity': 'inherited neurodegenerative disease', 'label': 'disease'}, {'entity': 'role', 'label': 'clinical trial'}, {'entity': 'enzymes', 'label': 'protein'}, {'entity': 'structural basis', 'label': 'diagnostic'}, {'entity': 'iden-tity', 'label': 'measurement'}, {'entity': 'identity', 'label': 'measurement'}, {'entity': 'nucleotide pair', 'label': 'gene'}, {'entity': 'active enzyme', 'label': 'protein'}, {'entity': 'vertebrates', 'label': 'organism'}, {'entity': 'r.m.', 'label': 'measurement'}, {'entity': 'Corrected gene sequence of the zebrafish CLN2', 'label': 'gene'}, {'entity': 'S1 pocket', 'label': 'protein'}, {'entity': 'Sequences of the catalytic domains of CLN2', 'label': 'gene'}, {'entity': 'magenta', 'label': 'color'}, {'entity': 'A homology-derived model of human CLN2', 'label': 'protein'}, {'entity': 'enzymes', 'label': 'gene'}, {'entity': 'enzymatic', 'label': 'protein'}, {'entity': 'enzymatic', 'label': 'protein'}, {'entity': 'defects', 'label': 'gene'}, {'entity': 'unibas', 'label': 'organization'}, {'entity': 'neuropeptide', 'label': 'protein'}, {'entity': 'enzymatic', 'label': 'protein'}, {'entity': 'computer program', 'label': 'method'}, {'entity': 'homology model', 'label': 'method'}, {'entity': 'functional region', 'label': 'protein'}, {'entity': 'energy minimization', 'label': 'method'}, {'entity': 'non-bonded interactions', 'label': 'measurement'}, {'entity': 'atomic-scale model', 'label': 'measurement'}, {'entity': 'computer program', 'label': 'method'}, {'entity': 'November 2002', 'label': 'event'}, {'entity': 'short-term', 'label': 'treatment'}, {'entity': 'low APIs', 'label': 'measurement'}, {'entity': 'emerging viral infectious disease', 'label': 'disease'}, {'entity': 'May', 'label': 'month'}, {'entity': 'linear model', 'label': 'method'}, {'entity': 'cut-off point', 'label': 'measurement'}, {'entity': 'low API', 'label': 'measurement'}, {'entity': 'high API', 'label': 'measurement'}, {'entity': '95% confidence intervals', 'label': 'measurement'}, {'entity': 'died from the disease', 'label': 'symptom'}, {'entity': '2003', 'label': 'year'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'low API', 'label': 'measurement'}, {'entity': 'Berkeley, California', 'label': 'location'}, {'entity': 'capital city', 'label': 'location'}, {'entity': 'diseases afflicting the human condition', 'label': 'disease'}, {'entity': 'March 2003', 'label': 'event'}, {'entity': 'restaurants', 'label': 'location'}, {'entity': 'frantic measures to contain it', 'label': 'treatment'}, {'entity': 'holidays', 'label': 'event'}, {'entity': '2003', 'label': 'year'}, {'entity': '1950', 'label': 'year'}, {'entity': 'dark', 'label': 'symptom'}, {'entity': 'Roche Diagnostics Penzberg', 'label': 'organization'}, {'entity': '2008', 'label': 'year'}, {'entity': 'sequence motif', 'label': 'gene'}, {'entity': 'regulatory role', 'label': 'protein'}, {'entity': 'structural motif', 'label': 'protein'}, {'entity': 'protein degradation', 'label': 'protein'}, {'entity': 'degradation', 'label': 'treatment'}, {'entity': 'nuclear factor kappa B precursor', 'label': 'protein'}, {'entity': 'sequence motif', 'label': 'protein'}, {'entity': 'protein degradation', 'label': 'protein'}, {'entity': 'cytoplasmic structure', 'label': 'protein'}, {'entity': 'protein(s)', 'label': 'protein'}, {'entity': 'protein pool', 'label': 'treatment'}, {'entity': '3', 'label': 'measurement'}, {'entity': '3', 'label': 'event'}, {'entity': 'pools', 'label': 'protein'}, {'entity': 'transcription/translation', 'label': 'method'}, {'entity': 'individual TNT reaction', 'label': 'method'}, {'entity': 'mixed TNT reaction', 'label': 'method'}, {'entity': 'categories', 'label': 'method'}, {'entity': 'chaperone protein', 'label': 'protein'}, {'entity': 'huntingtin(41)', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'protein'}, {'entity': '6', 'label': 'number'}, {'entity': 'binding site', 'label': 'protein'}, {'entity': 'in vitro ubiquitination', 'label': 'method'}, {'entity': '8', 'label': 'figure'}, {'entity': 'solving', 'label': 'method'}, {'entity': 'decision-making', 'label': 'clinical trial'}, {'entity': 'Triad', 'label': 'clinical trial'}, {'entity': 'evidence of no effect', 'label': 'disease'}, {'entity': 'busy work environment', 'label': 'environment'}, {'entity': 'government data', 'label': 'organization'}, {'entity': 'private sectors', 'label': 'organization'}, {'entity': 'repeated use', 'label': 'measurement'}, {'entity': 'surveillance issues', 'label': 'disease'}, {'entity': 'spatial information system', 'label': 'method'}, {'entity': 'organisational', 'label': 'organization'}, {'entity': 'discipline', 'label': 'organization'}, {'entity': 'interdependent', 'label': 'vocabulary'}, {'entity': 'GIS applications in health and healthcare', 'label': 'section'}, {'entity': 'data confidentiality', 'label': 'measurement'}, {'entity': 'environmental surveillance', 'label': 'environment'}, {'entity': 'Berkeley, California', 'label': 'city'}, {'entity': '2001', 'label': 'year'}, {'entity': 'paper-drawn', 'label': 'method'}, {'entity': 'spatial relationship', 'label': 'method'}, {'entity': 'records', 'label': 'measurement'}, {'entity': 'disease rates', 'label': 'measurement'}, {'entity': 'Disease incidence', 'label': 'disease'}, {'entity': 'numerical value', 'label': 'measurement'}, {'entity': 'choropleth map', 'label': 'diagram'}, {'entity': 'smaller ones', 'label': 'region'}, {'entity': 'filtered map', 'label': 'method'}, {'entity': 'spatial filter', 'label': 'method'}, {'entity': 'spatially', 'label': 'method'}, {'entity': 'point data', 'label': 'measurement'}, {'entity': 'circles', 'label': 'method'}, {'entity': 'isarithmic map', 'label': 'diagnostic'}, {'entity': 'continuous', 'label': 'measurement'}, {'entity': 'smoothed disease maps', 'label': 'diagnostic'}, {'entity': 'continuous map surface', 'label': 'measurement'}, {'entity': 'data structure', 'label': 'method'}, {'entity': 'covariates', 'label': 'measurement'}, {'entity': 'smoothed map', 'label': 'diagram'}, {'entity': 'mapped', 'label': 'method'}, {'entity': 'ESRI Arc-GIS', 'label': 'organization'}, {'entity': 'emerging', 'label': 'event'}, {'entity': 'global positioning system', 'label': 'technology'}, {'entity': 'digital imagery', 'label': 'measurement'}, {'entity': 'decisionmaking', 'label': 'method'}, {'entity': 'access', 'label': 'method'}, {'entity': 'telecommunications', 'label': 'organization'}, {'entity': 'electronic medical records', 'label': 'diagnostic'}, {'entity': 'transparency', 'label': 'public health'}, {'entity': 'healthcare delivery', 'label': 'clinical trial'}, {'entity': 'disease prevention', 'label': 'treatment'}, {'entity': 'forecasting spatio-temporal trends', 'label': 'method'}, {'entity': 'dynamic model', 'label': 'method'}, {'entity': 'migration', 'label': 'event'}, {'entity': 'presentation', 'label': 'method'}, {'entity': 'time frame', 'label': 'measurement'}, {'entity': '2004', 'label': 'year'}, {'entity': 'clinic', 'label': 'location'}, {'entity': 'spatially disaggregated data', 'label': 'measurement'}, {'entity': 'ED waiting time', 'label': 'symptom'}, {'entity': 'Patient Administration System', 'label': 'system'}, {'entity': 'healthcare planning', 'label': 'treatment'}, {'entity': '1998', 'label': 'publication'}, {'entity': 'service audit', 'label': 'clinical trial'}, {'entity': '1998', 'label': 'publication'}, {'entity': '1999', 'label': 'year'}, {'entity': '2003', 'label': 'year'}, {'entity': 'electronic patient record', 'label': 'clinical trial'}, {'entity': 'England and Wales', 'label': 'region'}, {'entity': 'spatial information', 'label': 'measurement'}, {'entity': '1990', 'label': 'year'}, {'entity': 'healthcare practitioners', 'label': 'person'}, {'entity': 'objectives', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'publication'}, {'entity': 'European Standard Population', 'label': 'organization'}, {'entity': 'operational', 'label': 'method'}, {'entity': 'geo-information infrastructure', 'label': 'organization'}, {'entity': 'recipes', 'label': 'publication'}, {'entity': 'legal framework', 'label': 'diagnostic'}, {'entity': 'partnerships', 'label': 'organization'}, {'entity': 'products', 'label': 'vaccine'}, {'entity': 'institutional framework', 'label': 'organization'}, {'entity': 'stakeholders', 'label': 'organization'}, {'entity': 'market', 'label': 'environment'}, {'entity': 'strategy', 'label': 'method'}, {'entity': 'benefits applications', 'label': 'clinical trial'}, {'entity': 'applications', 'label': 'clinical trial'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'catalogue', 'label': 'organization'}, {'entity': 'sharing', 'label': 'method'}, {'entity': 'technical resources', 'label': 'measurement'}, {'entity': '2000', 'label': 'measurement'}, {'entity': 'technical expertise', 'label': 'vaccine'}, {'entity': 'digital data management', 'label': 'method'}, {'entity': 'gathering', 'label': 'organization'}, {'entity': 'Policymakers', 'label': 'organization'}, {'entity': 'solution providers', 'label': 'organization'}, {'entity': 'personnel', 'label': 'organization'}, {'entity': 'specialties', 'label': 'organization'}, {'entity': 'upgrade', 'label': 'method'}, {'entity': 'Confidentiality issues', 'label': 'event'}, {'entity': '2010', 'label': 'year'}, {'entity': 'data standard', 'label': 'measurement'}, {'entity': 'redundancy', 'label': 'treatment'}, {'entity': '2002', 'label': 'event'}, {'entity': 'redistribution of data', 'label': 'measurement'}, {'entity': 'National data utilities/services', 'label': 'organization'}, {'entity': '2003', 'label': 'publication'}, {'entity': 'business benefits', 'label': 'measurement'}, {'entity': 'ethnographic', 'label': 'method'}, {'entity': 'conceptual data model', 'label': 'method'}, {'entity': 'population health status', 'label': 'epidemiology'}, {'entity': 'cartographic file', 'label': 'measurement'}, {'entity': 'cartographic', 'label': 'measurement'}, {'entity': 'data model', 'label': 'method'}, {'entity': 'descriptions', 'label': 'measurement'}, {'entity': 'Web accessible', 'label': 'method'}, {'entity': 'data standard', 'label': 'measurement'}, {'entity': 'acquiring data', 'label': 'method'}, {'entity': 'stewardship', 'label': 'vaccine'}, {'entity': 'data type', 'label': 'measurement'}, {'entity': 'development process', 'label': 'method'}, {'entity': 'software engineering', 'label': 'method'}, {'entity': 'graphical user interface', 'label': 'method'}, {'entity': 'artificial intelligence', 'label': 'method'}, {'entity': 'research process', 'label': 'clinical trial'}, {'entity': 'ambiguity', 'label': 'concept'}, {'entity': 'challenges', 'label': 'event'}, {'entity': 'resolve conflicts', 'label': 'method'}, {'entity': 'issues of power', 'label': 'event'}, {'entity': 'working relationship', 'label': 'treatment'}, {'entity': 'ambiguity', 'label': 'event'}, {'entity': 'ambiguity', 'label': 'vocabulary'}, {'entity': 'uncertainty', 'label': 'vocabulary'}, {'entity': 'geo-information service', 'label': 'organization'}, {'entity': 'information system', 'label': 'system'}, {'entity': 'Data model', 'label': 'method'}, {'entity': 'information systems', 'label': 'system'}, {'entity': 'technical', 'label': 'method'}, {'entity': 'environmental health hazard indicators', 'label': 'measurement'}, {'entity': 'terms', 'label': 'definition'}, {'entity': 'characteristics', 'label': 'measurement'}, {'entity': 'indicator', 'label': 'measurement'}, {'entity': 'patterns', 'label': 'diagnostic'}, {'entity': 'Unified Modelling Language', 'label': 'method'}, {'entity': 'Manitoba Centre for Health Policy', 'label': 'organization'}, {'entity': 'closed', 'label': 'organization'}, {'entity': 'open standard', 'label': 'measurement'}, {'entity': '1994', 'label': 'year'}, {'entity': 'eXtensible Markup Language', 'label': 'method'}, {'entity': 'XML encoding', 'label': 'method'}, {'entity': 'spatial metadata', 'label': 'measurement'}, {'entity': 'feature store', 'label': 'organization'}, {'entity': 'alternative', 'label': 'method'}, {'entity': 'OS', 'label': 'organization'}, {'entity': 'topographic identifier', 'label': 'measurement'}, {'entity': 'spatial information', 'label': 'measurement'}, {'entity': 'presentation', 'label': 'method'}, {'entity': 'geometry', 'label': 'measurement'}, {'entity': 'geographic data', 'label': 'measurement'}, {'entity': 'vector map', 'label': 'method'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'proprietary database', 'label': 'organization'}, {'entity': 'geometry', 'label': 'measurement'}, {'entity': 'geographic data', 'label': 'measurement'}, {'entity': 'Web browser', 'label': 'tool'}, {'entity': 'schema', 'label': 'publication'}, {'entity': 'International Classification of Diseases', 'label': 'publication'}, {'entity': 'US Department of Defense', 'label': 'organization'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'map services', 'label': 'organization'}, {'entity': 'query', 'label': 'method'}, {'entity': 'heterogeneous database', 'label': 'technology'}, {'entity': 'query', 'label': 'method'}, {'entity': 'distributed collaborative geoprocessing', 'label': 'method'}, {'entity': 'ESRI UK', 'label': 'organization'}, {'entity': 'terrorist', 'label': 'event'}, {'entity': 'developed ones', 'label': 'country'}, {'entity': 'data file', 'label': 'measurement'}, {'entity': 'embedded', 'label': 'method'}, {'entity': 'Microsoft Windows', 'label': 'organization'}, {'entity': 'workflow', 'label': 'method'}, {'entity': 'tools', 'label': 'vaccine'}, {'entity': 'learning', 'label': 'method'}, {'entity': 'graphical data', 'label': 'measurement'}, {'entity': 'visually impaired', 'label': 'person'}, {'entity': 'Securing', 'label': 'method'}, {'entity': 'information infrastructure', 'label': 'clinical trial'}, {'entity': 'privacy', 'label': 'public health'}, {'entity': 'issue', 'label': 'public health'}, {'entity': 'healthcare delivery', 'label': 'clinical trial'}, {'entity': 'address-level data', 'label': 'measurement'}, {'entity': 'locations', 'label': 'location'}, {'entity': 'groups', 'label': 'clinical trial'}, {'entity': 'confidentiality', 'label': 'measurement'}, {'entity': 'perturbation', 'label': 'method'}, {'entity': 'aggregation', 'label': 'method'}, {'entity': 'health record', 'label': 'diagnostic'}, {'entity': 'disclosure', 'label': 'event'}, {'entity': 'affine', 'label': 'method'}, {'entity': 'Armstrong et al', 'label': 'organization'}, {'entity': 'subdivision', 'label': 'location'}, {'entity': '2001', 'label': 'year'}, {'entity': 'Output Area', 'label': 'location'}, {'entity': 'public health data', 'label': 'public health'}, {'entity': 'Health System Resident Component', 'label': 'organization'}, {'entity': 'public health surveillance', 'label': 'public health'}, {'entity': 'data confidentiality', 'label': 'vulnerability'}, {'entity': 'eXtensible Access Control Markup Language', 'label': 'organization'}, {'entity': 'Internet music', 'label': 'vaccine'}, {'entity': 'DRM', 'label': 'organization'}, {'entity': 'privacy', 'label': 'virus'}, {'entity': '2001', 'label': 'year'}, {'entity': 'medical conditions', 'label': 'disease'}, {'entity': '1996', 'label': 'year'}, {'entity': 'data security', 'label': 'measurement'}, {'entity': 'bias', 'label': 'method'}, {'entity': 'disease rate', 'label': 'measurement'}, {'entity': 'small numbers', 'label': 'measurement'}, {'entity': 'apparent pattern', 'label': 'symptom'}, {'entity': 'objective mechanism', 'label': 'method'}, {'entity': 'surveillance systems', 'label': 'system'}, {'entity': 'health data', 'label': 'disease'}, {'entity': \"Rogerson's spatial pattern surveillance method\", 'label': 'method'}, {'entity': 'GeoVISTA', 'label': 'organization'}, {'entity': 'programming-free', 'label': 'method'}, {'entity': 'geocomputation', 'label': 'method'}, {'entity': 'statisticians', 'label': 'person'}, {'entity': 'aggregation bias', 'label': 'concept'}, {'entity': 'health outcome', 'label': 'disease'}, {'entity': 'datasets', 'label': 'measurement'}, {'entity': '1992', 'label': 'year'}, {'entity': 'significance', 'label': 'measurement'}, {'entity': 'Android', 'label': 'organization'}, {'entity': 'National Cancer Mortality Atlas', 'label': 'organization'}, {'entity': 'data errors', 'label': 'measurement'}, {'entity': 'malaria risk atlas', 'label': 'diagnostic'}, {'entity': 'maintain', 'label': 'method'}, {'entity': 'maintainer', 'label': 'organization'}, {'entity': 'concept', 'label': 'method'}, {'entity': 'spatial information infrastructure', 'label': 'system'}, {'entity': 'geographic location', 'label': 'location'}, {'entity': 'collaborative work', 'label': 'method'}, {'entity': 'indexing', 'label': 'method'}, {'entity': 'rights', 'label': 'vaccine'}, {'entity': 'US President', 'label': 'person'}, {'entity': '1998', 'label': 'year'}, {'entity': '1995', 'label': 'year'}, {'entity': 'Data Locator', 'label': 'organization'}, {'entity': 'Data Locator', 'label': 'organization'}, {'entity': 'query', 'label': 'clinical trial'}, {'entity': '2001', 'label': 'year'}, {'entity': 'JAVA', 'label': 'method'}, {'entity': 'UK Association for Geographic Information', 'label': 'organization'}, {'entity': 'GIgateway Data Locator', 'label': 'organization'}, {'entity': 'geographic information on their area', 'label': 'measurement'}, {'entity': 'Canadian Geospatial Data Infrastructure', 'label': 'organization'}, {'entity': 'sectors', 'label': 'organization'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'search', 'label': 'method'}, {'entity': 'base layer', 'label': 'location'}, {'entity': 'Framework', 'label': 'organization'}, {'entity': 'features', 'label': 'measurement'}, {'entity': 'user interface', 'label': 'method'}, {'entity': 'skills', 'label': 'measurement'}, {'entity': 'users', 'label': 'person'}, {'entity': 'implementations', 'label': 'organization'}, {'entity': 'bioterrorist attack', 'label': 'event'}, {'entity': 'public health surveillance', 'label': 'public health'}, {'entity': 'practitioners', 'label': 'person'}, {'entity': 'person identifier', 'label': 'measurement'}, {'entity': 'administrative tasks', 'label': 'measurement'}, {'entity': 'technologies', 'label': 'method'}, {'entity': 'US Federal Emergency Management Agency', 'label': 'organization'}, {'entity': 'outbreak response', 'label': 'event'}, {'entity': 'desktop', 'label': 'method'}, {'entity': 'building', 'label': 'location'}, {'entity': 'disease surveillance system', 'label': 'epidemiology'}, {'entity': 'healthcare professionals', 'label': 'organization'}, {'entity': 'incident tracking system', 'label': 'system'}, {'entity': 'Human intervention', 'label': 'method'}, {'entity': 'virtual private network', 'label': 'method'}, {'entity': 'statistical detection', 'label': 'method'}, {'entity': 'statistical detection', 'label': 'method'}, {'entity': 'OTC products', 'label': 'disease'}, {'entity': 'cash register', 'label': 'location'}, {'entity': 'public health surveillance', 'label': 'public health'}, {'entity': 'detect unusual sales patterns', 'label': 'method'}, {'entity': 'sales', 'label': 'measurement'}, {'entity': 'industry data systems', 'label': 'organization'}, {'entity': 'infectious disease clusters', 'label': 'disease'}, {'entity': 'user interface', 'label': 'method'}, {'entity': 'surveillance data', 'label': 'diagnostic'}, {'entity': 'colours', 'label': 'method'}, {'entity': 'colours', 'label': 'method'}, {'entity': 'colours', 'label': 'method'}, {'entity': 'colours', 'label': 'method'}, {'entity': 'colours', 'label': 'method'}, {'entity': 'interface engineer', 'label': 'person'}, {'entity': 'social justice', 'label': 'environment'}, {'entity': 'environmental parameters', 'label': 'environment'}, {'entity': '1993', 'label': 'year'}, {'entity': '1999', 'label': 'year'}, {'entity': 'statistically', 'label': 'method'}, {'entity': 'solving', 'label': 'method'}, {'entity': 'decision-making', 'label': 'clinical trial'}, {'entity': 'spatial statistical', 'label': 'method'}, {'entity': 'health conditions', 'label': 'disease'}, {'entity': 'healthcare delivery', 'label': 'clinical trial'}, {'entity': 'skills', 'label': 'method'}, {'entity': 'surveillance issues', 'label': 'disease'}, {'entity': 'spatial information system', 'label': 'measurement'}, {'entity': 'organisational', 'label': 'organization'}, {'entity': 'data layer', 'label': 'measurement'}, {'entity': 'georeferenced health data', 'label': 'measurement'}, {'entity': 'Professor', 'label': 'person'}, {'entity': 'information infrastructure', 'label': 'treatment'}, {'entity': 'sectors', 'label': 'organization'}, {'entity': 'training programmes', 'label': 'clinical trial'}, {'entity': 'geo-information infrastructure', 'label': 'organization'}, {'entity': 'data security', 'label': 'measurement'}, {'entity': 'spatial metadata', 'label': 'measurement'}, {'entity': 'data model', 'label': 'measurement'}, {'entity': 'Data/analysis problems and errors', 'label': 'measurement'}, {'entity': 'data errors', 'label': 'measurement'}, {'entity': 'facts', 'label': 'diagnostic'}, {'entity': 'modular', 'label': 'method'}, {'entity': 'spatial sciences', 'label': 'organization'}, {'entity': 'automated', 'label': 'method'}, {'entity': 'testers', 'label': 'method'}, {'entity': 'Longerterm solution', 'label': 'treatment'}, {'entity': 'information infrastructure', 'label': 'public health'}, {'entity': 'applications', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'year'}, {'entity': 'technologies', 'label': 'technology'}, {'entity': 'development direction', 'label': 'method'}, {'entity': 'electronic health and social care records', 'label': 'measurement'}, {'entity': 'educate the public', 'label': 'public health'}, {'entity': 'infectious disease epidemics', 'label': 'epidemiology'}, {'entity': 'outbreak response', 'label': 'epidemiology'}, {'entity': '2003', 'label': 'year'}, {'entity': '29 April 2003', 'label': 'date'}, {'entity': '2003', 'label': 'year'}, {'entity': 'states', 'label': 'region'}, {'entity': 'SARS worldwide infections/ recoveries/deaths time line graph', 'label': 'diagnostic'}, {'entity': 'visually monitor', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'building', 'label': 'location'}, {'entity': '2003', 'label': 'year'}, {'entity': '11 July 2003', 'label': 'date'}, {'entity': 'age group', 'label': 'virus'}, {'entity': 'SARS spread', 'label': 'epidemiology'}, {'entity': 'animated GIF', 'label': 'method'}, {'entity': 'ESRI China', 'label': 'organization'}, {'entity': 'suspected', 'label': 'disease'}, {'entity': 'Internet age', 'label': 'disease'}, {'entity': 'desktop', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'building', 'label': 'location'}, {'entity': 'lies', 'label': 'disease'}, {'entity': 'free of charge', 'label': 'measurement'}, {'entity': 'systemic steroids', 'label': 'drug'}, {'entity': 'infectious complications', 'label': 'disease'}, {'entity': 'Systemic steroid', 'label': 'drug'}, {'entity': 'clinical outcome', 'label': 'treatment'}, {'entity': 'bias', 'label': 'clinical trial'}, {'entity': 'appropriate timing', 'label': 'treatment'}, {'entity': 'clinically important', 'label': 'measurement'}, {'entity': 'evidence of harm', 'label': 'disease'}, {'entity': 'anecdotal', 'label': 'clinical trial'}, {'entity': 'liver-related morbidity and mortality', 'label': 'disease'}, {'entity': 'nasal inhalation', 'label': 'disease'}, {'entity': 'Detection of HCV RNA', 'label': 'diagnostic'}, {'entity': 'nasal', 'label': 'location'}, {'entity': 'virological', 'label': 'virus'}, {'entity': 'nasal cavity', 'label': 'location'}, {'entity': 'principal investigator', 'label': 'person'}, {'entity': 'similar type', 'label': 'vocabulary'}, {'entity': 'group', 'label': 'virus'}, {'entity': 'genome polymorphism', 'label': 'measurement'}, {'entity': '2002', 'label': 'year'}, {'entity': 'sequence polymorphism', 'label': 'measurement'}, {'entity': 'formats', 'label': 'measurement'}, {'entity': '29757', 'label': 'measurement'}, {'entity': 'sequence polymorphism', 'label': 'gene'}, {'entity': 'fully sequenced isolate', 'label': 'virus'}, {'entity': 'poly-a strings', 'label': 'gene'}, {'entity': 'TWC', 'label': 'virus'}, {'entity': 'IV', 'label': 'measurement'}, {'entity': 'phylogenetic classification', 'label': 'method'}, {'entity': 'classification', 'label': 'virus'}, {'entity': '4', 'label': 'measurement'}, {'entity': 'Phylogenetic tree of 36 SARS-CoV complete genome isolates', 'label': 'diagram'}, {'entity': 'Phylogenetic tree of 36 SARS-CoV complete genome isolates', 'label': 'diagram'}, {'entity': 'sequence set', 'label': 'measurement'}, {'entity': 'analysis of dissimilarities', 'label': 'method'}, {'entity': 'homologous', 'label': 'gene'}, {'entity': 'insertions', 'label': 'virus'}, {'entity': 'B group', 'label': 'group'}, {'entity': '2003', 'label': 'year'}, {'entity': 'fully sequenced isolate', 'label': 'virus'}, {'entity': 'function', 'label': 'method'}, {'entity': 'other', 'label': 'organization'}, {'entity': 'file', 'label': 'publication'}, {'entity': 'first outbreak', 'label': 'event'}, {'entity': 'proteolytic cleavage', 'label': 'protein'}, {'entity': 'recognise', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'picornavirus proteinases', 'label': 'protein'}, {'entity': 'computational prediction', 'label': 'method'}, {'entity': 'Q', 'label': 'protein'}, {'entity': 'picornavirus 3C proteinase counterpart', 'label': 'protein'}, {'entity': 'site logo', 'label': 'diagnostic'}, {'entity': 'position specific', 'label': 'measurement'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'consensus pattern', 'label': 'method'}, {'entity': 'sites', 'label': 'measurement'}, {'entity': 'classification', 'label': 'method'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'possibly cleaved', 'label': 'treatment'}, {'entity': 'protein cleavage', 'label': 'pathway'}, {'entity': 'coronavirus proteinase 3CL pro', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'tumor-suppressor protein', 'label': 'protein'}, {'entity': 'Lung related proteins', 'label': 'protein'}, {'entity': 'phosphorylation sites', 'label': 'protein'}, {'entity': 'conformational issues', 'label': 'event'}, {'entity': 'drug development', 'label': 'treatment'}, {'entity': 'cross-validated set', 'label': 'method'}, {'entity': 'data set', 'label': 'measurement'}, {'entity': 'prediction outcomes', 'label': 'clinical trial'}, {'entity': 'President of Ireland', 'label': 'person'}, {'entity': 'causality', 'label': 'clinical trial'}, {'entity': 'MRSA control', 'label': 'treatment'}, {'entity': '2008', 'label': 'year'}, {'entity': 'expense', 'label': 'measurement'}, {'entity': 'isolated', 'label': 'virus'}, {'entity': '48', 'label': 'measurement'}, {'entity': 'isolated', 'label': 'treatment'}, {'entity': 'Java', 'label': 'software'}, {'entity': 'DNA sequence', 'label': 'measurement'}, {'entity': 'molecular biologist', 'label': 'person'}, {'entity': 'Mac OS X', 'label': 'organization'}, {'entity': 'MS Windows', 'label': 'organization'}, {'entity': 'Java', 'label': 'method'}, {'entity': 'text file', 'label': 'measurement'}, {'entity': 'exported', 'label': 'method'}, {'entity': 'Extensible Markup Language', 'label': 'method'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'plain text', 'label': 'format'}, {'entity': 'genome sequences', 'label': 'virus'}, {'entity': 'alignments', 'label': 'measurement'}, {'entity': 'visual cues', 'label': 'diagnostic'}, {'entity': 'color-coded', 'label': 'method'}, {'entity': 'third', 'label': 'sequence'}, {'entity': 'second', 'label': 'measurement'}, {'entity': 'view', 'label': 'method'}, {'entity': 'long sequence blocks', 'label': 'measurement'}, {'entity': 'cross-platform', 'label': 'method'}, {'entity': 'DNA sequence', 'label': 'measurement'}, {'entity': 'alignment file', 'label': 'publication'}, {'entity': 'column', 'label': 'method'}, {'entity': 'table header', 'label': 'clinical trial'}, {'entity': 'summary window', 'label': 'organization'}, {'entity': 'polled', 'label': 'method'}, {'entity': 'XML', 'label': 'format'}, {'entity': 'JPEG', 'label': 'format'}, {'entity': 'screen', 'label': 'location'}, {'entity': 'preferences', 'label': 'publication'}, {'entity': 'word processor', 'label': 'tool'}, {'entity': 'formats', 'label': 'method'}, {'entity': 'search results table', 'label': 'method'}, {'entity': 'Perl 5', 'label': 'method'}, {'entity': 'query sequence', 'label': 'virus'}, {'entity': 'alignment file', 'label': 'file'}, {'entity': 'text file', 'label': 'measurement'}, {'entity': 'WWW', 'label': 'organization'}, {'entity': 'WWW', 'label': 'organization'}, {'entity': 'insertions', 'label': 'vaccine'}, {'entity': 'Java', 'label': 'method'}, {'entity': 'freely available', 'label': 'measurement'}, {'entity': 'Java', 'label': 'method'}, {'entity': 'SARS pandemic', 'label': 'event'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'estimate K', 'label': 'method'}, {'entity': 'genomics', 'label': 'method'}, {'entity': 'SARS pandemic', 'label': 'event'}, {'entity': 'root', 'label': 'gene'}, {'entity': 'non-linear effects', 'label': 'method'}, {'entity': 'ancestor sequence', 'label': 'virus'}, {'entity': \"S'\", 'label': 'variable'}, {'entity': 'tree', 'label': 'diagram'}, {'entity': 'estimate K and T 0', 'label': 'method'}, {'entity': 'common ancestor', 'label': 'gene'}, {'entity': 'final sequences', 'label': 'sequence'}, {'entity': 'November 11, 2002', 'label': 'date'}, {'entity': 'pair wise', 'label': 'method'}, {'entity': 'June 2003', 'label': 'date'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': '2003', 'label': 'year'}, {'entity': 'day', 'label': 'measurement'}, {'entity': 'host factors', 'label': 'host'}, {'entity': 'haploinsufficiency', 'label': 'gene'}, {'entity': 'IGF-II pathway', 'label': 'pathway'}, {'entity': 'carcinogenesis', 'label': 'treatment'}, {'entity': 'virus replication', 'label': 'virus'}, {'entity': 'virus replication', 'label': 'viral process'}, {'entity': 'Persistently infected RIE-1 cells fail to survive in serum-free media', 'label': 'symptom'}, {'entity': 'media', 'label': 'vaccine'}, {'entity': 'repression', 'label': 'gene'}, {'entity': '48 hours post-infection', 'label': 'time point'}, {'entity': 'mature hormone', 'label': 'protein'}, {'entity': 'IGF-II 62T', 'label': 'protein'}, {'entity': 'virion proteins', 'label': 'protein'}, {'entity': 'cell clone', 'label': 'host'}, {'entity': 'pathway', 'label': 'pathway'}, {'entity': 'Anchorage-independent growth phenotypes of RIE-1 cell clones', 'label': 'treatment'}, {'entity': 'media', 'label': 'virus'}, {'entity': 'morphogenesis', 'label': 'process'}, {'entity': 'mature hormone', 'label': 'gene'}, {'entity': 'negative activity', 'label': 'virus'}, {'entity': 'genetically selected trait', 'label': 'trait'}, {'entity': 'disassembly', 'label': 'disease'}, {'entity': 'virus resistant', 'label': 'treatment'}, {'entity': 'gene trap retrovirus shuttle vector', 'label': 'gene'}, {'entity': 'fresh media', 'label': 'vaccine'}, {'entity': 'visualized', 'label': 'diagnostic'}, {'entity': 'murine', 'label': 'gene'}, {'entity': 'reverse transcriptase PCR', 'label': 'method'}, {'entity': 'Rochester, NY', 'label': 'location'}, {'entity': 'sea plaque agarose', 'label': 'measurement'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'adsorb', 'label': 'virus'}, {'entity': 'Indianapolis, IN', 'label': 'location'}, {'entity': 'Rochester, NY', 'label': 'location'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'November 2002', 'label': 'date'}, {'entity': 'isolated', 'label': 'method'}, {'entity': 'media', 'label': 'viral cell culture media'}, {'entity': 'artefacts', 'label': 'disease'}, {'entity': 'filtered variations', 'label': 'measurement'}, {'entity': 'random noise', 'label': 'measurement'}, {'entity': 'null hypothesis', 'label': 'method'}, {'entity': 'noisy mutation', 'label': 'virus'}, {'entity': 'isolated', 'label': 'virus'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'principal isolate', 'label': 'virus'}, {'entity': 'data point', 'label': 'measurement'}, {'entity': '18372', 'label': 'measurement'}, {'entity': 'Inferring phylogenetic relationships', 'label': 'method'}, {'entity': 'artifacts', 'label': 'virus'}, {'entity': 'variations', 'label': 'measurement'}, {'entity': 'real mutation', 'label': 'virus'}, {'entity': '54 isolates', 'label': 'measurement'}, {'entity': 'clinical contact tracing', 'label': 'clinical trial'}, {'entity': 'SZ3', 'label': 'virus'}, {'entity': \"3' end\", 'label': 'gene'}, {'entity': 'relative instability', 'label': 'measurement'}, {'entity': 'Oct 21, 2002', 'label': 'date'}, {'entity': 'branches', 'label': 'pathway'}, {'entity': 'insertions', 'label': 'protein'}, {'entity': 'host response', 'label': 'immune response'}, {'entity': 'replicating', 'label': 'virus'}, {'entity': 'disease pathology', 'label': 'pathology'}, {'entity': 'human coronavirus 229E', 'label': 'virus'}, {'entity': 'world-wide problem', 'label': 'epidemiology'}, {'entity': 'host response', 'label': 'immune response'}, {'entity': 'primary events', 'label': 'immune response'}, {'entity': 'model system', 'label': 'method'}, {'entity': 'isolated', 'label': 'method'}, {'entity': 'spots', 'label': 'measurement'}, {'entity': 'viral infected samples', 'label': 'disease'}, {'entity': 'features', 'label': 'gene'}, {'entity': 'grouped together', 'label': 'method'}, {'entity': '2 days', 'label': 'measurement'}, {'entity': 'day 6', 'label': 'measurement'}, {'entity': 'replicate', 'label': 'virus'}, {'entity': 'donor d', 'label': 'person'}, {'entity': 'harvested', 'label': 'method'}, {'entity': 'technical noise', 'label': 'measurement'}, {'entity': '2C', 'label': 'figure'}, {'entity': 'up regulation', 'label': 'measurement'}, {'entity': 'localized', 'label': 'treatment'}, {'entity': 'host recognition of the virus', 'label': 'host'}, {'entity': 'expression range', 'label': 'measurement'}, {'entity': 'Human coronavirus 229E', 'label': 'virus'}, {'entity': 'smallpox eradication campaign', 'label': 'event'}, {'entity': 'smallpox outbreak', 'label': 'disease'}, {'entity': 'smallpox outbreak', 'label': 'disease'}, {'entity': 'mathematical model', 'label': 'method'}, {'entity': 'mathematical', 'label': 'method'}, {'entity': 'discrete-time network', 'label': 'method'}, {'entity': 'network simulation', 'label': 'method'}, {'entity': 'spread disease', 'label': 'disease'}, {'entity': 'protection against infection', 'label': 'vaccine'}, {'entity': 'detection of cases', 'label': 'diagnostic'}, {'entity': 'smallpox outbreak', 'label': 'disease'}, {'entity': 'significance', 'label': 'clinical trial'}, {'entity': 'smallpox vaccination', 'label': 'vaccine'}, {'entity': 'course of infection', 'label': 'treatment'}, {'entity': 'lesions', 'label': 'symptom'}, {'entity': 'mild', 'label': 'symptom'}, {'entity': 'severe', 'label': 'disease'}, {'entity': 'secondary case', 'label': 'virus'}, {'entity': 'secondary case', 'label': 'epidemiology'}, {'entity': 'epidemiologically relevant states', 'label': 'epidemiology'}, {'entity': 'social group', 'label': 'organization'}, {'entity': 'epidemic potential', 'label': 'epidemiology'}, {'entity': '2001', 'label': 'year'}, {'entity': 'uninfected', 'label': 'disease'}, {'entity': 'social group', 'label': 'treatment'}, {'entity': 'traced', 'label': 'vaccine'}, {'entity': 'Latin', 'label': 'language'}, {'entity': 'casual', 'label': 'transmission'}, {'entity': 'Latin', 'label': 'language'}, {'entity': 'household tracing delay', 'label': 'treatment'}, {'entity': '10', 'label': 'measurement'}, {'entity': 'eradication', 'label': 'treatment'}, {'entity': 'tracing contacts of contacts', 'label': 'method'}, {'entity': 'Prob.', 'label': 'measurement'}, {'entity': 'severe smallpox', 'label': 'disease'}, {'entity': 'social group', 'label': 'location'}, {'entity': 'prodromal period', 'label': 'symptom'}, {'entity': 'index case', 'label': 'disease'}, {'entity': 'realizations', 'label': 'measurement'}, {'entity': 'naturally occurring', 'label': 'virus'}, {'entity': 'uncertainty', 'label': 'measurement'}, {'entity': 'hazard for random transmission', 'label': 'transmission'}, {'entity': 'hazard for close contact transmission', 'label': 'transmission'}, {'entity': 'control parameters', 'label': 'method'}, {'entity': 'realizations', 'label': 'clinical trial'}, {'entity': 'φ', 'label': 'variable'}, {'entity': 'simulations', 'label': 'method'}, {'entity': 'replications', 'label': 'clinical trial'}, {'entity': 'simulated', 'label': 'method'}, {'entity': '1000', 'label': 'measurement'}, {'entity': 'workplace/social(w/s)', 'label': 'location'}, {'entity': 'high workplace/ social contact finding probability', 'label': 'measurement'}, {'entity': 'workplace/social contact finding probability', 'label': 'measurement'}, {'entity': 'workplace/social contact', 'label': 'environment'}, {'entity': 'household contact finding probability', 'label': 'measurement'}, {'entity': 'parameter set', 'label': 'clinical trial'}, {'entity': 'slow scenario', 'label': 'event'}, {'entity': 'simulations', 'label': 'clinical trial'}, {'entity': 'slow scenario', 'label': 'event'}, {'entity': 'fast scenario', 'label': 'scenario'}, {'entity': 'realizations', 'label': 'clinical trial'}, {'entity': 'contact finding probability', 'label': 'measurement'}, {'entity': 'diagnosis rate of cases', 'label': 'diagnostic'}, {'entity': 'scenarios', 'label': 'clinical trial'}, {'entity': 'smallpox epidemic', 'label': 'epidemiology'}, {'entity': '40', 'label': 'measurement'}, {'entity': 'Infectivity prior to the rash', 'label': 'disease'}, {'entity': 'loss of containment', 'label': 'event'}, {'entity': 'prodromal period', 'label': 'disease'}, {'entity': 'case b', 'label': 'clinical trial'}, {'entity': 'severe smallpox', 'label': 'disease'}, {'entity': '40', 'label': 'measurement'}, {'entity': 'Infectivity prior to the rash', 'label': 'measurement'}, {'entity': 'scenario b', 'label': 'event'}, {'entity': 'scenario a', 'label': 'scenario'}, {'entity': 'decontainment', 'label': 'event'}, {'entity': 'isolation of contacts', 'label': 'treatment'}, {'entity': 'simulated epidemics', 'label': 'clinical trial'}, {'entity': 'simulated epidemics', 'label': 'clinical trial'}, {'entity': 'scenarios', 'label': 'clinical trial'}, {'entity': 'household size', 'label': 'measurement'}, {'entity': 'health care worker', 'label': 'person'}, {'entity': 'smallpox control', 'label': 'disease'}, {'entity': 'health care workers', 'label': 'person'}, {'entity': 'disease control', 'label': 'public health'}, {'entity': 'disease control', 'label': 'public health'}, {'entity': 'response logistics', 'label': 'method'}, {'entity': 'transmit disease', 'label': 'transmission'}, {'entity': 'epidemic containment', 'label': 'treatment'}, {'entity': 'contact finding', 'label': 'detection'}, {'entity': 'polio eradication initiative', 'label': 'organization'}, {'entity': 'lack of information', 'label': 'disease'}, {'entity': 'copyright', 'label': 'publication'}, {'entity': 'permanently archived', 'label': 'method'}, {'entity': 'US National Library of Medicine', 'label': 'organization'}, {'entity': 'science', 'label': 'epidemiology'}, {'entity': 'reproduce', 'label': 'method'}, {'entity': '30', 'label': 'measurement'}, {'entity': 'reviewers', 'label': 'person'}, {'entity': 'reviewers', 'label': 'person'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'article-processing', 'label': 'measurement'}, {'entity': 'editorial', 'label': 'organization'}, {'entity': 'patent application', 'label': 'publication'}, {'entity': 'licensed', 'label': 'clinical trial'}, {'entity': 'biodefense study section', 'label': 'organization'}, {'entity': 'reimbursement', 'label': 'virus'}, {'entity': 'females', 'label': 'person'}, {'entity': 'immunofluorescence negative', 'label': 'diagnostic'}, {'entity': 'perceived cost', 'label': 'measurement'}, {'entity': 'primer set', 'label': 'method'}, {'entity': 'non-specific priming', 'label': 'method'}, {'entity': 'primer set', 'label': 'method'}, {'entity': 'investigation', 'label': 'clinical trial'}, {'entity': 'true positives', 'label': 'clinical trial'}, {'entity': 'specificity check', 'label': 'diagnostic'}, {'entity': 'throat', 'label': 'location'}, {'entity': 'throat', 'label': 'location'}, {'entity': 'female', 'label': 'person'}, {'entity': 'parainfluenza virus type 1', 'label': 'virus'}, {'entity': 'F adenovirus', 'label': 'virus'}, {'entity': 'clinical practice', 'label': 'method'}, {'entity': 'human respiratory viruses', 'label': 'virus'}, {'entity': 'reporting', 'label': 'method'}, {'entity': 'H1', 'label': 'virus'}, {'entity': 'H3', 'label': 'virus'}, {'entity': 'multi-reaction', 'label': 'measurement'}, {'entity': 'mixes', 'label': 'vaccine'}, {'entity': 'strip', 'label': 'diagnostic'}, {'entity': 'Mix stability', 'label': 'measurement'}, {'entity': 'nested format', 'label': 'method'}, {'entity': 'results', 'label': 'measurement'}, {'entity': 'skilled', 'label': 'person'}, {'entity': '2008', 'label': 'year'}, {'entity': 'factor', 'label': 'clinical trial'}, {'entity': 'equipment', 'label': 'measurement'}, {'entity': 'diagnosing viral infections', 'label': 'diagnostic'}, {'entity': 'laboratories', 'label': 'organization'}, {'entity': 'immunofluorescence negative', 'label': 'diagnostic'}, {'entity': 'January 2002', 'label': 'date'}, {'entity': 'national external quality assurance scheme', 'label': 'organization'}, {'entity': 'primer mix', 'label': 'method'}, {'entity': 'aliquoted', 'label': 'method'}, {'entity': 'U.K', 'label': 'country'}, {'entity': 'bands', 'label': 'diagnostic'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'skills', 'label': 'method'}, {'entity': 'acquired', 'label': 'method'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'knowledge', 'label': 'measurement'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical education', 'label': 'treatment'}, {'entity': 'specialty', 'label': 'organization'}, {'entity': 'specialty', 'label': 'organization'}, {'entity': 'basic costs of living', 'label': 'measurement'}, {'entity': 'practice', 'label': 'method'}, {'entity': 'research initiative', 'label': 'clinical trial'}, {'entity': 'stakeholders', 'label': 'person'}, {'entity': 'long-term care', 'label': 'clinical trial'}, {'entity': 'clinical situation', 'label': 'clinical trial'}, {'entity': 'ethical issues', 'label': 'clinical trial'}, {'entity': 'ethical issues', 'label': 'clinical trial'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'Washington, D.C.', 'label': 'location'}, {'entity': 'hospital ethicists', 'label': 'person'}, {'entity': 'facilitating meetings', 'label': 'method'}, {'entity': 'students', 'label': 'person'}, {'entity': 'knowledge', 'label': 'vaccine'}, {'entity': '2008', 'label': 'year'}, {'entity': 'capacity issues', 'label': 'clinical trial'}, {'entity': '2008', 'label': 'year'}, {'entity': 'professional knowledge', 'label': 'vocabulary'}, {'entity': 'career', 'label': 'clinical trial'}, {'entity': 'teaching', 'label': 'method'}, {'entity': 'research projects', 'label': 'clinical trial'}, {'entity': 'philosophy', 'label': 'method'}, {'entity': 'Practical clinical experiences', 'label': 'clinical trial'}, {'entity': 'challenges', 'label': 'clinical trial'}, {'entity': 'clinical case', 'label': 'clinical trial'}, {'entity': 'imminent', 'label': 'event'}, {'entity': 'hypothetical', 'label': 'event'}, {'entity': 'healthcare provider', 'label': 'person'}, {'entity': 'writing', 'label': 'method'}, {'entity': 'judgment', 'label': 'immune response'}, {'entity': 'fellow', 'label': 'person'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'competencies', 'label': 'clinical trial'}, {'entity': 'skill development', 'label': 'clinical trial'}, {'entity': 'educational activities', 'label': 'clinical trial'}, {'entity': 'scholarly activities', 'label': 'clinical trial'}, {'entity': 'clinical ethicists', 'label': 'person'}, {'entity': 'character traits', 'label': 'trait'}, {'entity': 'character traits', 'label': 'trait'}, {'entity': 'ethical decision-making', 'label': 'clinical trial'}, {'entity': 'self-knowledge', 'label': 'person'}, {'entity': 'work relatedstress', 'label': 'disease'}, {'entity': 'opinion', 'label': 'vaccine'}, {'entity': 'daily practice', 'label': 'method'}, {'entity': 'families', 'label': 'organization'}, {'entity': 'ethical issues', 'label': 'event'}, {'entity': 'insider', 'label': 'organization'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': 'clinical ethicist', 'label': 'person'}, {'entity': '2007', 'label': 'year'}, {'entity': 'block placements', 'label': 'clinical trial'}, {'entity': 'advocacy', 'label': 'clinical trial'}, {'entity': 'clinical ethicist', 'label': 'treatment'}, {'entity': 'educational model', 'label': 'method'}, {'entity': 'quality of reporting', 'label': 'measurement'}, {'entity': 'newly discovered infectious disease', 'label': 'disease'}, {'entity': 'day', 'label': 'measurement'}, {'entity': 'written protocol', 'label': 'publication'}, {'entity': 'search interface', 'label': 'method'}, {'entity': 'study types', 'label': 'clinical trial'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'outcome', 'label': 'measurement'}, {'entity': 'senior kindergarten', 'label': 'location'}, {'entity': 'elementary school', 'label': 'location'}, {'entity': 'reference database', 'label': 'organization'}, {'entity': 'bias', 'label': 'clinical trial'}, {'entity': 'titles', 'label': 'clinical trial'}, {'entity': '2005', 'label': 'year'}, {'entity': 'Disagreements', 'label': 'event'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'language', 'label': 'measurement'}, {'entity': 'English language', 'label': 'language'}, {'entity': 'abstract', 'label': 'publication'}, {'entity': 'resolved differences', 'label': 'method'}, {'entity': 'study designs', 'label': 'clinical trial'}, {'entity': 'Disagreements', 'label': 'clinical trial'}, {'entity': 'outcome', 'label': 'measurement'}, {'entity': 'absences', 'label': 'symptom'}, {'entity': 'abstracted', 'label': 'clinical trial'}, {'entity': 'quantitative synthesis', 'label': 'method'}, {'entity': 'articles', 'label': 'publication'}, {'entity': 'titles', 'label': 'clinical trial'}, {'entity': '2004', 'label': 'publication'}, {'entity': 'dropouts', 'label': 'event'}, {'entity': '2000', 'label': 'year'}, {'entity': '2004', 'label': 'year'}, {'entity': '16', 'label': 'clinical trial'}, {'entity': 'dropouts', 'label': 'clinical trial'}, {'entity': 'observer agreement statistics', 'label': 'measurement'}, {'entity': 'Cramer et al.', 'label': 'organization'}, {'entity': '1999', 'label': 'year'}, {'entity': 'schools', 'label': 'location'}, {'entity': 'hand sanitizers', 'label': 'vaccine'}, {'entity': 'non-randomized', 'label': 'method'}, {'entity': 'designs', 'label': 'method'}, {'entity': 'methodologically weak', 'label': 'method'}, {'entity': 'drop-outs', 'label': 'event'}, {'entity': '2009', 'label': 'year'}, {'entity': 'content experts', 'label': 'person'}, {'entity': 'quantitative synthesis', 'label': 'method'}, {'entity': '1999', 'label': 'year'}, {'entity': 'worldwide emergence', 'label': 'event'}, {'entity': 'rodent vector', 'label': 'virus'}, {'entity': 'insect vector', 'label': 'virus'}, {'entity': 'scoring matrix', 'label': 'method'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'domain IIa', 'label': 'protein'}, {'entity': 'regions of local similarity', 'label': 'protein'}, {'entity': 'Membrane Protein eXplorer', 'label': 'organization'}, {'entity': 'bonds', 'label': 'protein'}, {'entity': 'three-dimensional model', 'label': 'method'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'flavivirus class II viral fusion proteins', 'label': 'protein'}, {'entity': 'Bunyavirus Gc', 'label': 'protein'}, {'entity': '4', 'label': 'measurement'}, {'entity': 'virion protein', 'label': 'protein'}, {'entity': 'presumptive protein', 'label': 'protein'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'functional', 'label': 'protein'}, {'entity': 'color-coded', 'label': 'method'}, {'entity': 'deciphered', 'label': 'method'}, {'entity': 'Common order of proteins in Bunyavirus M segment polyproteins', 'label': 'protein'}, {'entity': '2012', 'label': 'publication'}, {'entity': 'segments', 'label': 'protein'}, {'entity': 'domains I-III', 'label': 'protein'}, {'entity': '6', 'label': 'measurement'}, {'entity': '6', 'label': 'measurement'}, {'entity': 'discontinuous replication', 'label': 'method'}, {'entity': '1960', 'label': 'year'}, {'entity': '1960', 'label': 'year'}, {'entity': 'nidovirales', 'label': 'gene'}, {'entity': 'RNA replication', 'label': 'virus'}, {'entity': 'autocatalytic cleavage', 'label': 'protein'}, {'entity': 'autocatalytic cleavage', 'label': 'protein'}, {'entity': \"Nucleotide content of individual HCoV-NL63 genes and the 5'/3' untranslated regions (UTR)\", 'label': 'measurement'}, {'entity': 'G anti-peaks', 'label': 'protein'}, {'entity': 'process', 'label': 'pathway'}, {'entity': '1b', 'label': 'gene'}, {'entity': 'discontinuous', 'label': 'method'}, {'entity': \"3' one-third\", 'label': 'gene'}, {'entity': 'viral transcription', 'label': 'virus'}, {'entity': 'strand-transfer', 'label': 'pathway'}, {'entity': 'mini-hairpin', 'label': 'gene'}, {'entity': 'strand transfer', 'label': 'treatment'}, {'entity': 'E body TRS', 'label': 'gene'}, {'entity': 'core sequence', 'label': 'gene'}, {'entity': 'homology', 'label': 'gene'}, {'entity': 'Expression levels of the HCoV-NL63 genomic and sg mRNAs', 'label': 'measurement'}, {'entity': 'Microsoft Excel', 'label': 'tool'}, {'entity': 'sliding window', 'label': 'method'}, {'entity': 'Mouse Hepatitis Virus', 'label': 'virus'}, {'entity': 'isolated', 'label': 'method'}, {'entity': 'cDNA bank', 'label': 'measurement'}, {'entity': 'complementary', 'label': 'virus'}, {'entity': 'SARS RNA', 'label': 'virus'}, {'entity': 'tertiary interaction', 'label': 'protein'}, {'entity': 'RNA tertiary structures', 'label': 'structure'}, {'entity': 'isomorphous replacement', 'label': 'method'}, {'entity': 'tertiary structural interaction', 'label': 'protein'}, {'entity': 'schematic diagram', 'label': 'diagram'}, {'entity': 'Long-range tertiary contacts', 'label': 'protein'}, {'entity': 'SARS RNA', 'label': 'gene'}, {'entity': 'structural contacts', 'label': 'protein'}, {'entity': 's2m SARS RNA', 'label': 'gene'}, {'entity': 'B(B) Formation', 'label': 'method'}, {'entity': 'seven-nucleotide asymmetric bubble', 'label': 'protein'}, {'entity': 'three-purine asymmetric bulge', 'label': 'protein'}, {'entity': 'seven-nucleotide bubble', 'label': 'protein'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'channel', 'label': 'pathway'}, {'entity': 'interactions', 'label': 'protein'}, {'entity': 'sequence complement', 'label': 'gene'}, {'entity': 'crystallizing', 'label': 'method'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'complementarity', 'label': 'protein'}, {'entity': 'cation-mediated', 'label': 'method'}, {'entity': 'eukaryotic homologs', 'label': 'protein'}, {'entity': 'hypothesis', 'label': 'diagnostic'}, {'entity': 'structural motif', 'label': 'protein'}, {'entity': 'SARS s2m RNA', 'label': 'protein'}, {'entity': 'sevennucleotide bubble', 'label': 'protein'}, {'entity': 'Patterson-map', 'label': 'method'}, {'entity': 'Mac OS X', 'label': 'organization'}, {'entity': 'data processing', 'label': 'method'}, {'entity': 'ethical framework', 'label': 'method'}, {'entity': 'decision making process', 'label': 'method'}, {'entity': 'opportunities for improvement', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'year'}, {'entity': 'fairness', 'label': 'value'}, {'entity': 'ethical framework', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'legitimate', 'label': 'word'}, {'entity': 'framework', 'label': 'method'}, {'entity': 'communicable disease outbreak', 'label': 'disease'}, {'entity': 'case study', 'label': 'method'}, {'entity': 'social processes', 'label': 'method'}, {'entity': 'key', 'label': 'person'}, {'entity': 'transcribed', 'label': 'method'}, {'entity': 'reasons', 'label': 'clinical trial'}, {'entity': 'United Nations Secretary-General', 'label': 'person'}, {'entity': 'voluntary', 'label': 'method'}, {'entity': 'concept', 'label': 'method'}, {'entity': 'evaluation', 'label': 'method'}, {'entity': 'verified', 'label': 'method'}, {'entity': 'decision making process', 'label': 'method'}, {'entity': 'reasonableness', 'label': 'clinical trial'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'ramp up', 'label': 'event'}, {'entity': 'SARS unit', 'label': 'treatment'}, {'entity': 'medical unit', 'label': 'location'}, {'entity': 'command center', 'label': 'organization'}, {'entity': 'no visitor policy', 'label': 'treatment'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'protecting staff', 'label': 'treatment'}, {'entity': 'reason', 'label': 'clinical trial'}, {'entity': 'urgent', 'label': 'clinical trial'}, {'entity': 'decisions made', 'label': 'clinical trial'}, {'entity': '2008', 'label': 'year'}, {'entity': 'reasons behind them', 'label': 'clinical trial'}, {'entity': 'discussion', 'label': 'event'}, {'entity': 'appealing the process', 'label': 'method'}, {'entity': 'implementation', 'label': 'treatment'}, {'entity': 'world-wide web', 'label': 'environment'}, {'entity': 'perceived', 'label': 'person'}, {'entity': 'goodwill', 'label': 'clinical trial'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'accountability for reasonableness', 'label': 'concept'}, {'entity': 'four conditions', 'label': 'clinical trial'}, {'entity': 'decision making', 'label': 'method'}, {'entity': 'signaling mechanisms', 'label': 'pathway'}, {'entity': 'cellular mechanisms', 'label': 'pathway'}, {'entity': 'growth factors', 'label': 'protein'}, {'entity': 'conserved', 'label': 'gene'}, {'entity': 'MAPK pathway', 'label': 'pathway'}, {'entity': 'PI3K inhibitor', 'label': 'drug'}, {'entity': 'PI3K inhibitor', 'label': 'drug'}, {'entity': 'PI3K inhibitor', 'label': 'drug'}, {'entity': 'floating cell', 'label': 'treatment'}, {'entity': 'D', 'label': 'virus'}, {'entity': '3A', 'label': 'measurement'}, {'entity': 'capsid RNA', 'label': 'virus'}, {'entity': '48 hours', 'label': 'measurement'}, {'entity': 'cell survival mechanisms', 'label': 'immune response'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'pUSEamp(+)', 'label': 'method'}, {'entity': 'mitogenic pathways', 'label': 'pathway'}, {'entity': '2000', 'label': 'publication'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'Mycoplasma-free rabbit kidney epithelial', 'label': 'organism'}, {'entity': 'plaque forming unit', 'label': 'measurement'}, {'entity': 'harvested', 'label': 'method'}, {'entity': 'DNA fragmentation', 'label': 'measurement'}, {'entity': 'Eukaryotes', 'label': 'organism'}, {'entity': 'superfamily', 'label': 'protein'}, {'entity': 'cell cycle', 'label': 'immune response'}, {'entity': 'homologues', 'label': 'gene'}, {'entity': '2009', 'label': 'year'}, {'entity': '11', 'label': 'measurement'}, {'entity': 'Appr-1-p processing enzyme family', 'label': 'protein'}, {'entity': 'mem-ber', 'label': 'gene'}, {'entity': 'alignment program', 'label': 'method'}, {'entity': 'ML trees', 'label': 'method'}, {'entity': 'random trees', 'label': 'treatment'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'Blosum 50 score', 'label': 'measurement'}, {'entity': 'campestris pv campestris', 'label': 'organism'}, {'entity': 'homologues', 'label': 'gene'}, {'entity': 'viral domains', 'label': 'virus'}, {'entity': 'Multiple alignment of sequence regions', 'label': 'method'}, {'entity': 'coordination', 'label': 'method'}, {'entity': 'coordination', 'label': 'protein'}, {'entity': 'Excel', 'label': 'software'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'virus'}, {'entity': 'essential residues', 'label': 'virus'}, {'entity': 'functional', 'label': 'virus'}, {'entity': 'homologues', 'label': 'gene'}, {'entity': 'Aconitum latent virus', 'label': 'virus'}, {'entity': 'integration', 'label': 'virus'}, {'entity': 'recent integration', 'label': 'event'}, {'entity': 'structural', 'label': 'protein'}, {'entity': 'indicated', 'label': 'virus'}, {'entity': 'domain', 'label': 'gene'}, {'entity': 'functional', 'label': 'treatment'}, {'entity': 'integrations', 'label': 'process'}, {'entity': 'degradation of RNA', 'label': 'process'}, {'entity': 'methylated', 'label': 'virus'}, {'entity': 'host proteins', 'label': 'protein'}, {'entity': 'AlkB integration', 'label': 'gene'}, {'entity': 'apple', 'label': 'host'}, {'entity': 'environmental compounds', 'label': 'environment'}, {'entity': 'integration', 'label': 'virus'}, {'entity': 'tree', 'label': 'diagram'}, {'entity': 'tree', 'label': 'diagram'}, {'entity': 'tree', 'label': 'diagram'}, {'entity': 'co-evolution', 'label': 'diagnostic'}, {'entity': 'domain', 'label': 'virus'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'integration patterns', 'label': 'pathway'}, {'entity': 'Dotter', 'label': 'software'}, {'entity': 'joints', 'label': 'symptom'}, {'entity': 'bones', 'label': 'immune response'}, {'entity': '2003', 'label': 'year'}, {'entity': 'examination of their lungs', 'label': 'diagnostic'}, {'entity': 'signs of lung consolidation', 'label': 'symptom'}, {'entity': 'lesions', 'label': 'symptom'}, {'entity': 'chest radiological examination', 'label': 'diagnostic'}, {'entity': '1+2+4+5', 'label': 'clinical trial'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'U.S.A', 'label': 'country'}, {'entity': 'interstitial', 'label': 'disease'}, {'entity': 'cavitation', 'label': 'symptom'}, {'entity': 'interstitium', 'label': 'pathway'}, {'entity': 'sagittal', 'label': 'method'}, {'entity': 'interval', 'label': 'measurement'}, {'entity': 'first stage', 'label': 'clinical trial'}, {'entity': '2002', 'label': 'year'}, {'entity': 'Mainland China', 'label': 'country'}, {'entity': 'Chinese traditional medicine', 'label': 'treatment'}, {'entity': 'clinical characteristics', 'label': 'disease'}, {'entity': 'digestive tract', 'label': 'location'}, {'entity': 'host factors', 'label': 'host'}, {'entity': 'degraded', 'label': 'protein'}, {'entity': 'folding', 'label': 'protein'}, {'entity': 'cellularlocation', 'label': 'location'}, {'entity': 'cytoplasmic production', 'label': 'protein'}, {'entity': 'secrete proteins', 'label': 'protein'}, {'entity': 'degradation', 'label': 'protein'}, {'entity': 'protein secretion', 'label': 'protein'}, {'entity': 'proteolysis', 'label': 'protein'}, {'entity': 'protein sample concentration', 'label': 'measurement'}, {'entity': 'secreted', 'label': 'protein'}, {'entity': 'L7', 'label': 'gene'}, {'entity': 'L12', 'label': 'gene'}, {'entity': '20', 'label': 'measurement'}, {'entity': 'protein quantification', 'label': 'measurement'}, {'entity': 'cytoplasmic form', 'label': 'protein'}, {'entity': 'secreted form', 'label': 'protein'}, {'entity': 'ovine interferon omega', 'label': 'protein'}, {'entity': 'production', 'label': 'treatment'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'cell fractions', 'label': 'measurement'}, {'entity': 'propeptide', 'label': 'protein'}, {'entity': 'heterologous', 'label': 'vocabulary'}, {'entity': 'Schematic representation', 'label': 'diagram'}, {'entity': 'proteases', 'label': 'protein'}, {'entity': 'production', 'label': 'measurement'}, {'entity': 'cytoplasmic chaperones', 'label': 'protein'}, {'entity': 'cytoplasmic chaperone', 'label': 'protein'}, {'entity': 'yields', 'label': 'measurement'}, {'entity': 'synthesis', 'label': 'protein'}, {'entity': 'protein conformation', 'label': 'protein'}, {'entity': 'fusion', 'label': 'method'}, {'entity': 'fusion', 'label': 'protein'}, {'entity': 'rule', 'label': 'method'}, {'entity': 'fusion', 'label': 'protein'}, {'entity': 'growth phase', 'label': 'clinical trial'}, {'entity': 'intracellular proteolysis', 'label': 'protein'}, {'entity': 'suppressor', 'label': 'gene'}, {'entity': 'directed', 'label': 'method'}, {'entity': 'L.(?<=[?!)', 'label': 'virus'}, {'entity': 'enzymes', 'label': 'protein'}, {'entity': 'cytoplasmic production', 'label': 'method'}, {'entity': 'stationary phase', 'label': 'growth phase'}, {'entity': 'exponential', 'label': 'growth phase'}, {'entity': 'L', 'label': 'protein'}, {'entity': 'L', 'label': 'gene'}, {'entity': 'cell compartment', 'label': 'gene'}, {'entity': 'production', 'label': 'measurement'}, {'entity': 'host factors', 'label': 'host'}, {'entity': 'protein production', 'label': 'process'}, {'entity': 'systems', 'label': 'treatment'}, {'entity': 'consistency of readings', 'label': 'measurement'}, {'entity': 'interpreted', 'label': 'method'}, {'entity': 'immunocompromised', 'label': 'disease'}, {'entity': 'home use', 'label': 'measurement'}, {'entity': 'communicate', 'label': 'vaccine'}, {'entity': 'San Diego, CA', 'label': 'location'}, {'entity': 'ear', 'label': 'location'}, {'entity': 'instrument', 'label': 'measurement'}, {'entity': 'central tendency', 'label': 'measurement'}, {'entity': 'females', 'label': 'person'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital thermometer', 'label': 'measurement'}, {'entity': 'home thermometer', 'label': 'measurement'}, {'entity': 'hospital tympanic thermometer', 'label': 'measurement'}, {'entity': 'home tympanic thermometer', 'label': 'measurement'}, {'entity': 'reference standard', 'label': 'diagnostic'}, {'entity': '2011', 'label': 'publication'}, {'entity': 'rectum', 'label': 'location'}, {'entity': 'recent outbreak', 'label': 'event'}, {'entity': 'body', 'label': 'diagnostic'}, {'entity': 'human error', 'label': 'symptom'}, {'entity': 'Abbreviations', 'label': 'measurement'}, {'entity': 'commerce', 'label': 'organization'}, {'entity': 'human history', 'label': 'epidemiology'}, {'entity': 'Historically', 'label': 'event'}, {'entity': '1995', 'label': 'year'}, {'entity': 'scientific', 'label': 'clinical trial'}, {'entity': '1989', 'label': 'year'}, {'entity': 'public health concern', 'label': 'public health'}, {'entity': 'meat production', 'label': 'clinical trial'}, {'entity': 'scientific', 'label': 'organization'}, {'entity': 'genetically modifi ed organisms', 'label': 'organism'}, {'entity': 'trade restrictions', 'label': 'measurement'}, {'entity': 'traded goods', 'label': 'product'}, {'entity': 'harmful to health', 'label': 'disease'}, {'entity': 'intellectual property rights', 'label': 'organization'}, {'entity': '1994', 'label': 'year'}, {'entity': 'copyright', 'label': 'measurement'}, {'entity': '2001', 'label': 'year'}, {'entity': 'royalty payment', 'label': 'measurement'}, {'entity': 'knowledge', 'label': 'diagnostic'}, {'entity': '2008', 'label': 'year'}, {'entity': 'world market', 'label': 'environment'}, {'entity': 'intellectual property rights', 'label': 'organization'}, {'entity': 'trade law', 'label': 'method'}, {'entity': 'communications technology', 'label': 'technology'}, {'entity': 'contracting out', 'label': 'method'}, {'entity': 'commercial', 'label': 'method'}, {'entity': 'market forces', 'label': 'method'}, {'entity': 'diffi culties', 'label': 'clinical trial'}, {'entity': 'world trading system', 'label': 'organization'}, {'entity': 'trade liberalisation', 'label': 'event'}, {'entity': 'potent gene knock-down', 'label': 'gene'}, {'entity': 'drug platform', 'label': 'drug'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {'entity': 'permissions', 'label': 'vaccine'}, {'entity': 'cooled', 'label': 'method'}, {'entity': 'Inhibition of firefly luciferase', 'label': 'method'}, {'entity': 'modifications', 'label': 'vaccine'}, {'entity': 'modified', 'label': 'method'}, {'entity': 'overhang', 'label': 'measurement'}, {'entity': 'firefly siLNA', 'label': 'vaccine'}, {'entity': 'firefly siLNA', 'label': 'vaccine'}, {'entity': 'firefly target', 'label': 'protein'}, {'entity': \"5' sense end\", 'label': 'gene'}, {'entity': 'sense end', 'label': 'protein'}, {'entity': 'tested positions', 'label': 'location'}, {'entity': '13', 'label': 'measurement'}, {'entity': 'irrelevant', 'label': 'measurement'}, {'entity': 'effective siLNA', 'label': 'measurement'}, {'entity': 'duplicate', 'label': 'measurement'}, {'entity': 'sense', 'label': 'measurement'}, {'entity': 'sense modification', 'label': 'method'}, {'entity': 'ultimate', 'label': 'clinical trial'}, {'entity': 'activity', 'label': 'measurement'}, {'entity': 'modifications', 'label': 'vaccine'}, {'entity': 'quadruplicate', 'label': 'measurement'}, {'entity': 'donor', 'label': 'host'}, {'entity': 'common ancestor', 'label': 'gene'}, {'entity': 'evolutionary processes', 'label': 'pathway'}, {'entity': 'prokaryote', 'label': 'gene'}, {'entity': '14', 'label': 'publication'}, {'entity': 'discrimination', 'label': 'method'}, {'entity': 'long word', 'label': 'measurement'}, {'entity': 'computational charge', 'label': 'measurement'}, {'entity': 'double-strand signature', 'label': 'method'}, {'entity': 'columns', 'label': 'method'}, {'entity': 'lines', 'label': 'method'}, {'entity': 'local signature', 'label': 'gene'}, {'entity': 'biases', 'label': 'vaccine'}, {'entity': 'categorical dependent variable', 'label': 'variable'}, {'entity': 'Comparison of signatures', 'label': 'method'}, {'entity': 'differences in distance', 'label': 'measurement'}, {'entity': 'local signature', 'label': 'measurement'}, {'entity': '22 species', 'label': 'number'}, {'entity': 'credible donors', 'label': 'measurement'}, {'entity': 'window', 'label': 'measurement'}, {'entity': 'sliding window', 'label': 'method'}, {'entity': 'segments', 'label': 'virus'}, {'entity': 'Distance of outlier from first potential donor', 'label': 'measurement'}, {'entity': 'unusual', 'label': 'virus'}, {'entity': 'foreign sequence', 'label': 'virus'}, {'entity': 'abnormal', 'label': 'disease'}, {'entity': '2.5', 'label': 'measurement'}, {'entity': 'intra-species', 'label': 'term'}, {'entity': 'advantages', 'label': 'clinical trial'}, {'entity': 'Ragan', 'label': 'person'}, {'entity': '7', 'label': 'measurement'}, {'entity': 'Kappa of zero', 'label': 'measurement'}, {'entity': 'Kappa of one', 'label': 'measurement'}, {'entity': 'single vote', 'label': 'measurement'}, {'entity': 'five methods', 'label': 'method'}, {'entity': 'every other approach', 'label': 'method'}, {'entity': 'Nakamura approach', 'label': 'method'}, {'entity': 'isolated', 'label': 'measurement'}, {'entity': '14', 'label': 'publication'}, {'entity': 'majority', 'label': 'gene'}, {'entity': 'outliers', 'label': 'measurement'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'homolog', 'label': 'gene'}, {'entity': 'atypicality', 'label': 'disease'}, {'entity': 'window', 'label': 'measurement'}, {'entity': 'criterion for detecting HTs', 'label': 'virus'}, {'entity': 'mean G+C%', 'label': 'measurement'}, {'entity': 'bank of signatures', 'label': 'measurement'}, {'entity': 'taxonomy', 'label': 'epidemiology'}, {'entity': 'taxonomically', 'label': 'gene'}, {'entity': 'selection of donors', 'label': 'method'}, {'entity': 'taxonomic', 'label': 'gene'}, {'entity': '50', 'label': 'measurement'}, {'entity': 'host response to this pathogen', 'label': 'immune response'}, {'entity': 'gene array', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'September 2003', 'label': 'date'}, {'entity': 'leukocytes', 'label': 'protein'}, {'entity': 'complex mechanisms', 'label': 'pathway'}, {'entity': 'significant gene', 'label': 'gene'}, {'entity': 'filter criteria', 'label': 'method'}, {'entity': 'Gene name', 'label': 'gene'}, {'entity': 'tissue remodeling', 'label': 'pathway'}, {'entity': 'Gene name Fold change', 'label': 'measurement'}, {'entity': 'ankyrin', 'label': 'protein'}, {'entity': 'nucleosome assembly', 'label': 'gene'}, {'entity': 'gene array', 'label': 'method'}, {'entity': 'Santa Clara, CA', 'label': 'location'}, {'entity': 'Carlsbad, CA', 'label': 'location'}, {'entity': 'human focus array', 'label': 'diagnostic'}, {'entity': 'Gene chip fluidics Station 400', 'label': 'equipment'}, {'entity': 'Eugene, OR', 'label': 'location'}, {'entity': 'chip', 'label': 'measurement'}, {'entity': '2002', 'label': 'year'}, {'entity': '2002', 'label': 'year'}, {'entity': 'nidovirales', 'label': 'virus'}, {'entity': 'person-to-person', 'label': 'transmission'}, {'entity': 'coding sequences', 'label': 'gene'}, {'entity': 'selection', 'label': 'immune response'}, {'entity': 'cysteine proteinases', 'label': 'protein'}, {'entity': 'acidic environments', 'label': 'environment'}, {'entity': 'novel infectious diseases', 'label': 'disease'}, {'entity': '2003', 'label': 'year'}, {'entity': 'danger', 'label': 'treatment'}, {'entity': 'small intestine', 'label': 'location'}, {'entity': 'days', 'label': 'measurement'}, {'entity': 'third phase', 'label': 'event'}, {'entity': 'second phase', 'label': 'clinical trial'}, {'entity': 'third', 'label': 'clinical trial'}, {'entity': 'children younger than 12 years', 'label': 'host'}, {'entity': 'disease progression', 'label': 'disease'}, {'entity': 'long-term complications', 'label': 'symptom'}, {'entity': 'cellular fusion', 'label': 'process'}, {'entity': 'viral replica-tion', 'label': 'virus'}, {'entity': 'gastrointestinal', 'label': 'treatment'}, {'entity': 'parenchymal cells', 'label': 'cell'}, {'entity': 'group I coronaviruses', 'label': 'virus'}, {'entity': 'drug development candidate', 'label': 'drug'}, {'entity': 'motifs', 'label': 'gene'}, {'entity': 'viral RNA synthesis', 'label': 'disease'}, {'entity': 'cysteine proteinase', 'label': 'protein'}, {'entity': 'cleavage sites', 'label': 'protein'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'catalytic', 'label': 'protein'}, {'entity': 'conformational rearrange-ments', 'label': 'protein'}, {'entity': 'structures', 'label': 'diagnostic'}, {'entity': 'binding mode', 'label': 'protein'}, {'entity': 'HIV-1 protease inhibitor', 'label': 'drug'}, {'entity': 'proved', 'label': 'method'}, {'entity': 'drug delivery system', 'label': 'drug'}, {'entity': 'recombinant viral proteins', 'label': 'protein'}, {'entity': 'DNA prime and adenovirus', 'label': 'vaccine'}, {'entity': 'limitation', 'label': 'clinical trial'}, {'entity': 'SARS-CoV infections', 'label': 'disease'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'global health hazard', 'label': 'public health'}, {'entity': 'linkage', 'label': 'method'}, {'entity': 'emerging virus', 'label': 'virus'}, {'entity': 'arrays', 'label': 'method'}, {'entity': 'non-specific', 'label': 'measurement'}, {'entity': 'fluorescent signal', 'label': 'measurement'}, {'entity': 'signal', 'label': 'measurement'}, {'entity': 'complementary arm sequences', 'label': 'gene'}, {'entity': 'Palo Alto, CA', 'label': 'location'}, {'entity': 'beacon', 'label': 'vaccine'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'hybridized', 'label': 'method'}, {'entity': 'channels', 'label': 'measurement'}, {'entity': 'conjugation', 'label': 'method'}, {'entity': 'MFI', 'label': 'measurement'}, {'entity': 'representative detection curve', 'label': 'diagnostic'}, {'entity': 'MFI', 'label': 'measurement'}, {'entity': 'control sequence', 'label': 'virus'}, {'entity': 'nucleotide mismatches', 'label': 'measurement'}, {'entity': 'replicates', 'label': 'measurement'}, {'entity': 'mispriming', 'label': 'diagnostic'}, {'entity': 'homologous sequence', 'label': 'gene'}, {'entity': 'complex synthesis', 'label': 'method'}, {'entity': 'Human Herpesvirus-6', 'label': 'virus'}, {'entity': 'tool dependent', 'label': 'method'}, {'entity': 'mRNA expression analysis', 'label': 'method'}, {'entity': 'mRNA expression', 'label': 'measurement'}, {'entity': 'gene evaluation', 'label': 'gene'}, {'entity': 'values', 'label': 'measurement'}, {'entity': 'experiment', 'label': 'clinical trial'}, {'entity': 'expressed', 'label': 'gene'}, {'entity': 'gene selection', 'label': 'gene'}, {'entity': '17D', 'label': 'virus'}, {'entity': 'RNA transcription', 'label': 'measurement'}, {'entity': 'particle', 'label': 'virus'}, {'entity': '1960', 'label': 'year'}, {'entity': 'spring', 'label': 'season'}, {'entity': 'coronavirus group 2', 'label': 'virus'}, {'entity': 'Screening of specimens', 'label': 'diagnostic'}, {'entity': \"3'\", 'label': 'gene'}, {'entity': 'group 1 coronavirus', 'label': 'virus'}, {'entity': 'November 2000', 'label': 'measurement'}, {'entity': '2004', 'label': 'year'}, {'entity': 'outer primer set', 'label': 'gene'}, {'entity': 'inner primer set', 'label': 'gene'}, {'entity': 'PCR Optimizer Kit', 'label': 'method'}, {'entity': 'specific primer', 'label': 'method'}, {'entity': 'yearly', 'label': 'epidemiology'}, {'entity': 'type B', 'label': 'virus'}, {'entity': '1 year', 'label': 'person'}, {'entity': 'subcluster', 'label': 'pathway'}, {'entity': 'Gel electrophoresis after pancoronavirus RT-PCR assay', 'label': 'method'}, {'entity': 'pseudoreplicates', 'label': 'measurement'}, {'entity': 'underlying conditions', 'label': 'disease'}, {'entity': '16 years', 'label': 'measurement'}, {'entity': 'February 2004', 'label': 'measurement'}, {'entity': 'season', 'label': 'epidemiology'}, {'entity': 'autumn', 'label': 'season'}, {'entity': '2002', 'label': 'year'}, {'entity': '2002', 'label': 'year'}, {'entity': 'worldwide distribution', 'label': 'region'}, {'entity': 'molecular markers', 'label': 'measurement'}, {'entity': '1988', 'label': 'year'}, {'entity': 'spring', 'label': 'season'}, {'entity': '1988', 'label': 'year'}, {'entity': 'subtype', 'label': 'virus'}, {'entity': 'ATM', 'label': 'gene'}, {'entity': 'sequence tracts', 'label': 'measurement'}, {'entity': 'insertions and deletions', 'label': 'measurement'}, {'entity': 'functional element', 'label': 'gene'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {}, {'entity': 'Palo Alto, CA', 'label': 'location'}, {'entity': 'logarithm', 'label': 'measurement'}, {'entity': 'random variation', 'label': 'measurement'}, {'entity': 'mean hybridization specificity', 'label': 'measurement'}, {'entity': 'original value', 'label': 'measurement'}, {'entity': 'Santa Clara, CA', 'label': 'location'}, {'entity': 'cognate', 'label': 'measurement'}, {'entity': 'cognate MM', 'label': 'measurement'}, {'entity': 'bin', 'label': 'measurement'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': '2009', 'label': 'year'}, {'entity': 'nucleotide tracts', 'label': 'gene'}, {'entity': 'structures', 'label': 'pathway'}, {'entity': 'sequence tracts', 'label': 'gene'}, {'entity': 'two base deletion probes', 'label': 'measurement'}, {'entity': 'ATM coding', 'label': 'gene'}, {'entity': 'base insertion', 'label': 'measurement'}, {'entity': 'screens for genetic variation', 'label': 'clinical trial'}, {'entity': 'sequence context', 'label': 'measurement'}, {'entity': 'bulges', 'label': 'protein'}, {'entity': '41', 'label': 'measurement'}, {'entity': 'strong', 'label': 'measurement'}, {'entity': 'sequence tract', 'label': 'protein'}, {'entity': 'definitive explanation', 'label': 'clinical trial'}, {'entity': 'functional significance', 'label': 'measurement'}, {'entity': 'identity', 'label': 'clinical trial'}, {'entity': '3a', 'label': 'protein'}, {'entity': 'surface expression', 'label': 'protein'}, {'entity': 'endoglycosidase H-resistant', 'label': 'protein'}, {'entity': 'cell-cell fusion', 'label': 'treatment'}, {'entity': 'cytoplasmic tail', 'label': 'protein'}, {'entity': 'protein localization', 'label': 'pathway'}, {'entity': 'fusion', 'label': 'immune response'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'amino-acids', 'label': 'protein'}, {'entity': 'cytoplasmic tail', 'label': 'protein'}, {'entity': 'virus evolution', 'label': 'treatment'}, {'entity': 'competing interests', 'label': 'organization'}, {'entity': 'medical needs', 'label': 'disease'}, {'entity': 'science', 'label': 'method'}, {'entity': '1990', 'label': 'year'}, {'entity': 'medical needs', 'label': 'disease'}, {'entity': 'access to existing treatments', 'label': 'treatment'}, {'entity': 'human right', 'label': 'vaccine'}, {'entity': 'binding framework', 'label': 'treaty'}, {'entity': 'industrial development', 'label': 'measurement'}, {'entity': 'innovation', 'label': 'treatment'}, {'entity': 'richest', 'label': 'person'}, {'entity': 'high-income countries', 'label': 'country'}, {'entity': '2002', 'label': 'year'}, {'entity': 'patent system', 'label': 'organization'}, {'entity': 'promoted', 'label': 'method'}, {'entity': 'patent system', 'label': 'organization'}, {'entity': 'international policy instrument', 'label': 'organization'}, {'entity': 'incentives', 'label': 'organization'}, {'entity': '2008', 'label': 'year'}, {'entity': '2001', 'label': 'year'}, {'entity': 'limit access to medicines', 'label': 'treatment'}, {'entity': 'international policy', 'label': 'organization'}, {'entity': 'international problem', 'label': 'event'}, {'entity': 'strategic sectors', 'label': 'organization'}, {'entity': 'collective action', 'label': 'clinical trial'}, {'entity': '1890', 'label': 'year'}, {'entity': 'public-sector funded', 'label': 'organization'}, {'entity': 'drug discovery', 'label': 'treatment'}, {'entity': 'reward', 'label': 'clinical trial'}, {'entity': 'patent rights', 'label': 'vaccine'}, {'entity': 'Innovative', 'label': 'method'}, {'entity': '2002', 'label': 'publication'}, {'entity': '60,000 people die from Afi can sleeping sickness', 'label': 'disease'}, {'entity': 'dissemination of information', 'label': 'method'}, {'entity': 'innovations', 'label': 'drug'}, {'entity': 'incentives', 'label': 'treatment'}, {'entity': 'government rights', 'label': 'organization'}, {'entity': '1980', 'label': 'year'}, {'entity': 'treaty', 'label': 'organization'}, {'entity': 'binding agreements', 'label': 'organization'}, {'entity': 'juridical avenue', 'label': 'organization'}, {'entity': 'patented', 'label': 'vaccine'}, {'entity': 'resources', 'label': 'vaccine'}, {'entity': '2001', 'label': 'year'}, {'entity': 'treaty', 'label': 'organization'}, {'entity': 'health R&D', 'label': 'organization'}, {'entity': 'public research', 'label': 'organization'}, {'entity': 'technology', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': '2010', 'label': 'year'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'GIS', 'label': 'organization'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'travel', 'label': 'transmission'}, {'entity': 'sequence variation', 'label': 'virus'}, {'entity': 'genetic variations', 'label': 'gene'}, {'entity': 'copy number', 'label': 'measurement'}, {'entity': 'calls', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'heterogeneous variant', 'label': 'variant'}, {'entity': 'T:C:G:A variant', 'label': 'variant'}, {'entity': 'Singapore SARS outbreak', 'label': 'event'}, {'entity': 'primary patient', 'label': 'virus'}, {'entity': 'Singapore SARS', 'label': 'disease'}, {'entity': 'Singapore SARS', 'label': 'disease'}, {'entity': 'sequence variation', 'label': 'virus'}, {'entity': 'viral genetic characterization', 'label': 'method'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'sequence information', 'label': 'measurement'}, {'entity': '3', 'label': 'measurement'}, {'entity': 'development', 'label': 'virus'}, {'entity': 'biological phenomena', 'label': 'pathway'}, {'entity': 'lysosomal', 'label': 'protein'}, {'entity': 'organogenesis', 'label': 'process'}, {'entity': 'germ cell differentiation', 'label': 'pathway'}, {'entity': 'germ cell', 'label': 'gene'}, {'entity': 'DNA content', 'label': 'measurement'}, {'entity': 'testicular development', 'label': 'pathway'}, {'entity': 'testicular development', 'label': 'pathway'}, {'entity': 'spermatogenic cells', 'label': 'cell'}, {'entity': 'Becton Dickinson, Calif', 'label': 'organization'}, {'entity': 'Clontech, Calif', 'label': 'organization'}, {'entity': 'Huntsville, Ala', 'label': 'location'}, {'entity': 'cell lineage', 'label': 'vocabulary'}, {'entity': 'imaged', 'label': 'method'}, {'entity': 'colon', 'label': 'location'}, {'entity': 'testicular development', 'label': 'pathway'}, {'entity': 'somatic cell', 'label': 'cell'}, {'entity': 'Negative control of sense probes', 'label': 'method'}, {'entity': 'developmental program', 'label': 'pathway'}, {'entity': 'stage', 'label': 'gene'}, {'entity': 'stages', 'label': 'stage-specific'}, {'entity': 'chromosome synapsis', 'label': 'gene'}, {'entity': 'interaction', 'label': 'clinical trial'}, {'entity': 'germ cell', 'label': 'cell'}, {'entity': '2C cells', 'label': 'cell'}, {'entity': 'differentiation', 'label': 'treatment'}, {'entity': '181', 'label': 'measurement'}, {'entity': 'embryo development', 'label': 'pathway'}, {'entity': 'development', 'label': 'clinical trial'}, {'entity': 'B lymphocyte', 'label': 'cell'}, {'entity': 'cellular differentiation', 'label': 'pathway'}, {'entity': 'ninecarbon', 'label': 'protein'}, {'entity': 'Hantavirus type', 'label': 'virus'}, {'entity': 'quasi-species', 'label': 'virus'}, {'entity': 'lines', 'label': 'method'}, {'entity': 'viral RNA polymerase', 'label': 'protein'}, {'entity': 'bases', 'label': 'protein'}, {'entity': 'S segment', 'label': 'gene'}, {'entity': '2003', 'label': 'year'}, {'entity': 'international linkages', 'label': 'organization'}, {'entity': 'goods', 'label': 'environment'}, {'entity': '2003', 'label': 'year'}, {'entity': 'international bodies and organizations', 'label': 'organization'}, {'entity': 'agreements', 'label': 'organization'}, {'entity': 'multilateral treaty', 'label': 'agreement'}, {'entity': 'advertising', 'label': 'method'}, {'entity': 'accounting', 'label': 'method'}, {'entity': 'New Zealand counterpart', 'label': 'organization'}, {'entity': '2008', 'label': 'year'}, {'entity': 'health relations', 'label': 'public health'}, {'entity': 'statutory authority', 'label': 'organization'}, {'entity': '2003', 'label': 'year'}, {'entity': 'free trade agreements', 'label': 'agreement'}, {'entity': 'travel industry', 'label': 'industry'}, {'entity': '1908', 'label': 'year'}, {'entity': 'remote areas', 'label': 'location'}, {'entity': 'previous policies', 'label': 'policy'}, {'entity': 'ethical danger', 'label': 'event'}, {'entity': 'professions', 'label': 'occupation'}, {'entity': '2003', 'label': 'year'}, {'entity': 'cascade of events', 'label': 'pathway'}, {'entity': 'severe acute respiratory syndrome-associated coronavirus', 'label': 'virus'}, {'entity': 'Viral plaque', 'label': 'disease'}, {'entity': 'type II IFNs', 'label': 'drug'}, {'entity': 'type II IFN treatment', 'label': 'treatment'}, {'entity': 'synergy', 'label': 'immune response'}, {'entity': 'HCMV replication', 'label': 'pathway'}, {'entity': 'vehicle', 'label': 'drug'}, {'entity': 'antagonism', 'label': 'clinical trial'}, {'entity': 'synergy', 'label': 'measurement'}, {'entity': 'viral dissemination', 'label': 'immune response'}, {'entity': 'effects', 'label': 'virus'}, {'entity': 'lytic cycle', 'label': 'pathway'}, {'entity': 'type II IFNs', 'label': 'drug'}, {'entity': 'Minneapolis, MN', 'label': 'city'}, {'entity': 'Houston, TX', 'label': 'location'}, {'entity': 'antagonism', 'label': 'immune response'}, {'entity': 'synergism', 'label': 'immune response'}, {'entity': 'isoboles', 'label': 'diagnostic'}, {'entity': 'convex isobole', 'label': 'diagnostic'}, {'entity': 'Concave isobole', 'label': 'method'}, {'entity': 'San Leandro, CA', 'label': 'location'}, {'entity': 'projection', 'label': 'virus'}, {'entity': 'virus orientation determination', 'label': 'method'}, {'entity': 'acquiring images', 'label': 'method'}, {'entity': 'acquisition', 'label': 'method'}, {'entity': 'particle image', 'label': 'virus'}, {'entity': 'criteria', 'label': 'method'}, {'entity': 'projection', 'label': 'method'}, {'entity': 'particle image', 'label': 'diagnostic'}, {'entity': 'picture', 'label': 'method'}, {'entity': 'images', 'label': 'measurement'}, {'entity': 'data recording', 'label': 'method'}, {'entity': 'image', 'label': 'measurement'}, {'entity': 'microscope', 'label': 'tool'}, {'entity': '2006', 'label': 'year'}, {'entity': 'projections', 'label': 'measurement'}, {'entity': 'criterion', 'label': 'method'}, {'entity': 'projection', 'label': 'method'}, {'entity': 'implementation', 'label': 'method'}, {'entity': '3D model', 'label': 'method'}, {'entity': 'projections', 'label': 'method'}, {'entity': 'biasing', 'label': 'method'}, {'entity': 'close to focus', 'label': 'measurement'}, {'entity': 'far to focus', 'label': 'method'}, {'entity': '3D structure', 'label': 'measurement'}, {'entity': 'empty shell', 'label': 'virus'}, {'entity': 'icosahedrally symmetric model', 'label': 'model'}, {'entity': 'asymmetric triangle', 'label': 'method'}, {'entity': 'icosahedral symmetry', 'label': 'method'}, {'entity': '3D structure', 'label': 'structure'}, {'entity': 'projection', 'label': 'method'}, {'entity': 'orientations projection', 'label': 'method'}, {'entity': 'micrograph', 'label': 'measurement'}, {'entity': 'projection matching', 'label': 'method'}, {'entity': 'low resolution model', 'label': 'measurement'}, {'entity': 'Particle', 'label': 'method'}, {'entity': 'P22 data', 'label': 'measurement'}, {'entity': 'HSV reconstruction', 'label': 'method'}, {'entity': 'micrograph', 'label': 'measurement'}, {'entity': 'Surface visualization', 'label': 'method'}, {'entity': 'empty shell', 'label': 'protein'}, {'entity': 'orientations determinations', 'label': 'measurement'}, {'entity': 'noisy', 'label': 'measurement'}, {'entity': 'contrast', 'label': 'measurement'}, {'entity': 'Gaussian filtered projection matching', 'label': 'method'}, {'entity': 'wavelet projection matching', 'label': 'method'}, {'entity': 'virus reconstruction', 'label': 'treatment'}, {'entity': 'acquiring images', 'label': 'method'}, {'entity': 'signal', 'label': 'measurement'}, {'entity': 'image', 'label': 'signal'}, {'entity': 'computable', 'label': 'method'}, {'entity': 'spectrum', 'label': 'measurement'}, {'entity': 'orientation', 'label': 'method'}, {'entity': 'decomposition', 'label': 'method'}, {'entity': 'A 2', 'label': 'variable'}, {'entity': 'approximation component', 'label': 'method'}, {'entity': 'initial orientation', 'label': 'method'}, {'entity': 'orientation estimation', 'label': 'clinical trial'}, {'entity': 'processors', 'label': 'measurement'}, {'entity': 'hierarchical implementation', 'label': 'method'}, {'entity': 'icosahedrally symmetric object', 'label': 'virus'}, {'entity': 'projection', 'label': 'measurement'}, {'entity': 'raw image', 'label': 'measurement'}, {'entity': 'coefficients', 'label': 'measurement'}, {'entity': 'projection', 'label': 'method'}, {'entity': 'individual recombination rate', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'Brassica rapa', 'label': 'host'}, {'entity': 'cellular machinery', 'label': 'protein'}, {'entity': 'cell coinfection', 'label': 'process'}, {'entity': 'generation', 'label': 'virus'}, {'entity': 'generations', 'label': 'measurement'}, {'entity': 'viral population', 'label': 'virus'}, {'entity': 'host cell infection', 'label': 'host'}, {'entity': 'mathematical', 'label': 'method'}, {'entity': 'panmictically', 'label': 'term'}, {'entity': 'clonally', 'label': 'method'}, {'entity': 'interference', 'label': 'immune response'}, {'entity': 'DNA synthesis', 'label': 'process'}, {'entity': '95% confidence intervals', 'label': 'measurement'}, {'entity': 'sequence interruption', 'label': 'virus'}, {'entity': 'sequence interruption', 'label': 'virus'}, {'entity': 'sequence interruption', 'label': 'virus'}, {'entity': 'synonymous mutation', 'label': 'gene'}, {'entity': 'spectrometry', 'label': 'method'}, {'entity': 'Image program', 'label': 'method'}, {'entity': 'neutral', 'label': 'measurement'}, {'entity': 'd', 'label': 'marker'}, {'entity': 'neutrality', 'label': 'measurement'}, {'entity': 'viral DNA', 'label': 'virus'}, {'entity': 'recombination distribution', 'label': 'measurement'}, {'entity': 'estimator', 'label': 'method'}, {'entity': 'generations', 'label': 'measurement'}, {'entity': 'April 2003', 'label': 'event'}, {'entity': 'protection of individual freedom', 'label': 'concept'}, {'entity': 'liberties', 'label': 'person'}, {'entity': 'restriction of movement', 'label': 'event'}, {'entity': 'expression of caring', 'label': 'symptom'}, {'entity': '2009', 'label': 'year'}, {'entity': '2009', 'label': 'year'}, {'entity': 'known facts', 'label': 'disease'}, {'entity': 'due diligence', 'label': 'method'}, {'entity': 'expectations', 'label': 'clinical trial'}, {'entity': 'professional code of conduct', 'label': 'vaccine'}, {'entity': 'ethical considerations', 'label': 'clinical trial'}, {'entity': 'families', 'label': 'host'}, {'entity': 'acceptance', 'label': 'organization'}, {'entity': 'criteria', 'label': 'organization'}, {'entity': 'rights', 'label': 'public health'}, {'entity': 'ethical argument', 'label': 'method'}, {'entity': 'identity', 'label': 'vaccine'}, {'entity': 'ethical value', 'label': 'concept'}, {'entity': 'contaminated', 'label': 'disease'}, {'entity': 'bias', 'label': 'event'}, {'entity': 'ethical values', 'label': 'clinical trial'}, {'entity': 'identity', 'label': 'person'}, {'entity': 'decision making', 'label': 'clinical trial'}, {'entity': 'evaluators', 'label': 'person'}, {'entity': 'information access', 'label': 'method'}, {'entity': 'handheld', 'label': 'method'}, {'entity': 'clinical decision making', 'label': 'treatment'}, {'entity': 'Berkeley, California', 'label': 'location'}, {'entity': 'clinical practice', 'label': 'treatment'}, {'entity': 'clinical decision making', 'label': 'clinical trial'}, {'entity': 'management decisions', 'label': 'clinical trial'}, {'entity': 'medical reference material', 'label': 'publication'}, {'entity': 'disseminating knowledge', 'label': 'method'}, {'entity': 'Palm Inc', 'label': 'organization'}, {'entity': 'residents', 'label': 'person'}, {'entity': 'file', 'label': 'publication'}, {'entity': 'e-mail', 'label': 'method'}, {'entity': 'telephone', 'label': 'method'}, {'entity': 'handheld devices', 'label': 'device'}, {'entity': 'final scenario', 'label': 'event'}, {'entity': 'information sources', 'label': 'vaccine'}, {'entity': 'methanol toxicity', 'label': 'disease'}, {'entity': 'study participants', 'label': 'person'}, {'entity': 'handheld device', 'label': 'device'}, {'entity': 'study time course', 'label': 'clinical trial'}, {'entity': 'audiovisual recording', 'label': 'method'}, {'entity': 'applications', 'label': 'method'}, {'entity': 'objective', 'label': 'clinical trial'}, {'entity': 'orders', 'label': 'treatment'}, {'entity': 'SAS Institute, Inc', 'label': 'organization'}, {'entity': 'personal information management', 'label': 'clinical trial'}, {'entity': 'operating system', 'label': 'vocabulary'}, {'entity': 'operating system', 'label': 'vocabulary'}, {'entity': 'medical information', 'label': 'disease'}, {'entity': 'handheld medical applications', 'label': 'clinical trial'}, {'entity': 'electronic media', 'label': 'method'}, {'entity': 'baseline scenarios', 'label': 'clinical trial'}, {'entity': 'Information source', 'label': 'method'}, {'entity': 'primary source of information', 'label': 'method'}, {'entity': 'Android', 'label': 'organization'}, {'entity': 'Washington, D.C.', 'label': 'city'}, {'entity': 'clinical scenarios', 'label': 'clinical trial'}, {'entity': 'medical reference information', 'label': 'publication'}, {'entity': 'clinical decision-making', 'label': 'clinical trial'}, {'entity': 'clinical situation', 'label': 'environment'}, {'entity': 'clinical scenario', 'label': 'treatment'}, {'entity': 'bias', 'label': 'clinical trial'}, {'entity': '2008', 'label': 'year'}, {'entity': 'lack of familiarity with the technology', 'label': 'vaccine'}, {'entity': '2003', 'label': 'year'}, {'entity': 'handheld', 'label': 'organization'}, {'entity': 'knowledge translation', 'label': 'clinical trial'}, {'entity': 'supply', 'label': 'measurement'}, {'entity': 'information technology', 'label': 'method'}, {'entity': 'clinical practice', 'label': 'treatment'}, {'entity': 'desktop', 'label': 'organization'}, {'entity': 'technology', 'label': 'method'}, {'entity': 'technologies', 'label': 'method'}, {'entity': 'search', 'label': 'method'}, {'entity': 'device', 'label': 'organization'}, {'entity': 'handhand', 'label': 'method'}, {'entity': 'isolated', 'label': 'measurement'}, {'entity': 'connector', 'label': 'device'}, {'entity': 'Clinical management of sepsis', 'label': 'clinical trial'}, {'entity': 'sector', 'label': 'organization'}, {'entity': 'artificial intelligence', 'label': 'method'}, {'entity': '2002', 'label': 'publication'}, {'entity': 'carers', 'label': 'person'}, {'entity': 'UK health service', 'label': 'organization'}, {'entity': 'patient information system', 'label': 'system'}, {'entity': '2010', 'label': 'publication'}, {'entity': 'information technology', 'label': 'technology'}, {'entity': 'total system failure', 'label': 'event'}, {'entity': 'system failures', 'label': 'event'}, {'entity': 'audit', 'label': 'diagnostic'}, {'entity': 'health IT', 'label': 'clinical trial'}, {'entity': 'clinical practice', 'label': 'treatment'}, {'entity': 'technologies', 'label': 'technology'}, {'entity': '29', 'label': 'measurement'}, {'entity': 'human orthologue', 'label': 'gene'}, {'entity': '2005', 'label': 'publication'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {'entity': 'synthesis', 'label': 'protein'}, {'entity': 'embryogenesis', 'label': 'process'}, {'entity': 'non-frameshift product', 'label': 'gene'}, {'entity': 'À1 frameshifting', 'label': 'gene'}, {'entity': 'RNA structure probing', 'label': 'method'}, {'entity': 'translated', 'label': 'method'}, {'entity': 'gel loading buffer', 'label': 'diagnostic'}, {'entity': 'run-off transcripts', 'label': 'method'}, {'entity': 'identity', 'label': 'virus'}, {'entity': 'stable', 'label': 'protein'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'C residue', 'label': 'gene'}, {'entity': 'reading frame', 'label': 'gene'}, {'entity': '1b', 'label': 'gene'}, {'entity': '1b', 'label': 'stem'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'loop adenosine', 'label': 'protein'}, {'entity': 'RNA structure mapping', 'label': 'method'}, {'entity': 'enzymatic', 'label': 'protein'}, {'entity': 'chemical probe', 'label': 'vaccine'}, {'entity': 'cleaved', 'label': 'method'}, {'entity': 'enzymes', 'label': 'protein'}, {'entity': 'viral frameshift', 'label': 'virus'}, {'entity': 'sequence comparisons', 'label': 'method'}, {'entity': 'stem 1 bulge', 'label': 'gene'}, {'entity': 'cellular metabolism', 'label': 'immune response'}, {'entity': 'errors', 'label': 'method'}, {'entity': 'synthetic green fluorescent protein', 'label': 'protein'}, {'entity': 'DNA sequence', 'label': 'measurement'}, {'entity': 'functional selection', 'label': 'method'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {'entity': 'synthesis errors', 'label': 'virus'}, {'entity': 'iterated', 'label': 'method'}, {'entity': 'mismatch exposure', 'label': 'method'}, {'entity': 'induced', 'label': 'method'}, {'entity': 'bp overlap', 'label': 'measurement'}, {'entity': 'incubate', 'label': 'method'}, {'entity': 'reactions', 'label': 'method'}, {'entity': 'binding protein', 'label': 'protein'}, {'entity': 'H', 'label': 'protein'}, {'entity': 'homoduplex', 'label': 'gene'}, {'entity': 'colonies', 'label': 'measurement'}, {'entity': 'input', 'label': 'measurement'}, {'entity': 'mismatch', 'label': 'virus'}, {'entity': 'constructs', 'label': 'measurement'}, {'entity': 'mathematical model', 'label': 'model'}, {'entity': 'fragmented', 'label': 'method'}, {'entity': 'MutS filtering', 'label': 'method'}, {'entity': 'error free', 'label': 'measurement'}, {'entity': 'consensus shuffling error rate', 'label': 'measurement'}, {'entity': 'binding conditions', 'label': 'method'}, {'entity': 'error rates per clone', 'label': 'measurement'}, {'entity': 'error reduction', 'label': 'method'}, {'entity': 'Agents', 'label': 'method'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {'entity': 'rat ribosomal RNA', 'label': 'gene'}, {'entity': '30 June 2004', 'label': 'date'}, {'entity': 'taxonomic table', 'label': 'publication'}, {'entity': 'National Institute of Infectious Disease', 'label': 'organization'}, {'entity': 'Carlsbad, CA', 'label': 'location'}, {'entity': 'Madison, WI', 'label': 'location'}, {'entity': 'ligated', 'label': 'method'}, {'entity': 'ligated', 'label': 'method'}, {'entity': 'amplicons', 'label': 'measurement'}, {'entity': 'Piscataway, NJ', 'label': 'location'}, {'entity': 'artificial RNA', 'label': 'vaccine'}, {'entity': 'programs', 'label': 'method'}, {'entity': 'bands', 'label': 'diagnostic'}, {'entity': 'virus sequence', 'label': 'virus'}, {'entity': 'specificity of primers', 'label': 'measurement'}, {'entity': 'database programming', 'label': 'method'}, {'entity': 'cytoplasmic vesicles', 'label': 'protein'}, {'entity': 'membranous structures', 'label': 'protein'}, {'entity': 'double membrane', 'label': 'structure'}, {'entity': 'intermediate', 'label': 'treatment'}, {'entity': 'virally induced vesicles', 'label': 'virus'}, {'entity': 'poliovirus-induced vesicles', 'label': 'virus'}, {'entity': 'viral growth', 'label': 'disease'}, {'entity': 'autophagic membrane', 'label': 'pathway'}, {'entity': 'double-membraned', 'label': 'vocabulary'}, {'entity': 'viral destruction', 'label': 'disease'}, {'entity': 'cell lysis', 'label': 'virus'}, {'entity': 'lysis', 'label': 'treatment'}, {'entity': 'RNA replication', 'label': 'protein'}, {'entity': 'double membrane', 'label': 'pathway'}, {'entity': 'replication complex', 'label': 'virus'}, {'entity': 'nonlytic', 'label': 'treatment'}, {'entity': 'particle', 'label': 'virus'}, {'entity': 'double-membraned', 'label': 'protein'}, {'entity': 'ESCRT complex', 'label': 'protein'}, {'entity': 'Palo Alto, California', 'label': 'location'}, {'entity': 'Santa Cruz, Califor-nia', 'label': 'location'}, {'entity': 'Lexington, Kentucky', 'label': 'location'}, {'entity': '4% formaldehyde solution', 'label': 'measurement'}, {'entity': 'SGI Octane', 'label': 'organization'}, {'entity': '7.0', 'label': 'measurement'}, {'entity': 'Piscataway, New Jersey', 'label': 'location'}, {'entity': 'population-based descriptive analysis', 'label': 'method'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'medical center', 'label': 'location'}, {'entity': 'quality of care', 'label': 'clinical trial'}, {'entity': 'March 14, 2003', 'label': 'date'}, {'entity': 'May', 'label': 'month'}, {'entity': 'medical center', 'label': 'location'}, {'entity': 'classic example', 'label': 'example'}, {'entity': 'pre-natal', 'label': 'clinical trial'}, {'entity': 'vaginal', 'label': 'method'}, {'entity': 'reimbursement', 'label': 'method'}, {'entity': 'quality of care', 'label': 'treatment'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'claims', 'label': 'clinical trial'}, {'entity': 'Case Payment-DRG 0373B', 'label': 'measurement'}, {'entity': 'accreditation level', 'label': 'measurement'}, {'entity': 'January 2002', 'label': 'date'}, {'entity': 'tested for', 'label': 'method'}, {'entity': 'Bureau of National Health Insurance', 'label': 'organization'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'peak SARS period', 'label': 'event'}, {'entity': 'pre-SARS period', 'label': 'period'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'peak-and post-SARS periods', 'label': 'period'}, {'entity': 'higher level hospitals', 'label': 'location'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'potential risk', 'label': 'virus'}, {'entity': 'peak SARS period', 'label': 'event'}, {'entity': 'medical center', 'label': 'organization'}, {'entity': 'SARS epidemic', 'label': 'event'}, {'entity': 'quality of care', 'label': 'measurement'}, {'entity': 'baby', 'label': 'symptom'}, {'entity': 'fears', 'label': 'symptom'}, {'entity': 'maternity', 'label': 'disease'}, {'entity': 'perinatal', 'label': 'disease'}, {'entity': 'various aspects', 'label': 'epidemiology'}, {'entity': \"previous year's level\", 'label': 'measurement'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'last case', 'label': 'event'}, {'entity': 'February 2003', 'label': 'event'}, {'entity': 'small', 'label': 'pathway'}, {'entity': 'transmitted to humans', 'label': 'transmission'}, {'entity': 'SARS epidemiology', 'label': 'epidemiology'}, {'entity': 'SARS epidemic', 'label': 'epidemiology'}, {'entity': 'etiology of the virus', 'label': 'disease'}, {'entity': 'turkey coronavirus', 'label': 'virus'}, {'entity': 'manifestation', 'label': 'disease'}, {'entity': 'human strain', 'label': 'virus'}, {'entity': 'SARS-CoV replication', 'label': 'process'}, {'entity': '2002', 'label': 'year'}, {'entity': 'cellular biology', 'label': 'biology'}, {'entity': 'SARS-CoV epidemic', 'label': 'disease'}, {'entity': 'tertiary interactions', 'label': 'protein'}, {'entity': 'CoV vaccine', 'label': 'vaccine'}, {'entity': 'engineered', 'label': 'method'}, {'entity': 'modified vaccinia virus', 'label': 'virus'}, {'entity': 'virus receptor', 'label': 'protein'}, {'entity': 'pandemic potential', 'label': 'virus'}, {'entity': 'development', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'year'}, {'entity': 'prevent human disease', 'label': 'disease'}, {'entity': 'traditional taxonomy', 'label': 'method'}, {'entity': 'bisectionally-distributed', 'label': 'method'}, {'entity': 'Lepidoptera', 'label': 'organism'}, {'entity': 'taxonomy', 'label': 'method'}, {'entity': 'characters', 'label': 'measurement'}, {'entity': 'bisect', 'label': 'method'}, {'entity': 'bisectionally distributed', 'label': 'method'}, {'entity': 'logarithm', 'label': 'measurement'}, {'entity': 'target-specific characters', 'label': 'method'}, {'entity': 'ideal characters', 'label': 'method'}, {'entity': 'characters', 'label': 'measurement'}, {'entity': 'bisectionally-distributed', 'label': 'method'}, {'entity': 'progressively', 'label': 'method'}, {'entity': 'subsequences', 'label': 'gene'}, {'entity': 'subsequences', 'label': 'gene'}, {'entity': 'aligned', 'label': 'method'}, {'entity': '4.4', 'label': 'measurement'}, {'entity': 'pools', 'label': 'measurement'}, {'entity': 'taxonomic key', 'label': 'method'}, {'entity': 'subsequences', 'label': 'gene'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'sequence variation', 'label': 'measurement'}, {'entity': 'dataset', 'label': 'publication'}, {'entity': 'datasets', 'label': 'publication'}, {'entity': 'Sets', 'label': 'method'}, {'entity': 'dataset', 'label': 'measurement'}, {'entity': 'real sequences', 'label': 'disease'}, {'entity': 'random data', 'label': 'measurement'}, {'entity': 'dataset', 'label': 'measurement'}, {'entity': 'pools', 'label': 'method'}, {'entity': 'sequence set', 'label': 'vocabulary'}, {'entity': 'selection algorithm', 'label': 'method'}, {'entity': 'bioinformatically', 'label': 'method'}, {'entity': 'combinatorial probe set', 'label': 'method'}, {'entity': 'datasets', 'label': 'publication'}, {'entity': 'insect', 'label': 'host'}, {'entity': 'dataset', 'label': 'organization'}, {'entity': 'singletons', 'label': 'measurement'}, {'entity': 'Set', 'label': 'method'}, {'entity': 'set', 'label': 'method'}, {'entity': 'search', 'label': 'method'}, {'entity': 'subtypes', 'label': 'gene'}, {'entity': '44', 'label': 'measurement'}, {'entity': 'deceased patients', 'label': 'person'}, {'entity': 'geneti-cally related', 'label': 'gene'}, {'entity': 'ethnicity', 'label': 'measurement'}, {'entity': 'healthy individuals', 'label': 'person'}, {'entity': 'disease outcome', 'label': 'symptom'}, {'entity': 'forward', 'label': 'gene'}, {'entity': 'SPSS', 'label': 'software'}, {'entity': 'Disease association', 'label': 'disease'}, {'entity': 'healthy individuals', 'label': 'disease'}, {'entity': 'sex', 'label': 'host'}, {'entity': 'disease progression', 'label': 'disease'}, {'entity': 'genetic composition', 'label': 'gene'}, {'entity': 'ACE I/D polymorphism', 'label': 'gene'}, {'entity': '3', 'label': 'virus'}, {'entity': 'structures', 'label': 'gene'}, {'entity': 'disruption', 'label': 'gene'}, {'entity': 'biological function', 'label': 'immune response'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'stimulating element', 'label': 'gene'}, {'entity': 'Nidovirales', 'label': 'virus'}, {'entity': 'Nidovirales', 'label': 'virus'}, {'entity': 'structures', 'label': 'pathway'}, {'entity': 'interaction', 'label': 'protein'}, {'entity': 'bases', 'label': 'protein'}, {'entity': 'deviations from the predicted structure', 'label': 'measurement'}, {'entity': 'A13456', 'label': 'gene'}, {'entity': '39', 'label': 'measurement'}, {'entity': 'terminal basepairs', 'label': 'gene'}, {'entity': 'canonical wobble G:U basepair', 'label': 'gene'}, {'entity': 'proton', 'label': 'protein'}, {'entity': 'secondary structure', 'label': 'pathway'}, {'entity': 'À1 PRF', 'label': 'gene'}, {'entity': 'fungal', 'label': 'organism'}, {'entity': 'formatted', 'label': 'method'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'constructs', 'label': 'virus'}, {'entity': 'significance', 'label': 'clinical trial'}, {'entity': 'group 1 coronavirus', 'label': 'virus'}, {'entity': 'group 2 coronavirus', 'label': 'virus'}, {'entity': 'groups', 'label': 'organization'}, {'entity': 'À1 PRF', 'label': 'protein'}, {'entity': 'replication competent', 'label': 'virus'}, {'entity': 'translation competent', 'label': 'disease'}, {'entity': 'group 1 coronavirus', 'label': 'virus'}, {'entity': 'kilocalories per mole', 'label': 'measurement'}, {'entity': 'complete medium', 'label': 'measurement'}, {'entity': 'Yeast cell', 'label': 'organism'}, {'entity': 'control plasmid', 'label': 'vaccine'}, {'entity': 'control construct', 'label': 'gene'}, {'entity': 'GACCCG', 'label': 'sequence'}, {'entity': 'Carlsbad, California', 'label': 'location'}, {'entity': 'spectra', 'label': 'diagnostic'}, {'entity': 'raw data', 'label': 'measurement'}, {'entity': 'taxonomic', 'label': 'disease'}, {'entity': 'Microbial Rosetta Stone', 'label': 'publication'}, {'entity': 'nomenclature', 'label': 'vocabulary'}, {'entity': 'taxonomic', 'label': 'measurement'}, {'entity': 'National Institute of Allergy and Infectious Disease', 'label': 'organization'}, {'entity': 'alternative name', 'label': 'vocabulary'}, {'entity': 'common name', 'label': 'term'}, {'entity': 'names', 'label': 'virus'}, {'entity': 'agent', 'label': 'disease causing agents'}, {'entity': 'taxonomic', 'label': 'vocabulary'}, {'entity': 'organism name', 'label': 'virus'}, {'entity': 'symbols key', 'label': 'diagram legend'}, {'entity': 'cellular life forms', 'label': 'organism'}, {'entity': 'sequence data', 'label': 'measurement'}, {'entity': 'data model', 'label': 'method'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'developed nations', 'label': 'country'}, {'entity': 'developed world', 'label': 'region'}, {'entity': 'Symbol key', 'label': 'disease'}, {'entity': 'Bacterial phylogeny', 'label': 'pathway'}, {'entity': 'agents', 'label': 'virus'}, {'entity': 'etiological agent', 'label': 'disease'}, {'entity': 'protist', 'label': 'disease'}, {'entity': 'gene sequence analysis', 'label': 'method'}, {'entity': 'antibiotic use in humans', 'label': 'disease'}, {'entity': 'infectious organism', 'label': 'virus'}, {'entity': '2003', 'label': 'year'}, {'entity': 'AIDS epidemic', 'label': 'disease'}, {'entity': 'grain', 'label': 'virus'}, {'entity': 'older children', 'label': 'disease'}, {'entity': 'infant', 'label': 'person'}, {'entity': 'virus families', 'label': 'virus'}, {'entity': 'biological weapon', 'label': 'treatment'}, {'entity': 'biological weapons', 'label': 'treatment'}, {'entity': 'biological source', 'label': 'virus'}, {'entity': 'outbreaks of disease', 'label': 'disease'}, {'entity': '2002', 'label': 'year'}, {'entity': 'chemical synthesis', 'label': 'method'}, {'entity': 'splicing', 'label': 'method'}, {'entity': 'Agent list', 'label': 'organization'}, {'entity': 'nomenclature', 'label': 'term'}, {'entity': 'branching', 'label': 'virus'}, {'entity': 'developing nations', 'label': 'region'}, {'entity': 'agents', 'label': 'disease'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'etiological agent', 'label': 'disease'}, {'entity': 'etiological agent', 'label': 'disease'}, {'entity': 'biological weapons', 'label': 'virus'}, {'entity': 'biological weapon', 'label': 'virus'}, {'entity': 'commission', 'label': 'event'}, {'entity': 'select agent', 'label': 'organization'}, {'entity': 'development', 'label': 'clinical trial'}, {'entity': 'literature analysis', 'label': 'method'}, {'entity': 'inhibitive', 'label': 'measurement'}, {'entity': '3CL(Pro)', 'label': 'protein'}, {'entity': '3CL(Pro)', 'label': 'protein'}, {'entity': 'Piscataway, NJ', 'label': 'location'}, {'entity': 'products', 'label': 'vaccine'}, {'entity': 'St Louis, MO', 'label': 'location'}, {'entity': 'St Louis, MO', 'label': 'location'}, {'entity': 'tested', 'label': 'method'}, {'entity': '3', 'label': 'measurement'}, {'entity': 'gallate group', 'label': 'measurement'}, {'entity': 'gallate group', 'label': 'measurement'}, {'entity': 'gallate group', 'label': 'protein'}, {'entity': 'intestinal tract', 'label': 'location'}, {'entity': 'structural motif', 'label': 'protein'}, {'entity': 'structural motif', 'label': 'protein'}, {'entity': 'stable', 'label': 'measurement'}, {'entity': 'self-cleaves', 'label': 'protein'}, {'entity': 'helical segments', 'label': 'protein'}, {'entity': 'enzymatic reaction', 'label': 'protein'}, {'entity': 'autosomal dyskeratosis congenita', 'label': 'disease'}, {'entity': 'biological functions', 'label': 'virus'}, {'entity': 'loop-stem', 'label': 'structure'}, {'entity': 'loop', 'label': 'protein'}, {'entity': 'secondary structure', 'label': 'structure'}, {'entity': 'formal vocabulary', 'label': 'vocabulary'}, {'entity': 'biological processes', 'label': 'pathway'}, {'entity': 'representation', 'label': 'method'}, {'entity': 'signaling pathway', 'label': 'pathway'}, {'entity': 'structured vocabulary', 'label': 'method'}, {'entity': 'vocabulary', 'label': 'ontology'}, {'entity': 'biological model', 'label': 'vocabulary'}, {'entity': 'terminology', 'label': 'method'}, {'entity': 'controlled vocabulary', 'label': 'vocabulary'}, {'entity': 'biological process', 'label': 'process'}, {'entity': 'structures', 'label': 'term'}, {'entity': 'unique identifier', 'label': 'measurement'}, {'entity': 'biological processes', 'label': 'pathway'}, {'entity': 'annotation', 'label': 'clinical trial'}, {'entity': 'inputted', 'label': 'measurement'}, {'entity': 'outputted', 'label': 'measurement'}, {'entity': 'class', 'label': 'method'}, {'entity': 'interactions', 'label': 'pathway'}, {'entity': 'attributes', 'label': 'method'}, {'entity': 'functions', 'label': 'gene product'}, {'entity': 'function', 'label': 'method'}, {'entity': 'cellular component', 'label': 'location'}, {'entity': 'biological model', 'label': 'treatment'}, {'entity': 'development', 'label': 'pathway'}, {'entity': 'SARA', 'label': 'protein'}, {'entity': 'accumulate', 'label': 'method'}, {'entity': 'proteasome', 'label': 'protein'}, {'entity': 'biological process domain', 'label': 'term'}, {'entity': 'less specific', 'label': 'term'}, {'entity': 'more specific', 'label': 'term'}, {'entity': 'Object-oriented terms', 'label': 'term'}, {'entity': 'diagram', 'label': 'method'}, {'entity': 'object-oriented diagram', 'label': 'diagram'}, {'entity': 'object function', 'label': 'method'}, {'entity': 'function', 'label': 'molecular function'}, {'entity': 'molecular function domain', 'label': 'ontology'}, {'entity': 'object functions', 'label': 'term'}, {'entity': 'The GO terms associated with the process, TGF-beta receptor complex assembly', 'label': 'term'}, {'entity': 'object function', 'label': 'method'}, {'entity': 'object function', 'label': 'method'}, {'entity': 'object function', 'label': 'method'}, {'entity': 'biological process', 'label': 'term'}, {'entity': 'objectoriented software', 'label': 'method'}, {'entity': 'sequence diagram', 'label': 'diagram'}, {'entity': 'functions', 'label': 'gene'}, {'entity': 'modeled', 'label': 'method'}, {'entity': 'molecular function domain', 'label': 'ontology'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'communicate', 'label': 'method'}, {'entity': 'sequence diagram', 'label': 'method'}, {'entity': 'protein heterodimerization', 'label': 'protein'}, {'entity': 'function call', 'label': 'method'}, {'entity': 'Boolean value', 'label': 'measurement'}, {'entity': 'events', 'label': 'event'}, {'entity': 'Biological process', 'label': 'pathway'}, {'entity': 'main event scenario', 'label': 'event'}, {'entity': 'dynamic architecture', 'label': 'method'}, {'entity': 'sequence diagram', 'label': 'diagram'}, {'entity': 'degradation', 'label': 'pathway'}, {'entity': 'biological process', 'label': 'process'}, {'entity': 'interactions', 'label': 'protein'}, {'entity': 'function', 'label': 'gene'}, {'entity': 'functions', 'label': 'gene'}, {'entity': 'compartment', 'label': 'gene'}, {'entity': 'cellular compartment', 'label': 'gene'}, {'entity': 'spatial representation', 'label': 'method'}, {'entity': 'paradigm', 'label': 'method'}, {'entity': 'vocabulary', 'label': 'gene'}, {'entity': 'biological process', 'label': 'biological process'}, {'entity': 'function', 'label': 'gene'}, {'entity': 'object-oriented framework', 'label': 'method'}, {'entity': 'biological networks', 'label': 'pathway'}, {'entity': 'function', 'label': 'gene'}, {'entity': 'biological process domain', 'label': 'ontology'}, {'entity': 'biological process', 'label': 'pathway'}, {'entity': 'process', 'label': 'pathway'}, {'entity': 'dynamics', 'label': 'measurement'}, {'entity': 'attributes', 'label': 'method'}, {'entity': 'temporal', 'label': 'measurement'}, {'entity': 'protein heterodimerization', 'label': 'protein'}, {'entity': 'heterodimerize', 'label': 'protein'}, {'entity': 'heterodimerize', 'label': 'protein'}, {'entity': 'function', 'label': 'protein'}, {'entity': 'computer scientists', 'label': 'person'}, {'entity': 'system', 'label': 'clinical trial'}, {'entity': 'chronologically order events', 'label': 'method'}, {'entity': 'process', 'label': 'clinical trial'}, {'entity': 'process', 'label': 'pathway'}, {'entity': 'annotation', 'label': 'method'}, {'entity': 'object-orientation', 'label': 'method'}, {'entity': 'human effort', 'label': 'measurement'}, {'entity': 'biological processes', 'label': 'pathway'}, {'entity': 'software engineering', 'label': 'method'}, {'entity': 'gene ontologies', 'label': 'gene'}, {'entity': 'card', 'label': 'method'}, {'entity': 'scope', 'label': 'measurement'}, {'entity': 'degradation', 'label': 'process'}, {'entity': 'requirement-gathering phase', 'label': 'method'}, {'entity': 'notation', 'label': 'method'}, {'entity': 'hidden', 'label': 'method'}, {'entity': 'dynamics', 'label': 'method'}, {'entity': 'UML notation', 'label': 'method'}, {'entity': 'sequence diagram', 'label': 'method'}, {'entity': 'class diagram', 'label': 'diagram'}, {'entity': 'set combination', 'label': 'method'}, {'entity': 'unsequenced', 'label': 'virus'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'origin', 'label': 'location'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'analytical tools', 'label': 'method'}, {'entity': 'human examination', 'label': 'method'}, {'entity': 'conserved sequence', 'label': 'gene'}, {'entity': 'non-repeated sequence', 'label': 'gene'}, {'entity': 'bases', 'label': 'measurement'}, {'entity': 'false negatives', 'label': 'measurement'}, {'entity': 'detection of SNPs', 'label': 'method'}, {'entity': 'chance', 'label': 'method'}, {'entity': 'genome alignments', 'label': 'measurement'}, {'entity': 'SNP forensics', 'label': 'method'}, {'entity': 'set', 'label': 'method'}, {'entity': 'phenotypic traits', 'label': 'trait'}, {'entity': 'variation', 'label': 'measurement'}, {'entity': 'bioinformatic software', 'label': 'tool'}, {'entity': 'Phylogram of SARS', 'label': 'diagram'}, {'entity': 'Phylogram of SARS', 'label': 'diagram'}, {'entity': 'operating system', 'label': 'software'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'bases', 'label': 'gene'}, {'entity': 'gestalt', 'label': 'consensus'}, {'entity': 'downstream', 'label': 'gene'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'regular expression pattern matching', 'label': 'method'}, {'entity': 'input sequence', 'label': 'measurement'}, {'entity': 'conserved sequence', 'label': 'sequence'}, {'entity': 'conserved sequence', 'label': 'gene'}, {'entity': 'repeat', 'label': 'gene'}, {'entity': 'chip', 'label': 'diagnostic'}, {'entity': 'sequence fragments', 'label': 'gene'}, {'entity': 'variable position', 'label': 'measurement'}, {'entity': 'haploblock', 'label': 'measurement'}, {'entity': 'discrimination', 'label': 'treatment'}, {'entity': 'value', 'label': 'measurement'}, {'entity': 'cut', 'label': 'method'}, {'entity': '3', 'label': 'measurement'}, {'entity': 'reaction conditions', 'label': 'treatment'}, {'entity': 'enzymes', 'label': 'protein'}, {'entity': 'character', 'label': 'gene'}, {'entity': 'character', 'label': 'gene'}, {'entity': 'genome group', 'label': 'organization'}, {'entity': 'demands', 'label': 'measurement'}, {'entity': 'combinatoric', 'label': 'method'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'groups', 'label': 'gene'}, {'entity': 'solution set', 'label': 'measurement'}, {'entity': 'genome groups', 'label': 'virus'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'others(j,A)', 'label': 'gene'}, {'entity': 'others(i,A)', 'label': 'gene'}, {'entity': 'solution set', 'label': 'measurement'}, {'entity': 'genome groups', 'label': 'gene'}, {'entity': 'step', 'label': 'method'}, {'entity': 'genome groups with the fewest genomes', 'label': 'gene'}, {'entity': 'genome groups with many genomes', 'label': 'virus'}, {'entity': 'genome group', 'label': 'gene'}, {'entity': 'character', 'label': 'gene'}, {'entity': 'simulated', 'label': 'method'}, {'entity': 'set number', 'label': 'measurement'}, {'entity': 'genome discrimination', 'label': 'measurement'}, {'entity': 'jump', 'label': 'parameter'}, {'entity': 'dnadist algorithm', 'label': 'method'}, {'entity': 'file', 'label': 'organization'}, {'entity': 'monoclonal Ab', 'label': 'drug'}, {'entity': 'host immune response', 'label': 'immune response'}, {'entity': 'intratracheal dsRNA', 'label': 'disease'}, {'entity': '20', 'label': 'measurement'}, {'entity': 'Minneapolis, MN', 'label': 'city'}, {'entity': 'San Diego, CA', 'label': 'location'}, {'entity': '24', 'label': 'publication'}, {'entity': 'apposed', 'label': 'method'}, {'entity': 'dewaxed', 'label': 'method'}, {'entity': 'San Ramon, CA', 'label': 'location'}, {'entity': 'Santa Cruz, CA', 'label': 'location'}, {'entity': 'Burlingame, CA', 'label': 'location'}, {'entity': 'Quantitative analysis of gene expression', 'label': 'method'}, {'entity': '7', 'label': 'measurement'}, {'entity': 'bronchi', 'label': 'anatomy'}, {'entity': 'separated', 'label': 'method'}, {'entity': 'West Grove, PA', 'label': 'location'}, {'entity': 'Gaithersburg, MD', 'label': 'location'}, {'entity': 'Winooski, VT', 'label': 'location'}, {'entity': 'Franklin Lakes, NJ', 'label': 'location'}, {'entity': 'San Jose, CA', 'label': 'location'}, {'entity': 'infiltrates', 'label': 'symptom'}, {'entity': 'dry', 'label': 'measurement'}, {'entity': '3', 'label': 'measurement'}, {'entity': 'mAb', 'label': 'protein'}, {'entity': '5F', 'label': 'measurement'}, {'entity': '7', 'label': 'clinical trial'}, {'entity': '7C', 'label': 'measurement'}, {'entity': '7B', 'label': 'measurement'}, {'entity': 'neutrophil infiltration', 'label': 'symptom'}, {'entity': 'host defense', 'label': 'host'}, {'entity': 'September 2003', 'label': 'date'}, {'entity': 'mononuclear cell', 'label': 'cell'}, {'entity': '6', 'label': 'measurement'}, {'entity': 'BALF', 'label': 'diagnostic'}, {'entity': 'Milwaukee, WI', 'label': 'location'}, {'entity': 'position', 'label': 'location'}, {'entity': 'ground glass opacification', 'label': 'disease'}, {'entity': 'inferior pulmonary vein', 'label': 'location'}, {'entity': 'summation of scores', 'label': 'measurement'}, {'entity': 'lobe', 'label': 'location'}, {'entity': 'San Jose, CA', 'label': 'location'}, {'entity': 'SEM', 'label': 'measurement'}, {'entity': 'abovedescribed symptoms', 'label': 'symptom'}, {'entity': 'lung field', 'label': 'location'}, {'entity': 'respiratory tract', 'label': 'location'}, {'entity': 'alveolar', 'label': 'location'}, {'entity': '2004', 'label': 'year'}, {'entity': 'T lymphocytes', 'label': 'cell'}, {'entity': 'CD8+ T cells', 'label': 'immune response'}, {'entity': 'outcome', 'label': 'clinical trial'}, {'entity': 'retrospective', 'label': 'clinical trial'}, {'entity': 'mammal', 'label': 'organism'}, {'entity': 'anchor positions', 'label': 'measurement'}, {'entity': 'tested', 'label': 'method'}, {'entity': 'training data set', 'label': 'measurement'}, {'entity': 'UNIX', 'label': 'environment'}, {'entity': 'input sequence', 'label': 'measurement'}, {'entity': 'line breaks', 'label': 'method'}, {'entity': 'carriage returns', 'label': 'method'}, {'entity': 'binding peptide', 'label': 'protein'}, {'entity': 'H2-D d', 'label': 'protein'}, {'entity': 'companion database', 'label': 'organization'}, {'entity': 'segment', 'label': 'gene'}, {'entity': 'taxonomic', 'label': 'gene'}, {'entity': '32', 'label': 'measurement'}, {'entity': 'reproduced', 'label': 'method'}, {'entity': '1999', 'label': 'year'}, {'entity': 'processing', 'label': 'method'}, {'entity': 'DNA sequence', 'label': 'measurement'}, {'entity': 'taxonomic', 'label': 'vocabulary'}, {'entity': 'genomic signature database', 'label': 'gene'}, {'entity': 'signature studies', 'label': 'clinical trial'}, {'entity': 'genomic signature database', 'label': 'database'}, {'entity': 'nucleotide sequences', 'label': 'measurement'}, {'entity': 'images', 'label': 'method'}, {'entity': 'toolbox', 'label': 'method'}, {'entity': 'companion database', 'label': 'tool'}, {'entity': 'companion database', 'label': 'organization'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'functions', 'label': 'method'}, {'entity': 'counterparts', 'label': 'gene'}, {'entity': 'SARS outbreak', 'label': 'event'}, {'entity': 'personal', 'label': 'person'}, {'entity': 'information technology', 'label': 'technology'}, {'entity': 'situation', 'label': 'event'}, {'entity': 'electronic', 'label': 'method'}, {'entity': 'synthesize', 'label': 'method'}, {'entity': 'Directive', 'label': 'organization'}, {'entity': 'limited-access', 'label': 'method'}, {'entity': 'April 3, 2003', 'label': 'date'}, {'entity': 'SARS steering committee', 'label': 'organization'}, {'entity': 'secure accessible', 'label': 'method'}, {'entity': 'password', 'label': 'method'}, {'entity': 'document distribution', 'label': 'method'}, {'entity': 'email link', 'label': 'method'}, {'entity': 'tracking', 'label': 'method'}, {'entity': 'watching', 'label': 'method'}, {'entity': 'debriefing session', 'label': 'event'}, {'entity': 'Space Shuttle', 'label': 'organization'}, {'entity': 'think-aloud', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': '53', 'label': 'measurement'}, {'entity': '2008', 'label': 'year'}, {'entity': 'web resources', 'label': 'organization'}, {'entity': 'forced-choice question', 'label': 'question'}, {'entity': 'links', 'label': 'vaccine'}, {'entity': '2008', 'label': 'year'}, {'entity': 'password', 'label': 'measurement'}, {'entity': 'personal barriers to accessing', 'label': 'virus'}, {'entity': 'Extranet access', 'label': 'organization'}, {'entity': 'Internet communication', 'label': 'method'}, {'entity': 'online communication', 'label': 'method'}, {'entity': 'administrator', 'label': 'person'}, {'entity': 'Internet site', 'label': 'environment'}, {'entity': 'Internet communication system', 'label': 'method'}, {'entity': 'privacy', 'label': 'disease'}, {'entity': 'SARS Steering Committee', 'label': 'organization'}, {'entity': '2008', 'label': 'year'}, {'entity': 'response to disasters', 'label': 'event'}, {'entity': 'chain of medical response', 'label': 'clinical trial'}, {'entity': 'March 2004', 'label': 'event'}, {'entity': 'bed', 'label': 'organization'}, {'entity': '1995', 'label': 'year'}, {'entity': '2008', 'label': 'year'}, {'entity': 'budgetary constraints', 'label': 'measurement'}, {'entity': 'logistical concerns', 'label': 'event'}, {'entity': 'critical care physicians', 'label': 'person'}, {'entity': 'sustainable disaster critical care response', 'label': 'treatment'}, {'entity': '2003', 'label': 'year'}, {'entity': 'healthcare professionals', 'label': 'person'}, {'entity': 'imperatives', 'label': 'clinical trial'}, {'entity': '2003', 'label': 'year'}, {'entity': 'discussion', 'label': 'event'}, {'entity': 'curriculum', 'label': 'clinical trial'}, {'entity': 'disaster planners', 'label': 'person'}, {'entity': 'knowledge collection', 'label': 'clinical trial'}, {'entity': 'time intensive', 'label': 'measurement'}, {'entity': 'adjunct', 'label': 'method'}, {'entity': 'skills', 'label': 'measurement'}, {'entity': 'disaster medical response', 'label': 'event'}, {'entity': 'execution', 'label': 'clinical trial'}, {'entity': 'results', 'label': 'measurement'}, {'entity': 'injuries', 'label': 'symptom'}, {'entity': 'personnel', 'label': 'person'}, {'entity': 'adaptation', 'label': 'method'}, {'entity': 'educational proposals', 'label': 'clinical trial'}, {'entity': 'disaster planning', 'label': 'clinical trial'}, {'entity': '2008', 'label': 'year'}, {'entity': 'economies of scale', 'label': 'method'}, {'entity': 'medical catastrophe', 'label': 'event'}, {'entity': 'conceptual elements', 'label': 'idea'}, {'entity': 'locally', 'label': 'location'}, {'entity': 'leadership', 'label': 'clinical trial'}, {'entity': 'modern medicine', 'label': 'organization'}, {'entity': 'limitations', 'label': 'treatment'}, {'entity': 'array format', 'label': 'method'}, {'entity': '1999', 'label': 'year'}, {'entity': '2002', 'label': 'year'}, {'entity': 'chip', 'label': 'method'}, {'entity': 'detect antigens', 'label': 'diagnostic'}, {'entity': 'image', 'label': 'diagnostic'}, {'entity': 'DNA replication', 'label': 'process'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'cardinal signature', 'label': 'signature'}, {'entity': 'detection and identification of viruses', 'label': 'diagnostic'}, {'entity': 'unique hybridization signature', 'label': 'method'}, {'entity': 'matrix effect', 'label': 'diagnostic'}, {'entity': 'CANARY sensor', 'label': 'method'}, {'entity': 'sensor', 'label': 'diagnostic'}, {'entity': 'drug profiling', 'label': 'method'}, {'entity': 'technology', 'label': 'method'}, {'entity': 'cell-omic', 'label': 'method'}, {'entity': 'biomimetic', 'label': 'protein'}, {'entity': 'science', 'label': 'method'}, {'entity': 'real-world application', 'label': 'clinical trial'}, {'entity': 'unique identifier', 'label': 'virus'}, {'entity': 'infection profiles', 'label': 'diagnostic'}, {'entity': 'capturing', 'label': 'method'}, {'entity': 'unrealistic', 'label': 'clinical trial'}, {'entity': 'response to infection', 'label': 'immune response'}, {'entity': 'structural rearrangements', 'label': 'virus'}, {'entity': 'noncytotoxic', 'label': 'drug'}, {'entity': 'confor-mation', 'label': 'protein'}, {'entity': 'DENV infection', 'label': 'disease'}, {'entity': 'Multivalent DENV vaccine', 'label': 'vaccine'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'Sindbis virus -SINV', 'label': 'virus'}, {'entity': 'stem domain', 'label': 'protein'}, {'entity': 'stem peptide', 'label': 'protein'}, {'entity': 'SEM', 'label': 'measurement'}, {'entity': 'domain III', 'label': 'protein'}, {'entity': 'monoclonal', 'label': 'drug'}, {'entity': 'class I', 'label': 'drug'}, {'entity': 'HIV inhibitor', 'label': 'drug'}, {'entity': 'San Francisco, CA', 'label': 'location'}, {'entity': 'Peptide stock solution', 'label': 'vaccine'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'Rockland, ME', 'label': 'location'}, {'entity': 'foci', 'label': 'disease'}, {'entity': 'Temecula, CA', 'label': 'location'}, {'entity': 'Minneapolis, MN', 'label': 'city'}, {'entity': 'assumptions', 'label': 'method'}, {'entity': 'model structure', 'label': 'method'}, {'entity': 'a priori', 'label': 'method'}, {'entity': 'assumptions', 'label': 'concept'}, {'entity': 'mathematical model', 'label': 'method'}, {'entity': '2001', 'label': 'year'}, {'entity': 'quantitative predictions', 'label': 'epidemiology'}, {'entity': 'model formulation', 'label': 'method'}, {'entity': 'models', 'label': 'clinical trial'}, {'entity': 'mathematical equations', 'label': 'publication'}, {'entity': 'latent class', 'label': 'disease'}, {'entity': 'infectious class', 'label': 'disease'}, {'entity': 'functional form', 'label': 'method'}, {'entity': 'class', 'label': 'method'}, {'entity': 'aggregation', 'label': 'process'}, {'entity': 'viral load data', 'label': 'measurement'}, {'entity': 'concerns', 'label': 'public health'}, {'entity': 'exponential', 'label': 'method'}, {'entity': '2003', 'label': 'year'}, {'entity': 'dynamics', 'label': 'epidemiology'}, {'entity': 'period', 'label': 'measurement'}, {'entity': 'viral life cycle dynamics', 'label': 'pathway'}, {'entity': 'model structure', 'label': 'method'}, {'entity': 'global control measures', 'label': 'treatment'}, {'entity': 'periods of fixed lengths', 'label': 'measurement'}, {'entity': 'classical exponentially distributed model', 'label': 'model'}, {'entity': 'English boarding school', 'label': 'location'}, {'entity': 'least squares error', 'label': 'measurement'}, {'entity': 'SEIR best-fit', 'label': 'method'}, {'entity': 'SIR best-fit', 'label': 'model'}, {'entity': 'exponentially distributed SIR model', 'label': 'model'}, {'entity': 'model structure', 'label': 'method'}, {'entity': 'transmission parameter', 'label': 'parameter'}, {'entity': 'uncertainty', 'label': 'epidemiology'}, {'entity': 'stimulator', 'label': 'virus'}, {'entity': 'pseudoknot stimulators', 'label': 'gene'}, {'entity': 'Z', 'label': 'gene'}, {'entity': 'non-commercial purposes', 'label': 'measurement'}, {'entity': 'host factors', 'label': 'host'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': 'viral RNA replication', 'label': 'virus'}, {'entity': 'reaction volume', 'label': 'measurement'}, {'entity': 'cooling', 'label': 'method'}, {'entity': 'incubated', 'label': 'method'}, {'entity': 'visualized', 'label': 'method'}, {'entity': 'Progema', 'label': 'organization'}, {'entity': 'base', 'label': 'protein'}, {'entity': 'scaffold', 'label': 'gene'}, {'entity': 'loop', 'label': 'gene'}, {'entity': 'transcription-coupled translation', 'label': 'method'}, {'entity': 'RNA attenuation', 'label': 'gene'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'non-shifted', 'label': 'protein'}, {'entity': 'schematic diagram', 'label': 'diagram'}, {'entity': 'downregulation', 'label': 'treatment'}, {'entity': 'inserts', 'label': 'clinical trial'}, {'entity': 'IBV pseudoknot', 'label': 'gene'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'artifact', 'label': 'vaccine'}, {'entity': 'conformations', 'label': 'disease'}, {'entity': 'junction geometry', 'label': 'pathway'}, {'entity': 'barley yellow dwarf virus', 'label': 'virus'}, {'entity': 'synthesis', 'label': 'method'}, {'entity': 'modulating', 'label': 'method'}, {'entity': 'stimulators', 'label': 'virus'}, {'entity': 'downstream determinants', 'label': 'gene'}, {'entity': 'destination information', 'label': 'measurement'}, {'entity': 'source information', 'label': 'measurement'}, {'entity': 'hardware', 'label': 'organization'}, {'entity': 'PCR prediction', 'label': 'diagnostic'}, {'entity': 't i', 'label': 'variable'}, {'entity': 'formula', 'label': 'method'}, {'entity': 'average method', 'label': 'method'}, {'entity': 'computation', 'label': 'method'}, {'entity': 'downstream', 'label': 'gene'}, {'entity': 'upstream', 'label': 'gene'}, {'entity': 'C++', 'label': 'method'}, {'entity': 'nucleotide code', 'label': 'vocabulary'}, {'entity': 'output', 'label': 'measurement'}, {'entity': 'output file', 'label': 'publication'}, {'entity': 'downstream', 'label': 'measurement'}, {'entity': 'digit', 'label': 'measurement'}, {'entity': 'product size limits', 'label': 'measurement'}, {'entity': 'base recognition', 'label': 'protein'}, {'entity': 'identity', 'label': 'measurement'}, {'entity': 'insertions', 'label': 'event'}, {'entity': 'C++', 'label': 'method'}, {'entity': '8a', 'label': 'gene'}, {'entity': '8b', 'label': 'gene'}, {'entity': 'WHO guidelines', 'label': 'organization'}, {'entity': \"T's\", 'label': 'gene'}, {'entity': 'genotypic reversion event', 'label': 'gene'}, {'entity': 'diversification of function', 'label': 'function'}, {'entity': 'concerted evolution', 'label': 'term'}, {'entity': 'illegitimate recombination mechanisms', 'label': 'gene'}, {'entity': 'A domain', 'label': 'protein'}, {'entity': 'primate', 'label': 'gene'}, {'entity': 'offspring', 'label': 'host'}, {'entity': 'offspring', 'label': 'host'}, {'entity': 'PSG evolution', 'label': 'gene'}, {'entity': 'functional motifs', 'label': 'protein'}, {'entity': 'PSG function', 'label': 'protein'}, {'entity': 'mouse A domain', 'label': 'gene'}, {'entity': 'publicly available sequences', 'label': 'measurement'}, {'entity': 'evolutionary tree', 'label': 'method'}, {'entity': 'tree', 'label': 'clinical trial'}, {'entity': 'gene conversion', 'label': 'gene'}, {'entity': 'N1 domain exon', 'label': 'gene'}, {'entity': '2011', 'label': 'year'}, {'entity': 'Box and whisker plots for Dayhoff PAM 250 scores', 'label': 'method'}, {'entity': 'mouse PSG', 'label': 'gene'}, {'entity': 'domain structure', 'label': 'measurement'}, {'entity': 'Domain name abbreviations', 'label': 'method'}, {'entity': 'domain', 'label': 'protein'}, {'entity': '3', 'label': 'measurement'}, {'entity': \"spider's web\", 'label': 'diagram'}, {'entity': 'human PSG', 'label': 'gene'}, {'entity': '5', 'label': 'measurement'}, {'entity': 'branching', 'label': 'pathway'}, {'entity': '10', 'label': 'figure'}, {'entity': 'whisker plot', 'label': 'diagram'}, {'entity': '11', 'label': 'measurement'}, {'entity': 'data point', 'label': 'measurement'}, {'entity': 'values', 'label': 'measurement'}, {'entity': 'face', 'label': 'gene'}, {'entity': '7A', 'label': 'figure'}, {'entity': 'pairwise-alignment', 'label': 'measurement'}, {'entity': 'substitution', 'label': 'virus'}, {'entity': 'domain', 'label': 'measurement'}, {'entity': 'sequence space', 'label': 'measurement'}, {'entity': 'representation', 'label': 'measurement'}, {'entity': 'human N domain', 'label': 'protein'}, {'entity': 'amino acid sequence', 'label': 'protein'}, {'entity': 'd n/d s ratio', 'label': 'measurement'}, {'entity': 'amino acid sequence', 'label': 'protein'}, {'entity': 'lineages', 'label': 'organism group'}, {'entity': 'integrin-associated tetraspanin', 'label': 'protein'}, {'entity': 'bracketed', 'label': 'method'}, {'entity': 'primate', 'label': 'organism'}, {'entity': 'primate', 'label': 'gene'}, {'entity': 'parent-offspring conflict', 'label': 'concept'}, {'entity': 'model system', 'label': 'gene'}, {'entity': 'FTP', 'label': 'organization'}, {'entity': 'search pattern', 'label': 'method'}, {'entity': 'Web based', 'label': 'method'}, {'entity': 'Nucleotide alignment', 'label': 'measurement'}, {'entity': 'tree-building', 'label': 'method'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': 'parainfluenza virus 3', 'label': 'virus'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': 'parainfluenza virus 3', 'label': 'virus'}, {'entity': '8°C freezer', 'label': 'measurement'}, {'entity': 'generating random numbers', 'label': 'method'}, {'entity': 'season', 'label': 'event'}, {'entity': 'March 2000', 'label': 'event'}, {'entity': 'reverse-transcribed', 'label': 'method'}, {'entity': 'SYBR Green intercalation', 'label': 'measurement'}, {'entity': 'NPS', 'label': 'measurement'}, {'entity': 'Cary, North Carolina', 'label': 'location'}, {'entity': 'human coronavirus', 'label': 'virus'}, {'entity': 'outpatient', 'label': 'location'}, {'entity': 'onset of disease', 'label': 'symptom'}, {'entity': 'low viral load', 'label': 'virus'}, {'entity': '2000', 'label': 'year'}, {'entity': 'winter', 'label': 'event'}, {'entity': 'months', 'label': 'transmission'}, {'entity': 'positives', 'label': 'measurement'}, {'entity': 'spread worldwide', 'label': 'transmission'}, {'entity': 'different parts of the world', 'label': 'location'}, {'entity': 'different parts of the world', 'label': 'location'}, {'entity': 'different parts of the world', 'label': 'location'}, {'entity': 'different parts of the world', 'label': 'location'}, {'entity': 'different parts of the world', 'label': 'location'}, {'entity': 'human respiratory virus', 'label': 'virus'}, {'entity': 'Severe Acute Respiratory Syndrome coronavirus', 'label': 'virus'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'gated', 'label': 'method'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'cell cycle', 'label': 'immune response'}, {'entity': 'G0/G1 phase', 'label': 'phase'}, {'entity': '2005', 'label': 'year'}, {'entity': 'G0/G1 phase', 'label': 'pathway'}, {'entity': 'cell cycle', 'label': 'treatment'}, {'entity': 'cell cycle', 'label': 'pathway'}, {'entity': 'retrovirus infection', 'label': 'disease'}, {'entity': 'glycosylation-blocked receptors', 'label': 'protein'}, {'entity': 'non-specific enhancement', 'label': 'clinical trial'}, {'entity': 'Glycosylation-specific enhancement', 'label': 'method'}, {'entity': 'peptide N-glycosidase F', 'label': 'protein'}, {'entity': 'replated', 'label': 'method'}, {'entity': 'experiment', 'label': 'clinical trial'}, {'entity': 'Berkeley, California', 'label': 'location'}, {'entity': 'hypothesis', 'label': 'virus'}, {'entity': 'infect cells', 'label': 'virus'}, {'entity': \"Dulbecco's\", 'label': 'measurement'}, {'entity': 'dishes', 'label': 'vessel'}, {'entity': 'Eugene, OR', 'label': 'location'}, {'entity': 'St Louis, MO', 'label': 'location'}, {'entity': 'lysed', 'label': 'method'}, {'entity': 'New', 'label': 'vaccine'}, {'entity': 'sequence dependence', 'label': 'measurement'}, {'entity': 'DNA duplex', 'label': 'structure'}, {'entity': 'O', 'label': 'measurement'}, {'entity': 'modulate', 'label': 'method'}, {'entity': 'deprotection', 'label': 'method'}, {'entity': 'two-state model', 'label': 'model'}, {'entity': 'complementary RNA', 'label': 'protein'}, {'entity': '37', 'label': 'measurement'}, {'entity': 'conformations', 'label': 'measurement'}, {'entity': 'predicted thermodynamics', 'label': 'measurement'}, {'entity': 'selfcomplementary', 'label': 'diagnostic'}, {'entity': 'substitution', 'label': 'method'}, {'entity': '2 0 -O-methyl RNA/RNA', 'label': 'measurement'}, {'entity': 'stabilities', 'label': 'measurement'}, {'entity': 'AU pair', 'label': 'gene'}, {'entity': 'kcal/mol', 'label': 'measurement'}, {'entity': 'O', 'label': 'measurement'}, {'entity': \"5' terminal LNA\", 'label': 'measurement'}, {'entity': 'tight binding', 'label': 'method'}, {'entity': 'O', 'label': 'measurement'}, {'entity': 'kcal/mol', 'label': 'measurement'}, {'entity': 'rigidity', 'label': 'measurement'}, {}, {'entity': 'mucus simulant', 'label': 'method'}, {'entity': 'ill health', 'label': 'disease'}, {'entity': 'recent outbreak', 'label': 'event'}, {'entity': 'providing care', 'label': 'treatment'}, {'entity': 'peacetime', 'label': 'event'}, {'entity': 'respiratory protection equipment', 'label': 'device'}, {'entity': '2002', 'label': 'year'}, {'entity': 'secretory', 'label': 'disease'}, {'entity': 'Schematic diagram of the macromolecules of mucus with the different types of crosslinks', 'label': 'diagram'}, {'entity': 'ionic interaction', 'label': 'method'}, {'entity': 'ocular', 'label': 'disease'}, {'entity': 'indirect', 'label': 'treatment'}, {'entity': 'mucus simulant', 'label': 'diagnostic'}, {'entity': 'mucus clearance', 'label': 'disease'}, {'entity': 'Mucus simulant', 'label': 'diagnostic'}, {'entity': 'pharynx', 'label': 'anatomy'}, {'entity': 'eyepiece', 'label': 'measurement'}, {'entity': 'musculature', 'label': 'treatment'}, {'entity': 'circle', 'label': 'symptom'}, {'entity': 'muzzle', 'label': 'measurement'}, {'entity': 'valve', 'label': 'device'}, {'entity': 'dehydrated', 'label': 'method'}, {'entity': 'scanned', 'label': 'method'}, {'entity': '2009', 'label': 'year'}, {'entity': 'high molecular weight dextran', 'label': 'drug'}, {'entity': 'crosslinking', 'label': 'measurement'}, {'entity': 'p < 0.01', 'label': 'measurement'}, {'entity': 'mucus transport', 'label': 'measurement'}, {'entity': 'cough maneuver', 'label': 'method'}, {'entity': 'corridor', 'label': 'location'}, {'entity': 'airflow interaction', 'label': 'method'}, {'entity': 'model system', 'label': 'clinical trial'}, {'entity': 'respiratory tract', 'label': 'treatment'}, {'entity': 'Aerosolization and dispersion pattern effects for sodium tetraborate crosslinked mucous gel simulants', 'label': 'measurement'}, {'entity': 'Aerosol pattern', 'label': 'measurement'}, {'entity': 'crosslink', 'label': 'protein'}, {'entity': 'palate', 'label': 'location'}, {'entity': 'transmission prevention', 'label': 'treatment'}, {'entity': 'causative organism', 'label': 'organism'}, {'entity': 'perishable', 'label': 'measurement'}, {'entity': 'respiratory disease management', 'label': 'treatment'}, {'entity': 'declare', 'label': 'method'}, {'entity': 'determinants of population health', 'label': 'epidemiology'}, {'entity': 'poor', 'label': 'host'}, {'entity': 'rich', 'label': 'person'}, {'entity': 'framework', 'label': 'organization'}, {'entity': 'economic', 'label': 'measurement'}, {'entity': 'causality', 'label': 'measurement'}, {'entity': 'health determinants', 'label': 'measurement'}, {'entity': 'causal chain', 'label': 'method'}, {'entity': 'distal', 'label': 'treatment'}, {'entity': 'transnational structures', 'label': 'organization'}, {'entity': 'ignorance', 'label': 'measurement'}, {'entity': 'economic development', 'label': 'measurement'}, {'entity': 'feedback', 'label': 'clinical trial'}, {'entity': 'distal determinants', 'label': 'concept'}, {'entity': '1999', 'label': 'year'}, {'entity': 'exchange of experiences', 'label': 'event'}, {'entity': 'cultural', 'label': 'environment'}, {'entity': 'Global environmental threats to ecosystems', 'label': 'environment'}, {'entity': 'economic benefits', 'label': 'measurement'}, {'entity': 'inequities between and within countries', 'label': 'measurement'}, {'entity': 'Kraay', 'label': 'person'}, {'entity': 'Dollar', 'label': 'organization'}, {'entity': 'economic', 'label': 'measurement'}, {'entity': '1980', 'label': 'year'}, {'entity': '1950', 'label': 'year'}, {'entity': '1997', 'label': 'year'}, {'entity': 'manufactured goods', 'label': 'product'}, {'entity': 'globalising', 'label': 'method'}, {'entity': 'travel', 'label': 'event'}, {'entity': '1970', 'label': 'year'}, {'entity': 'intolerance', 'label': 'symptom'}, {'entity': 'factors', 'label': 'variable'}, {'entity': 'social inequity', 'label': 'event'}, {'entity': 'social activities', 'label': 'event'}, {'entity': 'social connection', 'label': 'virus'}, {'entity': 'social divide', 'label': 'event'}, {'entity': 'global communication', 'label': 'method'}, {'entity': 'New technologies', 'label': 'method'}, {'entity': 'write', 'label': 'person'}, {'entity': 'computer graphics', 'label': 'method'}, {'entity': 'goods and services', 'label': 'measurement'}, {'entity': 'factors', 'label': 'factor'}, {'entity': 'economic model', 'label': 'method'}, {'entity': 'private good', 'label': 'disease'}, {'entity': 'developing', 'label': 'country'}, {'entity': 'social exclusion', 'label': 'event'}, {'entity': 'international law', 'label': 'treatment'}, {'entity': 'ideas', 'label': 'information'}, {'entity': 'unhealthy diet', 'label': 'disease'}, {'entity': 'choices', 'label': 'vaccine'}, {'entity': 'advertising', 'label': 'method'}, {'entity': 'tobacco consumption', 'label': 'disease'}, {'entity': 'tobacco consumption', 'label': 'disease'}, {'entity': 'health effects of different lifestyle options', 'label': 'disease'}, {'entity': 'legislation', 'label': 'vaccine'}, {'entity': 'health effects of globalisation', 'label': 'epidemiology'}, {'entity': '1957', 'label': 'year'}, {'entity': 'goods', 'label': 'transmission'}, {'entity': 'speed of responses', 'label': 'measurement'}, {'entity': 'agricultural sectors', 'label': 'organization'}, {'entity': 'Alcohol use', 'label': 'disease'}, {'entity': '2001', 'label': 'year'}, {'entity': 'food production', 'label': 'measurement'}, {'entity': 'self-sufficient', 'label': 'country'}, {'entity': 'shocks', 'label': 'event'}, {'entity': '1966', 'label': 'year'}, {'entity': '1996', 'label': 'year'}, {'entity': 'supply', 'label': 'measurement'}, {'entity': 'knowledge', 'label': 'method'}, {'entity': 'market competition', 'label': 'method'}, {'entity': 'socially', 'label': 'measurement'}, {}, {'entity': 'commodities', 'label': 'object'}, {'entity': 'knowledge', 'label': 'measurement'}, {'entity': 'sustainable development', 'label': 'concept'}, {'entity': 'conceptual framework', 'label': 'method'}, {'entity': 'globalisation and health', 'label': 'topic'}, {'entity': 'science', 'label': 'method'}, {'entity': 'beneficial', 'label': 'measurement'}, {'entity': 'all but inevitable', 'label': 'event'}, {'entity': 'Another influenza pandemic', 'label': 'event'}, {'entity': '2008', 'label': 'year'}, {'entity': 'caring for sick relatives', 'label': 'symptom'}, {'entity': 'emerging infectious disease threats', 'label': 'disease'}, {'entity': 'medical practitioners', 'label': 'person'}, {'entity': '2001', 'label': 'year'}, {'entity': 'output', 'label': 'measurement'}, {'entity': 'a workforce', 'label': 'organization'}, {'entity': 'week', 'label': 'measurement'}, {'entity': 'medical practitioners', 'label': 'person'}, {'entity': '4', 'label': 'number'}, {'entity': 'six-week period', 'label': 'measurement'}, {'entity': 'stockpile', 'label': 'treatment'}, {'entity': 'clinical case', 'label': 'disease'}, {'entity': 'rapid action', 'label': 'method'}, {'entity': 'telephone information service', 'label': 'service'}, {'entity': 'website', 'label': 'organization'}, {'entity': 'health workers', 'label': 'person'}, {'entity': 'nosocomial outbreak', 'label': 'event'}, {'entity': 'professional obligations', 'label': 'concept'}, {'entity': 'care for sick relatives', 'label': 'activity'}, {'entity': 'fear of infection', 'label': 'symptom'}, {'entity': 'emerging infectious threats', 'label': 'virus'}, {'entity': 'HSV replication', 'label': 'process'}, {'entity': 'host defence', 'label': 'immune response'}, {'entity': 'take over', 'label': 'treatment'}, {'entity': 'invading virus', 'label': 'virus'}, {'entity': 'conflict', 'label': 'host'}, {'entity': 'cleavage sites', 'label': 'pathway'}, {'entity': 'viral particles', 'label': 'virus'}, {'entity': 'viral DNA', 'label': 'gene'}, {'entity': 'viral particle', 'label': 'virus'}, {'entity': '53', 'label': 'publication'}, {'entity': 'perinuclear space', 'label': 'location'}, {'entity': 're-envelopment', 'label': 'virus'}, {'entity': 'sensory nerve ganglia', 'label': 'host'}, {'entity': '1967', 'label': 'year'}, {'entity': 'pregnant women', 'label': 'person'}, {'entity': 'CNS', 'label': 'disease'}, {'entity': 'high-dose acyclovir', 'label': 'treatment'}, {'entity': 'high-dose acyclovir', 'label': 'drug'}, {'entity': 'symptoms of infection', 'label': 'symptom'}, {'entity': 'symptomatic infection', 'label': 'disease'}, {'entity': 'phagocyte', 'label': 'cell'}, {'entity': 'circulation', 'label': 'measurement'}, {'entity': 'chemically simple', 'label': 'measurement'}, {'entity': 'stimulating substances', 'label': 'protein'}, {'entity': 'proliferation', 'label': 'pathway'}, {'entity': 'factors', 'label': 'vocabulary'}, {'entity': 'biological activity', 'label': 'disease'}, {'entity': 'mnemonic', 'label': 'method'}, {'entity': 'families', 'label': 'protein'}, {'entity': 'factors', 'label': 'protein'}, {'entity': 'stimulus', 'label': 'virus'}, {'entity': 'importance', 'label': 'measurement'}, {'entity': 'expression patterns', 'label': 'pathway'}, {'entity': 'subunits', 'label': 'protein'}, {'entity': 'effects', 'label': 'immune response'}, {'entity': 'ligand superfamily', 'label': 'protein'}, {'entity': 'ligand superfamily', 'label': 'protein'}, {'entity': 'ligand superfamily', 'label': 'protein'}, {'entity': 'transmembrane pro-TNF', 'label': 'protein'}, {'entity': 'cleaved', 'label': 'protein'}, {'entity': 'inhibitor-κB', 'label': 'protein'}, {'entity': 'evolved', 'label': 'immune response'}, {'entity': '5', 'label': 'gene'}, {'entity': 'signalling mechanisms', 'label': 'pathway'}, {'entity': 'JAK/STAT pathway', 'label': 'pathway'}, {'entity': 'IL-12 family', 'label': 'protein'}, {'entity': 'JAK/STAT-pathway', 'label': 'pathway'}, {'entity': 'resistance of HSV infections', 'label': 'disease'}, {'entity': 'cutaneous', 'label': 'disease'}, {'entity': 'β', 'label': 'gene'}, {'entity': 'macocyte', 'label': 'cell'}, {'entity': 'mediate signals', 'label': 'pathway'}, {'entity': 'HSV encephalitis', 'label': 'disease'}, {'entity': 'regulatory signals', 'label': 'pathway'}, {'entity': 'IFN-α/β system', 'label': 'protein'}, {'entity': 'HSV replication', 'label': 'disease'}, {'entity': 'trait', 'label': 'gene'}, {'entity': 'inhibitor of HSV replication', 'label': 'virus'}, {'entity': 'strains', 'label': 'host'}, {'entity': 'viral host protein synthesis', 'label': 'protein'}, {'entity': 'extrinsic', 'label': 'method'}, {'entity': 'differentiation', 'label': 'pathway'}, {'entity': 'killing', 'label': 'virus'}, {'entity': 'antiproliferative', 'label': 'drug'}, {'entity': 'effect', 'label': 'virus'}, {'entity': 'pathways', 'label': 'pathway'}, {'entity': 'consequences', 'label': 'symptom'}, {'entity': 'inflammatory cell', 'label': 'cell'}, {'entity': 'nonspecific response to the virus', 'label': 'immune response'}, {'entity': 'signalling pathway', 'label': 'pathway'}, {'entity': 'effect', 'label': 'clinical trial'}, {'entity': '3', 'label': 'figure'}, {'entity': 'innate IFN-γ production', 'label': 'protein'}, {'entity': 'redundancy', 'label': 'virus'}, {'entity': 'controlled', 'label': 'gene'}, {'entity': 'transcriptional repressor', 'label': 'protein'}, {'entity': \"Theiler's murine encephalomyelitis virus\", 'label': 'virus'}, {'entity': 'signalling mechanisms', 'label': 'pathway'}, {'entity': 'declare', 'label': 'method'}, {'entity': 'peer referral', 'label': 'method'}, {'entity': 'drug injecting', 'label': 'treatment'}, {'entity': 'HIV transmission', 'label': 'transmission'}, {'entity': '1996', 'label': 'year'}, {'entity': 'official and entirely legal movement', 'label': 'event'}, {'entity': 'crossing point', 'label': 'location'}, {'entity': 'town', 'label': 'location'}, {'entity': 'village', 'label': 'location'}, {'entity': 'sell drugs', 'label': 'drug'}, {'entity': 'Drug dealer', 'label': 'person'}, {'entity': 'employment', 'label': 'occupation'}, {'entity': 'find work', 'label': 'event'}, {'entity': 'nationality', 'label': 'measurement'}, {'entity': 'cross-border movement', 'label': 'event'}, {'entity': 'peer recruitment', 'label': 'method'}, {'entity': 'gathering', 'label': 'event'}, {'entity': 'probability proportional to size', 'label': 'method'}, {'entity': 'IDU', 'label': 'person'}, {'entity': 'unique identifier', 'label': 'measurement'}, {'entity': '2006', 'label': 'year'}, {'entity': 'unique identifier', 'label': 'method'}, {'entity': 'HIV-positive', 'label': 'disease'}, {'entity': '1822', 'label': 'measurement'}, {'entity': 'subjects', 'label': 'person'}, {'entity': 'police pressure', 'label': 'disease'}, {'entity': 'needle', 'label': 'vaccine'}, {'entity': 'groups', 'label': 'organization'}, {'entity': 'injection equipment', 'label': 'vaccine'}, {'entity': 'needle', 'label': 'vaccine'}, {'entity': 'clinically', 'label': 'clinical trial'}, {'entity': 'SARS CoV infection', 'label': 'disease'}, {'entity': 'SARS CoV infection', 'label': 'disease'}, {'entity': '5', 'label': 'figure'}, {'entity': 'endoglycosidase H sensitive', 'label': 'protein'}, {'entity': 'SARS-CoV interaction', 'label': 'disease'}, {'entity': 'lysomotropic', 'label': 'drug'}, {'entity': 'CQ', 'label': 'drug'}, {'entity': '9', 'label': 'measurement'}, {'entity': '9', 'label': 'measurement'}, {'entity': '2008', 'label': 'publication'}, {'entity': 'SARS-CoV replication', 'label': 'virus'}, {'entity': 'inhibiting the infection', 'label': 'treatment'}, {'entity': \"Dulbecco's modified Eagle medium\", 'label': 'measurement'}, {'entity': 'declare', 'label': 'method'}, {'entity': 'binding to ACE2', 'label': 'protein'}, {'entity': 'neutralizing', 'label': 'protein'}, {'entity': 'SCV', 'label': 'protein'}, {'entity': 'functional', 'label': 'method'}, {'entity': 'Se', 'label': 'protein'}, {'entity': 'interactions', 'label': 'virus'}, {'entity': 'interaction', 'label': 'virus'}, {'entity': 'Expression and binding of soluble S fragments containing the RBD', 'label': 'method'}, {'entity': 'Fc', 'label': 'protein'}, {'entity': 'bound', 'label': 'method'}, {'entity': 'bound', 'label': 'protein'}, {'entity': 'asparagine', 'label': 'protein'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'probe radius', 'label': 'measurement'}, {'entity': 'Glycosylation of S fragment containing the RBD', 'label': 'protein'}, {'entity': 'Effects of glycosylation on expression and binding of RBD-containing fragments', 'label': 'measurement'}, {'entity': 'interactions', 'label': 'protein'}, {'entity': 'secondary structure', 'label': 'protein'}, {'entity': 'strains', 'label': 'virus'}, {'entity': 'domain', 'label': 'protein'}, {'entity': 'cyan', 'label': 'color'}, {'entity': 'sites', 'label': 'protein'}, {'entity': 'sites', 'label': 'protein'}, {'entity': 'Mapping of the S RBD mutants on the structure', 'label': 'method'}, {'entity': 'molecular surface diagram', 'label': 'diagram'}, {'entity': 'solid surface diagram', 'label': 'diagram'}, {'entity': 'surface diagram', 'label': 'diagram'}, {'entity': 'beta sheets', 'label': 'protein'}, {'entity': 'on rate constant', 'label': 'measurement'}, {'entity': 'interaction', 'label': 'pathway'}, {'entity': '357', 'label': 'protein'}, {'entity': '6', 'label': 'number'}, {'entity': 'sites', 'label': 'pathway'}, {'entity': 'Boston, Massachusetts', 'label': 'city'}, {'entity': 'Bethesda, MD', 'label': 'location'}, {'entity': 'Carlsbad, CA', 'label': 'city'}, {'entity': 'La Jolla, CA', 'label': 'location'}, {'entity': 'Carlsbad, CA', 'label': 'location'}, {'entity': 'Indianapolis, IN', 'label': 'city'}, {'entity': 'SARS RBD fragment', 'label': 'protein'}, {'entity': 'declare', 'label': 'method'}, {'entity': 'microbial communities', 'label': 'organism'}, {'entity': 'sequence space', 'label': 'measurement'}, {'entity': 'sequence space', 'label': 'measurement'}, {'entity': 'viral detection and discovery', 'label': 'diagnostic'}, {'entity': 'viral detection microarray', 'label': 'method'}, {'entity': 'Free energy of hybridization', 'label': 'measurement'}, {'entity': 'column vector', 'label': 'measurement'}, {'entity': 'intensity vector normalization', 'label': 'method'}, {'entity': 'training dataset', 'label': 'dataset'}, {'entity': 'virus profile', 'label': 'virus'}, {'entity': 'interfamily separation', 'label': 'measurement'}, {'entity': 'viral energy profile', 'label': 'virus'}, {'entity': 'cross-hybridizing', 'label': 'vocabulary'}, {'entity': 'distributions', 'label': 'measurement'}, {'entity': 'distributions', 'label': 'measurement'}, {'entity': 'RSV-positive samples', 'label': 'disease'}, {'entity': 'masked', 'label': 'method'}, {'entity': 'identify', 'label': 'method'}, {'entity': 'sequence information', 'label': 'measurement'}, {'entity': 'hybridizing', 'label': 'method'}, {'entity': 'signal', 'label': 'measurement'}, {'entity': 'serotypes', 'label': 'virus'}, {'entity': 'nucleic acid complexity', 'label': 'disease'}, {'entity': 'data scientist', 'label': 'person'}, {'entity': 'positives', 'label': 'measurement'}, {'entity': 'negatives', 'label': 'measurement'}, {'entity': 'verification', 'label': 'method'}, {'entity': 'assessment', 'label': 'clinical trial'}, {'entity': 'taxonomy hierarchy', 'label': 'method'}, {'entity': 'true predictions', 'label': 'clinical trial'}, {'entity': 'second iteration', 'label': 'clinical trial'}, {'entity': 'estimated', 'label': 'method'}, {'entity': 'three iterations', 'label': 'measurement'}, {'entity': 'viral discovery', 'label': 'disease'}, {'entity': 'biochemical pathways', 'label': 'pathway'}, {'entity': 'Valencia, CA', 'label': 'location'}, {'entity': 'Hanover, MD', 'label': 'location'}, {'entity': 'clinical hybridization', 'label': 'measurement'}, {'entity': 'energy profile matrix', 'label': 'measurement'}, {'entity': 'Energy vector', 'label': 'measurement'}, {'entity': 'non-zero energy', 'label': 'measurement'}, {'entity': '2012', 'label': 'year'}, {'entity': 'second iteration', 'label': 'method'}, {'entity': 'first iteration', 'label': 'method'}, {'entity': 'Java', 'label': 'method'}, {}, {'entity': 'privacy', 'label': 'measurement'}, {'entity': 'desktop application', 'label': 'method'}, {'entity': 'July 2005', 'label': 'event'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'RSS', 'label': 'method'}, {'entity': 'geographic metadata', 'label': 'measurement'}, {'entity': 'geographic metadata', 'label': 'measurement'}, {'entity': 'geographic metadata', 'label': 'measurement'}, {'entity': 'Web browser', 'label': 'tool'}, {'entity': 'online atlas', 'label': 'publication'}, {'entity': 'November 1999', 'label': 'date'}, {'entity': 'Microsoft Virtual Earth', 'label': 'organization'}, {'entity': 'JavaScript', 'label': 'method'}, {'entity': 'site-specific', 'label': 'measurement'}, {'entity': 'SHA coordinates', 'label': 'location'}, {'entity': 'Google Maps API', 'label': 'organization'}, {'entity': 'XML', 'label': 'method'}, {'entity': 'Web browser', 'label': 'method'}, {'entity': 'location data', 'label': 'measurement'}, {'entity': 'Web user', 'label': 'person'}, {'entity': 'desktop application', 'label': 'tool'}, {'entity': 'programming', 'label': 'method'}, {'entity': 'Web information sharing', 'label': 'method'}, {'entity': 'August 2005', 'label': 'event'}, {'entity': 'users', 'label': 'person'}, {'entity': '2003', 'label': 'year'}, {'entity': 'illegal activity', 'label': 'event'}, {'entity': 'national security', 'label': 'concept'}, {'entity': 'Internet GIS', 'label': 'organization'}, {'entity': 'system', 'label': 'organization'}, {'entity': 'map copyright', 'label': 'publication'}, {'entity': 'geo-data', 'label': 'measurement'}, {'entity': 'geo-data', 'label': 'measurement'}, {'entity': 'Google Earth Pro', 'label': 'organization'}, {'entity': 'visual language of geography', 'label': 'method'}]\n",
      "<class 'str'>\n",
      "<class 'dict'>\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "# Convert each string in the subject_labels list to a dictionary\n",
    "subject_labels_dicts = [json.loads(label) for label in subject_labels]\n",
    "\n",
    "# Convert each string in the object_labels list to a dictionary\n",
    "object_labels_dicts = [json.loads(label) for label in object_labels]\n",
    "\n",
    "# Print the converted lists of dictionaries\n",
    "print(\"Subject Labels Dicts:\", subject_labels_dicts)\n",
    "print(\"Object Labels Dicts:\", object_labels_dicts)\n",
    "print(type(subject_labels_dicts[0][\"entity\"]))\n",
    "print(type(object_labels_dicts[0]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Subject labels saved to 'output_from_using_REBEL/subject_labels.json'\n",
      "Object labels saved to 'output_from_using_REBEL/object_labels.json'\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "# Save the dictionaries to separate JSON files\n",
    "with open('output_from_using_REBEL/subject_labels.json', 'w') as f:\n",
    "    json.dump(subject_labels_dicts, f, indent=4)\n",
    "\n",
    "with open('output_from_using_REBEL/object_labels.json', 'w') as f:\n",
    "    json.dump(object_labels_dicts, f, indent=4)\n",
    "\n",
    "print(\"Subject labels saved to 'output_from_using_REBEL/subject_labels.json'\")\n",
    "print(\"Object labels saved to 'output_from_using_REBEL/object_labels.json'\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Updated triples with matched labels saved to 'output_from_using_REBEL/Final_rebel_triples_labels.csv'\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import json\n",
    "\n",
    "# Load the triples and entity label data\n",
    "triples_df = pd.read_csv('output_from_using_REBEL/rebel_triples_labels.csv')\n",
    "\n",
    "# Load the subject and object labels from JSON files\n",
    "with open('output_from_using_REBEL/subject_labels.json', 'r') as f:\n",
    "    subject_labels_dicts = json.load(f)\n",
    "\n",
    "with open('output_from_using_REBEL/object_labels.json', 'r') as f:\n",
    "    object_labels_dicts = json.load(f)\n",
    "\n",
    "# Create dictionaries from the subject and object labels\n",
    "subject_label_dict = {entry['entity'].strip().lower(): entry['label'].strip() for entry in subject_labels_dicts if 'entity' in entry}\n",
    "object_label_dict = {entry['entity'].strip().lower(): entry['label'].strip() for entry in object_labels_dicts if 'entity' in entry}\n",
    "\n",
    "# Function to get the entity label\n",
    "def get_entity_label(entity, label_dict):\n",
    "    return label_dict.get(entity.strip().lower(), 'unknown')\n",
    "\n",
    "# Fill NaN values with an empty string\n",
    "triples_df['subject'] = triples_df['subject'].fillna('')\n",
    "triples_df['object'] = triples_df['object'].fillna('')\n",
    "\n",
    "# Update the unknown labels in the triples DataFrame\n",
    "triples_df['subject_label'] = triples_df.apply(\n",
    "    lambda row: get_entity_label(row['subject'], subject_label_dict) if row['subject_label'] == 'unknown' else row['subject_label'],\n",
    "    axis=1\n",
    ")\n",
    "\n",
    "triples_df['object_label'] = triples_df.apply(\n",
    "    lambda row: get_entity_label(row['object'], object_label_dict) if row['object_label'] == 'unknown' else row['object_label'],\n",
    "    axis=1\n",
    ")\n",
    "# Remove rows with NaN values in 'subject' or 'object'\n",
    "triples_df = triples_df.dropna(subset=['subject', 'object'])\n",
    "# Reorder the columns to have subject_label and object_label next to the triples\n",
    "columns_order = [\"subject\",\"relationship\",\"object\",\"subject_label\",\"object_label\",\"sentence\",\"chunk\"]\n",
    "triples_df = triples_df[columns_order]\n",
    "\n",
    "# Save the updated DataFrame to a new CSV file\n",
    "triples_df.to_csv('output_from_using_REBEL/Final_rebel_triples_labels_large.csv', index=False)\n",
    "\n",
    "print(\"Updated triples with matched labels saved to 'output_from_using_REBEL/Final_rebel_triples_labels.csv'\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "from tqdm import tqdm\n",
    "\n",
    "# Load the data\n",
    "Final_data = pd.read_csv(\"output_from_using_REBEL/Final_rebel_triples_labels_large.csv\")\n",
    "\n",
    "# Convert subject and object to lowercase for consistency\n",
    "Final_data['subject'] = Final_data['subject'].str.lower()\n",
    "Final_data['object'] = Final_data['object'].str.lower()\n",
    "\n",
    "# Add these lines before processing to create the initial subject_chunk and object_chunk columns\n",
    "Final_data['subject_chunk'] = Final_data['chunk']\n",
    "Final_data['object_chunk'] = Final_data['chunk']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import ast\n",
    "import re\n",
    "\n",
    "\n",
    "def clean_text(text):\n",
    "    \"\"\"Remove only single and double quotes from a string, keeping other special characters.\"\"\"\n",
    "    return re.sub(r\"[\\'\\\"]\", '', text)  # Remove only ' and \"\n",
    "\n",
    "def combine_chunks_with_labels(df):\n",
    "    \"\"\"Combine chunks for each entity while keeping labels.\"\"\"\n",
    "    combined_chunks = {}\n",
    "\n",
    "    for _, row in df.iterrows():\n",
    "        subject, object_ = row['subject'], row['object']\n",
    "        subject_label, object_label = row['subject_label'], row['object_label']\n",
    "        chunk = row['chunk']\n",
    "\n",
    "        # Convert chunk from string representation to actual list\n",
    "        try:\n",
    "            chunk = ast.literal_eval(chunk) if isinstance(chunk, str) else chunk\n",
    "        except (SyntaxError, ValueError):\n",
    "            chunk = []  \n",
    "\n",
    "        # Ensure chunk is a list\n",
    "        if not isinstance(chunk, list):\n",
    "            chunk = [chunk]  \n",
    "\n",
    "        # Clean chunks by removing special characters (except ())\n",
    "        cleaned_chunk = [clean_text(item) for item in chunk if isinstance(item, str)]\n",
    "        \n",
    "        # Store subject with its label\n",
    "        if subject not in combined_chunks:\n",
    "            combined_chunks[subject] = {\"label\": subject_label, \"chunks\": set()}\n",
    "        combined_chunks[subject][\"chunks\"].update(cleaned_chunk)\n",
    "\n",
    "        # Store object with its label\n",
    "        if object_ not in combined_chunks:\n",
    "            combined_chunks[object_] = {\"label\": object_label, \"chunks\": set()}\n",
    "        combined_chunks[object_][\"chunks\"].update(cleaned_chunk)\n",
    "\n",
    "    # Convert sets to lists for easier handling\n",
    "    for key in combined_chunks:\n",
    "        combined_chunks[key][\"chunks\"] = list(combined_chunks[key][\"chunks\"])\n",
    "\n",
    "    return combined_chunks\n",
    "\n",
    "# Load DataFrame\n",
    "df = pd.read_csv(\"output_from_using_REBEL/Final_rebel_triples_labels_large.csv\")\n",
    "\n",
    "# Convert subject and object to lowercase for consistency\n",
    "df['subject'] = df['subject'].str.lower()\n",
    "df['object'] = df['object'].str.lower()\n",
    "\n",
    "# Combine chunks while keeping labels\n",
    "combined_chunks = combine_chunks_with_labels(df)\n",
    "\n",
    "# Convert dictionary to DataFrame for saving\n",
    "combined_df = pd.DataFrame([\n",
    "    {\"entity\": key, \"label\": value[\"label\"], \"chunks\": value[\"chunks\"]}\n",
    "    for key, value in combined_chunks.items()\n",
    "])\n",
    "\n",
    "# Save to CSV\n",
    "combined_df.to_csv(\"combined_chunks_with_labels.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Recombination rates between adjacent markers are large, on the order of 0.05 to 0.1.',\n",
       " 'As shown, the results were not affected qualitatively, and only slightly quantitatively.',\n",
       " 'Assuming that ten viral replication cycles occurred during our experiment—based on data on the timing of coat protein detection—the per base and replication cycle recombination rate was on the order of 2 × 10(−5) to 4 × 10(−5).',\n",
       " 'This first determination of a virus recombination rate during a single multi-cellular host infection indicates that recombination is very frequent in the everyday life of this virus.',\n",
       " 'In case this assumption is incorrect, we did calculations assuming five, seven, ten, or 20 replication cycles during these 21 d.',\n",
       " 'The inferred per generation recombination and interference rates, assuming that CaMV undergoes ten replication cycles during the 21 d between infection and sampling, are given for each of the ten plants in Table 2 .']"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combined_df[combined_df['entity'] == 'replication cycle'][\"chunks\"][7477]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>subject</th>\n",
       "      <th>relationship</th>\n",
       "      <th>object</th>\n",
       "      <th>subject_label</th>\n",
       "      <th>object_label</th>\n",
       "      <th>sentence</th>\n",
       "      <th>chunk</th>\n",
       "      <th>subject_chunk</th>\n",
       "      <th>object_chunk</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>rhinovirus</td>\n",
       "      <td>has effect</td>\n",
       "      <td>upper respiratory tract infections</td>\n",
       "      <td>virus</td>\n",
       "      <td>disease</td>\n",
       "      <td>Rhinovirus has effect upper respiratory tract ...</td>\n",
       "      <td>['BACKGROUND: Rhinovirus, the most common caus...</td>\n",
       "      <td>([In this work, we have shown that several hal...</td>\n",
       "      <td>[BACKGROUND: Rhinovirus, the most common cause...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>aerosol transmission</td>\n",
       "      <td>facet of</td>\n",
       "      <td>rhinovirus</td>\n",
       "      <td>transmission</td>\n",
       "      <td>virus</td>\n",
       "      <td>aerosol transmission facet of rhinovirus.</td>\n",
       "      <td>['BACKGROUND: Rhinovirus, the most common caus...</td>\n",
       "      <td>[BACKGROUND: Rhinovirus, the most common cause...</td>\n",
       "      <td>([In this work, we have shown that several hal...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>detecting infectious agents</td>\n",
       "      <td>studies</td>\n",
       "      <td>detecting aerosolized rhinovirus</td>\n",
       "      <td>method</td>\n",
       "      <td>method</td>\n",
       "      <td>detecting infectious agents studies detecting ...</td>\n",
       "      <td>['As a first step in studies of possible airbo...</td>\n",
       "      <td>[As a first step in studies of possible airbor...</td>\n",
       "      <td>[As a first step in studies of possible airbor...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>aerosol chamber</td>\n",
       "      <td>use</td>\n",
       "      <td>aerosol</td>\n",
       "      <td>method</td>\n",
       "      <td>environment</td>\n",
       "      <td>aerosol chamber use aerosol.</td>\n",
       "      <td>['As a first step in studies of possible airbo...</td>\n",
       "      <td>[As a first step in studies of possible airbor...</td>\n",
       "      <td>([Using rhinovirus type 15, they inoculated vo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rhinoviral</td>\n",
       "      <td>subclass of</td>\n",
       "      <td>aerosols</td>\n",
       "      <td>virus</td>\n",
       "      <td>substance</td>\n",
       "      <td>rhinoviral subclass of aerosols.</td>\n",
       "      <td>['To determine the effect of UV irradiation on...</td>\n",
       "      <td>[To determine the effect of UV irradiation on ...</td>\n",
       "      <td>([Using rhinovirus type 15, they inoculated vo...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15861</th>\n",
       "      <td>geodrm</td>\n",
       "      <td>facet of</td>\n",
       "      <td>web-based geospatial technologies</td>\n",
       "      <td>method</td>\n",
       "      <td>method</td>\n",
       "      <td>GeoDRM facet of Web-based geospatial technolog...</td>\n",
       "      <td>['Such developments are of interest to the geo...</td>\n",
       "      <td>[Such developments are of interest to the geos...</td>\n",
       "      <td>[Such developments are of interest to the geos...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15862</th>\n",
       "      <td>ogc geodrm working group</td>\n",
       "      <td>field of work</td>\n",
       "      <td>digital rights management</td>\n",
       "      <td>organization</td>\n",
       "      <td>method</td>\n",
       "      <td>OGC GeoDRM Working Group field of work digital...</td>\n",
       "      <td>['The mission of OGC GeoDRM Working Group is t...</td>\n",
       "      <td>[The mission of OGC GeoDRM Working Group is to...</td>\n",
       "      <td>[The mission of OGC GeoDRM Working Group is to...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15863</th>\n",
       "      <td>google</td>\n",
       "      <td>product or material produced</td>\n",
       "      <td>maps</td>\n",
       "      <td>organization</td>\n",
       "      <td>location</td>\n",
       "      <td>Google product or material produced maps.</td>\n",
       "      <td>['The mission of OGC GeoDRM Working Group is t...</td>\n",
       "      <td>([Google released Google Maps (http://maps.goo...</td>\n",
       "      <td>([The mission of OGC GeoDRM Working Group is t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15864</th>\n",
       "      <td>map</td>\n",
       "      <td>part of</td>\n",
       "      <td>visual language of geography</td>\n",
       "      <td>environment</td>\n",
       "      <td>method</td>\n",
       "      <td>map part of visual language of geography.</td>\n",
       "      <td>['Building on the powerful and universal visua...</td>\n",
       "      <td>([One can display data collected at smaller ge...</td>\n",
       "      <td>[Building on the powerful and universal visual...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15865</th>\n",
       "      <td>google</td>\n",
       "      <td>subsidiary</td>\n",
       "      <td>msn</td>\n",
       "      <td>organization</td>\n",
       "      <td>organization</td>\n",
       "      <td>Google subsidiary MSN.</td>\n",
       "      <td>['Building on the powerful and universal visua...</td>\n",
       "      <td>([Google released Google Maps (http://maps.goo...</td>\n",
       "      <td>([Using the programmable interfaces provided b...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>15866 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           subject                  relationship  \\\n",
       "0                       rhinovirus                    has effect   \n",
       "1             aerosol transmission                      facet of   \n",
       "2      detecting infectious agents                       studies   \n",
       "3                  aerosol chamber                           use   \n",
       "4                       rhinoviral                   subclass of   \n",
       "...                            ...                           ...   \n",
       "15861                       geodrm                      facet of   \n",
       "15862     ogc geodrm working group                 field of work   \n",
       "15863                       google  product or material produced   \n",
       "15864                          map                       part of   \n",
       "15865                       google                    subsidiary   \n",
       "\n",
       "                                   object subject_label  object_label  \\\n",
       "0      upper respiratory tract infections         virus       disease   \n",
       "1                              rhinovirus  transmission         virus   \n",
       "2        detecting aerosolized rhinovirus        method        method   \n",
       "3                                 aerosol        method   environment   \n",
       "4                                aerosols         virus     substance   \n",
       "...                                   ...           ...           ...   \n",
       "15861   web-based geospatial technologies        method        method   \n",
       "15862           digital rights management  organization        method   \n",
       "15863                                maps  organization      location   \n",
       "15864        visual language of geography   environment        method   \n",
       "15865                                 msn  organization  organization   \n",
       "\n",
       "                                                sentence  \\\n",
       "0      Rhinovirus has effect upper respiratory tract ...   \n",
       "1              aerosol transmission facet of rhinovirus.   \n",
       "2      detecting infectious agents studies detecting ...   \n",
       "3                           aerosol chamber use aerosol.   \n",
       "4                       rhinoviral subclass of aerosols.   \n",
       "...                                                  ...   \n",
       "15861  GeoDRM facet of Web-based geospatial technolog...   \n",
       "15862  OGC GeoDRM Working Group field of work digital...   \n",
       "15863          Google product or material produced maps.   \n",
       "15864          map part of visual language of geography.   \n",
       "15865                             Google subsidiary MSN.   \n",
       "\n",
       "                                                   chunk  \\\n",
       "0      ['BACKGROUND: Rhinovirus, the most common caus...   \n",
       "1      ['BACKGROUND: Rhinovirus, the most common caus...   \n",
       "2      ['As a first step in studies of possible airbo...   \n",
       "3      ['As a first step in studies of possible airbo...   \n",
       "4      ['To determine the effect of UV irradiation on...   \n",
       "...                                                  ...   \n",
       "15861  ['Such developments are of interest to the geo...   \n",
       "15862  ['The mission of OGC GeoDRM Working Group is t...   \n",
       "15863  ['The mission of OGC GeoDRM Working Group is t...   \n",
       "15864  ['Building on the powerful and universal visua...   \n",
       "15865  ['Building on the powerful and universal visua...   \n",
       "\n",
       "                                           subject_chunk  \\\n",
       "0      ([In this work, we have shown that several hal...   \n",
       "1      [BACKGROUND: Rhinovirus, the most common cause...   \n",
       "2      [As a first step in studies of possible airbor...   \n",
       "3      [As a first step in studies of possible airbor...   \n",
       "4      [To determine the effect of UV irradiation on ...   \n",
       "...                                                  ...   \n",
       "15861  [Such developments are of interest to the geos...   \n",
       "15862  [The mission of OGC GeoDRM Working Group is to...   \n",
       "15863  ([Google released Google Maps (http://maps.goo...   \n",
       "15864  ([One can display data collected at smaller ge...   \n",
       "15865  ([Google released Google Maps (http://maps.goo...   \n",
       "\n",
       "                                            object_chunk  \n",
       "0      [BACKGROUND: Rhinovirus, the most common cause...  \n",
       "1      ([In this work, we have shown that several hal...  \n",
       "2      [As a first step in studies of possible airbor...  \n",
       "3      ([Using rhinovirus type 15, they inoculated vo...  \n",
       "4      ([Using rhinovirus type 15, they inoculated vo...  \n",
       "...                                                  ...  \n",
       "15861  [Such developments are of interest to the geos...  \n",
       "15862  [The mission of OGC GeoDRM Working Group is to...  \n",
       "15863  ([The mission of OGC GeoDRM Working Group is t...  \n",
       "15864  [Building on the powerful and universal visual...  \n",
       "15865  ([Using the programmable interfaces provided b...  \n",
       "\n",
       "[15866 rows x 9 columns]"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load entity-chunk mapping\n",
    "ents_chunks_df = pd.read_csv(\"combined_chunks.csv\")\n",
    "\n",
    "# Convert ents_chunks to a dictionary for quick lookup\n",
    "ents_chunks_dict = dict(zip(ents_chunks_df['entity'], ents_chunks_df['chunks']))\n",
    "\n",
    "# Load the final data\n",
    "Final_data = pd.read_csv(\"output_from_using_REBEL/Final_rebel_triples_labels_large.csv\")\n",
    "\n",
    "# Convert subject and object to lowercase to ensure consistent matching\n",
    "Final_data['subject'] = Final_data['subject'].str.lower()\n",
    "Final_data['object'] = Final_data['object'].str.lower()\n",
    "\n",
    "# Initialize subject_chunk and object_chunk\n",
    "Final_data['subject_chunk'] = Final_data['subject'].map(ents_chunks_dict)\n",
    "Final_data['object_chunk'] = Final_data['object'].map(ents_chunks_dict)\n",
    "\n",
    "# Fill missing values with empty lists\n",
    "Final_data['subject_chunk'] = Final_data['subject_chunk'].apply(lambda x: eval(x) if isinstance(x, str) else [])\n",
    "Final_data['object_chunk'] = Final_data['object_chunk'].apply(lambda x: eval(x) if isinstance(x, str) else [])\n",
    "\n",
    "Final_data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "Final_data.to_csv(\"updated_final_data.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "updated_final_data = pd.read_csv(\"updated_final_data.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'([\\'In this work, we have shown that several hallmarks of cellular autophagosomes, including the localization of GFP-LC3 into discrete punctate structures and staining with MDC followed by fixation, can be observed in human cells whether they are treated with tamoxifen or infected with poliovirus or rhinovirus.\\', \\'Furthermore, we have employed a new criterion for autophagy, the colocalization of LC3 and LAMP1, to demonstrate that this intracellular rearrangement was observed in cells treated with tamoxifen, infected with picornaviruses (poliovirus, rhinovirus 2, or rhinovirus 14) or transfected with plasmids that express poliovirus proteins 2BC and 3A (see Figures 2 and 8) .\\'], [\\'For cryofixation and EM analysis, H1-HeLa cells were grown in EMEM supplemented with 0.2 M HEPES and 0.1 M MgCl 2 in flasks.\\', \\'Cells were infected with virus at an MOI of 50 PFU/cell for poliovirus or 50 TCID 50 /cell for rhinovirus, then washed three times with PBS, trypsinized, and collected by centrifugation.\\'], [\\'Rhinoviruses have been associated with 40% to 65% of \"common colds\" studied by methods of virus culture [1, 2] .\\', \\'With the use of reverse transcriptase -polymerase chain reaction (RT-PCR), rhinovirus has been shown to be associated with 53% of all colds throughout the year and up to 92% of colds during the autumn outbreak period [3] .\\'], [\\'CONCLUSION: The air sampling and extraction methodology developed in this study should be applicable to the detection of rhinovirus and other airborne viruses in the indoor air of offices and schools.\\', \\'This method, however, cannot distinguish UV inactivated virus from infectious viral particles.\\'], [\\'Only two complete rhinovirus reference genomes are available, one for each group: HRV89 (group A) and HRV14 (group B).\\', \\'Energy profiles of both viruses are included in our energy profile matrix as well as profiles of several enteroviruses and other more distant members of the picornavirus family.\\'], [\\'These low level viruses are either just starting or more likely reaching the end of an infectious episode (latency is less likely) and this raises the previously unaddressed question of their role in viral respiratory pathogenesis.\\', \\'Fifth, the extent of rhinovirus infections was very significant.\\'], [\\'The semi-nested primer pair of OL26 and JWA-1b generates an amplicon of ~292 base pairs from within the original amplicon.\\', \\'The 292 bp amplicon does not differentiate between rhinovirus and enterovirus [34] .\\'], [\\'As shown in Figure 3 , colocalization of GFP-LC3 and LAMP1 was not observed in control cells or in cells that expressed either 2BC or 3A in isolation.\\', \\'However, coexpression of 2BC and 3A caused GFP-LC3 and LAMP1 to colocalize ( Figure 3A) , as was seen in cells infected with poliovirus or rhinoviruses treated with tamoxifen or treated with rapamycin (see Figure 2 ).\\'], [\\'Our data support the hypothesis that poliovirus, and likely the closely related rhinovirus, induce the formation of autophagosome-like structures to serve as the membrane scaffolds for RNA replication.\\', \\'We further suggest that double-membraned vesicles, by forming a luminal cytoplasmic compartment, may facilitate prelytic viral exit from infected cells.\\'], [\\'To test the feasibility of its routine use we needed to clinically validate its performance in a routine setting on specimens tested in parallel with our standard immunofluorescence protocol for the diagnosis of acute virus respiratory infections.\\', \\'Although the routine immunofluoresence panel lacked capacity for the detection of rhinoviruses, human metapneumovirus and CoV-229E, these were included on the strip for clinical reasons during the period of the study.\\'], [\\'In 2 cases a rhinovirus was detected in only one of the specimens.\\', \"As part of a separate rhinovirus validation protocol one of these specimens was subjected to retesting coupled with limited sequencing of the 5\\' non-coding region amplicon which confirmed the presence of a rhinovirus sequence.\"], [\\'Rhinviruses have been implicated in the exacerbation of asthma symptoms [4] and may even play a role in asthma related deaths [5] .\\', \\'Due to their role in the exacerbation of asthma symptoms, rhinoviruses are a major cause of pediatric hospitalizations [6] .\\'], [\\'BACKGROUND: Rhinovirus, the most common cause of upper respiratory tract infections, has been implicated in asthma exacerbations and possibly asthma deaths.\\', \\'Although the method of transmission of rhinoviruses is disputed, several studies have demonstrated that aerosol transmission is a likely method of transmission among adults.\\'], [\\'LAMP1 acquisition is a hallmark of the maturation of these structures, which eventually fuse with lysosomes to produce mature autophagosomes with single membranes and electron-dense contents.\\', \\'We hypothesize that infection by poliovirus or rhinovirus induces accumulation of autophagosomes to promote viral RNA replication by accelerating the formation of autophagosome-like structures from ER membranes, blocking the maturation of these structures into degradative organelles, or both (upper dotted line).\\'], [\\'However, IL-8 production can be induced by measles virus infection of fibroblasts [28] and by influenza A virus, RSV and rhinovirus in pulmonary epithelial cells or AM [28] [29] [30] [31] .\\', \\'The reasons for the lack of neutrophil recruitment in response to elevated IL-8 levels in SARS patients are not known and this deserves further investigation.\\'], [\\'This study was conducted to assess the ability to detect airborne rhinovirus on air sampling filters and documents the first successful attempt to detect airborne rhinovirus using RT-PCR amplification-based techniques.\\', \\'These techniques allow for a highly sensitive method to conduct air sampling for rhinovirus.\\'], [\\'For 5 of the patients the same virus was detected in each of the 2 specimens.\\', \\'The viruses identified were rhinovirus (3), adenovirus (1) and parainfluenza type 3 (1); the latter was also immunofluorescence positive.\\'], [\"We know that it is caused mainly by rhinoviruses and coronaviruses, but that\\'s about it. Having worked on coronaviruses for my doctoral thesis, I knew some of the many open questions.\", \\'A few months later, however, our ignorance concerning these viruses took on an urgent relevance when an exploding viral pandemic in southern China, Hong Kong, Vietnam, Canada, and Singapore left the scientific community stunned.\\'], [\\'Specific markers for autophagosome formation have become available, facilitating the identification of mem-branes derived from the autophagic pathway (reviewed in [43, 46] ).\\', \\'Here, we explore the role of several constituents of autophagosome machinery in poliovirus-and rhinovirusinfected cells by monitoring: the presence of autophagosomal protein LC3 in virally induced vesicles, the acquisition of colocalization of LC3 and LAMP1 in virally infected cells, the viral induction of punctate structures that stain with monodansylcadaverine (MDC), and the effects of perturbing the autophagosomal pathway pharmacologically and via RNA interference on intracellular and extracellular virus yield.\\'], [\\'Fourth, the molecular strip was better at detecting multiple infections.\\', \\'Even allowing for the inability of immunofluorescence to detect rhinoviruses, it should have detected the mixed adenovirus and influenza virus infections.\\'], [\\'Colocalization of LC3 and LAMP1 was observed early in the poliovirus replicative cycle, in cells infected with rhinoviruses 2 and 14, and in cells that express poliovirus proteins 2BC and 3A, known to be sufficient to induce double-membraned vesicles.\\', \\'Stimulation of autophagy increased poliovirus yield, and inhibition of the autophagosomal pathway by 3-methyladenine or by RNA interference against mRNAs that encode two different proteins known to be required for autophagy decreased poliovirus yield.\\'])'"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "updated_final_data[\"subject_chunk\"][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "list"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(updated_final_data[\"subject_chunk\"][0][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "import ast\n",
    "\n",
    "# Convert back to a list\n",
    "updated_final_data[\"subject_chunk\"] = updated_final_data[\"subject_chunk\"].apply(ast.literal_eval)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 58/58 [02:32<00:00,  2.63s/it]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing complete. CSV saved as 'updated_final_data_with_embeddings.csv'.\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import ast  # Safely evaluate tuple and list strings\n",
    "import json  # Store embeddings as JSON\n",
    "import concurrent.futures\n",
    "from langchain_ollama import OllamaEmbeddings\n",
    "from tqdm import tqdm  # Progress bar\n",
    "\n",
    "# Initialize Ollama Embeddings Model\n",
    "embedding_model = OllamaEmbeddings(model=\"nomic-embed-text:latest\")\n",
    "\n",
    "# Load the CSV file\n",
    "df = pd.read_csv(\"updated_final_data.csv\")\n",
    "\n",
    "# Function to process and embed chunks\n",
    "def process_chunks(chunk_data):\n",
    "    if pd.isna(chunk_data):  # Skip NaN values\n",
    "        return json.dumps([])  # Store as JSON empty list\n",
    "\n",
    "    # Convert string representation of tuple/list to Python object\n",
    "    try:\n",
    "        chunk_data = ast.literal_eval(chunk_data)\n",
    "    except (ValueError, SyntaxError):\n",
    "        return json.dumps([])  # Store as JSON empty list\n",
    "\n",
    "    # Ensure chunk_data is a list (flatten tuple if necessary)\n",
    "    if isinstance(chunk_data, tuple):\n",
    "        chunk_list = list(chunk_data)  # Convert tuple to list\n",
    "    elif isinstance(chunk_data, list):\n",
    "        chunk_list = chunk_data  # Use as is\n",
    "    else:\n",
    "        return json.dumps([])  # If not a list/tuple, return empty list\n",
    "\n",
    "    # Flatten lists inside tuples (extract only valid strings)\n",
    "    flat_chunk_list = []\n",
    "    for item in chunk_list:\n",
    "        if isinstance(item, list):  # Nested lists inside tuple or list\n",
    "            for sub_item in item:\n",
    "                if isinstance(sub_item, str):\n",
    "                    flat_chunk_list.append(sub_item)  # Extract string\n",
    "        elif isinstance(item, str):\n",
    "            flat_chunk_list.append(item)  # Directly add strings\n",
    "\n",
    "    return flat_chunk_list  # Return the clean list of text chunks\n",
    "\n",
    "\n",
    "# Apply processing to extract clean chunk lists\n",
    "df[\"subject_chunk\"] = df[\"subject_chunk\"].apply(process_chunks)\n",
    "df[\"object_chunk\"] = df[\"object_chunk\"].apply(process_chunks)\n",
    "\n",
    "# Flatten all text chunks into one list for batch embedding\n",
    "all_texts = list(set(sum(df[\"subject_chunk\"].tolist(), []) + sum(df[\"object_chunk\"].tolist(), [])))\n",
    "\n",
    "# Function to generate embeddings in parallel\n",
    "def embed_texts(text_list):\n",
    "    try:\n",
    "        return {text: embedding_model.embed_query(text) for text in text_list}\n",
    "    except Exception as e:\n",
    "        print(f\"Error embedding batch: {e}\")\n",
    "        return {}\n",
    "\n",
    "# Split text list into chunks for parallel processing\n",
    "batch_size = 256  # Adjust batch size based on GPU memory\n",
    "text_batches = [all_texts[i : i + batch_size] for i in range(0, len(all_texts), batch_size)]\n",
    "\n",
    "# Parallel embedding processing\n",
    "embedding_results = {}\n",
    "with concurrent.futures.ThreadPoolExecutor() as executor:\n",
    "    results = list(tqdm(executor.map(embed_texts, text_batches), total=len(text_batches)))\n",
    "\n",
    "# Merge results into a dictionary\n",
    "for result in results:\n",
    "    embedding_results.update(result)\n",
    "\n",
    "# Function to replace chunk lists with embeddings\n",
    "def replace_with_embeddings(chunk_list):\n",
    "    return json.dumps([{ \"chunk\": text, \"embedding\": embedding_results.get(text, []) } for text in chunk_list])\n",
    "\n",
    "# Replace subject and object chunks with their embeddings\n",
    "df[\"subject_chunk_embeddings\"] = df[\"subject_chunk\"].apply(replace_with_embeddings)\n",
    "df[\"object_chunk_embeddings\"] = df[\"object_chunk\"].apply(replace_with_embeddings)\n",
    "\n",
    "# Save the updated dataframe to CSV\n",
    "df.to_csv(\"updated_final_data_with_embeddings.csv\", index=False)\n",
    "\n",
    "print(\"Processing complete. CSV saved as 'updated_final_data_with_embeddings.csv'.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "df_final_embed = pd.read_csv(\"updated_final_data_with_embeddings.csv\", engine='c')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Drop the specified columns\n",
    "df = df_final_embed.drop(columns=[\"chunk\", \"subject_chunk\", \"object_chunk\"])\n",
    "\n",
    "# Save the updated dataframe to CSV\n",
    "# df.to_csv(\"updated_final_data_with_embeddings.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>subject</th>\n",
       "      <th>relationship</th>\n",
       "      <th>object</th>\n",
       "      <th>subject_label</th>\n",
       "      <th>object_label</th>\n",
       "      <th>sentence</th>\n",
       "      <th>subject_chunk_embeddings</th>\n",
       "      <th>object_chunk_embeddings</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>rhinovirus</td>\n",
       "      <td>has effect</td>\n",
       "      <td>upper respiratory tract infections</td>\n",
       "      <td>virus</td>\n",
       "      <td>disease</td>\n",
       "      <td>Rhinovirus has effect upper respiratory tract ...</td>\n",
       "      <td>[{\"chunk\": \"In this work, we have shown that s...</td>\n",
       "      <td>[{\"chunk\": \"BACKGROUND: Rhinovirus, the most c...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>aerosol transmission</td>\n",
       "      <td>facet of</td>\n",
       "      <td>rhinovirus</td>\n",
       "      <td>transmission</td>\n",
       "      <td>virus</td>\n",
       "      <td>aerosol transmission facet of rhinovirus.</td>\n",
       "      <td>[{\"chunk\": \"BACKGROUND: Rhinovirus, the most c...</td>\n",
       "      <td>[{\"chunk\": \"In this work, we have shown that s...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>detecting infectious agents</td>\n",
       "      <td>studies</td>\n",
       "      <td>detecting aerosolized rhinovirus</td>\n",
       "      <td>method</td>\n",
       "      <td>method</td>\n",
       "      <td>detecting infectious agents studies detecting ...</td>\n",
       "      <td>[{\"chunk\": \"As a first step in studies of poss...</td>\n",
       "      <td>[{\"chunk\": \"As a first step in studies of poss...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>aerosol chamber</td>\n",
       "      <td>use</td>\n",
       "      <td>aerosol</td>\n",
       "      <td>method</td>\n",
       "      <td>environment</td>\n",
       "      <td>aerosol chamber use aerosol.</td>\n",
       "      <td>[{\"chunk\": \"As a first step in studies of poss...</td>\n",
       "      <td>[{\"chunk\": \"Using rhinovirus type 15, they ino...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rhinoviral</td>\n",
       "      <td>subclass of</td>\n",
       "      <td>aerosols</td>\n",
       "      <td>virus</td>\n",
       "      <td>substance</td>\n",
       "      <td>rhinoviral subclass of aerosols.</td>\n",
       "      <td>[{\"chunk\": \"To determine the effect of UV irra...</td>\n",
       "      <td>[{\"chunk\": \"Using rhinovirus type 15, they ino...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15857</th>\n",
       "      <td>geodrm</td>\n",
       "      <td>facet of</td>\n",
       "      <td>web-based geospatial technologies</td>\n",
       "      <td>method</td>\n",
       "      <td>method</td>\n",
       "      <td>GeoDRM facet of Web-based geospatial technolog...</td>\n",
       "      <td>[{\"chunk\": \"Such developments are of interest ...</td>\n",
       "      <td>[{\"chunk\": \"Such developments are of interest ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15858</th>\n",
       "      <td>ogc geodrm working group</td>\n",
       "      <td>field of work</td>\n",
       "      <td>digital rights management</td>\n",
       "      <td>organization</td>\n",
       "      <td>method</td>\n",
       "      <td>OGC GeoDRM Working Group field of work digital...</td>\n",
       "      <td>[{\"chunk\": \"The mission of OGC GeoDRM Working ...</td>\n",
       "      <td>[{\"chunk\": \"The mission of OGC GeoDRM Working ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15859</th>\n",
       "      <td>google</td>\n",
       "      <td>product or material produced</td>\n",
       "      <td>maps</td>\n",
       "      <td>organization</td>\n",
       "      <td>location</td>\n",
       "      <td>Google product or material produced maps.</td>\n",
       "      <td>[{\"chunk\": \"Google released Google Maps http:m...</td>\n",
       "      <td>[{\"chunk\": \"The mission of OGC GeoDRM Working ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15860</th>\n",
       "      <td>map</td>\n",
       "      <td>part of</td>\n",
       "      <td>visual language of geography</td>\n",
       "      <td>environment</td>\n",
       "      <td>method</td>\n",
       "      <td>map part of visual language of geography.</td>\n",
       "      <td>[{\"chunk\": \"One can display data collected at ...</td>\n",
       "      <td>[{\"chunk\": \"Building on the powerful and unive...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15861</th>\n",
       "      <td>google</td>\n",
       "      <td>subsidiary</td>\n",
       "      <td>msn</td>\n",
       "      <td>organization</td>\n",
       "      <td>organization</td>\n",
       "      <td>Google subsidiary MSN.</td>\n",
       "      <td>[{\"chunk\": \"Google released Google Maps http:m...</td>\n",
       "      <td>[{\"chunk\": \"Using the programmable interfaces ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>15862 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           subject                  relationship  \\\n",
       "0                       rhinovirus                    has effect   \n",
       "1             aerosol transmission                      facet of   \n",
       "2      detecting infectious agents                       studies   \n",
       "3                  aerosol chamber                           use   \n",
       "4                       rhinoviral                   subclass of   \n",
       "...                            ...                           ...   \n",
       "15857                       geodrm                      facet of   \n",
       "15858     ogc geodrm working group                 field of work   \n",
       "15859                       google  product or material produced   \n",
       "15860                          map                       part of   \n",
       "15861                       google                    subsidiary   \n",
       "\n",
       "                                   object subject_label  object_label  \\\n",
       "0      upper respiratory tract infections         virus       disease   \n",
       "1                              rhinovirus  transmission         virus   \n",
       "2        detecting aerosolized rhinovirus        method        method   \n",
       "3                                 aerosol        method   environment   \n",
       "4                                aerosols         virus     substance   \n",
       "...                                   ...           ...           ...   \n",
       "15857   web-based geospatial technologies        method        method   \n",
       "15858           digital rights management  organization        method   \n",
       "15859                                maps  organization      location   \n",
       "15860        visual language of geography   environment        method   \n",
       "15861                                 msn  organization  organization   \n",
       "\n",
       "                                                sentence  \\\n",
       "0      Rhinovirus has effect upper respiratory tract ...   \n",
       "1              aerosol transmission facet of rhinovirus.   \n",
       "2      detecting infectious agents studies detecting ...   \n",
       "3                           aerosol chamber use aerosol.   \n",
       "4                       rhinoviral subclass of aerosols.   \n",
       "...                                                  ...   \n",
       "15857  GeoDRM facet of Web-based geospatial technolog...   \n",
       "15858  OGC GeoDRM Working Group field of work digital...   \n",
       "15859          Google product or material produced maps.   \n",
       "15860          map part of visual language of geography.   \n",
       "15861                             Google subsidiary MSN.   \n",
       "\n",
       "                                subject_chunk_embeddings  \\\n",
       "0      [{\"chunk\": \"In this work, we have shown that s...   \n",
       "1      [{\"chunk\": \"BACKGROUND: Rhinovirus, the most c...   \n",
       "2      [{\"chunk\": \"As a first step in studies of poss...   \n",
       "3      [{\"chunk\": \"As a first step in studies of poss...   \n",
       "4      [{\"chunk\": \"To determine the effect of UV irra...   \n",
       "...                                                  ...   \n",
       "15857  [{\"chunk\": \"Such developments are of interest ...   \n",
       "15858  [{\"chunk\": \"The mission of OGC GeoDRM Working ...   \n",
       "15859  [{\"chunk\": \"Google released Google Maps http:m...   \n",
       "15860  [{\"chunk\": \"One can display data collected at ...   \n",
       "15861  [{\"chunk\": \"Google released Google Maps http:m...   \n",
       "\n",
       "                                 object_chunk_embeddings  \n",
       "0      [{\"chunk\": \"BACKGROUND: Rhinovirus, the most c...  \n",
       "1      [{\"chunk\": \"In this work, we have shown that s...  \n",
       "2      [{\"chunk\": \"As a first step in studies of poss...  \n",
       "3      [{\"chunk\": \"Using rhinovirus type 15, they ino...  \n",
       "4      [{\"chunk\": \"Using rhinovirus type 15, they ino...  \n",
       "...                                                  ...  \n",
       "15857  [{\"chunk\": \"Such developments are of interest ...  \n",
       "15858  [{\"chunk\": \"The mission of OGC GeoDRM Working ...  \n",
       "15859  [{\"chunk\": \"The mission of OGC GeoDRM Working ...  \n",
       "15860  [{\"chunk\": \"Building on the powerful and unive...  \n",
       "15861  [{\"chunk\": \"Using the programmable interfaces ...  \n",
       "\n",
       "[15862 rows x 8 columns]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import ast  # Safely evaluate tuple and list strings\n",
    "\n",
    "chunk_data = ast.literal_eval(df[\"subject_chunk_embeddings\"][0])\n",
    "chunk_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "df = pd.read_csv(\"updated_final_data_with_embeddings.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import ast\n",
    "from neo4j import GraphDatabase\n",
    "\n",
    "# Initialize Neo4j driver (adjust URI, user, password as needed)\n",
    "uri = \"bolt://localhost:7687\"\n",
    "user = \"neo4j\"\n",
    "password = \"password\"\n",
    "driver = GraphDatabase.driver(uri, auth=(user, password))\n",
    "\n",
    "def create_nodes_and_relationships(tx, row):\n",
    "    subject = row['subject']\n",
    "    object_ = row['object']\n",
    "    subject_label = row['subject_label']\n",
    "    object_label = row['object_label']\n",
    "    relationship = row['relationship']\n",
    "\n",
    "    # Escape labels/relationship\n",
    "    subject_label_escaped = f\"`{subject_label.replace('`', '')}`\"\n",
    "    object_label_escaped  = f\"`{object_label.replace('`', '')}`\"\n",
    "    relationship_escaped  = f\"`{relationship.replace('`', '')}`\"\n",
    "\n",
    "    query_main = f\"\"\"\n",
    "    MERGE (s:Entity:{subject_label_escaped} {{name: $subjectName}})\n",
    "    MERGE (o:Entity:{object_label_escaped} {{name: $objectName}})\n",
    "    MERGE (s)-[r:{relationship_escaped} {{name: $relationship}}]->(o)\n",
    "    \"\"\"\n",
    "\n",
    "    tx.run(query_main, subjectName=subject, objectName=object_, relationship=relationship)\n",
    "\n",
    "    # Then do the same approach for chunk nodes...\n",
    "    # subject_chunk_list = ast.literal_eval(row['subject_chunk_embeddings'])\n",
    "    # etc...\n",
    "    \n",
    "    # 2) Create Chunk nodes for the subject\n",
    "    #    'subject_chunk_embeddings' is a JSON-like string with a list of dicts.\n",
    "    subject_chunk_list = ast.literal_eval(row['subject_chunk_embeddings'])\n",
    "    for chunk_dict in subject_chunk_list:\n",
    "        chunk_text = chunk_dict.get('chunk', '')\n",
    "        embedding = chunk_dict.get('embedding', [])\n",
    "        \n",
    "        query_sub_chunk = f\"\"\"\n",
    "        MATCH (s:Entity:{subject_label} {{name: $subjectName}})\n",
    "        MERGE (chunkNode:Chunk {{chunk: $chunkText}})\n",
    "        SET chunkNode.embedding = $embedding\n",
    "        MERGE (s)-[:HAS_CHUNK]->(chunkNode)\n",
    "        \"\"\"\n",
    "        tx.run(query_sub_chunk,\n",
    "               subjectName=subject,\n",
    "               chunkText=chunk_text,\n",
    "               embedding=embedding)\n",
    "    \n",
    "    # 3) Create Chunk nodes for the object\n",
    "    object_chunk_list = ast.literal_eval(row['object_chunk_embeddings'])\n",
    "    for chunk_dict in object_chunk_list:\n",
    "        chunk_text = chunk_dict.get('chunk', '')\n",
    "        embedding = chunk_dict.get('embedding', [])\n",
    "        \n",
    "        query_obj_chunk = f\"\"\"\n",
    "        MATCH (o:Entity:{object_label} {{name: $objectName}})\n",
    "        MERGE (chunkNode:Chunk {{chunk: $chunkText}})\n",
    "        SET chunkNode.embedding = $embedding\n",
    "        MERGE (o)-[:HAS_CHUNK]->(chunkNode)\n",
    "        \"\"\"\n",
    "        tx.run(query_obj_chunk,\n",
    "               objectName=object_,\n",
    "               chunkText=chunk_text,\n",
    "               embedding=embedding)\n",
    "\n",
    "# -----------------------\n",
    "# Process the DataFrame\n",
    "# -----------------------\n",
    "with driver.session() as session:\n",
    "    for idx, row in df.iterrows():\n",
    "        session.write_transaction(create_nodes_and_relationships, row)\n",
    "\n",
    "driver.close()\n",
    "print(\"Data import to Neo4j completed.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Count Labels"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['language', 'forensic', 'occupation', 'sample', 'mutation', 'infrastructure', 'vaccination', 'vocabulary_word', 'condition', 'stem', 'drug', 'contact', 'word', 'effect', 'procedure', 'policy', 'term', 'vulnerability', 'question', 'bacteria', 'service', 'anatomy', 'signature', 'section', 'information', 'scenario', 'symptom', 'media', 'cell', 'tool', 'viral barcodes', 'virus transport medium', 'object', 'list', 'person', 'organ', 'public health', 'method', 'data', 'technology', 'vessel', 'organization', 'pathway', 'cell line', 'region', 'country', 'primer', 'function', 'device', 'dataset', 'algorithm', 'city', 'disease', 'definition', 'activity', 'phase', 'growth phase', 'process', 'value', 'number', 'pattern', 'idea', 'database', 'ontology', 'sequence polymorphism', 'file', 'genotype', 'vocabulary', 'state', 'diagnosis', 'product', 'cell type', 'environment', 'ethical issue', 'cell cycle stage', 'relation', 'vaccine', 'variable', 'contact tracing', 'season', 'marker', 'viral cell culture media', 'table', 'stage-specific', 'structure', 'event', 'pathology', 'molecule', 'parameter', 'time point', 'file format', 'spacegroup', 'index', 'character trait', 'model', 'trait', 'organism', 'population', 'gene', 'agreement', 'disease causing agents', 'variant', 'clinical trial', 'chemical', 'cluster', 'year', 'strain', 'people', 'error', 'diagram', 'substance', 'publication', 'figure', 'diagram legend', 'conceptual model', 'format', 'role', 'diagnostic', 'SNP', 'measurement', 'detection', 'signal', 'time', 'time period', 'consensus', 'sequence variant', 'equipment', 'color', 'software', 'system', 'date', 'organism group', 'group', 'concept', 'viral process', 'biology', 'immune response', 'topic', 'sequence', 'workflow', 'treaty', 'classification', 'example', 'emergency response', 'industry', 'enzyme reaction', 'host', 'month', 'treatment', 'protein', 'period', 'virus', 'epidemiology', 'unknown', 'legal', 'province', 'transmission', 'clustering', 'location', 'conceptual framework']\n",
      "160\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the CSV file\n",
    "\n",
    "# Extract unique labels from 'subject_label' and 'object_label' columns\n",
    "subject_labels = Final_data['subject_label'].unique().tolist()\n",
    "object_labels = Final_data['object_label'].unique().tolist()\n",
    "\n",
    "# Combine and deduplicate the labels\n",
    "all_labels = list(set(subject_labels + object_labels))\n",
    "\n",
    "# Print the list of labels\n",
    "print(all_labels)\n",
    "print(len(all_labels))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "KG_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
